{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "RDKit WARNING: [09:45:30] Enabling RDKit 2019.09.3 jupyter extensions\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "import os\n",
    "import re\n",
    "import string\n",
    "from collections import defaultdict\n",
    "from io import StringIO\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import prody\n",
    "import py3Dmol\n",
    "import seaborn as sns\n",
    "from ipywidgets import interact\n",
    "from rdkit import Chem\n",
    "from rdkit.Chem.Draw import MolsToGridImage\n",
    "from scipy.spatial.distance import cdist\n",
    "from scipy.stats import describe\n",
    "from sklearn.cluster import KMeans\n",
    "from sklearn.decomposition import PCA\n",
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Prody Basics\n",
    "We begin by reading in the apo Mpro structure 6YB7 and taking a look at some of the capabilities provided by [Prody](http://prody.csb.pitt.edu/) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 4682 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<AtomGroup: 6YB7 (4682 atoms)>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# read pdb file into a prody AtomGroup\n",
    "apo_file = 'diamond_apo_mpro/6YB7.pdb'\n",
    "apo_struct = prody.parsePDB(apo_file)\n",
    "apo_struct"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No. atoms: 4682\n",
      "No. residues: 306\n",
      "No. chains: 1\n"
     ]
    }
   ],
   "source": [
    "# AtomGroup properties\n",
    "print(f'No. atoms: {apo_struct.numAtoms()}')\n",
    "print(f'No. residues: {apo_struct.numResidues()}')\n",
    "print(f'No. chains: {apo_struct.numChains()}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Names: ['N' 'CA' 'C' ... 'HG2' 'HE22' 'HE21']\n",
      "Resnums: [  1   1   1 ... 306 306 306]\n",
      "Resnames: ['SER' 'SER' 'SER' ... 'GLN' 'GLN' 'GLN']\n",
      "Chains: ['A' 'A' 'A' ... 'A' 'A' 'A']\n"
     ]
    }
   ],
   "source": [
    "# per-atom properties\n",
    "print(f'Names: {apo_struct.getNames()}')\n",
    "print(f'Resnums: {apo_struct.getResnums()}')\n",
    "print(f'Resnames: {apo_struct.getResnames()}')\n",
    "print(f'Chains: {apo_struct.getChids()}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Chain: A from 6YB7 (306 residues, 4682 atoms)>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Hierarchical Indexing\n",
    "apo_struct['A']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Residue: LEU 30 from Chain A from 6YB7 (19 atoms)>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "apo_struct['A', 30]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[ -2.471,   4.23 , -17.183],\n",
       "       [ -1.988,   5.462, -16.536],\n",
       "       [ -2.415,   5.516, -15.063],\n",
       "       ...,\n",
       "       [  4.987, -17.409,  -9.532],\n",
       "       [  7.697, -19.48 ,  -8.545],\n",
       "       [  6.203, -18.674,  -8.11 ]])"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get atomic coordinates\n",
    "apo_struct.getCoords()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Selection: 'backbone' from 6YB7 (1224 atoms)>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# parse out the backbone coordinates\n",
    "bb = apo_struct.select('backbone')\n",
    "bb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[ -2.471,   4.23 , -17.183],\n",
       "       [ -1.988,   5.462, -16.536],\n",
       "       [ -2.415,   5.516, -15.063],\n",
       "       ...,\n",
       "       [  5.582, -14.855,  -9.291],\n",
       "       [  5.941, -15.331,  -7.881],\n",
       "       [  7.071, -15.673,  -7.538]])"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bb.getCoords()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[  9.925, -13.741,  12.009],\n",
       "       [ 10.021, -14.099,  10.601],\n",
       "       [ 10.667, -15.489,  10.525],\n",
       "       [ 11.842, -15.639,  10.844],\n",
       "       [ 10.913, -13.051,   9.896],\n",
       "       [ 10.986, -13.214,   8.363],\n",
       "       [  9.659, -12.827,   7.694],\n",
       "       [ 12.157, -12.42 ,   7.771],\n",
       "       [ 10.796, -13.567,  12.496],\n",
       "       [  9.034, -14.119,  10.139],\n",
       "       [ 11.92 , -13.086,  10.311],\n",
       "       [ 10.546, -12.049,  10.126],\n",
       "       [ 11.174, -14.258,   8.125],\n",
       "       [  9.721, -12.902,   6.609],\n",
       "       [  8.843, -13.474,   8.012],\n",
       "       [  9.37 , -11.809,   7.949],\n",
       "       [ 12.174, -12.509,   6.687],\n",
       "       [ 12.087, -11.359,   7.986],\n",
       "       [ 13.113, -12.779,   8.152]])"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# parse out a specific residue and get it's coordinates\n",
    "apo_struct['A', 30].getCoords()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "___\n",
    "### How Rigid/Flexible is Mpro?\n",
    "\n",
    "Let's evaluate residue mobility accross the various apo and holo structures in the Diamond/XChem dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 22 PDBs were parsed in 0.80s.                                \n",
      "@> 44 PDBs were parsed in 1.67s.                            \n"
     ]
    }
   ],
   "source": [
    "# locate files\n",
    "non_cov_files = glob.glob('diamond_non_cov_mpro/*.pdb')\n",
    "cov_files = glob.glob('diamond_cov_mpro/*.pdb')\n",
    "\n",
    "# read files into Prody AtomGroups\n",
    "non_cov_structs = prody.parsePDB(non_cov_files)\n",
    "cov_structs = prody.parsePDB(cov_files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<AtomGroup: Mpro-x0995_1 (4658 atoms)>,\n",
       " <AtomGroup: Mpro-x0540_1 (4684 atoms)>,\n",
       " <AtomGroup: Mpro-x0354_1 (4684 atoms)>,\n",
       " <AtomGroup: Mpro-x1249_1 (4678 atoms)>,\n",
       " <AtomGroup: Mpro-x0678_1 (4679 atoms)>,\n",
       " <AtomGroup: Mpro-x0195_1 (4674 atoms)>,\n",
       " <AtomGroup: Mpro-x0991_1 (4659 atoms)>,\n",
       " <AtomGroup: Mpro-x1077_1 (4673 atoms)>,\n",
       " <AtomGroup: Mpro-x1093_1 (4683 atoms)>,\n",
       " <AtomGroup: Mpro-x0395_1 (4673 atoms)>,\n",
       " <AtomGroup: Mpro-x0397_1 (4675 atoms)>,\n",
       " <AtomGroup: Mpro-x0426_1 (4676 atoms)>,\n",
       " <AtomGroup: Mpro-x0874_1 (4671 atoms)>,\n",
       " <AtomGroup: Mpro-x0072_1 (4671 atoms)>,\n",
       " <AtomGroup: Mpro-x0104_1 (4674 atoms)>,\n",
       " <AtomGroup: Mpro-x0305_1 (4665 atoms)>,\n",
       " <AtomGroup: Mpro-x0967_1 (4681 atoms)>,\n",
       " <AtomGroup: Mpro-x0161_1 (4669 atoms)>,\n",
       " <AtomGroup: Mpro-x0107_1 (4666 atoms)>,\n",
       " <AtomGroup: Mpro-x0387_1 (4674 atoms)>,\n",
       " <AtomGroup: Mpro-x0946_1 (4662 atoms)>,\n",
       " <AtomGroup: Mpro-x0434_1 (4673 atoms)>]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "non_cov_structs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "# strip off hydrogens, we don't need them\n",
    "apo_struct = apo_struct.select('heavy').copy()\n",
    "non_cov_structs = [ s.select('heavy').copy() for s in non_cov_structs ]\n",
    "cov_structs = [ s.select('heavy').copy() for s in cov_structs ]\n",
    "\n",
    "# extract protein\n",
    "apo_struct = apo_struct.select('protein').copy()\n",
    "non_cov_prot_structs = [ s.select('protein').copy() for s in non_cov_structs ]\n",
    "cov_prot_structs = [ s.select('protein').copy() for s in cov_structs ]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Using Prody Enesembles to Identify Flexible Regions\n",
    "The ProDy PDBEnsemble class makes it easy to do certain types of comparisons/calculations on an ensemble of homologous or sequence-identical (like frames from an MD) protein structures.  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Starting iterative superposition:                                                       \n",
      "@> Step #1: RMSD difference = 1.1849e-01\n",
      "@> Step #2: RMSD difference = 3.8693e-07\n",
      "@> Iterative superposition completed in 0.01s.\n",
      "@> Final superposition to calculate transformations.\n",
      "@> Superposition completed in 0.00 seconds.\n",
      "@> Ensemble (22 conformations) were built in 0.86s.\n",
      "@> Starting iterative superposition:                                                       \n",
      "@> Step #1: RMSD difference = 1.6537e-01\n",
      "@> Step #2: RMSD difference = 3.8849e-07\n",
      "@> Iterative superposition completed in 0.01s.\n",
      "@> Final superposition to calculate transformations.\n",
      "@> Superposition completed in 0.01 seconds.\n",
      "@> Ensemble (44 conformations) were built in 1.70s.\n",
      "@> Starting iterative superposition:\n",
      "@> Step #1: RMSD difference = 9.2634e-12\n",
      "@> Iterative superposition completed in 0.01s.\n",
      "@> Final superposition to calculate transformations.\n",
      "@> Superposition completed in 0.01 seconds.\n",
      "@> Starting iterative superposition:\n",
      "@> Step #1: RMSD difference = 8.9071e-12\n",
      "@> Iterative superposition completed in 0.01s.\n",
      "@> Final superposition to calculate transformations.\n",
      "@> Superposition completed in 0.01 seconds.\n"
     ]
    }
   ],
   "source": [
    "# build PDBEnsembles \n",
    "non_cov_ensemble = prody.buildPDBEnsemble(non_cov_prot_structs)\n",
    "cov_ensemble = prody.buildPDBEnsemble(cov_prot_structs)\n",
    "\n",
    "# align ensembles\n",
    "non_cov_ensemble.iterpose()\n",
    "cov_ensemble.iterpose()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>rmsf_non_cov</th>\n",
       "      <th>rmsf_cov</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>resnum</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.172231</td>\n",
       "      <td>0.136248</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.143564</td>\n",
       "      <td>0.125013</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.107904</td>\n",
       "      <td>0.112619</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.110428</td>\n",
       "      <td>0.090036</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0.097995</td>\n",
       "      <td>0.082591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>0.111305</td>\n",
       "      <td>0.131022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>0.128207</td>\n",
       "      <td>0.153436</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>0.149995</td>\n",
       "      <td>0.177635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>0.129545</td>\n",
       "      <td>0.173439</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>0.150014</td>\n",
       "      <td>0.180164</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>304 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        rmsf_non_cov  rmsf_cov\n",
       "resnum                        \n",
       "1           0.172231  0.136248\n",
       "2           0.143564  0.125013\n",
       "3           0.107904  0.112619\n",
       "4           0.110428  0.090036\n",
       "5           0.097995  0.082591\n",
       "...              ...       ...\n",
       "300         0.111305  0.131022\n",
       "301         0.128207  0.153436\n",
       "302         0.149995  0.177635\n",
       "303         0.129545  0.173439\n",
       "304         0.150014  0.180164\n",
       "\n",
       "[304 rows x 2 columns]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Prody calculates RMSFs per atom, let's average accross residues\n",
    "non_cov_rmsf = pd.DataFrame({\n",
    "    'rmsf': non_cov_ensemble.getRMSFs(),\n",
    "    'resnum': non_cov_ensemble.getAtoms().getResnums(),\n",
    "    'resname': non_cov_ensemble.getAtoms().getResnames()\n",
    "}).groupby('resnum').agg(np.mean)\n",
    "\n",
    "cov_rmsf = pd.DataFrame({\n",
    "    'rmsf': cov_ensemble.getRMSFs(),\n",
    "    'resnum': cov_ensemble.getAtoms().getResnums(),\n",
    "    'resname': cov_ensemble.getAtoms().getResnames()\n",
    "}).groupby('resnum').agg(np.mean)\n",
    "\n",
    "rmsf_df = non_cov_rmsf.merge(\n",
    "    cov_rmsf, \n",
    "    left_index=True, \n",
    "    right_index=True,\n",
    "    suffixes=('_non_cov', '_cov')\n",
    ")\n",
    "rmsf_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Plot RMSF to compare the covalent and non-covalent sets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA7AAAAFzCAYAAAAHXuXxAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjMsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+AADFEAAAgAElEQVR4nOzdeZhcdZn3/8+39u50Zw9r2AeEQBZCwiqrCFEZ9nUcFkfA0Z/Ac/mMMzjwqOjMb3wcxwUHFxhEYVB2EQWEQYjsyi4k7BAhAUnSSTq91HKW7/PHOae6utPdqXRSp7qq3q/r6qvWrr4TY1N33cvXWGsFAAAAAMB4l6h3AAAAAAAAVIMEFgAAAADQEEhgAQAAAAANgQQWAAAAANAQSGABAAAAAA2BBBYAAAAA0BBS9Q5gU02fPt3uvPPO9Q4DAAAAAFADzzzzzGpr7YzhHmu4BHbnnXfW008/Xe8wAAAAAAA1YIz580iP0UIMAAAAAGgIJLAAAAAAgIZAAgsAAAAAaAgNNwMLAAAAYHxyHEfLly9XoVCodyhoALlcTjNnzlQ6na76e0hgAQAAAGwRy5cvV2dnp3beeWcZY+odDsYxa626urq0fPly7bLLLlV/Hy3EAAAAALaIQqGgadOmkbxio4wxmjZt2iZX60lgAQAAAGwxJK+o1lj+rZDAAgAAAEAdLVu2TPvss8+Yv/+73/2u+vv7h33siCOO0NNPPy1J+vjHP65169aN+edU47zzztNtt91Ws9cngQUAAACABjZaAlvpnnvu0eTJk2OIqHZIYAEAAAA0hWXLlmmvvfbSBRdcoL333lvHHHOM8vm8JOn555/XgQceqDlz5uikk07S2rVrJQUVyn/6p3/S/vvvrz322EOPPPLIsK/9xhtv6Oijj9bcuXM1f/58vfnmm7LW6otf/KL22WcfzZ49WzfffLMk6cwzz9Tdd99d/t6oKrls2TIdeuihmj9/vubPn6/HH398g5/jeZ6++MUvauHChZozZ45+/OMfS5IWL16sI444Qqeeeqr23HNPffKTn5S1VldeeaXee+89HXnkkTryyCNH/fvZeeedtXr1aknS17/+dX3oQx/Shz/8YZ111ln61re+JUm65pprtHDhQs2dO1ennHJKOTE+77zzdPHFF+vggw/WrrvuWq6yWmv1+c9/Xh/60Id09NFHa+XKleWfd+mll2rWrFmaM2eO/uEf/mEj/+tVhy3EAAAAALa4K369REvfW79FX3PWdhP1lb/ee9TnvP766/rFL36ha665Rqeffrpuv/12/e3f/q3OOeccff/739fhhx+uL3/5y7riiiv03e9+V5Lkuq7++Mc/6p577tEVV1yhBx54YIPX/eQnP6lLL71UJ510kgqFgnzf1x133KHnn39eL7zwglavXq2FCxfqsMMO0xlnnKFbbrlFn/jEJ1QqlfS73/1OP/zhD2Wt1f/8z/8ol8vp9ddf11lnnVVu741ce+21mjRpkp566ikVi0UdcsghOuaYYyRJzz33nJYsWaLttttOhxxyiB577DFdfPHF+va3v62HHnpI06dPr+rv8amnntLtt9+uF154QY7jaP78+dpvv/0kSSeffLIuuOACSdLll1+ua6+9VhdddJEk6f3339ejjz6qV155Rccff7xOPfVU/fKXv9Srr76qpUuX6oMPPtCsWbP0d3/3d+rq6tIvf/lLvfLKKzLGbLHWZSqwwCboK7pavnbj7RkAAACoj1122UXz5s2TJO23335atmyZuru7tW7dOh1++OGSpHPPPVcPP/xw+XtOPvnkQc8fqqenRytWrNBJJ50kKTi/tL29XY8++qjOOussJZNJbb311jr88MP11FNP6WMf+5geeughFYtF3XvvvTrssMPU1tYmx3F0wQUXaPbs2TrttNO0dOnSDX7W/fffr+uvv17z5s3TAQccoK6uLr3++uuSpP33318zZ85UIpHQvHnzho21Go899phOOOEE5XI5dXZ26q//+q/Lj7300ks69NBDNXv2bN14441asmRJ+bETTzxRiURCs2bN0gcffCBJevjhh8t/B9ttt52OOuooSdKkSZOUy+X06U9/WnfccYfa29vHFOtQVGCBTfDjh9/SL/74jp667Oh6hwIAADCubaxSWivZbLZ8PZlMlluIq/meZDIp13UlSZ/61Kf03HPPabvttiu3Blcrl8vpiCOO0H333aebb75ZZ555piTpO9/5jrbeemu98MIL8n1fuVxug++11ur73/++jj322EH3L168eIM/WxTrlnTeeefpzjvv1Ny5c/XTn/5UixcvLj9W+fOttaO+TiqV0h//+Ef97ne/02233ab//M//1IMPPrjZ8VGBBTbB2r6SVvUUVXC8eocCAACAKk2aNElTpkwpz7fecMMN5WrsSK677jo9//zzuueee9TZ2amZM2fqzjvvlCQVi0X19/fr0EMP1c033yzP87Rq1So9/PDD2n///SVJZ5xxhq677jo98sgjWrRokSSpu7tb2267rRKJhG644QZ53obvKY899lj98Ic/lOM4kqTXXntNfX19o8ba2dmpnp6eqv8+DjnkEP36179WoVBQb2+vfvOb35Qf6+np0bbbbivHcXTjjTdu9LUOO+yw8t/B+++/r4ceekiS1Nvbq+7ubn384x/Xd77zHb3wwgtVxzcaKrDAJnA8X5K0rt/RNpOSdY4GAAAA1frZz36mv//7v1d/f7923XVXXXfddZv0/TfccIM+85nP6Mtf/rLS6bRuvfVWnXTSSXriiSc0d+5cGWP0zW9+U9tss40k6ZhjjtHZZ5+tE044QZlMRpL0uc99Tqeccoquv/56LVq0SBMmTNjg55x//vlatmyZ5s+fL2utZsyYUU6cR3LhhRdq0aJF2m677coJ5GgWLlyo448/XnPmzNHWW2+t2bNna9KkSZKC5U4HHHCAZsyYoQMOOGCjifFJJ52kBx98ULNmzdKOO+6ogw46SFKQCJ9wwgkqFAqy1urb3/72RuOqhtlY6Xe8WbBggR066AzE5Qu3PK87nl2hey85VHttO7He4QAAAIwrL7/8svbaa696h4Eq9Pb2qqOjQ/39/TrssMN09dVXa/78+bHHMdy/GWPMM9baBcM9nwossAlcL/jAZ21fqc6RAAAAAGN34YUXaunSpSoUCjr33HPrkryOBQkssAmiFuK1/U6dIwEAAADG7uc//3m9QxgTljgBmyBKYNf0U4EFAAAA4kYCC2yCEi3EAAAAQN2QwAKbwHGjFmISWAAAACBuJLDAJijPwFKBBQAAAGJHAgtsApY4AQAAtK6vfvWr+ta3vjWm7122bNmIi5OWLVumffbZR5L09NNP6+KLLx5zjNXq6Oio+c+oBRJYYBOUZ2BpIQYAAMAmGC2BrbRgwQJdeeWVMUTUmEhggU0wUIElgQUAABiPrr/+es2ZM0dz587V2WefLSlIHo866ijNmTNHH/nIR/TOO++ou7tbO+20k3w/eH/X19enHXbYQY7j6JprrtHChQs1d+5cnXLKKerv79/g57z55ptatGiR9ttvPx166KF65ZVXJEnnnXeeLr74Yh188MHaddddddttt0mSLr30Uj3yyCOaN2+evvOd74wY/+LFi3XcccdJklatWqWPfvSj2nvvvXX++edrp5120urVqyVJJ554ovbbbz/tvffeuvrqq8vf39HRocsuu0xz587VgQceqA8++ECS9Pbbb+uggw7S7Nmzdfnll5ef//777+uwww7TvHnztM8+++iRRx4Z8999HDgHFtgEAzOwtBADAACM6t5Lpb+8uGVfc5vZ0se+MeLDS5Ys0b/8y7/o8ccf1/Tp07VmzRpJ0kUXXaRzzz1X5557rn7yk5/o4osv1p133ql58+bp97//vY488kj95je/0bHHHqt0Oq2TTz5ZF1xwgSTp8ssv17XXXquLLrpo0M+68MIL9aMf/Ui77767/vCHP+hzn/ucHnzwQUlBUvjoo4/qlVde0fHHH69TTz1V3/jGN/Stb31Lv/nNb6r+415xxRU66qij9KUvfUm//e1vde2115Yf+8lPfqKpU6cqn89r4cKFOuWUUzRt2jT19fXpwAMP1L/+67/qH//xH3XNNdfo8ssv1yWXXKLPfvazOuecc3TVVVeVX+fnP/+5jj32WF122WXyPG/YZH08oQILbAI3bCHuLboqhRuJAQAAMD48+OCDOu200zR9+nRJ0tSpUyVJTzzxhP7mb/5GknT22Wfr0UcflSSdccYZuvnmmyVJN910k8444wxJ0ksvvaRDDz1Us2fP1o033qglS5YM+jm9vb16/PHHddppp2nevHn6zGc+o/fff7/8+IknnqhEIqFZs2aVK6Bj8eijj+rMM8+UJC1atEhTpkwpP3bllVeWq6zvvvuuXn/9dUlSJpMpV3D3228/LVu2TJL02GOP6ayzzir/HUQWLlyo6667Tl/96lf14osvqrOzc8zxxoEKLLAJSp4vYyRrpXX9JW01MVfvkAAAAManUSql48Xxxx+vf/7nf9aaNWv0zDPP6KijjpIUtAHfeeedmjt3rn76059q8eLFg77P931NnjxZzz///LCvm81my9ettVs87sWLF+uBBx7QE088ofb2dh1xxBEqFAqSpHQ6LWOMJCmZTMp13fL3RfdXOuyww/Twww/r7rvv1nnnnacvfOELOuecc7Z4zFsKFVhgEzier2kTgl9Ia5iDBQAAGFeOOuoo3Xrrrerq6pKkcgvxwQcfrJtuukmSdOONN+rQQw+VFMyLLly4UJdccomOO+44JZNJSVJPT4+23XZbOY6jG2+8cYOfM3HiRO2yyy669dZbJQVJ6gsvvDBqbJ2dnerp6dmkP88hhxyiW265RZJ0//33a+3atZKk7u5uTZkyRe3t7XrllVf05JNPVvValX8HkT//+c/aeuutdcEFF+j888/Xs88+u0kxxo0EFtgEjutrq84ggWUOFgAAYHzZe++9ddlll+nwww/X3Llz9YUvfEGS9P3vf1/XXXed5syZoxtuuEHf+973yt9zxhln6L//+7/L7cOS9PWvf10HHHCADjnkEO25557D/qwbb7xR1157rebOnau9995bv/rVr0aNbc6cOUomk5o7d+6oS5wqfeUrX9H999+vffbZR7feequ22WYbdXZ2atGiRXJdV3vttZcuvfRSHXjggRt9re9973u66qqrNHv2bK1YsaJ8/+LFizV37lztu+++uvnmm3XJJZdUFVu9mFqUtGtpwYIF9umnn653GGhRe1x2rw75q2l66NVV+sEn5+vjs7etd0gAAADjxssvv6y99tqr3mE0jWKxqGQyqVQqpSeeeEKf/exnR2xbblTD/ZsxxjxjrV0w3POZgQWqZK1VyfO1dTj3ylE6AAAAqKV33nlHp59+unzfVyaT0TXXXFPvkOqOBBaokusH3QoDLcQksAAAAKid3XffXc8991y9wxhXajoDa4xZZIx51RjzhjHm0mEe39EY85Ax5jljzJ+MMR+vZTzA5ojOgJ2QTWlCJqk1zMACAAAAsapZAmuMSUq6StLHJM2SdJYxZtaQp10u6RZr7b6SzpT0g1rFA2wuxw0qsKlkQpPbM1pHCzEAAMAGGm3HDupnLP9WalmB3V/SG9bat6y1JUk3STphyHOspInh9UmS3qthPMBmKYUV2EzSaOqEDMfoAAAADJHL5dTV1UUSi42y1qqrq0u5XG6Tvq+WM7DbS3q34vZySQcMec5XJd1vjLlI0gRJRw/3QsaYCyVdKEk77rjjFg8UqEbUQpxOJjS5Pa21/bQQAwAAVJo5c6aWL1+uVatW1TsUNIBcLqeZM2du0vfUe4nTWZJ+aq39D2PMQZJuMMbsY631K59krb1a0tVScIxOHeIE5HrBP710MqGpEzL6c1d/nSMCAAAYX9LptHbZZZd6h4EmVssW4hWSdqi4PTO8r9KnJd0iSdbaJyTlJE2vYUzAmEUtxOlUQlPaM2whBgAAAGJWywT2KUm7G2N2McZkFCxpumvIc96R9BFJMsbspSCBpd8A45JTMQM7pT2jnqJbvg8AAABA7dUsgbXWupI+L+k+SS8r2Da8xBjzNWPM8eHT/rekC4wxL0j6haTzLBPfGKcqZ2CnTkhLktayyAkAAACITU1nYK2190i6Z8h9X664vlTSIbWMAdhSBi9xykiS1vU72qpz0zanAQAAABibWrYQA02l5A5e4iRJa5iDBQAAAGJT7y3EQMMoz8CmjHLp4P8662ghBgAAAGJDBRaoUpTAphKVFVjOggUAAADiQgILVKlyBnZKOAPLEicAAAAgPiSwQJVKXjADG7QQJ9WWTnIWLAAAABAjEligSm5FBVaSJrentS5PCzEAAAAQFxJYoErOkAQ2l06q6Pr1DAkAAABoKSSwQJWiFuIogU0njRwSWAAAACA2JLBAlaJkNRMmsJlUQiWPBBYAAACICwksUKVyC3HKBJfJRPk+AAAAALVHAgtUaegMbCaZYAYWAAAAiBEJLFClaAY2lQgqsJkUFVgAAAAgTiSwQJUcz1c6aWRMmMAmEypRgQUAAABiQwILVMlx/XL7sMQMLAAAABA3EligSkEFduD/MpkUFVgAAAAgTiSwQJUc32pW4h3pmZ9Kiiqwtr5BAQAAAC2EBBaokuP6OkkPSr/9kqSgAssWYgAAACA+JLBAlRzPV9Z4kpOXrFUmaZiBBQAAAGJEAgtUyfGsssaRZCWvxAwsAAAAEDMSWKBKJc9XxnjBDbfAFmIAAAAgZiSwQJUcz1dGbnijoEwqIde38n0WOQEAAABxIIEFqhTMwIYJrJsvH6lTogoLAAAAxIIEFqiS41qlowqsW1Q2RQILAAAAxIkEFqhSyfMHElhnoALrsMgJAAAAiAUJLFClYAbWCW64RWWowAIAAACxIoEFquRUVmDdygosS5wAAACAOJDAAlVyPavUkC3EklTyvDpGBQAAALQOEligSiXPV8pGFdiCMkkT3E8FFgAAAIgFCSxQpaCFOJqBLTADCwAAAMSMBBaokuPZgQps5RZiElgAAAAgFiSwQJUc11fSVmwhDhPYEsfoAAAAALEggQWqFMzARglsXukUCSwAAAAQJxJYoEqOV1GBdQoDFVhaiAEAAIBYkMACVfB8K99KSX9gC3GWCiwAAAAQKxJYoAqO5yshXwmFZ766BZY4AQAAADEjgQWqUPJ8peUO3FF5jA4VWAAAACAWJLBAFVzPKlOZwDpUYAEAAIC4kcACVXA2qMDmyxXYIhVYAAAAIBYksEAVSq6vjJyBOyrOgXU8W6eoAAAAgNZCAgtUwfF8pU1lC3GeGVgAAAAgZiSwQBWcoTOwbkHJhFHCMAMLAAAAxIUEFqiC4/kbJLCSlEklVCKBBQAAAGJBAgtUYdAxOqk2yQkS2HQyQQsxAAAAEBMSWKAKjluRwOYmSm5ekpSlAgsAAADEhgQWqILjWWWiJU7ZTsktSgoqsA4VWAAAACAWJLBAFQbNwGYnlluImYEFAAAA4kMCC1TB8Ya2EA/MwLKFGAAAAIgHCSxQBcezAwlsdqLkFSXfV4YlTgAAAEBsSGCBKgQtxE5wIzcxuHQLSqcSKnm2foEBAAAALYQEFqhCyfMrljhNCi7dgrLJhEquV7/AAAAAgBZCAgtUIZiBDRPVQRVYI4cKLAAAABALEligCo5b0UKcDRNYJ88MLAAAABAjEligCoOWOJUrsEW2EAMAAAAxIoEFqlCqPEYnqsC6+eAcWCqwAAAAQCxIYIEqOJ6vbHmJU2d4ZyFoIaYCCwAtx/OtvvnbV9TVW6x3KADQUkhggSo40RbiRFpKtwV3ugUqsADQopZ19ekHi9/Uw6+vqncoANBSSGCBKjieVc54UiorpXLBnW6BGVgAaFFuuIGeTfQAEC8SWKAKjucrl3ClZHpQAksFFgBaU/ThpeeTwAJAnEhggSoEM7CelMxI6TCBdaIKLG9eAKDVRImrSwILALEigQWq4Lg2TGCzUiqagQ23EHu+rOUNDAC0EtcPKrAuYyQAECsSWKAK5SVOyXQwBytJblGZpAkfJ4EFgFYS/d6nhRgA4kUCC1ShFB2jk8wMbCF2ggps9DgAoHXQQgwA9UECC1TB8Xxl5EqpTJDEypS3EEtikRMAtJhoiRMtxAAQr5omsMaYRcaYV40xbxhjLh3hOacbY5YaY5YYY35ey3iAsXI8q0y0xMmYYBNxuIU4eJw3MADQSqjAAkB9pGr1wsaYpKSrJH1U0nJJTxlj7rLWLq14zu6SviTpEGvtWmPMVrWKB9gcQQXWkZJh+3A6JzkFZajAAkBLYgYWAOqjlhXY/SW9Ya19y1pbknSTpBOGPOcCSVdZa9dKkrV2ZQ3jAcas5PpKK1ziJAWbiF1mYAGgVVGBBYD6qGUCu72kdytuLw/vq7SHpD2MMY8ZY540xiwa7oWMMRcaY542xjy9atWqGoULjMzxogQ23ECcyoZbiKnAAkAr4hgdAKiPei9xSknaXdIRks6SdI0xZvLQJ1lrr7bWLrDWLpgxY0bMIQLBJ+xBC3FYgU23SU6+vMSJGVgAaC1RCzEVWACIVy0T2BWSdqi4PTO8r9JySXdZax1r7duSXlOQ0ALjSsn1lVJ4jI40UIFNUYEFgFbkhRVYZmABIF61TGCfkrS7MWYXY0xG0pmS7hrynDsVVF9ljJmuoKX4rRrGBIyJ4/lKWTdIXKVwBrbiGB0qsADQUqIKbHQJAIhHzRJYa60r6fOS7pP0sqRbrLVLjDFfM8YcHz7tPkldxpilkh6S9EVrbVetYgLGyvFsWIGNWohzkpOnAgsALSqqvEaVWABAPGp2jI4kWWvvkXTPkPu+XHHdSvpC+AWMW47nK22dihbinOSuKi9x4hN4AGgt0e4DZmABIF71XuIENATH85W0TsUW4tzgY3SowAJASxmowJLAAkCcSGCBKpTccAZ20BbigtJJI4ktxADQaqLKq0sHDgDEaqMtxMaYhKS5kraTlJf0krV2Za0DA8YTx/OVSpaGbCEuUIEFgBY10ELM738AiNOICawxZjdJ/yTpaEmvS1olKSdpD2NMv6QfS/qZtZbf3Gh61nOkpCoS2GALcYYtxADQkmghBoD6GK0C+y+SfijpM+GypTJjzFaS/kbS2ZJ+VrvwgPqz1sr4TnAjVVGBZQsxALQsjtEBgPoYMYG11p41ymMrJX23JhEB44zjWWXkBjeiCmy6TbKe0sYPn0MCCwCtJDo+hwosAMRro0ucjDGnGWM6w+v/xxhzhzFmfu1DA8YHx/MrEthwiVMqJ0nK2KIkKrAA0GqiyiszsAAQr2q2EP8fa22PMebDkj4i6VoFrcVAS3A8X+lyAltxjI6klF8qPwcA0Do8thADQF1Uk8B64eUnJF1trb1bUqZ2IQHjS8nzlTZDW4iDBNaEm4iLJLAA0FKiyqtLCzEAxKqaBHaFMebHks6QdI8xJlvl9wFNYfAMbNRC3BZchpuIHZc3MADQSqIWYmZgASBe1SSip0u6T9Kx1tp1kqZK+mJNowLGEcetmIFNZQdfhhXYkucN/80AgKZUbiEmgQWAWI12jI4kyVrbb4x5SNIOFcubVtc2LGD8GDwDW7GFWJKcgtJJQwUWAFpMtPvAZYQEAGK10QTWGPN1SedJelNS9C7dSjqqdmEB40fJ85VReA7skC3EcvNhBZY3MADQSqIKLC3EABCvjSawClqId7PWlmodDDAeOZ7dcIlTOYEtKp1Mk8ACQIsZOEaHBBYA4lTNDOxLkibXOhBgvBr2GJ1wC7GcvDLJBOfAAkCL8XxaiAGgHqqpwP6bpOeMMS9JKkZ3WmuPr1lUwDjieL6yG2whjiqwwRInzoEFgNbissQJAOqimgT2Z5L+r6QXJfEuHS3H8eyGS5wqE1gqsADQcqIPLpmBBYB4VZPA9ltrr6x5JMA45bgVLcSp4bYQU4EFgFbDMToAUB/VJLCPGGP+TdJdGtxC/GzNogLGEdf3h1niNPgc2P5+tz7BAQDqorzEiQ8wASBW1SSw+4aXB1bcxzE6aBklzyqzQQtxWIF1gwpsyeMTeABoJVRgAaA+NprAWmuPjCMQYLxyXH/DBDaZkhIpyckrm0qo5Hr1CxAAEDtmYAGgPjZ6jI4xZpIx5tvGmKfDr/8wxkyKIzhgPHB9f8MlTlKwyMktKp00nAMLAC2msgJrLUksAMSlmnNgfyKpR9Lp4dd6SdfVMihgPAlaiJ3gxgYJbD44RsflzQsAtJLK1mGqsAAQn2pmYHez1p5ScfsKY8zztQoIGG8cN1jiZGVkEsmBB9Jt5S3EVGABoLVUbp93fatUcpQnAwC2mGoqsHljzIejG8aYQyTlaxcSML64vq+MvGDzsDEDD6Sy5S3EDufAAkBLqay6ssgJAOJTTQX27yVdXzH3ulbSubULCRhfHM+qTY6UTA9+INVWTmCLVGABoKU4FdvnPTbRA0BsRk1gjTEJSR+y1s41xkyUJGvt+lgiA8YJx/M1Ue7g+VcpqMA6eWWSCTmeL2utTGWFFgDQtDy/soWYDzEBIC6jthBba31J/xheX0/yilbkeL6yxpNJZgc/kG6T3KIyyYSspYUMAFqJ61llU8HbKJY4AUB8qpmBfcAY8w/GmB2MMVOjr5pHBowTjmeVMe4wLcTBFuJ0+AbGoY0YAFqG61vl0sHmJocEFgBiU80M7Bnh5f9XcZ+VtOuWDwcYfxzPVy4xQgtxWIGVpJLrqz0zzAsAAJqO6/uamE6pO88MLADEqZoEdi9rbaHyDmNMrkbxAOOO41VsIa6UbpOcgQosR+kAQGuw1srxBiqwzMACQHyqaSF+vMr7gKbkuFbZEVuIC8pWVGABAM0v6hiOZmDZgQAA8RmxAmuM2UbS9pLajDH7SorWq06U1B5DbMC44Ph+mMC2DX4gTGDTqeD/Gg4tZADQEqKdB21RBZbf/wAQm9FaiI+VdJ6kmZL+QwMJbI+kf65tWMD44XhW6eGO0UnnJKegTDJ4A0MFFgBaQ7R1OBsmsGwhBoD4jJjAWmt/JulnxphTrLW3xxgTMK44rh9uIR66xCncQhw24rOFGABaQ1RxjVqIHWZgASA21czAzjTGTDSB/zLGPHtLvjIAACAASURBVGuMOabmkQHjhOv7ygxXgU0Fu8yyCVeSVKQCCwAtIVralKMCCwCxqyaB/Ttr7XpJx0iaJulsSd+oaVTAOFIqtxAPWeKUDmZic3IkUYEFgFYRLW3KMQMLALGrJoGNZl8/Lul6a+2SivuApue4fpDADj1GJ7ydU0kSM7AA0CqiDyxzYQsxFVgAiE81Cewzxpj7FSSw9xljOiXxTh0tw/V9pYZtIQ4qsJkwgaWFGABagzekAssMLADEZ7QtxJFPS5on6S1rbb8xZpqkT9U2LGD8KHlWaesM00IczMBOSnuSpO68E3doAIA6iI5Ny4Vb/DxaiAEgNtUksPPCy12NKXcOdxtjUtZatzZhAeOH6/nDH6MTLnGanAk+ee/qLcYdGgCgDoZWYF1aiAEgNtUksD+QNF/SnxTMvu4jaYmkScaYz1pr769hfEDdOZ6vpHVGTGDbVFI2lVBXX6kO0QEA4hbNwEbH6Li0EANAbKqZgX1P0r7W2gXW2v0k7SvpLUkflfTNWgYHjAdOuYV4SAIbbiE2bkHTO7JaTQUWAFrC0AosS5wAID7VJLB7hJuHJUnW2qWS9rTWvlW7sIDxw3VdJeUNU4ENtxK7RU3ryKirlwosALSCqOKa5RgdAIhdNS3ES4wxP5R0U3j7DElLjTFZSWytQdOzXvjPPDX8FmK5eU2bMFWrqMACQEsoL3HiGB0AiF01FdjzJL0h6X+FX2+F9zmSjqxVYMB4YbywsjpSBdYpaFpHlgosALQIjtEBgPrZaAXWWpuX9B/h11C9WzwiYLzxR0hg01EFtlBuIbbWqmJbNwCgCUVLnJiBBYD4bbQCa4w5xBjzP8aY14wxb0VfcQSHBmGt5DZv9XHkCmywhVhuQdMnZFXyfPUUOVkKAJrdQAU23ELMDCwAxKaaFuJrJX1b0oclLaz4AgLP3SB9Z+/mTWK9MCkdKYF18prWETxGGzEANL9oBjabis6BpYUYAOJSTQLbba2911q70lrbFX3VPDI0jrXLpL6VUve79Y5ki/N9q5TdWAW2qGkdwTxsF4ucAKDpbVCBpYUYAGJTzRbih4wx/y7pDknld+fW2mdrFhUai1MILte8JU3brb6xbGGO7yutqAKbHvxgIiEls+EW4iC5XU0FFgCaXlRxLc/A0kIMALGpJoE9ILxcUHGflXTUlg8HDcnpDy7XNN9otOPZgQQ22jpcKZWT3KKmRxXYPiqwANDsBlqIgwqsQwUWAGJTzRbiDY7KMcZsXZtw0JDcigpsk3E9X5kogU2kN3xCOic5eU2dwAwsALQKL6zAppIJJROmfBsAUHvVzMBKkowxk40xnzbG/E7SczWMCY3GyQeXXW/WN44aKHm+ssYJboxYgS0ok0poYi7FDCwAtICoAptOGKUShhlYAIjRqBVYY0ybpBMk/Y2kfSV1SjpR0sO1Dw0No6krsHagAjtKAitJ0zuyWt1HBRYAml20xCkZJrDMwAJAfEaswBpjfi7pNUkflfR9STtLWmutXWytpVcGA6IK7Lo/Dxw50yQcz1dGYQV26BZiKWwhDhLYaR0ZKrAA0AIcb3ALMRVYAIjPaC3EsyStlfSypJettZ6C5U3AYFEF1neb7igdp3IGdsQKbJDAT5uQZQYWAFpAVIFNJ43SyQTnwAJAjEZMYK218ySdrqBt+AFjzKOSOlnghA04eSk7KbjeZG3EjmdHr8CGW4ilsAJLCzEAND23ooU4WOLE5/sAEJdRlzhZa1+x1n7FWrunpEsk/UzSU8aYx2OJDo3ByUtbzwquN10Cu5ElTum2cgv1tI6s1vaX5Hp8Eg8AzSxqIU4nEkolTHmpEwCg9qreQmytfcZa+w+SdpJ0ae1CQsNxC9KUnaV0e1MmsAMV2OFaiLPlCuz0joysldb2OzFGCACIm+dbGSMlEkbJJBVYAIjTaEucLjfGTB16vw08bIw5yhhzXG3DQ0Nw8kElcuquTZjAbmwLcdugGVhJ6upjkRMANDPHs0ongrdQ6USCJU4AEKPRjtF5UdKvjTEFSc9KWiUpJ2l3SfMkPSDp/695hBj/3EKQyE3dRVr1ar2j2aIGVWCHbSEevIVYklb3lKRt4ooQABA3z/eVTBhJCmdgGR0BgLiMmMBaa38l6VfGmN0lHSJpW0nrJf23pAuttfl4QsS45+SDRG7qrtJr90m+JyWS9Y5qiwhmYMMK7EaWOE0PE1gqsADQ3BzPKpUcSGCZgQWA+IxWgZUkWWtfl/T6WF7cGLNI0vckJSX9l7X2GyM87xRJt0laaK19eiw/C3XiOZL1ggpsxwzJK0nrV0iTd6x3ZFtEtIXYT2SUMGbDJww5RkeSVnOUDgA0Nc+3SifDFuJkghlYAIhR1UucNpUxJinpKkkfU3Cm7FnGmFnDPK9TwYbjP9QqFtSQ0x9cRjOwUlPNwUbnwNrhqq9SkMB6Jcn3NKktrWTCqKuXCiwANDN3SAsxM7AAEJ+aJbCS9pf0hrX2LWttSdJNkk4Y5nlfl/R/JRVqGAtqJZz/LLcQS02XwGZVGr59WAr+3JLkFpVIGE2dkFEXFVgAaGrBEqcggU0lDMenAUCMapnAbi/p3Yrby8P7yowx8yXtYK29u4ZxoJbC9lml2qTO7YKjZrrerG9MW1C0hdgOt8BJCv7cUrDIStL0jiwzsADQ5DzfKpmkAgsA9TDaMTr3V1z/0pb+wcaYhKRvS/rfVTz3QmPM08aYp1etWrWlQ8HmqKzAJhLBebBrl9Uzoi3K8XxljDP8GbDSwGZiJ0jkp3dkmIEFgCbneP7AMTrMwAJArEarwM6ouH7aGF57haQdKm7PDO+LdEraR9JiY8wySQdKussYs2DoC1lrr7bWLrDWLpgxY8bQh1FPlRVYScpNlEp99YtnC3Ojc2BHqsCmB1dgp03IUIEFgCbn+XbwDCwtxAAQm9ES2M39OPEpSbsbY3YxxmQknSnprvKLW9ttrZ1urd3ZWruzpCclHc8W4gZTWYGVgqVGTvOcsBSdA2tSoyxxkgYS2I6sVveUZC2fxgNAswqO0QneQqVoIQaAWI12jM6uxpi7JJmK62XW2uNHe2FrrWuM+byk+xQco/MTa+0SY8zXJD1trb1rtO9HgyhvIW4fuCz+pX7xbGGlcAuxGXEGdnAC+6GtO5V3PL2xsle7b90ZU5QAgDh5vq90OAObShpaiAEgRqMlsJUbg781lhe31t4j6Z4h9315hOceMZafgToLE7dyIpfODVRlm4DrWWWNI5OaOPwTospz+Gc+aLdpkqTH3+wigQWAJuVWtBCnEgkqsAAQoxETWGvt7ytvG2PSCmZWV1hrV9Y6MDSIqF04mgVNtTVdC3FWzigV2GgGNvgz7zC1XTOntOnxN1fr3IN3jidIAECsKpc4MQMLAPEabQvxj4wxe4fXJ0l6QdL1kp4zxpwVU3wY7zaowLYNLHZqAiXPV9a4G99C7A4sbjp4t2l68q018vlEHgCaUuUSp1SSGVgAiNNoS5wOtdYuCa9/StJr1trZkvaT9I81jwyNYWgFNt3WfC3E1Wwhrqg6H7TbNHXnHS19f30MEQIA4hYscYpaiJmBBYA4jZbAVh5m+VFJd0qStbZ5NvRg8w2twKZyA4udmkD5HNgRW4ijCuxA0n7QrtMlSU+82VXr8AAAdeD5VqnyMToJOR4JLADEZbQEdp0x5jhjzL6SDpH0W0kyxqQktcURHBrABhXYdsl6kufUL6YtyAm3ECs50jE6g8+BlaRtJuW064wJevzN1TFECACIm+P5g47R8XxmYAEgLqNtIf6MpCslbSPpf1VUXj8i6e5aB4YG4eSlREpKpoPb5a28+YH7GpjjWWU0SgV2yBbiyEG7TtOdz60IFn0kR/ucCADQaDzfDjpGhxlYAIjPiO+srbWvWWsXWWvnWWt/WnH/fdba/x1LdBj/3MJAFVIaaCVukk3EQQXWGWWJU3QO7OA/78G7TVdfydOLK7prHCEAIG7BMTqVFVgSWACIy4gVWGPMlaN9o7X24i0fDhqOkx+oQkpBC7HUNJuIywlsaoQW4mRGkhm0hViSDtx1qqRgDnb+jlNqHCUAIE7BMToDM7AuM7AAEJvRWoj/XtJLkm6R9J4kE0tEaCxDK7AjtNQ2Ktf1lJI3cgXWmHDz8uCEfVpHVrvNmKDn310XQ5QAgDhVHqOTThq5zMACQGxGS2C3lXSapDMkuZJulnSbtZZ35BgwtAIbJbNNsonYRpXVkWZgo8eGVGAlaadpE7R8bXNUogEAA4JjdIIW4mTCyLeS71slEnzWDwC1NtoMbJe19kfW2iMVnAM7WdJSY8zZsUWH8c/JD2wglgauu81RgTVeeJrUqAls27At0zOntGn52uZI5AEAAzzfLx+jE12yyAkA4jFaBVaSZIyZL+ksBWfB3ivpmVoHhQbi5oe0EEcV2OaoPJYrsCMdoyMFFehhWqa3n9ymnoKr7ryjSW2Nv5EZABBwPatUcmAGVhKLnAAgJqMtcfqapE9IelnSTZK+ZK114woMDcIpSJn2gdtNtoVYXjUtxLlhK84zpwR/LyvW5klgAaCJuL4tV16j43SCOdhkHaMCgNYw2gGVlytoG54r6d8kPWuM+ZMx5kVjzJ9iiQ7j3wYV2GgLcZO1EI+0xEkanMC++aD0q89LkrafEvy9rFjXJMk8AEBSkKxWzsBKYhMxAMRktBbiXWKLAo3LKQw5Rqe5KrCJcgV2lBbiVEUL8Yu3S8//t/SJ/9DMMIFlDhYAmoe1Vo5ny8foMAMLAPEaMYG11v55uPuNMQkFM7HDPo4WM/QYnVRzzcCWW4hHq8Cmc1JhfXB99WvBZf8aTevcRrl0QivYRAwATSPKU6PZ16gSywwsAMRjxBZiY8xEY8yXjDH/aYw5xgQukvSWpNPjCxHj2ohbiJsjaUt4TnBlo1uIC5K1Awlsfo2MMdp+chtH6QBAE3G84MzXgSVOlTOwAIBaG62F+AZJayU9Iel8Sf8syUg60Vr7fAyxoRGMlMAOs5W3ESX8ao7RyQZ/D32rpEJ4THL/GknBIidmYAGgeUSV1g2O0WEGFgBiMVoCu6u1drYkGWP+S9L7kna01jZHZoLNZ224xKliBjaRlBJpyWmOuc9EVS3EbZJbHKi+SlI+SGC3n9KmPy1fV8MIAQBxihLVDZY40UIMALEYbQuxE12x1nqSlpO8YhDPkaw/eImTFGwibpItxAk/aiHeyBInNz84ge3vkiTNnNKmtf2O+kucQAUAzSBqFW7zeqUfHKxpPa9IYgYWAOIyWgI71xizPvzqkTQnum6MWR9XgBjHojnXyiVOUpDQNskSp3IL8UaP0SlKq18feF7YQrz95PAoHeZgATSwt1f3sVE9FFVaJxeXSyuXaFJ3kMBGs7EAgNoaMYG11iattRPDr05rbari+sQ4g8Q4Fc25Dq3AppongU35VRyjEyXsq1+TZnwoqEDn10oKZmAlscgJQEP74q0v6Gu/XlrvMMaFKFFt93olSenwvxNUYAEgHqNVYIHRRRXYdPvg+9PtTbGF2FqrpA1biDdWgbWe9MHSIIFtm1qxxCk8C5ZFTgAaWHfe0bp+Z+NPbAFRopp1eyRJaRt06jADCwDxIIHF2EVV1tTQGdhcU2whdn2rjMLZ1aF/xkrRYz3vSdP3kNqnlJc4zejIKpNM0HoHoKEVXV/9DrP8kuSES5xyYQU2RQUWAGI12hZiYHRRApseMgObamuKFmLXs8qoiiVOlX/+6bsPqsAmEkbbTc4xAwugoRVdT/0lU+8wxoUoUc25wTqQZJjAuszAAkAsqMBi7KJNwxtUYNuaooW45PkDFdhRW4grHpu+h9Q+rVyBlYI5WGZgATSyousrX/LqHca4EM3AZrywhdgP/ltICzEAxIMEFmM3UgU23dYULcSO5ytjohnY9MhPLG9hNtLU3aT2qeVjdKRgDnYFM7AAGljR8dVPAitpoAKbcYIENunRQgwAcSKBxdiNVoF1Gn/m0/WssnLlJTKSGaV1LtrCPGWn4HrbVCm/TvKDN3vbT27Tqp6iCg5v/gA0Hmutiq5HBTYUnQNbTmDDFmKO0QGAeJDAYuycEbYQp3IDyW0DczxfWZXkJ0aZf5UGEvjpewSX7VMlWanQLUmaOTWo0L5HFRZAA3J9K98GYxXMeQ4scco4wQxswgv+e0cFFgDiQQKLsSsnsMNVYBs/WYtmYDc5gW2bGlyGi5y2n8xZsAAaV9EdSFr76SQpJ6qpMIGNWoiZgQWAeJDAYuzKLcTDzcA2frIWbSH2R1vgJA3MAE/fPbhsDxPYcJHTVp3B96/uLdYiTACoqWJF0kob8UCrcKoUJbDREieq0wAQBxJYjN1IFdhUm+QVpQb/j3mwxMmVn9xIBXarvaQ5Z0h7LApuD6nAduaC06p6i5yhCKDxDKrAksBuUIGNWohdjwosAMSBc2AxdiNWYHMDj2eGzMc2kKCF2JHdWAKbmSCdfPXA7SEV2I4wge0pkMACaDyDE1h+j0UzsMlisOfAuMzAAkCcSGAxdk5eSqSk5JB/RtFSJyff0Als0ELsShtLYIdqH1yBzaaSyqQSJLAAGlLR9XRh8tfqUbvypYPqHU7deb5VViUlwtnXBDOwABArEliMnVvYcAOxNLDUyG3sOVinXIHdyAzsUNmJQWJfcRZsZzalnoKzhSMEgNorOr5OSj6mLttJC7GCWdeJGjgqLqrAsqEZAOLBDCzGzunf8AxYaWCpkdPYR+mUPF9Z42x6BdYYqW1KuYVYCuZgmYEF0IiKrq+ciuowBRJYBS3EE01fcCPTIRN+WEsFFgDiQQKLsXMKGy5wkioS2P4NH2sg0RZipTaxAisFi5z6BxLYjlxKvbQQA2hARddTmympXQXlHX6PeZUV2M5tmIEFgJiRwGLs3PyGC5ykgfvcxq7AOp6vrNyxJbDtU6X82vLNjmyKGVgADano+GpTUe2mSAVWQQV2UlSB7dim/N86KrAAEA8SWIzdiBXY8L4GPws2moE1W6AC25lLq4cWYgANKGghLqlDec6BVVBpHVSB9UpKyOcYHQCICQksxm6kCmy5hbjRE9hgC7HZ1CVOktQ+ZAaWJU4AGlTRKSlrXLWLGVgp+HCzPAPbuY0kKWdK8hr87HMAaBQksBg7pzCQrFYqtxA3egLrK2McmfRYEthpQQXWBp/Is8QJQKPyisHv8ozxlC809u/1LSGowEYtxFtLkiYkXDm0EANALEhgMXZOfvgEttxC3NgzsK7nK7s5LcResbzIqiMXzMBayxscAI3FK/aVr/uFnjpGMj64vtVE0y+bapNyEyVJ7QmHJU4AEBMSWIydmx/hGJ3wbNgG30JcCluIE2Nd4iSVz4LtyKbl+VYFhxYzAI3FKw78LvcLvXWMZHxwPF+T1CflJpU7jjoSLjOwABATEliM3YgtxGFS2wRbiDNylMiMsQIrlRc5deZSksQcLICG41fsM7BFEljPD7YQm9ykcsdRUIHlA0oAiAMJLMZuxApscyxx8lxHKeMrMVySvjFRBTY/JIFlDhZAg7GlvorrtBA7ntVk0y+1TS5XYNsTJWZgASAmJLAYu5EqsMmMZBKNn8A6JUkaWwvxCBXYXs6CBdBg/NLA73JTkcy2Ks/3NdH0By3EUQXWOPJoIQaAWJDAYmysDSqwwyWwxgSfSjd4C7Efxj+mJU7lCuxaScEMrCT1kMACaDQV+wxMg+822BIczwbH6OQqKrDGkUsFFgBiQQKLsfEcyfrDtxBLwafSDV6BlVsMLlOZTf/etinBZViB7ciGFdgiM7AAGkzFRvmkQwW2fIxORQW2LeHIZQYWAGJBAouxiT6FH2k+NN3e+AmsEyawyTFUYJNpKTtpgxnY9VRgATQYU3Gmd9KlAut6njrVP2gLcZspUYEFgJiQwGJsovbgkSqwqVzQYtzA/HIFdgwJrCS1Tykfo8MMLIBGZSqS1pRHBTbh9CkpP1jixAwsAMSOBBZjE1VXR6zA5ga1nTUkL6rAjqGFWJI6t5O6V0gaaCFmBhZAo0lU7DPIeK1Zgb37T+/r5B88Js+3yrjhJuaKCmxOzMACQFxIYDE20SbKdPvwj6fbBy3+aEjlCuwIVeaNmbyj1P1u8BLJhNrSSWZgATScZJjAeiaprJ+X14KJ2rPvrNWz76zTkve6lXbWB3fmBiqwQQsxM7AAEAcSWIxN36rgcsKM4R9P5Rp+C7G84BidMS1xkqTJO0jr35O8oOramUupl3NgATSYhBf8Li+kp6hdBeUdr84Rxa87H3z4+MSbXcoNU4HNmlJLJvYAUA8ksBib3pXBZcfWwz+ebmv4JU7G3YwlTlJQgbWetD5sI86lWOIEoOGk/IKKJis3NUEdpqD+Uuv9HosS2Mff7FLGqUhgEwkpmVGbHLnMwAJALEhgMTZ9UQI7QgW2CRLYgQrsGBPYSTsEl2EbcWc2xRInoNE5BWn5M/WOIlYpryDHZOWnJwQV2FLrVWDXhwnsU8vWKOOGLcRtk4PLVJuyooUYAOJCAoux6V0ZVCazE4d/PNXW8C3Eic1d4jR5x+By3TuSpM5cWj0FZmCBhvbCz6X/Okr6YGm9I4lN2i/ISeTkp9s1QUX1t2AC2513lEkl1F/ytHZNOEKTmxRcpnPKqcgSJwCICQksxqZ3ZdA+bMzwj6dzDV+BNd5mHqMzaWZwuS6owHZkmYEFGl6UuL50e33jiFHKL8pNZKXMBE0w+eET2J4Pgq8m1VNw9eG/mi5Jyro98mWCs74lKZVTVszAAkBcSGAxNn0rR24flsItxI2dwCb8sFo61gpsKit1bFNRgU1xjA7Q6LreCC5ful2yrZGwZGxBbjInZTo1YaQW4l9eKN32qfiDi0l33tFO09q117YTNUl9ypv2YP5VklI5ZWxJDjOwABALEliMTe8qacJWIz+eykluvqHf4CU2dwZWCo/SCRLYjhwzsEDD63pTSk+Q1r4tvfdsvaOJRcYW5SVzMpkOtZvi8Euc1i6T3v1j45//PQzX89VbdDUxl9bBu03TRNOv/kTHwBPSUQWWGVgAiAMJLMZmoxXYnGR9yWvcmU/jRwnsGM+BlYKjdKIKbDal3pIrnzYzoDE5+WAp237nSom09NId9Y6o5qy1ytqivGSbErkJmqD8hsfoWBu0D/uO9N5z9Qm0hnoKrj6eeFKfeu50Hdf2oiaqb3ACm2pTxpaYgQWAmJDAYtP5XnAO7EhH6EhBC7EkOf3xxFQDySiBHWsLsRRWYFdIvqfOXFrWSn0teAQF0BTWvCXJStvvJ/3V0UEC2+RVN8ezyqkkP9WmZNtETVBB/UNn+Ys9QceNJL37ZPxB1lh33tH+iVc0uf9t7fvIhfpw4iXlk4MrsBlmYAEgNiSw2HT9a4Lq6sZaiKWG3kRcTmA3p4V40g5BVaLnL+rIpSSJRU5AowrnX8/+1RoV9jxR6nmvKRO2SkXXCxLYZFapXKeSxqpYGPLBZG/F8qZ3/hBvgDFYX3A03axX/4SZ0sEXK2ucYL9BJKrAMgMLALEggcWmC8+A9SaM1kLcFlw28CKnpF8KNk0mUmN/kegone531RkmsCxyAhpUmMA+0ztNb087PDgu7MXb6hxUbRVdX22mKJtuU7otqDp6+fWDnxQlsJN2kN79Q0PvPhhOd97RNK2XN2Eb6ZivK/G5J7XnOd8beEI6p7Rf5BxYAIhJTRNYY8wiY8yrxpg3jDGXDvP4F4wxS40xfzLG/M4Ys1Mt48EWEr5ZOefmt/Xo66uHf04zJLC2JM+kRz4qqBrls2DfVUe2eRPYX7/wnl7/oKfeYQC1tfoNdaemq185/aWQlHY6SFr+VL2jqqmi66tNJdlUu1K54Nxvt9A7+Ek9fwku9zpeyq8Z2NTcJNbnXU0z3bIdYdfRVntKE7cdeEKqTWlbpIUYAGJSswTWGJOUdJWkj0maJeksY8ysIU97TtICa+0cSbdJ+mat4sEW1Bsc4v6eO1F3v/je8M9JhQms27gJbMIryUtsxvyrVHEW7J8rKrCNu9hqODc8sUwX/eI5/WDxm/UOBaitrjf0bmI7SdIH3YVgjCK/ts5B1VbR8dSmYvChZGaCJMkfmsBGFdhZxweX7zRXW3V33tF0063kSIsL0zmlbZFjdAAgJrWswO4v6Q1r7VvW2pKkmySdUPkEa+1D1tpomOZJSTNrGA+2lLCFeLWdpN+/ukp2uHaxdDgD26BHKjieL+OV5G9uApuZILVPD1uI05KaaAb2mZ9q1Q+P05fvekmS9Naq3o18A9Dgut7QG14w+/iX9QWpfWqwE6CJFR1XOePIVCawxSHdFr0fBMvuZu4vtU1purng9X15TTW9ykwcYXFhqk1pnyVOABCXWiaw20t6t+L28vC+kXxa0r01jAdbSu8HchNZ9ahN73UX9MbKYRKX8hbixqzArs87yhhXfnIzFjhFwqN0mq2FeM2S32nGB4/onK3/rNP2m6m3VvUN/2EG0Az610j5NVpSDNpIP1hflNqmSk6f5BbrHFztlIrB73CTaZOyneGdQ1uIPwi20icS0g4HNN0iJ7c3GJVJjZTAbs4MbO/KYLM/AKBq42KJkzHmbyUtkPTvIzx+oTHmaWPM06tWrYo3OGyod5XymamSgtnQ3782zP8m5S3EjZnArss7ysjZvCN0IpN2kNYNLHHqbZIEdvUHQfv4ZTMe06ztJqqn6Gp1b6nOUQE1Es51vukHFdgP1hek9inBY03cRuzmg2TVpNvLFVhT6hv8pN4PpGg+dIf9pa7Xpb6uOMOsKT9skTYjtRCn2pSQJ+Nv4njI+vel786W/nTLZkYIAK2llgnsCkk7VNyeGd43iDHmaEmXSTreWjvsx9jW2quttQustQtmzBhl8y3i0bdSPckpyiQT2n2rDi1+dZgEtsErsOv6HWXkbt4ROpHJO0rd72pCOilJ6mmSFuKcE7xpz7xxr/acELQU0kaMphUmsG/ZlFUmGQAAIABJREFUbdWWTgYJbFuYwDZxG7FbDKZ8EtmKBNYZLoENj5XZ4cDg8o0H4gqx5hJ94bLCkTbvhyMzSa+4aV0or90bHDW3+rXNjBAAWkstE9inJO1ujNnFGJORdKakuyqfYIzZV9KPFSSvK2sYC7ak3pVal5iiye1pHfGhGfrj22vUXxqSlE2YHlx2v7vh9zeA7nxJWZWU2FIJrFtQIr9aHdlU0yxxane7tTQ7R7JWs1bcLkl6a3XfRr4LaFBdb8g3KS23MzR3h0lhAjs1eCzf/AlsMjNBygQtxAl3mHNgowrs9vtJ0/5KuvOz0qPflZrgaJlkPkpgRzj7POw4ysnRJo3BvhpMTZW639+M6IAW5BQkl46vVlazBNZa60r6vKT7JL0s6RZr7RJjzNeMMeGqQv27pA5JtxpjnjfG3DXCy2E86V2p1ZqsqRMyOnyPrVTyfD351pB2sbbJ0pSdpfeer0uImyuqwCbSWyiBlaR176gzlyq3EPcVXRWcBp19slYT/W6taN9L2uNYTVz6c01I+XqbBBbNqusNdee2k6uU9t1xilb3luRkm78C64XtwsnswBKntFPRaeGWpP4uqTOswKZz0gUPSnsdJz3wFenmTzZ8Epsphv99iz6YHSo8Ni5nSiq5Vf5ZS32yb/1ekvSX5W9vbohoIOsLjm548s/qzjfHh9l1ceOp0q8vqXcUqKOazsBaa++x1u5hrd3NWvuv4X1fttbeFV4/2lq7tbV2Xvh1/OiviLrzPal/tVb6EzW5Pa0FO09RWzqp3w/XRrztPOn9xkxgu/OOMsZRMpPb/BebFHbSh4ucegqufN/q1B89oS/d8eLmv349lPqUkSPbNk1aeL5M30p98v+xd56BcVR31//N9qq26r24yr3gbnoH0zsEEkIJECAhgbwhhZQnCWkQIAGSYEoeApheTLWxMTbuVXKVLMnqfbWStrd5P9xd1ZUt2ZIl+dH5stbs3ZnZ9e7MPfd//ueY94xJiMdw6qLpMLWqdBLNWjLjRItEc1AQulO5AhsMVWBVWiOotARQogp0qcA6Qtd+U5fqpC4arn0FljwEhz4RPbGjGHqvFZ+kFu8rEkIVWC1ebK4eVaG974LL1vs1JWuRAh6aZTMKR/0gn/EYRirWHmrggie/5hfv7+W+/+7EHxjdizvDhtoCKPps1C+OjeH4MSJMnMYwiuBsBjlIbcBMrEGDTq1kYZ6F1QcaONzQI1ohdSbYKkZldSJcgVWpB4HARoncSNrrRAXW42ddcSMHats4VNd+9NeOUIRdOTFYIO8ciErnYsVGShvHKrBjOAURDIK1hFI5hbRYPclR4rpQ5wv1+o/Ca1x/EfSGCKzOCJKEV6lHHejibWCvE4/hHtgwJAkmXCD+bRudrSRhGP0tOFSx4j1FQrgCi5em9i4EtrkE3v4OfHAf9OyNPfQpbqWJVYE5GL2njuHVGPrGnz47yHde2oZJq+L7Z41jw+Em/vDpweE+rdEHlw08rWLhsHHs8ztujFKPmjDGCOwYBga7aFWu8pqJNQqH3hvnZVLf5ubcJ77mmuc2srsytNqcMlM8jsIqbKvLh17hRxoMAquPBaUW2msw6dS0u328uEFIxqpto/MCYreKSavSFC+iM/LOZKK7kCqrHd/YivIYTjW0VYHfzUFvIumxBhKjRGtBrUMSv+1TuQLrFdcotVaQdZ/CgKZrBTZ0T8AcIWKmo33iyBCe4dBClmXMARtuTWzfgzp6YL00Obp4UVpD0uCDK2Hfu53bgwEo+ozdutOokeOJpY1G2+hczDxRvLC+lP9uKR/u0xhyBIIyL288wrmTk1j5wBJ+fMFEvrM4m+Ubynh7R9Vwn97ogq2i89/l3wzfeYxmbPknPLtARKCNUowR2DEMDA4xWTniMRFrUANwXn4Smx89h0cvnkRZk4Pff3JAjE2ZIR5HaB/sjnIrje2R8xttTi86yT84MTqSJPrDQhXYww121hc3kRSlpdXlwz4KXYkdLeKip4kKuXLmnIk+0MYEuYxKq/MorxzDGEYhQi6xO5yJpMV0VmDr2z1giAPnqRujI4dW6dU6EwA+lQGt7CIYditqD1dgIxBYU7K4hnadcI4yOL0BLLTi1Vr6HhQmsJKX5q5RYi0hAmsZB588DGE346rt4GxipXsWXr2QXheXlAzF6Y94vLa1gue+OvXfe3mzA6c3wPlTktCqRCLBzy6ezMJcC794fy8e/yj1wxgOhK8nkmKMwPYH794FryyD6p1CCfLlb+DTRyBpat9tEaMAYwR2kOH2BUavMU9/EFptbwhGEWvoJHfxJi13nZ7HBVOTOVjbJqIEDHEQkzUiK7CyLHPbi9v4+5rIvVk2lw+tNEgxOiBkxO11mLUqHN4AOrWC7581DoCaUViFdbeKvjdDbGjSmrMUgEWKfWMy4jGcemgS14kifzLpsXpiDRrUSom6No9wIj6FK7CEJMRqnajA+lVGjLhxhyfcoYzUiA69CkUoB3v0Etg2tw+L1IZf34eBE3TE6Ojw0mTvsijackSQ2+v+A+42MZHc/iJsfBpZoeJDx2QmjR8PQGV56RC+i5GLpnYPVS2uEbfwOdjzuP21bQDkp0R1bFMpFdy8IBOXL0Bx/Zh/RL8Rvp7kngVHvuktzx9DJxxNIme6bD38+yz45+mw/q8w59viujQYKsNhwhiBHUSUNzuY/qsvWFlwClvihwhsoxzdjcCGMTnZTJvbT12bW2xInTkiK7BObwC7x09JH2Sr1eVDi29wKrAgKrBtNZi0KgCunp1OfqpY+RqNMmJfu/geGGNCBNacTMAykcWKfWNOxGM49dBUhF8TTTNRpMXqUSgkEs06GtrcoQrsKUxg/eL6pNQKw6qgyoBRcuP0diGw+jhQ9XGtjMkc1QS21eklnlZkw1Ey6FWiB9as8tPck8DGZkPSFDjnl1DyJaz8IRxciTV5KW0Yyc7JA6Cp7tSX0faExx+gLeTK3yvJYBhRUGVj6mOfD6op4f6aNlQKiXGJpm7bw4Q2THDH0A/YKkBthPzLhCowlNE9hggo+gyQ4baPYPEPxGd1xk/g0r+BQjncZ3dCGCOwg4iMWAMmnYqNJU3DfSpDB0cDQaUWO3pijepeT09MFhfjg2FzopSZYCsfcRM8q0PIvI40RyZbHkcbUUEbRKUNzgHNKdBeR5xRjSTB7UtySIsRk57RWIEN2JvxyUpiYjurEsq8M5inPER5w6krpxzD/1E0FdNmygEkMmLF7zYpSisW6vSx4Dp1v/NSiMCGjYqCGlGBdYUJbHt9Z4ROJIxyAmtvs6GTfChMRyGwoSqGRSv3kBCXCwILsPgBeKQMHjoIP9zPu+MfByA7K1ccp6l6KE5/RKPrZ7W5dOTMEQ7WtuMPyhyoHby+5P21bYxLNKFTdycNWRYjerWSAyeZwNqcXgIDCi0eQWitFNeVrCXi7yMbhvd8RjIOfSrmsdlL4Lxfw0+r4axH+zakG0UYI7CDCIVCYmGuhU0lzUJCeyrC3oBHGw9IxESowE5MEkH3He66qWEjpz0n6QT7h+YQga2xuSL2niS6SlAgQ/LUwTmgOQV8Dm6eZeHt7y0iL8FEglmLSiFR3TL6CCzOJlowE2Ps8h3IOQM9HpS1O4fvvMYwhqFAUxH1GmFIlBoTJrA66sMV2FNYQiyFnSpDVUY0Jgz0qMCaIsiHw4jJFFE73tGpzHDbhERaFRWhxzeM0Gdj0QZpDFdgZbmzAhuGIQ6iUiA6jcI6F6nROmISUgmiwOBtFN+n/0MIy61NWtWIqsCG/x+qbYMnaz5Q29ZNPhyGUiExKcXM/pqTR2Bd3gBL/7iW17aO0oUlW7m4rljyRO/9WB9sZPhcULIGJl7USVgVpw7tO3XeyQjBwjwLta1ujjSPrH6OQYO9AZcmDoC4CAQ22qAmJVrXSWBHqBOxNeQUGZSh0tqdQAaDMumekKlE0iASWCDa18icLOFmqVRIpMToRmUFVum2YsPcfTU5ezFBJFJbtg3fiY1hDIMNlw3s9RwhDYtRg0Ej2gAEgfV0VmBP0UVLhd+FB03nxEdjxCS5cXpD5nP2+t4ROl0RkyUeR2mUjq8tZFgXfRQCG6rAxqoDnVVFpxW87d0JbBfsr20TbSQKJX5DAonYKKxqHcxTH/FosnvIk6q5cZyfatvI6YMNt0BVDdLicpPdQ32bh/zU3gQWhIz4QNg75CSgtMlOu8fP7ooI+cSjAbYKQWAlCbIWj/XB9oXSdeBzwsSLh/tMhgRjBHaQsXickFR+c/gUlRHb62lXCQIWqQcWYGKyuVNCbIgTF5oR1gfbVbp0pEfPpt3rZ5JUjkdl6oyBOFFECQJLe/f+6NRo/Yn3wHrs8NfJsOvVY4/1uWH367D5edHIX7LmuA6p8bRgV/Zwr9PH0mSaxEz/HtrdvuPa7xjGMOIQ6q864E8mLSQfBkFg7R4/Hk0MBP3gOTV72BQBtyCwIUihCqzLGxCTRnt95AidMGLDBHZ0VnuCoX7/DsO6SAhVYGM0gU4Tp5Yj4jECgXV5A5Q22jsIjTIqmSSphYLq/2MEtt3LX9XPcb/19wBsCldhbRWw4W8ibmgYUN8m/g8Hi8AeiGDg1BWTU6Joc/tPmh9G2Pvj8CD2+J40uGzgboWYDPF39mJor+l0/B5DJw59DBqzkA+fghgjsIOMbIuBlGgdm0pGjhxm0OBqgcaD1GjHo5DArFNFHDYx2czhhvbOPNCUmSOwAtuFwPbog211+pisqKA1atLg9QmYwwS2rtvmtBg9NbbIsrGvDjXw5YF+ZHSVrhUX8LV/AL/36GN3vAzvfw8++4mwUn/njuNaudT5bDhVMb2221MXM0sq5kht44D3OYYxjEiEInR2uxJI70Jgk6OFQ3krom1ipPX5DxaUARceqdONXaEzY5Q8OD0+cU8IeCNH6ITRkQU7Ok2KZEfYcT2l70EqLSARrfJjdXhFxFB4Qh2BwB6sayMow5QOAptChrqNwqpRWhE7TjTaPaRLTUS17GWKobVTRrz+r7D6Mdjw5LCcV0N7SEI8SAQ2LA+e3AeBDS9knGjP7VOri/sVSRQ2pyppsI++drfWkJIjfF3p6IMdkxF3QzAIhz6D8ecOXprGCMMYgR1kSJLEwjwLm0qbO3PyThWUfQ1ykD36OcQaNCgUkcndpGQzvoDc6UabMl2sRntGzmqf1elFo1QQpVNFILAeJkkVuOImD94BwyYnbTXdNqfF6qlrc+MPk/0Q7B4/97++i+++sp1XNx9j4lf0mchDa6uCwreOPrZgBSRPE2Yilz4JzuaOCfpAYAq04tHE9tquHX8GGilAa/HGAe9zDGMYkWgqQlao2dEa1WG8BpBkFrLR5qBw5z1V+2CVAQ/eLgRWGcqD9bjsHa70RyWwxkRQakctgVW6hJpKeTQTJ0kClQ6Tyo8/KNPm9nVWYMMS6i7YV9OjImdOJlGyUVh98mSkIwEtbe3ES+Kz+HZcIVtKrch+D8F97xNUqOGrP4jsypOMutawhNg5KP8f+2vbSI3WEeut7fxedMGkZDOSxAn1wda3uXlmTTHv7ao65thwBdbu6ZIYMVoQUnKU+CxijpkwEQyWsT7YnqjeIRyaT1H5MIwR2CHB4rx4rA4vh+oHz8FuRKBkDWij2Ms4Ygy9HYjDmNTTidicKh6dI0dWbbV7iTNqyIk3cqSpe9+Nu/4wRslDIHHK4B1QYwRtdK8KbGqMnkBQpr7d0237m9sqaXf7mZkRw8/f38vyDX3IY4JBKPoC8i8XxHTDk33LrppLoGYnTL9eSLtzzhDbB3rhDwYwye34dXG9nooffxoAvpp9A9vnGMYQAY9/epDP99Ude+BQoqkYX3Q2Dr+CzDhDx+akaEFgG/yhbc5T04lYFXTjU3RmBWqN4vpeVlMP9tD/zdEIrEIh5H7DLCFevqGMq54d+CRX7WqiDVPfMUEdA3WYFKIvuMnuEUTFlAQaQ6+h+2vbiNKpOiv6pmRMARs2u4OGHveCUxl+W2dLzemBzVTbXLz4ygso3DZ+6L4Th8YC7955Ug3A/IEgTXZPR2a7zXni7TD7a9rITzbCixfAUzPgX2fC+ieEImrbcgwVX5FjMbK/9vgl5K9uLscflKlucR2TdJc22jGHIv0ON4ycwkK/ELqOPLLaxg9X7A71wS7qXYF97x745OFhOMERgkMfg6SE8edFfNrlDQx61vHJxhiBHQIszLMAp1gfrCzD4TWQczrNTrnP/leAvAQTKoXEobrQaqIhRHScI0dWbXUIApsdb+xVgZUa9gKgTJk2uAc1J/fqgY0UpeMPBHnxmzJOy47lzbsXctHUZH67cj8vfROBxNbuEqtsEy6CpT+C5mI48FHk4xe8CUgw9Wrxd1xuyMFv08Deh6sFBTJBvaXXU9roZGyY0bQcGtg+xzCGHnB6/fzr65JjKxCGGk1F1KhEv9UZEzrddpOiBKmr9YYIyilagVUHuhPYqCihvPhgaxFNdSE5nzmZGpuLnRUtna0jXRGTNewEdluZlZ0VtgFP2rTeZloVvdslekGlx6gQZKfJ7u3tQByCLMtsLm1mWno0UrhFxZyMhEw8rVSMECOjk4KwIiljPoktu4inlZTyD7Ero2nNvZQ72u5Abi6BVY91e1nD7s9oenw69k8fG3RzsGaHl6AMMzPF//mJ9sG6fQFKmxycbzos7v/TbxBPfPlr+OhB+Pgh+O+1zE5UHLeE2O0L8NqWCpQKCYe3M1s3EoJBmdJGB2dNEtey0UhgZbWBghYluyttVLU4hYy4taLzGuNqEWqznf8BzylWSOoPZBn2vQ+5Z4A+lnv/u4P/Wbm/25C3dlRy2v+sHtXO52MEdgiQGqMnJ954avXBWkvFBSLvLFqcXmKNfRNYjUpBboKx04nYECI6I6hHrNnhxWLSkGUx9orS0TTtxy8r0KUOYgUWIhLYcCRH116bL/bXU9Xi4o6luWhUCp65cRYXTEniNyv3s7JA3PBlWWbj4Sb2r3sbWVIQzDsHJl8GlnGifyjYYxIpy1D4JuQshShREa+yuXCnzhcV2AHIpDxtQjaoMPauwCJJ1GmzibWX9nt/YxhDJByoFX2ChdWtwyerDPjAWsoORwL5KVFkWjqraSatCpNWRYUrRO5G0PVtMKEOevAruvRQaYRkOlrhpmzDCmSlhm3NWs5/8muuenYjs36zijv/s727Ic0IyIKtaRXnM1BCYvC2dBgXHhVqHTpJeBA0273dM2C7YFeljdJGB8ump3ZuDLWYJEo2Kk7VBIMIUDtD98N5dyEh887iCi7S7MY0+zqeu3UBgcylvBY4h+D2l8TnCSDLOD/5BTpXPfotTyH/bTryF78c8LEjLrTQKR8OpwUcV5SOtRSengXVOymqbycQlFngXAdqo2jduesreLgEHjoA174CcpClxgoqrM5eBoiyLB/TFPGjPTU0O7zccJpYaDta725dmxuXL8C8nDiidKpRSWD95nTC61CfFtYJIyeA8lDrUvEqkAPgd0PR58NznsOJugLRg59/BdU2F58U1vHfLRU4PJ0LG+/tqiYtVt+xEDsaMUZghwgL8yxsKbP26m0ctQi71eaGCOxRJMQAE5OjujgRjzwCa3V4iTVoyIk3hKJ0Om9SJttBSuUUoqMiGy4cN6JSI0iIxcWj62Tv3+tLybIYOHeykOWplAqeumEWc7NieWjFHl7bUsEty7dw0wtb8B/8lO2Bccz4yw7ueW032zNvh7oCDr3wHa5/bj33v75LhJVX7wRrKc5JV/PHzw5y/pPrWPLHtSyvSIG26gFNLu3WUC6iOXL2Y3vUONL95cg9SfQYxjAAhCNFbE7f8FWlWo5A0M83tlgunNo7KiYxSkuFK7SYd6pWYGUPfmWXSU6IwP4qs4DTnOtZl3I733p1H0lRWv52/Uwum5nK10WNPLW6S299TKZQ4AyjD8J46zoeU70iKjYDgClgw6XuB4FV6dEhCKy1zS48CSIQ2Le2V6FTK7hkehdTqJAEO1nRMuoqsOXNDprtxyd71rtDPdTjzoW4XLIKnkLyu2H6deg1Sl749lw+j7uZgCzjXPtXAIo3fUC2t4hVmT/gXsuLfByYh7zxaYL1B/t93NX765ny2OcRY3vCFakwgT2eCqxj03KwllLw6k94YlURSgKk1a6CiRd2SsqN8WJOkHcWIDGNYqBL61UIv/v4APN//yXlzZFl1LIs89I3R5iYZOaaOekAR43mKwkZOI1LNDEu0TQqCaxdLxZ/NCoFHxfWQuIU0MXAkQ1izMGPxW/KnAJ73x3Gkx0m7HtfyIcnXcrHoaKHyxfoaMcpa3Kwq8LGlbPShvMsTxhjBHaIMD8nDrvHf+r0wZashZgs5NgcWpy+o0qIQZgSVLW4xMrhCJQQt4QlxBYxGevaBxvXXkQRWd0zTgcD5mRBYLsQO4NGRZxR03HD2VHewq4KG7cvzkHZxSRLp1bywq2nkWUx8Oh7hRyobefx8+KZrihDm38Rl0xLYUd5C9dsyuIp/5VMrHmfO5v/xCd7Knnqy2IofBNZqeWeHWn86+tS4k1aFuVZ+LgtRxygov8yYqdNTDq0UZFNTYLxk4mSnDTUHhnY5zOGMXRBYXUb4Z/AnuHKxwwZnJXIqREJbHKUjuo2H+iihWztFIRGduNXdCWwwnV5atXrFCtyuePwInLiTay4eyFXzErj91dO47IZqXxcUNuZFdvhRDw8VVi3L8Bi30a+pVxFlXVgE/aYYAseTQS1SU+odWhkL5IEvuZykIO9CKzLG2DlnhounpqCWddlETjkUj9O7xgxWaj9gdsX4MpnN/Krj/Yfe3AP+AJBonyNeBV68fuZvExkVsZmQ7rwUojSqfnVLefzTvAs1AWvIbdW4f/qL9Rh4YKbHuTZ+66kcuFv8MhqClc81i+lhizLPLm6CK8/yPby3otOYT+KCUlmzFrVwAlsMIC8ZwVuWc1011ZaDm/jUnMxSncLTLmq93hdNMRPINUufCO6GjltKW3mhQ1lOL0BHv80MkHfdqSF/bVtfHtxNumxghwfLY6nJERYcxOMjE80dxDaUQNbBU0qseBz9ex0dlfaqG7ziD7Y8m/A74HDq2HiRZB/BRxeJWJ3BgGyLHc4VI9YyDLsf1+o7YwWVhbUMi0tmvRYPe/tqgZE9VWS4LKZqcfY2cjGGIEdbLTVQDDIzAzRP1FwKgSTB3zCgTjvbJy+IF5/8KgSYoCJSWKSU1TfLsyLJOWIIbAef4B2jx9LVwIbXt10Won21lGuzh38A5tTIejr9Tmkxug6bjgvrC8lWq/m2rnpvV4ebVDz3zvm89srprLu4TO5IUbc0KafdT2PXz2dzT89h/fuXUzSZb/BtuD/ca7/a7aZH+ay9ZcT2P4Su/XzWVfh5cnrZ/LanQu4aX4mB4LpBDTRAzJyCkuIDTGRK7DGNCG9birZ0+99jmEMPbG3upXF4+LRqBTDFy8SIrBByzjGJ5p6PZ0RaxCEQx83ohQmgwmt7CGg6l2BRaFCuuJZrpufyxt3LiDe1CkzvmZOOg5vQMj7oNOJd5gIbG2rm0RaUElBrA3V/X9hwEc0dnz6+GOPVemR/G7iDBpoDcldexDYz/fV0e7xc03P67sxAZDI1dlHVQX2/V3VWB1edpYPfPHG6vCSLFlx6xKFEc/ky8QT067tFl+Xm2BCdfoPQQ5S9vyNTPYWcmTCdzHoDSgVEvdcPJ89yVcxpfkL3vh83TGP+1VRI8raXXykeZTSst6+EvWtbpQKiPfXkxarHzCBlUvWYPI2sjz6+6CN5r1pm3kivwS0UaLSHAnpc9HW7yJWr+KzvXW0unw4vX4eeaeAzDgD3zsjj0/31rGltPccasPhJhQSXD4zFYtRg0alOGoFtrTJgVmnIsGkZVyiiSa7lxbHMeL3RgrcreC2USMnoFJI3LFULMB/WlgLWYuFdLtgBXjtMOlSmHqViPk6+MmgHP7TvXXM//2XfLB7ANeQk436veJzyL+C8mYHBVWtLJuRwpWz0vjmcBN1rW7e31XNojwLKdH6Y+9vBGOMwA4mGg/B30+DHS+RGWcgxqBmT+UpkOtWtR287ZB3Ni1OcaE7toQ4TGDtwoXSEDdiCGyLQ/STxJk0xBo1LNKVs3D7D4TspF6sgtboxg3+gcNROj37YKP11NhcVDQ7+XxfHTfNz8SgiZyxmxil41sLssTqffEXEJ0BifkAKBQSszJjuWFeJjEX/hQu/wfRuXOp0ebwkXcuP2u+kJ9eNInLZohVt/GJZmQUNMbOGpCRk79d5CKa4yI7jybkzQDANeZEPIbjhMsboLihnVkZMeSnRA1dBdbdelRZq6fuIPVyDEun5nUa7nRBVryBJruXgC62u4T4ZMnnW44IdcwQQouHoLLLRMeYAAoVLP0x46Yv5PdXTiO6x/1gXk4cmXEG3t4RivSIHV4CW2NzkSiJe7GrKbLpz/riRn7To5LoC13rAoZ+EFi1Dnwu4k1aNG2h9xmb023IWzsqSY/VsyCnhwGeUgXGBDJUo8fEKSxdBVHxG6iMuLHdQ7LUgs8YklKnzYFrX4bFD/Yae9XZi/nGeA65rgJaiGLmFQ90e37eTY8hK5TIG/7GuzuPHiPz7NrDPKp7h2mKI8Qe+bjX8/Vtbi40FKN8ejrn6fYPWHLetukVrLKJ2AW3wPy7UBz8EOX+90WcibqPfsO0OUjOJh6er2NLWTPn/HUd97y6k/JmJ3+6ZjoPnjOelGgd//PxgV7xjCWNdtJjDRg0KhQKibQYPVXHkBDnJpiQJIlxoUW5w6OlChsy7CrxWUiL1ZOXYGJKapSQEYf7YNf8DjQmyDldVPKjM2Df4MiIvy5qRJbh4bcKIi4mjAiE5cOTl7GyQMw1L5meypWz0gjK8NiHe6mwOrlyVu8iyWjDGIEdTMRPgPS5sOqXSK2VTEuLHj7p22CibJ3IGc05vcNSPuYYEuLUGD1qpUR96FRnAAAgAElEQVR52JBCP3IIbLND3GgtRg3sWcHL/JJJrevhjZvgrdsAaDBOGPwDh2RiPftg02L1VLe4ePGbMpQKiW8vyj72vgL+jqo4ESbWAMy6BeWN/yXz7rf4H+0PWbDoLO46vbOynBNvRKmQOKSdJtyLw5mOx0DQ0Yxd1hHTR49wYnI6VtmMorH/PUljGENX7A8ZOE1Ni2ZGejR7q1tFL/dg4/UbO37zvdBahVz0BQeDmRHlwwBZcaIa6VRFdVZgiz6HxzP6/Xs6Ibx/L6y4ZUAmbAOFVvYid63AGi3wg71w5v/r8zWSJHHNnHQ2lTaLCrUxAVS6YcuCrba5SJDEvThgi1w9eX7NQd7+prBbNcrRLMZKxshqk25Q6cDvxmLSYHJWib+7xAtVtTjZWNLMNXPSI2eom5NJVNhoaPeMiniLTaXNHKpv56rZoo+uoHpgc51Gu4dkyQpRofuiJMGUK0Fr7jVWoZCYeO2v8KKmbuqd6AzdxyiiU1HMvpXrVOt54u21fLY3cvTW1jIrzvKdLJB3AzCl9etehLC+3cN8jTAhvNzxTmRDJL8Xdrwi5Kpd4WrBWPY5HwaXcNGMTJh/D6gN4HOI99YX0ucCcFNaIx9+fwlpMTrWFTVy28IsFuRa0GuU/OTCSRRWt/Luru7f39JGB3kJxo6/U2N0R6/AdhnfQWCHsA/WFwjy5rZKvP5BWNQLLYDtd8V0RJpdPC2FXRU2VjUnENSYRLTXuHNApQ19p64QHi4DVci4W3u9ZkuZlfk5caTH6bnrf3eMvP7hsHw4ewkY41lZUMvszBjSYvTkJpiYkRHD5/vq0akVfd7TRhPGCOxgQpJg2dPiS/ThA8xIi6aovh2Xd+TfjI6K2j2CnOtjsIZu7nHHkBArFRIZcYZO4wGDZcRI7MLvYWrRP+C9uyjX53Ol9t+w7ClQaqhRpCAdLdfweBG+UbfXdNucFqPH4Q3w+tYKlk1P7Z8rXM0u8LRB7pnHHJodb2TzT8/hl8vyu1WRNCoF2RYDG/0hsh528DsGJGcTVtncZxVekiSq1FmY20v6tb8xjKEn9oYmw9PSo5meHoPTGxj8Xq2AX4S9H/4SWntUbbxO5NdvIuhz8S/9HUxLi464i6yQK3Eb5s4KbFjCVrV9cM+3Jyq3Cem/195L1TFYkINB9HiRVT2kZlEpfS+chXD1nHQkCd7ZWSXGxmQOG4Gtt7YSK4nvj9Le+7NqtnuYX/kiq7WPcKCmUzXlqQ9JyONyer2mF1SiAmsxaYn1VAvZtEJMsWRZ5okvipAQfXsRYU4mNiC+QwOt+g0HXvrmCLEGNT+7eDKSBAWVAyOwTW0ukmhBFdM/I5mUnClofnyAyVf9POLzyqU/QKmQ+LP5de5/fQfrihp7jfn72sP8QLcSWWOiOPsm5nCAiuruFfn6VjcTJbFtXPtWUr2ltLp6uAAXvAEfPQC7Xu22OVj4DirZS3nG5aLNymiBBfeK9qG8s/t+c4lTQKWHqu1MTYvm3XsX8/qdC/jZJfkdQy6bkcqkZDNvbO1UMYhIHDt5CZ3tDWkx+j5diB0eP7Wt7o7xaTF6dGrFcRExp9ffr57jLw808Mg7Bby/u1pk1J+IOiVEYHe2mskIEdhl01NRKyXu/O8e1rnyAKhMOKvzNVOugqAfDg1MRnzguZuof3KpuE8ADW1uypocnDs5iZe/PQ+1UuLht0dIm1TZenj7u/DXSdB8GKZexeEGOwdq27i0i9v5VSHTpvPzkzFpI6v8RhPGCOxgIzYLzvs1lK7lIv9qAkH5hMKpRwTqCiFZZKL2V0IMkBVn6KzAGuJGjEun1eHFgJu0wn9A/uV8MvN5Ctv0eGZ8Cx7cwy3KPxHTj/c3YIRJcc8KbChKx+MP8t2l/ZgoAZR9JR5zzujXcJUy8k99QpKZNa2poj+nYEX/9uVuoU0R1ec+AWzGPFK8R4a0MjSGUxeF1a3EmzQkR+mYkSHI46D7CVhLRcwCcigjOQRZhg/uhboC7vN8n8vPOzuifBg6CWxz0ATOFlGZKV4tnqwrGNzz7Ylv/tb5b+vQxFZ5vS4UkoysHnivVFqMnkV5Ft7eUSUiS2KzwRohy/okoL2ps2oV5WvoFicBsGp/PUsVBSRKNo6UFXdsd9SIPGtTysRjH0QtKrAZOifT/HshoVPF89aOKt7dVc0D54zvmHj3gjkZo1eQrpEuI660Oll9oJ6b5mdiMWnJSzBRMMA+dUdLPWopgM6S0f8XmRI6FgV6ISYT6eyfsdDzDfdGb+Hu/93eEYkDogpfWVzAufJmpHl3opx1s+iJ3vlht93Ut7vJDpRBxgL8Sj13KD/pvaCwJ3Sv3P5S5z1OlnFufokDwUxmz+tyXz775/DgHlAdZdFfqYLUWVAtFr2UComFeRY0qs73qlBILMi1CHVKqGpcbXPh8QfJ69Kfnxqjp6Hd0y0aMIyyJlFQCFdgFQqJvISBOxG7fQHO/PNXXPv8Jhrbjy4dP1K0hx3au7lm5XT4TRz86/QBHav7ztYTNCZS6tKTFfodZVoMbPrpObx253xM0y7BgZ4bvopia1lovpk6S8y9wkka/UAwECSldTdJvipsW/4DwObQ/hZkGsi0GLj+tAwKqlqHXy3h98IbN0PpWiGjvuwZmHUr74eMmrq6nV8+M5VZmTF8Z3H28J3vIGKMwA4F5n4XspcyueBxEmhh9wBXJkcUnFZorYSkqQD9lhADZFmMlDc7xCqdwTJiJMRWh5epUhmSHISZN5OZGIUsI/L3VFpq3Op+vb8BQ6kWUrq2mlCV/n7Y8LeOLNhFeRampEau9PRC6TqxqGC0HHvsUTA+0USJ1YNv4QNihfLIsc2ctN4W7MqYo47xWSZiwomnZXBD5sfwfwN7q1uZmhaNJEnkxpswapQUVg7yAlj9XvFoTBCLN+GJ6JbnYd97PO67gdxFV3HdaX1PsM06NXFGDfV+vfAJKFsHnlZAEgt/Q4WmYtGzP+068Xfz0KgdPK6QguY4CCzArQuzqWpxcfvL2/BaJgqfiMDRMy2HAp6WTtVLktTSy5jny8JypiqOANBW2dkH66ovoka2MC27H4oclR58Lq6oeUpc+5b8BIBDde388oO9LMqzcP/Z4/t+vSkZlbsZJYERnwX7+tYKFJLELQuywGNnVpqBggHmNftahOpBGzuIvXiLHoDspTzo/TdJ/hrRGxnC53vruFv5EbJSCwvuJWPKQqrleExln3aMcfsCOJwu4j0VkL2Y1onXcbnyGxpquigHbBVQvkGo0uoLO5UWhz7BZN3LCi7g3Pwu8kxJOjp5DSN9DtQW9JYld0F+ahROb4CykLItrErpWYEFqG/tvZ/w+Nwu448nSufLAw00tHvYVWnj8r9vYF9N33NcTdlaLFI7ywMX4809T1wXj0eN57JB8Re05CwDpA4JMRBKVYjntGsepv2e3eiiLHxr+RZWFtQgg6h+l6ztd/X38OH9xGDHLytQfP0XCPjYWtbMQ9oPmPrfWdBUzLS0aAJBmQO1bcfe4VCifIO451z+D7jmRZh9K+6AzGtbKzhnUlI3RV+MQcN79y5mVmY/YsFGAcYI7FBAoYBlT6EIePiN4c0Br0yOKIQneaEKbFh+G6PvRwXWYsDhDdDs8HYS2BFQkbM6vMxUhiZ8qbM75IGby6y4fQHcviDR/Xh/xwVziqjAbvkn7PwPbHme8YlGZmfG8NB5/ey79Tqhcku/5MPHwvgkM0EZSnJvhag0+OLnx7zIG/w2XOqjE1hNinAibhxzIh7DACEMnOwdv0uFQmJZYgMP7bkE9n94jFcPAPX7hNnF0h9D40Go3YOroQTfF79iTXAWxeNu59GLJx9zN1kWA1XuEMHb/V8hJZ1woZiMDiaKVwnS6nXCxmdAqYHzfyserUNDYH0dBLaPquExcMGUZP50zXQ2ljTztwKNcGFvKj72CwcZwZDqJaAykoK1W0Wt1eXDUbYVNaIqG2w81PGctrWMBnV6n6Z63aDWgdvGhMbPedp/FU3GcXj8Ae57bScmrZq/3TCzWzRaL0SlIMlBctUtVFgHnj16MvHlgQbmZceREqWDF87hzrbnaGz3UNc2gIiRttCiQlTK0ccNBAolXPk8CpWafxme59M9nQuoO/fs5mrVBhSzbwFTImqVkh36xWTbtnQYuTW0eciTalDKAUjMR7XoXlQEid37cucxwmqNa18WZkE7XoKAH3n1rygjlfb8G9BrjiOCL20uBDxQt7f79u0vwp/Hg9PKlFThO7EvFLVT0ti9ogqdBLbK1nsRpKTRgULqVI4AjEswUW1zdUZe9QPv7aom0azl3bvnkRGs5nfPvUSVtXdOrS8QJLF1D82KeH7nu4mNcaE+4OPxxzjwEQS8HEq4ECCykkGhIDkpmTfvXsjEZDPff20X1z6/icPm08BlZcvGtWwobjrmoSoKxUL+K+rriHJXw+7X0B76kAekFUg+J+x5namh+9PeAfZ+DzoOfiyuz7lndmz6YLdwB799SfZwndVJwRiBHSpY8mDxg1wUXNfv3sIRiXAVIURgbU4vUTrVUeWjYYQjasqbHYLABv2ib3OY0ezwcpq6TPRkmRLISzCRE2/ki311Hb0uQ0pga3bBql+Iz6S9FkPrYd69dzFzs3tkDQYDkQl/xSZhDZ9z5gmfzvgksRJb1BKAs38BNTuP6dhnCrTi0xx9BS8uZzoA9sq9Rx0XEa3VJ8/F9VTEzv+M6vD2/bVtBIJyxwQB4NbgB0TLbcgf3Dt4MtSG/RA/HmZcj6zUsHvl8+x+9tu4AxIfpP2Ip26cdXTCEUJWnIEyZyhC5uDHkHsWZM6H1orB6/v3uYXJ3Bs3wZ9yBVGeeZNwNo/NHrIKrM8tJqXScVZgAa6bm8Hy2+byjT1Ulao/ue7ksiyjcAhDrUDydJIka7cK7NqDDcyUBWn1KbREOcpw+wIEAkESPJV4ovsZqRbqE26Lyef5wDKa2j2s3FPL4QY7j181jUTzMbwNspYAsMx4cERLiKttLg7Vt3PWpAQhk288SI71a0BmzwDUZkpHqDoa1b8e2H4jOh0ufZKJgSJm1KzocEg+s+5FJEkBSx7qGFqfdi4afMiHhey/vt3NRCnUY5o0hai0iazmNPIrXhNzIVmGghV40hZw1Ts27BOvgr3vwKa/IzUV8bj3ehaNP05znJCRU1hGDAi1wtd/AUcDbP0X4xPNqJVSR8WztNFOjEHdzZMkLVZ8D2tsvRcTShrtZMQZ0Ko6CfacbHEvf2bN4X6dZovDy7qiel42PcuM/0xmhe9+XlM+RtG2L3qNPVTXzgyK8STPIdti4N2qkPlWw4F+HasbCt+CuFwKZfF7zLT0vahmMWl5555F/O7KqVRYndy4Wvz21n26gluWb+nWRxwJnsqd+FHC4gfZFRyHb/Vvedj5JLVRM4S7ccGbpEVriTWoKRxOAivLIiIo7+wOlYwsy7y44QiTU6JYmHtiCr2RjjECO5RY8hBt2hS+53iOVsfIXlHtE3WFYEoGk3BhbHH6jpkBG0Z4la+82Sl6YGFEyIitdi/TKBG2/QjTofOnJLGppLlj4jAkPbAgVpsdDcKV+ZZ3xLZIvRkBH7x8CfxzKbT0MD4p/QoUashaeMKnE3YiLq5vh+nXi4WK1b8WE+ZI8DrR4cGvi4v8fAiZ6Rk0ylHIA71RlayBJ6fAv88c3Qs/JwH+QJDn15V0D1Z3NMPHP4YP7oO2oTH2GWp0GDiFCWxrFZNa1vJhYCEBWYK3v3NUmV2/Ub8XEvNZW+FjHXOYUv0mCymgZeGjPHX3MhFV1Q9kWoyUOUIENuCFiRd1LPh1KFhOFLW7xb5PfwRmf0tcu5b8QDwXlzdkvaU+jyCwCs3xVWDDOHNiIleeeyZeWUlTyc7BOLV+o9nhJU5uIYgCddoMUiQrlc2d1aLP9taxSHMYOX4C9phJ5FJDcb2d0ooKoiUHuuR+KmNMicIE8Iy/4EdFk93DyxuPMC7RxDmT++FiHD8eYrM5Q9opnJtHKL46JBYDzpqYCAdWAqB2NTJZWU1hdf/VZnpXPQEUoQzcQcaUq3BmncMPVW+zbttutm3dxJWK9dimfhuiOwmzecJSmmUzjj3vA1DX6mayohJZoQbLOCRJ4sWoe3BIRnjteij6DJqK2Gu5kJ0VNj7TXSz66Fc/RkPMTD4PzmVeztHvjX0iKk0scBd/0blwvfcdaKsWi+2bn0MTcDAhycz+jgqsndx4Y7ce/eRoQdYiGTkdrG1jfGJ3B+dFefHcND+T574qYe2hYzunf1xYy/TgIfJbvoT8ywkuexq/rEAq7R3ndaC4iAxFI4a8BVw6PZWV5QqCaqNoJRgI2utE6sLUa6hocRFrUBN1jOuzWqng5vlZrHv4LB655nRaoydxT0Y5S8fH88sP9vWpjPQHgsTa9lOvz+PCWTk86b8atauRRjmGhktehFnfgtZKpIpNTEuPobB6GIsyNbuEKeikSzs2fXNYuIPfvji7T++GUwVjBHYooTFQPf8XTFZU0rzm78N9NseHur2dkzGEiVNsP/tD02MNKCQ40uwU1UYYEU7EgfZ6kuWGDgILwpXNH5R5L2RRH6Mfgh5YEDciJLjyeWEuYBkfmcB+/WdRaW0uhX+fBUc2dD5X+hVkzAeNsffrBgitSkmWxUBxOK/3zEdF5ag8ci+s7AzJbwxHX9kz69QcUWRiaB2AXNDngpU/FLltjiZ46SJ4754RITsfidhaZuXxTw9y76s7hUkOwM5XhAwt4IM1vx3eEzxOFFa3YjFqSAlNxNj6LyRknpZuZrnlx+KmvfpXJ3SMksoasFWwojKK77y0jS/VZ6GWApCxgMzzvz+gfWVbDFjlzp4yJlwIySILedBkxFXbxOO8O+HiP9N8/Uei8goQlytMnIZAteB3CyJ1ogQW4Iq52ZSSRkvZrhPe10BQY3ORiA2fNg4pOgO95MXaLMySnF4/64rqmK0oQsqYjyppEnmKGvbXtnKkSPzfJedM6d+B5nwHfrAXY9YsAFYfqKewupXbFvVzIilJMP4C8t27qLfaBtRPejKx9mAj6bF6EcFycKVYQAEujz48IKM1k7eRdnW8kP0ONiQJw+V/RS0Fydr2PyRs/zNuSYflwp90GzY1PY4vAnPRlX4OXif1baICG4yfIDwrAIMlg18Yfil6MN+4GZQavlYtAuC92lgh/QX+13w7qdF60mOPU60gSTD/bji8Gva8Ie573zwtct6vfhHcNtjxMlNSo9hX04Ysy5Q0Orr1v4K4pyeatb2idJxeP6VNjg4Zclf88tJ8JiWbeWjFbmpbj15s+WB3Nd8xbUFWG+DSv6GYcxtlmgkkNW/rNba9WOTLR49fzKUzUgjKElZDDjQOcGF777uADNOuocLq7Nb/eizoNUqunZtB9JTzMTfs4OmrJpBg1nLPqzu7RWZ1HKq6lXxKCCRNJy1GjyvjDB723cV35Z8xOS8XJl0CaiMUrGBaWhRF9e3DZ+R08GPRBjPhgo5NL35TRrxJw7IZqUd54amBMQI7xEhdcC0bA/nEF/6bD3dX89Cbu4dfM99f+L2iV6EXge1fZUKjUpAao6ciLCGGEUFgE9tDJh1dCOysjBgSzFo+2i36coasAjvvLrjrK8gL2bznnS2Mk7pWPCu3CdnQjBvhe+vBEA//uRw++D4UfSGq4rn9cx/uDyYkmilqaBd/ZIkbMzWRJ5nuVjHxU5jij7nfJn0uie7SDhv6Y6Ht899DyxF+4LmDDRd+BvO/B3te607ex9CBzaEg9e3lLTz+6UHxOW9bLiROC+4RMtM+/h9HMroaOOF1wI6XkSYv49LT5/OHsnE0Tr4VNj/XixzWtbr7Nel/YlURDz8rXEQLfOk8fMFEfv7DB2Hpj+Cqf/btctoHsiwGbGECmzYXzEnCLdWcMnhOxJVbRSyLKZGvixqZ+7vVvLo5pMyw5ILfNSRROv5wBVZ74gQ2xqChPXoi5raikzrhq7G5SJRsBE1JECUmdW6r6I38bG8daf4qDIF2yFyAMTWfJMnG4coamivEfSIxu58EVqkCcxIWk1j8fGt7FWadqiO6ol+YcD5q2cPMQIHwjhhhcPsCfHO4ibMmJiJZS4UM/7Q7IDaH01X7Kajqn5GTPxAkNtCEUzcEcXVhxOWwO+cOFns3MMe5gW0pNyMZu9+3xieZWMkSVH4nHPqEhnYPkxSVKJI7/8+zLAbWtCYRuHo5IMPEiyhoFgsS28pacJ/3e+SL/swbdWnMy4k7sarXogcgazF88jBsXw4N+2DR/ZBxmriub/w705J0WB1eihvsNLZ7ujkQh5Eao6e6B4E9VNeOLBORwOrUSv5x82y8/iDfe3VnN5fuqhYnT39ZzPriRsqbHRQcqedc+RukSZeCVhy7MX4+432H8Di7z20NDTvwoUZKncnEJDN5CUYKvanQ0EcP7ObnRL/vxme6z4kK34Lk6ZAwkQqrs28n76Mh72wI+oht3MazN8+msd3Dj97a0+v7WrivgBjJQdz4+QAsm5nGW4EzSciaLFyhNUbIvwz2fcCMZN3wGjkd/FjM2UIKx0N17aw52MAtC7LQqYdgYWiEYYzADjGiDRrWGc4jylvP8hVv8+7Oan781h78gVHQ49d4UJhuhCN0HF6K6uy9VvyOhiyLIVSBHTkS4iz3AYIoIGVGxzaFQuK8/CTaQxfuIeuB1ZohdWbn33lni8ln5Wbxt8cO790l5EQX/VH0Ut+xGmbdIlYhX7sWkAfFwCmM8UkmypudwnZfHwOWcX0SH0fDEQBU/TDeaImfg152468+Non69zsfo9/2D94JnM4672R++F4R1oWPgi4adrw8kLfzfwabS61MT4/m24uyWb6hjB2rXoO2KrFIcvqPxcLHZ4+Oqgq229fdwIndr4lA+QX3ccfSXOJNGh6xXoasj+lWhf18Xx0L/vAld7yyvcNoLhI2ljTxzJpirs8UE63f3X0D9501Dq1WB+f8srOqOQBkxhlpIhqfUg9Truh8Inna4DgRy7KowGbMA+CNbRXIMvxm5X6xGBqqgA2FkVPAKyqwykEgsAAJebNJppk1u47DxOU4UW1zkyi1oIpK7ui3lEMGQq9uLueiqNBCQMYCFKHoG3vVfgKNxfhRIsVmDeh4Bo0Kg0aJPyhz7ZwMjAPJW8xagl9l4GzF7hHZB7u1zIrLFxD9rweFfJhJl0DuGYxz7sbucncYDB0NVqeXZKz4DENIYIGUix7hcDCVJjkK45kP9Hpeq1LSlngaTUrhRN5qbSRFsiIldhLYKanRwvk3bgncuQYu/RtF9XaSorR4A0E2urM5knczje0e5uWcYM9hyIQKSQEf/0jkx069Rjy39Edgr2OpcxUAH+0R3+FI87G0GH2vCuz+EMnKj0Bgw/t54vqZ7K1u5Y5XtuP2BSiqb+fq5zbyxKoivrV8K+c98TVnKXaj87fDjBs6XqvMPR21FKB6T6eM2OrwMs6zn+aoyaDSIkkS5+Uns7k9QbRR9SxmyDL29c/hdrYJM8ln5gjZ9tOzhDfHtGvwB4JUt7i6mVD1G5kLhcFeyRpmZMTw04snseZgA+/srO42rOXwVgBM2aKyfvG0FLQqBUvHd5G6T78OPK3M8WwBhsnIqblEVLInXdKx6Q+fHsCsU3HbwuyTfz7DgDECexIw/ewbCEgqnptdxbM3z+ZgXTv/2TQ8ge4DQg8Dp/d3V+MNBLl6Tv9t78NROp0V2OElsIGgzMRAEc2G3F4S3PPzO2+m0UNVge2J7CWin7VkjZAAfvh90c925XOCvAHoomDZU/DjQyLja8lD3arHJ4pxiSYCQZkjTaEJU+osqI7cpxas3IpHViGlTIv4fFckTTsXgIodnx11nK2+kkV7fopHaWTxvc/x3zsWYHN6+fnHh5GnXw8HPhS9nWPogNsXYHeljQW5Fh69eDKzM2Pwb3qegDkdJlwkvjtn/wwqNoq+raGA3wPLL+jbMCoYhJUP4d+/kje3V/LgG7todx89QiVs4DQtPVrIoDc/C6mzIWMeJq2K+88ez9pyLyWTvgclX0LpV1gdXn72XiFpMXrWFzdx0VNfs7Gkt9OkzenloRV7yLEYuSrNBtpoYfhygog3aVBp9Dw9+Q1YcK/I/q5pExWDxkNCGn8iaKsW1dX002h1+li9v4ErZqYSZ9Dw/dd2YjeFCNYQGDkFPeKaoNKeeLsCQOZkQcK3bzl2XNdgQVRgW1FFp3Q43kb5GtlS2szOChvL4irEYo8lT0SjAHJTETGuCtr16R1S0oEg3qRFkuDWhQMjv6h1uNOXcrZyV7c+3ZGCtYca0KoULMyNF66wydMhNgtyzkDjtzNdKuXSZzaw5I9r+O3K/R15pT3R1O4lSWpBNg2iA3EEZCbG8lj8X/i28nFmj8+MOObMScm86VlIsPhLEptCC8lJnQQ2vJhWWN0KqbOwK6Ootrm4fm4GOrWCr4ua2Fom7k/H3f/aFTGZcOkT4t8L7+uM4Mk5A1JmklHyOpIEH3YQ2N6/zbRYUYHtWl3cV9NGlE7V4VKM19lrcfOCKcn85drpbC5r5tYXt3Lt85tELPZ9i3n+ltmcm5/IDxK2C1+U3DM7jzdd9Lc7D3US2D1H6pkmlSGnn9axbWpaFIeCIUVCDydiT+VOTI5yHvN+i7XzXxDfK1ul+I6d/XOYdze1rW78QXlAEuIOqHWiWhlq2bptYTZzs2L5zUf7aAi5Z3v9QXSNBQQkVcd3IN6kZe2Pz+S7S3I695VzBphTiDu0YviMnA6F4p8mXgzA+uJGvjrUyP1nj+u3T81oxxiBPQm4ZH4+yryzSK3+jIumJLF0fDxPrirqbr4yElFXKOy543KRZZkV2yqZnh7N5JTIK3iRkBVnoMXpozWoB4Vq2AmszeFhhlSCNbY3AVuUF49Zq0KpkDAPZNX8RKA1iX7WkjXw1e9h33tw3q8Fse011gyzb4VzHxvUvqEJScLUob6PH+sAACAASURBVKg+JCNOnS2MAULRE12hqtrCXjmHhNhjfweWzJpMMRl4i7/qe1BdIeqXziVHqqPpvKdJTkknPzWKH543gU8K61hrukSY1+x5/Xje2imLnRUtKANOvtP4JzSfPsTyjM+YL+1jbdRlQsoIMOtWMdHY8crQnMShT4RyYFMf/f2la2H7cnxv3s4/3/mUD3bX8M3ho0cYdDNw2v6S6O084xHRHwbcOC+TzDgD9xbNxmtKg1W/5FcfFNLq8vHCbXN5775FGLUqbnlhC/9Ye7hjAu3xB3jk7QKaHR6evnEW6qaDkJTfsd8TgSRJZFqM7LMbQaHkX1+XcvHT66k1TAA5cHyOm11RKSoCpJ/GRwU1eANB7liayzM3zaKyxcUv1rSAUis+q0FG0BsmsINTgVWkiDzxQN1efvfxfq55biNL/7RmSO+FtS124qVWJHMymJKRkUiRmnn8s4NoVQryXHvFNViSIDaLgKQiM1hNrlSHHNdPB+IeyE+J4tLpqWTHD5z4ayZfSLrUhL1yCHOEjxNrDzawMM+C3t0gVAGTl4knck4H4PnFdn5xaT4Tk8ws31DG69siu722tDRhllwoYwfZgTgCHrv+dH5928V9Oor/4NwJ+KZei4IAl1lfFhu7ENi8BCM6tYK9IbOe4tB9cmpaNAtyLXxd1MiWMisWoyYimTwuTLsG7tsmCGwYkgQzbkDZsJelsTbKm52olVJ3OW3pV1C2ntRoHR5/sJsMfX9NG/mpUULibKuEv06Crf/qdegrZ6Xz+yunsbXMSpxRwzv3LGJGRgwXTk3h2SuymdS+WZxflzlIWqKFvdIEous2d2yrObgVreQjbmLnXCY/JYqiYGjRsAeBLVr9Il5ZyW7jUh7dHYf7lo/g3o1w3Stw+sOg1nWoEo5LQgww7jxoOgTlm1AoJP50zXQ8/iA/e38vVS1Olm8oY7Jcij16Iqi0HS9LjdGj7pq8oVDC/LuRSr7krtgdw2PkVPqVWHCLzSIQlPndxwdIj9Vz26Lsk38uw4QxAnuykH852CqQ6vbw68um4PYHRN/aSEZdobiQK5TsrW7jYF07187NGNAuskJROhVWl3DeHWYCa68rIkZy4EyY1es5jUrBOZMTSTJrT657W95Z4rP++s+CoC7qLXUaSuTEG1EpJL46JPpbSZstHntWYX1uolr2UiBNIr8fixhalZLG+PlkOQpot0cISi9bD8svwO0LcL/+D2Qt6JRf3n16HnOyYvnBGg/elNOEjHgUSWGHGptLrVyu3ERK2buw7z1idzyNW2HgZ+UzO004lCohdTq8CuyNg38SO/8jHqt3RMz29Gx8jiY5Cq+k5f3EFzAqfMc0eSmsChk4ad3w1R/ExHjChR3Pa1QKHr9qGk1uiZ/aLofaPczd/weez9/HZMdWpiSb+Oj7S7hkeip//vwQd7+6g6dWF7P48bV8sb+eRy6YxNTUKBHlktTP3sZ+ICvOQHmzA6fXz7/XCyK5xRWanJ9oH2zVNiF9S57GOzurmJhkZkpqFKdlx/HtRdl8UFBHIDZ7SAmsRjdIE3NTEkG9hSnKSpZvKMMbCFJpdfFeDxnfYMLeUo+SIJiSQKXBr48niRZ2Vdi4cYoOpa1MxB4BKNX4orMZL1WTLdVhTp10XMd87pbZPHX9zGMPjADNZPF9j6nq7eo6nDjc0M6RZidnT0rslA+HCawxHpKmkdS0me8uyeGF2+ayMNfC458ejLg44WwSPcja2BNXQBwLE5LMzMnqO/ZNqZB44Ppl1BsnMV5RjUsZJfrXQ1ApFUxOieqoshXX2zv2e/r4BEqbHKzaX3/i/a89kTCh9wJb/uUAXKvfAUBmnKGTWLWUw+s3wkcPkhYrCF7YiTgQlDlY10Z+SkjV9cXPwNPa5+LmjfMyee/eRbx376JOsmhvhA1PiDjELvJhEIt4FdFzSXUXCbMrQK4Q8lptTmdaQpbFiE2TiEdh6NYH6/R4Saz4hEL9PB67fjG1re7OHv8uKG92drzv48LsWyE6Ez56EPwechNM/Oj8CazaX8+SP67lj58dYIbyCPqs2cfe18L7IWMBt9uewVFfenKNnAJ+qNjcUeh4Z2cVB+va+cmFk7pFJJ3qGCOwJwuTLhFuYfs/IDfBxO1Lcnh3Z3WvPoURA1kWpCokH16xvQKtSsFlA3Q264jSsYZkxK7hNXHylYuMNTk1sgT315dN5dU75p/MU4Jx54jHnNPhkicGpSo0EOjUSr67JId3dlbx5YF6IdmRlKLvpCtqd6OSfdgT5/YrBxggecb56CUv2zb0yIjzOuD9ewiYU7jU9WvGzVjU7eavVEg8cd0M/EGZl9xnQHNxn87IIw5l6yHUYzdU2FLazO26dZAwGX5yBH5aRdMdO7DK5u55fjNvEhOOwrcG9wRayqFkrXBelRTCNbMrmkvQlq7i1cC5tF74DObWQ/zJvOKYUqvCsIHT+r+CqwXO/12v38P/b++u46o8+weOfy4O3ZKCAQYIdnfrzIU5dZuuw20udHset/3WexaufZ4Zm0sXzjlr09mB3YEtFoIoqHRzzv374zoKKCggiuj3/Xr5Ek5e6M197uu6vtG+rg8rxnXBqflwVlsaM8p2KT0Ovwc/D4b983FxsGXi8Ka8cWd9VhyI5/Nlh2hYzZ2fH23DY51qQfJJ3Y/ar365/XME+Thz8nwmv2yM5nx6Do52Nqw87QQO7tdeiThmCwQ24+j5bHZEJzGoebWLvytd6/liMSDRscb16QVrDX+2cyx53YMrUgqbqg0YEJDIzjd7Mf/ZjrQIqsKsbTHXrepuXrI1ksRNp4go90AClP4ceriqtf1QgYgXO/8wWtscwEnlYOcXUqb3VEphU4IewkVyD+SobR3Czy3VvcBvEt+vO469yYY+Dfx1tXO/+uBbYIJfu4uOFsjNRCnFfwY2JDvXwnt/Xx6BkHEuBgBX39ItiF8vSin8Oz4IgCmgwWXnnEbVPNh3KgWLxeDQmVQcbG2o4eVMl3o6LzI1K698woevxj0QarSlXVYEUCD/1TD0pCw3A84foaZtIsDF68tjZ9PIyrXoAk5HV8G+efr/Ln5vsX2Zm9WsgqezvT7XT+sJn4ToaJtaXQoV9rwgu0ZHTFjIOrKGk+cz8E7cRZKd/8WwfdCf62FV3Yk21ShUiXjpP3Px5zxV2oygfR0fOoX48NXKKFIKpJws33+GjxYdwNfNgQCPMlZ6dnCF/p/qXdi1XwDwaMfaPN8jhDfvqs/yR4JwJw37GiVI0TLZwqCpmBR8YvsV+2MTyzamsji9C3JSIbgjOXkWPl96iCY1PLmz8fUNyb/ZyAT2RnH20hOUffPAMOjdQDe7LknBgwqReFyv0FVtRFaumXk7T9G3YdVSFzcq3AvWu8KrENvGbSXDcMAxsOiLVw9nO2qXokhVuQhoCsN/hWG/lCnfqjyM7RVKeIA7/5q1m4RsE/iFX7YDm3JoDQBVwooIby5GrZa9MGPD2T3LC9+x6kNIPsnqsDeIs3jSv9HlJ94gbxde6x/OZ3ENybZ1qxzFnNLiYfoA3Q7oOsnKNZN5cheh5kPQ4kF9seXgRvXAQEa0rsnMLSeJtq5U4xeuj69dv5bvIHb+ggF8ZzOY1GqdYPfMwm1cNn9DHrascutPjdb3QLtn6Z+9EOPk1mInKhcKOHX0ToVNU6Hp/RDQuMjHejrb896gJtQY8w9JY0/CC5E6p/WI3rVSSvFIx1osH9uFFeO68MPDrekY4qMnfmesVcj9G5bbP0eQlws5ZgsTlx+mbW0veob7s+l4EoZ/A4jbVfYXzsvWz6/eitnbY7FRMKBAVdtmNavoVmUWf0g8Vu6tdNySDnDecMXeqZx2YAH8G2J//iDu9vryY3Dz6hyOTytVC5aSyso1Y59p7W3pqj9zTZ6BBNok0rCaOzXjV+rbA/Ijcky+oXgoa/6pd91yH1NJHAl9jNrmo0QvuTna7yWkZvPHthgGt6iGX9Iuvbjd+vHCE73aXXULr/ljIH4/tX1deaZbXebvOsXqQ4UjQE6e0Itszj5F56VWiIaDQZmwD7z8nNOwmgdp2XkcO5fOwTOp1PVzxWSjqO3jcjGn9IZMYAEaDMQn/TB1VGx+BeKdv+iUjZaPAFAzeQsmG8XWE3pSdeE6s76/E/zzb13R/P5ZeqF698wrv9/qj3Q7xW6vwuMrYeTcIh/mG9aBLMOO1O2z2fLjeDqrnTjUanvZ48ID3NmTG4Bh7QWblp1H7s7fyVKO1O6gC1a93LseiRm5PDBtE2/N38srsyN59MetVPN0YuaT7YoNBy+R0F76/3rNJxB/AJON4sUedXi46jHqbHtfPyawhNETVYJJ6fEhrW0OkrJmStnHVFrHrYv5QR2ZuzOWuOQsXuwZcsv3fb2UTGBvpPr36DCvM3uoZ807vCHlt7NS9Cpaaez5U/8d3JnFe0+TmpXHva1Kv1rqbG+Lr5uDtZBTBYcQm/Pwi1nCOksDvN3LJ6erXCild+gdS55bXN4cbE18Obwpqdl5/PvP3RiB1sp/BSYbaYfXccQSQPPw0BK/rnKqwjnXegSlbOXkhaqap/fAhq+g2UimxwZQzdMpv+rsJe5rXZN29aoxM6cDxt55FVPMyWKGjVP0n6vZ+Yve8Ty0WBfjug52RCcxmGWYbeyh8bBC9z3brS62JsWXywuE9Da9T19wnt5TPgOwmGHHz2TW7Mo7a1L47Ewz3Ts4er2+PzsVY8d0Flja0qJBff2h2nU8eSYn+uYt5+T5oqNO9lsLOPVKma13dbv/31WHUtvXFU93d134JLijbnZfQLCPy+ULUqe2A0pP7stJsHWhLjU7jzHdQ2hTy4vTKVmk+rXUFb2zyniej9sN5hzyAlsye3sMHUN88Xd3vHi3q4MtDQI92JHuBXlZuuBTebGYCUhYyypLUxzsynFhzb+B3ilKPA6GQf8wdxxsbZi1Lab83sPqdHIWvkqHNObvwFYj2D6JD+4KQUUth7B+hVsn+RQ4v3nXKfcxlUSHex5nE43w3vQRpJ6pkDEU9OP64+Rac6/ZNFUvFl1y7qF2N2gzGvb/DZPawvRBjK5xnFrezny65ODFhyWkZNI4cSl5Ng66yu7Nws0fHvxLV3C/xIXPpz2xyRw+k3bx+k1X1vXH182BsKo36PO7/t0YKIY6baNtbW9IjoXFr+r2O/0+AWdvnGLWcWfjAGZsjiY5I5d9cSnYm2wIif5d5572+QA8a+jor8hZxS98JcfqCW7zUboWQbXmxbYZaxTkxxZLPXyPzmZQ8o8kejfDqdcblw8/0J19uYGotDOQcZ6lkdH0MDaSXqv3xcKajat78mq/MCyGwR9bT/Lb5mjua1OT2U+3p1YZ8sov0+dDXd9lUht4xxveD4TpA/XEsN2z+X28S6BKmweIsquHz+GZTFl9pMgF2tikTDJzyjGa4vha8A7B7OLHlNVHqB/gTpdQ36s/7xYjE9gbKfwuXWxj/nO4GOkEeTtz4PT1n8DmLXoNy9SuejW/JCxmvdtVqzP41GVhZBwBHo60LWOJ+OCLrXS8K3YCe3QlztkJzDJ3uX59XiuxUH83XumrS8uvzwzSIZyJx/WdhoHH2e3sMYURVtWtVK/rVK8bzdRhpiyLJDcnS+9OOnmS0ul11kadpV+jqsWuHCqlmDC4MfNMPVGWHM5vyM/Zyco1syc2uchm5CViGDoP59wRHRlQ1Id4wkH4thcs+jcsfuXKYZoWi84p8m+oizxs/bZs47qKbVGxDDStwxx2d357Kis/d0fuax3EvJ2x+X0AGw7Rla7LqxDWkRWQEssWrzsBmJHamGwbJx1GnJUMqz5E5aTxXW6v/MreDm6k1urLXaYN7Im+vDgY5BdwCkjeoVselKBVUyG1OutdyKSiC8dcdPAf3ZKmHBeMalonsE1reNK+jvfFdho77JrqQk4n1pfthWN0Aad/kmpwKjmLB4uoatsyuAprz1sXgMozDzZmK465SawwN8PBrhwvFS7kHs9+Aj4JwWNiKA+E5DJ/16kS55EZGYklCq89cDoVX6wTWFfrsegWgH1uCo3SN0JuOtTrX/hJ1lY62DpV2ATL2cGOE23fxdaSzdk5L1fIGC5Iz87jpw3H6VXfnzoOqboqfLMHLqvij8kW+n4IY/fpxacze7D/bQiz1UtUP7WYPTH6/yFu/jt0tonkbMe3dGXYm0lwB3D1u+zmun6u2NvasOHIOU6nZBHin/8ZOL5vGP883+nadgVLwz0QVbMdT/nspovtXvimm67YftdE/blTqzMcXc1TnWuTnmNm+sbj7DuVQoifC7Zbvoaa7S9WrqXRvbr1WvSGot9r02QwLIWLSRXD182BXx2H87+8e3jCcypVn1kIPpdHMNQPcOewYc19jttJ0MrnqKLSqNJ2VKHHPdG5Dn+P6cSet3uz9+3evD+wUfn1NnX104sV3f9P1xxp+QgM/RFeOgS9/1OqXuDKxoagbg/TwOYEfy5axvg/IzEXqL4dfS6Dnp+u5qNF5VPzJis7R/9/BXdk6b7THE1IZ3TXOrfd7ivIBPbGcvGBod/r3ZDpA2nqqzgQl3rd3zZl3zJsss5jRC2/+oMBDi/VeWItHyU7z8zaw2fpFuZX5pyeml4uOqTxQghxOYe5ldjOX8gwebDZrtVtleheGg+1D6ZnuD8TIq071NY8WOPsIVzMyaT6tij1ceAW1gN7ZcZv9xTiPmwBMZs53Ow13l1+mlyzQb8iwocL8nN35JWHhrCLUFLWfsO24+eYufUkYyZM5eCU+3nkvUk0eXsJr8/dU7r+ysvf1iuw/20OE2rB5Hb5hSUMQzdVn9JJ99fs94meBK7/b/GvdzxCT6A6PK8Xq7ZP160KylFOnoWcyNm4qwzsWz9c5GMe66TL/U+zFhPCxRtCe+uVdPOV29hclcUC674EZx9mpTUkwMORR7o14K/cVlh2zdBN6Df8jz0u7TjpXJ+WwfkTbNc2o3BXGeTs+bvIl46MTaa6sxm7c/sv9jwtFWslVI6tKf4xSSd1UaULF3DlJNDDifva1OStuxuglCLEzxVPZzsWJ9fUBZiOrirbCx+LwPAM4tMNyTQIdNcFdC7ROtiLQ7nWyVk59oI1H1xEHjZkBXUtvwtH0HnbLn66NVDtboDBI7aLSc7MZfn++Ks+PS01mZQJjVj/2fAr9vwFmL09hiD7VAwHd7Cz5s1Ze8GyZRrYu0GtToWf5G3Ne/WuU6oL2fJ2V/fOTLcZgM/ReXrHpYLM2HKSlKw8nupSB7Z9rxcOWj9W/BOcvXTV2BciYcBk3B3tmGQ/EduZI2D7dBpGTWaRbTf8uz51436Ia2RnLeS0YHccAKH++VEdjnYmfFwdinvq9dFgIMTvg58G6JZpjy3LnyzW6gyppwi3T6B7mB/frTvOnthkengl6M+nJsPyQ7/D+oGdC+z+/fL3yEyCrT/o9yppL+Sg9nxmHsaYIX2LrZMRVtWdKOsE1vjjIZqnr+Gfas9hE9qzyMcrpUrXS7mkAhrr47Tnm3pHusGAMi+o2DUegqFMvFt7H79vPckXyw4BYBgGr8+NZLwxjfgdC8gtzfVJEdYePsuwd76G7BQSfVszadURgr2dr3oNdauSCeyNFtZflwWP2834c68Rey6pfEMLLpUcg1eOLiiTtr2EhVy2fqvzgsL6s+VYIuk5ZnoUceFUUsHezpxOySLb3lPvRmRXQM+szEQ4sIAIx274VyndDuLtRCnFp0ObkOgaQjZ2ZFmLXp3dr0Mz3UM7XenpRavZFmxsed52NjaWXB7JeYk7llflz+0x9GtUlSbVPa/6Ei2CqhDQYzTBnOKjqd/zyazVTMibwGDTGuY4vMkM5wls3RTB8zN2luxD4tASWPu5XoEe+DX0ek9HB3zTXVfX/W0ELBqvK0Q/s1nnezUdoUOEi2gvBOjdV0dPCL8bWj8BWUn5xZNO7YSoZddUSTk7z8xzP62nd8psUl2CdchYEQI9nbi7aSAzNp/M351uNlI3jz+4sMzvD8DmqXB8DZZur7H2aArt6/jwfI9QIjwHcdziz9l6I8h9eBkj0l+gZ7hfoV0JuzpdSLDxJThmXpEvHRmbQn+fOJRhgeplmMD6het+npeEERdyoXdeWP/iH1MGNjaK9wc2omkNz4vftwzyYkN0ht5NLssENisFjqzgiHdXjp/LYEz3onOcWgZ7cZoqOiQz4dA1/iT5UiMXsM0Syv1dis5DLjM7Rxh3EF7cC4O/gYZDCDw+h7puZn7bfJXdc2DN3Gl4kEr7tCV8/PmHRfb8BZ23ueJAPE08s3QLnQsu7OwfXwMhPQu1ywD0zrxHjcKhxBXAyd6EbdeXSDA8OL/8ywoZQ67ZwrdrjtK6lhfNAhx1a6uQXlCS9kK2DtD0Pkyj1zLX/1lqJm+D+c9y0FKDyKZvoipwcaAsGlVzJzU7D8hvPVdh6t+jz3XN7ocnVhWuqF6ri/772CpGd63D+fQcEjNy6WHZoFMzwu7Mf6y9iz4X7pt7eYTetu91oaAOJe+KMK5XKF+PbKn7eBfDyd6Eg3dNMpUzRk4Gz+aMwaP78yV+j5uSqy+qTjfapi1neItq/HdFFCsPxrMw8jSZUWt40HYpD5r/ZO1V2shdSWpWLv/+czfdHPQ5/q6/DXbHJPNklzo3bvf/JlO5ziC3irD+MGgqAal7GGwTkd9/8zpIPaQv5nZaauN4ZNHFqpLFSjyud2CbjwKTHcsPnMHB1ob2dXzKPIYLJ7MTmdbVrYoo5LTnTzDn8FVSa3qG+9/4969EPJztmPhAa/ZbgojfsZDzR7aRcnAt5w1XGjYuQXW+Szm46lCdHm/gOnYr7fvezzejWrLzzV5Mur/kO7p+bYZjcXDndb+1LK72LZ62OfDYCuj5NuHGUea4fMiuPbsZ/fN2svOusCiUHAtzngT/RnD3f/WKdPsx8GQEVG2oi5BELdN5MiNm5IeUtX9O57dunHz5a6afhf1/6XxTO0c9afFvqKs2/joMvu6iq+T+em+ZKhRn5Zp58sfN3HPsLRrYnMCt/7tXrFb9VJc6ZOaa+WmDNfc95A7dPmDLtFK/90XxB2DpmxDah30Bg0jMyKVjiDf2tjY8N3II99lPpO3O3ry4zkRqlple9asWfr6NDXt9+9EoaxuW5ML/Blm5Zg6fSaWjg3XXuHoZjjOl9O7DsYjiFwoOLtA7bD5lqy5bGm1qeXHsbDpp1TrqipvFLXwU59BiMOcwMa4+YVXd8sOxL+Hr5kCwjxt7HJvD7hmQfe2fJ0bSSTxTDrLLqe31ya2ysck/fts+hcpN592a21kbdZal+4rP+YxLzsQn6g/i7aqT4duUV/KmMnbaP3y8+MBlC1dzd8SSZzEIckjNDx+G/B1YKHwxX9CIGXpRq4INbx/K36oLHieX6wJxN9hfu05xKjmLF1rYwnd99CJYu6dL9yImW6r3Hccd2RP423Uoj+eOpUfj4Osy3uupYaC+jnGyM10s3FRh3Pzh5Si456vLQ7m9ausFmGMRtAr2oqW1hVDo+ZV60dPlkmu5Zvfr1I8dP+fflpOuP+dqd4OAkueD1vVzo2cx56mCwgM9+D/bsXxRYyLrHDvTOvgGFcC6nhoPg+STvN08jfAAd178fSdv/bWXMW66sGArm4Os3lL2gn4f/HOAuORMHqoWS65nHRrUq0eDQHcGNb/+vZRvVjKBrSgNBpFVtQVP287jYGwJ80KzUuD3kbD5mxLv5CTtX0WK4cRneUOxM2foyemVbPtBX1i00CXlLzYuty97CFnzoCooBfuSrGEgFTGB3fkriW6hRJqD6NOw6tUff5trWsMTc+PhVM89gdf07tSJncseUzi1ylqhueOL0Gkcnu7uPNapNnfU98fdsZR5yPbO2DQZTqPklVQ5tx1190Q90en4Ajy6FEcTzPWdypr9J3nml+3k5BWxE2sxw5+PgTkHhv5wMWQoJjGD/emu8NAC6PMRPL4c2o4uPEn0rqNXvrd+pz/wC9rxM1hyobn+vUEpvQt79pDu19b9dej9gQ5v/aot7Jtf+Pmn98CWb4sMOY5PyWLE1xvocuxz+pq2QO/3of7dV/ynCvV3o0eYHz+sP0ZGTp7OjWr5sJ7cJRy84nOLlJcDsx8DBze4+7+sP6rPWRcWtur6ubH4hc70axTA37vjcLIz0THk8kWvjPB7MSmDxI0/F7p976kU8iwG9XL3g089cCq+b+MVWcPnisxVzkzSoZhh5Rs+XJwLVUm321ovAI+uLt0L7JtLlqMff52vxrPd615xoadVcBU+ybxLR5psufbc6yPr5wBQo82AsreDKamAJhDUgbZnZxHu58yb8/aQbt3pSs/OI+JQgj6GgR/+WkErtR/7liNxHvYtbnYWpnv9wKSVhxk6ZcPFQnGGYTBz60ma1fTEKfts4QnshR6fNrZQt+iwRao2BI+KvzB0tDNxPuReTJjJ3VHOlcSvwjAMpq4+yiiv/bRbOkiHnw7/VVcbLqUWQVVw9qvFs2cHkutWg6YliLq52TS0FnIK8Xe9/r8TJVHcAubFhbw1YLHw775hDK6RilNy1MU+soXU6qIjXtZ8lr8Lu+5LSDsDXcdfl6HXD3Tnz9T6fHvMizvq+5e4Ld9NrV4/sHPGYd8fTLq/OWazgSktjo65G6BeP2wwsD/09xUjLg3DYMORc3wdcYSxM3fy9C/b+HbtMX7fEs2vm6J5vGMQnvFbsKvdkakjW7LguU63dTrcLXDUVFJKYd/9Faqrszjuu0oZc9AXkL8/oAsoLHwJFozTzYyvwvHUJrYbYZiDupCkPGDv7OIfbM7VF+KhfcCjOkcT0jh+LuOawocB3B3tCK/qzraz1sOtrIWc0s/qAkDznoHstJI/L34/xG7jH1N3anq56F5o4qpaDHmZ6Ed2MNXjef42t2F/9WEVXyigxcP6wrPNaGg0JP927zowcAo+Kfv4u+5fLNsfz3O/7bg8w7fq3AAAIABJREFUnHjrd7pabr9PLuYMmS0GD32/hX4T1/DZ8qPktXqi+FXnDi/oHqJrPsu/LSkaIj6BOt3BLyw/D7fZAzD8N50L1vklvXMxep3e/fvjQdhlzTs6uhq+6w0LxsLEZnqX9NwROB1J1KaF/PnlC4yPf5mHbRdD26dLVFADYHTXOiRm5NLr8wgmrzrC+XrDwGRftgnOqvd17v7d/wVXP9ZFnaOun2uhirgeznZMHNGMKQ+04JOhTYrMm6wd1pgtllDsd/9SKBd+6b4z2NqAT3Ik1GhV+vFdcDEPtojJYtQyvYN+adGe66RBoDvO9iaWJ/rrCXlpwoiz0zCilrHY0pravm70bXjlHKdWwV6szQwmvUYXnad9jbnXqZELiMGPbh3LkDJQFm1Ho5JP8r8WcZxKzuLzpYfYdiKRfhPXMOq7zbR9fzmvzonE7cBMLNjg2W4U+NRF9X6fkPStbAz+hjMJ8fT7cg3zdsayKyaZw/FpDG1eXVfxLRhC7OCq8waDO4HTzT+RatO6HVstoWRv/umaUhBKa9XBBBLPRPNm1gRUlWAdoVLG0HulFCNa65Y5vRtUvTkmgKUU6u+Gva3NxQrEN7VaXSDzPJzZQ6tgLz5tGA2ooiMOlIJur+hiTjum6zoB676EBoN0+s91UD9AX4Ol55hvnQ0FB1f9+7F3LrXc4cdHWzO98V6dEtPnA9KrhNGL9SzbX3yEyefLDjPim428v/AA66LOsjsmmXf/3se//4yktq8LY8OSdApe8A06L9/krkNmtCgpm5CeHLYNpU3sD2B+qfgeoBYLzB2tL8rumaSbMK/70roi+lvxiedpCfhmHSfGvQet6vjyV0wrHji4CJWdpn/ZLnVoEaQnXNxFWnFAhyx1u8YJLOgdgvXblF4yKe0E1mLWE48V7+rQFsMCMdtg+C8la3Ow9gsMWye+iG/GwE7FV7wVlwsOqsUTL7zNqoMJDLwZJv7+9XXunGsRYUph/aDTOELWfMo2n0hWHKzKzK/bMOyxf2NrZ6dDOJe/o8Oimgy/+LQFkXFExafRMqgKE1dEsTbqLPWqurEnNoUT59LxcXWgqocj3cP8eKxTU51Puu4L3Ssu/B69oIIBd35BWnYefb+MoEl1T74Y1hTbS3f7vOvAg/N1WPGcJ8k6tgH73b+Q4RbMzsZjCIn6Dv8F4y4+vK71T5ZPfWj0CnT+V4n/qVoGezFtVEumrT3KR4sOMGmVLRvr3YXLrt+gxxtFnwOKcmKDbvrefBSE9SMnz8LmY+e5t2X1Ih9+pQuSur6uvEwfPk+fqPOuGg7CMAwWRJ5iUFA2NnHny5b/eoFXbXCvrneaWz1a+L4DC8DFF6q3LPvrl4KtyYYWQVVYc+Q8Rq0uqKOr9ASkJOefw0tQeVn8kt2MMX1Drprj1Moagreu2iP0Ovmgzl8rZqEjPjWL53/byRt31Sc8oPDvtDkvj7/mzaB3xnYOBQ6guv0NukSo1w88a1LnwBQeaPkF3607xnfrjhHg4cQnQ5uw8kA8MzcfZ51DBOba3bFxt1YHbvEQWPLwXzSeiCoxjFX/4vkZO6nq7oijnQ13hrnCoszLzxcDpkCV4Bvzs12jtrW9eN/Ug5apkyFm67Ut8JTClNVHGO88HxvMcO9P1/zvNbh5dVYejOe+NjdR79dSsLe1YdqoluXTxuV6q9VZ93id9zQMmgb75unJaHGV3Wt3gxpt9MLshUiRO96+bsOrb72WcHWwvab0tJtOi4d0W6Jpd9B88DcQM0tvCFUJxqnpEFqufI+Xt+7kriaXVzefufUkE5cfZkiL6rzWL5wqLvaATpvYejyRJtU9cVj4gC6GeoOiiG52sgNbkZRifY1H8Tefxtg1o/jHRXwMe2aR1+0N3ohuQkTQGLjrS93OYt0XxT4t+6iuXGgEtaNFUBX+zmuHysvUE9Wi7PhZF2+yhlWtOBBPqL8r1atce8/UlsFenMqxvs6lE9jcLL2LVVx14hXv6l3ngKYwej08MFuHt3zdTYc8X6my6rkjEDmTwzWHEW9xo99VdjLE5ZRSdAvzw8/9Jml54Fa1+ElAt9egxxt4+9egv9Me7j/zMVFf9MOcngiLXtEhUv0/vfh8s8Xgv8sPE+Lnyswn2zFxRDOi4tNYGHkaDyc77m4aSHigO4kZuby3YD/TNxzXz6/eGuaM1oWejkXofLkqQXy9+ggnz2fy9+44/jVrNxZLETsm9i5kD5vBEY82OO78nu15wbQ/8xIPrPWmzemXGJHzGq+rMUz0eYOZ4f/j/DMHcByzQYdzlbLwSc/6/sx4oh0LnuuIYcBH5zrqHeTIEkR9gE5bmPOErkLZWzd53xGdSGaumfZ1S3/hYWuyIaZqL06aaujzmsXC7phkTp7PZLCfNS+2+jVcoF8Inzu4EKb11CkX85+Df8br9InQPjqc+gbp3aAqRxPSOeXVRoc2nz189ScBxv75JCpPzlZpyp2Nr37OCvJ2xt/dgdkJNfTq/Lovi821nrF0A8+cHMtPs2YX6lmYuuJzUt+vy4DIZ8BkR72+pcx1vBY2JrjjXYjbxev2vxLi58bg5tVZ9EInhrSozlf3NWPz4Fz8OY9di5H5z1NKF1gbNR+77GQmZr/Ovzr5ciY1i/6NAnHPtX7WuF2yqBLWTy+GVQK2JhtoOJB0w4HcrT/CsTUYMx/CPOfpEkVhlcX26ETiju9ngLEc1eIh8Kp1za/p4WzH9Efb3Lh+qddB51BfanjdRD3ki+MeAMN+hpQ4mNoZzuzRxQWLY+3VTUqsjvJrP0b31r5O/NwcqebpxB31/cu3wnlFC+4I98/SFdandNIbQq0fB8Cm4SAAPI8v5PCZVEiO0X2+gTWHE3h1diSdQnz4YFAjqtjl6k2bNZ8R4ObAXU0CqZmxB44s110OLs17vk3JDmwFswnpze4jU2i0YJxuMN1xrG57cUFmIqyfCPXv4Z3EXvy08QSzt8fy95ihBDdYrSupNhle5Oro+X2r8DAcqFG/HU1reLKVeiQ7BOKxYJzOAwouUMU0JQ4OL9EhkiZb4pIz2XzsPI92uvYPLtA7BOk4Yla2mDLO6RzCiI91TtrpPTp/0NETqrXQoaFNRuiT6rE1even2QNw9//0bb71dOW92Y/DX8/rVcNOY3V4zKUFCiI+BpMDk3P7U83TlsZXqI4nbgE2Jug0DjqNw9kwWPHLx3Q8/CFpn7fEI+8sli6vEG9bDV+LgclGsTAyjsPxafx3RDNsbBR3Nwmkf6MAbBSFdurNFoMnftrKW3/to46vK+2H/az7722eqkOHWzzEmZQsvllzjDsbBxAe4M7Hiw/iYGfD2Dvq4eumK53mmvWE7bU5kRw78xRvhvTGt8UAfvSqgreLAy4OJpzt++JoZ1OukQINAj14tV84r87J5QW/cLwiPtG7xwXPNUU4O+tFvJNjUI8s1vmvwLoj57BR0LZ22fpCt6vrxyexd/Nl/Fdw4C8WHA/FzqRoyiFwcAffsDK97kXtx+i/U2J1+kBWMuRlAUahnfcboX+jAN6av5c5KaE8CzqF42p5ZbmZmA8u5u/c9ozuX69E+WFKKfo2DODXzdGkP/gvXH4dAJ+F68WA5g9Ccz3pS8rIoeauz+ho2ktgwgcs292FO5rUImnHPDwj3mK90RBavE67PiNQ9jf4Qr3BAIh5FocN/2PxwFZQrwls/kL3F046iZc5G5y8oF7fy58b3AFGzkF93ZWnzdPp/9KH+nduhzVc3r1i+rmWl97N6rJgR1vu3TUddk0nA2dcyCDHbMZ+0ORya/eTk2fhh/XHmLg8io8cZqNMdrrViKh8wvrpaJP5z+lF1qLyXwuq3Q2COupCnh1euO7DmzW6Ha7Xoz1ORQvpCU+tgVmPgjnb2ioM8K5Dtk9DBpxdx9L/PUst24WYMPOjz1jeiWlGqL8bkwYFY7fibb0xk2XtX52eoBePV32od19bXaGF1W1GGTcwp6I8tGzZ0ti6dWtFD6PcbDl+nmemLGRe+HICjs/VPbkGT4N6ffQDIj6GFe+xtMtsHl+cxeDm1Vm2/wzVPJ2Y/UAQjlPa6qIKIy4v8JDwcSsOptrRaPxqPJzt6PNFBA2cE/k0+11IOqHDkRsP1Q9e8yksfwfj2W38ftSe/yzcT06ehbnPdLgszKysOk1YwV/Zj+EZGKJ3I5JjIai9nrR6VNf9GU9sgHOH9YS1++vwTXfyTA5EdP2T+sGBVPUosAtoGHrSvfJ9iNupb6vaSE9kWz3O4egY6szsRnTIKHrt7c3IdkG8fmflWHUX5ee3WX/QI/IlMmxc6J/zIelmEzW9nHmwfTC/b4nGYsDiFzpfNUwzNSuXwZPXcyYlmy+GN6W98ykc1n+qe8h5VGf8n7v5c3sMy8d2paa3MxMWHWDSKl1MqKq7I57OdhxNSCfHbMHH1Z4JQxrTPezGVcQ2DIMHvt1EzskdzLR9A1WrC9w3s9iL3xNLJxO0bjx/uo7g7hcnYWeyITYpkzsnriHE342ZT7Yr0zhSsnLp+fFyZjOOaj4edEp6h9AAD77LfEGH+I6aey0/5k3nsR+3EBmbzMba36MOLtQr9HV76IrwC16CUzsgtBeE9IbYrRhbvkUlHmOM/bt89u9nsSthgZPt0YkMmrSej4c0Zmhwlg7R3jtX774M+Q4aDubXeQsYvv1+Mqp3xDV2DX/a3ckdj3+EMbkdpyxe5D2yhEZBFVil3ZwHP90Dsdt0vnZ2sv58q9pYf0bUbKf7NxZnyf/pHOCH/9GfDz/do/u83vcHmCrvxbLFYjDigx8Zq35jibklc3LbMMoyjxdsZ2G0fQbV+z8lC02/gtikTB6YtoljZ9MZWTuNd049ierw/HUNJRU3gGHoBTy7ElROzs3Ujy1rET1RmMVS+PN1zWe6/zwwx9wBb1LobIpkdc2naRkahMu6D/WCa/hdus7H/vmwcRI0Gqpb8t3xjt6BvY0opbYZhlFk3o9MYCtYSlYujd9awsu96zGoeiqmuU/gnXsa09Pr9W7i5w1J9W5Ey2NP0qymJz8/2obVhxJ49Met3NemJu/7LoNlb+mLopA78l84MxHLR7X42WEEo17RbT9emR3J37tPsevlVtjMHAkn1kLLR/UH1NTOWFyr8ojS+Y5ta3vxwaDG5ZrvMfb3nYzeP5IQojE8a7Kx2QTqt+qJh3OB3F+LBVZ9ABETMNs6Q142Q/PeZnue7jtXy8eFOxsH8Ey3uvmhJ4ahL3iOroQjq+DEWvJMThzJ9SZInaFT9pck4Mmcp9vTrKacmG83hmHw46q97Iw+j7+fL76uDizac5qtJxIBmDiiGXcXkZNSlOhzGQyavJ6zadk42NrQupYXDat5EOjpxJvz9vBQ+1q8cVf9i++742QS208kEhmbTHJmLvWquhFW1Y2uoX4Xc1xupJPnM+j9RQTPua3iqfTJ0ONNHb1wqRMbyPv+TjYYDXgo+yVGdajDv/uEce/UDRxLSGfesx2oXdaK1MDMLSdZP2cSX9hP4ru8PjRo3ZM2O/6ld3u6vXoNP+HN569dpxjz2w5mPNiAtivv06Fjw3+BZW9ixG7HXK0ltqe26x7ZwHGXxnyY2J0uAx69WPimJAzDoPPHKwn2dmH6o230bXk58EN/VPx+Mh9ZzvYpT9DU5gguL0dycvb/UePwzxxSwdS0xLKz3zzatim6t/ANlRYPPw3QYaudX9a55iWVk66rfJvs9A6Gkxc8tqxSFGu6mrf/2sv3644T4OHIdw+1Yuuxc5gX/ouHbJcU/3tsGLodk1I6B9Kx6AgkwzB47MetbDh6lj86xdNgxzs6Kuq5neB8C7Q4EeJmkH5OX982GU4kIUSfTaTPobcx7bMWVw3uBH0n5Kc3XKh/s3uG3n19IfK2Cx+WCexNruNHK8jMMZOYkUN1zvCP/Stk+jTAp/Vw+OdlRlne5JBjE/4a0/FiKOKH/xxgyuojfDQgjGFb7tUhwPX6QvidcGYvxo6fUalxTKk7haceGAHAH1tP8vKs3Sx5sTOh3vaw7G29uuPiA+kJ/OA3nreiG/PWXfV5sH1wuRc7+nVTNCfn/4fRTUx8ZRrJ1C3naV3Li18ea3PZLkNUxAy8l49jGoNIb/EUver7sy8uhTWHz7L6UAJ1fF34eGgTmhcxId25bSPH573HXTbrONfocU61fhUnOxP1qlaC6oHihomMSSYyNplhrWqUqhF4enYem46dY83hs2w8ep7DZ1LJsxi4OdoS8XK3CpmYlsaSvad54fcdfGGayB1s0rs3LR7KX6FPiiZ3cldOZtqxtuvvHEu34/t1x2law5OdJ5P4emQLejW4tsqRFovBoK8ieCn+FTqa9ubfMXKODsm+hWTmmGn53lL6Nw5gQs8qOnc/4yyGnTMfu7zMlDP16FDNxEDPKP447sim9ABGtQvm//qHl7q9xMeLDzB51RE2vdoTXzcHXp+7h7XbdvCX3avkmhypkhtPTMtXqH7neIzsNM5MaElVcxwbw1+j7bCSFwi7qR1eCr8M0Skpj68oWaG/SiAmMYPJq44wpnsIVT0cMQyDf/2xk46Rr3KPaT3cNfFi+zsA8rLJm/8Ctrt1dJYFG6Kd6+M94hvcahSORFq89zTv/ryI6dXnUSthha43MWAS+De4kT+iELcfiwU2Tda9e8PvujySwpyr+6/XbHP1MPBbkExgb3LjZu5iyd7T3NemJnc3DWTF7xMZk/wJeZjYbanF+1Un8uV9zQs1z84zW3j0x62sizrLjMG+tIydDvv/0qXTUSQGduHF460YMPRhBjTT/eyOJqTR/dPVvDegIQ+0DdIvdGI9xtynyUhJpEX657zUvymPdap9XX7OqPhUen4WQXiAO/vjUugU4sOaw2d5uEMwb96V/0GZnp1Hv4lrMOeZWfB8l8I7tEDEoQTG/7mb0ylZTBjShCEt8quh7j2VzPCpG/H3cGTWg+F4VvG+oUVbxO0nO8/M4TNpuDrYElwZKlSifxfHTl/L+OR3aG+zj/PKky1OHWnncgq3c7vJNOx5yPYDfvrXSEw2ivunbWLzsfM83bUO/+pzjTmqVttOJDJ48nr613Plq97uurhbnR7XHAp5Mxo7cydL955hy//1xPH0NoxVH/IF9/HlXieGt6rB/rgUdscm0zDQg/8MbEjjMvbJPHQmlV6fR/DWXfVxsDPxyuxI2tb2ourp1Xxh+YCzNr74vLLnYuX6xOO7iN+1hHp3v3Rr/btv/wn8G0K15hU9kusqK9fMgIkreTvjP7S27EQN+U5Xk02Ng0WvwsmNfJk3kMxq7ambtp2uaQtwsDXh9sTCi7s86edPseCrcQw0L8HW1hbVdTy0G1OpQ66FELcGmcDe5PLMFiyGLtMOupDC3v/dS7OkpcyrN4H+9z5e5Ep8alYuQyZv4FRyJnOebk9db0eI2cKedDdGzDyFp7Mdfz3bEU9nvSNkGAYdP1rJqeRM2tfxplf9qpxKymTn8TMcio7j3s5NeaVf+HX7OQ3DoPm7S0nMyOXBdkG8dXcD3luwn2/XHuPzYU0Y2ExPRF+bE8mvm6OZ8Xhb2hRTKCY1K5fHftzKnthkFr/YmepVnElMz+HO/67FYhj8Obo9gZ4lyPkQ4jaVnp3H1NVHcDy1ke7xP1E3YweRllpEu7fgq3PNGXlPX0ZaF7oS03NYcSCeAc2qlWq3+mr+3n2K8AB36lxDOHJlEHEogVHfbebhDsHc27IG66LO8t6C/YzpXpdxveoBkJGTh5Od6ZojX/p8EUF6Th5nkrNpW8eb7x/SVZ2PR/yMV7VQqoS0ueafR9w8lu47w3M/rSWi6pf4Ju26eLvZ5MhzmU/g3WYY79zTEICvZy/i7l1P4e0Idj1eg6jlmA8vxbBYSKw3DN8736j0Ba+EELcOmcBWRrmZZBxZh3O9K+9IxCRmMOCrdWTmmOkR7k/L4CpMWHQQb1d7fnu87WWTuFNJmczcepLZ22OJPp+BvcmG8EB37gj34+muda97g/FvIo6SmpXLCz1DsbFR5JktPPDtJjYdO0+jah40CHTnt80neaJzbV69ymQ6JjGD3p9H0LSmJz890oaHf9jCxiPnmDW6XZl3MIS4XWXl5DJp9TGmrD6Cn5sDK8Z1vbioJq5NntnCwz9sYc3hsxdv693An8n3tyj3c+6kVVFMWHSQGl5OhRYwxa3JMAyGTtnAuXMJLO4Wg72DE+l2XoxckEmSfQALnuuEk72OQsrMMfPQZzOYmP0m/kYCSba+zMxqTWrD+xk3vH8F/yRCCFGYTGBvcVHxqUxbc4zFe0+TmJFLbR8Xfn28beGKvZcwDIMT5zII8HTEwbZiQ2yTM3L5Yf1x1kYlsCM6ibp+rsx9pkOJ+oP9sukEr83ZQ4ugKmw7kch/Bjbk/jZBN2DUQtya4pIzUagrnj9E2ZxJySLiUAJHz6bzbLe6uFyHNhIJqdm8OX8PY7qHlFsFeXFz23biPIMnb+D5HiHU8XNl0sooDsenMXt0e5rUKLyYG3EogbHfLSXM4Rzrc4J5plsoz/cIKXW+tRBCXG8ygb1N5JotbD+RSKi/201fSKY46dl5mGxUiZtbG4bByG83szbqLIOaV+PToU3KvfiUEEIIcTN74qetLNl3BoDavi683KsefRsFFPnYf8/azYqD8Xx+b1M6hvgU+RghhKhoMoEVt7T4lCz+2BbDIx1qXQyVEkIIIW4X0ecy+GzpQfo1CqBnuP8VQ9MNw8BiUK757EIIUd5kAiuEEEIIIYQQolK40gRWkh6EEEIIIYQQQlQKMoEVQgghhBBCCFEpyARWCCGEEEIIIUSlIBNYIYQQQgghhBCVgkxghRBCCCGEEEJUCjKBFUIIIYQQQghRKcgEVgghhBBCCCFEpSATWCGEEEIIIYQQlYJMYIUQQgghhBBCVAoygRVCCCGEEEIIUSnIBFYIIYQQQgghRKUgE1ghhBBCCCGEEJWCTGCFEEIIIYQQQlQKyjCMih5DqSilEoATFT2Oq/ABzlb0IMRNR44LURQ5LsSl5JgQRZHjQhRFjgtRlFvhuAgyDMO3qDsq3QS2MlBKbTUMo2VFj0PcXOS4EEWR40JcSo4JURQ5LkRR5LgQRbnVjwsJIRZCCCGEEEIIUSnIBFYIIYQQQgghRKUgE9jr4+uKHoC4KclxIYoix4W4lBwToihyXIiiyHEhinJLHxeSAyuEEEIIIYQQolKQHVghhBBCCCGEEJWCTGDLkVKqj1LqoFIqSik1vqLHIyqOUuq4UipSKbVTKbXVepuXUmqpUuqw9e8qFT1OcX0ppb5TSsUrpfYUuK3I40BpE63nj91KqeYVN3JxPRVzXLyllIq1njN2KqX6FbjvFetxcVAp1btiRi2uN6VUDaXUSqXUPqXUXqXU89bb5ZxxG7vCcSHnjNuYUspRKbVZKbXLely8bb29llJqk/X//3ellL31dgfr91HW+4MrcvzXSiaw5UQpZQK+AvoC9YERSqn6FTsqUcG6GYbRtEAZ8/HAcsMwQoDl1u/Fre0HoM8ltxV3HPQFQqx/ngAm36AxihvvBy4/LgA+t54zmhqGsRDA+jkyHGhgfc4k6+eNuPXkAeMMw6gPtAWesf7/yznj9lbccQFyzridZQPdDcNoAjQF+iil2gIfoY+LukAi8Kj18Y8CidbbP7c+rtKSCWz5aQ1EGYZx1DCMHGAGcE8Fj0ncXO4BfrR+/SMwoALHIm4AwzAigPOX3FzccXAP8JOhbQQ8lVIBN2ak4kYq5rgozj3ADMMwsg3DOAZEoT9vxC3GMIw4wzC2W79OBfYD1ZBzxm3tCsdFceSccRuw/t6nWb+1s/4xgO7ALOvtl54vLpxHZgE9lFLqBg233MkEtvxUA04W+D6GK59gxK3NAJYopbYppZ6w3uZvGEac9evTgH/FDE1UsOKOAzmHiGetoaDfFUgxkOPiNmQN72sGbELOGcLqkuMC5JxxW1NKmZRSO4F4YClwBEgyDCPP+pCC//cXjwvr/cmA940dcfmRCawQ10dHwzCao0O8nlFKdS54p6HLf0sJ8NucHAeigMlAHXQoWBzwacUOR1QUpZQr8CfwgmEYKQXvk3PG7auI40LOGbc5wzDMhmE0Baqjd9nDKnhIN4xMYMtPLFCjwPfVrbeJ25BhGLHWv+OBOegTy5kL4V3Wv+MrboSiAhV3HMg55DZmGMYZ68WIBfiG/JA/OS5uI0opO/Qk5RfDMGZbb5Zzxm2uqONCzhniAsMwkoCVQDt0KoGt9a6C//cXjwvr/R7AuRs81HIjE9jyswUIsVb/skcn0M+v4DGJCqCUclFKuV34GugF7EEfDw9aH/YgMK9iRigqWHHHwXxglLWyaFsguUDYoLjFXZK7OBB9zgB9XAy3VpCshS7Ys/lGj09cf9Z8tG+B/YZhfFbgLjln3MaKOy7knHF7U0r5KqU8rV87AXeg86NXAkOsD7v0fHHhPDIEWGGN6KiUbK/+EFEShmHkKaWeBRYDJuA7wzD2VvCwRMXwB+ZYc+NtgV8Nw1iklNoCzFRKPQqcAO6twDGKG0Ap9RvQFfBRSsUAbwIfUvRxsBDohy64kQE8fMMHLG6IYo6Lrkqppujw0OPAkwCGYexVSs0E9qGrkT5jGIa5IsYtrrsOwEgg0prXBvAqcs643RV3XIyQc8ZtLQD40Vph2gaYaRjG30qpfcAMpdR7wA704gfWv6crpaLQRQSHV8Sgy4uqxJNvIYQQQgghhBC3EQkhFkIIIYQQQghRKcgEVgghhBBCCCFEpSATWCGEEEIIIYQQlYJMYIUQQgghhBBCVAoygRVCCCGEEEIIUSnIBFYIIcRtTSllVkrtVErtUUr9daG3XhleZ5pSqn4Rtz+klPrftY/0+lFK/aCUGnL1RwohhBAVSyawQgghbneZhmE0NQyjIbo/3jNleRHDMB4zDGNf+Q7t5qeUkp7yQgghbhiZwAohhBD5NgDVLnyjlHpZKbVdS97BAAADP0lEQVRFKbVbKfW29TYXpdQCpdQu667tMOvtq5RSLa1fP6yUOqSU2gx0KPB6hXY6lVJpV3qvSyml0pRS/7G+90allP+VXlcp1VUptVopNU8pdVQp9aFS6n6l1GalVKRSqk6Bl++plNpqHfed1ueblFIfFxjXkwVed41Saj5w203ahRBCVByZwAohhBDoyRrQA5hv/b4XEAK0BpoCLZRSnYE+wCnDMJpYd20XXfI6AcDb6IlrR+CysOIi3ru497qUC7DRMIwmQATweAl+tCbAU0A4MBIINQyjNTANGFPgccHW9+8PTFFKOQKPAsmGYbQCWgGPK6VqWR/fHHjeMIzQEoxBCCGEKBcygRVCCHG7c1JK7QROA/7AUuvtvax/dgDbgTD0JDMSuEMp9ZFSqpNhGMmXvF4bYJVhGAmGYeQAv5dgDMW916VygL+tX29DTzqvZothGHGGYWQDR4Al1tsjL3n+TMMwLIZhHAaOWsfQCxhl/ffZBHgXGNdmwzCOleD9hRBCiHIjeStCCCFud5mGYTRVSjkDi9E5sBMBBXxgGMbUS5+glGoO9APeU0otNwzjnRK+Vx7WxWOllA1gf+Eli3uvS+QahmFYvzaT/zle3OsCZBf42lLgewuFrwMMCjOs4xpjGMbigncopboC6VcZqxBCCFHuZAdWCCGEAAzDyACeA8ZZCxMtBh5RSrkCKKWqKaX8lFKBQIZhGD8DH6NDaQvaBHRRSnkrpeyAoQXuOw60sH59N2Bn/brI9yrF8It73dIYqpSysebF1gYOWsc12vpzoJQKVUq5lOG1hRBCiHIhO7BCCCGElWEYO5RSu4ERhmFMV0qFAxuUUgBpwANAXeBjpZQFyAVGX/IacUqpt9AFoZKAnQXu/gaYp5Tahc6dTbc+Z0kx7xVfwqEX+bqlFA1sBtyBpwzDyFJKTUOHGW9XemAJwIAyvLYQQghRLlR+JJIQQgghhBBCCHHzkhBiIYQQQgghhBCVgkxghRBCCCGEEEJUCjKBFUIIIYQQQghRKcgEVgghhBBCCCFEpSATWCGEEEIIIYQQlYJMYIUQQgghhBBCVAoygRVCCCGEEEIIUSnIBFYIIYQQQgghRKXw/9GNBWMEaGhrAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 1152x432 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "# plot the RMSFs\n",
    "fig, ax = plt.subplots(figsize=(16,6))\n",
    "\n",
    "ax.plot(rmsf_df['rmsf_non_cov'].values, label='non-covalent ligands')\n",
    "ax.plot(rmsf_df['rmsf_cov'].values, label='covalent ligands')\n",
    "ax.set_xlabel('Residue number')\n",
    "ax.set_ylabel('RMSF (Angstroms)')\n",
    "ax.legend()\n",
    "\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Write PDB files for the covalent and non-covalent structures.  Set the temperature factors for each atom to the RMSF values for that residue. By visualizing the temperature factors, we can see which regions of the protein are more flexible. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'cov_rmsf_betas.pdb'"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# let's create a nice 3D structural visualizations\n",
    "struct = non_cov_prot_structs[0]\n",
    "for residue in struct['A']:\n",
    "    rmsf = rmsf_df['rmsf_non_cov'][residue.getResnum()]\n",
    "    residue.setBetas(rmsf*150)\n",
    "prody.writePDB('non_cov_rmsf_betas.pdb', struct)\n",
    "\n",
    "struct = cov_prot_structs[0]\n",
    "for residue in struct['A']:\n",
    "    rmsf = rmsf_df['rmsf_cov'][residue.getResnum()]\n",
    "    residue.setBetas(rmsf*150)\n",
    "prody.writePDB('cov_rmsf_betas.pdb', struct)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Get some simple descriptive statistics on the RMSF values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "DescribeResult(nobs=2347, minmax=(7.619111462298995, 110.85465890115563), mean=16.70439678233319, variance=102.58964041607254, skewness=4.253060096890112, kurtosis=27.056135854988497)"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "describe(struct.getBetas())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Use Py3DMol to visualized the RMSF values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_15885999372043362\"  style=\"position: relative; width: 500px; height: 500px\">\n        <p id=\"3dmolwarning_15885999372043362\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.csb.pitt.edu/build/3Dmol.js');\n}\n\nvar viewer_15885999372043362 = null;\nvar warn = document.getElementById(\"3dmolwarning_15885999372043362\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_15885999372043362 = $3Dmol.createViewer($(\"#3dmolviewer_15885999372043362\"),{backgroundColor:\"white\"});\n\tviewer_15885999372043362.addModel(\"REMARK AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein'\\nATOM      1  N   SER A   1      -2.327   4.473 -16.891  1.00 25.83           N\\nATOM      2  CA  SER A   1      -2.209   5.813 -16.291  1.00 25.83           C\\nATOM      3  C   SER A   1      -2.610   5.806 -14.813  1.00 25.83           C\\nATOM      4  O   SER A   1      -3.283   4.886 -14.346  1.00 25.83           O\\nATOM      5  CB  SER A   1      -2.999   6.866 -17.096  1.00 25.83           C\\nATOM      6  OG  SER A   1      -2.177   7.396 -18.115  1.00 25.83           O\\nATOM      7  N   GLY A   2      -2.158   6.834 -14.083  1.00 21.53           N\\nATOM      8  CA  GLY A   2      -2.257   6.960 -12.637  1.00 21.53           C\\nATOM      9  C   GLY A   2      -0.976   6.471 -11.956  1.00 21.53           C\\nATOM     10  O   GLY A   2      -0.282   5.593 -12.473  1.00 21.53           O\\nATOM     11  N   PHE A   3      -0.670   7.047 -10.787  1.00 16.19           N\\nATOM     12  CA  PHE A   3       0.528   6.757 -10.010  1.00 16.19           C\\nATOM     13  C   PHE A   3       0.178   6.764  -8.519  1.00 16.19           C\\nATOM     14  O   PHE A   3      -0.178   7.804  -7.966  1.00 16.19           O\\nATOM     15  CB  PHE A   3       1.661   7.714 -10.423  1.00 16.19           C\\nATOM     16  CG  PHE A   3       3.047   7.202 -10.100  1.00 16.19           C\\nATOM     17  CD1 PHE A   3       3.659   6.253 -10.947  1.00 16.19           C\\nATOM     18  CD2 PHE A   3       3.725   7.664  -8.955  1.00 16.19           C\\nATOM     19  CE1 PHE A   3       4.935   5.748 -10.633  1.00 16.19           C\\nATOM     20  CE2 PHE A   3       5.006   7.173  -8.656  1.00 16.19           C\\nATOM     21  CZ  PHE A   3       5.597   6.194  -9.474  1.00 16.19           C\\nATOM     22  N   ARG A   4       0.254   5.582  -7.896  1.00 16.56           N\\nATOM     23  CA  ARG A   4      -0.095   5.305  -6.507  1.00 16.56           C\\nATOM     24  C   ARG A   4       1.143   4.912  -5.709  1.00 16.56           C\\nATOM     25  O   ARG A   4       2.091   4.362  -6.264  1.00 16.56           O\\nATOM     26  CB  ARG A   4      -1.087   4.119  -6.489  1.00 16.56           C\\nATOM     27  CG  ARG A   4      -2.538   4.542  -6.759  1.00 16.56           C\\nATOM     28  CD  ARG A   4      -3.135   5.257  -5.540  1.00 16.56           C\\nATOM     29  NE  ARG A   4      -4.534   5.640  -5.740  1.00 16.56           N\\nATOM     30  CZ  ARG A   4      -5.276   6.274  -4.817  1.00 16.56           C\\nATOM     31  NH1 ARG A   4      -4.767   6.595  -3.617  1.00 16.56           N\\nATOM     32  NH2 ARG A   4      -6.545   6.592  -5.099  1.00 16.56           N\\nATOM     33  N   LYS A   5       1.095   5.129  -4.383  1.00 14.70           N\\nATOM     34  CA  LYS A   5       2.080   4.605  -3.442  1.00 14.70           C\\nATOM     35  C   LYS A   5       1.779   3.111  -3.244  1.00 14.70           C\\nATOM     36  O   LYS A   5       0.868   2.731  -2.507  1.00 14.70           O\\nATOM     37  CB  LYS A   5       1.997   5.395  -2.120  1.00 14.70           C\\nATOM     38  CG  LYS A   5       3.140   5.067  -1.142  1.00 14.70           C\\nATOM     39  CD  LYS A   5       4.551   5.484  -1.613  1.00 14.70           C\\nATOM     40  CE  LYS A   5       4.782   7.002  -1.668  1.00 14.70           C\\nATOM     41  NZ  LYS A   5       4.673   7.624  -0.339  1.00 14.70           N\\nATOM     42  N   MET A   6       2.519   2.293  -4.001  1.00  8.53           N\\nATOM     43  CA  MET A   6       2.279   0.881  -4.232  1.00  8.53           C\\nATOM     44  C   MET A   6       3.273   0.073  -3.398  1.00  8.53           C\\nATOM     45  O   MET A   6       4.483   0.277  -3.485  1.00  8.53           O\\nATOM     46  CB  MET A   6       2.532   0.641  -5.738  1.00  8.53           C\\nATOM     47  CG  MET A   6       2.191  -0.774  -6.231  1.00  8.53           C\\nATOM     48  SD  MET A   6       2.354  -0.986  -8.025  1.00  8.53           S\\nATOM     49  CE  MET A   6       0.823  -0.190  -8.585  1.00  8.53           C\\nATOM     50  N   ALA A   7       2.750  -0.884  -2.623  1.00  9.04           N\\nATOM     51  CA  ALA A   7       3.541  -1.919  -1.973  1.00  9.04           C\\nATOM     52  C   ALA A   7       3.657  -3.132  -2.907  1.00  9.04           C\\nATOM     53  O   ALA A   7       2.814  -3.346  -3.781  1.00  9.04           O\\nATOM     54  CB  ALA A   7       2.808  -2.363  -0.699  1.00  9.04           C\\nATOM     55  N   PHE A   8       4.686  -3.961  -2.674  1.00  8.11           N\\nATOM     56  CA  PHE A   8       4.836  -5.289  -3.259  1.00  8.11           C\\nATOM     57  C   PHE A   8       3.754  -6.226  -2.691  1.00  8.11           C\\nATOM     58  O   PHE A   8       3.409  -6.079  -1.514  1.00  8.11           O\\nATOM     59  CB  PHE A   8       6.214  -5.869  -2.876  1.00  8.11           C\\nATOM     60  CG  PHE A   8       7.369  -5.268  -3.646  1.00  8.11           C\\nATOM     61  CD1 PHE A   8       7.603  -5.655  -4.982  1.00  8.11           C\\nATOM     62  CD2 PHE A   8       8.201  -4.310  -3.039  1.00  8.11           C\\nATOM     63  CE1 PHE A   8       8.659  -5.075  -5.709  1.00  8.11           C\\nATOM     64  CE2 PHE A   8       9.248  -3.728  -3.772  1.00  8.11           C\\nATOM     65  CZ  PHE A   8       9.484  -4.110  -5.105  1.00  8.11           C\\nATOM     66  N   PRO A   9       3.242  -7.199  -3.485  1.00  8.93           N\\nATOM     67  CA  PRO A   9       2.375  -8.288  -3.006  1.00  8.93           C\\nATOM     68  C   PRO A   9       2.999  -8.990  -1.788  1.00  8.93           C\\nATOM     69  O   PRO A   9       4.143  -9.439  -1.855  1.00  8.93           O\\nATOM     70  CB  PRO A   9       2.240  -9.256  -4.193  1.00  8.93           C\\nATOM     71  CG  PRO A   9       2.551  -8.423  -5.423  1.00  8.93           C\\nATOM     72  CD  PRO A   9       3.460  -7.300  -4.929  1.00  8.93           C\\nATOM     73  N   SER A  10       2.279  -9.000  -0.661  1.00 10.35           N\\nATOM     74  CA  SER A  10       2.811  -9.411   0.636  1.00 10.35           C\\nATOM     75  C   SER A  10       2.701 -10.920   0.908  1.00 10.35           C\\nATOM     76  O   SER A  10       3.244 -11.362   1.916  1.00 10.35           O\\nATOM     77  CB  SER A  10       2.072  -8.612   1.728  1.00 10.35           C\\nATOM     78  OG  SER A  10       0.666  -8.763   1.693  1.00 10.35           O\\nATOM     79  N   GLY A  11       2.057 -11.708   0.027  1.00  8.88           N\\nATOM     80  CA  GLY A  11       1.755 -13.134   0.193  1.00  8.88           C\\nATOM     81  C   GLY A  11       2.941 -14.055   0.522  1.00  8.88           C\\nATOM     82  O   GLY A  11       2.783 -14.977   1.322  1.00  8.88           O\\nATOM     83  N   LYS A  12       4.130 -13.783  -0.046  1.00  7.57           N\\nATOM     84  CA  LYS A  12       5.373 -14.516   0.213  1.00  7.57           C\\nATOM     85  C   LYS A  12       5.961 -14.231   1.608  1.00  7.57           C\\nATOM     86  O   LYS A  12       6.669 -15.081   2.147  1.00  7.57           O\\nATOM     87  CB  LYS A  12       6.429 -14.126  -0.840  1.00  7.57           C\\nATOM     88  CG  LYS A  12       6.192 -14.784  -2.206  1.00  7.57           C\\nATOM     89  CD  LYS A  12       7.171 -14.266  -3.270  1.00  7.57           C\\nATOM     90  CE  LYS A  12       6.830 -14.756  -4.681  1.00  7.57           C\\nATOM     91  NZ  LYS A  12       7.206 -16.165  -4.886  1.00  7.57           N\\nATOM     92  N   VAL A  13       5.660 -13.063   2.195  1.00  7.20           N\\nATOM     93  CA  VAL A  13       6.101 -12.659   3.529  1.00  7.20           C\\nATOM     94  C   VAL A  13       5.051 -13.048   4.579  1.00  7.20           C\\nATOM     95  O   VAL A  13       5.450 -13.397   5.686  1.00  7.20           O\\nATOM     96  CB  VAL A  13       6.357 -11.130   3.543  1.00  7.20           C\\nATOM     97  CG1 VAL A  13       6.776 -10.588   4.926  1.00  7.20           C\\nATOM     98  CG2 VAL A  13       7.448 -10.791   2.518  1.00  7.20           C\\nATOM     99  N   GLU A  14       3.746 -13.065   4.237  1.00  8.29           N\\nATOM    100  CA  GLU A  14       2.640 -13.522   5.092  1.00  8.29           C\\nATOM    101  C   GLU A  14       2.825 -14.985   5.532  1.00  8.29           C\\nATOM    102  O   GLU A  14       2.586 -15.309   6.695  1.00  8.29           O\\nATOM    103  CB  GLU A  14       1.299 -13.399   4.339  1.00  8.29           C\\nATOM    104  CG  GLU A  14       0.790 -11.962   4.131  1.00  8.29           C\\nATOM    105  CD  GLU A  14      -0.471 -11.938   3.269  1.00  8.29           C\\nATOM    106  OE1 GLU A  14      -1.443 -12.623   3.655  1.00  8.29           O\\nATOM    107  OE2 GLU A  14      -0.435 -11.254   2.224  1.00  8.29           O\\nATOM    108  N   GLY A  15       3.311 -15.844   4.618  1.00 13.05           N\\nATOM    109  CA  GLY A  15       3.633 -17.246   4.865  1.00 13.05           C\\nATOM    110  C   GLY A  15       4.893 -17.484   5.710  1.00 13.05           C\\nATOM    111  O   GLY A  15       5.182 -18.633   6.035  1.00 13.05           O\\nATOM    112  N   CYS A  16       5.629 -16.420   6.076  1.00  8.08           N\\nATOM    113  CA  CYS A  16       6.796 -16.463   6.955  1.00  8.08           C\\nATOM    114  C   CYS A  16       6.491 -15.907   8.354  1.00  8.08           C\\nATOM    115  O   CYS A  16       7.329 -16.062   9.238  1.00  8.08           O\\nATOM    116  CB  CYS A  16       7.932 -15.619   6.339  1.00  8.08           C\\nATOM    117  SG  CYS A  16       8.457 -16.318   4.753  1.00  8.08           S\\nATOM    118  N   MET A  17       5.335 -15.262   8.592  1.00  7.67           N\\nATOM    119  CA  MET A  17       5.069 -14.590   9.865  1.00  7.67           C\\nATOM    120  C   MET A  17       4.553 -15.571  10.919  1.00  7.67           C\\nATOM    121  O   MET A  17       3.609 -16.323  10.685  1.00  7.67           O\\nATOM    122  CB  MET A  17       4.031 -13.469   9.682  1.00  7.67           C\\nATOM    123  CG  MET A  17       4.494 -12.369   8.718  1.00  7.67           C\\nATOM    124  SD  MET A  17       6.197 -11.755   8.921  1.00  7.67           S\\nATOM    125  CE  MET A  17       6.173 -11.170  10.640  1.00  7.67           C\\nATOM    126  N   VAL A  18       5.191 -15.522  12.093  1.00  7.91           N\\nATOM    127  CA  VAL A  18       4.904 -16.316  13.282  1.00  7.91           C\\nATOM    128  C   VAL A  18       4.823 -15.381  14.493  1.00  7.91           C\\nATOM    129  O   VAL A  18       5.332 -14.263  14.458  1.00  7.91           O\\nATOM    130  CB  VAL A  18       5.922 -17.480  13.438  1.00  7.91           C\\nATOM    131  CG1 VAL A  18       5.944 -18.399  12.206  1.00  7.91           C\\nATOM    132  CG2 VAL A  18       7.356 -17.029  13.771  1.00  7.91           C\\nATOM    133  N   GLN A  19       4.177 -15.844  15.568  1.00  9.14           N\\nATOM    134  CA  GLN A  19       4.152 -15.184  16.867  1.00  9.14           C\\nATOM    135  C   GLN A  19       5.329 -15.737  17.680  1.00  9.14           C\\nATOM    136  O   GLN A  19       5.560 -16.944  17.660  1.00  9.14           O\\nATOM    137  CB  GLN A  19       2.845 -15.592  17.577  1.00  9.14           C\\nATOM    138  CG  GLN A  19       2.531 -14.738  18.821  1.00  9.14           C\\nATOM    139  CD  GLN A  19       1.233 -15.139  19.529  1.00  9.14           C\\nATOM    140  OE1 GLN A  19       0.453 -14.275  19.920  1.00  9.14           O\\nATOM    141  NE2 GLN A  19       0.976 -16.438  19.686  1.00  9.14           N\\nATOM    142  N   VAL A  20       6.044 -14.868  18.407  1.00  9.86           N\\nATOM    143  CA  VAL A  20       7.081 -15.240  19.368  1.00  9.86           C\\nATOM    144  C   VAL A  20       6.720 -14.589  20.705  1.00  9.86           C\\nATOM    145  O   VAL A  20       6.605 -13.367  20.787  1.00  9.86           O\\nATOM    146  CB  VAL A  20       8.502 -14.856  18.880  1.00  9.86           C\\nATOM    147  CG1 VAL A  20       9.584 -15.347  19.862  1.00  9.86           C\\nATOM    148  CG2 VAL A  20       8.808 -15.389  17.470  1.00  9.86           C\\nATOM    149  N   THR A  21       6.574 -15.418  21.745  1.00 13.64           N\\nATOM    150  CA  THR A  21       6.302 -15.015  23.119  1.00 13.64           C\\nATOM    151  C   THR A  21       7.469 -15.508  23.983  1.00 13.64           C\\nATOM    152  O   THR A  21       7.869 -16.667  23.888  1.00 13.64           O\\nATOM    153  CB  THR A  21       4.948 -15.610  23.581  1.00 13.64           C\\nATOM    154  OG1 THR A  21       3.902 -15.118  22.768  1.00 13.64           O\\nATOM    155  CG2 THR A  21       4.587 -15.288  25.044  1.00 13.64           C\\nATOM    156  N   CYS A  22       7.978 -14.631  24.858  1.00 16.53           N\\nATOM    157  CA  CYS A  22       8.954 -14.954  25.890  1.00 16.53           C\\nATOM    158  C   CYS A  22       8.521 -14.235  27.166  1.00 16.53           C\\nATOM    159  O   CYS A  22       8.553 -13.005  27.233  1.00 16.53           O\\nATOM    160  CB  CYS A  22      10.387 -14.668  25.400  1.00 16.53           C\\nATOM    161  SG  CYS A  22      11.614 -15.289  26.578  1.00 16.53           S\\nATOM    162  N   GLY A  23       8.057 -15.019  28.152  1.00 31.27           N\\nATOM    163  CA  GLY A  23       7.388 -14.539  29.352  1.00 31.27           C\\nATOM    164  C   GLY A  23       6.019 -13.945  29.008  1.00 31.27           C\\nATOM    165  O   GLY A  23       5.165 -14.629  28.442  1.00 31.27           O\\nATOM    166  N   THR A  24       5.825 -12.666  29.352  1.00 27.32           N\\nATOM    167  CA  THR A  24       4.602 -11.900  29.113  1.00 27.32           C\\nATOM    168  C   THR A  24       4.700 -10.994  27.865  1.00 27.32           C\\nATOM    169  O   THR A  24       3.696 -10.390  27.484  1.00 27.32           O\\nATOM    170  CB  THR A  24       4.316 -11.048  30.376  1.00 27.32           C\\nATOM    171  OG1 THR A  24       5.363 -10.129  30.639  1.00 27.32           O\\nATOM    172  CG2 THR A  24       4.062 -11.900  31.631  1.00 27.32           C\\nATOM    173  N   THR A  25       5.875 -10.903  27.217  1.00 26.88           N\\nATOM    174  CA  THR A  25       6.104 -10.080  26.029  1.00 26.88           C\\nATOM    175  C   THR A  25       5.859 -10.958  24.795  1.00 26.88           C\\nATOM    176  O   THR A  25       6.521 -11.981  24.624  1.00 26.88           O\\nATOM    177  CB  THR A  25       7.574  -9.600  26.035  1.00 26.88           C\\nATOM    178  OG1 THR A  25       7.772  -8.781  27.169  1.00 26.88           O\\nATOM    179  CG2 THR A  25       7.968  -8.752  24.809  1.00 26.88           C\\nATOM    180  N   THR A  26       4.917 -10.531  23.942  1.00 20.57           N\\nATOM    181  CA  THR A  26       4.553 -11.166  22.679  1.00 20.57           C\\nATOM    182  C   THR A  26       4.824 -10.166  21.550  1.00 20.57           C\\nATOM    183  O   THR A  26       4.357  -9.028  21.594  1.00 20.57           O\\nATOM    184  CB  THR A  26       3.063 -11.574  22.718  1.00 20.57           C\\nATOM    185  OG1 THR A  26       2.891 -12.558  23.717  1.00 20.57           O\\nATOM    186  CG2 THR A  26       2.558 -12.189  21.404  1.00 20.57           C\\nATOM    187  N   LEU A  27       5.549 -10.626  20.527  1.00 12.56           N\\nATOM    188  CA  LEU A  27       5.851  -9.910  19.295  1.00 12.56           C\\nATOM    189  C   LEU A  27       5.846 -10.878  18.101  1.00 12.56           C\\nATOM    190  O   LEU A  27       5.441 -12.033  18.238  1.00 12.56           O\\nATOM    191  CB  LEU A  27       7.062  -8.961  19.480  1.00 12.56           C\\nATOM    192  CG  LEU A  27       8.376  -9.555  20.035  1.00 12.56           C\\nATOM    193  CD1 LEU A  27       8.984 -10.622  19.116  1.00 12.56           C\\nATOM    194  CD2 LEU A  27       9.391  -8.427  20.278  1.00 12.56           C\\nATOM    195  N   ASN A  28       6.252 -10.399  16.920  1.00  7.83           N\\nATOM    196  CA  ASN A  28       6.263 -11.171  15.676  1.00  7.83           C\\nATOM    197  C   ASN A  28       7.675 -11.718  15.414  1.00  7.83           C\\nATOM    198  O   ASN A  28       8.676 -11.122  15.812  1.00  7.83           O\\nATOM    199  CB  ASN A  28       5.889 -10.248  14.497  1.00  7.83           C\\nATOM    200  CG  ASN A  28       4.544  -9.547  14.688  1.00  7.83           C\\nATOM    201  OD1 ASN A  28       3.490 -10.170  14.595  1.00  7.83           O\\nATOM    202  ND2 ASN A  28       4.583  -8.247  14.977  1.00  7.83           N\\nATOM    203  N   GLY A  29       7.743 -12.850  14.709  1.00  8.98           N\\nATOM    204  CA  GLY A  29       8.957 -13.442  14.172  1.00  8.98           C\\nATOM    205  C   GLY A  29       8.810 -13.689  12.672  1.00  8.98           C\\nATOM    206  O   GLY A  29       7.699 -13.873  12.175  1.00  8.98           O\\nATOM    207  N   LEU A  30       9.945 -13.725  11.964  1.00  7.58           N\\nATOM    208  CA  LEU A  30      10.050 -14.080  10.556  1.00  7.58           C\\nATOM    209  C   LEU A  30      10.706 -15.464  10.480  1.00  7.58           C\\nATOM    210  O   LEU A  30      11.878 -15.609  10.813  1.00  7.58           O\\nATOM    211  CB  LEU A  30      10.937 -13.026   9.855  1.00  7.58           C\\nATOM    212  CG  LEU A  30      11.004 -13.183   8.322  1.00  7.58           C\\nATOM    213  CD1 LEU A  30       9.678 -12.780   7.661  1.00  7.58           C\\nATOM    214  CD2 LEU A  30      12.181 -12.399   7.729  1.00  7.58           C\\nATOM    215  N   TRP A  31       9.947 -16.471  10.038  1.00  7.28           N\\nATOM    216  CA  TRP A  31      10.356 -17.862   9.892  1.00  7.28           C\\nATOM    217  C   TRP A  31      10.815 -18.087   8.442  1.00  7.28           C\\nATOM    218  O   TRP A  31      10.010 -18.037   7.512  1.00  7.28           O\\nATOM    219  CB  TRP A  31       9.115 -18.725  10.208  1.00  7.28           C\\nATOM    220  CG  TRP A  31       9.281 -20.210  10.356  1.00  7.28           C\\nATOM    221  CD1 TRP A  31      10.439 -20.888  10.546  1.00  7.28           C\\nATOM    222  CD2 TRP A  31       8.233 -21.223  10.294  1.00  7.28           C\\nATOM    223  NE1 TRP A  31      10.177 -22.240  10.611  1.00  7.28           N\\nATOM    224  CE2 TRP A  31       8.834 -22.509  10.453  1.00  7.28           C\\nATOM    225  CE3 TRP A  31       6.835 -21.193  10.070  1.00  7.28           C\\nATOM    226  CZ2 TRP A  31       8.089 -23.698  10.407  1.00  7.28           C\\nATOM    227  CZ3 TRP A  31       6.080 -22.381  10.009  1.00  7.28           C\\nATOM    228  CH2 TRP A  31       6.704 -23.631  10.181  1.00  7.28           C\\nATOM    229  N   LEU A  32      12.122 -18.322   8.265  1.00  8.89           N\\nATOM    230  CA  LEU A  32      12.785 -18.614   6.995  1.00  8.89           C\\nATOM    231  C   LEU A  32      13.661 -19.839   7.205  1.00  8.89           C\\nATOM    232  O   LEU A  32      14.543 -19.819   8.066  1.00  8.89           O\\nATOM    233  CB  LEU A  32      13.646 -17.423   6.547  1.00  8.89           C\\nATOM    234  CG  LEU A  32      12.868 -16.153   6.185  1.00  8.89           C\\nATOM    235  CD1 LEU A  32      13.846 -14.983   6.100  1.00  8.89           C\\nATOM    236  CD2 LEU A  32      12.074 -16.308   4.880  1.00  8.89           C\\nATOM    237  N   ASP A  33      13.383 -20.916   6.454  1.00  9.36           N\\nATOM    238  CA  ASP A  33      13.950 -22.253   6.650  1.00  9.36           C\\nATOM    239  C   ASP A  33      13.591 -22.724   8.071  1.00  9.36           C\\nATOM    240  O   ASP A  33      12.444 -22.555   8.486  1.00  9.36           O\\nATOM    241  CB  ASP A  33      15.467 -22.319   6.306  1.00  9.36           C\\nATOM    242  CG  ASP A  33      15.812 -21.813   4.901  1.00  9.36           C\\nATOM    243  OD1 ASP A  33      15.128 -22.244   3.947  1.00  9.36           O\\nATOM    244  OD2 ASP A  33      16.765 -21.010   4.799  1.00  9.36           O\\nATOM    245  N   ASP A  34      14.550 -23.272   8.824  1.00  9.08           N\\nATOM    246  CA  ASP A  34      14.369 -23.766  10.187  1.00  9.08           C\\nATOM    247  C   ASP A  34      14.816 -22.726  11.242  1.00  9.08           C\\nATOM    248  O   ASP A  34      15.148 -23.122  12.356  1.00  9.08           O\\nATOM    249  CB  ASP A  34      15.146 -25.106  10.314  1.00  9.08           C\\nATOM    250  CG  ASP A  34      16.682 -25.063  10.242  1.00  9.08           C\\nATOM    251  OD1 ASP A  34      17.232 -24.079   9.701  1.00  9.08           O\\nATOM    252  OD2 ASP A  34      17.287 -26.045  10.726  1.00  9.08           O\\nATOM    253  N   VAL A  35      14.806 -21.417  10.922  1.00  8.34           N\\nATOM    254  CA  VAL A  35      15.188 -20.318  11.820  1.00  8.34           C\\nATOM    255  C   VAL A  35      14.079 -19.257  11.851  1.00  8.34           C\\nATOM    256  O   VAL A  35      13.626 -18.804  10.801  1.00  8.34           O\\nATOM    257  CB  VAL A  35      16.570 -19.719  11.430  1.00  8.34           C\\nATOM    258  CG1 VAL A  35      17.017 -18.599  12.393  1.00  8.34           C\\nATOM    259  CG2 VAL A  35      17.678 -20.783  11.343  1.00  8.34           C\\nATOM    260  N   VAL A  36      13.696 -18.828  13.065  1.00  6.67           N\\nATOM    261  CA  VAL A  36      12.772 -17.731  13.333  1.00  6.67           C\\nATOM    262  C   VAL A  36      13.579 -16.557  13.902  1.00  6.67           C\\nATOM    263  O   VAL A  36      14.136 -16.643  14.995  1.00  6.67           O\\nATOM    264  CB  VAL A  36      11.655 -18.204  14.298  1.00  6.67           C\\nATOM    265  CG1 VAL A  36      10.731 -17.058  14.761  1.00  6.67           C\\nATOM    266  CG2 VAL A  36      10.818 -19.314  13.644  1.00  6.67           C\\nATOM    267  N   TYR A  37      13.617 -15.457  13.143  1.00  8.11           N\\nATOM    268  CA  TYR A  37      14.291 -14.204  13.467  1.00  8.11           C\\nATOM    269  C   TYR A  37      13.269 -13.286  14.147  1.00  8.11           C\\nATOM    270  O   TYR A  37      12.173 -13.103  13.623  1.00  8.11           O\\nATOM    271  CB  TYR A  37      14.727 -13.539  12.142  1.00  8.11           C\\nATOM    272  CG  TYR A  37      15.718 -14.367  11.344  1.00  8.11           C\\nATOM    273  CD1 TYR A  37      15.255 -15.347  10.439  1.00  8.11           C\\nATOM    274  CD2 TYR A  37      17.103 -14.195  11.533  1.00  8.11           C\\nATOM    275  CE1 TYR A  37      16.161 -16.184   9.770  1.00  8.11           C\\nATOM    276  CE2 TYR A  37      18.010 -15.023  10.846  1.00  8.11           C\\nATOM    277  CZ  TYR A  37      17.542 -16.023   9.975  1.00  8.11           C\\nATOM    278  OH  TYR A  37      18.430 -16.833   9.329  1.00  8.11           O\\nATOM    279  N   CYS A  38      13.623 -12.701  15.298  1.00  9.34           N\\nATOM    280  CA  CYS A  38      12.792 -11.761  16.054  1.00  9.34           C\\nATOM    281  C   CYS A  38      13.672 -10.743  16.799  1.00  9.34           C\\nATOM    282  O   CYS A  38      14.817 -11.077  17.101  1.00  9.34           O\\nATOM    283  CB  CYS A  38      11.786 -12.518  16.955  1.00  9.34           C\\nATOM    284  SG  CYS A  38      12.604 -13.390  18.326  1.00  9.34           S\\nATOM    285  N   PRO A  39      13.171  -9.532  17.145  1.00  9.32           N\\nATOM    286  CA  PRO A  39      13.857  -8.572  18.029  1.00  9.32           C\\nATOM    287  C   PRO A  39      14.219  -9.211  19.378  1.00  9.32           C\\nATOM    288  O   PRO A  39      13.389  -9.906  19.968  1.00  9.32           O\\nATOM    289  CB  PRO A  39      12.854  -7.429  18.256  1.00  9.32           C\\nATOM    290  CG  PRO A  39      11.886  -7.523  17.097  1.00  9.32           C\\nATOM    291  CD  PRO A  39      11.911  -8.981  16.657  1.00  9.32           C\\nATOM    292  N   ARG A  40      15.454  -8.987  19.854  1.00 12.72           N\\nATOM    293  CA  ARG A  40      15.956  -9.577  21.094  1.00 12.72           C\\nATOM    294  C   ARG A  40      15.265  -9.053  22.364  1.00 12.72           C\\nATOM    295  O   ARG A  40      15.312  -9.741  23.379  1.00 12.72           O\\nATOM    296  CB  ARG A  40      17.487  -9.410  21.191  1.00 12.72           C\\nATOM    297  CG  ARG A  40      17.994  -7.967  21.341  1.00 12.72           C\\nATOM    298  CD  ARG A  40      19.487  -7.921  21.665  1.00 12.72           C\\nATOM    299  NE  ARG A  40      19.972  -6.539  21.730  1.00 12.72           N\\nATOM    300  CZ  ARG A  40      21.225  -6.150  22.008  1.00 12.72           C\\nATOM    301  NH1 ARG A  40      22.191  -7.048  22.243  1.00 12.72           N\\nATOM    302  NH2 ARG A  40      21.506  -4.842  22.047  1.00 12.72           N\\nATOM    303  N   HIS A  41      14.578  -7.895  22.303  1.00 26.42           N\\nATOM    304  CA  HIS A  41      13.758  -7.340  23.384  1.00 26.42           C\\nATOM    305  C   HIS A  41      12.463  -8.120  23.666  1.00 26.42           C\\nATOM    306  O   HIS A  41      11.714  -7.728  24.557  1.00 26.42           O\\nATOM    307  CB  HIS A  41      13.466  -5.846  23.136  1.00 26.42           C\\nATOM    308  CG  HIS A  41      12.449  -5.482  22.084  1.00 26.42           C\\nATOM    309  ND1 HIS A  41      12.809  -4.892  20.870  1.00 26.42           N\\nATOM    310  CD2 HIS A  41      11.077  -5.625  22.119  1.00 26.42           C\\nATOM    311  CE1 HIS A  41      11.661  -4.732  20.231  1.00 26.42           C\\nATOM    312  NE2 HIS A  41      10.601  -5.134  20.924  1.00 26.42           N\\nATOM    313  N   VAL A  42      12.218  -9.253  22.987  1.00 18.64           N\\nATOM    314  CA  VAL A  42      11.208 -10.238  23.379  1.00 18.64           C\\nATOM    315  C   VAL A  42      11.475 -10.804  24.793  1.00 18.64           C\\nATOM    316  O   VAL A  42      10.525 -11.132  25.496  1.00 18.64           O\\nATOM    317  CB  VAL A  42      11.111 -11.344  22.299  1.00 18.64           C\\nATOM    318  CG1 VAL A  42      12.328 -12.282  22.247  1.00 18.64           C\\nATOM    319  CG2 VAL A  42       9.803 -12.149  22.400  1.00 18.64           C\\nATOM    320  N   ILE A  43      12.749 -10.861  25.230  1.00 15.10           N\\nATOM    321  CA  ILE A  43      13.159 -11.307  26.566  1.00 15.10           C\\nATOM    322  C   ILE A  43      12.988 -10.205  27.629  1.00 15.10           C\\nATOM    323  O   ILE A  43      13.037 -10.519  28.815  1.00 15.10           O\\nATOM    324  CB  ILE A  43      14.642 -11.794  26.540  1.00 15.10           C\\nATOM    325  CG1 ILE A  43      15.721 -10.675  26.536  1.00 15.10           C\\nATOM    326  CG2 ILE A  43      14.840 -12.846  25.435  1.00 15.10           C\\nATOM    327  CD1 ILE A  43      17.159 -11.173  26.332  1.00 15.10           C\\nATOM    328  N   CYS A  44      12.809  -8.936  27.226  1.00 31.09           N\\nATOM    329  CA  CYS A  44      12.689  -7.794  28.127  1.00 31.09           C\\nATOM    330  C   CYS A  44      11.221  -7.565  28.500  1.00 31.09           C\\nATOM    331  O   CYS A  44      10.345  -7.569  27.634  1.00 31.09           O\\nATOM    332  CB  CYS A  44      13.163  -6.515  27.401  1.00 31.09           C\\nATOM    333  SG  CYS A  44      14.962  -6.508  27.242  1.00 31.09           S\\nATOM    334  N   THR A  45      10.987  -7.253  29.784  1.00 81.61           N\\nATOM    335  CA  THR A  45       9.804  -6.535  30.266  1.00 81.61           C\\nATOM    336  C   THR A  45       9.958  -5.038  29.903  1.00 81.61           C\\nATOM    337  O   THR A  45      11.039  -4.598  29.507  1.00 81.61           O\\nATOM    338  CB  THR A  45       9.688  -6.716  31.806  1.00 81.61           C\\nATOM    339  OG1 THR A  45      10.680  -6.008  32.523  1.00 81.61           O\\nATOM    340  CG2 THR A  45       9.693  -8.180  32.274  1.00 81.61           C\\nATOM    341  N   SER A  46       8.896  -4.228  30.053  1.00129.82           N\\nATOM    342  CA  SER A  46       8.966  -2.774  29.842  1.00129.82           C\\nATOM    343  C   SER A  46       9.843  -2.047  30.889  1.00129.82           C\\nATOM    344  O   SER A  46      10.348  -0.960  30.602  1.00129.82           O\\nATOM    345  CB  SER A  46       7.537  -2.197  29.775  1.00129.82           C\\nATOM    346  OG  SER A  46       6.852  -2.288  31.006  1.00129.82           O\\nATOM    347  N   GLU A  47      10.085  -2.678  32.053  0.24115.27           N\\nATOM    348  CA  GLU A  47      11.016  -2.241  33.092  0.24115.27           C\\nATOM    349  C   GLU A  47      12.472  -2.529  32.682  0.24115.27           C\\nATOM    350  O   GLU A  47      13.331  -1.671  32.886  0.24115.27           O\\nATOM    351  CB  GLU A  47      10.738  -2.975  34.430  0.24115.27           C\\nATOM    352  CG  GLU A  47       9.260  -3.032  34.877  0.24115.27           C\\nATOM    353  CD  GLU A  47       8.515  -4.252  34.331  0.24115.27           C\\nATOM    354  OE1 GLU A  47       7.712  -4.066  33.392  0.24115.27           O\\nATOM    355  OE2 GLU A  47       8.792  -5.362  34.835  0.24115.27           O\\nATOM    356  N   ASP A  48      12.741  -3.701  32.066  1.00 71.53           N\\nATOM    357  CA  ASP A  48      14.051  -4.083  31.520  1.00 71.53           C\\nATOM    358  C   ASP A  48      14.489  -3.169  30.372  1.00 71.53           C\\nATOM    359  O   ASP A  48      15.680  -2.891  30.265  1.00 71.53           O\\nATOM    360  CB  ASP A  48      14.067  -5.540  30.991  1.00 71.53           C\\nATOM    361  CG  ASP A  48      13.856  -6.642  32.028  1.00 71.53           C\\nATOM    362  OD1 ASP A  48      14.257  -6.433  33.193  1.00 71.53           O\\nATOM    363  OD2 ASP A  48      13.338  -7.704  31.618  1.00 71.53           O\\nATOM    364  N   MET A  49      13.544  -2.670  29.553  1.00 58.60           N\\nATOM    365  CA  MET A  49      13.799  -1.781  28.418  1.00 58.60           C\\nATOM    366  C   MET A  49      14.432  -0.431  28.786  1.00 58.60           C\\nATOM    367  O   MET A  49      15.036   0.188  27.916  1.00 58.60           O\\nATOM    368  CB  MET A  49      12.495  -1.539  27.624  1.00 58.60           C\\nATOM    369  CG  MET A  49      12.114  -2.702  26.700  1.00 58.60           C\\nATOM    370  SD  MET A  49      13.363  -3.138  25.460  1.00 58.60           S\\nATOM    371  CE  MET A  49      13.307  -1.700  24.363  1.00 58.60           C\\nATOM    372  N   LEU A  50      14.351   0.008  30.053  1.00 54.26           N\\nATOM    373  CA  LEU A  50      14.956   1.247  30.540  1.00 54.26           C\\nATOM    374  C   LEU A  50      16.495   1.183  30.577  1.00 54.26           C\\nATOM    375  O   LEU A  50      17.134   2.199  30.308  1.00 54.26           O\\nATOM    376  CB  LEU A  50      14.428   1.549  31.959  1.00 54.26           C\\nATOM    377  CG  LEU A  50      12.905   1.797  32.043  1.00 54.26           C\\nATOM    378  CD1 LEU A  50      12.439   1.803  33.506  1.00 54.26           C\\nATOM    379  CD2 LEU A  50      12.489   3.102  31.342  1.00 54.26           C\\nATOM    380  N   ASN A  51      17.087   0.011  30.874  1.00 45.22           N\\nATOM    381  CA  ASN A  51      18.532  -0.239  30.847  1.00 45.22           C\\nATOM    382  C   ASN A  51      18.802  -1.757  30.744  1.00 45.22           C\\nATOM    383  O   ASN A  51      19.208  -2.359  31.739  1.00 45.22           O\\nATOM    384  CB  ASN A  51      19.213   0.459  32.061  1.00 45.22           C\\nATOM    385  CG  ASN A  51      20.748   0.499  32.033  1.00 45.22           C\\nATOM    386  OD1 ASN A  51      21.376   0.518  33.088  1.00 45.22           O\\nATOM    387  ND2 ASN A  51      21.369   0.554  30.851  1.00 45.22           N\\nATOM    388  N   PRO A  52      18.574  -2.398  29.574  1.00 25.46           N\\nATOM    389  CA  PRO A  52      18.727  -3.850  29.412  1.00 25.46           C\\nATOM    390  C   PRO A  52      20.194  -4.235  29.150  1.00 25.46           C\\nATOM    391  O   PRO A  52      20.829  -3.696  28.241  1.00 25.46           O\\nATOM    392  CB  PRO A  52      17.870  -4.181  28.179  1.00 25.46           C\\nATOM    393  CG  PRO A  52      17.896  -2.913  27.335  1.00 25.46           C\\nATOM    394  CD  PRO A  52      18.044  -1.786  28.353  1.00 25.46           C\\nATOM    395  N   ASN A  53      20.706  -5.215  29.910  1.00 27.98           N\\nATOM    396  CA  ASN A  53      21.904  -5.977  29.567  1.00 27.98           C\\nATOM    397  C   ASN A  53      21.372  -7.279  28.956  1.00 27.98           C\\nATOM    398  O   ASN A  53      20.917  -8.174  29.668  1.00 27.98           O\\nATOM    399  CB  ASN A  53      22.731  -6.275  30.839  1.00 27.98           C\\nATOM    400  CG  ASN A  53      24.128  -6.819  30.499  1.00 27.98           C\\nATOM    401  OD1 ASN A  53      24.265  -7.819  29.795  1.00 27.98           O\\nATOM    402  ND2 ASN A  53      25.176  -6.144  30.971  1.00 27.98           N\\nATOM    403  N   TYR A  54      21.417  -7.359  27.621  1.00 18.60           N\\nATOM    404  CA  TYR A  54      20.811  -8.430  26.840  1.00 18.60           C\\nATOM    405  C   TYR A  54      21.492  -9.789  26.997  1.00 18.60           C\\nATOM    406  O   TYR A  54      20.784 -10.791  26.984  1.00 18.60           O\\nATOM    407  CB  TYR A  54      20.808  -8.019  25.364  1.00 18.60           C\\nATOM    408  CG  TYR A  54      19.674  -7.069  25.064  1.00 18.60           C\\nATOM    409  CD1 TYR A  54      18.369  -7.585  24.977  1.00 18.60           C\\nATOM    410  CD2 TYR A  54      19.899  -5.688  24.900  1.00 18.60           C\\nATOM    411  CE1 TYR A  54      17.293  -6.723  24.735  1.00 18.60           C\\nATOM    412  CE2 TYR A  54      18.816  -4.828  24.643  1.00 18.60           C\\nATOM    413  CZ  TYR A  54      17.508  -5.345  24.572  1.00 18.60           C\\nATOM    414  OH  TYR A  54      16.445  -4.523  24.356  1.00 18.60           O\\nATOM    415  N   GLU A  55      22.820  -9.838  27.200  1.00 21.06           N\\nATOM    416  CA  GLU A  55      23.549 -11.078  27.469  1.00 21.06           C\\nATOM    417  C   GLU A  55      23.145 -11.658  28.836  1.00 21.06           C\\nATOM    418  O   GLU A  55      22.861 -12.850  28.916  1.00 21.06           O\\nATOM    419  CB  GLU A  55      25.074 -10.819  27.441  1.00 21.06           C\\nATOM    420  CG  GLU A  55      25.679 -10.604  26.033  1.00 21.06           C\\nATOM    421  CD  GLU A  55      25.795 -11.857  25.153  1.00 21.06           C\\nATOM    422  OE1 GLU A  55      25.271 -12.921  25.548  1.00 21.06           O\\nATOM    423  OE2 GLU A  55      26.401 -11.719  24.070  1.00 21.06           O\\nATOM    424  N   ASP A  56      23.049 -10.824  29.887  1.00 23.58           N\\nATOM    425  CA  ASP A  56      22.645 -11.249  31.233  1.00 23.58           C\\nATOM    426  C   ASP A  56      21.164 -11.646  31.327  1.00 23.58           C\\nATOM    427  O   ASP A  56      20.850 -12.597  32.043  1.00 23.58           O\\nATOM    428  CB  ASP A  56      22.945 -10.143  32.270  1.00 23.58           C\\nATOM    429  CG  ASP A  56      24.434  -9.834  32.473  1.00 23.58           C\\nATOM    430  OD1 ASP A  56      25.278 -10.653  32.048  1.00 23.58           O\\nATOM    431  OD2 ASP A  56      24.701  -8.779  33.087  1.00 23.58           O\\nATOM    432  N   LEU A  57      20.262 -10.979  30.585  1.00 22.50           N\\nATOM    433  CA  LEU A  57      18.845 -11.348  30.509  1.00 22.50           C\\nATOM    434  C   LEU A  57      18.635 -12.660  29.732  1.00 22.50           C\\nATOM    435  O   LEU A  57      17.730 -13.420  30.076  1.00 22.50           O\\nATOM    436  CB  LEU A  57      18.038 -10.228  29.814  1.00 22.50           C\\nATOM    437  CG  LEU A  57      17.846  -8.940  30.644  1.00 22.50           C\\nATOM    438  CD1 LEU A  57      17.314  -7.808  29.755  1.00 22.50           C\\nATOM    439  CD2 LEU A  57      16.916  -9.141  31.853  1.00 22.50           C\\nATOM    440  N   LEU A  58      19.473 -12.941  28.718  1.00 21.26           N\\nATOM    441  CA  LEU A  58      19.356 -14.113  27.853  1.00 21.26           C\\nATOM    442  C   LEU A  58      19.872 -15.399  28.526  1.00 21.26           C\\nATOM    443  O   LEU A  58      19.359 -16.464  28.191  1.00 21.26           O\\nATOM    444  CB  LEU A  58      20.131 -13.838  26.545  1.00 21.26           C\\nATOM    445  CG  LEU A  58      19.918 -14.838  25.389  1.00 21.26           C\\nATOM    446  CD1 LEU A  58      18.446 -14.896  24.947  1.00 21.26           C\\nATOM    447  CD2 LEU A  58      20.812 -14.453  24.203  1.00 21.26           C\\nATOM    448  N   ILE A  59      20.799 -15.319  29.508  1.00 21.86           N\\nATOM    449  CA  ILE A  59      21.253 -16.439  30.360  1.00 21.86           C\\nATOM    450  C   ILE A  59      20.081 -17.152  31.069  1.00 21.86           C\\nATOM    451  O   ILE A  59      20.100 -18.373  31.221  1.00 21.86           O\\nATOM    452  CB  ILE A  59      22.343 -15.951  31.373  1.00 21.86           C\\nATOM    453  CG1 ILE A  59      23.673 -15.641  30.653  1.00 21.86           C\\nATOM    454  CG2 ILE A  59      22.608 -16.927  32.547  1.00 21.86           C\\nATOM    455  CD1 ILE A  59      24.694 -14.864  31.501  1.00 21.86           C\\nATOM    456  N   ARG A  60      19.062 -16.381  31.477  1.00 21.63           N\\nATOM    457  CA  ARG A  60      17.888 -16.839  32.216  1.00 21.63           C\\nATOM    458  C   ARG A  60      16.792 -17.419  31.299  1.00 21.63           C\\nATOM    459  O   ARG A  60      15.768 -17.873  31.811  1.00 21.63           O\\nATOM    460  CB  ARG A  60      17.298 -15.627  32.977  1.00 21.63           C\\nATOM    461  CG  ARG A  60      18.307 -14.845  33.845  1.00 21.63           C\\nATOM    462  CD  ARG A  60      18.878 -15.654  35.023  1.00 21.63           C\\nATOM    463  NE  ARG A  60      20.213 -15.172  35.403  1.00 21.63           N\\nATOM    464  CZ  ARG A  60      21.157 -15.874  36.052  1.00 21.63           C\\nATOM    465  NH1 ARG A  60      22.335 -15.292  36.302  1.00 21.63           N\\nATOM    466  NH2 ARG A  60      20.948 -17.138  36.448  1.00 21.63           N\\nATOM    467  N   LYS A  61      17.000 -17.429  29.970  1.00 21.28           N\\nATOM    468  CA  LYS A  61      16.096 -18.007  28.979  1.00 21.28           C\\nATOM    469  C   LYS A  61      16.731 -19.271  28.382  1.00 21.28           C\\nATOM    470  O   LYS A  61      17.888 -19.262  27.964  1.00 21.28           O\\nATOM    471  CB  LYS A  61      15.845 -16.992  27.841  1.00 21.28           C\\nATOM    472  CG  LYS A  61      15.367 -15.599  28.293  1.00 21.28           C\\nATOM    473  CD  LYS A  61      14.139 -15.615  29.219  1.00 21.28           C\\nATOM    474  CE  LYS A  61      13.617 -14.204  29.515  1.00 21.28           C\\nATOM    475  NZ  LYS A  61      12.505 -14.228  30.479  1.00 21.28           N\\nATOM    476  N   SER A  62      15.938 -20.347  28.326  1.00 21.63           N\\nATOM    477  CA  SER A  62      16.206 -21.599  27.620  1.00 21.63           C\\nATOM    478  C   SER A  62      15.326 -21.659  26.357  1.00 21.63           C\\nATOM    479  O   SER A  62      14.474 -20.793  26.154  1.00 21.63           O\\nATOM    480  CB  SER A  62      15.830 -22.761  28.560  1.00 21.63           C\\nATOM    481  OG  SER A  62      16.829 -22.922  29.545  1.00 21.63           O\\nATOM    482  N   ASN A  63      15.517 -22.691  25.511  1.00 20.19           N\\nATOM    483  CA  ASN A  63      14.747 -22.927  24.277  1.00 20.19           C\\nATOM    484  C   ASN A  63      13.227 -23.001  24.498  1.00 20.19           C\\nATOM    485  O   ASN A  63      12.469 -22.431  23.713  1.00 20.19           O\\nATOM    486  CB  ASN A  63      15.204 -24.217  23.556  1.00 20.19           C\\nATOM    487  CG  ASN A  63      16.641 -24.195  23.041  1.00 20.19           C\\nATOM    488  OD1 ASN A  63      17.109 -23.174  22.551  1.00 20.19           O\\nATOM    489  ND2 ASN A  63      17.355 -25.315  23.156  1.00 20.19           N\\nATOM    490  N   HIS A  64      12.792 -23.669  25.582  1.00 21.45           N\\nATOM    491  CA  HIS A  64      11.385 -23.854  25.940  1.00 21.45           C\\nATOM    492  C   HIS A  64      10.690 -22.563  26.426  1.00 21.45           C\\nATOM    493  O   HIS A  64       9.460 -22.523  26.441  1.00 21.45           O\\nATOM    494  CB  HIS A  64      11.252 -24.966  27.005  1.00 21.45           C\\nATOM    495  CG  HIS A  64      11.820 -24.677  28.375  1.00 21.45           C\\nATOM    496  ND1 HIS A  64      13.103 -25.081  28.759  1.00 21.45           N\\nATOM    497  CD2 HIS A  64      11.229 -24.017  29.432  1.00 21.45           C\\nATOM    498  CE1 HIS A  64      13.235 -24.660  30.010  1.00 21.45           C\\nATOM    499  NE2 HIS A  64      12.158 -24.014  30.456  1.00 21.45           N\\nATOM    500  N   ASN A  65      11.450 -21.510  26.788  1.00 21.78           N\\nATOM    501  CA  ASN A  65      10.916 -20.199  27.180  1.00 21.78           C\\nATOM    502  C   ASN A  65      10.584 -19.315  25.965  1.00 21.78           C\\nATOM    503  O   ASN A  65       9.896 -18.311  26.136  1.00 21.78           O\\nATOM    504  CB  ASN A  65      11.938 -19.430  28.053  1.00 21.78           C\\nATOM    505  CG  ASN A  65      12.235 -20.114  29.391  1.00 21.78           C\\nATOM    506  OD1 ASN A  65      13.382 -20.434  29.686  1.00 21.78           O\\nATOM    507  ND2 ASN A  65      11.213 -20.304  30.227  1.00 21.78           N\\nATOM    508  N   PHE A  66      11.029 -19.681  24.752  1.00 15.07           N\\nATOM    509  CA  PHE A  66      10.676 -19.011  23.504  1.00 15.07           C\\nATOM    510  C   PHE A  66       9.555 -19.822  22.852  1.00 15.07           C\\nATOM    511  O   PHE A  66       9.805 -20.817  22.174  1.00 15.07           O\\nATOM    512  CB  PHE A  66      11.911 -18.992  22.585  1.00 15.07           C\\nATOM    513  CG  PHE A  66      13.014 -18.107  23.123  1.00 15.07           C\\nATOM    514  CD1 PHE A  66      12.873 -16.706  23.078  1.00 15.07           C\\nATOM    515  CD2 PHE A  66      14.159 -18.681  23.712  1.00 15.07           C\\nATOM    516  CE1 PHE A  66      13.868 -15.884  23.633  1.00 15.07           C\\nATOM    517  CE2 PHE A  66      15.151 -17.857  24.273  1.00 15.07           C\\nATOM    518  CZ  PHE A  66      15.004 -16.457  24.235  1.00 15.07           C\\nATOM    519  N   LEU A  67       8.309 -19.395  23.086  1.00 13.49           N\\nATOM    520  CA  LEU A  67       7.105 -20.019  22.556  1.00 13.49           C\\nATOM    521  C   LEU A  67       6.845 -19.423  21.169  1.00 13.49           C\\nATOM    522  O   LEU A  67       6.550 -18.236  21.048  1.00 13.49           O\\nATOM    523  CB  LEU A  67       5.916 -19.685  23.479  1.00 13.49           C\\nATOM    524  CG  LEU A  67       6.043 -20.172  24.936  1.00 13.49           C\\nATOM    525  CD1 LEU A  67       4.807 -19.739  25.734  1.00 13.49           C\\nATOM    526  CD2 LEU A  67       6.247 -21.691  25.039  1.00 13.49           C\\nATOM    527  N   VAL A  68       6.988 -20.253  20.129  1.00 12.01           N\\nATOM    528  CA  VAL A  68       6.852 -19.873  18.728  1.00 12.01           C\\nATOM    529  C   VAL A  68       5.604 -20.572  18.173  1.00 12.01           C\\nATOM    530  O   VAL A  68       5.460 -21.783  18.332  1.00 12.01           O\\nATOM    531  CB  VAL A  68       8.132 -20.278  17.958  1.00 12.01           C\\nATOM    532  CG1 VAL A  68       8.049 -19.922  16.468  1.00 12.01           C\\nATOM    533  CG2 VAL A  68       9.398 -19.630  18.551  1.00 12.01           C\\nATOM    534  N   GLN A  69       4.721 -19.809  17.512  1.00 11.31           N\\nATOM    535  CA  GLN A  69       3.434 -20.269  17.000  1.00 11.31           C\\nATOM    536  C   GLN A  69       3.184 -19.705  15.597  1.00 11.31           C\\nATOM    537  O   GLN A  69       3.001 -18.499  15.436  1.00 11.31           O\\nATOM    538  CB  GLN A  69       2.341 -19.972  18.049  1.00 11.31           C\\nATOM    539  CG  GLN A  69       0.903 -20.336  17.617  1.00 11.31           C\\nATOM    540  CD  GLN A  69      -0.167 -20.096  18.692  1.00 11.31           C\\nATOM    541  OE1 GLN A  69      -1.355 -20.101  18.382  1.00 11.31           O\\nATOM    542  NE2 GLN A  69       0.221 -19.861  19.947  1.00 11.31           N\\nATOM    543  N   ALA A  70       3.162 -20.596  14.594  1.00 15.60           N\\nATOM    544  CA  ALA A  70       2.822 -20.320  13.202  1.00 15.60           C\\nATOM    545  C   ALA A  70       1.363 -20.738  12.995  1.00 15.60           C\\nATOM    546  O   ALA A  70       1.038 -21.917  13.113  1.00 15.60           O\\nATOM    547  CB  ALA A  70       3.711 -21.212  12.317  1.00 15.60           C\\nATOM    548  N   GLY A  71       0.471 -19.772  12.728  1.00 29.79           N\\nATOM    549  CA  GLY A  71      -0.974 -19.990  12.694  1.00 29.79           C\\nATOM    550  C   GLY A  71      -1.498 -20.316  14.097  1.00 29.79           C\\nATOM    551  O   GLY A  71      -1.397 -19.485  14.997  1.00 29.79           O\\nATOM    552  N   ASN A  72      -2.020 -21.540  14.272  0.24 31.03           N\\nATOM    553  CA  ASN A  72      -2.456 -22.136  15.541  0.24 31.03           C\\nATOM    554  C   ASN A  72      -1.735 -23.471  15.820  0.24 31.03           C\\nATOM    555  O   ASN A  72      -2.260 -24.313  16.549  0.24 31.03           O\\nATOM    556  CB  ASN A  72      -4.009 -22.201  15.629  0.24 31.03           C\\nATOM    557  CG  ASN A  72      -4.698 -20.953  16.204  0.24 31.03           C\\nATOM    558  OD1 ASN A  72      -5.635 -21.080  16.989  0.24 31.03           O\\nATOM    559  ND2 ASN A  72      -4.288 -19.749  15.805  0.24 31.03           N\\nATOM    560  N  AVAL A  73      -0.514 -23.649  15.278  0.25 23.65           N\\nATOM    561  CA AVAL A  73       0.378 -24.779  15.545  0.25 23.65           C\\nATOM    562  C  AVAL A  73       1.704 -24.255  16.110  0.25 23.65           C\\nATOM    563  O  AVAL A  73       2.295 -23.312  15.585  0.25 23.65           O\\nATOM    564  CB AVAL A  73       0.520 -25.718  14.313  0.25 23.65           C\\nATOM    565  CG1AVAL A  73       1.058 -25.039  13.042  0.25 23.65           C\\nATOM    566  CG2AVAL A  73       1.352 -26.972  14.634  0.25 23.65           C\\nATOM    567  N   GLN A  74       2.150 -24.887  17.203  1.00 16.27           N\\nATOM    568  CA  GLN A  74       3.329 -24.520  17.972  1.00 16.27           C\\nATOM    569  C   GLN A  74       4.567 -25.197  17.353  1.00 16.27           C\\nATOM    570  O   GLN A  74       4.554 -26.405  17.108  1.00 16.27           O\\nATOM    571  CB  GLN A  74       3.105 -25.052  19.408  1.00 16.27           C\\nATOM    572  CG  GLN A  74       3.909 -24.300  20.483  1.00 16.27           C\\nATOM    573  CD  GLN A  74       3.370 -22.897  20.783  1.00 16.27           C\\nATOM    574  OE1 GLN A  74       2.195 -22.604  20.573  1.00 16.27           O\\nATOM    575  NE2 GLN A  74       4.237 -22.016  21.278  1.00 16.27           N\\nATOM    576  N   LEU A  75       5.630 -24.417  17.107  1.00 13.26           N\\nATOM    577  CA  LEU A  75       6.916 -24.896  16.596  1.00 13.26           C\\nATOM    578  C   LEU A  75       7.854 -25.140  17.782  1.00 13.26           C\\nATOM    579  O   LEU A  75       7.934 -24.315  18.693  1.00 13.26           O\\nATOM    580  CB  LEU A  75       7.573 -23.811  15.711  1.00 13.26           C\\nATOM    581  CG  LEU A  75       6.730 -23.319  14.516  1.00 13.26           C\\nATOM    582  CD1 LEU A  75       7.562 -22.365  13.648  1.00 13.26           C\\nATOM    583  CD2 LEU A  75       6.175 -24.466  13.654  1.00 13.26           C\\nATOM    584  N   ARG A  76       8.573 -26.271  17.762  1.00 12.52           N\\nATOM    585  CA  ARG A  76       9.502 -26.664  18.815  1.00 12.52           C\\nATOM    586  C   ARG A  76      10.859 -26.011  18.537  1.00 12.52           C\\nATOM    587  O   ARG A  76      11.477 -26.297  17.514  1.00 12.52           O\\nATOM    588  CB  ARG A  76       9.653 -28.196  18.761  1.00 12.52           C\\nATOM    589  CG  ARG A  76      10.564 -28.792  19.853  1.00 12.52           C\\nATOM    590  CD  ARG A  76      10.715 -30.314  19.731  1.00 12.52           C\\nATOM    591  NE  ARG A  76       9.409 -30.984  19.772  1.00 12.52           N\\nATOM    592  CZ  ARG A  76       8.696 -31.276  20.871  1.00 12.52           C\\nATOM    593  NH1 ARG A  76       9.161 -31.009  22.100  1.00 12.52           N\\nATOM    594  NH2 ARG A  76       7.490 -31.837  20.725  1.00 12.52           N\\nATOM    595  N   VAL A  77      11.318 -25.160  19.467  1.00 12.81           N\\nATOM    596  CA  VAL A  77      12.620 -24.500  19.431  1.00 12.81           C\\nATOM    597  C   VAL A  77      13.660 -25.495  19.976  1.00 12.81           C\\nATOM    598  O   VAL A  77      13.496 -26.046  21.063  1.00 12.81           O\\nATOM    599  CB  VAL A  77      12.573 -23.209  20.295  1.00 12.81           C\\nATOM    600  CG1 VAL A  77      13.917 -22.453  20.330  1.00 12.81           C\\nATOM    601  CG2 VAL A  77      11.483 -22.243  19.797  1.00 12.81           C\\nATOM    602  N   ILE A  78      14.724 -25.716  19.195  1.00 16.19           N\\nATOM    603  CA  ILE A  78      15.823 -26.648  19.457  1.00 16.19           C\\nATOM    604  C   ILE A  78      17.171 -25.914  19.583  1.00 16.19           C\\nATOM    605  O   ILE A  78      18.205 -26.560  19.752  1.00 16.19           O\\nATOM    606  CB  ILE A  78      15.847 -27.748  18.360  1.00 16.19           C\\nATOM    607  CG1 ILE A  78      16.065 -27.185  16.934  1.00 16.19           C\\nATOM    608  CG2 ILE A  78      14.578 -28.620  18.435  1.00 16.19           C\\nATOM    609  CD1 ILE A  78      16.482 -28.244  15.906  1.00 16.19           C\\nATOM    610  N   GLY A  79      17.168 -24.578  19.506  1.00 16.66           N\\nATOM    611  CA  GLY A  79      18.345 -23.743  19.647  1.00 16.66           C\\nATOM    612  C   GLY A  79      17.952 -22.270  19.651  1.00 16.66           C\\nATOM    613  O   GLY A  79      16.926 -21.894  19.089  1.00 16.66           O\\nATOM    614  N   HIS A  80      18.784 -21.434  20.282  1.00 15.01           N\\nATOM    615  CA  HIS A  80      18.674 -19.982  20.257  1.00 15.01           C\\nATOM    616  C   HIS A  80      20.058 -19.338  20.335  1.00 15.01           C\\nATOM    617  O   HIS A  80      20.942 -19.826  21.040  1.00 15.01           O\\nATOM    618  CB  HIS A  80      17.668 -19.438  21.292  1.00 15.01           C\\nATOM    619  CG  HIS A  80      18.096 -19.535  22.732  1.00 15.01           C\\nATOM    620  ND1 HIS A  80      18.014 -20.700  23.466  1.00 15.01           N\\nATOM    621  CD2 HIS A  80      18.657 -18.605  23.577  1.00 15.01           C\\nATOM    622  CE1 HIS A  80      18.489 -20.437  24.683  1.00 15.01           C\\nATOM    623  NE2 HIS A  80      18.903 -19.185  24.821  1.00 15.01           N\\nATOM    624  N   SER A  81      20.222 -18.223  19.615  1.00 13.43           N\\nATOM    625  CA  SER A  81      21.430 -17.414  19.574  1.00 13.43           C\\nATOM    626  C   SER A  81      21.068 -15.957  19.288  1.00 13.43           C\\nATOM    627  O   SER A  81      20.159 -15.691  18.504  1.00 13.43           O\\nATOM    628  CB  SER A  81      22.458 -18.033  18.603  1.00 13.43           C\\nATOM    629  OG  SER A  81      22.104 -17.829  17.248  1.00 13.43           O\\nATOM    630  N   MET A  82      21.807 -15.021  19.895  1.00 12.46           N\\nATOM    631  CA  MET A  82      21.705 -13.592  19.641  1.00 12.46           C\\nATOM    632  C   MET A  82      22.793 -13.204  18.636  1.00 12.46           C\\nATOM    633  O   MET A  82      23.978 -13.444  18.869  1.00 12.46           O\\nATOM    634  CB  MET A  82      21.945 -12.861  20.973  1.00 12.46           C\\nATOM    635  CG  MET A  82      21.706 -11.345  20.911  1.00 12.46           C\\nATOM    636  SD  MET A  82      21.487 -10.572  22.534  1.00 12.46           S\\nATOM    637  CE  MET A  82      23.060 -10.962  23.335  1.00 12.46           C\\nATOM    638  N   GLN A  83      22.366 -12.582  17.530  1.00 13.05           N\\nATOM    639  CA  GLN A  83      23.221 -11.925  16.554  1.00 13.05           C\\nATOM    640  C   GLN A  83      22.810 -10.454  16.545  1.00 13.05           C\\nATOM    641  O   GLN A  83      21.782 -10.097  15.970  1.00 13.05           O\\nATOM    642  CB  GLN A  83      23.063 -12.586  15.172  1.00 13.05           C\\nATOM    643  CG  GLN A  83      24.013 -11.974  14.120  1.00 13.05           C\\nATOM    644  CD  GLN A  83      23.820 -12.526  12.710  1.00 13.05           C\\nATOM    645  OE1 GLN A  83      22.913 -13.312  12.448  1.00 13.05           O\\nATOM    646  NE2 GLN A  83      24.692 -12.119  11.789  1.00 13.05           N\\nATOM    647  N   ASN A  84      23.611  -9.619  17.222  1.00 12.88           N\\nATOM    648  CA  ASN A  84      23.461  -8.169  17.329  1.00 12.88           C\\nATOM    649  C   ASN A  84      22.192  -7.841  18.133  1.00 12.88           C\\nATOM    650  O   ASN A  84      22.163  -8.123  19.330  1.00 12.88           O\\nATOM    651  CB  ASN A  84      23.610  -7.458  15.959  1.00 12.88           C\\nATOM    652  CG  ASN A  84      24.972  -7.714  15.305  1.00 12.88           C\\nATOM    653  OD1 ASN A  84      26.009  -7.359  15.862  1.00 12.88           O\\nATOM    654  ND2 ASN A  84      24.981  -8.353  14.135  1.00 12.88           N\\nATOM    655  N   CYS A  85      21.143  -7.304  17.486  1.00 11.81           N\\nATOM    656  CA  CYS A  85      19.867  -6.944  18.111  1.00 11.81           C\\nATOM    657  C   CYS A  85      18.694  -7.834  17.660  1.00 11.81           C\\nATOM    658  O   CYS A  85      17.543  -7.470  17.896  1.00 11.81           O\\nATOM    659  CB  CYS A  85      19.560  -5.443  17.936  1.00 11.81           C\\nATOM    660  SG  CYS A  85      20.879  -4.427  18.648  1.00 11.81           S\\nATOM    661  N   VAL A  86      18.956  -9.020  17.083  1.00 10.64           N\\nATOM    662  CA  VAL A  86      17.947 -10.056  16.835  1.00 10.64           C\\nATOM    663  C   VAL A  86      18.365 -11.372  17.492  1.00 10.64           C\\nATOM    664  O   VAL A  86      19.551 -11.694  17.564  1.00 10.64           O\\nATOM    665  CB  VAL A  86      17.602 -10.226  15.328  1.00 10.64           C\\nATOM    666  CG1 VAL A  86      16.930  -8.972  14.748  1.00 10.64           C\\nATOM    667  CG2 VAL A  86      18.769 -10.720  14.451  1.00 10.64           C\\nATOM    668  N   LEU A  87      17.361 -12.146  17.923  1.00  9.52           N\\nATOM    669  CA  LEU A  87      17.493 -13.548  18.280  1.00  9.52           C\\nATOM    670  C   LEU A  87      17.071 -14.381  17.073  1.00  9.52           C\\nATOM    671  O   LEU A  87      16.028 -14.141  16.465  1.00  9.52           O\\nATOM    672  CB  LEU A  87      16.537 -13.901  19.433  1.00  9.52           C\\nATOM    673  CG  LEU A  87      16.935 -13.282  20.782  1.00  9.52           C\\nATOM    674  CD1 LEU A  87      15.802 -13.458  21.791  1.00  9.52           C\\nATOM    675  CD2 LEU A  87      18.237 -13.853  21.357  1.00  9.52           C\\nATOM    676  N   LYS A  88      17.897 -15.383  16.770  1.00  9.68           N\\nATOM    677  CA  LYS A  88      17.646 -16.440  15.810  1.00  9.68           C\\nATOM    678  C   LYS A  88      17.280 -17.664  16.650  1.00  9.68           C\\nATOM    679  O   LYS A  88      18.110 -18.160  17.407  1.00  9.68           O\\nATOM    680  CB  LYS A  88      18.956 -16.705  15.046  1.00  9.68           C\\nATOM    681  CG  LYS A  88      19.435 -15.460  14.282  1.00  9.68           C\\nATOM    682  CD  LYS A  88      20.559 -15.745  13.284  1.00  9.68           C\\nATOM    683  CE  LYS A  88      21.860 -16.225  13.938  1.00  9.68           C\\nATOM    684  NZ  LYS A  88      22.956 -16.251  12.958  1.00  9.68           N\\nATOM    685  N   LEU A  89      16.023 -18.108  16.549  1.00  9.20           N\\nATOM    686  CA  LEU A  89      15.490 -19.270  17.244  1.00  9.20           C\\nATOM    687  C   LEU A  89      15.430 -20.396  16.208  1.00  9.20           C\\nATOM    688  O   LEU A  89      14.580 -20.372  15.319  1.00  9.20           O\\nATOM    689  CB  LEU A  89      14.056 -18.956  17.741  1.00  9.20           C\\nATOM    690  CG  LEU A  89      13.911 -17.698  18.623  1.00  9.20           C\\nATOM    691  CD1 LEU A  89      12.432 -17.432  18.929  1.00  9.20           C\\nATOM    692  CD2 LEU A  89      14.733 -17.793  19.915  1.00  9.20           C\\nATOM    693  N   LYS A  90      16.329 -21.386  16.319  1.00 11.44           N\\nATOM    694  CA  LYS A  90      16.297 -22.581  15.483  1.00 11.44           C\\nATOM    695  C   LYS A  90      15.136 -23.465  15.958  1.00 11.44           C\\nATOM    696  O   LYS A  90      15.048 -23.780  17.143  1.00 11.44           O\\nATOM    697  CB  LYS A  90      17.619 -23.370  15.595  1.00 11.44           C\\nATOM    698  CG  LYS A  90      17.744 -24.425  14.477  1.00 11.44           C\\nATOM    699  CD  LYS A  90      18.917 -25.391  14.668  1.00 11.44           C\\nATOM    700  CE  LYS A  90      19.080 -26.332  13.469  1.00 11.44           C\\nATOM    701  NZ  LYS A  90      20.142 -27.323  13.700  1.00 11.44           N\\nATOM    702  N   VAL A  91      14.268 -23.856  15.019  1.00 10.64           N\\nATOM    703  CA  VAL A  91      13.133 -24.750  15.217  1.00 10.64           C\\nATOM    704  C   VAL A  91      13.382 -26.066  14.460  1.00 10.64           C\\nATOM    705  O   VAL A  91      14.226 -26.121  13.567  1.00 10.64           O\\nATOM    706  CB  VAL A  91      11.815 -24.046  14.792  1.00 10.64           C\\nATOM    707  CG1 VAL A  91      11.378 -22.984  15.817  1.00 10.64           C\\nATOM    708  CG2 VAL A  91      11.867 -23.427  13.386  1.00 10.64           C\\nATOM    709  N   ASP A  92      12.679 -27.144  14.845  1.00 10.85           N\\nATOM    710  CA  ASP A  92      12.873 -28.499  14.303  1.00 10.85           C\\nATOM    711  C   ASP A  92      12.214 -28.698  12.919  1.00 10.85           C\\nATOM    712  O   ASP A  92      12.413 -29.744  12.302  1.00 10.85           O\\nATOM    713  CB  ASP A  92      12.305 -29.535  15.312  1.00 10.85           C\\nATOM    714  CG  ASP A  92      10.778 -29.677  15.436  1.00 10.85           C\\nATOM    715  OD1 ASP A  92      10.048 -28.711  15.118  1.00 10.85           O\\nATOM    716  OD2 ASP A  92      10.366 -30.770  15.881  1.00 10.85           O\\nATOM    717  N   THR A  93      11.423 -27.718  12.457  1.00 10.73           N\\nATOM    718  CA  THR A  93      10.646 -27.758  11.228  1.00 10.73           C\\nATOM    719  C   THR A  93      11.094 -26.576  10.366  1.00 10.73           C\\nATOM    720  O   THR A  93      11.039 -25.427  10.808  1.00 10.73           O\\nATOM    721  CB  THR A  93       9.139 -27.683  11.587  1.00 10.73           C\\nATOM    722  OG1 THR A  93       8.791 -28.809  12.370  1.00 10.73           O\\nATOM    723  CG2 THR A  93       8.204 -27.702  10.366  1.00 10.73           C\\nATOM    724  N   ALA A  94      11.485 -26.860   9.116  1.00  9.39           N\\nATOM    725  CA  ALA A  94      11.705 -25.853   8.088  1.00  9.39           C\\nATOM    726  C   ALA A  94      10.341 -25.348   7.604  1.00  9.39           C\\nATOM    727  O   ALA A  94       9.436 -26.157   7.400  1.00  9.39           O\\nATOM    728  CB  ALA A  94      12.454 -26.487   6.908  1.00  9.39           C\\nATOM    729  N   ASN A  95      10.185 -24.025   7.428  1.00  9.09           N\\nATOM    730  CA  ASN A  95       8.956 -23.404   6.933  1.00  9.09           C\\nATOM    731  C   ASN A  95       8.704 -23.898   5.493  1.00  9.09           C\\nATOM    732  O   ASN A  95       9.539 -23.635   4.625  1.00  9.09           O\\nATOM    733  CB  ASN A  95       9.102 -21.863   6.998  1.00  9.09           C\\nATOM    734  CG  ASN A  95       7.873 -21.067   6.537  1.00  9.09           C\\nATOM    735  OD1 ASN A  95       6.774 -21.600   6.425  1.00  9.09           O\\nATOM    736  ND2 ASN A  95       8.054 -19.787   6.226  1.00  9.09           N\\nATOM    737  N   PRO A  96       7.577 -24.594   5.215  1.00  8.88           N\\nATOM    738  CA  PRO A  96       7.269 -25.109   3.873  1.00  8.88           C\\nATOM    739  C   PRO A  96       6.900 -24.000   2.872  1.00  8.88           C\\nATOM    740  O   PRO A  96       6.959 -24.223   1.664  1.00  8.88           O\\nATOM    741  CB  PRO A  96       6.107 -26.090   4.090  1.00  8.88           C\\nATOM    742  CG  PRO A  96       5.405 -25.591   5.342  1.00  8.88           C\\nATOM    743  CD  PRO A  96       6.512 -24.937   6.166  1.00  8.88           C\\nATOM    744  N   LYS A  97       6.568 -22.806   3.384  1.00 10.91           N\\nATOM    745  CA  LYS A  97       6.253 -21.601   2.635  1.00 10.91           C\\nATOM    746  C   LYS A  97       7.464 -20.651   2.539  1.00 10.91           C\\nATOM    747  O   LYS A  97       7.257 -19.485   2.210  1.00 10.91           O\\nATOM    748  CB  LYS A  97       5.119 -20.864   3.378  1.00 10.91           C\\nATOM    749  CG  LYS A  97       3.811 -21.655   3.507  1.00 10.91           C\\nATOM    750  CD  LYS A  97       2.713 -20.790   4.139  1.00 10.91           C\\nATOM    751  CE  LYS A  97       1.397 -21.544   4.340  1.00 10.91           C\\nATOM    752  NZ  LYS A  97       0.370 -20.655   4.907  1.00 10.91           N\\nATOM    753  N   THR A  98       8.718 -21.094   2.789  1.00  9.74           N\\nATOM    754  CA  THR A  98       9.906 -20.232   2.647  1.00  9.74           C\\nATOM    755  C   THR A  98      10.056 -19.854   1.160  1.00  9.74           C\\nATOM    756  O   THR A  98      10.209 -20.759   0.337  1.00  9.74           O\\nATOM    757  CB  THR A  98      11.212 -20.938   3.105  1.00  9.74           C\\nATOM    758  OG1 THR A  98      11.132 -21.281   4.458  1.00  9.74           O\\nATOM    759  CG2 THR A  98      12.462 -20.044   3.024  1.00  9.74           C\\nATOM    760  N   PRO A  99      10.040 -18.552   0.799  1.00  9.51           N\\nATOM    761  CA  PRO A  99      10.293 -18.115  -0.576  1.00  9.51           C\\nATOM    762  C   PRO A  99      11.806 -18.169  -0.846  1.00  9.51           C\\nATOM    763  O   PRO A  99      12.599 -18.407   0.063  1.00  9.51           O\\nATOM    764  CB  PRO A  99       9.769 -16.669  -0.604  1.00  9.51           C\\nATOM    765  CG  PRO A  99      10.006 -16.154   0.808  1.00  9.51           C\\nATOM    766  CD  PRO A  99       9.821 -17.398   1.678  1.00  9.51           C\\nATOM    767  N   LYS A 100      12.231 -17.869  -2.079  1.00  9.71           N\\nATOM    768  CA  LYS A 100      13.620 -17.485  -2.340  1.00  9.71           C\\nATOM    769  C   LYS A 100      13.861 -16.140  -1.638  1.00  9.71           C\\nATOM    770  O   LYS A 100      12.988 -15.272  -1.677  1.00  9.71           O\\nATOM    771  CB  LYS A 100      13.849 -17.350  -3.851  1.00  9.71           C\\nATOM    772  CG  LYS A 100      13.721 -18.680  -4.604  1.00  9.71           C\\nATOM    773  CD  LYS A 100      13.930 -18.483  -6.108  1.00  9.71           C\\nATOM    774  CE  LYS A 100      13.577 -19.724  -6.934  1.00  9.71           C\\nATOM    775  NZ  LYS A 100      13.624 -19.421  -8.373  1.00  9.71           N\\nATOM    776  N   TYR A 101      14.987 -16.002  -0.926  1.00 10.49           N\\nATOM    777  CA  TYR A 101      15.238 -14.839  -0.085  1.00 10.49           C\\nATOM    778  C   TYR A 101      16.708 -14.445  -0.006  1.00 10.49           C\\nATOM    779  O   TYR A 101      17.608 -15.266  -0.180  1.00 10.49           O\\nATOM    780  CB  TYR A 101      14.545 -14.959   1.294  1.00 10.49           C\\nATOM    781  CG  TYR A 101      15.253 -15.857   2.295  1.00 10.49           C\\nATOM    782  CD1 TYR A 101      16.207 -15.314   3.181  1.00 10.49           C\\nATOM    783  CD2 TYR A 101      14.989 -17.240   2.318  1.00 10.49           C\\nATOM    784  CE1 TYR A 101      16.908 -16.154   4.067  1.00 10.49           C\\nATOM    785  CE2 TYR A 101      15.687 -18.082   3.203  1.00 10.49           C\\nATOM    786  CZ  TYR A 101      16.654 -17.539   4.070  1.00 10.49           C\\nATOM    787  OH  TYR A 101      17.357 -18.354   4.905  1.00 10.49           O\\nATOM    788  N   LYS A 102      16.906 -13.164   0.321  1.00 10.60           N\\nATOM    789  CA  LYS A 102      18.169 -12.531   0.659  1.00 10.60           C\\nATOM    790  C   LYS A 102      17.958 -11.652   1.892  1.00 10.60           C\\nATOM    791  O   LYS A 102      16.854 -11.160   2.132  1.00 10.60           O\\nATOM    792  CB  LYS A 102      18.686 -11.706  -0.534  1.00 10.60           C\\nATOM    793  CG  LYS A 102      19.153 -12.546  -1.730  1.00 10.60           C\\nATOM    794  CD  LYS A 102      19.504 -11.673  -2.939  1.00 10.60           C\\nATOM    795  CE  LYS A 102      19.938 -12.508  -4.150  1.00 10.60           C\\nATOM    796  NZ  LYS A 102      20.153 -11.656  -5.330  1.00 10.60           N\\nATOM    797  N   PHE A 103      19.040 -11.420   2.641  1.00 10.65           N\\nATOM    798  CA  PHE A 103      19.144 -10.363   3.638  1.00 10.65           C\\nATOM    799  C   PHE A 103      20.069  -9.304   3.037  1.00 10.65           C\\nATOM    800  O   PHE A 103      21.222  -9.596   2.714  1.00 10.65           O\\nATOM    801  CB  PHE A 103      19.752 -10.932   4.938  1.00 10.65           C\\nATOM    802  CG  PHE A 103      18.887 -11.910   5.724  1.00 10.65           C\\nATOM    803  CD1 PHE A 103      17.505 -12.063   5.464  1.00 10.65           C\\nATOM    804  CD2 PHE A 103      19.478 -12.671   6.754  1.00 10.65           C\\nATOM    805  CE1 PHE A 103      16.731 -12.962   6.215  1.00 10.65           C\\nATOM    806  CE2 PHE A 103      18.704 -13.578   7.501  1.00 10.65           C\\nATOM    807  CZ  PHE A 103      17.330 -13.725   7.231  1.00 10.65           C\\nATOM    808  N   VAL A 104      19.540  -8.085   2.870  1.00 10.87           N\\nATOM    809  CA  VAL A 104      20.236  -6.930   2.313  1.00 10.87           C\\nATOM    810  C   VAL A 104      20.132  -5.768   3.304  1.00 10.87           C\\nATOM    811  O   VAL A 104      19.079  -5.556   3.902  1.00 10.87           O\\nATOM    812  CB  VAL A 104      19.697  -6.584   0.897  1.00 10.87           C\\nATOM    813  CG1 VAL A 104      19.790  -7.772  -0.076  1.00 10.87           C\\nATOM    814  CG2 VAL A 104      18.272  -6.003   0.888  1.00 10.87           C\\nATOM    815  N   ARG A 105      21.216  -4.999   3.456  1.00 11.78           N\\nATOM    816  CA  ARG A 105      21.216  -3.757   4.215  1.00 11.78           C\\nATOM    817  C   ARG A 105      21.075  -2.618   3.196  1.00 11.78           C\\nATOM    818  O   ARG A 105      21.990  -2.377   2.408  1.00 11.78           O\\nATOM    819  CB  ARG A 105      22.537  -3.646   5.001  1.00 11.78           C\\nATOM    820  CG  ARG A 105      22.558  -2.429   5.939  1.00 11.78           C\\nATOM    821  CD  ARG A 105      23.778  -2.402   6.862  1.00 11.78           C\\nATOM    822  NE  ARG A 105      23.732  -1.225   7.736  1.00 11.78           N\\nATOM    823  CZ  ARG A 105      23.064  -1.131   8.898  1.00 11.78           C\\nATOM    824  NH1 ARG A 105      22.339  -2.149   9.378  1.00 11.78           N\\nATOM    825  NH2 ARG A 105      23.121   0.010   9.591  1.00 11.78           N\\nATOM    826  N   ILE A 106      19.915  -1.941   3.197  1.00 12.87           N\\nATOM    827  CA  ILE A 106      19.598  -0.861   2.261  1.00 12.87           C\\nATOM    828  C   ILE A 106      20.226   0.465   2.714  1.00 12.87           C\\nATOM    829  O   ILE A 106      20.446   0.699   3.901  1.00 12.87           O\\nATOM    830  CB  ILE A 106      18.062  -0.682   2.121  1.00 12.87           C\\nATOM    831  CG1 ILE A 106      17.316  -0.458   3.460  1.00 12.87           C\\nATOM    832  CG2 ILE A 106      17.466  -1.835   1.292  1.00 12.87           C\\nATOM    833  CD1 ILE A 106      15.891   0.073   3.285  1.00 12.87           C\\nATOM    834  N   GLN A 107      20.485   1.344   1.742  0.24 17.34           N\\nATOM    835  CA  GLN A 107      20.991   2.698   1.944  0.24 17.34           C\\nATOM    836  C   GLN A 107      19.823   3.665   2.212  0.24 17.34           C\\nATOM    837  O   GLN A 107      18.683   3.356   1.861  0.24 17.34           O\\nATOM    838  CB  GLN A 107      21.714   3.110   0.639  0.24 17.34           C\\nATOM    839  CG  GLN A 107      23.003   2.294   0.388  0.24 17.34           C\\nATOM    840  CD  GLN A 107      24.071   2.460   1.477  0.24 17.34           C\\nATOM    841  OE1 GLN A 107      24.598   1.472   1.984  0.24 17.34           O\\nATOM    842  NE2 GLN A 107      24.369   3.699   1.873  0.24 17.34           N\\nATOM    843  N   PRO A 108      20.079   4.853   2.806  1.00 15.89           N\\nATOM    844  CA  PRO A 108      19.055   5.885   2.974  1.00 15.89           C\\nATOM    845  C   PRO A 108      18.630   6.462   1.617  1.00 15.89           C\\nATOM    846  O   PRO A 108      19.456   6.700   0.737  1.00 15.89           O\\nATOM    847  CB  PRO A 108      19.674   6.936   3.895  1.00 15.89           C\\nATOM    848  CG  PRO A 108      21.176   6.750   3.768  1.00 15.89           C\\nATOM    849  CD  PRO A 108      21.370   5.299   3.331  1.00 15.89           C\\nATOM    850  N   GLY A 109      17.318   6.644   1.459  1.00 13.75           N\\nATOM    851  CA  GLY A 109      16.639   6.988   0.223  1.00 13.75           C\\nATOM    852  C   GLY A 109      15.957   5.790  -0.442  1.00 13.75           C\\nATOM    853  O   GLY A 109      15.105   5.998  -1.303  1.00 13.75           O\\nATOM    854  N   GLN A 110      16.303   4.548  -0.058  1.00 11.36           N\\nATOM    855  CA  GLN A 110      15.715   3.338  -0.624  1.00 11.36           C\\nATOM    856  C   GLN A 110      14.412   2.995   0.089  1.00 11.36           C\\nATOM    857  O   GLN A 110      14.300   3.074   1.312  1.00 11.36           O\\nATOM    858  CB  GLN A 110      16.665   2.140  -0.449  1.00 11.36           C\\nATOM    859  CG  GLN A 110      17.952   2.250  -1.281  1.00 11.36           C\\nATOM    860  CD  GLN A 110      17.693   2.390  -2.785  1.00 11.36           C\\nATOM    861  OE1 GLN A 110      16.946   1.610  -3.370  1.00 11.36           O\\nATOM    862  NE2 GLN A 110      18.292   3.400  -3.411  1.00 11.36           N\\nATOM    863  N   THR A 111      13.443   2.598  -0.735  1.00 10.90           N\\nATOM    864  CA  THR A 111      12.083   2.246  -0.380  1.00 10.90           C\\nATOM    865  C   THR A 111      11.938   0.732  -0.190  1.00 10.90           C\\nATOM    866  O   THR A 111      12.775  -0.055  -0.632  1.00 10.90           O\\nATOM    867  CB  THR A 111      11.179   2.726  -1.530  1.00 10.90           C\\nATOM    868  OG1 THR A 111      11.603   2.110  -2.728  1.00 10.90           O\\nATOM    869  CG2 THR A 111      11.158   4.256  -1.677  1.00 10.90           C\\nATOM    870  N   PHE A 112      10.858   0.345   0.496  1.00  9.20           N\\nATOM    871  CA  PHE A 112      10.504  -1.031   0.818  1.00  9.20           C\\nATOM    872  C   PHE A 112       9.061  -1.090   1.317  1.00  9.20           C\\nATOM    873  O   PHE A 112       8.545  -0.113   1.857  1.00  9.20           O\\nATOM    874  CB  PHE A 112      11.503  -1.628   1.832  1.00  9.20           C\\nATOM    875  CG  PHE A 112      11.600  -0.900   3.159  1.00  9.20           C\\nATOM    876  CD1 PHE A 112      10.786  -1.294   4.237  1.00  9.20           C\\nATOM    877  CD2 PHE A 112      12.515   0.162   3.321  1.00  9.20           C\\nATOM    878  CE1 PHE A 112      10.930  -0.671   5.489  1.00  9.20           C\\nATOM    879  CE2 PHE A 112      12.638   0.802   4.566  1.00  9.20           C\\nATOM    880  CZ  PHE A 112      11.855   0.370   5.649  1.00  9.20           C\\nATOM    881  N   SER A 113       8.428  -2.257   1.168  1.00  8.34           N\\nATOM    882  CA  SER A 113       7.093  -2.546   1.674  1.00  8.34           C\\nATOM    883  C   SER A 113       7.227  -3.154   3.069  1.00  8.34           C\\nATOM    884  O   SER A 113       7.943  -4.136   3.233  1.00  8.34           O\\nATOM    885  CB  SER A 113       6.459  -3.624   0.783  1.00  8.34           C\\nATOM    886  OG  SER A 113       6.276  -3.083  -0.499  1.00  8.34           O\\nATOM    887  N   VAL A 114       6.515  -2.592   4.052  1.00  8.30           N\\nATOM    888  CA  VAL A 114       6.358  -3.145   5.393  1.00  8.30           C\\nATOM    889  C   VAL A 114       5.020  -3.884   5.424  1.00  8.30           C\\nATOM    890  O   VAL A 114       3.989  -3.290   5.112  1.00  8.30           O\\nATOM    891  CB  VAL A 114       6.312  -1.991   6.428  1.00  8.30           C\\nATOM    892  CG1 VAL A 114       5.957  -2.466   7.851  1.00  8.30           C\\nATOM    893  CG2 VAL A 114       7.646  -1.236   6.479  1.00  8.30           C\\nATOM    894  N   LEU A 115       5.040  -5.145   5.877  1.00  7.63           N\\nATOM    895  CA  LEU A 115       3.852  -5.881   6.284  1.00  7.63           C\\nATOM    896  C   LEU A 115       3.762  -5.727   7.810  1.00  7.63           C\\nATOM    897  O   LEU A 115       4.542  -6.328   8.546  1.00  7.63           O\\nATOM    898  CB  LEU A 115       4.040  -7.364   5.888  1.00  7.63           C\\nATOM    899  CG  LEU A 115       2.825  -8.277   6.155  1.00  7.63           C\\nATOM    900  CD1 LEU A 115       1.604  -7.848   5.325  1.00  7.63           C\\nATOM    901  CD2 LEU A 115       3.189  -9.744   5.893  1.00  7.63           C\\nATOM    902  N   ALA A 116       2.819  -4.902   8.285  1.00  9.79           N\\nATOM    903  CA  ALA A 116       2.540  -4.711   9.706  1.00  9.79           C\\nATOM    904  C   ALA A 116       1.739  -5.913  10.215  1.00  9.79           C\\nATOM    905  O   ALA A 116       0.746  -6.303   9.600  1.00  9.79           O\\nATOM    906  CB  ALA A 116       1.704  -3.440   9.887  1.00  9.79           C\\nATOM    907  N   CYS A 117       2.214  -6.513  11.312  1.00  8.38           N\\nATOM    908  CA  CYS A 117       1.707  -7.742  11.913  1.00  8.38           C\\nATOM    909  C   CYS A 117       1.491  -7.541  13.412  1.00  8.38           C\\nATOM    910  O   CYS A 117       2.195  -6.754  14.040  1.00  8.38           O\\nATOM    911  CB  CYS A 117       2.693  -8.900  11.658  1.00  8.38           C\\nATOM    912  SG  CYS A 117       2.829  -9.279   9.896  1.00  8.38           S\\nATOM    913  N   TYR A 118       0.536  -8.285  13.985  1.00  9.80           N\\nATOM    914  CA  TYR A 118       0.237  -8.345  15.415  1.00  9.80           C\\nATOM    915  C   TYR A 118      -0.116  -9.788  15.747  1.00  9.80           C\\nATOM    916  O   TYR A 118      -1.042 -10.337  15.152  1.00  9.80           O\\nATOM    917  CB  TYR A 118      -0.903  -7.377  15.792  1.00  9.80           C\\nATOM    918  CG  TYR A 118      -0.582  -5.922  15.535  1.00  9.80           C\\nATOM    919  CD1 TYR A 118       0.104  -5.165  16.504  1.00  9.80           C\\nATOM    920  CD2 TYR A 118      -0.917  -5.342  14.295  1.00  9.80           C\\nATOM    921  CE1 TYR A 118       0.475  -3.841  16.219  1.00  9.80           C\\nATOM    922  CE2 TYR A 118      -0.526  -4.025  14.002  1.00  9.80           C\\nATOM    923  CZ  TYR A 118       0.183  -3.279  14.959  1.00  9.80           C\\nATOM    924  OH  TYR A 118       0.597  -2.018  14.656  1.00  9.80           O\\nATOM    925  N   ASN A 119       0.627 -10.406  16.677  1.00 14.05           N\\nATOM    926  CA  ASN A 119       0.440 -11.777  17.168  1.00 14.05           C\\nATOM    927  C   ASN A 119       0.776 -12.821  16.086  1.00 14.05           C\\nATOM    928  O   ASN A 119       0.144 -13.875  16.017  1.00 14.05           O\\nATOM    929  CB  ASN A 119      -0.944 -11.989  17.838  1.00 14.05           C\\nATOM    930  CG  ASN A 119      -1.177 -11.044  19.021  1.00 14.05           C\\nATOM    931  OD1 ASN A 119      -1.844 -10.020  18.882  1.00 14.05           O\\nATOM    932  ND2 ASN A 119      -0.643 -11.394  20.191  1.00 14.05           N\\nATOM    933  N   GLY A 120       1.731 -12.490  15.202  1.00 12.43           N\\nATOM    934  CA  GLY A 120       2.110 -13.252  14.018  1.00 12.43           C\\nATOM    935  C   GLY A 120       1.151 -13.123  12.827  1.00 12.43           C\\nATOM    936  O   GLY A 120       1.420 -13.724  11.788  1.00 12.43           O\\nATOM    937  N   SER A 121       0.040 -12.379  12.972  1.00 11.66           N\\nATOM    938  CA  SER A 121      -1.023 -12.249  11.978  1.00 11.66           C\\nATOM    939  C   SER A 121      -0.913 -10.881  11.272  1.00 11.66           C\\nATOM    940  O   SER A 121      -1.004  -9.858  11.953  1.00 11.66           O\\nATOM    941  CB  SER A 121      -2.385 -12.325  12.695  1.00 11.66           C\\nATOM    942  OG  SER A 121      -2.624 -13.650  13.126  1.00 11.66           O\\nATOM    943  N   PRO A 122      -0.736 -10.837   9.928  1.00 15.22           N\\nATOM    944  CA  PRO A 122      -0.765  -9.615   9.102  1.00 15.22           C\\nATOM    945  C   PRO A 122      -2.044  -8.775   9.269  1.00 15.22           C\\nATOM    946  O   PRO A 122      -3.147  -9.322   9.249  1.00 15.22           O\\nATOM    947  CB  PRO A 122      -0.655 -10.106   7.650  1.00 15.22           C\\nATOM    948  CG  PRO A 122      -0.050 -11.495   7.732  1.00 15.22           C\\nATOM    949  CD  PRO A 122      -0.399 -12.008   9.119  1.00 15.22           C\\nATOM    950  N   SER A 123      -1.879  -7.450   9.405  1.00 15.75           N\\nATOM    951  CA  SER A 123      -2.954  -6.466   9.543  1.00 15.75           C\\nATOM    952  C   SER A 123      -3.033  -5.510   8.340  1.00 15.75           C\\nATOM    953  O   SER A 123      -4.104  -4.959   8.086  1.00 15.75           O\\nATOM    954  CB  SER A 123      -2.766  -5.704  10.872  1.00 15.75           C\\nATOM    955  OG  SER A 123      -1.665  -4.814  10.854  1.00 15.75           O\\nATOM    956  N   GLY A 124      -1.936  -5.321   7.592  1.00 10.28           N\\nATOM    957  CA  GLY A 124      -1.903  -4.457   6.419  1.00 10.28           C\\nATOM    958  C   GLY A 124      -0.492  -4.301   5.861  1.00 10.28           C\\nATOM    959  O   GLY A 124       0.488  -4.605   6.541  1.00 10.28           O\\nATOM    960  N   VAL A 125      -0.407  -3.792   4.623  1.00  8.37           N\\nATOM    961  CA  VAL A 125       0.834  -3.540   3.898  1.00  8.37           C\\nATOM    962  C   VAL A 125       0.854  -2.088   3.404  1.00  8.37           C\\nATOM    963  O   VAL A 125      -0.140  -1.583   2.878  1.00  8.37           O\\nATOM    964  CB  VAL A 125       1.074  -4.618   2.805  1.00  8.37           C\\nATOM    965  CG1 VAL A 125       0.096  -4.532   1.618  1.00  8.37           C\\nATOM    966  CG2 VAL A 125       2.533  -4.619   2.310  1.00  8.37           C\\nATOM    967  N   TYR A 126       2.004  -1.430   3.579  1.00 11.67           N\\nATOM    968  CA  TYR A 126       2.277  -0.056   3.182  1.00 11.67           C\\nATOM    969  C   TYR A 126       3.732   0.092   2.757  1.00 11.67           C\\nATOM    970  O   TYR A 126       4.598  -0.665   3.192  1.00 11.67           O\\nATOM    971  CB  TYR A 126       1.837   0.947   4.268  1.00 11.67           C\\nATOM    972  CG  TYR A 126       2.504   0.765   5.623  1.00 11.67           C\\nATOM    973  CD1 TYR A 126       3.800   1.269   5.865  1.00 11.67           C\\nATOM    974  CD2 TYR A 126       1.821   0.074   6.645  1.00 11.67           C\\nATOM    975  CE1 TYR A 126       4.415   1.057   7.113  1.00 11.67           C\\nATOM    976  CE2 TYR A 126       2.429  -0.114   7.899  1.00 11.67           C\\nATOM    977  CZ  TYR A 126       3.732   0.367   8.129  1.00 11.67           C\\nATOM    978  OH  TYR A 126       4.333   0.173   9.336  1.00 11.67           O\\nATOM    979  N   GLN A 127       3.991   1.104   1.926  1.00 17.26           N\\nATOM    980  CA  GLN A 127       5.314   1.449   1.440  1.00 17.26           C\\nATOM    981  C   GLN A 127       5.960   2.470   2.389  1.00 17.26           C\\nATOM    982  O   GLN A 127       5.291   3.350   2.932  1.00 17.26           O\\nATOM    983  CB  GLN A 127       5.119   2.066   0.042  1.00 17.26           C\\nATOM    984  CG  GLN A 127       6.382   2.176  -0.827  1.00 17.26           C\\nATOM    985  CD  GLN A 127       7.000   0.832  -1.224  1.00 17.26           C\\nATOM    986  OE1 GLN A 127       6.513  -0.240  -0.877  1.00 17.26           O\\nATOM    987  NE2 GLN A 127       8.097   0.894  -1.970  1.00 17.26           N\\nATOM    988  N   CYS A 128       7.275   2.328   2.568  1.00 22.83           N\\nATOM    989  CA  CYS A 128       8.153   3.134   3.406  1.00 22.83           C\\nATOM    990  C   CYS A 128       9.385   3.564   2.607  1.00 22.83           C\\nATOM    991  O   CYS A 128       9.604   3.097   1.489  1.00 22.83           O\\nATOM    992  CB  CYS A 128       8.544   2.316   4.663  1.00 22.83           C\\nATOM    993  SG  CYS A 128       7.552   2.830   6.085  1.00 22.83           S\\nATOM    994  N   ALA A 129      10.211   4.415   3.227  1.00 17.43           N\\nATOM    995  CA  ALA A 129      11.589   4.678   2.843  1.00 17.43           C\\nATOM    996  C   ALA A 129      12.471   4.735   4.082  1.00 17.43           C\\nATOM    997  O   ALA A 129      12.045   5.203   5.140  1.00 17.43           O\\nATOM    998  CB  ALA A 129      11.716   5.925   1.955  1.00 17.43           C\\nATOM    999  N   MET A 130      13.731   4.308   3.924  1.00 15.37           N\\nATOM   1000  CA  MET A 130      14.794   4.584   4.875  1.00 15.37           C\\nATOM   1001  C   MET A 130      15.136   6.064   4.708  1.00 15.37           C\\nATOM   1002  O   MET A 130      15.616   6.478   3.656  1.00 15.37           O\\nATOM   1003  CB  MET A 130      16.006   3.685   4.555  1.00 15.37           C\\nATOM   1004  CG  MET A 130      17.136   3.754   5.601  1.00 15.37           C\\nATOM   1005  SD  MET A 130      16.657   3.409   7.320  1.00 15.37           S\\nATOM   1006  CE  MET A 130      15.890   1.776   7.150  1.00 15.37           C\\nATOM   1007  N   ARG A 131      14.820   6.865   5.732  1.00 14.69           N\\nATOM   1008  CA  ARG A 131      15.026   8.308   5.724  1.00 14.69           C\\nATOM   1009  C   ARG A 131      16.534   8.608   5.866  1.00 14.69           C\\nATOM   1010  O   ARG A 131      17.247   7.792   6.449  1.00 14.69           O\\nATOM   1011  CB  ARG A 131      14.271   8.925   6.920  1.00 14.69           C\\nATOM   1012  CG  ARG A 131      12.741   8.695   6.947  1.00 14.69           C\\nATOM   1013  CD  ARG A 131      11.989   9.074   5.665  1.00 14.69           C\\nATOM   1014  NE  ARG A 131      12.349  10.420   5.219  1.00 14.69           N\\nATOM   1015  CZ  ARG A 131      12.025  10.986   4.052  1.00 14.69           C\\nATOM   1016  NH1 ARG A 131      11.202  10.391   3.182  1.00 14.69           N\\nATOM   1017  NH2 ARG A 131      12.540  12.181   3.757  1.00 14.69           N\\nATOM   1018  N   PRO A 132      17.047   9.742   5.341  1.00 16.36           N\\nATOM   1019  CA  PRO A 132      18.433  10.214   5.536  1.00 16.36           C\\nATOM   1020  C   PRO A 132      18.982  10.225   6.978  1.00 16.36           C\\nATOM   1021  O   PRO A 132      20.175   9.995   7.163  1.00 16.36           O\\nATOM   1022  CB  PRO A 132      18.444  11.622   4.943  1.00 16.36           C\\nATOM   1023  CG  PRO A 132      17.400  11.552   3.845  1.00 16.36           C\\nATOM   1024  CD  PRO A 132      16.341  10.600   4.390  1.00 16.36           C\\nATOM   1025  N   ASN A 133      18.124  10.436   7.992  1.00 15.55           N\\nATOM   1026  CA  ASN A 133      18.475  10.347   9.417  1.00 15.55           C\\nATOM   1027  C   ASN A 133      18.382   8.908   9.980  1.00 15.55           C\\nATOM   1028  O   ASN A 133      18.618   8.700  11.168  1.00 15.55           O\\nATOM   1029  CB  ASN A 133      17.577  11.308  10.245  1.00 15.55           C\\nATOM   1030  CG  ASN A 133      16.060  11.090  10.172  1.00 15.55           C\\nATOM   1031  OD1 ASN A 133      15.572  10.141   9.566  1.00 15.55           O\\nATOM   1032  ND2 ASN A 133      15.301  11.978  10.811  1.00 15.55           N\\nATOM   1033  N   PHE A 134      18.025   7.922   9.141  1.00 14.68           N\\nATOM   1034  CA  PHE A 134      17.982   6.480   9.400  1.00 14.68           C\\nATOM   1035  C   PHE A 134      16.751   5.999  10.179  1.00 14.68           C\\nATOM   1036  O   PHE A 134      16.685   4.830  10.561  1.00 14.68           O\\nATOM   1037  CB  PHE A 134      19.306   5.931   9.944  1.00 14.68           C\\nATOM   1038  CG  PHE A 134      20.466   5.972   8.976  1.00 14.68           C\\nATOM   1039  CD1 PHE A 134      20.603   4.957   8.007  1.00 14.68           C\\nATOM   1040  CD2 PHE A 134      21.422   7.005   9.056  1.00 14.68           C\\nATOM   1041  CE1 PHE A 134      21.703   4.968   7.132  1.00 14.68           C\\nATOM   1042  CE2 PHE A 134      22.517   7.019   8.172  1.00 14.68           C\\nATOM   1043  CZ  PHE A 134      22.661   5.997   7.214  1.00 14.68           C\\nATOM   1044  N   THR A 135      15.759   6.878  10.382  1.00 15.17           N\\nATOM   1045  CA  THR A 135      14.430   6.504  10.856  1.00 15.17           C\\nATOM   1046  C   THR A 135      13.589   6.029   9.656  1.00 15.17           C\\nATOM   1047  O   THR A 135      14.022   6.097   8.504  1.00 15.17           O\\nATOM   1048  CB  THR A 135      13.749   7.723  11.538  1.00 15.17           C\\nATOM   1049  OG1 THR A 135      13.374   8.734  10.622  1.00 15.17           O\\nATOM   1050  CG2 THR A 135      14.582   8.339  12.669  1.00 15.17           C\\nATOM   1051  N   ILE A 136      12.368   5.563   9.941  1.00 16.72           N\\nATOM   1052  CA  ILE A 136      11.315   5.276   8.973  1.00 16.72           C\\nATOM   1053  C   ILE A 136      10.016   5.891   9.514  1.00 16.72           C\\nATOM   1054  O   ILE A 136       9.805   5.941  10.726  1.00 16.72           O\\nATOM   1055  CB  ILE A 136      11.277   3.774   8.542  1.00 16.72           C\\nATOM   1056  CG1 ILE A 136      10.443   2.839   9.450  1.00 16.72           C\\nATOM   1057  CG2 ILE A 136      12.694   3.205   8.312  1.00 16.72           C\\nATOM   1058  CD1 ILE A 136      10.153   1.476   8.805  1.00 16.72           C\\nATOM   1059  N   LYS A 137       9.149   6.367   8.614  1.00 17.02           N\\nATOM   1060  CA  LYS A 137       7.821   6.882   8.926  1.00 17.02           C\\nATOM   1061  C   LYS A 137       6.820   5.768   8.603  1.00 17.02           C\\nATOM   1062  O   LYS A 137       6.217   5.742   7.530  1.00 17.02           O\\nATOM   1063  CB  LYS A 137       7.555   8.127   8.057  1.00 17.02           C\\nATOM   1064  CG  LYS A 137       8.271   9.380   8.581  1.00 17.02           C\\nATOM   1065  CD  LYS A 137       8.317  10.493   7.527  1.00 17.02           C\\nATOM   1066  CE  LYS A 137       8.804  11.826   8.103  1.00 17.02           C\\nATOM   1067  NZ  LYS A 137       9.236  12.742   7.032  1.00 17.02           N\\nATOM   1068  N   GLY A 138       6.677   4.827   9.546  1.00 17.71           N\\nATOM   1069  CA  GLY A 138       5.765   3.696   9.464  1.00 17.71           C\\nATOM   1070  C   GLY A 138       4.441   3.959  10.183  1.00 17.71           C\\nATOM   1071  O   GLY A 138       4.132   5.084  10.577  1.00 17.71           O\\nATOM   1072  N   SER A 139       3.671   2.882  10.361  1.00 18.24           N\\nATOM   1073  CA  SER A 139       2.434   2.818  11.126  1.00 18.24           C\\nATOM   1074  C   SER A 139       2.523   1.554  11.979  1.00 18.24           C\\nATOM   1075  O   SER A 139       2.361   0.445  11.468  1.00 18.24           O\\nATOM   1076  CB  SER A 139       1.218   2.893  10.182  1.00 18.24           C\\nATOM   1077  OG  SER A 139       0.004   2.824  10.903  1.00 18.24           O\\nATOM   1078  N   PHE A 140       2.850   1.741  13.262  1.00 19.57           N\\nATOM   1079  CA  PHE A 140       3.230   0.702  14.209  1.00 19.57           C\\nATOM   1080  C   PHE A 140       2.626   1.026  15.581  1.00 19.57           C\\nATOM   1081  O   PHE A 140       2.726   2.162  16.045  1.00 19.57           O\\nATOM   1082  CB  PHE A 140       4.767   0.615  14.258  1.00 19.57           C\\nATOM   1083  CG  PHE A 140       5.487   0.092  13.023  1.00 19.57           C\\nATOM   1084  CD1 PHE A 140       5.316  -1.233  12.572  1.00 19.57           C\\nATOM   1085  CD2 PHE A 140       6.398   0.928  12.353  1.00 19.57           C\\nATOM   1086  CE1 PHE A 140       6.098  -1.730  11.510  1.00 19.57           C\\nATOM   1087  CE2 PHE A 140       7.187   0.437  11.301  1.00 19.57           C\\nATOM   1088  CZ  PHE A 140       7.055  -0.902  10.892  1.00 19.57           C\\nATOM   1089  N   LEU A 141       2.022   0.020  16.227  1.00 23.03           N\\nATOM   1090  CA  LEU A 141       1.447   0.064  17.574  1.00 23.03           C\\nATOM   1091  C   LEU A 141       2.197  -0.939  18.472  1.00 23.03           C\\nATOM   1092  O   LEU A 141       3.190  -1.536  18.056  1.00 23.03           O\\nATOM   1093  CB  LEU A 141      -0.064  -0.284  17.506  1.00 23.03           C\\nATOM   1094  CG  LEU A 141      -0.912   0.546  16.514  1.00 23.03           C\\nATOM   1095  CD1 LEU A 141      -2.329  -0.035  16.399  1.00 23.03           C\\nATOM   1096  CD2 LEU A 141      -0.966   2.031  16.900  1.00 23.03           C\\nATOM   1097  N   ASN A 142       1.726  -1.144  19.713  1.00 32.08           N\\nATOM   1098  CA  ASN A 142       2.256  -2.158  20.633  1.00 32.08           C\\nATOM   1099  C   ASN A 142       1.937  -3.566  20.100  1.00 32.08           C\\nATOM   1100  O   ASN A 142       0.806  -3.840  19.696  1.00 32.08           O\\nATOM   1101  CB  ASN A 142       1.636  -2.001  22.043  1.00 32.08           C\\nATOM   1102  CG  ASN A 142       1.987  -0.698  22.772  1.00 32.08           C\\nATOM   1103  OD1 ASN A 142       2.886   0.041  22.380  1.00 32.08           O\\nATOM   1104  ND2 ASN A 142       1.254  -0.399  23.846  1.00 32.08           N\\nATOM   1105  N   GLY A 143       2.953  -4.439  20.082  1.00 21.33           N\\nATOM   1106  CA  GLY A 143       2.906  -5.784  19.514  1.00 21.33           C\\nATOM   1107  C   GLY A 143       3.389  -5.863  18.059  1.00 21.33           C\\nATOM   1108  O   GLY A 143       3.443  -6.965  17.515  1.00 21.33           O\\nATOM   1109  N   SER A 144       3.738  -4.728  17.425  1.00 13.26           N\\nATOM   1110  CA  SER A 144       4.145  -4.663  16.021  1.00 13.26           C\\nATOM   1111  C   SER A 144       5.629  -4.979  15.775  1.00 13.26           C\\nATOM   1112  O   SER A 144       6.025  -5.089  14.617  1.00 13.26           O\\nATOM   1113  CB  SER A 144       3.854  -3.260  15.464  1.00 13.26           C\\nATOM   1114  OG  SER A 144       4.781  -2.288  15.896  1.00 13.26           O\\nATOM   1115  N   CYS A 145       6.455  -5.117  16.825  1.00 15.56           N\\nATOM   1116  CA  CYS A 145       7.885  -5.400  16.701  1.00 15.56           C\\nATOM   1117  C   CYS A 145       8.116  -6.774  16.056  1.00 15.56           C\\nATOM   1118  O   CYS A 145       7.343  -7.706  16.265  1.00 15.56           O\\nATOM   1119  CB  CYS A 145       8.548  -5.324  18.082  1.00 15.56           C\\nATOM   1120  SG  CYS A 145       8.513  -3.608  18.656  1.00 15.56           S\\nATOM   1121  N   GLY A 146       9.139  -6.863  15.202  1.00 10.94           N\\nATOM   1122  CA  GLY A 146       9.413  -8.028  14.376  1.00 10.94           C\\nATOM   1123  C   GLY A 146       8.639  -8.066  13.055  1.00 10.94           C\\nATOM   1124  O   GLY A 146       8.818  -9.014  12.293  1.00 10.94           O\\nATOM   1125  N   SER A 147       7.816  -7.038  12.766  1.00  7.81           N\\nATOM   1126  CA  SER A 147       7.221  -6.790  11.452  1.00  7.81           C\\nATOM   1127  C   SER A 147       8.339  -6.320  10.519  1.00  7.81           C\\nATOM   1128  O   SER A 147       9.130  -5.456  10.903  1.00  7.81           O\\nATOM   1129  CB  SER A 147       6.185  -5.656  11.544  1.00  7.81           C\\nATOM   1130  OG  SER A 147       5.004  -6.111  12.163  1.00  7.81           O\\nATOM   1131  N   VAL A 148       8.424  -6.913   9.322  1.00  7.50           N\\nATOM   1132  CA  VAL A 148       9.573  -6.760   8.439  1.00  7.50           C\\nATOM   1133  C   VAL A 148       9.247  -5.903   7.219  1.00  7.50           C\\nATOM   1134  O   VAL A 148       8.130  -5.917   6.700  1.00  7.50           O\\nATOM   1135  CB  VAL A 148      10.149  -8.144   8.042  1.00  7.50           C\\nATOM   1136  CG1 VAL A 148      10.593  -8.920   9.291  1.00  7.50           C\\nATOM   1137  CG2 VAL A 148       9.242  -9.008   7.148  1.00  7.50           C\\nATOM   1138  N   GLY A 149      10.285  -5.192   6.763  1.00  8.85           N\\nATOM   1139  CA  GLY A 149      10.343  -4.489   5.500  1.00  8.85           C\\nATOM   1140  C   GLY A 149      11.108  -5.314   4.469  1.00  8.85           C\\nATOM   1141  O   GLY A 149      12.052  -6.028   4.815  1.00  8.85           O\\nATOM   1142  N   PHE A 150      10.679  -5.220   3.205  1.00  8.53           N\\nATOM   1143  CA  PHE A 150      11.143  -6.058   2.108  1.00  8.53           C\\nATOM   1144  C   PHE A 150      10.942  -5.396   0.742  1.00  8.53           C\\nATOM   1145  O   PHE A 150      10.061  -4.558   0.555  1.00  8.53           O\\nATOM   1146  CB  PHE A 150      10.485  -7.456   2.189  1.00  8.53           C\\nATOM   1147  CG  PHE A 150       8.977  -7.445   1.985  1.00  8.53           C\\nATOM   1148  CD1 PHE A 150       8.120  -7.166   3.071  1.00  8.53           C\\nATOM   1149  CD2 PHE A 150       8.426  -7.674   0.706  1.00  8.53           C\\nATOM   1150  CE1 PHE A 150       6.729  -7.084   2.873  1.00  8.53           C\\nATOM   1151  CE2 PHE A 150       7.033  -7.603   0.511  1.00  8.53           C\\nATOM   1152  CZ  PHE A 150       6.184  -7.300   1.593  1.00  8.53           C\\nATOM   1153  N   ASN A 151      11.727  -5.870  -0.230  1.00  9.22           N\\nATOM   1154  CA  ASN A 151      11.552  -5.686  -1.667  1.00  9.22           C\\nATOM   1155  C   ASN A 151      11.434  -7.073  -2.315  1.00  9.22           C\\nATOM   1156  O   ASN A 151      11.926  -8.050  -1.751  1.00  9.22           O\\nATOM   1157  CB  ASN A 151      12.753  -4.908  -2.243  1.00  9.22           C\\nATOM   1158  CG  ASN A 151      12.757  -3.425  -1.856  1.00  9.22           C\\nATOM   1159  OD1 ASN A 151      12.128  -2.614  -2.524  1.00  9.22           O\\nATOM   1160  ND2 ASN A 151      13.479  -3.045  -0.804  1.00  9.22           N\\nATOM   1161  N   ILE A 152      10.808  -7.164  -3.501  1.00 11.44           N\\nATOM   1162  CA  ILE A 152      10.706  -8.401  -4.284  1.00 11.44           C\\nATOM   1163  C   ILE A 152      11.246  -8.144  -5.697  1.00 11.44           C\\nATOM   1164  O   ILE A 152      10.775  -7.242  -6.388  1.00 11.44           O\\nATOM   1165  CB  ILE A 152       9.268  -9.005  -4.275  1.00 11.44           C\\nATOM   1166  CG1 ILE A 152       8.688  -9.126  -2.844  1.00 11.44           C\\nATOM   1167  CG2 ILE A 152       9.234 -10.369  -5.006  1.00 11.44           C\\nATOM   1168  CD1 ILE A 152       7.296  -9.771  -2.758  1.00 11.44           C\\nATOM   1169  N   ASP A 153      12.198  -8.984  -6.132  0.24 13.77           N\\nATOM   1170  CA  ASP A 153      12.700  -9.077  -7.500  0.24 13.77           C\\nATOM   1171  C   ASP A 153      12.194 -10.419  -8.049  0.24 13.77           C\\nATOM   1172  O   ASP A 153      12.834 -11.453  -7.852  0.24 13.77           O\\nATOM   1173  CB  ASP A 153      14.247  -8.973  -7.465  0.24 13.77           C\\nATOM   1174  CG  ASP A 153      14.991  -9.163  -8.796  0.24 13.77           C\\nATOM   1175  OD1 ASP A 153      14.330  -9.220  -9.856  0.24 13.77           O\\nATOM   1176  OD2 ASP A 153      16.238  -9.229  -8.717  0.24 13.77           O\\nATOM   1177  N   TYR A 154      11.004 -10.384  -8.675  0.24 16.78           N\\nATOM   1178  CA  TYR A 154      10.279 -11.497  -9.293  0.24 16.78           C\\nATOM   1179  C   TYR A 154       9.726 -12.470  -8.237  0.24 16.78           C\\nATOM   1180  O   TYR A 154       8.555 -12.381  -7.868  0.24 16.78           O\\nATOM   1181  CB  TYR A 154      11.077 -12.168 -10.442  0.24 16.78           C\\nATOM   1182  CG  TYR A 154      10.389 -13.367 -11.079  0.24 16.78           C\\nATOM   1183  CD1 TYR A 154       9.090 -13.245 -11.617  0.24 16.78           C\\nATOM   1184  CD2 TYR A 154      11.032 -14.622 -11.091  0.24 16.78           C\\nATOM   1185  CE1 TYR A 154       8.429 -14.376 -12.133  0.24 16.78           C\\nATOM   1186  CE2 TYR A 154      10.371 -15.754 -11.605  0.24 16.78           C\\nATOM   1187  CZ  TYR A 154       9.066 -15.632 -12.120  0.24 16.78           C\\nATOM   1188  OH  TYR A 154       8.418 -16.730 -12.605  0.24 16.78           O\\nATOM   1189  N   ASP A 155      10.584 -13.387  -7.762  0.24 15.30           N\\nATOM   1190  CA  ASP A 155      10.281 -14.449  -6.806  0.24 15.30           C\\nATOM   1191  C   ASP A 155      11.219 -14.424  -5.586  0.24 15.30           C\\nATOM   1192  O   ASP A 155      10.946 -15.131  -4.617  0.24 15.30           O\\nATOM   1193  CB  ASP A 155      10.227 -15.822  -7.530  0.24 15.30           C\\nATOM   1194  CG  ASP A 155      11.537 -16.362  -8.125  0.24 15.30           C\\nATOM   1195  OD1 ASP A 155      12.549 -15.627  -8.139  0.24 15.30           O\\nATOM   1196  OD2 ASP A 155      11.504 -17.531  -8.567  0.24 15.30           O\\nATOM   1197  N   CYS A 156      12.292 -13.614  -5.611  1.00 10.59           N\\nATOM   1198  CA  CYS A 156      13.244 -13.464  -4.516  1.00 10.59           C\\nATOM   1199  C   CYS A 156      12.810 -12.274  -3.649  1.00 10.59           C\\nATOM   1200  O   CYS A 156      12.754 -11.148  -4.139  1.00 10.59           O\\nATOM   1201  CB  CYS A 156      14.640 -13.187  -5.114  1.00 10.59           C\\nATOM   1202  SG  CYS A 156      15.911 -13.215  -3.812  1.00 10.59           S\\nATOM   1203  N   VAL A 157      12.529 -12.525  -2.360  1.00  7.42           N\\nATOM   1204  CA  VAL A 157      12.189 -11.511  -1.366  1.00  7.42           C\\nATOM   1205  C   VAL A 157      13.486 -11.107  -0.645  1.00  7.42           C\\nATOM   1206  O   VAL A 157      14.010 -11.854   0.180  1.00  7.42           O\\nATOM   1207  CB  VAL A 157      11.188 -12.097  -0.331  1.00  7.42           C\\nATOM   1208  CG1 VAL A 157      10.755 -11.036   0.698  1.00  7.42           C\\nATOM   1209  CG2 VAL A 157       9.936 -12.700  -0.993  1.00  7.42           C\\nATOM   1210  N   SER A 158      13.979  -9.895  -0.923  1.00  8.82           N\\nATOM   1211  CA  SER A 158      15.084  -9.287  -0.195  1.00  8.82           C\\nATOM   1212  C   SER A 158      14.501  -8.578   1.033  1.00  8.82           C\\nATOM   1213  O   SER A 158      13.880  -7.526   0.902  1.00  8.82           O\\nATOM   1214  CB  SER A 158      15.775  -8.249  -1.098  1.00  8.82           C\\nATOM   1215  OG  SER A 158      16.727  -8.906  -1.904  1.00  8.82           O\\nATOM   1216  N   PHE A 159      14.705  -9.163   2.220  1.00  8.45           N\\nATOM   1217  CA  PHE A 159      14.320  -8.586   3.504  1.00  8.45           C\\nATOM   1218  C   PHE A 159      15.400  -7.586   3.915  1.00  8.45           C\\nATOM   1219  O   PHE A 159      16.591  -7.888   3.830  1.00  8.45           O\\nATOM   1220  CB  PHE A 159      14.260  -9.696   4.565  1.00  8.45           C\\nATOM   1221  CG  PHE A 159      13.137 -10.682   4.336  1.00  8.45           C\\nATOM   1222  CD1 PHE A 159      11.816 -10.321   4.666  1.00  8.45           C\\nATOM   1223  CD2 PHE A 159      13.400 -11.945   3.766  1.00  8.45           C\\nATOM   1224  CE1 PHE A 159      10.761 -11.221   4.430  1.00  8.45           C\\nATOM   1225  CE2 PHE A 159      12.342 -12.843   3.531  1.00  8.45           C\\nATOM   1226  CZ  PHE A 159      11.023 -12.486   3.870  1.00  8.45           C\\nATOM   1227  N   CYS A 160      14.969  -6.397   4.346  1.00 10.80           N\\nATOM   1228  CA  CYS A 160      15.840  -5.264   4.632  1.00 10.80           C\\nATOM   1229  C   CYS A 160      15.562  -4.567   5.950  1.00 10.80           C\\nATOM   1230  O   CYS A 160      16.405  -3.797   6.390  1.00 10.80           O\\nATOM   1231  CB  CYS A 160      15.885  -4.292   3.447  1.00 10.80           C\\nATOM   1232  SG  CYS A 160      14.262  -3.531   3.134  1.00 10.80           S\\nATOM   1233  N   TYR A 161      14.415  -4.821   6.583  1.00  8.81           N\\nATOM   1234  CA  TYR A 161      14.029  -4.173   7.821  1.00  8.81           C\\nATOM   1235  C   TYR A 161      13.348  -5.182   8.739  1.00  8.81           C\\nATOM   1236  O   TYR A 161      12.701  -6.118   8.276  1.00  8.81           O\\nATOM   1237  CB  TYR A 161      13.252  -2.877   7.523  1.00  8.81           C\\nATOM   1238  CG  TYR A 161      12.845  -2.045   8.723  1.00  8.81           C\\nATOM   1239  CD1 TYR A 161      13.743  -1.104   9.267  1.00  8.81           C\\nATOM   1240  CD2 TYR A 161      11.557  -2.191   9.278  1.00  8.81           C\\nATOM   1241  CE1 TYR A 161      13.348  -0.302  10.354  1.00  8.81           C\\nATOM   1242  CE2 TYR A 161      11.168  -1.395  10.371  1.00  8.81           C\\nATOM   1243  CZ  TYR A 161      12.062  -0.450  10.910  1.00  8.81           C\\nATOM   1244  OH  TYR A 161      11.679   0.335  11.957  1.00  8.81           O\\nATOM   1245  N   MET A 162      13.500  -4.959  10.044  1.00 10.10           N\\nATOM   1246  CA  MET A 162      12.794  -5.602  11.131  1.00 10.10           C\\nATOM   1247  C   MET A 162      12.568  -4.520  12.178  1.00 10.10           C\\nATOM   1248  O   MET A 162      13.528  -3.931  12.676  1.00 10.10           O\\nATOM   1249  CB  MET A 162      13.560  -6.833  11.645  1.00 10.10           C\\nATOM   1250  CG  MET A 162      12.795  -7.552  12.763  1.00 10.10           C\\nATOM   1251  SD  MET A 162      13.475  -9.157  13.239  1.00 10.10           S\\nATOM   1252  CE  MET A 162      12.582 -10.186  12.048  1.00 10.10           C\\nATOM   1253  N   HIS A 163      11.296  -4.257  12.498  1.00 10.91           N\\nATOM   1254  CA  HIS A 163      10.884  -3.187  13.392  1.00 10.91           C\\nATOM   1255  C   HIS A 163      11.224  -3.521  14.850  1.00 10.91           C\\nATOM   1256  O   HIS A 163      10.990  -4.642  15.299  1.00 10.91           O\\nATOM   1257  CB  HIS A 163       9.368  -2.984  13.242  1.00 10.91           C\\nATOM   1258  CG  HIS A 163       8.893  -1.781  14.004  1.00 10.91           C\\nATOM   1259  ND1 HIS A 163       9.439  -0.510  13.796  1.00 10.91           N\\nATOM   1260  CD2 HIS A 163       7.955  -1.715  15.007  1.00 10.91           C\\nATOM   1261  CE1 HIS A 163       8.834   0.258  14.687  1.00 10.91           C\\nATOM   1262  NE2 HIS A 163       7.935  -0.397  15.412  1.00 10.91           N\\nATOM   1263  N   HIS A 164      11.792  -2.540  15.567  1.00 13.39           N\\nATOM   1264  CA  HIS A 164      12.245  -2.674  16.949  1.00 13.39           C\\nATOM   1265  C   HIS A 164      11.671  -1.588  17.861  1.00 13.39           C\\nATOM   1266  O   HIS A 164      11.212  -1.894  18.958  1.00 13.39           O\\nATOM   1267  CB  HIS A 164      13.785  -2.671  17.032  1.00 13.39           C\\nATOM   1268  CG  HIS A 164      14.489  -3.944  16.642  1.00 13.39           C\\nATOM   1269  ND1 HIS A 164      14.266  -4.612  15.448  1.00 13.39           N\\nATOM   1270  CD2 HIS A 164      15.469  -4.646  17.314  1.00 13.39           C\\nATOM   1271  CE1 HIS A 164      15.005  -5.717  15.475  1.00 13.39           C\\nATOM   1272  NE2 HIS A 164      15.755  -5.773  16.570  1.00 13.39           N\\nATOM   1273  N   MET A 165      11.762  -0.319  17.444  1.00 48.84           N\\nATOM   1274  CA  MET A 165      11.592   0.842  18.310  1.00 48.84           C\\nATOM   1275  C   MET A 165      10.695   1.888  17.661  1.00 48.84           C\\nATOM   1276  O   MET A 165      10.778   2.122  16.458  1.00 48.84           O\\nATOM   1277  CB  MET A 165      12.970   1.448  18.659  1.00 48.84           C\\nATOM   1278  CG  MET A 165      13.913   0.522  19.440  1.00 48.84           C\\nATOM   1279  SD  MET A 165      13.321  -0.014  21.069  1.00 48.84           S\\nATOM   1280  CE  MET A 165      13.514   1.530  21.993  1.00 48.84           C\\nATOM   1281  N   GLU A 166       9.905   2.571  18.495  1.00 23.02           N\\nATOM   1282  CA  GLU A 166       9.333   3.876  18.208  1.00 23.02           C\\nATOM   1283  C   GLU A 166      10.050   4.870  19.127  1.00 23.02           C\\nATOM   1284  O   GLU A 166      10.109   4.668  20.341  1.00 23.02           O\\nATOM   1285  CB  GLU A 166       7.818   3.839  18.482  1.00 23.02           C\\nATOM   1286  CG  GLU A 166       7.095   5.162  18.164  1.00 23.02           C\\nATOM   1287  CD  GLU A 166       5.579   4.995  18.208  1.00 23.02           C\\nATOM   1288  OE1 GLU A 166       5.042   4.998  19.336  1.00 23.02           O\\nATOM   1289  OE2 GLU A 166       4.979   4.850  17.122  1.00 23.02           O\\nATOM   1290  N   LEU A 167      10.595   5.945  18.543  1.00 21.17           N\\nATOM   1291  CA  LEU A 167      11.238   7.042  19.263  1.00 21.17           C\\nATOM   1292  C   LEU A 167      10.145   7.975  19.834  1.00 21.17           C\\nATOM   1293  O   LEU A 167       9.040   7.989  19.291  1.00 21.17           O\\nATOM   1294  CB  LEU A 167      12.113   7.855  18.277  1.00 21.17           C\\nATOM   1295  CG  LEU A 167      13.145   7.046  17.456  1.00 21.17           C\\nATOM   1296  CD1 LEU A 167      14.057   7.993  16.662  1.00 21.17           C\\nATOM   1297  CD2 LEU A 167      14.000   6.094  18.309  1.00 21.17           C\\nATOM   1298  N   PRO A 168      10.418   8.755  20.907  1.00 22.23           N\\nATOM   1299  CA  PRO A 168       9.498   9.750  21.497  1.00 22.23           C\\nATOM   1300  C   PRO A 168       8.734  10.703  20.546  1.00 22.23           C\\nATOM   1301  O   PRO A 168       7.626  11.117  20.886  1.00 22.23           O\\nATOM   1302  CB  PRO A 168      10.364  10.551  22.478  1.00 22.23           C\\nATOM   1303  CG  PRO A 168      11.450   9.580  22.908  1.00 22.23           C\\nATOM   1304  CD  PRO A 168      11.644   8.655  21.708  1.00 22.23           C\\nATOM   1305  N   THR A 169       9.290  11.026  19.363  1.00 21.93           N\\nATOM   1306  CA  THR A 169       8.657  11.851  18.322  1.00 21.93           C\\nATOM   1307  C   THR A 169       7.712  11.057  17.389  1.00 21.93           C\\nATOM   1308  O   THR A 169       7.168  11.636  16.449  1.00 21.93           O\\nATOM   1309  CB  THR A 169       9.765  12.530  17.476  1.00 21.93           C\\nATOM   1310  OG1 THR A 169      10.598  11.573  16.847  1.00 21.93           O\\nATOM   1311  CG2 THR A 169      10.607  13.536  18.274  1.00 21.93           C\\nATOM   1312  N   GLY A 170       7.498   9.755  17.634  1.00 18.98           N\\nATOM   1313  CA  GLY A 170       6.563   8.899  16.906  1.00 18.98           C\\nATOM   1314  C   GLY A 170       7.089   8.336  15.579  1.00 18.98           C\\nATOM   1315  O   GLY A 170       6.323   7.683  14.871  1.00 18.98           O\\nATOM   1316  N   VAL A 171       8.369   8.573  15.244  1.00 17.99           N\\nATOM   1317  CA  VAL A 171       9.075   7.957  14.116  1.00 17.99           C\\nATOM   1318  C   VAL A 171       9.742   6.663  14.607  1.00 17.99           C\\nATOM   1319  O   VAL A 171       9.870   6.429  15.809  1.00 17.99           O\\nATOM   1320  CB  VAL A 171      10.129   8.934  13.526  1.00 17.99           C\\nATOM   1321  CG1 VAL A 171       9.483  10.260  13.090  1.00 17.99           C\\nATOM   1322  CG2 VAL A 171      11.353   9.184  14.423  1.00 17.99           C\\nATOM   1323  N   HIS A 172      10.131   5.807  13.657  1.00 15.14           N\\nATOM   1324  CA  HIS A 172      10.435   4.403  13.896  1.00 15.14           C\\nATOM   1325  C   HIS A 172      11.890   4.084  13.583  1.00 15.14           C\\nATOM   1326  O   HIS A 172      12.500   4.704  12.716  1.00 15.14           O\\nATOM   1327  CB  HIS A 172       9.450   3.567  13.066  1.00 15.14           C\\nATOM   1328  CG  HIS A 172       8.028   3.858  13.473  1.00 15.14           C\\nATOM   1329  ND1 HIS A 172       7.123   4.528  12.642  1.00 15.14           N\\nATOM   1330  CD2 HIS A 172       7.425   3.623  14.690  1.00 15.14           C\\nATOM   1331  CE1 HIS A 172       6.033   4.679  13.383  1.00 15.14           C\\nATOM   1332  NE2 HIS A 172       6.158   4.164  14.604  1.00 15.14           N\\nATOM   1333  N   ALA A 173      12.429   3.102  14.313  1.00 15.43           N\\nATOM   1334  CA  ALA A 173      13.792   2.614  14.214  1.00 15.43           C\\nATOM   1335  C   ALA A 173      13.817   1.089  14.305  1.00 15.43           C\\nATOM   1336  O   ALA A 173      13.007   0.476  15.001  1.00 15.43           O\\nATOM   1337  CB  ALA A 173      14.701   3.328  15.227  1.00 15.43           C\\nATOM   1338  N   GLY A 174      14.744   0.477  13.567  1.00 12.46           N\\nATOM   1339  CA  GLY A 174      14.858  -0.961  13.437  1.00 12.46           C\\nATOM   1340  C   GLY A 174      16.166  -1.353  12.768  1.00 12.46           C\\nATOM   1341  O   GLY A 174      17.011  -0.512  12.460  1.00 12.46           O\\nATOM   1342  N   THR A 175      16.322  -2.665  12.588  1.00 10.67           N\\nATOM   1343  CA  THR A 175      17.532  -3.325  12.126  1.00 10.67           C\\nATOM   1344  C   THR A 175      17.330  -3.908  10.730  1.00 10.67           C\\nATOM   1345  O   THR A 175      16.199  -4.095  10.301  1.00 10.67           O\\nATOM   1346  CB  THR A 175      17.863  -4.459  13.131  1.00 10.67           C\\nATOM   1347  OG1 THR A 175      16.908  -5.504  13.098  1.00 10.67           O\\nATOM   1348  CG2 THR A 175      18.014  -4.001  14.585  1.00 10.67           C\\nATOM   1349  N   ASP A 176      18.433  -4.314  10.085  1.00 11.44           N\\nATOM   1350  CA  ASP A 176      18.430  -5.336   9.034  1.00 11.44           C\\nATOM   1351  C   ASP A 176      18.237  -6.725   9.694  1.00 11.44           C\\nATOM   1352  O   ASP A 176      18.176  -6.843  10.919  1.00 11.44           O\\nATOM   1353  CB  ASP A 176      19.761  -5.245   8.238  1.00 11.44           C\\nATOM   1354  CG  ASP A 176      21.080  -5.378   9.011  1.00 11.44           C\\nATOM   1355  OD1 ASP A 176      21.101  -6.019  10.083  1.00 11.44           O\\nATOM   1356  OD2 ASP A 176      22.070  -4.817   8.499  1.00 11.44           O\\nATOM   1357  N   LEU A 177      18.133  -7.796   8.896  1.00 10.96           N\\nATOM   1358  CA  LEU A 177      17.892  -9.151   9.405  1.00 10.96           C\\nATOM   1359  C   LEU A 177      19.146  -9.831   9.997  1.00 10.96           C\\nATOM   1360  O   LEU A 177      19.022 -10.924  10.548  1.00 10.96           O\\nATOM   1361  CB  LEU A 177      17.268 -10.013   8.285  1.00 10.96           C\\nATOM   1362  CG  LEU A 177      15.725  -9.941   8.206  1.00 10.96           C\\nATOM   1363  CD1 LEU A 177      15.056 -10.667   9.386  1.00 10.96           C\\nATOM   1364  CD2 LEU A 177      15.176  -8.513   8.037  1.00 10.96           C\\nATOM   1365  N   GLU A 178      20.317  -9.168   9.973  1.00 12.96           N\\nATOM   1366  CA  GLU A 178      21.511  -9.535  10.742  1.00 12.96           C\\nATOM   1367  C   GLU A 178      21.524  -8.850  12.127  1.00 12.96           C\\nATOM   1368  O   GLU A 178      22.421  -9.115  12.924  1.00 12.96           O\\nATOM   1369  CB  GLU A 178      22.763  -9.107   9.951  1.00 12.96           C\\nATOM   1370  CG  GLU A 178      22.906  -9.892   8.630  1.00 12.96           C\\nATOM   1371  CD  GLU A 178      24.146  -9.532   7.811  1.00 12.96           C\\nATOM   1372  OE1 GLU A 178      24.913  -8.645   8.245  1.00 12.96           O\\nATOM   1373  OE2 GLU A 178      24.299 -10.154   6.738  1.00 12.96           O\\nATOM   1374  N   GLY A 179      20.519  -8.014  12.434  1.00 12.17           N\\nATOM   1375  CA  GLY A 179      20.294  -7.385  13.729  1.00 12.17           C\\nATOM   1376  C   GLY A 179      21.047  -6.079  13.964  1.00 12.17           C\\nATOM   1377  O   GLY A 179      21.024  -5.583  15.086  1.00 12.17           O\\nATOM   1378  N   ASN A 180      21.703  -5.507  12.947  1.00 11.39           N\\nATOM   1379  CA  ASN A 180      22.379  -4.218  13.062  1.00 11.39           C\\nATOM   1380  C   ASN A 180      21.364  -3.113  12.757  1.00 11.39           C\\nATOM   1381  O   ASN A 180      20.782  -3.094  11.672  1.00 11.39           O\\nATOM   1382  CB  ASN A 180      23.534  -4.141  12.041  1.00 11.39           C\\nATOM   1383  CG  ASN A 180      24.645  -5.137  12.371  1.00 11.39           C\\nATOM   1384  OD1 ASN A 180      24.792  -6.157  11.703  1.00 11.39           O\\nATOM   1385  ND2 ASN A 180      25.439  -4.841  13.402  1.00 11.39           N\\nATOM   1386  N   PHE A 181      21.164  -2.194  13.718  1.00 10.54           N\\nATOM   1387  CA  PHE A 181      20.304  -1.015  13.611  1.00 10.54           C\\nATOM   1388  C   PHE A 181      20.740  -0.106  12.462  1.00 10.54           C\\nATOM   1389  O   PHE A 181      21.933   0.135  12.271  1.00 10.54           O\\nATOM   1390  CB  PHE A 181      20.279  -0.217  14.933  1.00 10.54           C\\nATOM   1391  CG  PHE A 181      19.117  -0.553  15.851  1.00 10.54           C\\nATOM   1392  CD1 PHE A 181      19.267  -1.526  16.859  1.00 10.54           C\\nATOM   1393  CD2 PHE A 181      17.886   0.124  15.712  1.00 10.54           C\\nATOM   1394  CE1 PHE A 181      18.197  -1.818  17.725  1.00 10.54           C\\nATOM   1395  CE2 PHE A 181      16.818  -0.161  16.584  1.00 10.54           C\\nATOM   1396  CZ  PHE A 181      16.976  -1.132  17.591  1.00 10.54           C\\nATOM   1397  N   TYR A 182      19.756   0.440  11.740  1.00 13.46           N\\nATOM   1398  CA  TYR A 182      19.958   1.569  10.844  1.00 13.46           C\\nATOM   1399  C   TYR A 182      20.118   2.816  11.710  1.00 13.46           C\\nATOM   1400  O   TYR A 182      19.322   3.057  12.616  1.00 13.46           O\\nATOM   1401  CB  TYR A 182      18.737   1.718   9.926  1.00 13.46           C\\nATOM   1402  CG  TYR A 182      18.685   0.630   8.883  1.00 13.46           C\\nATOM   1403  CD1 TYR A 182      19.591   0.662   7.806  1.00 13.46           C\\nATOM   1404  CD2 TYR A 182      17.771  -0.434   9.010  1.00 13.46           C\\nATOM   1405  CE1 TYR A 182      19.589  -0.376   6.862  1.00 13.46           C\\nATOM   1406  CE2 TYR A 182      17.758  -1.461   8.053  1.00 13.46           C\\nATOM   1407  CZ  TYR A 182      18.666  -1.429   6.977  1.00 13.46           C\\nATOM   1408  OH  TYR A 182      18.665  -2.413   6.040  1.00 13.46           O\\nATOM   1409  N   GLY A 183      21.181   3.583  11.449  1.00 22.94           N\\nATOM   1410  CA  GLY A 183      21.528   4.774  12.204  1.00 22.94           C\\nATOM   1411  C   GLY A 183      22.156   4.466  13.567  1.00 22.94           C\\nATOM   1412  O   GLY A 183      22.610   3.344  13.798  1.00 22.94           O\\nATOM   1413  N   PRO A 184      22.198   5.468  14.470  1.00 14.16           N\\nATOM   1414  CA  PRO A 184      22.826   5.361  15.793  1.00 14.16           C\\nATOM   1415  C   PRO A 184      21.901   4.754  16.868  1.00 14.16           C\\nATOM   1416  O   PRO A 184      22.282   4.714  18.038  1.00 14.16           O\\nATOM   1417  CB  PRO A 184      23.142   6.823  16.145  1.00 14.16           C\\nATOM   1418  CG  PRO A 184      22.008   7.608  15.507  1.00 14.16           C\\nATOM   1419  CD  PRO A 184      21.709   6.831  14.229  1.00 14.16           C\\nATOM   1420  N   PHE A 185      20.687   4.317  16.491  1.00 16.87           N\\nATOM   1421  CA  PHE A 185      19.626   3.915  17.408  1.00 16.87           C\\nATOM   1422  C   PHE A 185      19.911   2.560  18.070  1.00 16.87           C\\nATOM   1423  O   PHE A 185      20.613   1.714  17.515  1.00 16.87           O\\nATOM   1424  CB  PHE A 185      18.288   3.873  16.641  1.00 16.87           C\\nATOM   1425  CG  PHE A 185      17.911   5.215  16.031  1.00 16.87           C\\nATOM   1426  CD1 PHE A 185      17.758   6.353  16.852  1.00 16.87           C\\nATOM   1427  CD2 PHE A 185      17.769   5.346  14.635  1.00 16.87           C\\nATOM   1428  CE1 PHE A 185      17.498   7.613  16.279  1.00 16.87           C\\nATOM   1429  CE2 PHE A 185      17.505   6.603  14.060  1.00 16.87           C\\nATOM   1430  CZ  PHE A 185      17.376   7.738  14.882  1.00 16.87           C\\nATOM   1431  N   VAL A 186      19.371   2.397  19.283  1.00 20.33           N\\nATOM   1432  CA  VAL A 186      19.539   1.249  20.167  1.00 20.33           C\\nATOM   1433  C   VAL A 186      18.166   0.747  20.628  1.00 20.33           C\\nATOM   1434  O   VAL A 186      17.203   1.514  20.687  1.00 20.33           O\\nATOM   1435  CB  VAL A 186      20.491   1.598  21.349  1.00 20.33           C\\nATOM   1436  CG1 VAL A 186      21.935   1.855  20.887  1.00 20.33           C\\nATOM   1437  CG2 VAL A 186      20.004   2.768  22.226  1.00 20.33           C\\nATOM   1438  N   ASP A 187      18.089  -0.542  20.993  1.00 14.42           N\\nATOM   1439  CA  ASP A 187      16.916  -1.185  21.591  1.00 14.42           C\\nATOM   1440  C   ASP A 187      16.943  -0.963  23.117  1.00 14.42           C\\nATOM   1441  O   ASP A 187      17.122  -1.884  23.912  1.00 14.42           O\\nATOM   1442  CB  ASP A 187      16.849  -2.690  21.212  1.00 14.42           C\\nATOM   1443  CG  ASP A 187      18.106  -3.556  21.366  1.00 14.42           C\\nATOM   1444  OD1 ASP A 187      19.152  -3.065  21.844  1.00 14.42           O\\nATOM   1445  OD2 ASP A 187      17.998  -4.738  20.981  1.00 14.42           O\\nATOM   1446  N   ARG A 188      16.752   0.301  23.508  1.00 19.45           N\\nATOM   1447  CA  ARG A 188      16.660   0.785  24.875  1.00 19.45           C\\nATOM   1448  C   ARG A 188      15.718   1.990  24.883  1.00 19.45           C\\nATOM   1449  O   ARG A 188      15.852   2.881  24.048  1.00 19.45           O\\nATOM   1450  CB  ARG A 188      18.071   1.063  25.443  1.00 19.45           C\\nATOM   1451  CG  ARG A 188      18.078   1.681  26.852  1.00 19.45           C\\nATOM   1452  CD  ARG A 188      19.489   1.984  27.366  1.00 19.45           C\\nATOM   1453  NE  ARG A 188      19.416   2.627  28.682  1.00 19.45           N\\nATOM   1454  CZ  ARG A 188      20.342   3.406  29.262  1.00 19.45           C\\nATOM   1455  NH1 ARG A 188      21.501   3.690  28.652  1.00 19.45           N\\nATOM   1456  NH2 ARG A 188      20.093   3.907  30.478  1.00 19.45           N\\nATOM   1457  N   GLN A 189      14.791   2.032  25.849  1.00 36.62           N\\nATOM   1458  CA  GLN A 189      13.893   3.147  26.122  1.00 36.62           C\\nATOM   1459  C   GLN A 189      14.699   4.239  26.851  1.00 36.62           C\\nATOM   1460  O   GLN A 189      14.831   4.229  28.075  1.00 36.62           O\\nATOM   1461  CB  GLN A 189      12.729   2.612  26.986  1.00 36.62           C\\nATOM   1462  CG  GLN A 189      11.667   3.672  27.350  1.00 36.62           C\\nATOM   1463  CD  GLN A 189      10.507   3.155  28.211  1.00 36.62           C\\nATOM   1464  OE1 GLN A 189       9.701   3.956  28.677  1.00 36.62           O\\nATOM   1465  NE2 GLN A 189      10.413   1.845  28.455  1.00 36.62           N\\nATOM   1466  N   THR A 190      15.267   5.156  26.058  1.00 32.27           N\\nATOM   1467  CA  THR A 190      16.133   6.258  26.469  1.00 32.27           C\\nATOM   1468  C   THR A 190      15.911   7.469  25.545  1.00 32.27           C\\nATOM   1469  O   THR A 190      15.077   7.425  24.637  1.00 32.27           O\\nATOM   1470  CB  THR A 190      17.599   5.756  26.629  1.00 32.27           C\\nATOM   1471  OG1 THR A 190      18.400   6.726  27.277  1.00 32.27           O\\nATOM   1472  CG2 THR A 190      18.295   5.386  25.305  1.00 32.27           C\\nATOM   1473  N   ALA A 191      16.671   8.552  25.774  1.00 94.66           N\\nATOM   1474  CA  ALA A 191      16.702   9.749  24.941  1.00 94.66           C\\nATOM   1475  C   ALA A 191      17.405   9.427  23.615  1.00 94.66           C\\nATOM   1476  O   ALA A 191      18.616   9.209  23.579  1.00 94.66           O\\nATOM   1477  CB  ALA A 191      17.469  10.848  25.693  1.00 94.66           C\\nATOM   1478  N   GLN A 192      16.607   9.365  22.541  1.00 28.38           N\\nATOM   1479  CA  GLN A 192      17.020   9.024  21.187  1.00 28.38           C\\nATOM   1480  C   GLN A 192      16.353  10.008  20.229  1.00 28.38           C\\nATOM   1481  O   GLN A 192      15.290   9.719  19.678  1.00 28.38           O\\nATOM   1482  CB  GLN A 192      16.668   7.559  20.867  1.00 28.38           C\\nATOM   1483  CG  GLN A 192      17.357   6.518  21.763  1.00 28.38           C\\nATOM   1484  CD  GLN A 192      17.358   5.150  21.093  1.00 28.38           C\\nATOM   1485  OE1 GLN A 192      18.128   4.921  20.168  1.00 28.38           O\\nATOM   1486  NE2 GLN A 192      16.511   4.233  21.551  1.00 28.38           N\\nATOM   1487  N   ALA A 193      16.974  11.182  20.051  1.00 15.62           N\\nATOM   1488  CA  ALA A 193      16.542  12.206  19.107  1.00 15.62           C\\nATOM   1489  C   ALA A 193      17.132  11.883  17.731  1.00 15.62           C\\nATOM   1490  O   ALA A 193      18.329  11.619  17.612  1.00 15.62           O\\nATOM   1491  CB  ALA A 193      17.108  13.559  19.565  1.00 15.62           C\\nATOM   1492  N   ALA A 194      16.287  11.924  16.693  1.00 18.22           N\\nATOM   1493  CA  ALA A 194      16.677  11.772  15.299  1.00 18.22           C\\nATOM   1494  C   ALA A 194      17.244  13.098  14.771  1.00 18.22           C\\nATOM   1495  O   ALA A 194      16.810  14.178  15.179  1.00 18.22           O\\nATOM   1496  CB  ALA A 194      15.423  11.421  14.485  1.00 18.22           C\\nATOM   1497  N   GLY A 195      18.194  12.998  13.830  1.00 19.71           N\\nATOM   1498  CA  GLY A 195      18.773  14.111  13.083  1.00 19.71           C\\nATOM   1499  C   GLY A 195      17.792  14.711  12.070  1.00 19.71           C\\nATOM   1500  O   GLY A 195      16.615  14.345  12.041  1.00 19.71           O\\nATOM   1501  N   THR A 196      18.281  15.636  11.228  1.00 22.64           N\\nATOM   1502  CA  THR A 196      17.486  16.343  10.220  1.00 22.64           C\\nATOM   1503  C   THR A 196      16.961  15.332   9.190  1.00 22.64           C\\nATOM   1504  O   THR A 196      17.754  14.663   8.523  1.00 22.64           O\\nATOM   1505  CB  THR A 196      18.362  17.411   9.516  1.00 22.64           C\\nATOM   1506  OG1 THR A 196      18.916  18.282  10.482  1.00 22.64           O\\nATOM   1507  CG2 THR A 196      17.575  18.294   8.527  1.00 22.64           C\\nATOM   1508  N   ASP A 197      15.627  15.233   9.070  1.00 28.51           N\\nATOM   1509  CA  ASP A 197      14.974  14.423   8.054  1.00 28.51           C\\nATOM   1510  C   ASP A 197      14.890  15.274   6.782  1.00 28.51           C\\nATOM   1511  O   ASP A 197      13.889  15.939   6.502  1.00 28.51           O\\nATOM   1512  CB  ASP A 197      13.572  13.978   8.532  1.00 28.51           C\\nATOM   1513  CG  ASP A 197      12.986  12.831   7.695  1.00 28.51           C\\nATOM   1514  OD1 ASP A 197      13.583  12.471   6.656  1.00 28.51           O\\nATOM   1515  OD2 ASP A 197      11.942  12.303   8.125  1.00 28.51           O\\nATOM   1516  N   THR A 198      15.994  15.246   6.027  1.00 23.01           N\\nATOM   1517  CA  THR A 198      16.144  15.887   4.733  1.00 23.01           C\\nATOM   1518  C   THR A 198      15.329  15.119   3.677  1.00 23.01           C\\nATOM   1519  O   THR A 198      15.007  13.942   3.853  1.00 23.01           O\\nATOM   1520  CB  THR A 198      17.650  15.931   4.365  1.00 23.01           C\\nATOM   1521  OG1 THR A 198      18.196  14.635   4.346  1.00 23.01           O\\nATOM   1522  CG2 THR A 198      18.505  16.789   5.308  1.00 23.01           C\\nATOM   1523  N   THR A 199      14.970  15.806   2.587  1.00 14.59           N\\nATOM   1524  CA  THR A 199      14.190  15.229   1.500  1.00 14.59           C\\nATOM   1525  C   THR A 199      15.157  14.540   0.520  1.00 14.59           C\\nATOM   1526  O   THR A 199      16.212  15.081   0.188  1.00 14.59           O\\nATOM   1527  CB  THR A 199      13.407  16.354   0.791  1.00 14.59           C\\nATOM   1528  OG1 THR A 199      12.498  16.932   1.708  1.00 14.59           O\\nATOM   1529  CG2 THR A 199      12.539  15.846  -0.366  1.00 14.59           C\\nATOM   1530  N   ILE A 200      14.784  13.335   0.067  1.00 12.39           N\\nATOM   1531  CA  ILE A 200      15.573  12.482  -0.820  1.00 12.39           C\\nATOM   1532  C   ILE A 200      15.343  13.001  -2.253  1.00 12.39           C\\nATOM   1533  O   ILE A 200      14.291  12.776  -2.847  1.00 12.39           O\\nATOM   1534  CB  ILE A 200      15.082  11.011  -0.673  1.00 12.39           C\\nATOM   1535  CG1 ILE A 200      15.086  10.498   0.792  1.00 12.39           C\\nATOM   1536  CG2 ILE A 200      15.921  10.061  -1.553  1.00 12.39           C\\nATOM   1537  CD1 ILE A 200      13.951   9.503   1.084  1.00 12.39           C\\nATOM   1538  N   THR A 201      16.321  13.747  -2.783  1.00  9.12           N\\nATOM   1539  CA  THR A 201      16.186  14.588  -3.976  1.00  9.12           C\\nATOM   1540  C   THR A 201      16.018  13.770  -5.264  1.00  9.12           C\\nATOM   1541  O   THR A 201      15.186  14.128  -6.096  1.00  9.12           O\\nATOM   1542  CB  THR A 201      17.459  15.444  -4.090  1.00  9.12           C\\nATOM   1543  OG1 THR A 201      17.587  16.198  -2.909  1.00  9.12           O\\nATOM   1544  CG2 THR A 201      17.484  16.420  -5.271  1.00  9.12           C\\nATOM   1545  N   VAL A 202      16.790  12.681  -5.421  1.00 10.09           N\\nATOM   1546  CA  VAL A 202      16.729  11.761  -6.561  1.00 10.09           C\\nATOM   1547  C   VAL A 202      15.357  11.063  -6.671  1.00 10.09           C\\nATOM   1548  O   VAL A 202      14.915  10.768  -7.780  1.00 10.09           O\\nATOM   1549  CB  VAL A 202      17.906  10.753  -6.467  1.00 10.09           C\\nATOM   1550  CG1 VAL A 202      17.826   9.788  -5.272  1.00 10.09           C\\nATOM   1551  CG2 VAL A 202      18.108   9.962  -7.770  1.00 10.09           C\\nATOM   1552  N   ASN A 203      14.670  10.860  -5.534  1.00  8.68           N\\nATOM   1553  CA  ASN A 203      13.325  10.298  -5.461  1.00  8.68           C\\nATOM   1554  C   ASN A 203      12.259  11.337  -5.836  1.00  8.68           C\\nATOM   1555  O   ASN A 203      11.300  10.971  -6.513  1.00  8.68           O\\nATOM   1556  CB  ASN A 203      13.035   9.788  -4.031  1.00  8.68           C\\nATOM   1557  CG  ASN A 203      13.771   8.499  -3.659  1.00  8.68           C\\nATOM   1558  OD1 ASN A 203      14.789   8.149  -4.251  1.00  8.68           O\\nATOM   1559  ND2 ASN A 203      13.244   7.770  -2.676  1.00  8.68           N\\nATOM   1560  N   VAL A 204      12.429  12.622  -5.462  1.00  6.62           N\\nATOM   1561  CA  VAL A 204      11.555  13.720  -5.899  1.00  6.62           C\\nATOM   1562  C   VAL A 204      11.639  13.913  -7.420  1.00  6.62           C\\nATOM   1563  O   VAL A 204      10.599  14.024  -8.059  1.00  6.62           O\\nATOM   1564  CB  VAL A 204      11.865  15.046  -5.151  1.00  6.62           C\\nATOM   1565  CG1 VAL A 204      11.036  16.234  -5.678  1.00  6.62           C\\nATOM   1566  CG2 VAL A 204      11.616  14.923  -3.643  1.00  6.62           C\\nATOM   1567  N   LEU A 205      12.853  13.898  -7.996  1.00  7.13           N\\nATOM   1568  CA  LEU A 205      13.089  14.009  -9.436  1.00  7.13           C\\nATOM   1569  C   LEU A 205      12.493  12.833 -10.216  1.00  7.13           C\\nATOM   1570  O   LEU A 205      11.889  13.058 -11.262  1.00  7.13           O\\nATOM   1571  CB  LEU A 205      14.601  14.100  -9.711  1.00  7.13           C\\nATOM   1572  CG  LEU A 205      15.234  15.426  -9.252  1.00  7.13           C\\nATOM   1573  CD1 LEU A 205      16.760  15.313  -9.208  1.00  7.13           C\\nATOM   1574  CD2 LEU A 205      14.815  16.611 -10.134  1.00  7.13           C\\nATOM   1575  N   ALA A 206      12.592  11.607  -9.673  1.00  8.73           N\\nATOM   1576  CA  ALA A 206      11.974  10.399 -10.213  1.00  8.73           C\\nATOM   1577  C   ALA A 206      10.444  10.488 -10.231  1.00  8.73           C\\nATOM   1578  O   ALA A 206       9.824  10.108 -11.218  1.00  8.73           O\\nATOM   1579  CB  ALA A 206      12.406   9.197  -9.368  1.00  8.73           C\\nATOM   1580  N   TRP A 207       9.835  11.015  -9.160  1.00  8.66           N\\nATOM   1581  CA  TRP A 207       8.395  11.217  -9.041  1.00  8.66           C\\nATOM   1582  C   TRP A 207       7.886  12.403  -9.889  1.00  8.66           C\\nATOM   1583  O   TRP A 207       6.725  12.384 -10.295  1.00  8.66           O\\nATOM   1584  CB  TRP A 207       8.082  11.411  -7.550  1.00  8.66           C\\nATOM   1585  CG  TRP A 207       6.719  11.911  -7.198  1.00  8.66           C\\nATOM   1586  CD1 TRP A 207       5.608  11.164  -7.016  1.00  8.66           C\\nATOM   1587  CD2 TRP A 207       6.323  13.294  -6.975  1.00  8.66           C\\nATOM   1588  NE1 TRP A 207       4.559  11.988  -6.664  1.00  8.66           N\\nATOM   1589  CE2 TRP A 207       4.950  13.309  -6.597  1.00  8.66           C\\nATOM   1590  CE3 TRP A 207       6.996  14.535  -7.031  1.00  8.66           C\\nATOM   1591  CZ2 TRP A 207       4.299  14.493  -6.220  1.00  8.66           C\\nATOM   1592  CZ3 TRP A 207       6.337  15.734  -6.701  1.00  8.66           C\\nATOM   1593  CH2 TRP A 207       4.998  15.711  -6.269  1.00  8.66           C\\nATOM   1594  N   LEU A 208       8.736  13.394 -10.218  1.00  6.89           N\\nATOM   1595  CA  LEU A 208       8.418  14.441 -11.193  1.00  6.89           C\\nATOM   1596  C   LEU A 208       8.499  13.892 -12.631  1.00  6.89           C\\nATOM   1597  O   LEU A 208       7.743  14.352 -13.482  1.00  6.89           O\\nATOM   1598  CB  LEU A 208       9.388  15.637 -11.057  1.00  6.89           C\\nATOM   1599  CG  LEU A 208       9.223  16.504  -9.788  1.00  6.89           C\\nATOM   1600  CD1 LEU A 208      10.397  17.486  -9.654  1.00  6.89           C\\nATOM   1601  CD2 LEU A 208       7.892  17.268  -9.745  1.00  6.89           C\\nATOM   1602  N   TYR A 209       9.334  12.869 -12.901  1.00  7.71           N\\nATOM   1603  CA  TYR A 209       9.310  12.112 -14.157  1.00  7.71           C\\nATOM   1604  C   TYR A 209       8.071  11.211 -14.244  1.00  7.71           C\\nATOM   1605  O   TYR A 209       7.490  11.109 -15.322  1.00  7.71           O\\nATOM   1606  CB  TYR A 209      10.570  11.231 -14.298  1.00  7.71           C\\nATOM   1607  CG  TYR A 209      11.795  11.970 -14.790  1.00  7.71           C\\nATOM   1608  CD1 TYR A 209      11.791  12.515 -16.087  1.00  7.71           C\\nATOM   1609  CD2 TYR A 209      12.946  12.092 -13.985  1.00  7.71           C\\nATOM   1610  CE1 TYR A 209      12.916  13.203 -16.564  1.00  7.71           C\\nATOM   1611  CE2 TYR A 209      14.071  12.793 -14.458  1.00  7.71           C\\nATOM   1612  CZ  TYR A 209      14.050  13.346 -15.752  1.00  7.71           C\\nATOM   1613  OH  TYR A 209      15.128  14.013 -16.236  1.00  7.71           O\\nATOM   1614  N   ALA A 210       7.628  10.623 -13.116  1.00  9.52           N\\nATOM   1615  CA  ALA A 210       6.394   9.844 -13.000  1.00  9.52           C\\nATOM   1616  C   ALA A 210       5.162  10.709 -13.290  1.00  9.52           C\\nATOM   1617  O   ALA A 210       4.238  10.250 -13.952  1.00  9.52           O\\nATOM   1618  CB  ALA A 210       6.270   9.267 -11.581  1.00  9.52           C\\nATOM   1619  N   ALA A 211       5.180  11.972 -12.832  1.00 10.16           N\\nATOM   1620  CA  ALA A 211       4.136  12.960 -13.056  1.00 10.16           C\\nATOM   1621  C   ALA A 211       4.007  13.329 -14.537  1.00 10.16           C\\nATOM   1622  O   ALA A 211       2.887  13.328 -15.041  1.00 10.16           O\\nATOM   1623  CB  ALA A 211       4.436  14.199 -12.210  1.00 10.16           C\\nATOM   1624  N   VAL A 212       5.129  13.568 -15.244  1.00 12.58           N\\nATOM   1625  CA  VAL A 212       5.160  13.828 -16.689  1.00 12.58           C\\nATOM   1626  C   VAL A 212       4.635  12.636 -17.507  1.00 12.58           C\\nATOM   1627  O   VAL A 212       3.844  12.850 -18.423  1.00 12.58           O\\nATOM   1628  CB  VAL A 212       6.585  14.245 -17.142  1.00 12.58           C\\nATOM   1629  CG1 VAL A 212       6.768  14.287 -18.676  1.00 12.58           C\\nATOM   1630  CG2 VAL A 212       6.954  15.617 -16.562  1.00 12.58           C\\nATOM   1631  N   ILE A 213       5.042  11.399 -17.168  1.00 14.98           N\\nATOM   1632  CA  ILE A 213       4.582  10.152 -17.798  1.00 14.98           C\\nATOM   1633  C   ILE A 213       3.053   9.964 -17.657  1.00 14.98           C\\nATOM   1634  O   ILE A 213       2.420   9.434 -18.569  1.00 14.98           O\\nATOM   1635  CB  ILE A 213       5.392   8.945 -17.224  1.00 14.98           C\\nATOM   1636  CG1 ILE A 213       6.878   9.009 -17.655  1.00 14.98           C\\nATOM   1637  CG2 ILE A 213       4.807   7.568 -17.610  1.00 14.98           C\\nATOM   1638  CD1 ILE A 213       7.826   8.156 -16.796  1.00 14.98           C\\nATOM   1639  N   ASN A 214       2.468  10.432 -16.542  1.00 18.81           N\\nATOM   1640  CA  ASN A 214       1.050  10.309 -16.198  1.00 18.81           C\\nATOM   1641  C   ASN A 214       0.255  11.617 -16.408  1.00 18.81           C\\nATOM   1642  O   ASN A 214      -0.884  11.718 -15.953  1.00 18.81           O\\nATOM   1643  CB  ASN A 214       0.945   9.788 -14.745  1.00 18.81           C\\nATOM   1644  CG  ASN A 214       1.352   8.313 -14.646  1.00 18.81           C\\nATOM   1645  OD1 ASN A 214       0.599   7.443 -15.067  1.00 18.81           O\\nATOM   1646  ND2 ASN A 214       2.543   8.016 -14.128  1.00 18.81           N\\nATOM   1647  N   GLY A 215       0.814  12.584 -17.158  1.00 21.39           N\\nATOM   1648  CA  GLY A 215       0.104  13.727 -17.734  1.00 21.39           C\\nATOM   1649  C   GLY A 215       0.107  15.036 -16.931  1.00 21.39           C\\nATOM   1650  O   GLY A 215      -0.351  16.045 -17.467  1.00 21.39           O\\nATOM   1651  N  AASP A 216       0.636  15.060 -15.695  0.17 15.98           N\\nATOM   1652  CA AASP A 216       0.840  16.284 -14.908  0.17 15.98           C\\nATOM   1653  C  AASP A 216       2.175  16.914 -15.325  0.17 15.98           C\\nATOM   1654  O  AASP A 216       3.243  16.349 -15.095  0.17 15.98           O\\nATOM   1655  CB AASP A 216       0.875  15.963 -13.394  0.17 15.98           C\\nATOM   1656  CG AASP A 216      -0.490  15.631 -12.783  0.17 15.98           C\\nATOM   1657  OD1AASP A 216      -1.513  15.703 -13.498  0.17 15.98           O\\nATOM   1658  OD2AASP A 216      -0.503  15.335 -11.570  0.17 15.98           O\\nATOM   1659  N   ARG A 217       2.087  18.073 -15.993  1.00 15.72           N\\nATOM   1660  CA  ARG A 217       3.177  18.714 -16.729  1.00 15.72           C\\nATOM   1661  C   ARG A 217       3.283  20.222 -16.470  1.00 15.72           C\\nATOM   1662  O   ARG A 217       4.184  20.850 -17.025  1.00 15.72           O\\nATOM   1663  CB  ARG A 217       3.050  18.390 -18.237  1.00 15.72           C\\nATOM   1664  CG  ARG A 217       3.230  16.902 -18.571  1.00 15.72           C\\nATOM   1665  CD  ARG A 217       3.061  16.581 -20.058  1.00 15.72           C\\nATOM   1666  NE  ARG A 217       3.247  15.140 -20.277  1.00 15.72           N\\nATOM   1667  CZ  ARG A 217       2.584  14.349 -21.137  1.00 15.72           C\\nATOM   1668  NH1 ARG A 217       1.626  14.829 -21.941  1.00 15.72           N\\nATOM   1669  NH2 ARG A 217       2.891  13.047 -21.187  1.00 15.72           N\\nATOM   1670  N   TRP A 218       2.389  20.814 -15.654  1.00 13.94           N\\nATOM   1671  CA  TRP A 218       2.288  22.261 -15.412  1.00 13.94           C\\nATOM   1672  C   TRP A 218       3.569  22.902 -14.845  1.00 13.94           C\\nATOM   1673  O   TRP A 218       3.838  24.072 -15.114  1.00 13.94           O\\nATOM   1674  CB  TRP A 218       1.082  22.560 -14.490  1.00 13.94           C\\nATOM   1675  CG  TRP A 218       1.146  22.075 -13.065  1.00 13.94           C\\nATOM   1676  CD1 TRP A 218       0.529  20.974 -12.578  1.00 13.94           C\\nATOM   1677  CD2 TRP A 218       1.847  22.671 -11.926  1.00 13.94           C\\nATOM   1678  NE1 TRP A 218       0.801  20.847 -11.233  1.00 13.94           N\\nATOM   1679  CE2 TRP A 218       1.618  21.858 -10.775  1.00 13.94           C\\nATOM   1680  CE3 TRP A 218       2.663  23.814 -11.745  1.00 13.94           C\\nATOM   1681  CZ2 TRP A 218       2.169  22.162  -9.518  1.00 13.94           C\\nATOM   1682  CZ3 TRP A 218       3.256  24.104 -10.501  1.00 13.94           C\\nATOM   1683  CH2 TRP A 218       3.003  23.284  -9.386  1.00 13.94           C\\nATOM   1684  N   PHE A 219       4.345  22.132 -14.065  1.00 11.15           N\\nATOM   1685  CA  PHE A 219       5.559  22.557 -13.373  1.00 11.15           C\\nATOM   1686  C   PHE A 219       6.810  22.541 -14.269  1.00 11.15           C\\nATOM   1687  O   PHE A 219       7.858  23.003 -13.820  1.00 11.15           O\\nATOM   1688  CB  PHE A 219       5.787  21.613 -12.172  1.00 11.15           C\\nATOM   1689  CG  PHE A 219       5.918  20.136 -12.509  1.00 11.15           C\\nATOM   1690  CD1 PHE A 219       7.162  19.590 -12.894  1.00 11.15           C\\nATOM   1691  CD2 PHE A 219       4.776  19.310 -12.467  1.00 11.15           C\\nATOM   1692  CE1 PHE A 219       7.255  18.236 -13.268  1.00 11.15           C\\nATOM   1693  CE2 PHE A 219       4.876  17.951 -12.812  1.00 11.15           C\\nATOM   1694  CZ  PHE A 219       6.111  17.419 -13.226  1.00 11.15           C\\nATOM   1695  N   LEU A 220       6.725  22.032 -15.512  1.00 14.63           N\\nATOM   1696  CA  LEU A 220       7.829  22.043 -16.471  1.00 14.63           C\\nATOM   1697  C   LEU A 220       8.042  23.475 -16.974  1.00 14.63           C\\nATOM   1698  O   LEU A 220       7.175  24.038 -17.643  1.00 14.63           O\\nATOM   1699  CB  LEU A 220       7.520  21.120 -17.667  1.00 14.63           C\\nATOM   1700  CG  LEU A 220       7.423  19.616 -17.337  1.00 14.63           C\\nATOM   1701  CD1 LEU A 220       7.137  18.829 -18.627  1.00 14.63           C\\nATOM   1702  CD2 LEU A 220       8.685  19.081 -16.635  1.00 14.63           C\\nATOM   1703  N  AASN A 221       9.195  24.058 -16.617  0.17 26.99           N\\nATOM   1704  CA AASN A 221       9.583  25.421 -16.970  0.17 26.99           C\\nATOM   1705  C  AASN A 221      10.507  25.438 -18.199  0.17 26.99           C\\nATOM   1706  O  AASN A 221      10.783  24.408 -18.814  0.17 26.99           O\\nATOM   1707  CB AASN A 221      10.262  26.114 -15.758  0.17 26.99           C\\nATOM   1708  CG AASN A 221      11.489  25.424 -15.148  0.17 26.99           C\\nATOM   1709  OD1AASN A 221      12.258  24.734 -15.814  0.17 26.99           O\\nATOM   1710  ND2AASN A 221      11.671  25.620 -13.849  0.17 26.99           N\\nATOM   1711  N   ARG A 222      10.982  26.640 -18.544  1.00 22.56           N\\nATOM   1712  CA  ARG A 222      11.867  26.928 -19.668  1.00 22.56           C\\nATOM   1713  C   ARG A 222      13.356  26.908 -19.269  1.00 22.56           C\\nATOM   1714  O   ARG A 222      14.203  27.219 -20.105  1.00 22.56           O\\nATOM   1715  CB  ARG A 222      11.521  28.355 -20.150  1.00 22.56           C\\nATOM   1716  CG  ARG A 222      10.074  28.475 -20.655  1.00 22.56           C\\nATOM   1717  CD  ARG A 222       9.751  29.861 -21.222  1.00 22.56           C\\nATOM   1718  NE  ARG A 222       8.575  29.790 -22.098  1.00 22.56           N\\nATOM   1719  CZ  ARG A 222       8.590  29.421 -23.390  1.00 22.56           C\\nATOM   1720  NH1 ARG A 222       9.732  29.092 -24.014  1.00 22.56           N\\nATOM   1721  NH2 ARG A 222       7.439  29.373 -24.070  1.00 22.56           N\\nATOM   1722  N   PHE A 223      13.680  26.583 -18.006  1.00 23.05           N\\nATOM   1723  CA  PHE A 223      15.003  26.769 -17.413  1.00 23.05           C\\nATOM   1724  C   PHE A 223      15.779  25.449 -17.327  1.00 23.05           C\\nATOM   1725  O   PHE A 223      15.208  24.359 -17.369  1.00 23.05           O\\nATOM   1726  CB  PHE A 223      14.819  27.308 -15.976  1.00 23.05           C\\nATOM   1727  CG  PHE A 223      13.943  28.545 -15.834  1.00 23.05           C\\nATOM   1728  CD1 PHE A 223      13.921  29.556 -16.821  1.00 23.05           C\\nATOM   1729  CD2 PHE A 223      13.115  28.670 -14.701  1.00 23.05           C\\nATOM   1730  CE1 PHE A 223      13.044  30.651 -16.694  1.00 23.05           C\\nATOM   1731  CE2 PHE A 223      12.241  29.765 -14.571  1.00 23.05           C\\nATOM   1732  CZ  PHE A 223      12.199  30.752 -15.572  1.00 23.05           C\\nATOM   1733  N   THR A 224      17.097  25.586 -17.140  1.00 21.85           N\\nATOM   1734  CA  THR A 224      18.038  24.535 -16.775  1.00 21.85           C\\nATOM   1735  C   THR A 224      18.932  25.040 -15.628  1.00 21.85           C\\nATOM   1736  O   THR A 224      18.936  26.229 -15.303  1.00 21.85           O\\nATOM   1737  CB  THR A 224      18.773  23.987 -18.022  1.00 21.85           C\\nATOM   1738  OG1 THR A 224      19.378  22.749 -17.711  1.00 21.85           O\\nATOM   1739  CG2 THR A 224      19.857  24.919 -18.589  1.00 21.85           C\\nATOM   1740  N   THR A 225      19.678  24.123 -15.006  1.00 20.22           N\\nATOM   1741  CA  THR A 225      20.608  24.367 -13.908  1.00 20.22           C\\nATOM   1742  C   THR A 225      21.850  23.479 -14.091  1.00 20.22           C\\nATOM   1743  O   THR A 225      21.918  22.674 -15.017  1.00 20.22           O\\nATOM   1744  CB  THR A 225      19.859  24.274 -12.546  1.00 20.22           C\\nATOM   1745  OG1 THR A 225      20.687  24.597 -11.453  1.00 20.22           O\\nATOM   1746  CG2 THR A 225      19.293  22.887 -12.229  1.00 20.22           C\\nATOM   1747  N   THR A 226      22.841  23.633 -13.210  1.00 22.36           N\\nATOM   1748  CA  THR A 226      23.969  22.721 -13.029  1.00 22.36           C\\nATOM   1749  C   THR A 226      23.743  21.958 -11.710  1.00 22.36           C\\nATOM   1750  O   THR A 226      22.879  22.339 -10.916  1.00 22.36           O\\nATOM   1751  CB  THR A 226      25.276  23.553 -12.990  1.00 22.36           C\\nATOM   1752  OG1 THR A 226      25.340  24.381 -11.845  1.00 22.36           O\\nATOM   1753  CG2 THR A 226      25.504  24.407 -14.247  1.00 22.36           C\\nATOM   1754  N   LEU A 227      24.530  20.898 -11.448  1.00 21.61           N\\nATOM   1755  CA  LEU A 227      24.510  20.190 -10.162  1.00 21.61           C\\nATOM   1756  C   LEU A 227      24.896  21.104  -8.995  1.00 21.61           C\\nATOM   1757  O   LEU A 227      24.219  21.076  -7.972  1.00 21.61           O\\nATOM   1758  CB  LEU A 227      25.444  18.956 -10.163  1.00 21.61           C\\nATOM   1759  CG  LEU A 227      24.876  17.692 -10.834  1.00 21.61           C\\nATOM   1760  CD1 LEU A 227      25.889  16.538 -10.746  1.00 21.61           C\\nATOM   1761  CD2 LEU A 227      23.533  17.261 -10.227  1.00 21.61           C\\nATOM   1762  N   ASN A 228      25.934  21.940  -9.161  1.00 20.36           N\\nATOM   1763  CA  ASN A 228      26.396  22.871  -8.134  1.00 20.36           C\\nATOM   1764  C   ASN A 228      25.382  23.985  -7.839  1.00 20.36           C\\nATOM   1765  O   ASN A 228      25.134  24.237  -6.663  1.00 20.36           O\\nATOM   1766  CB  ASN A 228      27.771  23.473  -8.505  1.00 20.36           C\\nATOM   1767  CG  ASN A 228      28.913  22.447  -8.485  1.00 20.36           C\\nATOM   1768  OD1 ASN A 228      28.868  21.460  -7.753  1.00 20.36           O\\nATOM   1769  ND2 ASN A 228      29.941  22.662  -9.306  1.00 20.36           N\\nATOM   1770  N   ASP A 229      24.756  24.607  -8.857  1.00 19.18           N\\nATOM   1771  CA  ASP A 229      23.725  25.641  -8.677  1.00 19.18           C\\nATOM   1772  C   ASP A 229      22.452  25.069  -8.027  1.00 19.18           C\\nATOM   1773  O   ASP A 229      21.897  25.698  -7.126  1.00 19.18           O\\nATOM   1774  CB  ASP A 229      23.411  26.330 -10.029  1.00 19.18           C\\nATOM   1775  CG  ASP A 229      22.272  27.360  -9.967  1.00 19.18           C\\nATOM   1776  OD1 ASP A 229      22.415  28.331  -9.194  1.00 19.18           O\\nATOM   1777  OD2 ASP A 229      21.264  27.156 -10.676  1.00 19.18           O\\nATOM   1778  N   PHE A 230      22.032  23.862  -8.445  1.00 15.87           N\\nATOM   1779  CA  PHE A 230      20.915  23.141  -7.847  1.00 15.87           C\\nATOM   1780  C   PHE A 230      21.190  22.786  -6.374  1.00 15.87           C\\nATOM   1781  O   PHE A 230      20.287  22.923  -5.555  1.00 15.87           O\\nATOM   1782  CB  PHE A 230      20.595  21.863  -8.655  1.00 15.87           C\\nATOM   1783  CG  PHE A 230      19.448  21.084  -8.037  1.00 15.87           C\\nATOM   1784  CD1 PHE A 230      18.123  21.534  -8.199  1.00 15.87           C\\nATOM   1785  CD2 PHE A 230      19.726  20.027  -7.150  1.00 15.87           C\\nATOM   1786  CE1 PHE A 230      17.080  20.949  -7.454  1.00 15.87           C\\nATOM   1787  CE2 PHE A 230      18.695  19.489  -6.365  1.00 15.87           C\\nATOM   1788  CZ  PHE A 230      17.371  19.939  -6.518  1.00 15.87           C\\nATOM   1789  N   ASN A 231      22.410  22.335  -6.037  1.00 19.38           N\\nATOM   1790  CA  ASN A 231      22.774  21.919  -4.682  1.00 19.38           C\\nATOM   1791  C   ASN A 231      22.944  23.081  -3.689  1.00 19.38           C\\nATOM   1792  O   ASN A 231      22.850  22.833  -2.488  1.00 19.38           O\\nATOM   1793  CB  ASN A 231      24.037  21.028  -4.686  1.00 19.38           C\\nATOM   1794  CG  ASN A 231      23.786  19.616  -5.234  1.00 19.38           C\\nATOM   1795  OD1 ASN A 231      22.659  19.125  -5.238  1.00 19.38           O\\nATOM   1796  ND2 ASN A 231      24.838  18.958  -5.722  1.00 19.38           N\\nATOM   1797  N   LEU A 232      23.090  24.337  -4.155  1.00 21.71           N\\nATOM   1798  CA  LEU A 232      22.988  25.532  -3.306  1.00 21.71           C\\nATOM   1799  C   LEU A 232      21.534  25.751  -2.849  1.00 21.71           C\\nATOM   1800  O   LEU A 232      21.306  26.116  -1.697  1.00 21.71           O\\nATOM   1801  CB  LEU A 232      23.437  26.802  -4.068  1.00 21.71           C\\nATOM   1802  CG  LEU A 232      24.923  26.875  -4.473  1.00 21.71           C\\nATOM   1803  CD1 LEU A 232      25.191  28.138  -5.305  1.00 21.71           C\\nATOM   1804  CD2 LEU A 232      25.880  26.796  -3.276  1.00 21.71           C\\nATOM   1805  N   VAL A 233      20.557  25.488  -3.735  1.00 16.08           N\\nATOM   1806  CA  VAL A 233      19.126  25.531  -3.433  1.00 16.08           C\\nATOM   1807  C   VAL A 233      18.738  24.302  -2.589  1.00 16.08           C\\nATOM   1808  O   VAL A 233      17.991  24.457  -1.627  1.00 16.08           O\\nATOM   1809  CB  VAL A 233      18.306  25.596  -4.754  1.00 16.08           C\\nATOM   1810  CG1 VAL A 233      16.781  25.637  -4.521  1.00 16.08           C\\nATOM   1811  CG2 VAL A 233      18.724  26.806  -5.608  1.00 16.08           C\\nATOM   1812  N   ALA A 234      19.281  23.107  -2.888  1.00 15.07           N\\nATOM   1813  CA  ALA A 234      19.018  21.869  -2.151  1.00 15.07           C\\nATOM   1814  C   ALA A 234      19.432  21.974  -0.673  1.00 15.07           C\\nATOM   1815  O   ALA A 234      18.637  21.637   0.199  1.00 15.07           O\\nATOM   1816  CB  ALA A 234      19.731  20.680  -2.812  1.00 15.07           C\\nATOM   1817  N   MET A 235      20.625  22.532  -0.399  1.00 25.88           N\\nATOM   1818  CA  MET A 235      21.141  22.881   0.929  1.00 25.88           C\\nATOM   1819  C   MET A 235      20.157  23.757   1.725  1.00 25.88           C\\nATOM   1820  O   MET A 235      19.798  23.405   2.848  1.00 25.88           O\\nATOM   1821  CB  MET A 235      22.513  23.571   0.751  1.00 25.88           C\\nATOM   1822  CG  MET A 235      23.206  24.039   2.048  1.00 25.88           C\\nATOM   1823  SD  MET A 235      24.059  22.743   2.981  1.00 25.88           S\\nATOM   1824  CE  MET A 235      25.635  22.729   2.085  1.00 25.88           C\\nATOM   1825  N   LYS A 236      19.708  24.871   1.121  1.00 16.91           N\\nATOM   1826  CA  LYS A 236      18.770  25.842   1.687  1.00 16.91           C\\nATOM   1827  C   LYS A 236      17.419  25.233   2.114  1.00 16.91           C\\nATOM   1828  O   LYS A 236      16.878  25.649   3.136  1.00 16.91           O\\nATOM   1829  CB  LYS A 236      18.606  27.009   0.682  1.00 16.91           C\\nATOM   1830  CG  LYS A 236      17.448  27.988   0.965  1.00 16.91           C\\nATOM   1831  CD  LYS A 236      17.517  29.253   0.099  1.00 16.91           C\\nATOM   1832  CE  LYS A 236      16.198  30.040   0.058  1.00 16.91           C\\nATOM   1833  NZ  LYS A 236      15.808  30.548   1.383  1.00 16.91           N\\nATOM   1834  N   TYR A 237      16.890  24.258   1.358  1.00 11.29           N\\nATOM   1835  CA  TYR A 237      15.583  23.639   1.594  1.00 11.29           C\\nATOM   1836  C   TYR A 237      15.671  22.280   2.313  1.00 11.29           C\\nATOM   1837  O   TYR A 237      14.651  21.604   2.434  1.00 11.29           O\\nATOM   1838  CB  TYR A 237      14.861  23.504   0.232  1.00 11.29           C\\nATOM   1839  CG  TYR A 237      14.293  24.834  -0.230  1.00 11.29           C\\nATOM   1840  CD1 TYR A 237      13.139  25.347   0.394  1.00 11.29           C\\nATOM   1841  CD2 TYR A 237      14.935  25.596  -1.228  1.00 11.29           C\\nATOM   1842  CE1 TYR A 237      12.666  26.629   0.062  1.00 11.29           C\\nATOM   1843  CE2 TYR A 237      14.468  26.880  -1.559  1.00 11.29           C\\nATOM   1844  CZ  TYR A 237      13.342  27.404  -0.899  1.00 11.29           C\\nATOM   1845  OH  TYR A 237      12.905  28.663  -1.185  1.00 11.29           O\\nATOM   1846  N   ASN A 238      16.858  21.870   2.797  1.00 17.89           N\\nATOM   1847  CA  ASN A 238      17.118  20.594   3.483  1.00 17.89           C\\nATOM   1848  C   ASN A 238      16.827  19.388   2.572  1.00 17.89           C\\nATOM   1849  O   ASN A 238      16.229  18.407   2.994  1.00 17.89           O\\nATOM   1850  CB  ASN A 238      16.407  20.479   4.861  1.00 17.89           C\\nATOM   1851  CG  ASN A 238      16.915  21.472   5.906  1.00 17.89           C\\nATOM   1852  OD1 ASN A 238      18.117  21.586   6.136  1.00 17.89           O\\nATOM   1853  ND2 ASN A 238      15.999  22.211   6.532  1.00 17.89           N\\nATOM   1854  N   TYR A 239      17.214  19.487   1.299  1.00 13.89           N\\nATOM   1855  CA  TYR A 239      17.227  18.409   0.323  1.00 13.89           C\\nATOM   1856  C   TYR A 239      18.653  17.855   0.283  1.00 13.89           C\\nATOM   1857  O   TYR A 239      19.620  18.611   0.403  1.00 13.89           O\\nATOM   1858  CB  TYR A 239      16.889  18.997  -1.064  1.00 13.89           C\\nATOM   1859  CG  TYR A 239      15.423  18.950  -1.434  1.00 13.89           C\\nATOM   1860  CD1 TYR A 239      14.463  19.586  -0.624  1.00 13.89           C\\nATOM   1861  CD2 TYR A 239      15.013  18.262  -2.594  1.00 13.89           C\\nATOM   1862  CE1 TYR A 239      13.099  19.483  -0.942  1.00 13.89           C\\nATOM   1863  CE2 TYR A 239      13.652  18.163  -2.918  1.00 13.89           C\\nATOM   1864  CZ  TYR A 239      12.694  18.764  -2.086  1.00 13.89           C\\nATOM   1865  OH  TYR A 239      11.373  18.645  -2.396  1.00 13.89           O\\nATOM   1866  N   GLU A 240      18.779  16.536   0.086  1.00 18.49           N\\nATOM   1867  CA  GLU A 240      20.048  15.823  -0.046  1.00 18.49           C\\nATOM   1868  C   GLU A 240      20.833  16.322  -1.279  1.00 18.49           C\\nATOM   1869  O   GLU A 240      20.208  16.601  -2.304  1.00 18.49           O\\nATOM   1870  CB  GLU A 240      19.748  14.313  -0.197  1.00 18.49           C\\nATOM   1871  CG  GLU A 240      19.383  13.611   1.124  1.00 18.49           C\\nATOM   1872  CD  GLU A 240      20.573  13.547   2.080  1.00 18.49           C\\nATOM   1873  OE1 GLU A 240      21.472  12.719   1.818  1.00 18.49           O\\nATOM   1874  OE2 GLU A 240      20.586  14.350   3.038  1.00 18.49           O\\nATOM   1875  N   PRO A 241      22.179  16.445  -1.221  1.00 21.61           N\\nATOM   1876  CA  PRO A 241      23.003  16.794  -2.389  1.00 21.61           C\\nATOM   1877  C   PRO A 241      22.799  15.785  -3.532  1.00 21.61           C\\nATOM   1878  O   PRO A 241      22.879  14.575  -3.308  1.00 21.61           O\\nATOM   1879  CB  PRO A 241      24.461  16.749  -1.905  1.00 21.61           C\\nATOM   1880  CG  PRO A 241      24.403  16.752  -0.385  1.00 21.61           C\\nATOM   1881  CD  PRO A 241      22.989  16.314  -0.010  1.00 21.61           C\\nATOM   1882  N   LEU A 242      22.515  16.283  -4.742  1.00 21.42           N\\nATOM   1883  CA  LEU A 242      22.347  15.460  -5.930  1.00 21.42           C\\nATOM   1884  C   LEU A 242      23.733  15.274  -6.560  1.00 21.42           C\\nATOM   1885  O   LEU A 242      24.406  16.244  -6.913  1.00 21.42           O\\nATOM   1886  CB  LEU A 242      21.403  16.179  -6.914  1.00 21.42           C\\nATOM   1887  CG  LEU A 242      20.848  15.262  -8.025  1.00 21.42           C\\nATOM   1888  CD1 LEU A 242      19.821  14.248  -7.491  1.00 21.42           C\\nATOM   1889  CD2 LEU A 242      20.207  16.102  -9.129  1.00 21.42           C\\nATOM   1890  N   THR A 243      24.158  14.011  -6.664  1.00 27.38           N\\nATOM   1891  CA  THR A 243      25.442  13.585  -7.214  1.00 27.38           C\\nATOM   1892  C   THR A 243      25.277  13.239  -8.705  1.00 27.38           C\\nATOM   1893  O   THR A 243      24.159  13.178  -9.218  1.00 27.38           O\\nATOM   1894  CB  THR A 243      25.906  12.334  -6.422  1.00 27.38           C\\nATOM   1895  OG1 THR A 243      25.059  11.223  -6.652  1.00 27.38           O\\nATOM   1896  CG2 THR A 243      26.025  12.579  -4.908  1.00 27.38           C\\nATOM   1897  N   GLN A 244      26.397  12.970  -9.395  1.00 28.11           N\\nATOM   1898  CA  GLN A 244      26.387  12.458 -10.765  1.00 28.11           C\\nATOM   1899  C   GLN A 244      25.872  11.007 -10.831  1.00 28.11           C\\nATOM   1900  O   GLN A 244      25.264  10.647 -11.835  1.00 28.11           O\\nATOM   1901  CB  GLN A 244      27.801  12.560 -11.384  1.00 28.11           C\\nATOM   1902  CG  GLN A 244      27.868  12.180 -12.886  1.00 28.11           C\\nATOM   1903  CD  GLN A 244      27.047  13.079 -13.821  1.00 28.11           C\\nATOM   1904  OE1 GLN A 244      26.539  12.615 -14.838  1.00 28.11           O\\nATOM   1905  NE2 GLN A 244      26.881  14.357 -13.475  1.00 28.11           N\\nATOM   1906  N   ASP A 245      26.031  10.203  -9.759  1.00 26.75           N\\nATOM   1907  CA  ASP A 245      25.433   8.866  -9.642  1.00 26.75           C\\nATOM   1908  C   ASP A 245      23.900   8.924  -9.670  1.00 26.75           C\\nATOM   1909  O   ASP A 245      23.289   8.093 -10.334  1.00 26.75           O\\nATOM   1910  CB  ASP A 245      25.876   8.126  -8.354  1.00 26.75           C\\nATOM   1911  CG  ASP A 245      27.357   7.753  -8.262  1.00 26.75           C\\nATOM   1912  OD1 ASP A 245      28.095   7.976  -9.246  1.00 26.75           O\\nATOM   1913  OD2 ASP A 245      27.736   7.254  -7.181  1.00 26.75           O\\nATOM   1914  N   HIS A 246      23.289   9.926  -9.013  1.00 20.76           N\\nATOM   1915  CA  HIS A 246      21.846  10.173  -9.029  1.00 20.76           C\\nATOM   1916  C   HIS A 246      21.340  10.596 -10.417  1.00 20.76           C\\nATOM   1917  O   HIS A 246      20.259  10.165 -10.813  1.00 20.76           O\\nATOM   1918  CB  HIS A 246      21.500  11.270  -8.007  1.00 20.76           C\\nATOM   1919  CG  HIS A 246      21.747  10.913  -6.564  1.00 20.76           C\\nATOM   1920  ND1 HIS A 246      22.067  11.875  -5.601  1.00 20.76           N\\nATOM   1921  CD2 HIS A 246      21.676   9.681  -5.948  1.00 20.76           C\\nATOM   1922  CE1 HIS A 246      22.214  11.197  -4.472  1.00 20.76           C\\nATOM   1923  NE2 HIS A 246      21.976   9.894  -4.615  1.00 20.76           N\\nATOM   1924  N   VAL A 247      22.134  11.382 -11.169  1.00 20.08           N\\nATOM   1925  CA  VAL A 247      21.859  11.770 -12.556  1.00 20.08           C\\nATOM   1926  C   VAL A 247      21.887  10.552 -13.496  1.00 20.08           C\\nATOM   1927  O   VAL A 247      20.994  10.424 -14.334  1.00 20.08           O\\nATOM   1928  CB  VAL A 247      22.849  12.882 -13.011  1.00 20.08           C\\nATOM   1929  CG1 VAL A 247      22.779  13.216 -14.513  1.00 20.08           C\\nATOM   1930  CG2 VAL A 247      22.656  14.177 -12.205  1.00 20.08           C\\nATOM   1931  N   ASP A 248      22.866   9.647 -13.330  1.00 21.75           N\\nATOM   1932  CA  ASP A 248      23.009   8.413 -14.111  1.00 21.75           C\\nATOM   1933  C   ASP A 248      21.874   7.411 -13.842  1.00 21.75           C\\nATOM   1934  O   ASP A 248      21.413   6.758 -14.779  1.00 21.75           O\\nATOM   1935  CB  ASP A 248      24.363   7.724 -13.817  1.00 21.75           C\\nATOM   1936  CG  ASP A 248      25.608   8.556 -14.140  1.00 21.75           C\\nATOM   1937  OD1 ASP A 248      25.512   9.479 -14.977  1.00 21.75           O\\nATOM   1938  OD2 ASP A 248      26.660   8.237 -13.545  1.00 21.75           O\\nATOM   1939  N   ILE A 249      21.395   7.320 -12.588  1.00 19.78           N\\nATOM   1940  CA  ILE A 249      20.264   6.480 -12.178  1.00 19.78           C\\nATOM   1941  C   ILE A 249      18.931   7.020 -12.748  1.00 19.78           C\\nATOM   1942  O   ILE A 249      18.052   6.216 -13.051  1.00 19.78           O\\nATOM   1943  CB  ILE A 249      20.239   6.362 -10.620  1.00 19.78           C\\nATOM   1944  CG1 ILE A 249      21.457   5.569 -10.090  1.00 19.78           C\\nATOM   1945  CG2 ILE A 249      18.962   5.693 -10.076  1.00 19.78           C\\nATOM   1946  CD1 ILE A 249      21.755   5.803  -8.600  1.00 19.78           C\\nATOM   1947  N   LEU A 250      18.788   8.346 -12.933  1.00 13.97           N\\nATOM   1948  CA  LEU A 250      17.620   9.002 -13.538  1.00 13.97           C\\nATOM   1949  C   LEU A 250      17.663   9.014 -15.083  1.00 13.97           C\\nATOM   1950  O   LEU A 250      16.711   9.490 -15.702  1.00 13.97           O\\nATOM   1951  CB  LEU A 250      17.566  10.470 -13.049  1.00 13.97           C\\nATOM   1952  CG  LEU A 250      17.097  10.645 -11.590  1.00 13.97           C\\nATOM   1953  CD1 LEU A 250      17.427  12.057 -11.082  1.00 13.97           C\\nATOM   1954  CD2 LEU A 250      15.593  10.370 -11.431  1.00 13.97           C\\nATOM   1955  N   GLY A 251      18.729   8.477 -15.702  1.00 21.12           N\\nATOM   1956  CA  GLY A 251      18.956   8.400 -17.146  1.00 21.12           C\\nATOM   1957  C   GLY A 251      17.836   7.733 -17.964  1.00 21.12           C\\nATOM   1958  O   GLY A 251      17.359   8.361 -18.906  1.00 21.12           O\\nATOM   1959  N   PRO A 252      17.390   6.499 -17.636  1.00 23.69           N\\nATOM   1960  CA  PRO A 252      16.229   5.840 -18.261  1.00 23.69           C\\nATOM   1961  C   PRO A 252      14.913   6.649 -18.264  1.00 23.69           C\\nATOM   1962  O   PRO A 252      14.208   6.635 -19.272  1.00 23.69           O\\nATOM   1963  CB  PRO A 252      16.043   4.534 -17.478  1.00 23.69           C\\nATOM   1964  CG  PRO A 252      17.421   4.225 -16.919  1.00 23.69           C\\nATOM   1965  CD  PRO A 252      18.091   5.585 -16.735  1.00 23.69           C\\nATOM   1966  N   LEU A 253      14.594   7.377 -17.178  1.00 14.33           N\\nATOM   1967  CA  LEU A 253      13.398   8.226 -17.084  1.00 14.33           C\\nATOM   1968  C   LEU A 253      13.557   9.538 -17.868  1.00 14.33           C\\nATOM   1969  O   LEU A 253      12.582  10.024 -18.441  1.00 14.33           O\\nATOM   1970  CB  LEU A 253      13.123   8.575 -15.605  1.00 14.33           C\\nATOM   1971  CG  LEU A 253      12.558   7.406 -14.773  1.00 14.33           C\\nATOM   1972  CD1 LEU A 253      12.621   7.723 -13.271  1.00 14.33           C\\nATOM   1973  CD2 LEU A 253      11.125   7.031 -15.181  1.00 14.33           C\\nATOM   1974  N   SER A 254      14.787  10.071 -17.941  1.00 11.96           N\\nATOM   1975  CA  SER A 254      15.167  11.210 -18.771  1.00 11.96           C\\nATOM   1976  C   SER A 254      15.066  10.881 -20.273  1.00 11.96           C\\nATOM   1977  O   SER A 254      14.618  11.725 -21.047  1.00 11.96           O\\nATOM   1978  CB  SER A 254      16.594  11.636 -18.367  1.00 11.96           C\\nATOM   1979  OG  SER A 254      17.085  12.697 -19.154  1.00 11.96           O\\nATOM   1980  N   ALA A 255      15.421   9.645 -20.665  1.00 17.87           N\\nATOM   1981  CA  ALA A 255      15.352   9.134 -22.031  1.00 17.87           C\\nATOM   1982  C   ALA A 255      13.908   8.836 -22.471  1.00 17.87           C\\nATOM   1983  O   ALA A 255      13.584   9.043 -23.640  1.00 17.87           O\\nATOM   1984  CB  ALA A 255      16.165   7.831 -22.101  1.00 17.87           C\\nATOM   1985  N   GLN A 256      13.043   8.383 -21.544  1.00 25.07           N\\nATOM   1986  CA  GLN A 256      11.637   8.055 -21.793  1.00 25.07           C\\nATOM   1987  C   GLN A 256      10.787   9.314 -22.052  1.00 25.07           C\\nATOM   1988  O   GLN A 256       9.946   9.301 -22.951  1.00 25.07           O\\nATOM   1989  CB  GLN A 256      11.085   7.279 -20.572  1.00 25.07           C\\nATOM   1990  CG  GLN A 256       9.608   6.838 -20.706  1.00 25.07           C\\nATOM   1991  CD  GLN A 256       9.085   6.032 -19.513  1.00 25.07           C\\nATOM   1992  OE1 GLN A 256       9.825   5.692 -18.592  1.00 25.07           O\\nATOM   1993  NE2 GLN A 256       7.786   5.729 -19.526  1.00 25.07           N\\nATOM   1994  N   THR A 257      11.005  10.386 -21.273  1.00 18.15           N\\nATOM   1995  CA  THR A 257      10.247  11.640 -21.352  1.00 18.15           C\\nATOM   1996  C   THR A 257      10.902  12.680 -22.281  1.00 18.15           C\\nATOM   1997  O   THR A 257      10.231  13.631 -22.680  1.00 18.15           O\\nATOM   1998  CB  THR A 257      10.133  12.247 -19.931  1.00 18.15           C\\nATOM   1999  OG1 THR A 257      11.400  12.565 -19.386  1.00 18.15           O\\nATOM   2000  CG2 THR A 257       9.344  11.360 -18.956  1.00 18.15           C\\nATOM   2001  N   GLY A 258      12.196  12.526 -22.609  1.00 16.66           N\\nATOM   2002  CA  GLY A 258      12.983  13.474 -23.396  1.00 16.66           C\\nATOM   2003  C   GLY A 258      13.422  14.739 -22.639  1.00 16.66           C\\nATOM   2004  O   GLY A 258      14.008  15.627 -23.255  1.00 16.66           O\\nATOM   2005  N   ILE A 259      13.145  14.831 -21.327  1.00 15.50           N\\nATOM   2006  CA  ILE A 259      13.531  15.945 -20.463  1.00 15.50           C\\nATOM   2007  C   ILE A 259      14.850  15.539 -19.793  1.00 15.50           C\\nATOM   2008  O   ILE A 259      14.898  14.528 -19.092  1.00 15.50           O\\nATOM   2009  CB  ILE A 259      12.433  16.176 -19.381  1.00 15.50           C\\nATOM   2010  CG1 ILE A 259      11.068  16.515 -20.020  1.00 15.50           C\\nATOM   2011  CG2 ILE A 259      12.831  17.269 -18.359  1.00 15.50           C\\nATOM   2012  CD1 ILE A 259       9.894  16.366 -19.049  1.00 15.50           C\\nATOM   2013  N   ALA A 260      15.906  16.348 -19.983  1.00 16.52           N\\nATOM   2014  CA  ALA A 260      17.215  16.193 -19.346  1.00 16.52           C\\nATOM   2015  C   ALA A 260      17.101  16.326 -17.819  1.00 16.52           C\\nATOM   2016  O   ALA A 260      16.255  17.077 -17.331  1.00 16.52           O\\nATOM   2017  CB  ALA A 260      18.173  17.259 -19.896  1.00 16.52           C\\nATOM   2018  N   VAL A 261      17.931  15.588 -17.062  1.00 16.13           N\\nATOM   2019  CA  VAL A 261      17.928  15.574 -15.592  1.00 16.13           C\\nATOM   2020  C   VAL A 261      18.124  16.970 -14.986  1.00 16.13           C\\nATOM   2021  O   VAL A 261      17.411  17.322 -14.051  1.00 16.13           O\\nATOM   2022  CB  VAL A 261      18.939  14.542 -15.034  1.00 16.13           C\\nATOM   2023  CG1 VAL A 261      18.956  14.541 -13.492  1.00 16.13           C\\nATOM   2024  CG2 VAL A 261      18.645  13.118 -15.540  1.00 16.13           C\\nATOM   2025  N   LEU A 262      19.031  17.775 -15.557  1.00 19.24           N\\nATOM   2026  CA  LEU A 262      19.297  19.145 -15.137  1.00 19.24           C\\nATOM   2027  C   LEU A 262      18.195  20.146 -15.537  1.00 19.24           C\\nATOM   2028  O   LEU A 262      18.118  21.209 -14.926  1.00 19.24           O\\nATOM   2029  CB  LEU A 262      20.642  19.593 -15.729  1.00 19.24           C\\nATOM   2030  CG  LEU A 262      21.884  18.942 -15.076  1.00 19.24           C\\nATOM   2031  CD1 LEU A 262      23.157  19.400 -15.799  1.00 19.24           C\\nATOM   2032  CD2 LEU A 262      21.999  19.228 -13.568  1.00 19.24           C\\nATOM   2033  N   ASP A 263      17.307  19.815 -16.491  1.00 18.33           N\\nATOM   2034  CA  ASP A 263      16.095  20.597 -16.773  1.00 18.33           C\\nATOM   2035  C   ASP A 263      15.006  20.275 -15.736  1.00 18.33           C\\nATOM   2036  O   ASP A 263      14.304  21.186 -15.301  1.00 18.33           O\\nATOM   2037  CB  ASP A 263      15.530  20.294 -18.184  1.00 18.33           C\\nATOM   2038  CG  ASP A 263      16.432  20.679 -19.363  1.00 18.33           C\\nATOM   2039  OD1 ASP A 263      17.592  21.082 -19.129  1.00 18.33           O\\nATOM   2040  OD2 ASP A 263      15.937  20.549 -20.503  1.00 18.33           O\\nATOM   2041  N   MET A 264      14.904  19.009 -15.287  1.00 12.95           N\\nATOM   2042  CA  MET A 264      13.992  18.585 -14.220  1.00 12.95           C\\nATOM   2043  C   MET A 264      14.464  19.074 -12.834  1.00 12.95           C\\nATOM   2044  O   MET A 264      13.628  19.321 -11.967  1.00 12.95           O\\nATOM   2045  CB  MET A 264      13.873  17.044 -14.228  1.00 12.95           C\\nATOM   2046  CG  MET A 264      12.691  16.514 -13.392  1.00 12.95           C\\nATOM   2047  SD  MET A 264      11.055  17.029 -13.984  1.00 12.95           S\\nATOM   2048  CE  MET A 264      10.759  15.743 -15.217  1.00 12.95           C\\nATOM   2049  N   CYS A 265      15.780  19.280 -12.642  1.00 14.73           N\\nATOM   2050  CA  CYS A 265      16.369  19.906 -11.455  1.00 14.73           C\\nATOM   2051  C   CYS A 265      16.027  21.397 -11.387  1.00 14.73           C\\nATOM   2052  O   CYS A 265      15.817  21.917 -10.296  1.00 14.73           O\\nATOM   2053  CB  CYS A 265      17.901  19.774 -11.483  1.00 14.73           C\\nATOM   2054  SG  CYS A 265      18.401  18.065 -11.178  1.00 14.73           S\\nATOM   2055  N   ALA A 266      15.923  22.072 -12.543  1.00 19.80           N\\nATOM   2056  CA  ALA A 266      15.522  23.470 -12.632  1.00 19.80           C\\nATOM   2057  C   ALA A 266      14.015  23.637 -12.388  1.00 19.80           C\\nATOM   2058  O   ALA A 266      13.616  24.641 -11.804  1.00 19.80           O\\nATOM   2059  CB  ALA A 266      15.864  23.989 -14.024  1.00 19.80           C\\nATOM   2060  N   SER A 267      13.191  22.642 -12.766  1.00 16.52           N\\nATOM   2061  CA  SER A 267      11.767  22.560 -12.429  1.00 16.52           C\\nATOM   2062  C   SER A 267      11.540  22.349 -10.930  1.00 16.52           C\\nATOM   2063  O   SER A 267      10.679  23.012 -10.354  1.00 16.52           O\\nATOM   2064  CB  SER A 267      11.096  21.430 -13.223  1.00 16.52           C\\nATOM   2065  OG  SER A 267      10.741  21.914 -14.495  1.00 16.52           O\\nATOM   2066  N   LEU A 268      12.348  21.483 -10.296  1.00 10.55           N\\nATOM   2067  CA  LEU A 268      12.358  21.256  -8.854  1.00 10.55           C\\nATOM   2068  C   LEU A 268      12.841  22.502  -8.091  1.00 10.55           C\\nATOM   2069  O   LEU A 268      12.213  22.873  -7.106  1.00 10.55           O\\nATOM   2070  CB  LEU A 268      13.225  20.014  -8.550  1.00 10.55           C\\nATOM   2071  CG  LEU A 268      13.388  19.639  -7.059  1.00 10.55           C\\nATOM   2072  CD1 LEU A 268      12.051  19.569  -6.305  1.00 10.55           C\\nATOM   2073  CD2 LEU A 268      14.141  18.307  -6.926  1.00 10.55           C\\nATOM   2074  N   LYS A 269      13.891  23.188  -8.573  1.00 14.03           N\\nATOM   2075  CA  LYS A 269      14.404  24.458  -8.048  1.00 14.03           C\\nATOM   2076  C   LYS A 269      13.326  25.556  -7.969  1.00 14.03           C\\nATOM   2077  O   LYS A 269      13.211  26.206  -6.932  1.00 14.03           O\\nATOM   2078  CB  LYS A 269      15.633  24.887  -8.879  1.00 14.03           C\\nATOM   2079  CG  LYS A 269      16.106  26.337  -8.683  1.00 14.03           C\\nATOM   2080  CD  LYS A 269      17.393  26.658  -9.453  1.00 14.03           C\\nATOM   2081  CE  LYS A 269      17.718  28.155  -9.387  1.00 14.03           C\\nATOM   2082  NZ  LYS A 269      18.826  28.520 -10.283  1.00 14.03           N\\nATOM   2083  N   GLU A 270      12.518  25.734  -9.029  1.00 16.21           N\\nATOM   2084  CA  GLU A 270      11.428  26.711  -9.054  1.00 16.21           C\\nATOM   2085  C   GLU A 270      10.224  26.260  -8.208  1.00 16.21           C\\nATOM   2086  O   GLU A 270       9.554  27.121  -7.647  1.00 16.21           O\\nATOM   2087  CB  GLU A 270      10.941  26.927 -10.500  1.00 16.21           C\\nATOM   2088  CG  GLU A 270      11.932  27.624 -11.446  1.00 16.21           C\\nATOM   2089  CD  GLU A 270      12.477  28.961 -10.956  1.00 16.21           C\\nATOM   2090  OE1 GLU A 270      11.646  29.874 -10.762  1.00 16.21           O\\nATOM   2091  OE2 GLU A 270      13.716  29.049 -10.805  1.00 16.21           O\\nATOM   2092  N   LEU A 271       9.966  24.947  -8.055  1.00 14.11           N\\nATOM   2093  CA  LEU A 271       8.927  24.423  -7.157  1.00 14.11           C\\nATOM   2094  C   LEU A 271       9.287  24.624  -5.675  1.00 14.11           C\\nATOM   2095  O   LEU A 271       8.385  24.803  -4.860  1.00 14.11           O\\nATOM   2096  CB  LEU A 271       8.704  22.912  -7.399  1.00 14.11           C\\nATOM   2097  CG  LEU A 271       7.876  22.558  -8.650  1.00 14.11           C\\nATOM   2098  CD1 LEU A 271       8.017  21.060  -8.973  1.00 14.11           C\\nATOM   2099  CD2 LEU A 271       6.395  22.936  -8.491  1.00 14.11           C\\nATOM   2100  N   LEU A 272      10.584  24.639  -5.327  1.00 13.94           N\\nATOM   2101  CA  LEU A 272      11.067  24.925  -3.979  1.00 13.94           C\\nATOM   2102  C   LEU A 272      11.032  26.431  -3.688  1.00 13.94           C\\nATOM   2103  O   LEU A 272      10.637  26.825  -2.592  1.00 13.94           O\\nATOM   2104  CB  LEU A 272      12.526  24.432  -3.838  1.00 13.94           C\\nATOM   2105  CG  LEU A 272      12.689  22.901  -3.853  1.00 13.94           C\\nATOM   2106  CD1 LEU A 272      14.169  22.495  -3.960  1.00 13.94           C\\nATOM   2107  CD2 LEU A 272      12.032  22.240  -2.642  1.00 13.94           C\\nATOM   2108  N   GLN A 273      11.412  27.269  -4.664  1.00 14.78           N\\nATOM   2109  CA  GLN A 273      11.509  28.719  -4.517  1.00 14.78           C\\nATOM   2110  C   GLN A 273      10.141  29.423  -4.584  1.00 14.78           C\\nATOM   2111  O   GLN A 273       9.936  30.384  -3.843  1.00 14.78           O\\nATOM   2112  CB  GLN A 273      12.433  29.255  -5.628  1.00 14.78           C\\nATOM   2113  CG  GLN A 273      13.917  28.926  -5.341  1.00 14.78           C\\nATOM   2114  CD  GLN A 273      14.914  29.284  -6.451  1.00 14.78           C\\nATOM   2115  OE1 GLN A 273      16.117  29.127  -6.254  1.00 14.78           O\\nATOM   2116  NE2 GLN A 273      14.454  29.768  -7.605  1.00 14.78           N\\nATOM   2117  N   ASN A 274       9.207  28.953  -5.428  0.24 23.10           N\\nATOM   2118  CA  ASN A 274       7.885  29.566  -5.626  0.24 23.10           C\\nATOM   2119  C   ASN A 274       6.765  28.804  -4.897  0.24 23.10           C\\nATOM   2120  O   ASN A 274       5.670  29.348  -4.749  0.24 23.10           O\\nATOM   2121  CB  ASN A 274       7.525  29.579  -7.134  0.24 23.10           C\\nATOM   2122  CG  ASN A 274       8.546  30.318  -8.008  0.24 23.10           C\\nATOM   2123  OD1 ASN A 274       9.031  31.384  -7.638  0.24 23.10           O\\nATOM   2124  ND2 ASN A 274       8.912  29.732  -9.150  0.24 23.10           N\\nATOM   2125  N   GLY A 275       7.011  27.556  -4.462  1.00 19.76           N\\nATOM   2126  CA  GLY A 275       5.998  26.653  -3.920  1.00 19.76           C\\nATOM   2127  C   GLY A 275       5.122  26.045  -5.024  1.00 19.76           C\\nATOM   2128  O   GLY A 275       5.464  26.104  -6.207  1.00 19.76           O\\nATOM   2129  N   MET A 276       3.991  25.448  -4.626  1.00 19.59           N\\nATOM   2130  CA  MET A 276       3.040  24.790  -5.523  1.00 19.59           C\\nATOM   2131  C   MET A 276       1.936  25.758  -5.995  1.00 19.59           C\\nATOM   2132  O   MET A 276       1.369  25.541  -7.063  1.00 19.59           O\\nATOM   2133  CB  MET A 276       2.356  23.639  -4.755  1.00 19.59           C\\nATOM   2134  CG  MET A 276       3.320  22.535  -4.278  1.00 19.59           C\\nATOM   2135  SD  MET A 276       3.966  21.422  -5.558  1.00 19.59           S\\nATOM   2136  CE  MET A 276       2.565  20.288  -5.731  1.00 19.59           C\\nATOM   2137  N   ASN A 277       1.622  26.804  -5.206  1.00 18.55           N\\nATOM   2138  CA  ASN A 277       0.612  27.844  -5.462  1.00 18.55           C\\nATOM   2139  C   ASN A 277      -0.833  27.315  -5.391  1.00 18.55           C\\nATOM   2140  O   ASN A 277      -1.706  27.787  -6.116  1.00 18.55           O\\nATOM   2141  CB  ASN A 277       0.915  28.688  -6.731  1.00 18.55           C\\nATOM   2142  CG  ASN A 277       2.143  29.586  -6.560  1.00 18.55           C\\nATOM   2143  OD1 ASN A 277       2.070  30.606  -5.880  1.00 18.55           O\\nATOM   2144  ND2 ASN A 277       3.274  29.213  -7.161  1.00 18.55           N\\nATOM   2145  N   GLY A 278      -1.081  26.313  -4.532  1.00 17.96           N\\nATOM   2146  CA  GLY A 278      -2.373  25.653  -4.360  1.00 17.96           C\\nATOM   2147  C   GLY A 278      -2.657  24.536  -5.375  1.00 17.96           C\\nATOM   2148  O   GLY A 278      -3.723  23.926  -5.309  1.00 17.96           O\\nATOM   2149  N   ARG A 279      -1.722  24.258  -6.297  1.00 18.10           N\\nATOM   2150  CA  ARG A 279      -1.806  23.184  -7.288  1.00 18.10           C\\nATOM   2151  C   ARG A 279      -1.241  21.878  -6.703  1.00 18.10           C\\nATOM   2152  O   ARG A 279      -0.632  21.872  -5.632  1.00 18.10           O\\nATOM   2153  CB  ARG A 279      -0.969  23.590  -8.519  1.00 18.10           C\\nATOM   2154  CG  ARG A 279      -1.510  24.857  -9.207  1.00 18.10           C\\nATOM   2155  CD  ARG A 279      -0.755  25.223 -10.489  1.00 18.10           C\\nATOM   2156  NE  ARG A 279       0.545  25.836 -10.184  1.00 18.10           N\\nATOM   2157  CZ  ARG A 279       0.872  27.136 -10.278  1.00 18.10           C\\nATOM   2158  NH1 ARG A 279       2.131  27.512 -10.035  1.00 18.10           N\\nATOM   2159  NH2 ARG A 279      -0.022  28.076 -10.611  1.00 18.10           N\\nATOM   2160  N   THR A 280      -1.465  20.765  -7.416  1.00 16.67           N\\nATOM   2161  CA  THR A 280      -1.053  19.415  -7.037  1.00 16.67           C\\nATOM   2162  C   THR A 280      -0.289  18.742  -8.183  1.00 16.67           C\\nATOM   2163  O   THR A 280      -0.524  19.028  -9.357  1.00 16.67           O\\nATOM   2164  CB  THR A 280      -2.261  18.581  -6.538  1.00 16.67           C\\nATOM   2165  OG1 THR A 280      -3.197  18.346  -7.574  1.00 16.67           O\\nATOM   2166  CG2 THR A 280      -2.981  19.181  -5.321  1.00 16.67           C\\nATOM   2167  N   ILE A 281       0.609  17.820  -7.817  1.00 14.98           N\\nATOM   2168  CA  ILE A 281       1.335  16.922  -8.709  1.00 14.98           C\\nATOM   2169  C   ILE A 281       1.034  15.513  -8.190  1.00 14.98           C\\nATOM   2170  O   ILE A 281       1.317  15.222  -7.029  1.00 14.98           O\\nATOM   2171  CB  ILE A 281       2.851  17.262  -8.703  1.00 14.98           C\\nATOM   2172  CG1 ILE A 281       3.134  18.698  -9.197  1.00 14.98           C\\nATOM   2173  CG2 ILE A 281       3.661  16.246  -9.530  1.00 14.98           C\\nATOM   2174  CD1 ILE A 281       4.570  19.186  -8.947  1.00 14.98           C\\nATOM   2175  N   LEU A 282       0.421  14.662  -9.029  1.00 15.26           N\\nATOM   2176  CA  LEU A 282       0.009  13.279  -8.754  1.00 15.26           C\\nATOM   2177  C   LEU A 282      -0.883  13.158  -7.500  1.00 15.26           C\\nATOM   2178  O   LEU A 282      -0.735  12.227  -6.706  1.00 15.26           O\\nATOM   2179  CB  LEU A 282       1.240  12.341  -8.713  1.00 15.26           C\\nATOM   2180  CG  LEU A 282       1.936  12.135 -10.073  1.00 15.26           C\\nATOM   2181  CD1 LEU A 282       3.237  11.347  -9.873  1.00 15.26           C\\nATOM   2182  CD2 LEU A 282       1.021  11.434 -11.093  1.00 15.26           C\\nATOM   2183  N   GLY A 283      -1.793  14.128  -7.312  1.00 19.60           N\\nATOM   2184  CA  GLY A 283      -2.720  14.209  -6.188  1.00 19.60           C\\nATOM   2185  C   GLY A 283      -2.103  14.670  -4.858  1.00 19.60           C\\nATOM   2186  O   GLY A 283      -2.782  14.580  -3.835  1.00 19.60           O\\nATOM   2187  N   SER A 284      -0.847  15.150  -4.851  1.00 16.38           N\\nATOM   2188  CA  SER A 284      -0.152  15.627  -3.658  1.00 16.38           C\\nATOM   2189  C   SER A 284       0.164  17.122  -3.779  1.00 16.38           C\\nATOM   2190  O   SER A 284       0.655  17.585  -4.808  1.00 16.38           O\\nATOM   2191  CB  SER A 284       1.122  14.796  -3.425  1.00 16.38           C\\nATOM   2192  OG  SER A 284       1.736  15.181  -2.210  1.00 16.38           O\\nATOM   2193  N   ALA A 285      -0.083  17.847  -2.678  1.00 20.92           N\\nATOM   2194  CA  ALA A 285       0.254  19.251  -2.458  1.00 20.92           C\\nATOM   2195  C   ALA A 285       1.675  19.422  -1.878  1.00 20.92           C\\nATOM   2196  O   ALA A 285       2.039  20.527  -1.479  1.00 20.92           O\\nATOM   2197  CB  ALA A 285      -0.756  19.805  -1.437  1.00 20.92           C\\nATOM   2198  N   LEU A 286       2.489  18.355  -1.854  1.00 19.64           N\\nATOM   2199  CA  LEU A 286       3.899  18.364  -1.482  1.00 19.64           C\\nATOM   2200  C   LEU A 286       4.710  17.500  -2.448  1.00 19.64           C\\nATOM   2201  O   LEU A 286       4.162  16.685  -3.192  1.00 19.64           O\\nATOM   2202  CB  LEU A 286       4.102  18.097   0.032  1.00 19.64           C\\nATOM   2203  CG  LEU A 286       3.737  16.698   0.582  1.00 19.64           C\\nATOM   2204  CD1 LEU A 286       4.802  15.625   0.286  1.00 19.64           C\\nATOM   2205  CD2 LEU A 286       3.513  16.781   2.100  1.00 19.64           C\\nATOM   2206  N   LEU A 287       6.032  17.697  -2.431  1.00 14.07           N\\nATOM   2207  CA  LEU A 287       6.982  17.015  -3.294  1.00 14.07           C\\nATOM   2208  C   LEU A 287       7.369  15.687  -2.621  1.00 14.07           C\\nATOM   2209  O   LEU A 287       8.117  15.668  -1.642  1.00 14.07           O\\nATOM   2210  CB  LEU A 287       8.204  17.943  -3.478  1.00 14.07           C\\nATOM   2211  CG  LEU A 287       7.864  19.330  -4.089  1.00 14.07           C\\nATOM   2212  CD1 LEU A 287       9.090  20.251  -4.093  1.00 14.07           C\\nATOM   2213  CD2 LEU A 287       7.259  19.239  -5.499  1.00 14.07           C\\nATOM   2214  N   GLU A 288       6.800  14.584  -3.137  1.00 13.13           N\\nATOM   2215  CA  GLU A 288       6.912  13.222  -2.608  1.00 13.13           C\\nATOM   2216  C   GLU A 288       8.313  12.666  -2.884  1.00 13.13           C\\nATOM   2217  O   GLU A 288       8.845  12.868  -3.975  1.00 13.13           O\\nATOM   2218  CB  GLU A 288       5.901  12.323  -3.349  1.00 13.13           C\\nATOM   2219  CG  GLU A 288       4.418  12.681  -3.123  1.00 13.13           C\\nATOM   2220  CD  GLU A 288       3.902  12.300  -1.737  1.00 13.13           C\\nATOM   2221  OE1 GLU A 288       4.121  11.138  -1.336  1.00 13.13           O\\nATOM   2222  OE2 GLU A 288       3.267  13.168  -1.103  1.00 13.13           O\\nATOM   2223  N   ASP A 289       8.889  11.952  -1.904  1.00 11.67           N\\nATOM   2224  CA  ASP A 289      10.288  11.514  -1.891  1.00 11.67           C\\nATOM   2225  C   ASP A 289      10.461  10.039  -1.483  1.00 11.67           C\\nATOM   2226  O   ASP A 289      11.579   9.635  -1.169  1.00 11.67           O\\nATOM   2227  CB  ASP A 289      11.132  12.519  -1.077  1.00 11.67           C\\nATOM   2228  CG  ASP A 289      11.101  12.351   0.444  1.00 11.67           C\\nATOM   2229  OD1 ASP A 289      10.057  11.933   0.988  1.00 11.67           O\\nATOM   2230  OD2 ASP A 289      12.153  12.631   1.052  1.00 11.67           O\\nATOM   2231  N   GLU A 290       9.391   9.231  -1.512  1.00 13.47           N\\nATOM   2232  CA  GLU A 290       9.405   7.803  -1.187  1.00 13.47           C\\nATOM   2233  C   GLU A 290       8.974   6.958  -2.395  1.00 13.47           C\\nATOM   2234  O   GLU A 290       8.320   5.930  -2.233  1.00 13.47           O\\nATOM   2235  CB  GLU A 290       8.639   7.503   0.124  1.00 13.47           C\\nATOM   2236  CG  GLU A 290       9.094   8.377   1.316  1.00 13.47           C\\nATOM   2237  CD  GLU A 290       8.784   7.796   2.700  1.00 13.47           C\\nATOM   2238  OE1 GLU A 290       7.918   6.902   2.799  1.00 13.47           O\\nATOM   2239  OE2 GLU A 290       9.443   8.249   3.661  1.00 13.47           O\\nATOM   2240  N   PHE A 291       9.400   7.381  -3.597  1.00 10.30           N\\nATOM   2241  CA  PHE A 291       9.485   6.598  -4.826  1.00 10.30           C\\nATOM   2242  C   PHE A 291      10.905   6.760  -5.366  1.00 10.30           C\\nATOM   2243  O   PHE A 291      11.273   7.865  -5.762  1.00 10.30           O\\nATOM   2244  CB  PHE A 291       8.507   7.118  -5.897  1.00 10.30           C\\nATOM   2245  CG  PHE A 291       7.042   7.062  -5.549  1.00 10.30           C\\nATOM   2246  CD1 PHE A 291       6.335   5.861  -5.735  1.00 10.30           C\\nATOM   2247  CD2 PHE A 291       6.368   8.222  -5.114  1.00 10.30           C\\nATOM   2248  CE1 PHE A 291       4.946   5.836  -5.529  1.00 10.30           C\\nATOM   2249  CE2 PHE A 291       4.978   8.193  -4.906  1.00 10.30           C\\nATOM   2250  CZ  PHE A 291       4.265   7.000  -5.123  1.00 10.30           C\\nATOM   2251  N   THR A 292      11.688   5.675  -5.445  1.00  9.64           N\\nATOM   2252  CA  THR A 292      12.990   5.679  -6.116  1.00  9.64           C\\nATOM   2253  C   THR A 292      12.792   5.717  -7.650  1.00  9.64           C\\nATOM   2254  O   THR A 292      11.730   5.319  -8.133  1.00  9.64           O\\nATOM   2255  CB  THR A 292      13.809   4.420  -5.716  1.00  9.64           C\\nATOM   2256  OG1 THR A 292      13.295   3.234  -6.286  1.00  9.64           O\\nATOM   2257  CG2 THR A 292      13.978   4.232  -4.207  1.00  9.64           C\\nATOM   2258  N   PRO A 293      13.809   6.135  -8.432  1.00 10.12           N\\nATOM   2259  CA  PRO A 293      13.893   5.910  -9.886  1.00 10.12           C\\nATOM   2260  C   PRO A 293      13.533   4.489 -10.375  1.00 10.12           C\\nATOM   2261  O   PRO A 293      12.867   4.367 -11.401  1.00 10.12           O\\nATOM   2262  CB  PRO A 293      15.322   6.307 -10.271  1.00 10.12           C\\nATOM   2263  CG  PRO A 293      15.770   7.274  -9.180  1.00 10.12           C\\nATOM   2264  CD  PRO A 293      14.944   6.923  -7.948  1.00 10.12           C\\nATOM   2265  N   PHE A 294      13.900   3.434  -9.621  1.00 11.40           N\\nATOM   2266  CA  PHE A 294      13.542   2.043  -9.917  1.00 11.40           C\\nATOM   2267  C   PHE A 294      12.059   1.740  -9.639  1.00 11.40           C\\nATOM   2268  O   PHE A 294      11.476   0.973 -10.399  1.00 11.40           O\\nATOM   2269  CB  PHE A 294      14.414   1.070  -9.089  1.00 11.40           C\\nATOM   2270  CG  PHE A 294      15.879   1.018  -9.493  1.00 11.40           C\\nATOM   2271  CD1 PHE A 294      16.244   0.757 -10.832  1.00 11.40           C\\nATOM   2272  CD2 PHE A 294      16.888   1.243  -8.533  1.00 11.40           C\\nATOM   2273  CE1 PHE A 294      17.599   0.780 -11.216  1.00 11.40           C\\nATOM   2274  CE2 PHE A 294      18.241   1.291  -8.921  1.00 11.40           C\\nATOM   2275  CZ  PHE A 294      18.597   1.066 -10.264  1.00 11.40           C\\nATOM   2276  N   ASP A 295      11.427   2.358  -8.622  1.00 11.59           N\\nATOM   2277  CA  ASP A 295       9.990   2.222  -8.336  1.00 11.59           C\\nATOM   2278  C   ASP A 295       9.118   2.871  -9.411  1.00 11.59           C\\nATOM   2279  O   ASP A 295       8.076   2.317  -9.761  1.00 11.59           O\\nATOM   2280  CB  ASP A 295       9.599   2.896  -7.004  1.00 11.59           C\\nATOM   2281  CG  ASP A 295      10.271   2.353  -5.757  1.00 11.59           C\\nATOM   2282  OD1 ASP A 295      10.650   1.162  -5.737  1.00 11.59           O\\nATOM   2283  OD2 ASP A 295      10.362   3.159  -4.810  1.00 11.59           O\\nATOM   2284  N   VAL A 296       9.544   4.032  -9.934  1.00 11.32           N\\nATOM   2285  CA  VAL A 296       8.849   4.761 -10.989  1.00 11.32           C\\nATOM   2286  C   VAL A 296       8.894   3.970 -12.299  1.00 11.32           C\\nATOM   2287  O   VAL A 296       7.844   3.782 -12.903  1.00 11.32           O\\nATOM   2288  CB  VAL A 296       9.440   6.182 -11.139  1.00 11.32           C\\nATOM   2289  CG1 VAL A 296       8.842   6.941 -12.342  1.00 11.32           C\\nATOM   2290  CG2 VAL A 296       9.224   6.992  -9.848  1.00 11.32           C\\nATOM   2291  N   VAL A 297      10.072   3.453 -12.690  1.00 12.99           N\\nATOM   2292  CA  VAL A 297      10.244   2.542 -13.824  1.00 12.99           C\\nATOM   2293  C   VAL A 297       9.402   1.261 -13.666  1.00 12.99           C\\nATOM   2294  O   VAL A 297       8.770   0.851 -14.636  1.00 12.99           O\\nATOM   2295  CB  VAL A 297      11.757   2.257 -14.039  1.00 12.99           C\\nATOM   2296  CG1 VAL A 297      12.057   1.070 -14.978  1.00 12.99           C\\nATOM   2297  CG2 VAL A 297      12.474   3.511 -14.574  1.00 12.99           C\\nATOM   2298  N   ARG A 298       9.355   0.662 -12.460  1.00 12.33           N\\nATOM   2299  CA  ARG A 298       8.642  -0.586 -12.178  1.00 12.33           C\\nATOM   2300  C   ARG A 298       7.121  -0.432 -12.326  1.00 12.33           C\\nATOM   2301  O   ARG A 298       6.496  -1.273 -12.969  1.00 12.33           O\\nATOM   2302  CB  ARG A 298       8.975  -1.083 -10.753  1.00 12.33           C\\nATOM   2303  CG  ARG A 298       8.575  -2.551 -10.513  1.00 12.33           C\\nATOM   2304  CD  ARG A 298       8.732  -3.029  -9.067  1.00 12.33           C\\nATOM   2305  NE  ARG A 298      10.122  -2.865  -8.624  1.00 12.33           N\\nATOM   2306  CZ  ARG A 298      10.566  -1.955  -7.742  1.00 12.33           C\\nATOM   2307  NH1 ARG A 298      11.881  -1.830  -7.544  1.00 12.33           N\\nATOM   2308  NH2 ARG A 298       9.732  -1.151  -7.072  1.00 12.33           N\\nATOM   2309  N   GLN A 299       6.535   0.636 -11.755  1.00 14.69           N\\nATOM   2310  CA  GLN A 299       5.099   0.905 -11.805  1.00 14.69           C\\nATOM   2311  C   GLN A 299       4.658   1.444 -13.179  1.00 14.69           C\\nATOM   2312  O   GLN A 299       3.574   1.081 -13.635  1.00 14.69           O\\nATOM   2313  CB  GLN A 299       4.730   1.895 -10.677  1.00 14.69           C\\nATOM   2314  CG  GLN A 299       3.236   2.304 -10.662  1.00 14.69           C\\nATOM   2315  CD  GLN A 299       2.746   3.044  -9.415  1.00 14.69           C\\nATOM   2316  OE1 GLN A 299       1.565   3.370  -9.336  1.00 14.69           O\\nATOM   2317  NE2 GLN A 299       3.607   3.317  -8.436  1.00 14.69           N\\nATOM   2318  N   CYS A 300       5.486   2.258 -13.859  1.00 16.70           N\\nATOM   2319  CA  CYS A 300       5.185   2.806 -15.188  1.00 16.70           C\\nATOM   2320  C   CYS A 300       5.386   1.782 -16.326  1.00 16.70           C\\nATOM   2321  O   CYS A 300       4.904   2.036 -17.429  1.00 16.70           O\\nATOM   2322  CB  CYS A 300       6.073   4.038 -15.474  1.00 16.70           C\\nATOM   2323  SG  CYS A 300       5.538   5.463 -14.475  1.00 16.70           S\\nATOM   2324  N   SER A 301       6.038   0.630 -16.071  1.00 19.23           N\\nATOM   2325  CA  SER A 301       6.217  -0.470 -17.031  1.00 19.23           C\\nATOM   2326  C   SER A 301       5.471  -1.755 -16.623  1.00 19.23           C\\nATOM   2327  O   SER A 301       5.361  -2.664 -17.444  1.00 19.23           O\\nATOM   2328  CB  SER A 301       7.721  -0.719 -17.285  1.00 19.23           C\\nATOM   2329  OG  SER A 301       8.356  -1.403 -16.225  1.00 19.23           O\\nATOM   2330  N   GLY A 302       4.929  -1.824 -15.394  1.00 22.50           N\\nATOM   2331  CA  GLY A 302       4.077  -2.904 -14.903  1.00 22.50           C\\nATOM   2332  C   GLY A 302       4.788  -4.241 -14.657  1.00 22.50           C\\nATOM   2333  O   GLY A 302       4.243  -5.282 -15.022  1.00 22.50           O\\nATOM   2334  N   VAL A 303       5.990  -4.225 -14.049  1.00 19.43           N\\nATOM   2335  CA  VAL A 303       6.775  -5.424 -13.713  1.00 19.43           C\\nATOM   2336  C   VAL A 303       6.061  -6.245 -12.621  1.00 19.43           C\\nATOM   2337  O   VAL A 303       5.534  -5.684 -11.659  1.00 19.43           O\\nATOM   2338  CB  VAL A 303       8.190  -5.024 -13.215  1.00 19.43           C\\nATOM   2339  CG1 VAL A 303       9.098  -6.225 -12.878  1.00 19.43           C\\nATOM   2340  CG2 VAL A 303       8.924  -4.086 -14.186  1.00 19.43           C\\nATOM   2341  N   THR A 304       6.084  -7.576 -12.779  1.00 22.50           N\\nATOM   2342  CA  THR A 304       5.494  -8.561 -11.877  1.00 22.50           C\\nATOM   2343  C   THR A 304       6.612  -9.402 -11.242  1.00 22.50           C\\nATOM   2344  O   THR A 304       6.418 -10.262 -10.384  1.00 22.50           O\\nATOM   2345  CB  THR A 304       4.477  -9.431 -12.663  1.00 22.50           C\\nATOM   2346  OG1 THR A 304       5.106 -10.218 -13.659  1.00 22.50           O\\nATOM   2347  CG2 THR A 304       3.345  -8.616 -13.310  1.00 22.50           C\\n\",\"pdb\");\n\tviewer_15885999372043362.setStyle({\"cartoon\": {\"colorscheme\": {\"prop\": \"b\", \"gradient\": \"sinebow\", \"min\": 0, \"max\": 130}}});\n\tviewer_15885999372043362.setBackgroundColor(\"0xeeeeee\");\n\tviewer_15885999372043362.zoomTo();\nviewer_15885999372043362.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_15885999372043362\"  style=\"position: relative; width: 500px; height: 500px\">\n",
       "        <p id=\"3dmolwarning_15885999372043362\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "      resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.csb.pitt.edu/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_15885999372043362 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_15885999372043362\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_15885999372043362 = $3Dmol.createViewer($(\"#3dmolviewer_15885999372043362\"),{backgroundColor:\"white\"});\n",
       "\tviewer_15885999372043362.addModel(\"REMARK AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein'\\nATOM      1  N   SER A   1      -2.327   4.473 -16.891  1.00 25.83           N\\nATOM      2  CA  SER A   1      -2.209   5.813 -16.291  1.00 25.83           C\\nATOM      3  C   SER A   1      -2.610   5.806 -14.813  1.00 25.83           C\\nATOM      4  O   SER A   1      -3.283   4.886 -14.346  1.00 25.83           O\\nATOM      5  CB  SER A   1      -2.999   6.866 -17.096  1.00 25.83           C\\nATOM      6  OG  SER A   1      -2.177   7.396 -18.115  1.00 25.83           O\\nATOM      7  N   GLY A   2      -2.158   6.834 -14.083  1.00 21.53           N\\nATOM      8  CA  GLY A   2      -2.257   6.960 -12.637  1.00 21.53           C\\nATOM      9  C   GLY A   2      -0.976   6.471 -11.956  1.00 21.53           C\\nATOM     10  O   GLY A   2      -0.282   5.593 -12.473  1.00 21.53           O\\nATOM     11  N   PHE A   3      -0.670   7.047 -10.787  1.00 16.19           N\\nATOM     12  CA  PHE A   3       0.528   6.757 -10.010  1.00 16.19           C\\nATOM     13  C   PHE A   3       0.178   6.764  -8.519  1.00 16.19           C\\nATOM     14  O   PHE A   3      -0.178   7.804  -7.966  1.00 16.19           O\\nATOM     15  CB  PHE A   3       1.661   7.714 -10.423  1.00 16.19           C\\nATOM     16  CG  PHE A   3       3.047   7.202 -10.100  1.00 16.19           C\\nATOM     17  CD1 PHE A   3       3.659   6.253 -10.947  1.00 16.19           C\\nATOM     18  CD2 PHE A   3       3.725   7.664  -8.955  1.00 16.19           C\\nATOM     19  CE1 PHE A   3       4.935   5.748 -10.633  1.00 16.19           C\\nATOM     20  CE2 PHE A   3       5.006   7.173  -8.656  1.00 16.19           C\\nATOM     21  CZ  PHE A   3       5.597   6.194  -9.474  1.00 16.19           C\\nATOM     22  N   ARG A   4       0.254   5.582  -7.896  1.00 16.56           N\\nATOM     23  CA  ARG A   4      -0.095   5.305  -6.507  1.00 16.56           C\\nATOM     24  C   ARG A   4       1.143   4.912  -5.709  1.00 16.56           C\\nATOM     25  O   ARG A   4       2.091   4.362  -6.264  1.00 16.56           O\\nATOM     26  CB  ARG A   4      -1.087   4.119  -6.489  1.00 16.56           C\\nATOM     27  CG  ARG A   4      -2.538   4.542  -6.759  1.00 16.56           C\\nATOM     28  CD  ARG A   4      -3.135   5.257  -5.540  1.00 16.56           C\\nATOM     29  NE  ARG A   4      -4.534   5.640  -5.740  1.00 16.56           N\\nATOM     30  CZ  ARG A   4      -5.276   6.274  -4.817  1.00 16.56           C\\nATOM     31  NH1 ARG A   4      -4.767   6.595  -3.617  1.00 16.56           N\\nATOM     32  NH2 ARG A   4      -6.545   6.592  -5.099  1.00 16.56           N\\nATOM     33  N   LYS A   5       1.095   5.129  -4.383  1.00 14.70           N\\nATOM     34  CA  LYS A   5       2.080   4.605  -3.442  1.00 14.70           C\\nATOM     35  C   LYS A   5       1.779   3.111  -3.244  1.00 14.70           C\\nATOM     36  O   LYS A   5       0.868   2.731  -2.507  1.00 14.70           O\\nATOM     37  CB  LYS A   5       1.997   5.395  -2.120  1.00 14.70           C\\nATOM     38  CG  LYS A   5       3.140   5.067  -1.142  1.00 14.70           C\\nATOM     39  CD  LYS A   5       4.551   5.484  -1.613  1.00 14.70           C\\nATOM     40  CE  LYS A   5       4.782   7.002  -1.668  1.00 14.70           C\\nATOM     41  NZ  LYS A   5       4.673   7.624  -0.339  1.00 14.70           N\\nATOM     42  N   MET A   6       2.519   2.293  -4.001  1.00  8.53           N\\nATOM     43  CA  MET A   6       2.279   0.881  -4.232  1.00  8.53           C\\nATOM     44  C   MET A   6       3.273   0.073  -3.398  1.00  8.53           C\\nATOM     45  O   MET A   6       4.483   0.277  -3.485  1.00  8.53           O\\nATOM     46  CB  MET A   6       2.532   0.641  -5.738  1.00  8.53           C\\nATOM     47  CG  MET A   6       2.191  -0.774  -6.231  1.00  8.53           C\\nATOM     48  SD  MET A   6       2.354  -0.986  -8.025  1.00  8.53           S\\nATOM     49  CE  MET A   6       0.823  -0.190  -8.585  1.00  8.53           C\\nATOM     50  N   ALA A   7       2.750  -0.884  -2.623  1.00  9.04           N\\nATOM     51  CA  ALA A   7       3.541  -1.919  -1.973  1.00  9.04           C\\nATOM     52  C   ALA A   7       3.657  -3.132  -2.907  1.00  9.04           C\\nATOM     53  O   ALA A   7       2.814  -3.346  -3.781  1.00  9.04           O\\nATOM     54  CB  ALA A   7       2.808  -2.363  -0.699  1.00  9.04           C\\nATOM     55  N   PHE A   8       4.686  -3.961  -2.674  1.00  8.11           N\\nATOM     56  CA  PHE A   8       4.836  -5.289  -3.259  1.00  8.11           C\\nATOM     57  C   PHE A   8       3.754  -6.226  -2.691  1.00  8.11           C\\nATOM     58  O   PHE A   8       3.409  -6.079  -1.514  1.00  8.11           O\\nATOM     59  CB  PHE A   8       6.214  -5.869  -2.876  1.00  8.11           C\\nATOM     60  CG  PHE A   8       7.369  -5.268  -3.646  1.00  8.11           C\\nATOM     61  CD1 PHE A   8       7.603  -5.655  -4.982  1.00  8.11           C\\nATOM     62  CD2 PHE A   8       8.201  -4.310  -3.039  1.00  8.11           C\\nATOM     63  CE1 PHE A   8       8.659  -5.075  -5.709  1.00  8.11           C\\nATOM     64  CE2 PHE A   8       9.248  -3.728  -3.772  1.00  8.11           C\\nATOM     65  CZ  PHE A   8       9.484  -4.110  -5.105  1.00  8.11           C\\nATOM     66  N   PRO A   9       3.242  -7.199  -3.485  1.00  8.93           N\\nATOM     67  CA  PRO A   9       2.375  -8.288  -3.006  1.00  8.93           C\\nATOM     68  C   PRO A   9       2.999  -8.990  -1.788  1.00  8.93           C\\nATOM     69  O   PRO A   9       4.143  -9.439  -1.855  1.00  8.93           O\\nATOM     70  CB  PRO A   9       2.240  -9.256  -4.193  1.00  8.93           C\\nATOM     71  CG  PRO A   9       2.551  -8.423  -5.423  1.00  8.93           C\\nATOM     72  CD  PRO A   9       3.460  -7.300  -4.929  1.00  8.93           C\\nATOM     73  N   SER A  10       2.279  -9.000  -0.661  1.00 10.35           N\\nATOM     74  CA  SER A  10       2.811  -9.411   0.636  1.00 10.35           C\\nATOM     75  C   SER A  10       2.701 -10.920   0.908  1.00 10.35           C\\nATOM     76  O   SER A  10       3.244 -11.362   1.916  1.00 10.35           O\\nATOM     77  CB  SER A  10       2.072  -8.612   1.728  1.00 10.35           C\\nATOM     78  OG  SER A  10       0.666  -8.763   1.693  1.00 10.35           O\\nATOM     79  N   GLY A  11       2.057 -11.708   0.027  1.00  8.88           N\\nATOM     80  CA  GLY A  11       1.755 -13.134   0.193  1.00  8.88           C\\nATOM     81  C   GLY A  11       2.941 -14.055   0.522  1.00  8.88           C\\nATOM     82  O   GLY A  11       2.783 -14.977   1.322  1.00  8.88           O\\nATOM     83  N   LYS A  12       4.130 -13.783  -0.046  1.00  7.57           N\\nATOM     84  CA  LYS A  12       5.373 -14.516   0.213  1.00  7.57           C\\nATOM     85  C   LYS A  12       5.961 -14.231   1.608  1.00  7.57           C\\nATOM     86  O   LYS A  12       6.669 -15.081   2.147  1.00  7.57           O\\nATOM     87  CB  LYS A  12       6.429 -14.126  -0.840  1.00  7.57           C\\nATOM     88  CG  LYS A  12       6.192 -14.784  -2.206  1.00  7.57           C\\nATOM     89  CD  LYS A  12       7.171 -14.266  -3.270  1.00  7.57           C\\nATOM     90  CE  LYS A  12       6.830 -14.756  -4.681  1.00  7.57           C\\nATOM     91  NZ  LYS A  12       7.206 -16.165  -4.886  1.00  7.57           N\\nATOM     92  N   VAL A  13       5.660 -13.063   2.195  1.00  7.20           N\\nATOM     93  CA  VAL A  13       6.101 -12.659   3.529  1.00  7.20           C\\nATOM     94  C   VAL A  13       5.051 -13.048   4.579  1.00  7.20           C\\nATOM     95  O   VAL A  13       5.450 -13.397   5.686  1.00  7.20           O\\nATOM     96  CB  VAL A  13       6.357 -11.130   3.543  1.00  7.20           C\\nATOM     97  CG1 VAL A  13       6.776 -10.588   4.926  1.00  7.20           C\\nATOM     98  CG2 VAL A  13       7.448 -10.791   2.518  1.00  7.20           C\\nATOM     99  N   GLU A  14       3.746 -13.065   4.237  1.00  8.29           N\\nATOM    100  CA  GLU A  14       2.640 -13.522   5.092  1.00  8.29           C\\nATOM    101  C   GLU A  14       2.825 -14.985   5.532  1.00  8.29           C\\nATOM    102  O   GLU A  14       2.586 -15.309   6.695  1.00  8.29           O\\nATOM    103  CB  GLU A  14       1.299 -13.399   4.339  1.00  8.29           C\\nATOM    104  CG  GLU A  14       0.790 -11.962   4.131  1.00  8.29           C\\nATOM    105  CD  GLU A  14      -0.471 -11.938   3.269  1.00  8.29           C\\nATOM    106  OE1 GLU A  14      -1.443 -12.623   3.655  1.00  8.29           O\\nATOM    107  OE2 GLU A  14      -0.435 -11.254   2.224  1.00  8.29           O\\nATOM    108  N   GLY A  15       3.311 -15.844   4.618  1.00 13.05           N\\nATOM    109  CA  GLY A  15       3.633 -17.246   4.865  1.00 13.05           C\\nATOM    110  C   GLY A  15       4.893 -17.484   5.710  1.00 13.05           C\\nATOM    111  O   GLY A  15       5.182 -18.633   6.035  1.00 13.05           O\\nATOM    112  N   CYS A  16       5.629 -16.420   6.076  1.00  8.08           N\\nATOM    113  CA  CYS A  16       6.796 -16.463   6.955  1.00  8.08           C\\nATOM    114  C   CYS A  16       6.491 -15.907   8.354  1.00  8.08           C\\nATOM    115  O   CYS A  16       7.329 -16.062   9.238  1.00  8.08           O\\nATOM    116  CB  CYS A  16       7.932 -15.619   6.339  1.00  8.08           C\\nATOM    117  SG  CYS A  16       8.457 -16.318   4.753  1.00  8.08           S\\nATOM    118  N   MET A  17       5.335 -15.262   8.592  1.00  7.67           N\\nATOM    119  CA  MET A  17       5.069 -14.590   9.865  1.00  7.67           C\\nATOM    120  C   MET A  17       4.553 -15.571  10.919  1.00  7.67           C\\nATOM    121  O   MET A  17       3.609 -16.323  10.685  1.00  7.67           O\\nATOM    122  CB  MET A  17       4.031 -13.469   9.682  1.00  7.67           C\\nATOM    123  CG  MET A  17       4.494 -12.369   8.718  1.00  7.67           C\\nATOM    124  SD  MET A  17       6.197 -11.755   8.921  1.00  7.67           S\\nATOM    125  CE  MET A  17       6.173 -11.170  10.640  1.00  7.67           C\\nATOM    126  N   VAL A  18       5.191 -15.522  12.093  1.00  7.91           N\\nATOM    127  CA  VAL A  18       4.904 -16.316  13.282  1.00  7.91           C\\nATOM    128  C   VAL A  18       4.823 -15.381  14.493  1.00  7.91           C\\nATOM    129  O   VAL A  18       5.332 -14.263  14.458  1.00  7.91           O\\nATOM    130  CB  VAL A  18       5.922 -17.480  13.438  1.00  7.91           C\\nATOM    131  CG1 VAL A  18       5.944 -18.399  12.206  1.00  7.91           C\\nATOM    132  CG2 VAL A  18       7.356 -17.029  13.771  1.00  7.91           C\\nATOM    133  N   GLN A  19       4.177 -15.844  15.568  1.00  9.14           N\\nATOM    134  CA  GLN A  19       4.152 -15.184  16.867  1.00  9.14           C\\nATOM    135  C   GLN A  19       5.329 -15.737  17.680  1.00  9.14           C\\nATOM    136  O   GLN A  19       5.560 -16.944  17.660  1.00  9.14           O\\nATOM    137  CB  GLN A  19       2.845 -15.592  17.577  1.00  9.14           C\\nATOM    138  CG  GLN A  19       2.531 -14.738  18.821  1.00  9.14           C\\nATOM    139  CD  GLN A  19       1.233 -15.139  19.529  1.00  9.14           C\\nATOM    140  OE1 GLN A  19       0.453 -14.275  19.920  1.00  9.14           O\\nATOM    141  NE2 GLN A  19       0.976 -16.438  19.686  1.00  9.14           N\\nATOM    142  N   VAL A  20       6.044 -14.868  18.407  1.00  9.86           N\\nATOM    143  CA  VAL A  20       7.081 -15.240  19.368  1.00  9.86           C\\nATOM    144  C   VAL A  20       6.720 -14.589  20.705  1.00  9.86           C\\nATOM    145  O   VAL A  20       6.605 -13.367  20.787  1.00  9.86           O\\nATOM    146  CB  VAL A  20       8.502 -14.856  18.880  1.00  9.86           C\\nATOM    147  CG1 VAL A  20       9.584 -15.347  19.862  1.00  9.86           C\\nATOM    148  CG2 VAL A  20       8.808 -15.389  17.470  1.00  9.86           C\\nATOM    149  N   THR A  21       6.574 -15.418  21.745  1.00 13.64           N\\nATOM    150  CA  THR A  21       6.302 -15.015  23.119  1.00 13.64           C\\nATOM    151  C   THR A  21       7.469 -15.508  23.983  1.00 13.64           C\\nATOM    152  O   THR A  21       7.869 -16.667  23.888  1.00 13.64           O\\nATOM    153  CB  THR A  21       4.948 -15.610  23.581  1.00 13.64           C\\nATOM    154  OG1 THR A  21       3.902 -15.118  22.768  1.00 13.64           O\\nATOM    155  CG2 THR A  21       4.587 -15.288  25.044  1.00 13.64           C\\nATOM    156  N   CYS A  22       7.978 -14.631  24.858  1.00 16.53           N\\nATOM    157  CA  CYS A  22       8.954 -14.954  25.890  1.00 16.53           C\\nATOM    158  C   CYS A  22       8.521 -14.235  27.166  1.00 16.53           C\\nATOM    159  O   CYS A  22       8.553 -13.005  27.233  1.00 16.53           O\\nATOM    160  CB  CYS A  22      10.387 -14.668  25.400  1.00 16.53           C\\nATOM    161  SG  CYS A  22      11.614 -15.289  26.578  1.00 16.53           S\\nATOM    162  N   GLY A  23       8.057 -15.019  28.152  1.00 31.27           N\\nATOM    163  CA  GLY A  23       7.388 -14.539  29.352  1.00 31.27           C\\nATOM    164  C   GLY A  23       6.019 -13.945  29.008  1.00 31.27           C\\nATOM    165  O   GLY A  23       5.165 -14.629  28.442  1.00 31.27           O\\nATOM    166  N   THR A  24       5.825 -12.666  29.352  1.00 27.32           N\\nATOM    167  CA  THR A  24       4.602 -11.900  29.113  1.00 27.32           C\\nATOM    168  C   THR A  24       4.700 -10.994  27.865  1.00 27.32           C\\nATOM    169  O   THR A  24       3.696 -10.390  27.484  1.00 27.32           O\\nATOM    170  CB  THR A  24       4.316 -11.048  30.376  1.00 27.32           C\\nATOM    171  OG1 THR A  24       5.363 -10.129  30.639  1.00 27.32           O\\nATOM    172  CG2 THR A  24       4.062 -11.900  31.631  1.00 27.32           C\\nATOM    173  N   THR A  25       5.875 -10.903  27.217  1.00 26.88           N\\nATOM    174  CA  THR A  25       6.104 -10.080  26.029  1.00 26.88           C\\nATOM    175  C   THR A  25       5.859 -10.958  24.795  1.00 26.88           C\\nATOM    176  O   THR A  25       6.521 -11.981  24.624  1.00 26.88           O\\nATOM    177  CB  THR A  25       7.574  -9.600  26.035  1.00 26.88           C\\nATOM    178  OG1 THR A  25       7.772  -8.781  27.169  1.00 26.88           O\\nATOM    179  CG2 THR A  25       7.968  -8.752  24.809  1.00 26.88           C\\nATOM    180  N   THR A  26       4.917 -10.531  23.942  1.00 20.57           N\\nATOM    181  CA  THR A  26       4.553 -11.166  22.679  1.00 20.57           C\\nATOM    182  C   THR A  26       4.824 -10.166  21.550  1.00 20.57           C\\nATOM    183  O   THR A  26       4.357  -9.028  21.594  1.00 20.57           O\\nATOM    184  CB  THR A  26       3.063 -11.574  22.718  1.00 20.57           C\\nATOM    185  OG1 THR A  26       2.891 -12.558  23.717  1.00 20.57           O\\nATOM    186  CG2 THR A  26       2.558 -12.189  21.404  1.00 20.57           C\\nATOM    187  N   LEU A  27       5.549 -10.626  20.527  1.00 12.56           N\\nATOM    188  CA  LEU A  27       5.851  -9.910  19.295  1.00 12.56           C\\nATOM    189  C   LEU A  27       5.846 -10.878  18.101  1.00 12.56           C\\nATOM    190  O   LEU A  27       5.441 -12.033  18.238  1.00 12.56           O\\nATOM    191  CB  LEU A  27       7.062  -8.961  19.480  1.00 12.56           C\\nATOM    192  CG  LEU A  27       8.376  -9.555  20.035  1.00 12.56           C\\nATOM    193  CD1 LEU A  27       8.984 -10.622  19.116  1.00 12.56           C\\nATOM    194  CD2 LEU A  27       9.391  -8.427  20.278  1.00 12.56           C\\nATOM    195  N   ASN A  28       6.252 -10.399  16.920  1.00  7.83           N\\nATOM    196  CA  ASN A  28       6.263 -11.171  15.676  1.00  7.83           C\\nATOM    197  C   ASN A  28       7.675 -11.718  15.414  1.00  7.83           C\\nATOM    198  O   ASN A  28       8.676 -11.122  15.812  1.00  7.83           O\\nATOM    199  CB  ASN A  28       5.889 -10.248  14.497  1.00  7.83           C\\nATOM    200  CG  ASN A  28       4.544  -9.547  14.688  1.00  7.83           C\\nATOM    201  OD1 ASN A  28       3.490 -10.170  14.595  1.00  7.83           O\\nATOM    202  ND2 ASN A  28       4.583  -8.247  14.977  1.00  7.83           N\\nATOM    203  N   GLY A  29       7.743 -12.850  14.709  1.00  8.98           N\\nATOM    204  CA  GLY A  29       8.957 -13.442  14.172  1.00  8.98           C\\nATOM    205  C   GLY A  29       8.810 -13.689  12.672  1.00  8.98           C\\nATOM    206  O   GLY A  29       7.699 -13.873  12.175  1.00  8.98           O\\nATOM    207  N   LEU A  30       9.945 -13.725  11.964  1.00  7.58           N\\nATOM    208  CA  LEU A  30      10.050 -14.080  10.556  1.00  7.58           C\\nATOM    209  C   LEU A  30      10.706 -15.464  10.480  1.00  7.58           C\\nATOM    210  O   LEU A  30      11.878 -15.609  10.813  1.00  7.58           O\\nATOM    211  CB  LEU A  30      10.937 -13.026   9.855  1.00  7.58           C\\nATOM    212  CG  LEU A  30      11.004 -13.183   8.322  1.00  7.58           C\\nATOM    213  CD1 LEU A  30       9.678 -12.780   7.661  1.00  7.58           C\\nATOM    214  CD2 LEU A  30      12.181 -12.399   7.729  1.00  7.58           C\\nATOM    215  N   TRP A  31       9.947 -16.471  10.038  1.00  7.28           N\\nATOM    216  CA  TRP A  31      10.356 -17.862   9.892  1.00  7.28           C\\nATOM    217  C   TRP A  31      10.815 -18.087   8.442  1.00  7.28           C\\nATOM    218  O   TRP A  31      10.010 -18.037   7.512  1.00  7.28           O\\nATOM    219  CB  TRP A  31       9.115 -18.725  10.208  1.00  7.28           C\\nATOM    220  CG  TRP A  31       9.281 -20.210  10.356  1.00  7.28           C\\nATOM    221  CD1 TRP A  31      10.439 -20.888  10.546  1.00  7.28           C\\nATOM    222  CD2 TRP A  31       8.233 -21.223  10.294  1.00  7.28           C\\nATOM    223  NE1 TRP A  31      10.177 -22.240  10.611  1.00  7.28           N\\nATOM    224  CE2 TRP A  31       8.834 -22.509  10.453  1.00  7.28           C\\nATOM    225  CE3 TRP A  31       6.835 -21.193  10.070  1.00  7.28           C\\nATOM    226  CZ2 TRP A  31       8.089 -23.698  10.407  1.00  7.28           C\\nATOM    227  CZ3 TRP A  31       6.080 -22.381  10.009  1.00  7.28           C\\nATOM    228  CH2 TRP A  31       6.704 -23.631  10.181  1.00  7.28           C\\nATOM    229  N   LEU A  32      12.122 -18.322   8.265  1.00  8.89           N\\nATOM    230  CA  LEU A  32      12.785 -18.614   6.995  1.00  8.89           C\\nATOM    231  C   LEU A  32      13.661 -19.839   7.205  1.00  8.89           C\\nATOM    232  O   LEU A  32      14.543 -19.819   8.066  1.00  8.89           O\\nATOM    233  CB  LEU A  32      13.646 -17.423   6.547  1.00  8.89           C\\nATOM    234  CG  LEU A  32      12.868 -16.153   6.185  1.00  8.89           C\\nATOM    235  CD1 LEU A  32      13.846 -14.983   6.100  1.00  8.89           C\\nATOM    236  CD2 LEU A  32      12.074 -16.308   4.880  1.00  8.89           C\\nATOM    237  N   ASP A  33      13.383 -20.916   6.454  1.00  9.36           N\\nATOM    238  CA  ASP A  33      13.950 -22.253   6.650  1.00  9.36           C\\nATOM    239  C   ASP A  33      13.591 -22.724   8.071  1.00  9.36           C\\nATOM    240  O   ASP A  33      12.444 -22.555   8.486  1.00  9.36           O\\nATOM    241  CB  ASP A  33      15.467 -22.319   6.306  1.00  9.36           C\\nATOM    242  CG  ASP A  33      15.812 -21.813   4.901  1.00  9.36           C\\nATOM    243  OD1 ASP A  33      15.128 -22.244   3.947  1.00  9.36           O\\nATOM    244  OD2 ASP A  33      16.765 -21.010   4.799  1.00  9.36           O\\nATOM    245  N   ASP A  34      14.550 -23.272   8.824  1.00  9.08           N\\nATOM    246  CA  ASP A  34      14.369 -23.766  10.187  1.00  9.08           C\\nATOM    247  C   ASP A  34      14.816 -22.726  11.242  1.00  9.08           C\\nATOM    248  O   ASP A  34      15.148 -23.122  12.356  1.00  9.08           O\\nATOM    249  CB  ASP A  34      15.146 -25.106  10.314  1.00  9.08           C\\nATOM    250  CG  ASP A  34      16.682 -25.063  10.242  1.00  9.08           C\\nATOM    251  OD1 ASP A  34      17.232 -24.079   9.701  1.00  9.08           O\\nATOM    252  OD2 ASP A  34      17.287 -26.045  10.726  1.00  9.08           O\\nATOM    253  N   VAL A  35      14.806 -21.417  10.922  1.00  8.34           N\\nATOM    254  CA  VAL A  35      15.188 -20.318  11.820  1.00  8.34           C\\nATOM    255  C   VAL A  35      14.079 -19.257  11.851  1.00  8.34           C\\nATOM    256  O   VAL A  35      13.626 -18.804  10.801  1.00  8.34           O\\nATOM    257  CB  VAL A  35      16.570 -19.719  11.430  1.00  8.34           C\\nATOM    258  CG1 VAL A  35      17.017 -18.599  12.393  1.00  8.34           C\\nATOM    259  CG2 VAL A  35      17.678 -20.783  11.343  1.00  8.34           C\\nATOM    260  N   VAL A  36      13.696 -18.828  13.065  1.00  6.67           N\\nATOM    261  CA  VAL A  36      12.772 -17.731  13.333  1.00  6.67           C\\nATOM    262  C   VAL A  36      13.579 -16.557  13.902  1.00  6.67           C\\nATOM    263  O   VAL A  36      14.136 -16.643  14.995  1.00  6.67           O\\nATOM    264  CB  VAL A  36      11.655 -18.204  14.298  1.00  6.67           C\\nATOM    265  CG1 VAL A  36      10.731 -17.058  14.761  1.00  6.67           C\\nATOM    266  CG2 VAL A  36      10.818 -19.314  13.644  1.00  6.67           C\\nATOM    267  N   TYR A  37      13.617 -15.457  13.143  1.00  8.11           N\\nATOM    268  CA  TYR A  37      14.291 -14.204  13.467  1.00  8.11           C\\nATOM    269  C   TYR A  37      13.269 -13.286  14.147  1.00  8.11           C\\nATOM    270  O   TYR A  37      12.173 -13.103  13.623  1.00  8.11           O\\nATOM    271  CB  TYR A  37      14.727 -13.539  12.142  1.00  8.11           C\\nATOM    272  CG  TYR A  37      15.718 -14.367  11.344  1.00  8.11           C\\nATOM    273  CD1 TYR A  37      15.255 -15.347  10.439  1.00  8.11           C\\nATOM    274  CD2 TYR A  37      17.103 -14.195  11.533  1.00  8.11           C\\nATOM    275  CE1 TYR A  37      16.161 -16.184   9.770  1.00  8.11           C\\nATOM    276  CE2 TYR A  37      18.010 -15.023  10.846  1.00  8.11           C\\nATOM    277  CZ  TYR A  37      17.542 -16.023   9.975  1.00  8.11           C\\nATOM    278  OH  TYR A  37      18.430 -16.833   9.329  1.00  8.11           O\\nATOM    279  N   CYS A  38      13.623 -12.701  15.298  1.00  9.34           N\\nATOM    280  CA  CYS A  38      12.792 -11.761  16.054  1.00  9.34           C\\nATOM    281  C   CYS A  38      13.672 -10.743  16.799  1.00  9.34           C\\nATOM    282  O   CYS A  38      14.817 -11.077  17.101  1.00  9.34           O\\nATOM    283  CB  CYS A  38      11.786 -12.518  16.955  1.00  9.34           C\\nATOM    284  SG  CYS A  38      12.604 -13.390  18.326  1.00  9.34           S\\nATOM    285  N   PRO A  39      13.171  -9.532  17.145  1.00  9.32           N\\nATOM    286  CA  PRO A  39      13.857  -8.572  18.029  1.00  9.32           C\\nATOM    287  C   PRO A  39      14.219  -9.211  19.378  1.00  9.32           C\\nATOM    288  O   PRO A  39      13.389  -9.906  19.968  1.00  9.32           O\\nATOM    289  CB  PRO A  39      12.854  -7.429  18.256  1.00  9.32           C\\nATOM    290  CG  PRO A  39      11.886  -7.523  17.097  1.00  9.32           C\\nATOM    291  CD  PRO A  39      11.911  -8.981  16.657  1.00  9.32           C\\nATOM    292  N   ARG A  40      15.454  -8.987  19.854  1.00 12.72           N\\nATOM    293  CA  ARG A  40      15.956  -9.577  21.094  1.00 12.72           C\\nATOM    294  C   ARG A  40      15.265  -9.053  22.364  1.00 12.72           C\\nATOM    295  O   ARG A  40      15.312  -9.741  23.379  1.00 12.72           O\\nATOM    296  CB  ARG A  40      17.487  -9.410  21.191  1.00 12.72           C\\nATOM    297  CG  ARG A  40      17.994  -7.967  21.341  1.00 12.72           C\\nATOM    298  CD  ARG A  40      19.487  -7.921  21.665  1.00 12.72           C\\nATOM    299  NE  ARG A  40      19.972  -6.539  21.730  1.00 12.72           N\\nATOM    300  CZ  ARG A  40      21.225  -6.150  22.008  1.00 12.72           C\\nATOM    301  NH1 ARG A  40      22.191  -7.048  22.243  1.00 12.72           N\\nATOM    302  NH2 ARG A  40      21.506  -4.842  22.047  1.00 12.72           N\\nATOM    303  N   HIS A  41      14.578  -7.895  22.303  1.00 26.42           N\\nATOM    304  CA  HIS A  41      13.758  -7.340  23.384  1.00 26.42           C\\nATOM    305  C   HIS A  41      12.463  -8.120  23.666  1.00 26.42           C\\nATOM    306  O   HIS A  41      11.714  -7.728  24.557  1.00 26.42           O\\nATOM    307  CB  HIS A  41      13.466  -5.846  23.136  1.00 26.42           C\\nATOM    308  CG  HIS A  41      12.449  -5.482  22.084  1.00 26.42           C\\nATOM    309  ND1 HIS A  41      12.809  -4.892  20.870  1.00 26.42           N\\nATOM    310  CD2 HIS A  41      11.077  -5.625  22.119  1.00 26.42           C\\nATOM    311  CE1 HIS A  41      11.661  -4.732  20.231  1.00 26.42           C\\nATOM    312  NE2 HIS A  41      10.601  -5.134  20.924  1.00 26.42           N\\nATOM    313  N   VAL A  42      12.218  -9.253  22.987  1.00 18.64           N\\nATOM    314  CA  VAL A  42      11.208 -10.238  23.379  1.00 18.64           C\\nATOM    315  C   VAL A  42      11.475 -10.804  24.793  1.00 18.64           C\\nATOM    316  O   VAL A  42      10.525 -11.132  25.496  1.00 18.64           O\\nATOM    317  CB  VAL A  42      11.111 -11.344  22.299  1.00 18.64           C\\nATOM    318  CG1 VAL A  42      12.328 -12.282  22.247  1.00 18.64           C\\nATOM    319  CG2 VAL A  42       9.803 -12.149  22.400  1.00 18.64           C\\nATOM    320  N   ILE A  43      12.749 -10.861  25.230  1.00 15.10           N\\nATOM    321  CA  ILE A  43      13.159 -11.307  26.566  1.00 15.10           C\\nATOM    322  C   ILE A  43      12.988 -10.205  27.629  1.00 15.10           C\\nATOM    323  O   ILE A  43      13.037 -10.519  28.815  1.00 15.10           O\\nATOM    324  CB  ILE A  43      14.642 -11.794  26.540  1.00 15.10           C\\nATOM    325  CG1 ILE A  43      15.721 -10.675  26.536  1.00 15.10           C\\nATOM    326  CG2 ILE A  43      14.840 -12.846  25.435  1.00 15.10           C\\nATOM    327  CD1 ILE A  43      17.159 -11.173  26.332  1.00 15.10           C\\nATOM    328  N   CYS A  44      12.809  -8.936  27.226  1.00 31.09           N\\nATOM    329  CA  CYS A  44      12.689  -7.794  28.127  1.00 31.09           C\\nATOM    330  C   CYS A  44      11.221  -7.565  28.500  1.00 31.09           C\\nATOM    331  O   CYS A  44      10.345  -7.569  27.634  1.00 31.09           O\\nATOM    332  CB  CYS A  44      13.163  -6.515  27.401  1.00 31.09           C\\nATOM    333  SG  CYS A  44      14.962  -6.508  27.242  1.00 31.09           S\\nATOM    334  N   THR A  45      10.987  -7.253  29.784  1.00 81.61           N\\nATOM    335  CA  THR A  45       9.804  -6.535  30.266  1.00 81.61           C\\nATOM    336  C   THR A  45       9.958  -5.038  29.903  1.00 81.61           C\\nATOM    337  O   THR A  45      11.039  -4.598  29.507  1.00 81.61           O\\nATOM    338  CB  THR A  45       9.688  -6.716  31.806  1.00 81.61           C\\nATOM    339  OG1 THR A  45      10.680  -6.008  32.523  1.00 81.61           O\\nATOM    340  CG2 THR A  45       9.693  -8.180  32.274  1.00 81.61           C\\nATOM    341  N   SER A  46       8.896  -4.228  30.053  1.00129.82           N\\nATOM    342  CA  SER A  46       8.966  -2.774  29.842  1.00129.82           C\\nATOM    343  C   SER A  46       9.843  -2.047  30.889  1.00129.82           C\\nATOM    344  O   SER A  46      10.348  -0.960  30.602  1.00129.82           O\\nATOM    345  CB  SER A  46       7.537  -2.197  29.775  1.00129.82           C\\nATOM    346  OG  SER A  46       6.852  -2.288  31.006  1.00129.82           O\\nATOM    347  N   GLU A  47      10.085  -2.678  32.053  0.24115.27           N\\nATOM    348  CA  GLU A  47      11.016  -2.241  33.092  0.24115.27           C\\nATOM    349  C   GLU A  47      12.472  -2.529  32.682  0.24115.27           C\\nATOM    350  O   GLU A  47      13.331  -1.671  32.886  0.24115.27           O\\nATOM    351  CB  GLU A  47      10.738  -2.975  34.430  0.24115.27           C\\nATOM    352  CG  GLU A  47       9.260  -3.032  34.877  0.24115.27           C\\nATOM    353  CD  GLU A  47       8.515  -4.252  34.331  0.24115.27           C\\nATOM    354  OE1 GLU A  47       7.712  -4.066  33.392  0.24115.27           O\\nATOM    355  OE2 GLU A  47       8.792  -5.362  34.835  0.24115.27           O\\nATOM    356  N   ASP A  48      12.741  -3.701  32.066  1.00 71.53           N\\nATOM    357  CA  ASP A  48      14.051  -4.083  31.520  1.00 71.53           C\\nATOM    358  C   ASP A  48      14.489  -3.169  30.372  1.00 71.53           C\\nATOM    359  O   ASP A  48      15.680  -2.891  30.265  1.00 71.53           O\\nATOM    360  CB  ASP A  48      14.067  -5.540  30.991  1.00 71.53           C\\nATOM    361  CG  ASP A  48      13.856  -6.642  32.028  1.00 71.53           C\\nATOM    362  OD1 ASP A  48      14.257  -6.433  33.193  1.00 71.53           O\\nATOM    363  OD2 ASP A  48      13.338  -7.704  31.618  1.00 71.53           O\\nATOM    364  N   MET A  49      13.544  -2.670  29.553  1.00 58.60           N\\nATOM    365  CA  MET A  49      13.799  -1.781  28.418  1.00 58.60           C\\nATOM    366  C   MET A  49      14.432  -0.431  28.786  1.00 58.60           C\\nATOM    367  O   MET A  49      15.036   0.188  27.916  1.00 58.60           O\\nATOM    368  CB  MET A  49      12.495  -1.539  27.624  1.00 58.60           C\\nATOM    369  CG  MET A  49      12.114  -2.702  26.700  1.00 58.60           C\\nATOM    370  SD  MET A  49      13.363  -3.138  25.460  1.00 58.60           S\\nATOM    371  CE  MET A  49      13.307  -1.700  24.363  1.00 58.60           C\\nATOM    372  N   LEU A  50      14.351   0.008  30.053  1.00 54.26           N\\nATOM    373  CA  LEU A  50      14.956   1.247  30.540  1.00 54.26           C\\nATOM    374  C   LEU A  50      16.495   1.183  30.577  1.00 54.26           C\\nATOM    375  O   LEU A  50      17.134   2.199  30.308  1.00 54.26           O\\nATOM    376  CB  LEU A  50      14.428   1.549  31.959  1.00 54.26           C\\nATOM    377  CG  LEU A  50      12.905   1.797  32.043  1.00 54.26           C\\nATOM    378  CD1 LEU A  50      12.439   1.803  33.506  1.00 54.26           C\\nATOM    379  CD2 LEU A  50      12.489   3.102  31.342  1.00 54.26           C\\nATOM    380  N   ASN A  51      17.087   0.011  30.874  1.00 45.22           N\\nATOM    381  CA  ASN A  51      18.532  -0.239  30.847  1.00 45.22           C\\nATOM    382  C   ASN A  51      18.802  -1.757  30.744  1.00 45.22           C\\nATOM    383  O   ASN A  51      19.208  -2.359  31.739  1.00 45.22           O\\nATOM    384  CB  ASN A  51      19.213   0.459  32.061  1.00 45.22           C\\nATOM    385  CG  ASN A  51      20.748   0.499  32.033  1.00 45.22           C\\nATOM    386  OD1 ASN A  51      21.376   0.518  33.088  1.00 45.22           O\\nATOM    387  ND2 ASN A  51      21.369   0.554  30.851  1.00 45.22           N\\nATOM    388  N   PRO A  52      18.574  -2.398  29.574  1.00 25.46           N\\nATOM    389  CA  PRO A  52      18.727  -3.850  29.412  1.00 25.46           C\\nATOM    390  C   PRO A  52      20.194  -4.235  29.150  1.00 25.46           C\\nATOM    391  O   PRO A  52      20.829  -3.696  28.241  1.00 25.46           O\\nATOM    392  CB  PRO A  52      17.870  -4.181  28.179  1.00 25.46           C\\nATOM    393  CG  PRO A  52      17.896  -2.913  27.335  1.00 25.46           C\\nATOM    394  CD  PRO A  52      18.044  -1.786  28.353  1.00 25.46           C\\nATOM    395  N   ASN A  53      20.706  -5.215  29.910  1.00 27.98           N\\nATOM    396  CA  ASN A  53      21.904  -5.977  29.567  1.00 27.98           C\\nATOM    397  C   ASN A  53      21.372  -7.279  28.956  1.00 27.98           C\\nATOM    398  O   ASN A  53      20.917  -8.174  29.668  1.00 27.98           O\\nATOM    399  CB  ASN A  53      22.731  -6.275  30.839  1.00 27.98           C\\nATOM    400  CG  ASN A  53      24.128  -6.819  30.499  1.00 27.98           C\\nATOM    401  OD1 ASN A  53      24.265  -7.819  29.795  1.00 27.98           O\\nATOM    402  ND2 ASN A  53      25.176  -6.144  30.971  1.00 27.98           N\\nATOM    403  N   TYR A  54      21.417  -7.359  27.621  1.00 18.60           N\\nATOM    404  CA  TYR A  54      20.811  -8.430  26.840  1.00 18.60           C\\nATOM    405  C   TYR A  54      21.492  -9.789  26.997  1.00 18.60           C\\nATOM    406  O   TYR A  54      20.784 -10.791  26.984  1.00 18.60           O\\nATOM    407  CB  TYR A  54      20.808  -8.019  25.364  1.00 18.60           C\\nATOM    408  CG  TYR A  54      19.674  -7.069  25.064  1.00 18.60           C\\nATOM    409  CD1 TYR A  54      18.369  -7.585  24.977  1.00 18.60           C\\nATOM    410  CD2 TYR A  54      19.899  -5.688  24.900  1.00 18.60           C\\nATOM    411  CE1 TYR A  54      17.293  -6.723  24.735  1.00 18.60           C\\nATOM    412  CE2 TYR A  54      18.816  -4.828  24.643  1.00 18.60           C\\nATOM    413  CZ  TYR A  54      17.508  -5.345  24.572  1.00 18.60           C\\nATOM    414  OH  TYR A  54      16.445  -4.523  24.356  1.00 18.60           O\\nATOM    415  N   GLU A  55      22.820  -9.838  27.200  1.00 21.06           N\\nATOM    416  CA  GLU A  55      23.549 -11.078  27.469  1.00 21.06           C\\nATOM    417  C   GLU A  55      23.145 -11.658  28.836  1.00 21.06           C\\nATOM    418  O   GLU A  55      22.861 -12.850  28.916  1.00 21.06           O\\nATOM    419  CB  GLU A  55      25.074 -10.819  27.441  1.00 21.06           C\\nATOM    420  CG  GLU A  55      25.679 -10.604  26.033  1.00 21.06           C\\nATOM    421  CD  GLU A  55      25.795 -11.857  25.153  1.00 21.06           C\\nATOM    422  OE1 GLU A  55      25.271 -12.921  25.548  1.00 21.06           O\\nATOM    423  OE2 GLU A  55      26.401 -11.719  24.070  1.00 21.06           O\\nATOM    424  N   ASP A  56      23.049 -10.824  29.887  1.00 23.58           N\\nATOM    425  CA  ASP A  56      22.645 -11.249  31.233  1.00 23.58           C\\nATOM    426  C   ASP A  56      21.164 -11.646  31.327  1.00 23.58           C\\nATOM    427  O   ASP A  56      20.850 -12.597  32.043  1.00 23.58           O\\nATOM    428  CB  ASP A  56      22.945 -10.143  32.270  1.00 23.58           C\\nATOM    429  CG  ASP A  56      24.434  -9.834  32.473  1.00 23.58           C\\nATOM    430  OD1 ASP A  56      25.278 -10.653  32.048  1.00 23.58           O\\nATOM    431  OD2 ASP A  56      24.701  -8.779  33.087  1.00 23.58           O\\nATOM    432  N   LEU A  57      20.262 -10.979  30.585  1.00 22.50           N\\nATOM    433  CA  LEU A  57      18.845 -11.348  30.509  1.00 22.50           C\\nATOM    434  C   LEU A  57      18.635 -12.660  29.732  1.00 22.50           C\\nATOM    435  O   LEU A  57      17.730 -13.420  30.076  1.00 22.50           O\\nATOM    436  CB  LEU A  57      18.038 -10.228  29.814  1.00 22.50           C\\nATOM    437  CG  LEU A  57      17.846  -8.940  30.644  1.00 22.50           C\\nATOM    438  CD1 LEU A  57      17.314  -7.808  29.755  1.00 22.50           C\\nATOM    439  CD2 LEU A  57      16.916  -9.141  31.853  1.00 22.50           C\\nATOM    440  N   LEU A  58      19.473 -12.941  28.718  1.00 21.26           N\\nATOM    441  CA  LEU A  58      19.356 -14.113  27.853  1.00 21.26           C\\nATOM    442  C   LEU A  58      19.872 -15.399  28.526  1.00 21.26           C\\nATOM    443  O   LEU A  58      19.359 -16.464  28.191  1.00 21.26           O\\nATOM    444  CB  LEU A  58      20.131 -13.838  26.545  1.00 21.26           C\\nATOM    445  CG  LEU A  58      19.918 -14.838  25.389  1.00 21.26           C\\nATOM    446  CD1 LEU A  58      18.446 -14.896  24.947  1.00 21.26           C\\nATOM    447  CD2 LEU A  58      20.812 -14.453  24.203  1.00 21.26           C\\nATOM    448  N   ILE A  59      20.799 -15.319  29.508  1.00 21.86           N\\nATOM    449  CA  ILE A  59      21.253 -16.439  30.360  1.00 21.86           C\\nATOM    450  C   ILE A  59      20.081 -17.152  31.069  1.00 21.86           C\\nATOM    451  O   ILE A  59      20.100 -18.373  31.221  1.00 21.86           O\\nATOM    452  CB  ILE A  59      22.343 -15.951  31.373  1.00 21.86           C\\nATOM    453  CG1 ILE A  59      23.673 -15.641  30.653  1.00 21.86           C\\nATOM    454  CG2 ILE A  59      22.608 -16.927  32.547  1.00 21.86           C\\nATOM    455  CD1 ILE A  59      24.694 -14.864  31.501  1.00 21.86           C\\nATOM    456  N   ARG A  60      19.062 -16.381  31.477  1.00 21.63           N\\nATOM    457  CA  ARG A  60      17.888 -16.839  32.216  1.00 21.63           C\\nATOM    458  C   ARG A  60      16.792 -17.419  31.299  1.00 21.63           C\\nATOM    459  O   ARG A  60      15.768 -17.873  31.811  1.00 21.63           O\\nATOM    460  CB  ARG A  60      17.298 -15.627  32.977  1.00 21.63           C\\nATOM    461  CG  ARG A  60      18.307 -14.845  33.845  1.00 21.63           C\\nATOM    462  CD  ARG A  60      18.878 -15.654  35.023  1.00 21.63           C\\nATOM    463  NE  ARG A  60      20.213 -15.172  35.403  1.00 21.63           N\\nATOM    464  CZ  ARG A  60      21.157 -15.874  36.052  1.00 21.63           C\\nATOM    465  NH1 ARG A  60      22.335 -15.292  36.302  1.00 21.63           N\\nATOM    466  NH2 ARG A  60      20.948 -17.138  36.448  1.00 21.63           N\\nATOM    467  N   LYS A  61      17.000 -17.429  29.970  1.00 21.28           N\\nATOM    468  CA  LYS A  61      16.096 -18.007  28.979  1.00 21.28           C\\nATOM    469  C   LYS A  61      16.731 -19.271  28.382  1.00 21.28           C\\nATOM    470  O   LYS A  61      17.888 -19.262  27.964  1.00 21.28           O\\nATOM    471  CB  LYS A  61      15.845 -16.992  27.841  1.00 21.28           C\\nATOM    472  CG  LYS A  61      15.367 -15.599  28.293  1.00 21.28           C\\nATOM    473  CD  LYS A  61      14.139 -15.615  29.219  1.00 21.28           C\\nATOM    474  CE  LYS A  61      13.617 -14.204  29.515  1.00 21.28           C\\nATOM    475  NZ  LYS A  61      12.505 -14.228  30.479  1.00 21.28           N\\nATOM    476  N   SER A  62      15.938 -20.347  28.326  1.00 21.63           N\\nATOM    477  CA  SER A  62      16.206 -21.599  27.620  1.00 21.63           C\\nATOM    478  C   SER A  62      15.326 -21.659  26.357  1.00 21.63           C\\nATOM    479  O   SER A  62      14.474 -20.793  26.154  1.00 21.63           O\\nATOM    480  CB  SER A  62      15.830 -22.761  28.560  1.00 21.63           C\\nATOM    481  OG  SER A  62      16.829 -22.922  29.545  1.00 21.63           O\\nATOM    482  N   ASN A  63      15.517 -22.691  25.511  1.00 20.19           N\\nATOM    483  CA  ASN A  63      14.747 -22.927  24.277  1.00 20.19           C\\nATOM    484  C   ASN A  63      13.227 -23.001  24.498  1.00 20.19           C\\nATOM    485  O   ASN A  63      12.469 -22.431  23.713  1.00 20.19           O\\nATOM    486  CB  ASN A  63      15.204 -24.217  23.556  1.00 20.19           C\\nATOM    487  CG  ASN A  63      16.641 -24.195  23.041  1.00 20.19           C\\nATOM    488  OD1 ASN A  63      17.109 -23.174  22.551  1.00 20.19           O\\nATOM    489  ND2 ASN A  63      17.355 -25.315  23.156  1.00 20.19           N\\nATOM    490  N   HIS A  64      12.792 -23.669  25.582  1.00 21.45           N\\nATOM    491  CA  HIS A  64      11.385 -23.854  25.940  1.00 21.45           C\\nATOM    492  C   HIS A  64      10.690 -22.563  26.426  1.00 21.45           C\\nATOM    493  O   HIS A  64       9.460 -22.523  26.441  1.00 21.45           O\\nATOM    494  CB  HIS A  64      11.252 -24.966  27.005  1.00 21.45           C\\nATOM    495  CG  HIS A  64      11.820 -24.677  28.375  1.00 21.45           C\\nATOM    496  ND1 HIS A  64      13.103 -25.081  28.759  1.00 21.45           N\\nATOM    497  CD2 HIS A  64      11.229 -24.017  29.432  1.00 21.45           C\\nATOM    498  CE1 HIS A  64      13.235 -24.660  30.010  1.00 21.45           C\\nATOM    499  NE2 HIS A  64      12.158 -24.014  30.456  1.00 21.45           N\\nATOM    500  N   ASN A  65      11.450 -21.510  26.788  1.00 21.78           N\\nATOM    501  CA  ASN A  65      10.916 -20.199  27.180  1.00 21.78           C\\nATOM    502  C   ASN A  65      10.584 -19.315  25.965  1.00 21.78           C\\nATOM    503  O   ASN A  65       9.896 -18.311  26.136  1.00 21.78           O\\nATOM    504  CB  ASN A  65      11.938 -19.430  28.053  1.00 21.78           C\\nATOM    505  CG  ASN A  65      12.235 -20.114  29.391  1.00 21.78           C\\nATOM    506  OD1 ASN A  65      13.382 -20.434  29.686  1.00 21.78           O\\nATOM    507  ND2 ASN A  65      11.213 -20.304  30.227  1.00 21.78           N\\nATOM    508  N   PHE A  66      11.029 -19.681  24.752  1.00 15.07           N\\nATOM    509  CA  PHE A  66      10.676 -19.011  23.504  1.00 15.07           C\\nATOM    510  C   PHE A  66       9.555 -19.822  22.852  1.00 15.07           C\\nATOM    511  O   PHE A  66       9.805 -20.817  22.174  1.00 15.07           O\\nATOM    512  CB  PHE A  66      11.911 -18.992  22.585  1.00 15.07           C\\nATOM    513  CG  PHE A  66      13.014 -18.107  23.123  1.00 15.07           C\\nATOM    514  CD1 PHE A  66      12.873 -16.706  23.078  1.00 15.07           C\\nATOM    515  CD2 PHE A  66      14.159 -18.681  23.712  1.00 15.07           C\\nATOM    516  CE1 PHE A  66      13.868 -15.884  23.633  1.00 15.07           C\\nATOM    517  CE2 PHE A  66      15.151 -17.857  24.273  1.00 15.07           C\\nATOM    518  CZ  PHE A  66      15.004 -16.457  24.235  1.00 15.07           C\\nATOM    519  N   LEU A  67       8.309 -19.395  23.086  1.00 13.49           N\\nATOM    520  CA  LEU A  67       7.105 -20.019  22.556  1.00 13.49           C\\nATOM    521  C   LEU A  67       6.845 -19.423  21.169  1.00 13.49           C\\nATOM    522  O   LEU A  67       6.550 -18.236  21.048  1.00 13.49           O\\nATOM    523  CB  LEU A  67       5.916 -19.685  23.479  1.00 13.49           C\\nATOM    524  CG  LEU A  67       6.043 -20.172  24.936  1.00 13.49           C\\nATOM    525  CD1 LEU A  67       4.807 -19.739  25.734  1.00 13.49           C\\nATOM    526  CD2 LEU A  67       6.247 -21.691  25.039  1.00 13.49           C\\nATOM    527  N   VAL A  68       6.988 -20.253  20.129  1.00 12.01           N\\nATOM    528  CA  VAL A  68       6.852 -19.873  18.728  1.00 12.01           C\\nATOM    529  C   VAL A  68       5.604 -20.572  18.173  1.00 12.01           C\\nATOM    530  O   VAL A  68       5.460 -21.783  18.332  1.00 12.01           O\\nATOM    531  CB  VAL A  68       8.132 -20.278  17.958  1.00 12.01           C\\nATOM    532  CG1 VAL A  68       8.049 -19.922  16.468  1.00 12.01           C\\nATOM    533  CG2 VAL A  68       9.398 -19.630  18.551  1.00 12.01           C\\nATOM    534  N   GLN A  69       4.721 -19.809  17.512  1.00 11.31           N\\nATOM    535  CA  GLN A  69       3.434 -20.269  17.000  1.00 11.31           C\\nATOM    536  C   GLN A  69       3.184 -19.705  15.597  1.00 11.31           C\\nATOM    537  O   GLN A  69       3.001 -18.499  15.436  1.00 11.31           O\\nATOM    538  CB  GLN A  69       2.341 -19.972  18.049  1.00 11.31           C\\nATOM    539  CG  GLN A  69       0.903 -20.336  17.617  1.00 11.31           C\\nATOM    540  CD  GLN A  69      -0.167 -20.096  18.692  1.00 11.31           C\\nATOM    541  OE1 GLN A  69      -1.355 -20.101  18.382  1.00 11.31           O\\nATOM    542  NE2 GLN A  69       0.221 -19.861  19.947  1.00 11.31           N\\nATOM    543  N   ALA A  70       3.162 -20.596  14.594  1.00 15.60           N\\nATOM    544  CA  ALA A  70       2.822 -20.320  13.202  1.00 15.60           C\\nATOM    545  C   ALA A  70       1.363 -20.738  12.995  1.00 15.60           C\\nATOM    546  O   ALA A  70       1.038 -21.917  13.113  1.00 15.60           O\\nATOM    547  CB  ALA A  70       3.711 -21.212  12.317  1.00 15.60           C\\nATOM    548  N   GLY A  71       0.471 -19.772  12.728  1.00 29.79           N\\nATOM    549  CA  GLY A  71      -0.974 -19.990  12.694  1.00 29.79           C\\nATOM    550  C   GLY A  71      -1.498 -20.316  14.097  1.00 29.79           C\\nATOM    551  O   GLY A  71      -1.397 -19.485  14.997  1.00 29.79           O\\nATOM    552  N   ASN A  72      -2.020 -21.540  14.272  0.24 31.03           N\\nATOM    553  CA  ASN A  72      -2.456 -22.136  15.541  0.24 31.03           C\\nATOM    554  C   ASN A  72      -1.735 -23.471  15.820  0.24 31.03           C\\nATOM    555  O   ASN A  72      -2.260 -24.313  16.549  0.24 31.03           O\\nATOM    556  CB  ASN A  72      -4.009 -22.201  15.629  0.24 31.03           C\\nATOM    557  CG  ASN A  72      -4.698 -20.953  16.204  0.24 31.03           C\\nATOM    558  OD1 ASN A  72      -5.635 -21.080  16.989  0.24 31.03           O\\nATOM    559  ND2 ASN A  72      -4.288 -19.749  15.805  0.24 31.03           N\\nATOM    560  N  AVAL A  73      -0.514 -23.649  15.278  0.25 23.65           N\\nATOM    561  CA AVAL A  73       0.378 -24.779  15.545  0.25 23.65           C\\nATOM    562  C  AVAL A  73       1.704 -24.255  16.110  0.25 23.65           C\\nATOM    563  O  AVAL A  73       2.295 -23.312  15.585  0.25 23.65           O\\nATOM    564  CB AVAL A  73       0.520 -25.718  14.313  0.25 23.65           C\\nATOM    565  CG1AVAL A  73       1.058 -25.039  13.042  0.25 23.65           C\\nATOM    566  CG2AVAL A  73       1.352 -26.972  14.634  0.25 23.65           C\\nATOM    567  N   GLN A  74       2.150 -24.887  17.203  1.00 16.27           N\\nATOM    568  CA  GLN A  74       3.329 -24.520  17.972  1.00 16.27           C\\nATOM    569  C   GLN A  74       4.567 -25.197  17.353  1.00 16.27           C\\nATOM    570  O   GLN A  74       4.554 -26.405  17.108  1.00 16.27           O\\nATOM    571  CB  GLN A  74       3.105 -25.052  19.408  1.00 16.27           C\\nATOM    572  CG  GLN A  74       3.909 -24.300  20.483  1.00 16.27           C\\nATOM    573  CD  GLN A  74       3.370 -22.897  20.783  1.00 16.27           C\\nATOM    574  OE1 GLN A  74       2.195 -22.604  20.573  1.00 16.27           O\\nATOM    575  NE2 GLN A  74       4.237 -22.016  21.278  1.00 16.27           N\\nATOM    576  N   LEU A  75       5.630 -24.417  17.107  1.00 13.26           N\\nATOM    577  CA  LEU A  75       6.916 -24.896  16.596  1.00 13.26           C\\nATOM    578  C   LEU A  75       7.854 -25.140  17.782  1.00 13.26           C\\nATOM    579  O   LEU A  75       7.934 -24.315  18.693  1.00 13.26           O\\nATOM    580  CB  LEU A  75       7.573 -23.811  15.711  1.00 13.26           C\\nATOM    581  CG  LEU A  75       6.730 -23.319  14.516  1.00 13.26           C\\nATOM    582  CD1 LEU A  75       7.562 -22.365  13.648  1.00 13.26           C\\nATOM    583  CD2 LEU A  75       6.175 -24.466  13.654  1.00 13.26           C\\nATOM    584  N   ARG A  76       8.573 -26.271  17.762  1.00 12.52           N\\nATOM    585  CA  ARG A  76       9.502 -26.664  18.815  1.00 12.52           C\\nATOM    586  C   ARG A  76      10.859 -26.011  18.537  1.00 12.52           C\\nATOM    587  O   ARG A  76      11.477 -26.297  17.514  1.00 12.52           O\\nATOM    588  CB  ARG A  76       9.653 -28.196  18.761  1.00 12.52           C\\nATOM    589  CG  ARG A  76      10.564 -28.792  19.853  1.00 12.52           C\\nATOM    590  CD  ARG A  76      10.715 -30.314  19.731  1.00 12.52           C\\nATOM    591  NE  ARG A  76       9.409 -30.984  19.772  1.00 12.52           N\\nATOM    592  CZ  ARG A  76       8.696 -31.276  20.871  1.00 12.52           C\\nATOM    593  NH1 ARG A  76       9.161 -31.009  22.100  1.00 12.52           N\\nATOM    594  NH2 ARG A  76       7.490 -31.837  20.725  1.00 12.52           N\\nATOM    595  N   VAL A  77      11.318 -25.160  19.467  1.00 12.81           N\\nATOM    596  CA  VAL A  77      12.620 -24.500  19.431  1.00 12.81           C\\nATOM    597  C   VAL A  77      13.660 -25.495  19.976  1.00 12.81           C\\nATOM    598  O   VAL A  77      13.496 -26.046  21.063  1.00 12.81           O\\nATOM    599  CB  VAL A  77      12.573 -23.209  20.295  1.00 12.81           C\\nATOM    600  CG1 VAL A  77      13.917 -22.453  20.330  1.00 12.81           C\\nATOM    601  CG2 VAL A  77      11.483 -22.243  19.797  1.00 12.81           C\\nATOM    602  N   ILE A  78      14.724 -25.716  19.195  1.00 16.19           N\\nATOM    603  CA  ILE A  78      15.823 -26.648  19.457  1.00 16.19           C\\nATOM    604  C   ILE A  78      17.171 -25.914  19.583  1.00 16.19           C\\nATOM    605  O   ILE A  78      18.205 -26.560  19.752  1.00 16.19           O\\nATOM    606  CB  ILE A  78      15.847 -27.748  18.360  1.00 16.19           C\\nATOM    607  CG1 ILE A  78      16.065 -27.185  16.934  1.00 16.19           C\\nATOM    608  CG2 ILE A  78      14.578 -28.620  18.435  1.00 16.19           C\\nATOM    609  CD1 ILE A  78      16.482 -28.244  15.906  1.00 16.19           C\\nATOM    610  N   GLY A  79      17.168 -24.578  19.506  1.00 16.66           N\\nATOM    611  CA  GLY A  79      18.345 -23.743  19.647  1.00 16.66           C\\nATOM    612  C   GLY A  79      17.952 -22.270  19.651  1.00 16.66           C\\nATOM    613  O   GLY A  79      16.926 -21.894  19.089  1.00 16.66           O\\nATOM    614  N   HIS A  80      18.784 -21.434  20.282  1.00 15.01           N\\nATOM    615  CA  HIS A  80      18.674 -19.982  20.257  1.00 15.01           C\\nATOM    616  C   HIS A  80      20.058 -19.338  20.335  1.00 15.01           C\\nATOM    617  O   HIS A  80      20.942 -19.826  21.040  1.00 15.01           O\\nATOM    618  CB  HIS A  80      17.668 -19.438  21.292  1.00 15.01           C\\nATOM    619  CG  HIS A  80      18.096 -19.535  22.732  1.00 15.01           C\\nATOM    620  ND1 HIS A  80      18.014 -20.700  23.466  1.00 15.01           N\\nATOM    621  CD2 HIS A  80      18.657 -18.605  23.577  1.00 15.01           C\\nATOM    622  CE1 HIS A  80      18.489 -20.437  24.683  1.00 15.01           C\\nATOM    623  NE2 HIS A  80      18.903 -19.185  24.821  1.00 15.01           N\\nATOM    624  N   SER A  81      20.222 -18.223  19.615  1.00 13.43           N\\nATOM    625  CA  SER A  81      21.430 -17.414  19.574  1.00 13.43           C\\nATOM    626  C   SER A  81      21.068 -15.957  19.288  1.00 13.43           C\\nATOM    627  O   SER A  81      20.159 -15.691  18.504  1.00 13.43           O\\nATOM    628  CB  SER A  81      22.458 -18.033  18.603  1.00 13.43           C\\nATOM    629  OG  SER A  81      22.104 -17.829  17.248  1.00 13.43           O\\nATOM    630  N   MET A  82      21.807 -15.021  19.895  1.00 12.46           N\\nATOM    631  CA  MET A  82      21.705 -13.592  19.641  1.00 12.46           C\\nATOM    632  C   MET A  82      22.793 -13.204  18.636  1.00 12.46           C\\nATOM    633  O   MET A  82      23.978 -13.444  18.869  1.00 12.46           O\\nATOM    634  CB  MET A  82      21.945 -12.861  20.973  1.00 12.46           C\\nATOM    635  CG  MET A  82      21.706 -11.345  20.911  1.00 12.46           C\\nATOM    636  SD  MET A  82      21.487 -10.572  22.534  1.00 12.46           S\\nATOM    637  CE  MET A  82      23.060 -10.962  23.335  1.00 12.46           C\\nATOM    638  N   GLN A  83      22.366 -12.582  17.530  1.00 13.05           N\\nATOM    639  CA  GLN A  83      23.221 -11.925  16.554  1.00 13.05           C\\nATOM    640  C   GLN A  83      22.810 -10.454  16.545  1.00 13.05           C\\nATOM    641  O   GLN A  83      21.782 -10.097  15.970  1.00 13.05           O\\nATOM    642  CB  GLN A  83      23.063 -12.586  15.172  1.00 13.05           C\\nATOM    643  CG  GLN A  83      24.013 -11.974  14.120  1.00 13.05           C\\nATOM    644  CD  GLN A  83      23.820 -12.526  12.710  1.00 13.05           C\\nATOM    645  OE1 GLN A  83      22.913 -13.312  12.448  1.00 13.05           O\\nATOM    646  NE2 GLN A  83      24.692 -12.119  11.789  1.00 13.05           N\\nATOM    647  N   ASN A  84      23.611  -9.619  17.222  1.00 12.88           N\\nATOM    648  CA  ASN A  84      23.461  -8.169  17.329  1.00 12.88           C\\nATOM    649  C   ASN A  84      22.192  -7.841  18.133  1.00 12.88           C\\nATOM    650  O   ASN A  84      22.163  -8.123  19.330  1.00 12.88           O\\nATOM    651  CB  ASN A  84      23.610  -7.458  15.959  1.00 12.88           C\\nATOM    652  CG  ASN A  84      24.972  -7.714  15.305  1.00 12.88           C\\nATOM    653  OD1 ASN A  84      26.009  -7.359  15.862  1.00 12.88           O\\nATOM    654  ND2 ASN A  84      24.981  -8.353  14.135  1.00 12.88           N\\nATOM    655  N   CYS A  85      21.143  -7.304  17.486  1.00 11.81           N\\nATOM    656  CA  CYS A  85      19.867  -6.944  18.111  1.00 11.81           C\\nATOM    657  C   CYS A  85      18.694  -7.834  17.660  1.00 11.81           C\\nATOM    658  O   CYS A  85      17.543  -7.470  17.896  1.00 11.81           O\\nATOM    659  CB  CYS A  85      19.560  -5.443  17.936  1.00 11.81           C\\nATOM    660  SG  CYS A  85      20.879  -4.427  18.648  1.00 11.81           S\\nATOM    661  N   VAL A  86      18.956  -9.020  17.083  1.00 10.64           N\\nATOM    662  CA  VAL A  86      17.947 -10.056  16.835  1.00 10.64           C\\nATOM    663  C   VAL A  86      18.365 -11.372  17.492  1.00 10.64           C\\nATOM    664  O   VAL A  86      19.551 -11.694  17.564  1.00 10.64           O\\nATOM    665  CB  VAL A  86      17.602 -10.226  15.328  1.00 10.64           C\\nATOM    666  CG1 VAL A  86      16.930  -8.972  14.748  1.00 10.64           C\\nATOM    667  CG2 VAL A  86      18.769 -10.720  14.451  1.00 10.64           C\\nATOM    668  N   LEU A  87      17.361 -12.146  17.923  1.00  9.52           N\\nATOM    669  CA  LEU A  87      17.493 -13.548  18.280  1.00  9.52           C\\nATOM    670  C   LEU A  87      17.071 -14.381  17.073  1.00  9.52           C\\nATOM    671  O   LEU A  87      16.028 -14.141  16.465  1.00  9.52           O\\nATOM    672  CB  LEU A  87      16.537 -13.901  19.433  1.00  9.52           C\\nATOM    673  CG  LEU A  87      16.935 -13.282  20.782  1.00  9.52           C\\nATOM    674  CD1 LEU A  87      15.802 -13.458  21.791  1.00  9.52           C\\nATOM    675  CD2 LEU A  87      18.237 -13.853  21.357  1.00  9.52           C\\nATOM    676  N   LYS A  88      17.897 -15.383  16.770  1.00  9.68           N\\nATOM    677  CA  LYS A  88      17.646 -16.440  15.810  1.00  9.68           C\\nATOM    678  C   LYS A  88      17.280 -17.664  16.650  1.00  9.68           C\\nATOM    679  O   LYS A  88      18.110 -18.160  17.407  1.00  9.68           O\\nATOM    680  CB  LYS A  88      18.956 -16.705  15.046  1.00  9.68           C\\nATOM    681  CG  LYS A  88      19.435 -15.460  14.282  1.00  9.68           C\\nATOM    682  CD  LYS A  88      20.559 -15.745  13.284  1.00  9.68           C\\nATOM    683  CE  LYS A  88      21.860 -16.225  13.938  1.00  9.68           C\\nATOM    684  NZ  LYS A  88      22.956 -16.251  12.958  1.00  9.68           N\\nATOM    685  N   LEU A  89      16.023 -18.108  16.549  1.00  9.20           N\\nATOM    686  CA  LEU A  89      15.490 -19.270  17.244  1.00  9.20           C\\nATOM    687  C   LEU A  89      15.430 -20.396  16.208  1.00  9.20           C\\nATOM    688  O   LEU A  89      14.580 -20.372  15.319  1.00  9.20           O\\nATOM    689  CB  LEU A  89      14.056 -18.956  17.741  1.00  9.20           C\\nATOM    690  CG  LEU A  89      13.911 -17.698  18.623  1.00  9.20           C\\nATOM    691  CD1 LEU A  89      12.432 -17.432  18.929  1.00  9.20           C\\nATOM    692  CD2 LEU A  89      14.733 -17.793  19.915  1.00  9.20           C\\nATOM    693  N   LYS A  90      16.329 -21.386  16.319  1.00 11.44           N\\nATOM    694  CA  LYS A  90      16.297 -22.581  15.483  1.00 11.44           C\\nATOM    695  C   LYS A  90      15.136 -23.465  15.958  1.00 11.44           C\\nATOM    696  O   LYS A  90      15.048 -23.780  17.143  1.00 11.44           O\\nATOM    697  CB  LYS A  90      17.619 -23.370  15.595  1.00 11.44           C\\nATOM    698  CG  LYS A  90      17.744 -24.425  14.477  1.00 11.44           C\\nATOM    699  CD  LYS A  90      18.917 -25.391  14.668  1.00 11.44           C\\nATOM    700  CE  LYS A  90      19.080 -26.332  13.469  1.00 11.44           C\\nATOM    701  NZ  LYS A  90      20.142 -27.323  13.700  1.00 11.44           N\\nATOM    702  N   VAL A  91      14.268 -23.856  15.019  1.00 10.64           N\\nATOM    703  CA  VAL A  91      13.133 -24.750  15.217  1.00 10.64           C\\nATOM    704  C   VAL A  91      13.382 -26.066  14.460  1.00 10.64           C\\nATOM    705  O   VAL A  91      14.226 -26.121  13.567  1.00 10.64           O\\nATOM    706  CB  VAL A  91      11.815 -24.046  14.792  1.00 10.64           C\\nATOM    707  CG1 VAL A  91      11.378 -22.984  15.817  1.00 10.64           C\\nATOM    708  CG2 VAL A  91      11.867 -23.427  13.386  1.00 10.64           C\\nATOM    709  N   ASP A  92      12.679 -27.144  14.845  1.00 10.85           N\\nATOM    710  CA  ASP A  92      12.873 -28.499  14.303  1.00 10.85           C\\nATOM    711  C   ASP A  92      12.214 -28.698  12.919  1.00 10.85           C\\nATOM    712  O   ASP A  92      12.413 -29.744  12.302  1.00 10.85           O\\nATOM    713  CB  ASP A  92      12.305 -29.535  15.312  1.00 10.85           C\\nATOM    714  CG  ASP A  92      10.778 -29.677  15.436  1.00 10.85           C\\nATOM    715  OD1 ASP A  92      10.048 -28.711  15.118  1.00 10.85           O\\nATOM    716  OD2 ASP A  92      10.366 -30.770  15.881  1.00 10.85           O\\nATOM    717  N   THR A  93      11.423 -27.718  12.457  1.00 10.73           N\\nATOM    718  CA  THR A  93      10.646 -27.758  11.228  1.00 10.73           C\\nATOM    719  C   THR A  93      11.094 -26.576  10.366  1.00 10.73           C\\nATOM    720  O   THR A  93      11.039 -25.427  10.808  1.00 10.73           O\\nATOM    721  CB  THR A  93       9.139 -27.683  11.587  1.00 10.73           C\\nATOM    722  OG1 THR A  93       8.791 -28.809  12.370  1.00 10.73           O\\nATOM    723  CG2 THR A  93       8.204 -27.702  10.366  1.00 10.73           C\\nATOM    724  N   ALA A  94      11.485 -26.860   9.116  1.00  9.39           N\\nATOM    725  CA  ALA A  94      11.705 -25.853   8.088  1.00  9.39           C\\nATOM    726  C   ALA A  94      10.341 -25.348   7.604  1.00  9.39           C\\nATOM    727  O   ALA A  94       9.436 -26.157   7.400  1.00  9.39           O\\nATOM    728  CB  ALA A  94      12.454 -26.487   6.908  1.00  9.39           C\\nATOM    729  N   ASN A  95      10.185 -24.025   7.428  1.00  9.09           N\\nATOM    730  CA  ASN A  95       8.956 -23.404   6.933  1.00  9.09           C\\nATOM    731  C   ASN A  95       8.704 -23.898   5.493  1.00  9.09           C\\nATOM    732  O   ASN A  95       9.539 -23.635   4.625  1.00  9.09           O\\nATOM    733  CB  ASN A  95       9.102 -21.863   6.998  1.00  9.09           C\\nATOM    734  CG  ASN A  95       7.873 -21.067   6.537  1.00  9.09           C\\nATOM    735  OD1 ASN A  95       6.774 -21.600   6.425  1.00  9.09           O\\nATOM    736  ND2 ASN A  95       8.054 -19.787   6.226  1.00  9.09           N\\nATOM    737  N   PRO A  96       7.577 -24.594   5.215  1.00  8.88           N\\nATOM    738  CA  PRO A  96       7.269 -25.109   3.873  1.00  8.88           C\\nATOM    739  C   PRO A  96       6.900 -24.000   2.872  1.00  8.88           C\\nATOM    740  O   PRO A  96       6.959 -24.223   1.664  1.00  8.88           O\\nATOM    741  CB  PRO A  96       6.107 -26.090   4.090  1.00  8.88           C\\nATOM    742  CG  PRO A  96       5.405 -25.591   5.342  1.00  8.88           C\\nATOM    743  CD  PRO A  96       6.512 -24.937   6.166  1.00  8.88           C\\nATOM    744  N   LYS A  97       6.568 -22.806   3.384  1.00 10.91           N\\nATOM    745  CA  LYS A  97       6.253 -21.601   2.635  1.00 10.91           C\\nATOM    746  C   LYS A  97       7.464 -20.651   2.539  1.00 10.91           C\\nATOM    747  O   LYS A  97       7.257 -19.485   2.210  1.00 10.91           O\\nATOM    748  CB  LYS A  97       5.119 -20.864   3.378  1.00 10.91           C\\nATOM    749  CG  LYS A  97       3.811 -21.655   3.507  1.00 10.91           C\\nATOM    750  CD  LYS A  97       2.713 -20.790   4.139  1.00 10.91           C\\nATOM    751  CE  LYS A  97       1.397 -21.544   4.340  1.00 10.91           C\\nATOM    752  NZ  LYS A  97       0.370 -20.655   4.907  1.00 10.91           N\\nATOM    753  N   THR A  98       8.718 -21.094   2.789  1.00  9.74           N\\nATOM    754  CA  THR A  98       9.906 -20.232   2.647  1.00  9.74           C\\nATOM    755  C   THR A  98      10.056 -19.854   1.160  1.00  9.74           C\\nATOM    756  O   THR A  98      10.209 -20.759   0.337  1.00  9.74           O\\nATOM    757  CB  THR A  98      11.212 -20.938   3.105  1.00  9.74           C\\nATOM    758  OG1 THR A  98      11.132 -21.281   4.458  1.00  9.74           O\\nATOM    759  CG2 THR A  98      12.462 -20.044   3.024  1.00  9.74           C\\nATOM    760  N   PRO A  99      10.040 -18.552   0.799  1.00  9.51           N\\nATOM    761  CA  PRO A  99      10.293 -18.115  -0.576  1.00  9.51           C\\nATOM    762  C   PRO A  99      11.806 -18.169  -0.846  1.00  9.51           C\\nATOM    763  O   PRO A  99      12.599 -18.407   0.063  1.00  9.51           O\\nATOM    764  CB  PRO A  99       9.769 -16.669  -0.604  1.00  9.51           C\\nATOM    765  CG  PRO A  99      10.006 -16.154   0.808  1.00  9.51           C\\nATOM    766  CD  PRO A  99       9.821 -17.398   1.678  1.00  9.51           C\\nATOM    767  N   LYS A 100      12.231 -17.869  -2.079  1.00  9.71           N\\nATOM    768  CA  LYS A 100      13.620 -17.485  -2.340  1.00  9.71           C\\nATOM    769  C   LYS A 100      13.861 -16.140  -1.638  1.00  9.71           C\\nATOM    770  O   LYS A 100      12.988 -15.272  -1.677  1.00  9.71           O\\nATOM    771  CB  LYS A 100      13.849 -17.350  -3.851  1.00  9.71           C\\nATOM    772  CG  LYS A 100      13.721 -18.680  -4.604  1.00  9.71           C\\nATOM    773  CD  LYS A 100      13.930 -18.483  -6.108  1.00  9.71           C\\nATOM    774  CE  LYS A 100      13.577 -19.724  -6.934  1.00  9.71           C\\nATOM    775  NZ  LYS A 100      13.624 -19.421  -8.373  1.00  9.71           N\\nATOM    776  N   TYR A 101      14.987 -16.002  -0.926  1.00 10.49           N\\nATOM    777  CA  TYR A 101      15.238 -14.839  -0.085  1.00 10.49           C\\nATOM    778  C   TYR A 101      16.708 -14.445  -0.006  1.00 10.49           C\\nATOM    779  O   TYR A 101      17.608 -15.266  -0.180  1.00 10.49           O\\nATOM    780  CB  TYR A 101      14.545 -14.959   1.294  1.00 10.49           C\\nATOM    781  CG  TYR A 101      15.253 -15.857   2.295  1.00 10.49           C\\nATOM    782  CD1 TYR A 101      16.207 -15.314   3.181  1.00 10.49           C\\nATOM    783  CD2 TYR A 101      14.989 -17.240   2.318  1.00 10.49           C\\nATOM    784  CE1 TYR A 101      16.908 -16.154   4.067  1.00 10.49           C\\nATOM    785  CE2 TYR A 101      15.687 -18.082   3.203  1.00 10.49           C\\nATOM    786  CZ  TYR A 101      16.654 -17.539   4.070  1.00 10.49           C\\nATOM    787  OH  TYR A 101      17.357 -18.354   4.905  1.00 10.49           O\\nATOM    788  N   LYS A 102      16.906 -13.164   0.321  1.00 10.60           N\\nATOM    789  CA  LYS A 102      18.169 -12.531   0.659  1.00 10.60           C\\nATOM    790  C   LYS A 102      17.958 -11.652   1.892  1.00 10.60           C\\nATOM    791  O   LYS A 102      16.854 -11.160   2.132  1.00 10.60           O\\nATOM    792  CB  LYS A 102      18.686 -11.706  -0.534  1.00 10.60           C\\nATOM    793  CG  LYS A 102      19.153 -12.546  -1.730  1.00 10.60           C\\nATOM    794  CD  LYS A 102      19.504 -11.673  -2.939  1.00 10.60           C\\nATOM    795  CE  LYS A 102      19.938 -12.508  -4.150  1.00 10.60           C\\nATOM    796  NZ  LYS A 102      20.153 -11.656  -5.330  1.00 10.60           N\\nATOM    797  N   PHE A 103      19.040 -11.420   2.641  1.00 10.65           N\\nATOM    798  CA  PHE A 103      19.144 -10.363   3.638  1.00 10.65           C\\nATOM    799  C   PHE A 103      20.069  -9.304   3.037  1.00 10.65           C\\nATOM    800  O   PHE A 103      21.222  -9.596   2.714  1.00 10.65           O\\nATOM    801  CB  PHE A 103      19.752 -10.932   4.938  1.00 10.65           C\\nATOM    802  CG  PHE A 103      18.887 -11.910   5.724  1.00 10.65           C\\nATOM    803  CD1 PHE A 103      17.505 -12.063   5.464  1.00 10.65           C\\nATOM    804  CD2 PHE A 103      19.478 -12.671   6.754  1.00 10.65           C\\nATOM    805  CE1 PHE A 103      16.731 -12.962   6.215  1.00 10.65           C\\nATOM    806  CE2 PHE A 103      18.704 -13.578   7.501  1.00 10.65           C\\nATOM    807  CZ  PHE A 103      17.330 -13.725   7.231  1.00 10.65           C\\nATOM    808  N   VAL A 104      19.540  -8.085   2.870  1.00 10.87           N\\nATOM    809  CA  VAL A 104      20.236  -6.930   2.313  1.00 10.87           C\\nATOM    810  C   VAL A 104      20.132  -5.768   3.304  1.00 10.87           C\\nATOM    811  O   VAL A 104      19.079  -5.556   3.902  1.00 10.87           O\\nATOM    812  CB  VAL A 104      19.697  -6.584   0.897  1.00 10.87           C\\nATOM    813  CG1 VAL A 104      19.790  -7.772  -0.076  1.00 10.87           C\\nATOM    814  CG2 VAL A 104      18.272  -6.003   0.888  1.00 10.87           C\\nATOM    815  N   ARG A 105      21.216  -4.999   3.456  1.00 11.78           N\\nATOM    816  CA  ARG A 105      21.216  -3.757   4.215  1.00 11.78           C\\nATOM    817  C   ARG A 105      21.075  -2.618   3.196  1.00 11.78           C\\nATOM    818  O   ARG A 105      21.990  -2.377   2.408  1.00 11.78           O\\nATOM    819  CB  ARG A 105      22.537  -3.646   5.001  1.00 11.78           C\\nATOM    820  CG  ARG A 105      22.558  -2.429   5.939  1.00 11.78           C\\nATOM    821  CD  ARG A 105      23.778  -2.402   6.862  1.00 11.78           C\\nATOM    822  NE  ARG A 105      23.732  -1.225   7.736  1.00 11.78           N\\nATOM    823  CZ  ARG A 105      23.064  -1.131   8.898  1.00 11.78           C\\nATOM    824  NH1 ARG A 105      22.339  -2.149   9.378  1.00 11.78           N\\nATOM    825  NH2 ARG A 105      23.121   0.010   9.591  1.00 11.78           N\\nATOM    826  N   ILE A 106      19.915  -1.941   3.197  1.00 12.87           N\\nATOM    827  CA  ILE A 106      19.598  -0.861   2.261  1.00 12.87           C\\nATOM    828  C   ILE A 106      20.226   0.465   2.714  1.00 12.87           C\\nATOM    829  O   ILE A 106      20.446   0.699   3.901  1.00 12.87           O\\nATOM    830  CB  ILE A 106      18.062  -0.682   2.121  1.00 12.87           C\\nATOM    831  CG1 ILE A 106      17.316  -0.458   3.460  1.00 12.87           C\\nATOM    832  CG2 ILE A 106      17.466  -1.835   1.292  1.00 12.87           C\\nATOM    833  CD1 ILE A 106      15.891   0.073   3.285  1.00 12.87           C\\nATOM    834  N   GLN A 107      20.485   1.344   1.742  0.24 17.34           N\\nATOM    835  CA  GLN A 107      20.991   2.698   1.944  0.24 17.34           C\\nATOM    836  C   GLN A 107      19.823   3.665   2.212  0.24 17.34           C\\nATOM    837  O   GLN A 107      18.683   3.356   1.861  0.24 17.34           O\\nATOM    838  CB  GLN A 107      21.714   3.110   0.639  0.24 17.34           C\\nATOM    839  CG  GLN A 107      23.003   2.294   0.388  0.24 17.34           C\\nATOM    840  CD  GLN A 107      24.071   2.460   1.477  0.24 17.34           C\\nATOM    841  OE1 GLN A 107      24.598   1.472   1.984  0.24 17.34           O\\nATOM    842  NE2 GLN A 107      24.369   3.699   1.873  0.24 17.34           N\\nATOM    843  N   PRO A 108      20.079   4.853   2.806  1.00 15.89           N\\nATOM    844  CA  PRO A 108      19.055   5.885   2.974  1.00 15.89           C\\nATOM    845  C   PRO A 108      18.630   6.462   1.617  1.00 15.89           C\\nATOM    846  O   PRO A 108      19.456   6.700   0.737  1.00 15.89           O\\nATOM    847  CB  PRO A 108      19.674   6.936   3.895  1.00 15.89           C\\nATOM    848  CG  PRO A 108      21.176   6.750   3.768  1.00 15.89           C\\nATOM    849  CD  PRO A 108      21.370   5.299   3.331  1.00 15.89           C\\nATOM    850  N   GLY A 109      17.318   6.644   1.459  1.00 13.75           N\\nATOM    851  CA  GLY A 109      16.639   6.988   0.223  1.00 13.75           C\\nATOM    852  C   GLY A 109      15.957   5.790  -0.442  1.00 13.75           C\\nATOM    853  O   GLY A 109      15.105   5.998  -1.303  1.00 13.75           O\\nATOM    854  N   GLN A 110      16.303   4.548  -0.058  1.00 11.36           N\\nATOM    855  CA  GLN A 110      15.715   3.338  -0.624  1.00 11.36           C\\nATOM    856  C   GLN A 110      14.412   2.995   0.089  1.00 11.36           C\\nATOM    857  O   GLN A 110      14.300   3.074   1.312  1.00 11.36           O\\nATOM    858  CB  GLN A 110      16.665   2.140  -0.449  1.00 11.36           C\\nATOM    859  CG  GLN A 110      17.952   2.250  -1.281  1.00 11.36           C\\nATOM    860  CD  GLN A 110      17.693   2.390  -2.785  1.00 11.36           C\\nATOM    861  OE1 GLN A 110      16.946   1.610  -3.370  1.00 11.36           O\\nATOM    862  NE2 GLN A 110      18.292   3.400  -3.411  1.00 11.36           N\\nATOM    863  N   THR A 111      13.443   2.598  -0.735  1.00 10.90           N\\nATOM    864  CA  THR A 111      12.083   2.246  -0.380  1.00 10.90           C\\nATOM    865  C   THR A 111      11.938   0.732  -0.190  1.00 10.90           C\\nATOM    866  O   THR A 111      12.775  -0.055  -0.632  1.00 10.90           O\\nATOM    867  CB  THR A 111      11.179   2.726  -1.530  1.00 10.90           C\\nATOM    868  OG1 THR A 111      11.603   2.110  -2.728  1.00 10.90           O\\nATOM    869  CG2 THR A 111      11.158   4.256  -1.677  1.00 10.90           C\\nATOM    870  N   PHE A 112      10.858   0.345   0.496  1.00  9.20           N\\nATOM    871  CA  PHE A 112      10.504  -1.031   0.818  1.00  9.20           C\\nATOM    872  C   PHE A 112       9.061  -1.090   1.317  1.00  9.20           C\\nATOM    873  O   PHE A 112       8.545  -0.113   1.857  1.00  9.20           O\\nATOM    874  CB  PHE A 112      11.503  -1.628   1.832  1.00  9.20           C\\nATOM    875  CG  PHE A 112      11.600  -0.900   3.159  1.00  9.20           C\\nATOM    876  CD1 PHE A 112      10.786  -1.294   4.237  1.00  9.20           C\\nATOM    877  CD2 PHE A 112      12.515   0.162   3.321  1.00  9.20           C\\nATOM    878  CE1 PHE A 112      10.930  -0.671   5.489  1.00  9.20           C\\nATOM    879  CE2 PHE A 112      12.638   0.802   4.566  1.00  9.20           C\\nATOM    880  CZ  PHE A 112      11.855   0.370   5.649  1.00  9.20           C\\nATOM    881  N   SER A 113       8.428  -2.257   1.168  1.00  8.34           N\\nATOM    882  CA  SER A 113       7.093  -2.546   1.674  1.00  8.34           C\\nATOM    883  C   SER A 113       7.227  -3.154   3.069  1.00  8.34           C\\nATOM    884  O   SER A 113       7.943  -4.136   3.233  1.00  8.34           O\\nATOM    885  CB  SER A 113       6.459  -3.624   0.783  1.00  8.34           C\\nATOM    886  OG  SER A 113       6.276  -3.083  -0.499  1.00  8.34           O\\nATOM    887  N   VAL A 114       6.515  -2.592   4.052  1.00  8.30           N\\nATOM    888  CA  VAL A 114       6.358  -3.145   5.393  1.00  8.30           C\\nATOM    889  C   VAL A 114       5.020  -3.884   5.424  1.00  8.30           C\\nATOM    890  O   VAL A 114       3.989  -3.290   5.112  1.00  8.30           O\\nATOM    891  CB  VAL A 114       6.312  -1.991   6.428  1.00  8.30           C\\nATOM    892  CG1 VAL A 114       5.957  -2.466   7.851  1.00  8.30           C\\nATOM    893  CG2 VAL A 114       7.646  -1.236   6.479  1.00  8.30           C\\nATOM    894  N   LEU A 115       5.040  -5.145   5.877  1.00  7.63           N\\nATOM    895  CA  LEU A 115       3.852  -5.881   6.284  1.00  7.63           C\\nATOM    896  C   LEU A 115       3.762  -5.727   7.810  1.00  7.63           C\\nATOM    897  O   LEU A 115       4.542  -6.328   8.546  1.00  7.63           O\\nATOM    898  CB  LEU A 115       4.040  -7.364   5.888  1.00  7.63           C\\nATOM    899  CG  LEU A 115       2.825  -8.277   6.155  1.00  7.63           C\\nATOM    900  CD1 LEU A 115       1.604  -7.848   5.325  1.00  7.63           C\\nATOM    901  CD2 LEU A 115       3.189  -9.744   5.893  1.00  7.63           C\\nATOM    902  N   ALA A 116       2.819  -4.902   8.285  1.00  9.79           N\\nATOM    903  CA  ALA A 116       2.540  -4.711   9.706  1.00  9.79           C\\nATOM    904  C   ALA A 116       1.739  -5.913  10.215  1.00  9.79           C\\nATOM    905  O   ALA A 116       0.746  -6.303   9.600  1.00  9.79           O\\nATOM    906  CB  ALA A 116       1.704  -3.440   9.887  1.00  9.79           C\\nATOM    907  N   CYS A 117       2.214  -6.513  11.312  1.00  8.38           N\\nATOM    908  CA  CYS A 117       1.707  -7.742  11.913  1.00  8.38           C\\nATOM    909  C   CYS A 117       1.491  -7.541  13.412  1.00  8.38           C\\nATOM    910  O   CYS A 117       2.195  -6.754  14.040  1.00  8.38           O\\nATOM    911  CB  CYS A 117       2.693  -8.900  11.658  1.00  8.38           C\\nATOM    912  SG  CYS A 117       2.829  -9.279   9.896  1.00  8.38           S\\nATOM    913  N   TYR A 118       0.536  -8.285  13.985  1.00  9.80           N\\nATOM    914  CA  TYR A 118       0.237  -8.345  15.415  1.00  9.80           C\\nATOM    915  C   TYR A 118      -0.116  -9.788  15.747  1.00  9.80           C\\nATOM    916  O   TYR A 118      -1.042 -10.337  15.152  1.00  9.80           O\\nATOM    917  CB  TYR A 118      -0.903  -7.377  15.792  1.00  9.80           C\\nATOM    918  CG  TYR A 118      -0.582  -5.922  15.535  1.00  9.80           C\\nATOM    919  CD1 TYR A 118       0.104  -5.165  16.504  1.00  9.80           C\\nATOM    920  CD2 TYR A 118      -0.917  -5.342  14.295  1.00  9.80           C\\nATOM    921  CE1 TYR A 118       0.475  -3.841  16.219  1.00  9.80           C\\nATOM    922  CE2 TYR A 118      -0.526  -4.025  14.002  1.00  9.80           C\\nATOM    923  CZ  TYR A 118       0.183  -3.279  14.959  1.00  9.80           C\\nATOM    924  OH  TYR A 118       0.597  -2.018  14.656  1.00  9.80           O\\nATOM    925  N   ASN A 119       0.627 -10.406  16.677  1.00 14.05           N\\nATOM    926  CA  ASN A 119       0.440 -11.777  17.168  1.00 14.05           C\\nATOM    927  C   ASN A 119       0.776 -12.821  16.086  1.00 14.05           C\\nATOM    928  O   ASN A 119       0.144 -13.875  16.017  1.00 14.05           O\\nATOM    929  CB  ASN A 119      -0.944 -11.989  17.838  1.00 14.05           C\\nATOM    930  CG  ASN A 119      -1.177 -11.044  19.021  1.00 14.05           C\\nATOM    931  OD1 ASN A 119      -1.844 -10.020  18.882  1.00 14.05           O\\nATOM    932  ND2 ASN A 119      -0.643 -11.394  20.191  1.00 14.05           N\\nATOM    933  N   GLY A 120       1.731 -12.490  15.202  1.00 12.43           N\\nATOM    934  CA  GLY A 120       2.110 -13.252  14.018  1.00 12.43           C\\nATOM    935  C   GLY A 120       1.151 -13.123  12.827  1.00 12.43           C\\nATOM    936  O   GLY A 120       1.420 -13.724  11.788  1.00 12.43           O\\nATOM    937  N   SER A 121       0.040 -12.379  12.972  1.00 11.66           N\\nATOM    938  CA  SER A 121      -1.023 -12.249  11.978  1.00 11.66           C\\nATOM    939  C   SER A 121      -0.913 -10.881  11.272  1.00 11.66           C\\nATOM    940  O   SER A 121      -1.004  -9.858  11.953  1.00 11.66           O\\nATOM    941  CB  SER A 121      -2.385 -12.325  12.695  1.00 11.66           C\\nATOM    942  OG  SER A 121      -2.624 -13.650  13.126  1.00 11.66           O\\nATOM    943  N   PRO A 122      -0.736 -10.837   9.928  1.00 15.22           N\\nATOM    944  CA  PRO A 122      -0.765  -9.615   9.102  1.00 15.22           C\\nATOM    945  C   PRO A 122      -2.044  -8.775   9.269  1.00 15.22           C\\nATOM    946  O   PRO A 122      -3.147  -9.322   9.249  1.00 15.22           O\\nATOM    947  CB  PRO A 122      -0.655 -10.106   7.650  1.00 15.22           C\\nATOM    948  CG  PRO A 122      -0.050 -11.495   7.732  1.00 15.22           C\\nATOM    949  CD  PRO A 122      -0.399 -12.008   9.119  1.00 15.22           C\\nATOM    950  N   SER A 123      -1.879  -7.450   9.405  1.00 15.75           N\\nATOM    951  CA  SER A 123      -2.954  -6.466   9.543  1.00 15.75           C\\nATOM    952  C   SER A 123      -3.033  -5.510   8.340  1.00 15.75           C\\nATOM    953  O   SER A 123      -4.104  -4.959   8.086  1.00 15.75           O\\nATOM    954  CB  SER A 123      -2.766  -5.704  10.872  1.00 15.75           C\\nATOM    955  OG  SER A 123      -1.665  -4.814  10.854  1.00 15.75           O\\nATOM    956  N   GLY A 124      -1.936  -5.321   7.592  1.00 10.28           N\\nATOM    957  CA  GLY A 124      -1.903  -4.457   6.419  1.00 10.28           C\\nATOM    958  C   GLY A 124      -0.492  -4.301   5.861  1.00 10.28           C\\nATOM    959  O   GLY A 124       0.488  -4.605   6.541  1.00 10.28           O\\nATOM    960  N   VAL A 125      -0.407  -3.792   4.623  1.00  8.37           N\\nATOM    961  CA  VAL A 125       0.834  -3.540   3.898  1.00  8.37           C\\nATOM    962  C   VAL A 125       0.854  -2.088   3.404  1.00  8.37           C\\nATOM    963  O   VAL A 125      -0.140  -1.583   2.878  1.00  8.37           O\\nATOM    964  CB  VAL A 125       1.074  -4.618   2.805  1.00  8.37           C\\nATOM    965  CG1 VAL A 125       0.096  -4.532   1.618  1.00  8.37           C\\nATOM    966  CG2 VAL A 125       2.533  -4.619   2.310  1.00  8.37           C\\nATOM    967  N   TYR A 126       2.004  -1.430   3.579  1.00 11.67           N\\nATOM    968  CA  TYR A 126       2.277  -0.056   3.182  1.00 11.67           C\\nATOM    969  C   TYR A 126       3.732   0.092   2.757  1.00 11.67           C\\nATOM    970  O   TYR A 126       4.598  -0.665   3.192  1.00 11.67           O\\nATOM    971  CB  TYR A 126       1.837   0.947   4.268  1.00 11.67           C\\nATOM    972  CG  TYR A 126       2.504   0.765   5.623  1.00 11.67           C\\nATOM    973  CD1 TYR A 126       3.800   1.269   5.865  1.00 11.67           C\\nATOM    974  CD2 TYR A 126       1.821   0.074   6.645  1.00 11.67           C\\nATOM    975  CE1 TYR A 126       4.415   1.057   7.113  1.00 11.67           C\\nATOM    976  CE2 TYR A 126       2.429  -0.114   7.899  1.00 11.67           C\\nATOM    977  CZ  TYR A 126       3.732   0.367   8.129  1.00 11.67           C\\nATOM    978  OH  TYR A 126       4.333   0.173   9.336  1.00 11.67           O\\nATOM    979  N   GLN A 127       3.991   1.104   1.926  1.00 17.26           N\\nATOM    980  CA  GLN A 127       5.314   1.449   1.440  1.00 17.26           C\\nATOM    981  C   GLN A 127       5.960   2.470   2.389  1.00 17.26           C\\nATOM    982  O   GLN A 127       5.291   3.350   2.932  1.00 17.26           O\\nATOM    983  CB  GLN A 127       5.119   2.066   0.042  1.00 17.26           C\\nATOM    984  CG  GLN A 127       6.382   2.176  -0.827  1.00 17.26           C\\nATOM    985  CD  GLN A 127       7.000   0.832  -1.224  1.00 17.26           C\\nATOM    986  OE1 GLN A 127       6.513  -0.240  -0.877  1.00 17.26           O\\nATOM    987  NE2 GLN A 127       8.097   0.894  -1.970  1.00 17.26           N\\nATOM    988  N   CYS A 128       7.275   2.328   2.568  1.00 22.83           N\\nATOM    989  CA  CYS A 128       8.153   3.134   3.406  1.00 22.83           C\\nATOM    990  C   CYS A 128       9.385   3.564   2.607  1.00 22.83           C\\nATOM    991  O   CYS A 128       9.604   3.097   1.489  1.00 22.83           O\\nATOM    992  CB  CYS A 128       8.544   2.316   4.663  1.00 22.83           C\\nATOM    993  SG  CYS A 128       7.552   2.830   6.085  1.00 22.83           S\\nATOM    994  N   ALA A 129      10.211   4.415   3.227  1.00 17.43           N\\nATOM    995  CA  ALA A 129      11.589   4.678   2.843  1.00 17.43           C\\nATOM    996  C   ALA A 129      12.471   4.735   4.082  1.00 17.43           C\\nATOM    997  O   ALA A 129      12.045   5.203   5.140  1.00 17.43           O\\nATOM    998  CB  ALA A 129      11.716   5.925   1.955  1.00 17.43           C\\nATOM    999  N   MET A 130      13.731   4.308   3.924  1.00 15.37           N\\nATOM   1000  CA  MET A 130      14.794   4.584   4.875  1.00 15.37           C\\nATOM   1001  C   MET A 130      15.136   6.064   4.708  1.00 15.37           C\\nATOM   1002  O   MET A 130      15.616   6.478   3.656  1.00 15.37           O\\nATOM   1003  CB  MET A 130      16.006   3.685   4.555  1.00 15.37           C\\nATOM   1004  CG  MET A 130      17.136   3.754   5.601  1.00 15.37           C\\nATOM   1005  SD  MET A 130      16.657   3.409   7.320  1.00 15.37           S\\nATOM   1006  CE  MET A 130      15.890   1.776   7.150  1.00 15.37           C\\nATOM   1007  N   ARG A 131      14.820   6.865   5.732  1.00 14.69           N\\nATOM   1008  CA  ARG A 131      15.026   8.308   5.724  1.00 14.69           C\\nATOM   1009  C   ARG A 131      16.534   8.608   5.866  1.00 14.69           C\\nATOM   1010  O   ARG A 131      17.247   7.792   6.449  1.00 14.69           O\\nATOM   1011  CB  ARG A 131      14.271   8.925   6.920  1.00 14.69           C\\nATOM   1012  CG  ARG A 131      12.741   8.695   6.947  1.00 14.69           C\\nATOM   1013  CD  ARG A 131      11.989   9.074   5.665  1.00 14.69           C\\nATOM   1014  NE  ARG A 131      12.349  10.420   5.219  1.00 14.69           N\\nATOM   1015  CZ  ARG A 131      12.025  10.986   4.052  1.00 14.69           C\\nATOM   1016  NH1 ARG A 131      11.202  10.391   3.182  1.00 14.69           N\\nATOM   1017  NH2 ARG A 131      12.540  12.181   3.757  1.00 14.69           N\\nATOM   1018  N   PRO A 132      17.047   9.742   5.341  1.00 16.36           N\\nATOM   1019  CA  PRO A 132      18.433  10.214   5.536  1.00 16.36           C\\nATOM   1020  C   PRO A 132      18.982  10.225   6.978  1.00 16.36           C\\nATOM   1021  O   PRO A 132      20.175   9.995   7.163  1.00 16.36           O\\nATOM   1022  CB  PRO A 132      18.444  11.622   4.943  1.00 16.36           C\\nATOM   1023  CG  PRO A 132      17.400  11.552   3.845  1.00 16.36           C\\nATOM   1024  CD  PRO A 132      16.341  10.600   4.390  1.00 16.36           C\\nATOM   1025  N   ASN A 133      18.124  10.436   7.992  1.00 15.55           N\\nATOM   1026  CA  ASN A 133      18.475  10.347   9.417  1.00 15.55           C\\nATOM   1027  C   ASN A 133      18.382   8.908   9.980  1.00 15.55           C\\nATOM   1028  O   ASN A 133      18.618   8.700  11.168  1.00 15.55           O\\nATOM   1029  CB  ASN A 133      17.577  11.308  10.245  1.00 15.55           C\\nATOM   1030  CG  ASN A 133      16.060  11.090  10.172  1.00 15.55           C\\nATOM   1031  OD1 ASN A 133      15.572  10.141   9.566  1.00 15.55           O\\nATOM   1032  ND2 ASN A 133      15.301  11.978  10.811  1.00 15.55           N\\nATOM   1033  N   PHE A 134      18.025   7.922   9.141  1.00 14.68           N\\nATOM   1034  CA  PHE A 134      17.982   6.480   9.400  1.00 14.68           C\\nATOM   1035  C   PHE A 134      16.751   5.999  10.179  1.00 14.68           C\\nATOM   1036  O   PHE A 134      16.685   4.830  10.561  1.00 14.68           O\\nATOM   1037  CB  PHE A 134      19.306   5.931   9.944  1.00 14.68           C\\nATOM   1038  CG  PHE A 134      20.466   5.972   8.976  1.00 14.68           C\\nATOM   1039  CD1 PHE A 134      20.603   4.957   8.007  1.00 14.68           C\\nATOM   1040  CD2 PHE A 134      21.422   7.005   9.056  1.00 14.68           C\\nATOM   1041  CE1 PHE A 134      21.703   4.968   7.132  1.00 14.68           C\\nATOM   1042  CE2 PHE A 134      22.517   7.019   8.172  1.00 14.68           C\\nATOM   1043  CZ  PHE A 134      22.661   5.997   7.214  1.00 14.68           C\\nATOM   1044  N   THR A 135      15.759   6.878  10.382  1.00 15.17           N\\nATOM   1045  CA  THR A 135      14.430   6.504  10.856  1.00 15.17           C\\nATOM   1046  C   THR A 135      13.589   6.029   9.656  1.00 15.17           C\\nATOM   1047  O   THR A 135      14.022   6.097   8.504  1.00 15.17           O\\nATOM   1048  CB  THR A 135      13.749   7.723  11.538  1.00 15.17           C\\nATOM   1049  OG1 THR A 135      13.374   8.734  10.622  1.00 15.17           O\\nATOM   1050  CG2 THR A 135      14.582   8.339  12.669  1.00 15.17           C\\nATOM   1051  N   ILE A 136      12.368   5.563   9.941  1.00 16.72           N\\nATOM   1052  CA  ILE A 136      11.315   5.276   8.973  1.00 16.72           C\\nATOM   1053  C   ILE A 136      10.016   5.891   9.514  1.00 16.72           C\\nATOM   1054  O   ILE A 136       9.805   5.941  10.726  1.00 16.72           O\\nATOM   1055  CB  ILE A 136      11.277   3.774   8.542  1.00 16.72           C\\nATOM   1056  CG1 ILE A 136      10.443   2.839   9.450  1.00 16.72           C\\nATOM   1057  CG2 ILE A 136      12.694   3.205   8.312  1.00 16.72           C\\nATOM   1058  CD1 ILE A 136      10.153   1.476   8.805  1.00 16.72           C\\nATOM   1059  N   LYS A 137       9.149   6.367   8.614  1.00 17.02           N\\nATOM   1060  CA  LYS A 137       7.821   6.882   8.926  1.00 17.02           C\\nATOM   1061  C   LYS A 137       6.820   5.768   8.603  1.00 17.02           C\\nATOM   1062  O   LYS A 137       6.217   5.742   7.530  1.00 17.02           O\\nATOM   1063  CB  LYS A 137       7.555   8.127   8.057  1.00 17.02           C\\nATOM   1064  CG  LYS A 137       8.271   9.380   8.581  1.00 17.02           C\\nATOM   1065  CD  LYS A 137       8.317  10.493   7.527  1.00 17.02           C\\nATOM   1066  CE  LYS A 137       8.804  11.826   8.103  1.00 17.02           C\\nATOM   1067  NZ  LYS A 137       9.236  12.742   7.032  1.00 17.02           N\\nATOM   1068  N   GLY A 138       6.677   4.827   9.546  1.00 17.71           N\\nATOM   1069  CA  GLY A 138       5.765   3.696   9.464  1.00 17.71           C\\nATOM   1070  C   GLY A 138       4.441   3.959  10.183  1.00 17.71           C\\nATOM   1071  O   GLY A 138       4.132   5.084  10.577  1.00 17.71           O\\nATOM   1072  N   SER A 139       3.671   2.882  10.361  1.00 18.24           N\\nATOM   1073  CA  SER A 139       2.434   2.818  11.126  1.00 18.24           C\\nATOM   1074  C   SER A 139       2.523   1.554  11.979  1.00 18.24           C\\nATOM   1075  O   SER A 139       2.361   0.445  11.468  1.00 18.24           O\\nATOM   1076  CB  SER A 139       1.218   2.893  10.182  1.00 18.24           C\\nATOM   1077  OG  SER A 139       0.004   2.824  10.903  1.00 18.24           O\\nATOM   1078  N   PHE A 140       2.850   1.741  13.262  1.00 19.57           N\\nATOM   1079  CA  PHE A 140       3.230   0.702  14.209  1.00 19.57           C\\nATOM   1080  C   PHE A 140       2.626   1.026  15.581  1.00 19.57           C\\nATOM   1081  O   PHE A 140       2.726   2.162  16.045  1.00 19.57           O\\nATOM   1082  CB  PHE A 140       4.767   0.615  14.258  1.00 19.57           C\\nATOM   1083  CG  PHE A 140       5.487   0.092  13.023  1.00 19.57           C\\nATOM   1084  CD1 PHE A 140       5.316  -1.233  12.572  1.00 19.57           C\\nATOM   1085  CD2 PHE A 140       6.398   0.928  12.353  1.00 19.57           C\\nATOM   1086  CE1 PHE A 140       6.098  -1.730  11.510  1.00 19.57           C\\nATOM   1087  CE2 PHE A 140       7.187   0.437  11.301  1.00 19.57           C\\nATOM   1088  CZ  PHE A 140       7.055  -0.902  10.892  1.00 19.57           C\\nATOM   1089  N   LEU A 141       2.022   0.020  16.227  1.00 23.03           N\\nATOM   1090  CA  LEU A 141       1.447   0.064  17.574  1.00 23.03           C\\nATOM   1091  C   LEU A 141       2.197  -0.939  18.472  1.00 23.03           C\\nATOM   1092  O   LEU A 141       3.190  -1.536  18.056  1.00 23.03           O\\nATOM   1093  CB  LEU A 141      -0.064  -0.284  17.506  1.00 23.03           C\\nATOM   1094  CG  LEU A 141      -0.912   0.546  16.514  1.00 23.03           C\\nATOM   1095  CD1 LEU A 141      -2.329  -0.035  16.399  1.00 23.03           C\\nATOM   1096  CD2 LEU A 141      -0.966   2.031  16.900  1.00 23.03           C\\nATOM   1097  N   ASN A 142       1.726  -1.144  19.713  1.00 32.08           N\\nATOM   1098  CA  ASN A 142       2.256  -2.158  20.633  1.00 32.08           C\\nATOM   1099  C   ASN A 142       1.937  -3.566  20.100  1.00 32.08           C\\nATOM   1100  O   ASN A 142       0.806  -3.840  19.696  1.00 32.08           O\\nATOM   1101  CB  ASN A 142       1.636  -2.001  22.043  1.00 32.08           C\\nATOM   1102  CG  ASN A 142       1.987  -0.698  22.772  1.00 32.08           C\\nATOM   1103  OD1 ASN A 142       2.886   0.041  22.380  1.00 32.08           O\\nATOM   1104  ND2 ASN A 142       1.254  -0.399  23.846  1.00 32.08           N\\nATOM   1105  N   GLY A 143       2.953  -4.439  20.082  1.00 21.33           N\\nATOM   1106  CA  GLY A 143       2.906  -5.784  19.514  1.00 21.33           C\\nATOM   1107  C   GLY A 143       3.389  -5.863  18.059  1.00 21.33           C\\nATOM   1108  O   GLY A 143       3.443  -6.965  17.515  1.00 21.33           O\\nATOM   1109  N   SER A 144       3.738  -4.728  17.425  1.00 13.26           N\\nATOM   1110  CA  SER A 144       4.145  -4.663  16.021  1.00 13.26           C\\nATOM   1111  C   SER A 144       5.629  -4.979  15.775  1.00 13.26           C\\nATOM   1112  O   SER A 144       6.025  -5.089  14.617  1.00 13.26           O\\nATOM   1113  CB  SER A 144       3.854  -3.260  15.464  1.00 13.26           C\\nATOM   1114  OG  SER A 144       4.781  -2.288  15.896  1.00 13.26           O\\nATOM   1115  N   CYS A 145       6.455  -5.117  16.825  1.00 15.56           N\\nATOM   1116  CA  CYS A 145       7.885  -5.400  16.701  1.00 15.56           C\\nATOM   1117  C   CYS A 145       8.116  -6.774  16.056  1.00 15.56           C\\nATOM   1118  O   CYS A 145       7.343  -7.706  16.265  1.00 15.56           O\\nATOM   1119  CB  CYS A 145       8.548  -5.324  18.082  1.00 15.56           C\\nATOM   1120  SG  CYS A 145       8.513  -3.608  18.656  1.00 15.56           S\\nATOM   1121  N   GLY A 146       9.139  -6.863  15.202  1.00 10.94           N\\nATOM   1122  CA  GLY A 146       9.413  -8.028  14.376  1.00 10.94           C\\nATOM   1123  C   GLY A 146       8.639  -8.066  13.055  1.00 10.94           C\\nATOM   1124  O   GLY A 146       8.818  -9.014  12.293  1.00 10.94           O\\nATOM   1125  N   SER A 147       7.816  -7.038  12.766  1.00  7.81           N\\nATOM   1126  CA  SER A 147       7.221  -6.790  11.452  1.00  7.81           C\\nATOM   1127  C   SER A 147       8.339  -6.320  10.519  1.00  7.81           C\\nATOM   1128  O   SER A 147       9.130  -5.456  10.903  1.00  7.81           O\\nATOM   1129  CB  SER A 147       6.185  -5.656  11.544  1.00  7.81           C\\nATOM   1130  OG  SER A 147       5.004  -6.111  12.163  1.00  7.81           O\\nATOM   1131  N   VAL A 148       8.424  -6.913   9.322  1.00  7.50           N\\nATOM   1132  CA  VAL A 148       9.573  -6.760   8.439  1.00  7.50           C\\nATOM   1133  C   VAL A 148       9.247  -5.903   7.219  1.00  7.50           C\\nATOM   1134  O   VAL A 148       8.130  -5.917   6.700  1.00  7.50           O\\nATOM   1135  CB  VAL A 148      10.149  -8.144   8.042  1.00  7.50           C\\nATOM   1136  CG1 VAL A 148      10.593  -8.920   9.291  1.00  7.50           C\\nATOM   1137  CG2 VAL A 148       9.242  -9.008   7.148  1.00  7.50           C\\nATOM   1138  N   GLY A 149      10.285  -5.192   6.763  1.00  8.85           N\\nATOM   1139  CA  GLY A 149      10.343  -4.489   5.500  1.00  8.85           C\\nATOM   1140  C   GLY A 149      11.108  -5.314   4.469  1.00  8.85           C\\nATOM   1141  O   GLY A 149      12.052  -6.028   4.815  1.00  8.85           O\\nATOM   1142  N   PHE A 150      10.679  -5.220   3.205  1.00  8.53           N\\nATOM   1143  CA  PHE A 150      11.143  -6.058   2.108  1.00  8.53           C\\nATOM   1144  C   PHE A 150      10.942  -5.396   0.742  1.00  8.53           C\\nATOM   1145  O   PHE A 150      10.061  -4.558   0.555  1.00  8.53           O\\nATOM   1146  CB  PHE A 150      10.485  -7.456   2.189  1.00  8.53           C\\nATOM   1147  CG  PHE A 150       8.977  -7.445   1.985  1.00  8.53           C\\nATOM   1148  CD1 PHE A 150       8.120  -7.166   3.071  1.00  8.53           C\\nATOM   1149  CD2 PHE A 150       8.426  -7.674   0.706  1.00  8.53           C\\nATOM   1150  CE1 PHE A 150       6.729  -7.084   2.873  1.00  8.53           C\\nATOM   1151  CE2 PHE A 150       7.033  -7.603   0.511  1.00  8.53           C\\nATOM   1152  CZ  PHE A 150       6.184  -7.300   1.593  1.00  8.53           C\\nATOM   1153  N   ASN A 151      11.727  -5.870  -0.230  1.00  9.22           N\\nATOM   1154  CA  ASN A 151      11.552  -5.686  -1.667  1.00  9.22           C\\nATOM   1155  C   ASN A 151      11.434  -7.073  -2.315  1.00  9.22           C\\nATOM   1156  O   ASN A 151      11.926  -8.050  -1.751  1.00  9.22           O\\nATOM   1157  CB  ASN A 151      12.753  -4.908  -2.243  1.00  9.22           C\\nATOM   1158  CG  ASN A 151      12.757  -3.425  -1.856  1.00  9.22           C\\nATOM   1159  OD1 ASN A 151      12.128  -2.614  -2.524  1.00  9.22           O\\nATOM   1160  ND2 ASN A 151      13.479  -3.045  -0.804  1.00  9.22           N\\nATOM   1161  N   ILE A 152      10.808  -7.164  -3.501  1.00 11.44           N\\nATOM   1162  CA  ILE A 152      10.706  -8.401  -4.284  1.00 11.44           C\\nATOM   1163  C   ILE A 152      11.246  -8.144  -5.697  1.00 11.44           C\\nATOM   1164  O   ILE A 152      10.775  -7.242  -6.388  1.00 11.44           O\\nATOM   1165  CB  ILE A 152       9.268  -9.005  -4.275  1.00 11.44           C\\nATOM   1166  CG1 ILE A 152       8.688  -9.126  -2.844  1.00 11.44           C\\nATOM   1167  CG2 ILE A 152       9.234 -10.369  -5.006  1.00 11.44           C\\nATOM   1168  CD1 ILE A 152       7.296  -9.771  -2.758  1.00 11.44           C\\nATOM   1169  N   ASP A 153      12.198  -8.984  -6.132  0.24 13.77           N\\nATOM   1170  CA  ASP A 153      12.700  -9.077  -7.500  0.24 13.77           C\\nATOM   1171  C   ASP A 153      12.194 -10.419  -8.049  0.24 13.77           C\\nATOM   1172  O   ASP A 153      12.834 -11.453  -7.852  0.24 13.77           O\\nATOM   1173  CB  ASP A 153      14.247  -8.973  -7.465  0.24 13.77           C\\nATOM   1174  CG  ASP A 153      14.991  -9.163  -8.796  0.24 13.77           C\\nATOM   1175  OD1 ASP A 153      14.330  -9.220  -9.856  0.24 13.77           O\\nATOM   1176  OD2 ASP A 153      16.238  -9.229  -8.717  0.24 13.77           O\\nATOM   1177  N   TYR A 154      11.004 -10.384  -8.675  0.24 16.78           N\\nATOM   1178  CA  TYR A 154      10.279 -11.497  -9.293  0.24 16.78           C\\nATOM   1179  C   TYR A 154       9.726 -12.470  -8.237  0.24 16.78           C\\nATOM   1180  O   TYR A 154       8.555 -12.381  -7.868  0.24 16.78           O\\nATOM   1181  CB  TYR A 154      11.077 -12.168 -10.442  0.24 16.78           C\\nATOM   1182  CG  TYR A 154      10.389 -13.367 -11.079  0.24 16.78           C\\nATOM   1183  CD1 TYR A 154       9.090 -13.245 -11.617  0.24 16.78           C\\nATOM   1184  CD2 TYR A 154      11.032 -14.622 -11.091  0.24 16.78           C\\nATOM   1185  CE1 TYR A 154       8.429 -14.376 -12.133  0.24 16.78           C\\nATOM   1186  CE2 TYR A 154      10.371 -15.754 -11.605  0.24 16.78           C\\nATOM   1187  CZ  TYR A 154       9.066 -15.632 -12.120  0.24 16.78           C\\nATOM   1188  OH  TYR A 154       8.418 -16.730 -12.605  0.24 16.78           O\\nATOM   1189  N   ASP A 155      10.584 -13.387  -7.762  0.24 15.30           N\\nATOM   1190  CA  ASP A 155      10.281 -14.449  -6.806  0.24 15.30           C\\nATOM   1191  C   ASP A 155      11.219 -14.424  -5.586  0.24 15.30           C\\nATOM   1192  O   ASP A 155      10.946 -15.131  -4.617  0.24 15.30           O\\nATOM   1193  CB  ASP A 155      10.227 -15.822  -7.530  0.24 15.30           C\\nATOM   1194  CG  ASP A 155      11.537 -16.362  -8.125  0.24 15.30           C\\nATOM   1195  OD1 ASP A 155      12.549 -15.627  -8.139  0.24 15.30           O\\nATOM   1196  OD2 ASP A 155      11.504 -17.531  -8.567  0.24 15.30           O\\nATOM   1197  N   CYS A 156      12.292 -13.614  -5.611  1.00 10.59           N\\nATOM   1198  CA  CYS A 156      13.244 -13.464  -4.516  1.00 10.59           C\\nATOM   1199  C   CYS A 156      12.810 -12.274  -3.649  1.00 10.59           C\\nATOM   1200  O   CYS A 156      12.754 -11.148  -4.139  1.00 10.59           O\\nATOM   1201  CB  CYS A 156      14.640 -13.187  -5.114  1.00 10.59           C\\nATOM   1202  SG  CYS A 156      15.911 -13.215  -3.812  1.00 10.59           S\\nATOM   1203  N   VAL A 157      12.529 -12.525  -2.360  1.00  7.42           N\\nATOM   1204  CA  VAL A 157      12.189 -11.511  -1.366  1.00  7.42           C\\nATOM   1205  C   VAL A 157      13.486 -11.107  -0.645  1.00  7.42           C\\nATOM   1206  O   VAL A 157      14.010 -11.854   0.180  1.00  7.42           O\\nATOM   1207  CB  VAL A 157      11.188 -12.097  -0.331  1.00  7.42           C\\nATOM   1208  CG1 VAL A 157      10.755 -11.036   0.698  1.00  7.42           C\\nATOM   1209  CG2 VAL A 157       9.936 -12.700  -0.993  1.00  7.42           C\\nATOM   1210  N   SER A 158      13.979  -9.895  -0.923  1.00  8.82           N\\nATOM   1211  CA  SER A 158      15.084  -9.287  -0.195  1.00  8.82           C\\nATOM   1212  C   SER A 158      14.501  -8.578   1.033  1.00  8.82           C\\nATOM   1213  O   SER A 158      13.880  -7.526   0.902  1.00  8.82           O\\nATOM   1214  CB  SER A 158      15.775  -8.249  -1.098  1.00  8.82           C\\nATOM   1215  OG  SER A 158      16.727  -8.906  -1.904  1.00  8.82           O\\nATOM   1216  N   PHE A 159      14.705  -9.163   2.220  1.00  8.45           N\\nATOM   1217  CA  PHE A 159      14.320  -8.586   3.504  1.00  8.45           C\\nATOM   1218  C   PHE A 159      15.400  -7.586   3.915  1.00  8.45           C\\nATOM   1219  O   PHE A 159      16.591  -7.888   3.830  1.00  8.45           O\\nATOM   1220  CB  PHE A 159      14.260  -9.696   4.565  1.00  8.45           C\\nATOM   1221  CG  PHE A 159      13.137 -10.682   4.336  1.00  8.45           C\\nATOM   1222  CD1 PHE A 159      11.816 -10.321   4.666  1.00  8.45           C\\nATOM   1223  CD2 PHE A 159      13.400 -11.945   3.766  1.00  8.45           C\\nATOM   1224  CE1 PHE A 159      10.761 -11.221   4.430  1.00  8.45           C\\nATOM   1225  CE2 PHE A 159      12.342 -12.843   3.531  1.00  8.45           C\\nATOM   1226  CZ  PHE A 159      11.023 -12.486   3.870  1.00  8.45           C\\nATOM   1227  N   CYS A 160      14.969  -6.397   4.346  1.00 10.80           N\\nATOM   1228  CA  CYS A 160      15.840  -5.264   4.632  1.00 10.80           C\\nATOM   1229  C   CYS A 160      15.562  -4.567   5.950  1.00 10.80           C\\nATOM   1230  O   CYS A 160      16.405  -3.797   6.390  1.00 10.80           O\\nATOM   1231  CB  CYS A 160      15.885  -4.292   3.447  1.00 10.80           C\\nATOM   1232  SG  CYS A 160      14.262  -3.531   3.134  1.00 10.80           S\\nATOM   1233  N   TYR A 161      14.415  -4.821   6.583  1.00  8.81           N\\nATOM   1234  CA  TYR A 161      14.029  -4.173   7.821  1.00  8.81           C\\nATOM   1235  C   TYR A 161      13.348  -5.182   8.739  1.00  8.81           C\\nATOM   1236  O   TYR A 161      12.701  -6.118   8.276  1.00  8.81           O\\nATOM   1237  CB  TYR A 161      13.252  -2.877   7.523  1.00  8.81           C\\nATOM   1238  CG  TYR A 161      12.845  -2.045   8.723  1.00  8.81           C\\nATOM   1239  CD1 TYR A 161      13.743  -1.104   9.267  1.00  8.81           C\\nATOM   1240  CD2 TYR A 161      11.557  -2.191   9.278  1.00  8.81           C\\nATOM   1241  CE1 TYR A 161      13.348  -0.302  10.354  1.00  8.81           C\\nATOM   1242  CE2 TYR A 161      11.168  -1.395  10.371  1.00  8.81           C\\nATOM   1243  CZ  TYR A 161      12.062  -0.450  10.910  1.00  8.81           C\\nATOM   1244  OH  TYR A 161      11.679   0.335  11.957  1.00  8.81           O\\nATOM   1245  N   MET A 162      13.500  -4.959  10.044  1.00 10.10           N\\nATOM   1246  CA  MET A 162      12.794  -5.602  11.131  1.00 10.10           C\\nATOM   1247  C   MET A 162      12.568  -4.520  12.178  1.00 10.10           C\\nATOM   1248  O   MET A 162      13.528  -3.931  12.676  1.00 10.10           O\\nATOM   1249  CB  MET A 162      13.560  -6.833  11.645  1.00 10.10           C\\nATOM   1250  CG  MET A 162      12.795  -7.552  12.763  1.00 10.10           C\\nATOM   1251  SD  MET A 162      13.475  -9.157  13.239  1.00 10.10           S\\nATOM   1252  CE  MET A 162      12.582 -10.186  12.048  1.00 10.10           C\\nATOM   1253  N   HIS A 163      11.296  -4.257  12.498  1.00 10.91           N\\nATOM   1254  CA  HIS A 163      10.884  -3.187  13.392  1.00 10.91           C\\nATOM   1255  C   HIS A 163      11.224  -3.521  14.850  1.00 10.91           C\\nATOM   1256  O   HIS A 163      10.990  -4.642  15.299  1.00 10.91           O\\nATOM   1257  CB  HIS A 163       9.368  -2.984  13.242  1.00 10.91           C\\nATOM   1258  CG  HIS A 163       8.893  -1.781  14.004  1.00 10.91           C\\nATOM   1259  ND1 HIS A 163       9.439  -0.510  13.796  1.00 10.91           N\\nATOM   1260  CD2 HIS A 163       7.955  -1.715  15.007  1.00 10.91           C\\nATOM   1261  CE1 HIS A 163       8.834   0.258  14.687  1.00 10.91           C\\nATOM   1262  NE2 HIS A 163       7.935  -0.397  15.412  1.00 10.91           N\\nATOM   1263  N   HIS A 164      11.792  -2.540  15.567  1.00 13.39           N\\nATOM   1264  CA  HIS A 164      12.245  -2.674  16.949  1.00 13.39           C\\nATOM   1265  C   HIS A 164      11.671  -1.588  17.861  1.00 13.39           C\\nATOM   1266  O   HIS A 164      11.212  -1.894  18.958  1.00 13.39           O\\nATOM   1267  CB  HIS A 164      13.785  -2.671  17.032  1.00 13.39           C\\nATOM   1268  CG  HIS A 164      14.489  -3.944  16.642  1.00 13.39           C\\nATOM   1269  ND1 HIS A 164      14.266  -4.612  15.448  1.00 13.39           N\\nATOM   1270  CD2 HIS A 164      15.469  -4.646  17.314  1.00 13.39           C\\nATOM   1271  CE1 HIS A 164      15.005  -5.717  15.475  1.00 13.39           C\\nATOM   1272  NE2 HIS A 164      15.755  -5.773  16.570  1.00 13.39           N\\nATOM   1273  N   MET A 165      11.762  -0.319  17.444  1.00 48.84           N\\nATOM   1274  CA  MET A 165      11.592   0.842  18.310  1.00 48.84           C\\nATOM   1275  C   MET A 165      10.695   1.888  17.661  1.00 48.84           C\\nATOM   1276  O   MET A 165      10.778   2.122  16.458  1.00 48.84           O\\nATOM   1277  CB  MET A 165      12.970   1.448  18.659  1.00 48.84           C\\nATOM   1278  CG  MET A 165      13.913   0.522  19.440  1.00 48.84           C\\nATOM   1279  SD  MET A 165      13.321  -0.014  21.069  1.00 48.84           S\\nATOM   1280  CE  MET A 165      13.514   1.530  21.993  1.00 48.84           C\\nATOM   1281  N   GLU A 166       9.905   2.571  18.495  1.00 23.02           N\\nATOM   1282  CA  GLU A 166       9.333   3.876  18.208  1.00 23.02           C\\nATOM   1283  C   GLU A 166      10.050   4.870  19.127  1.00 23.02           C\\nATOM   1284  O   GLU A 166      10.109   4.668  20.341  1.00 23.02           O\\nATOM   1285  CB  GLU A 166       7.818   3.839  18.482  1.00 23.02           C\\nATOM   1286  CG  GLU A 166       7.095   5.162  18.164  1.00 23.02           C\\nATOM   1287  CD  GLU A 166       5.579   4.995  18.208  1.00 23.02           C\\nATOM   1288  OE1 GLU A 166       5.042   4.998  19.336  1.00 23.02           O\\nATOM   1289  OE2 GLU A 166       4.979   4.850  17.122  1.00 23.02           O\\nATOM   1290  N   LEU A 167      10.595   5.945  18.543  1.00 21.17           N\\nATOM   1291  CA  LEU A 167      11.238   7.042  19.263  1.00 21.17           C\\nATOM   1292  C   LEU A 167      10.145   7.975  19.834  1.00 21.17           C\\nATOM   1293  O   LEU A 167       9.040   7.989  19.291  1.00 21.17           O\\nATOM   1294  CB  LEU A 167      12.113   7.855  18.277  1.00 21.17           C\\nATOM   1295  CG  LEU A 167      13.145   7.046  17.456  1.00 21.17           C\\nATOM   1296  CD1 LEU A 167      14.057   7.993  16.662  1.00 21.17           C\\nATOM   1297  CD2 LEU A 167      14.000   6.094  18.309  1.00 21.17           C\\nATOM   1298  N   PRO A 168      10.418   8.755  20.907  1.00 22.23           N\\nATOM   1299  CA  PRO A 168       9.498   9.750  21.497  1.00 22.23           C\\nATOM   1300  C   PRO A 168       8.734  10.703  20.546  1.00 22.23           C\\nATOM   1301  O   PRO A 168       7.626  11.117  20.886  1.00 22.23           O\\nATOM   1302  CB  PRO A 168      10.364  10.551  22.478  1.00 22.23           C\\nATOM   1303  CG  PRO A 168      11.450   9.580  22.908  1.00 22.23           C\\nATOM   1304  CD  PRO A 168      11.644   8.655  21.708  1.00 22.23           C\\nATOM   1305  N   THR A 169       9.290  11.026  19.363  1.00 21.93           N\\nATOM   1306  CA  THR A 169       8.657  11.851  18.322  1.00 21.93           C\\nATOM   1307  C   THR A 169       7.712  11.057  17.389  1.00 21.93           C\\nATOM   1308  O   THR A 169       7.168  11.636  16.449  1.00 21.93           O\\nATOM   1309  CB  THR A 169       9.765  12.530  17.476  1.00 21.93           C\\nATOM   1310  OG1 THR A 169      10.598  11.573  16.847  1.00 21.93           O\\nATOM   1311  CG2 THR A 169      10.607  13.536  18.274  1.00 21.93           C\\nATOM   1312  N   GLY A 170       7.498   9.755  17.634  1.00 18.98           N\\nATOM   1313  CA  GLY A 170       6.563   8.899  16.906  1.00 18.98           C\\nATOM   1314  C   GLY A 170       7.089   8.336  15.579  1.00 18.98           C\\nATOM   1315  O   GLY A 170       6.323   7.683  14.871  1.00 18.98           O\\nATOM   1316  N   VAL A 171       8.369   8.573  15.244  1.00 17.99           N\\nATOM   1317  CA  VAL A 171       9.075   7.957  14.116  1.00 17.99           C\\nATOM   1318  C   VAL A 171       9.742   6.663  14.607  1.00 17.99           C\\nATOM   1319  O   VAL A 171       9.870   6.429  15.809  1.00 17.99           O\\nATOM   1320  CB  VAL A 171      10.129   8.934  13.526  1.00 17.99           C\\nATOM   1321  CG1 VAL A 171       9.483  10.260  13.090  1.00 17.99           C\\nATOM   1322  CG2 VAL A 171      11.353   9.184  14.423  1.00 17.99           C\\nATOM   1323  N   HIS A 172      10.131   5.807  13.657  1.00 15.14           N\\nATOM   1324  CA  HIS A 172      10.435   4.403  13.896  1.00 15.14           C\\nATOM   1325  C   HIS A 172      11.890   4.084  13.583  1.00 15.14           C\\nATOM   1326  O   HIS A 172      12.500   4.704  12.716  1.00 15.14           O\\nATOM   1327  CB  HIS A 172       9.450   3.567  13.066  1.00 15.14           C\\nATOM   1328  CG  HIS A 172       8.028   3.858  13.473  1.00 15.14           C\\nATOM   1329  ND1 HIS A 172       7.123   4.528  12.642  1.00 15.14           N\\nATOM   1330  CD2 HIS A 172       7.425   3.623  14.690  1.00 15.14           C\\nATOM   1331  CE1 HIS A 172       6.033   4.679  13.383  1.00 15.14           C\\nATOM   1332  NE2 HIS A 172       6.158   4.164  14.604  1.00 15.14           N\\nATOM   1333  N   ALA A 173      12.429   3.102  14.313  1.00 15.43           N\\nATOM   1334  CA  ALA A 173      13.792   2.614  14.214  1.00 15.43           C\\nATOM   1335  C   ALA A 173      13.817   1.089  14.305  1.00 15.43           C\\nATOM   1336  O   ALA A 173      13.007   0.476  15.001  1.00 15.43           O\\nATOM   1337  CB  ALA A 173      14.701   3.328  15.227  1.00 15.43           C\\nATOM   1338  N   GLY A 174      14.744   0.477  13.567  1.00 12.46           N\\nATOM   1339  CA  GLY A 174      14.858  -0.961  13.437  1.00 12.46           C\\nATOM   1340  C   GLY A 174      16.166  -1.353  12.768  1.00 12.46           C\\nATOM   1341  O   GLY A 174      17.011  -0.512  12.460  1.00 12.46           O\\nATOM   1342  N   THR A 175      16.322  -2.665  12.588  1.00 10.67           N\\nATOM   1343  CA  THR A 175      17.532  -3.325  12.126  1.00 10.67           C\\nATOM   1344  C   THR A 175      17.330  -3.908  10.730  1.00 10.67           C\\nATOM   1345  O   THR A 175      16.199  -4.095  10.301  1.00 10.67           O\\nATOM   1346  CB  THR A 175      17.863  -4.459  13.131  1.00 10.67           C\\nATOM   1347  OG1 THR A 175      16.908  -5.504  13.098  1.00 10.67           O\\nATOM   1348  CG2 THR A 175      18.014  -4.001  14.585  1.00 10.67           C\\nATOM   1349  N   ASP A 176      18.433  -4.314  10.085  1.00 11.44           N\\nATOM   1350  CA  ASP A 176      18.430  -5.336   9.034  1.00 11.44           C\\nATOM   1351  C   ASP A 176      18.237  -6.725   9.694  1.00 11.44           C\\nATOM   1352  O   ASP A 176      18.176  -6.843  10.919  1.00 11.44           O\\nATOM   1353  CB  ASP A 176      19.761  -5.245   8.238  1.00 11.44           C\\nATOM   1354  CG  ASP A 176      21.080  -5.378   9.011  1.00 11.44           C\\nATOM   1355  OD1 ASP A 176      21.101  -6.019  10.083  1.00 11.44           O\\nATOM   1356  OD2 ASP A 176      22.070  -4.817   8.499  1.00 11.44           O\\nATOM   1357  N   LEU A 177      18.133  -7.796   8.896  1.00 10.96           N\\nATOM   1358  CA  LEU A 177      17.892  -9.151   9.405  1.00 10.96           C\\nATOM   1359  C   LEU A 177      19.146  -9.831   9.997  1.00 10.96           C\\nATOM   1360  O   LEU A 177      19.022 -10.924  10.548  1.00 10.96           O\\nATOM   1361  CB  LEU A 177      17.268 -10.013   8.285  1.00 10.96           C\\nATOM   1362  CG  LEU A 177      15.725  -9.941   8.206  1.00 10.96           C\\nATOM   1363  CD1 LEU A 177      15.056 -10.667   9.386  1.00 10.96           C\\nATOM   1364  CD2 LEU A 177      15.176  -8.513   8.037  1.00 10.96           C\\nATOM   1365  N   GLU A 178      20.317  -9.168   9.973  1.00 12.96           N\\nATOM   1366  CA  GLU A 178      21.511  -9.535  10.742  1.00 12.96           C\\nATOM   1367  C   GLU A 178      21.524  -8.850  12.127  1.00 12.96           C\\nATOM   1368  O   GLU A 178      22.421  -9.115  12.924  1.00 12.96           O\\nATOM   1369  CB  GLU A 178      22.763  -9.107   9.951  1.00 12.96           C\\nATOM   1370  CG  GLU A 178      22.906  -9.892   8.630  1.00 12.96           C\\nATOM   1371  CD  GLU A 178      24.146  -9.532   7.811  1.00 12.96           C\\nATOM   1372  OE1 GLU A 178      24.913  -8.645   8.245  1.00 12.96           O\\nATOM   1373  OE2 GLU A 178      24.299 -10.154   6.738  1.00 12.96           O\\nATOM   1374  N   GLY A 179      20.519  -8.014  12.434  1.00 12.17           N\\nATOM   1375  CA  GLY A 179      20.294  -7.385  13.729  1.00 12.17           C\\nATOM   1376  C   GLY A 179      21.047  -6.079  13.964  1.00 12.17           C\\nATOM   1377  O   GLY A 179      21.024  -5.583  15.086  1.00 12.17           O\\nATOM   1378  N   ASN A 180      21.703  -5.507  12.947  1.00 11.39           N\\nATOM   1379  CA  ASN A 180      22.379  -4.218  13.062  1.00 11.39           C\\nATOM   1380  C   ASN A 180      21.364  -3.113  12.757  1.00 11.39           C\\nATOM   1381  O   ASN A 180      20.782  -3.094  11.672  1.00 11.39           O\\nATOM   1382  CB  ASN A 180      23.534  -4.141  12.041  1.00 11.39           C\\nATOM   1383  CG  ASN A 180      24.645  -5.137  12.371  1.00 11.39           C\\nATOM   1384  OD1 ASN A 180      24.792  -6.157  11.703  1.00 11.39           O\\nATOM   1385  ND2 ASN A 180      25.439  -4.841  13.402  1.00 11.39           N\\nATOM   1386  N   PHE A 181      21.164  -2.194  13.718  1.00 10.54           N\\nATOM   1387  CA  PHE A 181      20.304  -1.015  13.611  1.00 10.54           C\\nATOM   1388  C   PHE A 181      20.740  -0.106  12.462  1.00 10.54           C\\nATOM   1389  O   PHE A 181      21.933   0.135  12.271  1.00 10.54           O\\nATOM   1390  CB  PHE A 181      20.279  -0.217  14.933  1.00 10.54           C\\nATOM   1391  CG  PHE A 181      19.117  -0.553  15.851  1.00 10.54           C\\nATOM   1392  CD1 PHE A 181      19.267  -1.526  16.859  1.00 10.54           C\\nATOM   1393  CD2 PHE A 181      17.886   0.124  15.712  1.00 10.54           C\\nATOM   1394  CE1 PHE A 181      18.197  -1.818  17.725  1.00 10.54           C\\nATOM   1395  CE2 PHE A 181      16.818  -0.161  16.584  1.00 10.54           C\\nATOM   1396  CZ  PHE A 181      16.976  -1.132  17.591  1.00 10.54           C\\nATOM   1397  N   TYR A 182      19.756   0.440  11.740  1.00 13.46           N\\nATOM   1398  CA  TYR A 182      19.958   1.569  10.844  1.00 13.46           C\\nATOM   1399  C   TYR A 182      20.118   2.816  11.710  1.00 13.46           C\\nATOM   1400  O   TYR A 182      19.322   3.057  12.616  1.00 13.46           O\\nATOM   1401  CB  TYR A 182      18.737   1.718   9.926  1.00 13.46           C\\nATOM   1402  CG  TYR A 182      18.685   0.630   8.883  1.00 13.46           C\\nATOM   1403  CD1 TYR A 182      19.591   0.662   7.806  1.00 13.46           C\\nATOM   1404  CD2 TYR A 182      17.771  -0.434   9.010  1.00 13.46           C\\nATOM   1405  CE1 TYR A 182      19.589  -0.376   6.862  1.00 13.46           C\\nATOM   1406  CE2 TYR A 182      17.758  -1.461   8.053  1.00 13.46           C\\nATOM   1407  CZ  TYR A 182      18.666  -1.429   6.977  1.00 13.46           C\\nATOM   1408  OH  TYR A 182      18.665  -2.413   6.040  1.00 13.46           O\\nATOM   1409  N   GLY A 183      21.181   3.583  11.449  1.00 22.94           N\\nATOM   1410  CA  GLY A 183      21.528   4.774  12.204  1.00 22.94           C\\nATOM   1411  C   GLY A 183      22.156   4.466  13.567  1.00 22.94           C\\nATOM   1412  O   GLY A 183      22.610   3.344  13.798  1.00 22.94           O\\nATOM   1413  N   PRO A 184      22.198   5.468  14.470  1.00 14.16           N\\nATOM   1414  CA  PRO A 184      22.826   5.361  15.793  1.00 14.16           C\\nATOM   1415  C   PRO A 184      21.901   4.754  16.868  1.00 14.16           C\\nATOM   1416  O   PRO A 184      22.282   4.714  18.038  1.00 14.16           O\\nATOM   1417  CB  PRO A 184      23.142   6.823  16.145  1.00 14.16           C\\nATOM   1418  CG  PRO A 184      22.008   7.608  15.507  1.00 14.16           C\\nATOM   1419  CD  PRO A 184      21.709   6.831  14.229  1.00 14.16           C\\nATOM   1420  N   PHE A 185      20.687   4.317  16.491  1.00 16.87           N\\nATOM   1421  CA  PHE A 185      19.626   3.915  17.408  1.00 16.87           C\\nATOM   1422  C   PHE A 185      19.911   2.560  18.070  1.00 16.87           C\\nATOM   1423  O   PHE A 185      20.613   1.714  17.515  1.00 16.87           O\\nATOM   1424  CB  PHE A 185      18.288   3.873  16.641  1.00 16.87           C\\nATOM   1425  CG  PHE A 185      17.911   5.215  16.031  1.00 16.87           C\\nATOM   1426  CD1 PHE A 185      17.758   6.353  16.852  1.00 16.87           C\\nATOM   1427  CD2 PHE A 185      17.769   5.346  14.635  1.00 16.87           C\\nATOM   1428  CE1 PHE A 185      17.498   7.613  16.279  1.00 16.87           C\\nATOM   1429  CE2 PHE A 185      17.505   6.603  14.060  1.00 16.87           C\\nATOM   1430  CZ  PHE A 185      17.376   7.738  14.882  1.00 16.87           C\\nATOM   1431  N   VAL A 186      19.371   2.397  19.283  1.00 20.33           N\\nATOM   1432  CA  VAL A 186      19.539   1.249  20.167  1.00 20.33           C\\nATOM   1433  C   VAL A 186      18.166   0.747  20.628  1.00 20.33           C\\nATOM   1434  O   VAL A 186      17.203   1.514  20.687  1.00 20.33           O\\nATOM   1435  CB  VAL A 186      20.491   1.598  21.349  1.00 20.33           C\\nATOM   1436  CG1 VAL A 186      21.935   1.855  20.887  1.00 20.33           C\\nATOM   1437  CG2 VAL A 186      20.004   2.768  22.226  1.00 20.33           C\\nATOM   1438  N   ASP A 187      18.089  -0.542  20.993  1.00 14.42           N\\nATOM   1439  CA  ASP A 187      16.916  -1.185  21.591  1.00 14.42           C\\nATOM   1440  C   ASP A 187      16.943  -0.963  23.117  1.00 14.42           C\\nATOM   1441  O   ASP A 187      17.122  -1.884  23.912  1.00 14.42           O\\nATOM   1442  CB  ASP A 187      16.849  -2.690  21.212  1.00 14.42           C\\nATOM   1443  CG  ASP A 187      18.106  -3.556  21.366  1.00 14.42           C\\nATOM   1444  OD1 ASP A 187      19.152  -3.065  21.844  1.00 14.42           O\\nATOM   1445  OD2 ASP A 187      17.998  -4.738  20.981  1.00 14.42           O\\nATOM   1446  N   ARG A 188      16.752   0.301  23.508  1.00 19.45           N\\nATOM   1447  CA  ARG A 188      16.660   0.785  24.875  1.00 19.45           C\\nATOM   1448  C   ARG A 188      15.718   1.990  24.883  1.00 19.45           C\\nATOM   1449  O   ARG A 188      15.852   2.881  24.048  1.00 19.45           O\\nATOM   1450  CB  ARG A 188      18.071   1.063  25.443  1.00 19.45           C\\nATOM   1451  CG  ARG A 188      18.078   1.681  26.852  1.00 19.45           C\\nATOM   1452  CD  ARG A 188      19.489   1.984  27.366  1.00 19.45           C\\nATOM   1453  NE  ARG A 188      19.416   2.627  28.682  1.00 19.45           N\\nATOM   1454  CZ  ARG A 188      20.342   3.406  29.262  1.00 19.45           C\\nATOM   1455  NH1 ARG A 188      21.501   3.690  28.652  1.00 19.45           N\\nATOM   1456  NH2 ARG A 188      20.093   3.907  30.478  1.00 19.45           N\\nATOM   1457  N   GLN A 189      14.791   2.032  25.849  1.00 36.62           N\\nATOM   1458  CA  GLN A 189      13.893   3.147  26.122  1.00 36.62           C\\nATOM   1459  C   GLN A 189      14.699   4.239  26.851  1.00 36.62           C\\nATOM   1460  O   GLN A 189      14.831   4.229  28.075  1.00 36.62           O\\nATOM   1461  CB  GLN A 189      12.729   2.612  26.986  1.00 36.62           C\\nATOM   1462  CG  GLN A 189      11.667   3.672  27.350  1.00 36.62           C\\nATOM   1463  CD  GLN A 189      10.507   3.155  28.211  1.00 36.62           C\\nATOM   1464  OE1 GLN A 189       9.701   3.956  28.677  1.00 36.62           O\\nATOM   1465  NE2 GLN A 189      10.413   1.845  28.455  1.00 36.62           N\\nATOM   1466  N   THR A 190      15.267   5.156  26.058  1.00 32.27           N\\nATOM   1467  CA  THR A 190      16.133   6.258  26.469  1.00 32.27           C\\nATOM   1468  C   THR A 190      15.911   7.469  25.545  1.00 32.27           C\\nATOM   1469  O   THR A 190      15.077   7.425  24.637  1.00 32.27           O\\nATOM   1470  CB  THR A 190      17.599   5.756  26.629  1.00 32.27           C\\nATOM   1471  OG1 THR A 190      18.400   6.726  27.277  1.00 32.27           O\\nATOM   1472  CG2 THR A 190      18.295   5.386  25.305  1.00 32.27           C\\nATOM   1473  N   ALA A 191      16.671   8.552  25.774  1.00 94.66           N\\nATOM   1474  CA  ALA A 191      16.702   9.749  24.941  1.00 94.66           C\\nATOM   1475  C   ALA A 191      17.405   9.427  23.615  1.00 94.66           C\\nATOM   1476  O   ALA A 191      18.616   9.209  23.579  1.00 94.66           O\\nATOM   1477  CB  ALA A 191      17.469  10.848  25.693  1.00 94.66           C\\nATOM   1478  N   GLN A 192      16.607   9.365  22.541  1.00 28.38           N\\nATOM   1479  CA  GLN A 192      17.020   9.024  21.187  1.00 28.38           C\\nATOM   1480  C   GLN A 192      16.353  10.008  20.229  1.00 28.38           C\\nATOM   1481  O   GLN A 192      15.290   9.719  19.678  1.00 28.38           O\\nATOM   1482  CB  GLN A 192      16.668   7.559  20.867  1.00 28.38           C\\nATOM   1483  CG  GLN A 192      17.357   6.518  21.763  1.00 28.38           C\\nATOM   1484  CD  GLN A 192      17.358   5.150  21.093  1.00 28.38           C\\nATOM   1485  OE1 GLN A 192      18.128   4.921  20.168  1.00 28.38           O\\nATOM   1486  NE2 GLN A 192      16.511   4.233  21.551  1.00 28.38           N\\nATOM   1487  N   ALA A 193      16.974  11.182  20.051  1.00 15.62           N\\nATOM   1488  CA  ALA A 193      16.542  12.206  19.107  1.00 15.62           C\\nATOM   1489  C   ALA A 193      17.132  11.883  17.731  1.00 15.62           C\\nATOM   1490  O   ALA A 193      18.329  11.619  17.612  1.00 15.62           O\\nATOM   1491  CB  ALA A 193      17.108  13.559  19.565  1.00 15.62           C\\nATOM   1492  N   ALA A 194      16.287  11.924  16.693  1.00 18.22           N\\nATOM   1493  CA  ALA A 194      16.677  11.772  15.299  1.00 18.22           C\\nATOM   1494  C   ALA A 194      17.244  13.098  14.771  1.00 18.22           C\\nATOM   1495  O   ALA A 194      16.810  14.178  15.179  1.00 18.22           O\\nATOM   1496  CB  ALA A 194      15.423  11.421  14.485  1.00 18.22           C\\nATOM   1497  N   GLY A 195      18.194  12.998  13.830  1.00 19.71           N\\nATOM   1498  CA  GLY A 195      18.773  14.111  13.083  1.00 19.71           C\\nATOM   1499  C   GLY A 195      17.792  14.711  12.070  1.00 19.71           C\\nATOM   1500  O   GLY A 195      16.615  14.345  12.041  1.00 19.71           O\\nATOM   1501  N   THR A 196      18.281  15.636  11.228  1.00 22.64           N\\nATOM   1502  CA  THR A 196      17.486  16.343  10.220  1.00 22.64           C\\nATOM   1503  C   THR A 196      16.961  15.332   9.190  1.00 22.64           C\\nATOM   1504  O   THR A 196      17.754  14.663   8.523  1.00 22.64           O\\nATOM   1505  CB  THR A 196      18.362  17.411   9.516  1.00 22.64           C\\nATOM   1506  OG1 THR A 196      18.916  18.282  10.482  1.00 22.64           O\\nATOM   1507  CG2 THR A 196      17.575  18.294   8.527  1.00 22.64           C\\nATOM   1508  N   ASP A 197      15.627  15.233   9.070  1.00 28.51           N\\nATOM   1509  CA  ASP A 197      14.974  14.423   8.054  1.00 28.51           C\\nATOM   1510  C   ASP A 197      14.890  15.274   6.782  1.00 28.51           C\\nATOM   1511  O   ASP A 197      13.889  15.939   6.502  1.00 28.51           O\\nATOM   1512  CB  ASP A 197      13.572  13.978   8.532  1.00 28.51           C\\nATOM   1513  CG  ASP A 197      12.986  12.831   7.695  1.00 28.51           C\\nATOM   1514  OD1 ASP A 197      13.583  12.471   6.656  1.00 28.51           O\\nATOM   1515  OD2 ASP A 197      11.942  12.303   8.125  1.00 28.51           O\\nATOM   1516  N   THR A 198      15.994  15.246   6.027  1.00 23.01           N\\nATOM   1517  CA  THR A 198      16.144  15.887   4.733  1.00 23.01           C\\nATOM   1518  C   THR A 198      15.329  15.119   3.677  1.00 23.01           C\\nATOM   1519  O   THR A 198      15.007  13.942   3.853  1.00 23.01           O\\nATOM   1520  CB  THR A 198      17.650  15.931   4.365  1.00 23.01           C\\nATOM   1521  OG1 THR A 198      18.196  14.635   4.346  1.00 23.01           O\\nATOM   1522  CG2 THR A 198      18.505  16.789   5.308  1.00 23.01           C\\nATOM   1523  N   THR A 199      14.970  15.806   2.587  1.00 14.59           N\\nATOM   1524  CA  THR A 199      14.190  15.229   1.500  1.00 14.59           C\\nATOM   1525  C   THR A 199      15.157  14.540   0.520  1.00 14.59           C\\nATOM   1526  O   THR A 199      16.212  15.081   0.188  1.00 14.59           O\\nATOM   1527  CB  THR A 199      13.407  16.354   0.791  1.00 14.59           C\\nATOM   1528  OG1 THR A 199      12.498  16.932   1.708  1.00 14.59           O\\nATOM   1529  CG2 THR A 199      12.539  15.846  -0.366  1.00 14.59           C\\nATOM   1530  N   ILE A 200      14.784  13.335   0.067  1.00 12.39           N\\nATOM   1531  CA  ILE A 200      15.573  12.482  -0.820  1.00 12.39           C\\nATOM   1532  C   ILE A 200      15.343  13.001  -2.253  1.00 12.39           C\\nATOM   1533  O   ILE A 200      14.291  12.776  -2.847  1.00 12.39           O\\nATOM   1534  CB  ILE A 200      15.082  11.011  -0.673  1.00 12.39           C\\nATOM   1535  CG1 ILE A 200      15.086  10.498   0.792  1.00 12.39           C\\nATOM   1536  CG2 ILE A 200      15.921  10.061  -1.553  1.00 12.39           C\\nATOM   1537  CD1 ILE A 200      13.951   9.503   1.084  1.00 12.39           C\\nATOM   1538  N   THR A 201      16.321  13.747  -2.783  1.00  9.12           N\\nATOM   1539  CA  THR A 201      16.186  14.588  -3.976  1.00  9.12           C\\nATOM   1540  C   THR A 201      16.018  13.770  -5.264  1.00  9.12           C\\nATOM   1541  O   THR A 201      15.186  14.128  -6.096  1.00  9.12           O\\nATOM   1542  CB  THR A 201      17.459  15.444  -4.090  1.00  9.12           C\\nATOM   1543  OG1 THR A 201      17.587  16.198  -2.909  1.00  9.12           O\\nATOM   1544  CG2 THR A 201      17.484  16.420  -5.271  1.00  9.12           C\\nATOM   1545  N   VAL A 202      16.790  12.681  -5.421  1.00 10.09           N\\nATOM   1546  CA  VAL A 202      16.729  11.761  -6.561  1.00 10.09           C\\nATOM   1547  C   VAL A 202      15.357  11.063  -6.671  1.00 10.09           C\\nATOM   1548  O   VAL A 202      14.915  10.768  -7.780  1.00 10.09           O\\nATOM   1549  CB  VAL A 202      17.906  10.753  -6.467  1.00 10.09           C\\nATOM   1550  CG1 VAL A 202      17.826   9.788  -5.272  1.00 10.09           C\\nATOM   1551  CG2 VAL A 202      18.108   9.962  -7.770  1.00 10.09           C\\nATOM   1552  N   ASN A 203      14.670  10.860  -5.534  1.00  8.68           N\\nATOM   1553  CA  ASN A 203      13.325  10.298  -5.461  1.00  8.68           C\\nATOM   1554  C   ASN A 203      12.259  11.337  -5.836  1.00  8.68           C\\nATOM   1555  O   ASN A 203      11.300  10.971  -6.513  1.00  8.68           O\\nATOM   1556  CB  ASN A 203      13.035   9.788  -4.031  1.00  8.68           C\\nATOM   1557  CG  ASN A 203      13.771   8.499  -3.659  1.00  8.68           C\\nATOM   1558  OD1 ASN A 203      14.789   8.149  -4.251  1.00  8.68           O\\nATOM   1559  ND2 ASN A 203      13.244   7.770  -2.676  1.00  8.68           N\\nATOM   1560  N   VAL A 204      12.429  12.622  -5.462  1.00  6.62           N\\nATOM   1561  CA  VAL A 204      11.555  13.720  -5.899  1.00  6.62           C\\nATOM   1562  C   VAL A 204      11.639  13.913  -7.420  1.00  6.62           C\\nATOM   1563  O   VAL A 204      10.599  14.024  -8.059  1.00  6.62           O\\nATOM   1564  CB  VAL A 204      11.865  15.046  -5.151  1.00  6.62           C\\nATOM   1565  CG1 VAL A 204      11.036  16.234  -5.678  1.00  6.62           C\\nATOM   1566  CG2 VAL A 204      11.616  14.923  -3.643  1.00  6.62           C\\nATOM   1567  N   LEU A 205      12.853  13.898  -7.996  1.00  7.13           N\\nATOM   1568  CA  LEU A 205      13.089  14.009  -9.436  1.00  7.13           C\\nATOM   1569  C   LEU A 205      12.493  12.833 -10.216  1.00  7.13           C\\nATOM   1570  O   LEU A 205      11.889  13.058 -11.262  1.00  7.13           O\\nATOM   1571  CB  LEU A 205      14.601  14.100  -9.711  1.00  7.13           C\\nATOM   1572  CG  LEU A 205      15.234  15.426  -9.252  1.00  7.13           C\\nATOM   1573  CD1 LEU A 205      16.760  15.313  -9.208  1.00  7.13           C\\nATOM   1574  CD2 LEU A 205      14.815  16.611 -10.134  1.00  7.13           C\\nATOM   1575  N   ALA A 206      12.592  11.607  -9.673  1.00  8.73           N\\nATOM   1576  CA  ALA A 206      11.974  10.399 -10.213  1.00  8.73           C\\nATOM   1577  C   ALA A 206      10.444  10.488 -10.231  1.00  8.73           C\\nATOM   1578  O   ALA A 206       9.824  10.108 -11.218  1.00  8.73           O\\nATOM   1579  CB  ALA A 206      12.406   9.197  -9.368  1.00  8.73           C\\nATOM   1580  N   TRP A 207       9.835  11.015  -9.160  1.00  8.66           N\\nATOM   1581  CA  TRP A 207       8.395  11.217  -9.041  1.00  8.66           C\\nATOM   1582  C   TRP A 207       7.886  12.403  -9.889  1.00  8.66           C\\nATOM   1583  O   TRP A 207       6.725  12.384 -10.295  1.00  8.66           O\\nATOM   1584  CB  TRP A 207       8.082  11.411  -7.550  1.00  8.66           C\\nATOM   1585  CG  TRP A 207       6.719  11.911  -7.198  1.00  8.66           C\\nATOM   1586  CD1 TRP A 207       5.608  11.164  -7.016  1.00  8.66           C\\nATOM   1587  CD2 TRP A 207       6.323  13.294  -6.975  1.00  8.66           C\\nATOM   1588  NE1 TRP A 207       4.559  11.988  -6.664  1.00  8.66           N\\nATOM   1589  CE2 TRP A 207       4.950  13.309  -6.597  1.00  8.66           C\\nATOM   1590  CE3 TRP A 207       6.996  14.535  -7.031  1.00  8.66           C\\nATOM   1591  CZ2 TRP A 207       4.299  14.493  -6.220  1.00  8.66           C\\nATOM   1592  CZ3 TRP A 207       6.337  15.734  -6.701  1.00  8.66           C\\nATOM   1593  CH2 TRP A 207       4.998  15.711  -6.269  1.00  8.66           C\\nATOM   1594  N   LEU A 208       8.736  13.394 -10.218  1.00  6.89           N\\nATOM   1595  CA  LEU A 208       8.418  14.441 -11.193  1.00  6.89           C\\nATOM   1596  C   LEU A 208       8.499  13.892 -12.631  1.00  6.89           C\\nATOM   1597  O   LEU A 208       7.743  14.352 -13.482  1.00  6.89           O\\nATOM   1598  CB  LEU A 208       9.388  15.637 -11.057  1.00  6.89           C\\nATOM   1599  CG  LEU A 208       9.223  16.504  -9.788  1.00  6.89           C\\nATOM   1600  CD1 LEU A 208      10.397  17.486  -9.654  1.00  6.89           C\\nATOM   1601  CD2 LEU A 208       7.892  17.268  -9.745  1.00  6.89           C\\nATOM   1602  N   TYR A 209       9.334  12.869 -12.901  1.00  7.71           N\\nATOM   1603  CA  TYR A 209       9.310  12.112 -14.157  1.00  7.71           C\\nATOM   1604  C   TYR A 209       8.071  11.211 -14.244  1.00  7.71           C\\nATOM   1605  O   TYR A 209       7.490  11.109 -15.322  1.00  7.71           O\\nATOM   1606  CB  TYR A 209      10.570  11.231 -14.298  1.00  7.71           C\\nATOM   1607  CG  TYR A 209      11.795  11.970 -14.790  1.00  7.71           C\\nATOM   1608  CD1 TYR A 209      11.791  12.515 -16.087  1.00  7.71           C\\nATOM   1609  CD2 TYR A 209      12.946  12.092 -13.985  1.00  7.71           C\\nATOM   1610  CE1 TYR A 209      12.916  13.203 -16.564  1.00  7.71           C\\nATOM   1611  CE2 TYR A 209      14.071  12.793 -14.458  1.00  7.71           C\\nATOM   1612  CZ  TYR A 209      14.050  13.346 -15.752  1.00  7.71           C\\nATOM   1613  OH  TYR A 209      15.128  14.013 -16.236  1.00  7.71           O\\nATOM   1614  N   ALA A 210       7.628  10.623 -13.116  1.00  9.52           N\\nATOM   1615  CA  ALA A 210       6.394   9.844 -13.000  1.00  9.52           C\\nATOM   1616  C   ALA A 210       5.162  10.709 -13.290  1.00  9.52           C\\nATOM   1617  O   ALA A 210       4.238  10.250 -13.952  1.00  9.52           O\\nATOM   1618  CB  ALA A 210       6.270   9.267 -11.581  1.00  9.52           C\\nATOM   1619  N   ALA A 211       5.180  11.972 -12.832  1.00 10.16           N\\nATOM   1620  CA  ALA A 211       4.136  12.960 -13.056  1.00 10.16           C\\nATOM   1621  C   ALA A 211       4.007  13.329 -14.537  1.00 10.16           C\\nATOM   1622  O   ALA A 211       2.887  13.328 -15.041  1.00 10.16           O\\nATOM   1623  CB  ALA A 211       4.436  14.199 -12.210  1.00 10.16           C\\nATOM   1624  N   VAL A 212       5.129  13.568 -15.244  1.00 12.58           N\\nATOM   1625  CA  VAL A 212       5.160  13.828 -16.689  1.00 12.58           C\\nATOM   1626  C   VAL A 212       4.635  12.636 -17.507  1.00 12.58           C\\nATOM   1627  O   VAL A 212       3.844  12.850 -18.423  1.00 12.58           O\\nATOM   1628  CB  VAL A 212       6.585  14.245 -17.142  1.00 12.58           C\\nATOM   1629  CG1 VAL A 212       6.768  14.287 -18.676  1.00 12.58           C\\nATOM   1630  CG2 VAL A 212       6.954  15.617 -16.562  1.00 12.58           C\\nATOM   1631  N   ILE A 213       5.042  11.399 -17.168  1.00 14.98           N\\nATOM   1632  CA  ILE A 213       4.582  10.152 -17.798  1.00 14.98           C\\nATOM   1633  C   ILE A 213       3.053   9.964 -17.657  1.00 14.98           C\\nATOM   1634  O   ILE A 213       2.420   9.434 -18.569  1.00 14.98           O\\nATOM   1635  CB  ILE A 213       5.392   8.945 -17.224  1.00 14.98           C\\nATOM   1636  CG1 ILE A 213       6.878   9.009 -17.655  1.00 14.98           C\\nATOM   1637  CG2 ILE A 213       4.807   7.568 -17.610  1.00 14.98           C\\nATOM   1638  CD1 ILE A 213       7.826   8.156 -16.796  1.00 14.98           C\\nATOM   1639  N   ASN A 214       2.468  10.432 -16.542  1.00 18.81           N\\nATOM   1640  CA  ASN A 214       1.050  10.309 -16.198  1.00 18.81           C\\nATOM   1641  C   ASN A 214       0.255  11.617 -16.408  1.00 18.81           C\\nATOM   1642  O   ASN A 214      -0.884  11.718 -15.953  1.00 18.81           O\\nATOM   1643  CB  ASN A 214       0.945   9.788 -14.745  1.00 18.81           C\\nATOM   1644  CG  ASN A 214       1.352   8.313 -14.646  1.00 18.81           C\\nATOM   1645  OD1 ASN A 214       0.599   7.443 -15.067  1.00 18.81           O\\nATOM   1646  ND2 ASN A 214       2.543   8.016 -14.128  1.00 18.81           N\\nATOM   1647  N   GLY A 215       0.814  12.584 -17.158  1.00 21.39           N\\nATOM   1648  CA  GLY A 215       0.104  13.727 -17.734  1.00 21.39           C\\nATOM   1649  C   GLY A 215       0.107  15.036 -16.931  1.00 21.39           C\\nATOM   1650  O   GLY A 215      -0.351  16.045 -17.467  1.00 21.39           O\\nATOM   1651  N  AASP A 216       0.636  15.060 -15.695  0.17 15.98           N\\nATOM   1652  CA AASP A 216       0.840  16.284 -14.908  0.17 15.98           C\\nATOM   1653  C  AASP A 216       2.175  16.914 -15.325  0.17 15.98           C\\nATOM   1654  O  AASP A 216       3.243  16.349 -15.095  0.17 15.98           O\\nATOM   1655  CB AASP A 216       0.875  15.963 -13.394  0.17 15.98           C\\nATOM   1656  CG AASP A 216      -0.490  15.631 -12.783  0.17 15.98           C\\nATOM   1657  OD1AASP A 216      -1.513  15.703 -13.498  0.17 15.98           O\\nATOM   1658  OD2AASP A 216      -0.503  15.335 -11.570  0.17 15.98           O\\nATOM   1659  N   ARG A 217       2.087  18.073 -15.993  1.00 15.72           N\\nATOM   1660  CA  ARG A 217       3.177  18.714 -16.729  1.00 15.72           C\\nATOM   1661  C   ARG A 217       3.283  20.222 -16.470  1.00 15.72           C\\nATOM   1662  O   ARG A 217       4.184  20.850 -17.025  1.00 15.72           O\\nATOM   1663  CB  ARG A 217       3.050  18.390 -18.237  1.00 15.72           C\\nATOM   1664  CG  ARG A 217       3.230  16.902 -18.571  1.00 15.72           C\\nATOM   1665  CD  ARG A 217       3.061  16.581 -20.058  1.00 15.72           C\\nATOM   1666  NE  ARG A 217       3.247  15.140 -20.277  1.00 15.72           N\\nATOM   1667  CZ  ARG A 217       2.584  14.349 -21.137  1.00 15.72           C\\nATOM   1668  NH1 ARG A 217       1.626  14.829 -21.941  1.00 15.72           N\\nATOM   1669  NH2 ARG A 217       2.891  13.047 -21.187  1.00 15.72           N\\nATOM   1670  N   TRP A 218       2.389  20.814 -15.654  1.00 13.94           N\\nATOM   1671  CA  TRP A 218       2.288  22.261 -15.412  1.00 13.94           C\\nATOM   1672  C   TRP A 218       3.569  22.902 -14.845  1.00 13.94           C\\nATOM   1673  O   TRP A 218       3.838  24.072 -15.114  1.00 13.94           O\\nATOM   1674  CB  TRP A 218       1.082  22.560 -14.490  1.00 13.94           C\\nATOM   1675  CG  TRP A 218       1.146  22.075 -13.065  1.00 13.94           C\\nATOM   1676  CD1 TRP A 218       0.529  20.974 -12.578  1.00 13.94           C\\nATOM   1677  CD2 TRP A 218       1.847  22.671 -11.926  1.00 13.94           C\\nATOM   1678  NE1 TRP A 218       0.801  20.847 -11.233  1.00 13.94           N\\nATOM   1679  CE2 TRP A 218       1.618  21.858 -10.775  1.00 13.94           C\\nATOM   1680  CE3 TRP A 218       2.663  23.814 -11.745  1.00 13.94           C\\nATOM   1681  CZ2 TRP A 218       2.169  22.162  -9.518  1.00 13.94           C\\nATOM   1682  CZ3 TRP A 218       3.256  24.104 -10.501  1.00 13.94           C\\nATOM   1683  CH2 TRP A 218       3.003  23.284  -9.386  1.00 13.94           C\\nATOM   1684  N   PHE A 219       4.345  22.132 -14.065  1.00 11.15           N\\nATOM   1685  CA  PHE A 219       5.559  22.557 -13.373  1.00 11.15           C\\nATOM   1686  C   PHE A 219       6.810  22.541 -14.269  1.00 11.15           C\\nATOM   1687  O   PHE A 219       7.858  23.003 -13.820  1.00 11.15           O\\nATOM   1688  CB  PHE A 219       5.787  21.613 -12.172  1.00 11.15           C\\nATOM   1689  CG  PHE A 219       5.918  20.136 -12.509  1.00 11.15           C\\nATOM   1690  CD1 PHE A 219       7.162  19.590 -12.894  1.00 11.15           C\\nATOM   1691  CD2 PHE A 219       4.776  19.310 -12.467  1.00 11.15           C\\nATOM   1692  CE1 PHE A 219       7.255  18.236 -13.268  1.00 11.15           C\\nATOM   1693  CE2 PHE A 219       4.876  17.951 -12.812  1.00 11.15           C\\nATOM   1694  CZ  PHE A 219       6.111  17.419 -13.226  1.00 11.15           C\\nATOM   1695  N   LEU A 220       6.725  22.032 -15.512  1.00 14.63           N\\nATOM   1696  CA  LEU A 220       7.829  22.043 -16.471  1.00 14.63           C\\nATOM   1697  C   LEU A 220       8.042  23.475 -16.974  1.00 14.63           C\\nATOM   1698  O   LEU A 220       7.175  24.038 -17.643  1.00 14.63           O\\nATOM   1699  CB  LEU A 220       7.520  21.120 -17.667  1.00 14.63           C\\nATOM   1700  CG  LEU A 220       7.423  19.616 -17.337  1.00 14.63           C\\nATOM   1701  CD1 LEU A 220       7.137  18.829 -18.627  1.00 14.63           C\\nATOM   1702  CD2 LEU A 220       8.685  19.081 -16.635  1.00 14.63           C\\nATOM   1703  N  AASN A 221       9.195  24.058 -16.617  0.17 26.99           N\\nATOM   1704  CA AASN A 221       9.583  25.421 -16.970  0.17 26.99           C\\nATOM   1705  C  AASN A 221      10.507  25.438 -18.199  0.17 26.99           C\\nATOM   1706  O  AASN A 221      10.783  24.408 -18.814  0.17 26.99           O\\nATOM   1707  CB AASN A 221      10.262  26.114 -15.758  0.17 26.99           C\\nATOM   1708  CG AASN A 221      11.489  25.424 -15.148  0.17 26.99           C\\nATOM   1709  OD1AASN A 221      12.258  24.734 -15.814  0.17 26.99           O\\nATOM   1710  ND2AASN A 221      11.671  25.620 -13.849  0.17 26.99           N\\nATOM   1711  N   ARG A 222      10.982  26.640 -18.544  1.00 22.56           N\\nATOM   1712  CA  ARG A 222      11.867  26.928 -19.668  1.00 22.56           C\\nATOM   1713  C   ARG A 222      13.356  26.908 -19.269  1.00 22.56           C\\nATOM   1714  O   ARG A 222      14.203  27.219 -20.105  1.00 22.56           O\\nATOM   1715  CB  ARG A 222      11.521  28.355 -20.150  1.00 22.56           C\\nATOM   1716  CG  ARG A 222      10.074  28.475 -20.655  1.00 22.56           C\\nATOM   1717  CD  ARG A 222       9.751  29.861 -21.222  1.00 22.56           C\\nATOM   1718  NE  ARG A 222       8.575  29.790 -22.098  1.00 22.56           N\\nATOM   1719  CZ  ARG A 222       8.590  29.421 -23.390  1.00 22.56           C\\nATOM   1720  NH1 ARG A 222       9.732  29.092 -24.014  1.00 22.56           N\\nATOM   1721  NH2 ARG A 222       7.439  29.373 -24.070  1.00 22.56           N\\nATOM   1722  N   PHE A 223      13.680  26.583 -18.006  1.00 23.05           N\\nATOM   1723  CA  PHE A 223      15.003  26.769 -17.413  1.00 23.05           C\\nATOM   1724  C   PHE A 223      15.779  25.449 -17.327  1.00 23.05           C\\nATOM   1725  O   PHE A 223      15.208  24.359 -17.369  1.00 23.05           O\\nATOM   1726  CB  PHE A 223      14.819  27.308 -15.976  1.00 23.05           C\\nATOM   1727  CG  PHE A 223      13.943  28.545 -15.834  1.00 23.05           C\\nATOM   1728  CD1 PHE A 223      13.921  29.556 -16.821  1.00 23.05           C\\nATOM   1729  CD2 PHE A 223      13.115  28.670 -14.701  1.00 23.05           C\\nATOM   1730  CE1 PHE A 223      13.044  30.651 -16.694  1.00 23.05           C\\nATOM   1731  CE2 PHE A 223      12.241  29.765 -14.571  1.00 23.05           C\\nATOM   1732  CZ  PHE A 223      12.199  30.752 -15.572  1.00 23.05           C\\nATOM   1733  N   THR A 224      17.097  25.586 -17.140  1.00 21.85           N\\nATOM   1734  CA  THR A 224      18.038  24.535 -16.775  1.00 21.85           C\\nATOM   1735  C   THR A 224      18.932  25.040 -15.628  1.00 21.85           C\\nATOM   1736  O   THR A 224      18.936  26.229 -15.303  1.00 21.85           O\\nATOM   1737  CB  THR A 224      18.773  23.987 -18.022  1.00 21.85           C\\nATOM   1738  OG1 THR A 224      19.378  22.749 -17.711  1.00 21.85           O\\nATOM   1739  CG2 THR A 224      19.857  24.919 -18.589  1.00 21.85           C\\nATOM   1740  N   THR A 225      19.678  24.123 -15.006  1.00 20.22           N\\nATOM   1741  CA  THR A 225      20.608  24.367 -13.908  1.00 20.22           C\\nATOM   1742  C   THR A 225      21.850  23.479 -14.091  1.00 20.22           C\\nATOM   1743  O   THR A 225      21.918  22.674 -15.017  1.00 20.22           O\\nATOM   1744  CB  THR A 225      19.859  24.274 -12.546  1.00 20.22           C\\nATOM   1745  OG1 THR A 225      20.687  24.597 -11.453  1.00 20.22           O\\nATOM   1746  CG2 THR A 225      19.293  22.887 -12.229  1.00 20.22           C\\nATOM   1747  N   THR A 226      22.841  23.633 -13.210  1.00 22.36           N\\nATOM   1748  CA  THR A 226      23.969  22.721 -13.029  1.00 22.36           C\\nATOM   1749  C   THR A 226      23.743  21.958 -11.710  1.00 22.36           C\\nATOM   1750  O   THR A 226      22.879  22.339 -10.916  1.00 22.36           O\\nATOM   1751  CB  THR A 226      25.276  23.553 -12.990  1.00 22.36           C\\nATOM   1752  OG1 THR A 226      25.340  24.381 -11.845  1.00 22.36           O\\nATOM   1753  CG2 THR A 226      25.504  24.407 -14.247  1.00 22.36           C\\nATOM   1754  N   LEU A 227      24.530  20.898 -11.448  1.00 21.61           N\\nATOM   1755  CA  LEU A 227      24.510  20.190 -10.162  1.00 21.61           C\\nATOM   1756  C   LEU A 227      24.896  21.104  -8.995  1.00 21.61           C\\nATOM   1757  O   LEU A 227      24.219  21.076  -7.972  1.00 21.61           O\\nATOM   1758  CB  LEU A 227      25.444  18.956 -10.163  1.00 21.61           C\\nATOM   1759  CG  LEU A 227      24.876  17.692 -10.834  1.00 21.61           C\\nATOM   1760  CD1 LEU A 227      25.889  16.538 -10.746  1.00 21.61           C\\nATOM   1761  CD2 LEU A 227      23.533  17.261 -10.227  1.00 21.61           C\\nATOM   1762  N   ASN A 228      25.934  21.940  -9.161  1.00 20.36           N\\nATOM   1763  CA  ASN A 228      26.396  22.871  -8.134  1.00 20.36           C\\nATOM   1764  C   ASN A 228      25.382  23.985  -7.839  1.00 20.36           C\\nATOM   1765  O   ASN A 228      25.134  24.237  -6.663  1.00 20.36           O\\nATOM   1766  CB  ASN A 228      27.771  23.473  -8.505  1.00 20.36           C\\nATOM   1767  CG  ASN A 228      28.913  22.447  -8.485  1.00 20.36           C\\nATOM   1768  OD1 ASN A 228      28.868  21.460  -7.753  1.00 20.36           O\\nATOM   1769  ND2 ASN A 228      29.941  22.662  -9.306  1.00 20.36           N\\nATOM   1770  N   ASP A 229      24.756  24.607  -8.857  1.00 19.18           N\\nATOM   1771  CA  ASP A 229      23.725  25.641  -8.677  1.00 19.18           C\\nATOM   1772  C   ASP A 229      22.452  25.069  -8.027  1.00 19.18           C\\nATOM   1773  O   ASP A 229      21.897  25.698  -7.126  1.00 19.18           O\\nATOM   1774  CB  ASP A 229      23.411  26.330 -10.029  1.00 19.18           C\\nATOM   1775  CG  ASP A 229      22.272  27.360  -9.967  1.00 19.18           C\\nATOM   1776  OD1 ASP A 229      22.415  28.331  -9.194  1.00 19.18           O\\nATOM   1777  OD2 ASP A 229      21.264  27.156 -10.676  1.00 19.18           O\\nATOM   1778  N   PHE A 230      22.032  23.862  -8.445  1.00 15.87           N\\nATOM   1779  CA  PHE A 230      20.915  23.141  -7.847  1.00 15.87           C\\nATOM   1780  C   PHE A 230      21.190  22.786  -6.374  1.00 15.87           C\\nATOM   1781  O   PHE A 230      20.287  22.923  -5.555  1.00 15.87           O\\nATOM   1782  CB  PHE A 230      20.595  21.863  -8.655  1.00 15.87           C\\nATOM   1783  CG  PHE A 230      19.448  21.084  -8.037  1.00 15.87           C\\nATOM   1784  CD1 PHE A 230      18.123  21.534  -8.199  1.00 15.87           C\\nATOM   1785  CD2 PHE A 230      19.726  20.027  -7.150  1.00 15.87           C\\nATOM   1786  CE1 PHE A 230      17.080  20.949  -7.454  1.00 15.87           C\\nATOM   1787  CE2 PHE A 230      18.695  19.489  -6.365  1.00 15.87           C\\nATOM   1788  CZ  PHE A 230      17.371  19.939  -6.518  1.00 15.87           C\\nATOM   1789  N   ASN A 231      22.410  22.335  -6.037  1.00 19.38           N\\nATOM   1790  CA  ASN A 231      22.774  21.919  -4.682  1.00 19.38           C\\nATOM   1791  C   ASN A 231      22.944  23.081  -3.689  1.00 19.38           C\\nATOM   1792  O   ASN A 231      22.850  22.833  -2.488  1.00 19.38           O\\nATOM   1793  CB  ASN A 231      24.037  21.028  -4.686  1.00 19.38           C\\nATOM   1794  CG  ASN A 231      23.786  19.616  -5.234  1.00 19.38           C\\nATOM   1795  OD1 ASN A 231      22.659  19.125  -5.238  1.00 19.38           O\\nATOM   1796  ND2 ASN A 231      24.838  18.958  -5.722  1.00 19.38           N\\nATOM   1797  N   LEU A 232      23.090  24.337  -4.155  1.00 21.71           N\\nATOM   1798  CA  LEU A 232      22.988  25.532  -3.306  1.00 21.71           C\\nATOM   1799  C   LEU A 232      21.534  25.751  -2.849  1.00 21.71           C\\nATOM   1800  O   LEU A 232      21.306  26.116  -1.697  1.00 21.71           O\\nATOM   1801  CB  LEU A 232      23.437  26.802  -4.068  1.00 21.71           C\\nATOM   1802  CG  LEU A 232      24.923  26.875  -4.473  1.00 21.71           C\\nATOM   1803  CD1 LEU A 232      25.191  28.138  -5.305  1.00 21.71           C\\nATOM   1804  CD2 LEU A 232      25.880  26.796  -3.276  1.00 21.71           C\\nATOM   1805  N   VAL A 233      20.557  25.488  -3.735  1.00 16.08           N\\nATOM   1806  CA  VAL A 233      19.126  25.531  -3.433  1.00 16.08           C\\nATOM   1807  C   VAL A 233      18.738  24.302  -2.589  1.00 16.08           C\\nATOM   1808  O   VAL A 233      17.991  24.457  -1.627  1.00 16.08           O\\nATOM   1809  CB  VAL A 233      18.306  25.596  -4.754  1.00 16.08           C\\nATOM   1810  CG1 VAL A 233      16.781  25.637  -4.521  1.00 16.08           C\\nATOM   1811  CG2 VAL A 233      18.724  26.806  -5.608  1.00 16.08           C\\nATOM   1812  N   ALA A 234      19.281  23.107  -2.888  1.00 15.07           N\\nATOM   1813  CA  ALA A 234      19.018  21.869  -2.151  1.00 15.07           C\\nATOM   1814  C   ALA A 234      19.432  21.974  -0.673  1.00 15.07           C\\nATOM   1815  O   ALA A 234      18.637  21.637   0.199  1.00 15.07           O\\nATOM   1816  CB  ALA A 234      19.731  20.680  -2.812  1.00 15.07           C\\nATOM   1817  N   MET A 235      20.625  22.532  -0.399  1.00 25.88           N\\nATOM   1818  CA  MET A 235      21.141  22.881   0.929  1.00 25.88           C\\nATOM   1819  C   MET A 235      20.157  23.757   1.725  1.00 25.88           C\\nATOM   1820  O   MET A 235      19.798  23.405   2.848  1.00 25.88           O\\nATOM   1821  CB  MET A 235      22.513  23.571   0.751  1.00 25.88           C\\nATOM   1822  CG  MET A 235      23.206  24.039   2.048  1.00 25.88           C\\nATOM   1823  SD  MET A 235      24.059  22.743   2.981  1.00 25.88           S\\nATOM   1824  CE  MET A 235      25.635  22.729   2.085  1.00 25.88           C\\nATOM   1825  N   LYS A 236      19.708  24.871   1.121  1.00 16.91           N\\nATOM   1826  CA  LYS A 236      18.770  25.842   1.687  1.00 16.91           C\\nATOM   1827  C   LYS A 236      17.419  25.233   2.114  1.00 16.91           C\\nATOM   1828  O   LYS A 236      16.878  25.649   3.136  1.00 16.91           O\\nATOM   1829  CB  LYS A 236      18.606  27.009   0.682  1.00 16.91           C\\nATOM   1830  CG  LYS A 236      17.448  27.988   0.965  1.00 16.91           C\\nATOM   1831  CD  LYS A 236      17.517  29.253   0.099  1.00 16.91           C\\nATOM   1832  CE  LYS A 236      16.198  30.040   0.058  1.00 16.91           C\\nATOM   1833  NZ  LYS A 236      15.808  30.548   1.383  1.00 16.91           N\\nATOM   1834  N   TYR A 237      16.890  24.258   1.358  1.00 11.29           N\\nATOM   1835  CA  TYR A 237      15.583  23.639   1.594  1.00 11.29           C\\nATOM   1836  C   TYR A 237      15.671  22.280   2.313  1.00 11.29           C\\nATOM   1837  O   TYR A 237      14.651  21.604   2.434  1.00 11.29           O\\nATOM   1838  CB  TYR A 237      14.861  23.504   0.232  1.00 11.29           C\\nATOM   1839  CG  TYR A 237      14.293  24.834  -0.230  1.00 11.29           C\\nATOM   1840  CD1 TYR A 237      13.139  25.347   0.394  1.00 11.29           C\\nATOM   1841  CD2 TYR A 237      14.935  25.596  -1.228  1.00 11.29           C\\nATOM   1842  CE1 TYR A 237      12.666  26.629   0.062  1.00 11.29           C\\nATOM   1843  CE2 TYR A 237      14.468  26.880  -1.559  1.00 11.29           C\\nATOM   1844  CZ  TYR A 237      13.342  27.404  -0.899  1.00 11.29           C\\nATOM   1845  OH  TYR A 237      12.905  28.663  -1.185  1.00 11.29           O\\nATOM   1846  N   ASN A 238      16.858  21.870   2.797  1.00 17.89           N\\nATOM   1847  CA  ASN A 238      17.118  20.594   3.483  1.00 17.89           C\\nATOM   1848  C   ASN A 238      16.827  19.388   2.572  1.00 17.89           C\\nATOM   1849  O   ASN A 238      16.229  18.407   2.994  1.00 17.89           O\\nATOM   1850  CB  ASN A 238      16.407  20.479   4.861  1.00 17.89           C\\nATOM   1851  CG  ASN A 238      16.915  21.472   5.906  1.00 17.89           C\\nATOM   1852  OD1 ASN A 238      18.117  21.586   6.136  1.00 17.89           O\\nATOM   1853  ND2 ASN A 238      15.999  22.211   6.532  1.00 17.89           N\\nATOM   1854  N   TYR A 239      17.214  19.487   1.299  1.00 13.89           N\\nATOM   1855  CA  TYR A 239      17.227  18.409   0.323  1.00 13.89           C\\nATOM   1856  C   TYR A 239      18.653  17.855   0.283  1.00 13.89           C\\nATOM   1857  O   TYR A 239      19.620  18.611   0.403  1.00 13.89           O\\nATOM   1858  CB  TYR A 239      16.889  18.997  -1.064  1.00 13.89           C\\nATOM   1859  CG  TYR A 239      15.423  18.950  -1.434  1.00 13.89           C\\nATOM   1860  CD1 TYR A 239      14.463  19.586  -0.624  1.00 13.89           C\\nATOM   1861  CD2 TYR A 239      15.013  18.262  -2.594  1.00 13.89           C\\nATOM   1862  CE1 TYR A 239      13.099  19.483  -0.942  1.00 13.89           C\\nATOM   1863  CE2 TYR A 239      13.652  18.163  -2.918  1.00 13.89           C\\nATOM   1864  CZ  TYR A 239      12.694  18.764  -2.086  1.00 13.89           C\\nATOM   1865  OH  TYR A 239      11.373  18.645  -2.396  1.00 13.89           O\\nATOM   1866  N   GLU A 240      18.779  16.536   0.086  1.00 18.49           N\\nATOM   1867  CA  GLU A 240      20.048  15.823  -0.046  1.00 18.49           C\\nATOM   1868  C   GLU A 240      20.833  16.322  -1.279  1.00 18.49           C\\nATOM   1869  O   GLU A 240      20.208  16.601  -2.304  1.00 18.49           O\\nATOM   1870  CB  GLU A 240      19.748  14.313  -0.197  1.00 18.49           C\\nATOM   1871  CG  GLU A 240      19.383  13.611   1.124  1.00 18.49           C\\nATOM   1872  CD  GLU A 240      20.573  13.547   2.080  1.00 18.49           C\\nATOM   1873  OE1 GLU A 240      21.472  12.719   1.818  1.00 18.49           O\\nATOM   1874  OE2 GLU A 240      20.586  14.350   3.038  1.00 18.49           O\\nATOM   1875  N   PRO A 241      22.179  16.445  -1.221  1.00 21.61           N\\nATOM   1876  CA  PRO A 241      23.003  16.794  -2.389  1.00 21.61           C\\nATOM   1877  C   PRO A 241      22.799  15.785  -3.532  1.00 21.61           C\\nATOM   1878  O   PRO A 241      22.879  14.575  -3.308  1.00 21.61           O\\nATOM   1879  CB  PRO A 241      24.461  16.749  -1.905  1.00 21.61           C\\nATOM   1880  CG  PRO A 241      24.403  16.752  -0.385  1.00 21.61           C\\nATOM   1881  CD  PRO A 241      22.989  16.314  -0.010  1.00 21.61           C\\nATOM   1882  N   LEU A 242      22.515  16.283  -4.742  1.00 21.42           N\\nATOM   1883  CA  LEU A 242      22.347  15.460  -5.930  1.00 21.42           C\\nATOM   1884  C   LEU A 242      23.733  15.274  -6.560  1.00 21.42           C\\nATOM   1885  O   LEU A 242      24.406  16.244  -6.913  1.00 21.42           O\\nATOM   1886  CB  LEU A 242      21.403  16.179  -6.914  1.00 21.42           C\\nATOM   1887  CG  LEU A 242      20.848  15.262  -8.025  1.00 21.42           C\\nATOM   1888  CD1 LEU A 242      19.821  14.248  -7.491  1.00 21.42           C\\nATOM   1889  CD2 LEU A 242      20.207  16.102  -9.129  1.00 21.42           C\\nATOM   1890  N   THR A 243      24.158  14.011  -6.664  1.00 27.38           N\\nATOM   1891  CA  THR A 243      25.442  13.585  -7.214  1.00 27.38           C\\nATOM   1892  C   THR A 243      25.277  13.239  -8.705  1.00 27.38           C\\nATOM   1893  O   THR A 243      24.159  13.178  -9.218  1.00 27.38           O\\nATOM   1894  CB  THR A 243      25.906  12.334  -6.422  1.00 27.38           C\\nATOM   1895  OG1 THR A 243      25.059  11.223  -6.652  1.00 27.38           O\\nATOM   1896  CG2 THR A 243      26.025  12.579  -4.908  1.00 27.38           C\\nATOM   1897  N   GLN A 244      26.397  12.970  -9.395  1.00 28.11           N\\nATOM   1898  CA  GLN A 244      26.387  12.458 -10.765  1.00 28.11           C\\nATOM   1899  C   GLN A 244      25.872  11.007 -10.831  1.00 28.11           C\\nATOM   1900  O   GLN A 244      25.264  10.647 -11.835  1.00 28.11           O\\nATOM   1901  CB  GLN A 244      27.801  12.560 -11.384  1.00 28.11           C\\nATOM   1902  CG  GLN A 244      27.868  12.180 -12.886  1.00 28.11           C\\nATOM   1903  CD  GLN A 244      27.047  13.079 -13.821  1.00 28.11           C\\nATOM   1904  OE1 GLN A 244      26.539  12.615 -14.838  1.00 28.11           O\\nATOM   1905  NE2 GLN A 244      26.881  14.357 -13.475  1.00 28.11           N\\nATOM   1906  N   ASP A 245      26.031  10.203  -9.759  1.00 26.75           N\\nATOM   1907  CA  ASP A 245      25.433   8.866  -9.642  1.00 26.75           C\\nATOM   1908  C   ASP A 245      23.900   8.924  -9.670  1.00 26.75           C\\nATOM   1909  O   ASP A 245      23.289   8.093 -10.334  1.00 26.75           O\\nATOM   1910  CB  ASP A 245      25.876   8.126  -8.354  1.00 26.75           C\\nATOM   1911  CG  ASP A 245      27.357   7.753  -8.262  1.00 26.75           C\\nATOM   1912  OD1 ASP A 245      28.095   7.976  -9.246  1.00 26.75           O\\nATOM   1913  OD2 ASP A 245      27.736   7.254  -7.181  1.00 26.75           O\\nATOM   1914  N   HIS A 246      23.289   9.926  -9.013  1.00 20.76           N\\nATOM   1915  CA  HIS A 246      21.846  10.173  -9.029  1.00 20.76           C\\nATOM   1916  C   HIS A 246      21.340  10.596 -10.417  1.00 20.76           C\\nATOM   1917  O   HIS A 246      20.259  10.165 -10.813  1.00 20.76           O\\nATOM   1918  CB  HIS A 246      21.500  11.270  -8.007  1.00 20.76           C\\nATOM   1919  CG  HIS A 246      21.747  10.913  -6.564  1.00 20.76           C\\nATOM   1920  ND1 HIS A 246      22.067  11.875  -5.601  1.00 20.76           N\\nATOM   1921  CD2 HIS A 246      21.676   9.681  -5.948  1.00 20.76           C\\nATOM   1922  CE1 HIS A 246      22.214  11.197  -4.472  1.00 20.76           C\\nATOM   1923  NE2 HIS A 246      21.976   9.894  -4.615  1.00 20.76           N\\nATOM   1924  N   VAL A 247      22.134  11.382 -11.169  1.00 20.08           N\\nATOM   1925  CA  VAL A 247      21.859  11.770 -12.556  1.00 20.08           C\\nATOM   1926  C   VAL A 247      21.887  10.552 -13.496  1.00 20.08           C\\nATOM   1927  O   VAL A 247      20.994  10.424 -14.334  1.00 20.08           O\\nATOM   1928  CB  VAL A 247      22.849  12.882 -13.011  1.00 20.08           C\\nATOM   1929  CG1 VAL A 247      22.779  13.216 -14.513  1.00 20.08           C\\nATOM   1930  CG2 VAL A 247      22.656  14.177 -12.205  1.00 20.08           C\\nATOM   1931  N   ASP A 248      22.866   9.647 -13.330  1.00 21.75           N\\nATOM   1932  CA  ASP A 248      23.009   8.413 -14.111  1.00 21.75           C\\nATOM   1933  C   ASP A 248      21.874   7.411 -13.842  1.00 21.75           C\\nATOM   1934  O   ASP A 248      21.413   6.758 -14.779  1.00 21.75           O\\nATOM   1935  CB  ASP A 248      24.363   7.724 -13.817  1.00 21.75           C\\nATOM   1936  CG  ASP A 248      25.608   8.556 -14.140  1.00 21.75           C\\nATOM   1937  OD1 ASP A 248      25.512   9.479 -14.977  1.00 21.75           O\\nATOM   1938  OD2 ASP A 248      26.660   8.237 -13.545  1.00 21.75           O\\nATOM   1939  N   ILE A 249      21.395   7.320 -12.588  1.00 19.78           N\\nATOM   1940  CA  ILE A 249      20.264   6.480 -12.178  1.00 19.78           C\\nATOM   1941  C   ILE A 249      18.931   7.020 -12.748  1.00 19.78           C\\nATOM   1942  O   ILE A 249      18.052   6.216 -13.051  1.00 19.78           O\\nATOM   1943  CB  ILE A 249      20.239   6.362 -10.620  1.00 19.78           C\\nATOM   1944  CG1 ILE A 249      21.457   5.569 -10.090  1.00 19.78           C\\nATOM   1945  CG2 ILE A 249      18.962   5.693 -10.076  1.00 19.78           C\\nATOM   1946  CD1 ILE A 249      21.755   5.803  -8.600  1.00 19.78           C\\nATOM   1947  N   LEU A 250      18.788   8.346 -12.933  1.00 13.97           N\\nATOM   1948  CA  LEU A 250      17.620   9.002 -13.538  1.00 13.97           C\\nATOM   1949  C   LEU A 250      17.663   9.014 -15.083  1.00 13.97           C\\nATOM   1950  O   LEU A 250      16.711   9.490 -15.702  1.00 13.97           O\\nATOM   1951  CB  LEU A 250      17.566  10.470 -13.049  1.00 13.97           C\\nATOM   1952  CG  LEU A 250      17.097  10.645 -11.590  1.00 13.97           C\\nATOM   1953  CD1 LEU A 250      17.427  12.057 -11.082  1.00 13.97           C\\nATOM   1954  CD2 LEU A 250      15.593  10.370 -11.431  1.00 13.97           C\\nATOM   1955  N   GLY A 251      18.729   8.477 -15.702  1.00 21.12           N\\nATOM   1956  CA  GLY A 251      18.956   8.400 -17.146  1.00 21.12           C\\nATOM   1957  C   GLY A 251      17.836   7.733 -17.964  1.00 21.12           C\\nATOM   1958  O   GLY A 251      17.359   8.361 -18.906  1.00 21.12           O\\nATOM   1959  N   PRO A 252      17.390   6.499 -17.636  1.00 23.69           N\\nATOM   1960  CA  PRO A 252      16.229   5.840 -18.261  1.00 23.69           C\\nATOM   1961  C   PRO A 252      14.913   6.649 -18.264  1.00 23.69           C\\nATOM   1962  O   PRO A 252      14.208   6.635 -19.272  1.00 23.69           O\\nATOM   1963  CB  PRO A 252      16.043   4.534 -17.478  1.00 23.69           C\\nATOM   1964  CG  PRO A 252      17.421   4.225 -16.919  1.00 23.69           C\\nATOM   1965  CD  PRO A 252      18.091   5.585 -16.735  1.00 23.69           C\\nATOM   1966  N   LEU A 253      14.594   7.377 -17.178  1.00 14.33           N\\nATOM   1967  CA  LEU A 253      13.398   8.226 -17.084  1.00 14.33           C\\nATOM   1968  C   LEU A 253      13.557   9.538 -17.868  1.00 14.33           C\\nATOM   1969  O   LEU A 253      12.582  10.024 -18.441  1.00 14.33           O\\nATOM   1970  CB  LEU A 253      13.123   8.575 -15.605  1.00 14.33           C\\nATOM   1971  CG  LEU A 253      12.558   7.406 -14.773  1.00 14.33           C\\nATOM   1972  CD1 LEU A 253      12.621   7.723 -13.271  1.00 14.33           C\\nATOM   1973  CD2 LEU A 253      11.125   7.031 -15.181  1.00 14.33           C\\nATOM   1974  N   SER A 254      14.787  10.071 -17.941  1.00 11.96           N\\nATOM   1975  CA  SER A 254      15.167  11.210 -18.771  1.00 11.96           C\\nATOM   1976  C   SER A 254      15.066  10.881 -20.273  1.00 11.96           C\\nATOM   1977  O   SER A 254      14.618  11.725 -21.047  1.00 11.96           O\\nATOM   1978  CB  SER A 254      16.594  11.636 -18.367  1.00 11.96           C\\nATOM   1979  OG  SER A 254      17.085  12.697 -19.154  1.00 11.96           O\\nATOM   1980  N   ALA A 255      15.421   9.645 -20.665  1.00 17.87           N\\nATOM   1981  CA  ALA A 255      15.352   9.134 -22.031  1.00 17.87           C\\nATOM   1982  C   ALA A 255      13.908   8.836 -22.471  1.00 17.87           C\\nATOM   1983  O   ALA A 255      13.584   9.043 -23.640  1.00 17.87           O\\nATOM   1984  CB  ALA A 255      16.165   7.831 -22.101  1.00 17.87           C\\nATOM   1985  N   GLN A 256      13.043   8.383 -21.544  1.00 25.07           N\\nATOM   1986  CA  GLN A 256      11.637   8.055 -21.793  1.00 25.07           C\\nATOM   1987  C   GLN A 256      10.787   9.314 -22.052  1.00 25.07           C\\nATOM   1988  O   GLN A 256       9.946   9.301 -22.951  1.00 25.07           O\\nATOM   1989  CB  GLN A 256      11.085   7.279 -20.572  1.00 25.07           C\\nATOM   1990  CG  GLN A 256       9.608   6.838 -20.706  1.00 25.07           C\\nATOM   1991  CD  GLN A 256       9.085   6.032 -19.513  1.00 25.07           C\\nATOM   1992  OE1 GLN A 256       9.825   5.692 -18.592  1.00 25.07           O\\nATOM   1993  NE2 GLN A 256       7.786   5.729 -19.526  1.00 25.07           N\\nATOM   1994  N   THR A 257      11.005  10.386 -21.273  1.00 18.15           N\\nATOM   1995  CA  THR A 257      10.247  11.640 -21.352  1.00 18.15           C\\nATOM   1996  C   THR A 257      10.902  12.680 -22.281  1.00 18.15           C\\nATOM   1997  O   THR A 257      10.231  13.631 -22.680  1.00 18.15           O\\nATOM   1998  CB  THR A 257      10.133  12.247 -19.931  1.00 18.15           C\\nATOM   1999  OG1 THR A 257      11.400  12.565 -19.386  1.00 18.15           O\\nATOM   2000  CG2 THR A 257       9.344  11.360 -18.956  1.00 18.15           C\\nATOM   2001  N   GLY A 258      12.196  12.526 -22.609  1.00 16.66           N\\nATOM   2002  CA  GLY A 258      12.983  13.474 -23.396  1.00 16.66           C\\nATOM   2003  C   GLY A 258      13.422  14.739 -22.639  1.00 16.66           C\\nATOM   2004  O   GLY A 258      14.008  15.627 -23.255  1.00 16.66           O\\nATOM   2005  N   ILE A 259      13.145  14.831 -21.327  1.00 15.50           N\\nATOM   2006  CA  ILE A 259      13.531  15.945 -20.463  1.00 15.50           C\\nATOM   2007  C   ILE A 259      14.850  15.539 -19.793  1.00 15.50           C\\nATOM   2008  O   ILE A 259      14.898  14.528 -19.092  1.00 15.50           O\\nATOM   2009  CB  ILE A 259      12.433  16.176 -19.381  1.00 15.50           C\\nATOM   2010  CG1 ILE A 259      11.068  16.515 -20.020  1.00 15.50           C\\nATOM   2011  CG2 ILE A 259      12.831  17.269 -18.359  1.00 15.50           C\\nATOM   2012  CD1 ILE A 259       9.894  16.366 -19.049  1.00 15.50           C\\nATOM   2013  N   ALA A 260      15.906  16.348 -19.983  1.00 16.52           N\\nATOM   2014  CA  ALA A 260      17.215  16.193 -19.346  1.00 16.52           C\\nATOM   2015  C   ALA A 260      17.101  16.326 -17.819  1.00 16.52           C\\nATOM   2016  O   ALA A 260      16.255  17.077 -17.331  1.00 16.52           O\\nATOM   2017  CB  ALA A 260      18.173  17.259 -19.896  1.00 16.52           C\\nATOM   2018  N   VAL A 261      17.931  15.588 -17.062  1.00 16.13           N\\nATOM   2019  CA  VAL A 261      17.928  15.574 -15.592  1.00 16.13           C\\nATOM   2020  C   VAL A 261      18.124  16.970 -14.986  1.00 16.13           C\\nATOM   2021  O   VAL A 261      17.411  17.322 -14.051  1.00 16.13           O\\nATOM   2022  CB  VAL A 261      18.939  14.542 -15.034  1.00 16.13           C\\nATOM   2023  CG1 VAL A 261      18.956  14.541 -13.492  1.00 16.13           C\\nATOM   2024  CG2 VAL A 261      18.645  13.118 -15.540  1.00 16.13           C\\nATOM   2025  N   LEU A 262      19.031  17.775 -15.557  1.00 19.24           N\\nATOM   2026  CA  LEU A 262      19.297  19.145 -15.137  1.00 19.24           C\\nATOM   2027  C   LEU A 262      18.195  20.146 -15.537  1.00 19.24           C\\nATOM   2028  O   LEU A 262      18.118  21.209 -14.926  1.00 19.24           O\\nATOM   2029  CB  LEU A 262      20.642  19.593 -15.729  1.00 19.24           C\\nATOM   2030  CG  LEU A 262      21.884  18.942 -15.076  1.00 19.24           C\\nATOM   2031  CD1 LEU A 262      23.157  19.400 -15.799  1.00 19.24           C\\nATOM   2032  CD2 LEU A 262      21.999  19.228 -13.568  1.00 19.24           C\\nATOM   2033  N   ASP A 263      17.307  19.815 -16.491  1.00 18.33           N\\nATOM   2034  CA  ASP A 263      16.095  20.597 -16.773  1.00 18.33           C\\nATOM   2035  C   ASP A 263      15.006  20.275 -15.736  1.00 18.33           C\\nATOM   2036  O   ASP A 263      14.304  21.186 -15.301  1.00 18.33           O\\nATOM   2037  CB  ASP A 263      15.530  20.294 -18.184  1.00 18.33           C\\nATOM   2038  CG  ASP A 263      16.432  20.679 -19.363  1.00 18.33           C\\nATOM   2039  OD1 ASP A 263      17.592  21.082 -19.129  1.00 18.33           O\\nATOM   2040  OD2 ASP A 263      15.937  20.549 -20.503  1.00 18.33           O\\nATOM   2041  N   MET A 264      14.904  19.009 -15.287  1.00 12.95           N\\nATOM   2042  CA  MET A 264      13.992  18.585 -14.220  1.00 12.95           C\\nATOM   2043  C   MET A 264      14.464  19.074 -12.834  1.00 12.95           C\\nATOM   2044  O   MET A 264      13.628  19.321 -11.967  1.00 12.95           O\\nATOM   2045  CB  MET A 264      13.873  17.044 -14.228  1.00 12.95           C\\nATOM   2046  CG  MET A 264      12.691  16.514 -13.392  1.00 12.95           C\\nATOM   2047  SD  MET A 264      11.055  17.029 -13.984  1.00 12.95           S\\nATOM   2048  CE  MET A 264      10.759  15.743 -15.217  1.00 12.95           C\\nATOM   2049  N   CYS A 265      15.780  19.280 -12.642  1.00 14.73           N\\nATOM   2050  CA  CYS A 265      16.369  19.906 -11.455  1.00 14.73           C\\nATOM   2051  C   CYS A 265      16.027  21.397 -11.387  1.00 14.73           C\\nATOM   2052  O   CYS A 265      15.817  21.917 -10.296  1.00 14.73           O\\nATOM   2053  CB  CYS A 265      17.901  19.774 -11.483  1.00 14.73           C\\nATOM   2054  SG  CYS A 265      18.401  18.065 -11.178  1.00 14.73           S\\nATOM   2055  N   ALA A 266      15.923  22.072 -12.543  1.00 19.80           N\\nATOM   2056  CA  ALA A 266      15.522  23.470 -12.632  1.00 19.80           C\\nATOM   2057  C   ALA A 266      14.015  23.637 -12.388  1.00 19.80           C\\nATOM   2058  O   ALA A 266      13.616  24.641 -11.804  1.00 19.80           O\\nATOM   2059  CB  ALA A 266      15.864  23.989 -14.024  1.00 19.80           C\\nATOM   2060  N   SER A 267      13.191  22.642 -12.766  1.00 16.52           N\\nATOM   2061  CA  SER A 267      11.767  22.560 -12.429  1.00 16.52           C\\nATOM   2062  C   SER A 267      11.540  22.349 -10.930  1.00 16.52           C\\nATOM   2063  O   SER A 267      10.679  23.012 -10.354  1.00 16.52           O\\nATOM   2064  CB  SER A 267      11.096  21.430 -13.223  1.00 16.52           C\\nATOM   2065  OG  SER A 267      10.741  21.914 -14.495  1.00 16.52           O\\nATOM   2066  N   LEU A 268      12.348  21.483 -10.296  1.00 10.55           N\\nATOM   2067  CA  LEU A 268      12.358  21.256  -8.854  1.00 10.55           C\\nATOM   2068  C   LEU A 268      12.841  22.502  -8.091  1.00 10.55           C\\nATOM   2069  O   LEU A 268      12.213  22.873  -7.106  1.00 10.55           O\\nATOM   2070  CB  LEU A 268      13.225  20.014  -8.550  1.00 10.55           C\\nATOM   2071  CG  LEU A 268      13.388  19.639  -7.059  1.00 10.55           C\\nATOM   2072  CD1 LEU A 268      12.051  19.569  -6.305  1.00 10.55           C\\nATOM   2073  CD2 LEU A 268      14.141  18.307  -6.926  1.00 10.55           C\\nATOM   2074  N   LYS A 269      13.891  23.188  -8.573  1.00 14.03           N\\nATOM   2075  CA  LYS A 269      14.404  24.458  -8.048  1.00 14.03           C\\nATOM   2076  C   LYS A 269      13.326  25.556  -7.969  1.00 14.03           C\\nATOM   2077  O   LYS A 269      13.211  26.206  -6.932  1.00 14.03           O\\nATOM   2078  CB  LYS A 269      15.633  24.887  -8.879  1.00 14.03           C\\nATOM   2079  CG  LYS A 269      16.106  26.337  -8.683  1.00 14.03           C\\nATOM   2080  CD  LYS A 269      17.393  26.658  -9.453  1.00 14.03           C\\nATOM   2081  CE  LYS A 269      17.718  28.155  -9.387  1.00 14.03           C\\nATOM   2082  NZ  LYS A 269      18.826  28.520 -10.283  1.00 14.03           N\\nATOM   2083  N   GLU A 270      12.518  25.734  -9.029  1.00 16.21           N\\nATOM   2084  CA  GLU A 270      11.428  26.711  -9.054  1.00 16.21           C\\nATOM   2085  C   GLU A 270      10.224  26.260  -8.208  1.00 16.21           C\\nATOM   2086  O   GLU A 270       9.554  27.121  -7.647  1.00 16.21           O\\nATOM   2087  CB  GLU A 270      10.941  26.927 -10.500  1.00 16.21           C\\nATOM   2088  CG  GLU A 270      11.932  27.624 -11.446  1.00 16.21           C\\nATOM   2089  CD  GLU A 270      12.477  28.961 -10.956  1.00 16.21           C\\nATOM   2090  OE1 GLU A 270      11.646  29.874 -10.762  1.00 16.21           O\\nATOM   2091  OE2 GLU A 270      13.716  29.049 -10.805  1.00 16.21           O\\nATOM   2092  N   LEU A 271       9.966  24.947  -8.055  1.00 14.11           N\\nATOM   2093  CA  LEU A 271       8.927  24.423  -7.157  1.00 14.11           C\\nATOM   2094  C   LEU A 271       9.287  24.624  -5.675  1.00 14.11           C\\nATOM   2095  O   LEU A 271       8.385  24.803  -4.860  1.00 14.11           O\\nATOM   2096  CB  LEU A 271       8.704  22.912  -7.399  1.00 14.11           C\\nATOM   2097  CG  LEU A 271       7.876  22.558  -8.650  1.00 14.11           C\\nATOM   2098  CD1 LEU A 271       8.017  21.060  -8.973  1.00 14.11           C\\nATOM   2099  CD2 LEU A 271       6.395  22.936  -8.491  1.00 14.11           C\\nATOM   2100  N   LEU A 272      10.584  24.639  -5.327  1.00 13.94           N\\nATOM   2101  CA  LEU A 272      11.067  24.925  -3.979  1.00 13.94           C\\nATOM   2102  C   LEU A 272      11.032  26.431  -3.688  1.00 13.94           C\\nATOM   2103  O   LEU A 272      10.637  26.825  -2.592  1.00 13.94           O\\nATOM   2104  CB  LEU A 272      12.526  24.432  -3.838  1.00 13.94           C\\nATOM   2105  CG  LEU A 272      12.689  22.901  -3.853  1.00 13.94           C\\nATOM   2106  CD1 LEU A 272      14.169  22.495  -3.960  1.00 13.94           C\\nATOM   2107  CD2 LEU A 272      12.032  22.240  -2.642  1.00 13.94           C\\nATOM   2108  N   GLN A 273      11.412  27.269  -4.664  1.00 14.78           N\\nATOM   2109  CA  GLN A 273      11.509  28.719  -4.517  1.00 14.78           C\\nATOM   2110  C   GLN A 273      10.141  29.423  -4.584  1.00 14.78           C\\nATOM   2111  O   GLN A 273       9.936  30.384  -3.843  1.00 14.78           O\\nATOM   2112  CB  GLN A 273      12.433  29.255  -5.628  1.00 14.78           C\\nATOM   2113  CG  GLN A 273      13.917  28.926  -5.341  1.00 14.78           C\\nATOM   2114  CD  GLN A 273      14.914  29.284  -6.451  1.00 14.78           C\\nATOM   2115  OE1 GLN A 273      16.117  29.127  -6.254  1.00 14.78           O\\nATOM   2116  NE2 GLN A 273      14.454  29.768  -7.605  1.00 14.78           N\\nATOM   2117  N   ASN A 274       9.207  28.953  -5.428  0.24 23.10           N\\nATOM   2118  CA  ASN A 274       7.885  29.566  -5.626  0.24 23.10           C\\nATOM   2119  C   ASN A 274       6.765  28.804  -4.897  0.24 23.10           C\\nATOM   2120  O   ASN A 274       5.670  29.348  -4.749  0.24 23.10           O\\nATOM   2121  CB  ASN A 274       7.525  29.579  -7.134  0.24 23.10           C\\nATOM   2122  CG  ASN A 274       8.546  30.318  -8.008  0.24 23.10           C\\nATOM   2123  OD1 ASN A 274       9.031  31.384  -7.638  0.24 23.10           O\\nATOM   2124  ND2 ASN A 274       8.912  29.732  -9.150  0.24 23.10           N\\nATOM   2125  N   GLY A 275       7.011  27.556  -4.462  1.00 19.76           N\\nATOM   2126  CA  GLY A 275       5.998  26.653  -3.920  1.00 19.76           C\\nATOM   2127  C   GLY A 275       5.122  26.045  -5.024  1.00 19.76           C\\nATOM   2128  O   GLY A 275       5.464  26.104  -6.207  1.00 19.76           O\\nATOM   2129  N   MET A 276       3.991  25.448  -4.626  1.00 19.59           N\\nATOM   2130  CA  MET A 276       3.040  24.790  -5.523  1.00 19.59           C\\nATOM   2131  C   MET A 276       1.936  25.758  -5.995  1.00 19.59           C\\nATOM   2132  O   MET A 276       1.369  25.541  -7.063  1.00 19.59           O\\nATOM   2133  CB  MET A 276       2.356  23.639  -4.755  1.00 19.59           C\\nATOM   2134  CG  MET A 276       3.320  22.535  -4.278  1.00 19.59           C\\nATOM   2135  SD  MET A 276       3.966  21.422  -5.558  1.00 19.59           S\\nATOM   2136  CE  MET A 276       2.565  20.288  -5.731  1.00 19.59           C\\nATOM   2137  N   ASN A 277       1.622  26.804  -5.206  1.00 18.55           N\\nATOM   2138  CA  ASN A 277       0.612  27.844  -5.462  1.00 18.55           C\\nATOM   2139  C   ASN A 277      -0.833  27.315  -5.391  1.00 18.55           C\\nATOM   2140  O   ASN A 277      -1.706  27.787  -6.116  1.00 18.55           O\\nATOM   2141  CB  ASN A 277       0.915  28.688  -6.731  1.00 18.55           C\\nATOM   2142  CG  ASN A 277       2.143  29.586  -6.560  1.00 18.55           C\\nATOM   2143  OD1 ASN A 277       2.070  30.606  -5.880  1.00 18.55           O\\nATOM   2144  ND2 ASN A 277       3.274  29.213  -7.161  1.00 18.55           N\\nATOM   2145  N   GLY A 278      -1.081  26.313  -4.532  1.00 17.96           N\\nATOM   2146  CA  GLY A 278      -2.373  25.653  -4.360  1.00 17.96           C\\nATOM   2147  C   GLY A 278      -2.657  24.536  -5.375  1.00 17.96           C\\nATOM   2148  O   GLY A 278      -3.723  23.926  -5.309  1.00 17.96           O\\nATOM   2149  N   ARG A 279      -1.722  24.258  -6.297  1.00 18.10           N\\nATOM   2150  CA  ARG A 279      -1.806  23.184  -7.288  1.00 18.10           C\\nATOM   2151  C   ARG A 279      -1.241  21.878  -6.703  1.00 18.10           C\\nATOM   2152  O   ARG A 279      -0.632  21.872  -5.632  1.00 18.10           O\\nATOM   2153  CB  ARG A 279      -0.969  23.590  -8.519  1.00 18.10           C\\nATOM   2154  CG  ARG A 279      -1.510  24.857  -9.207  1.00 18.10           C\\nATOM   2155  CD  ARG A 279      -0.755  25.223 -10.489  1.00 18.10           C\\nATOM   2156  NE  ARG A 279       0.545  25.836 -10.184  1.00 18.10           N\\nATOM   2157  CZ  ARG A 279       0.872  27.136 -10.278  1.00 18.10           C\\nATOM   2158  NH1 ARG A 279       2.131  27.512 -10.035  1.00 18.10           N\\nATOM   2159  NH2 ARG A 279      -0.022  28.076 -10.611  1.00 18.10           N\\nATOM   2160  N   THR A 280      -1.465  20.765  -7.416  1.00 16.67           N\\nATOM   2161  CA  THR A 280      -1.053  19.415  -7.037  1.00 16.67           C\\nATOM   2162  C   THR A 280      -0.289  18.742  -8.183  1.00 16.67           C\\nATOM   2163  O   THR A 280      -0.524  19.028  -9.357  1.00 16.67           O\\nATOM   2164  CB  THR A 280      -2.261  18.581  -6.538  1.00 16.67           C\\nATOM   2165  OG1 THR A 280      -3.197  18.346  -7.574  1.00 16.67           O\\nATOM   2166  CG2 THR A 280      -2.981  19.181  -5.321  1.00 16.67           C\\nATOM   2167  N   ILE A 281       0.609  17.820  -7.817  1.00 14.98           N\\nATOM   2168  CA  ILE A 281       1.335  16.922  -8.709  1.00 14.98           C\\nATOM   2169  C   ILE A 281       1.034  15.513  -8.190  1.00 14.98           C\\nATOM   2170  O   ILE A 281       1.317  15.222  -7.029  1.00 14.98           O\\nATOM   2171  CB  ILE A 281       2.851  17.262  -8.703  1.00 14.98           C\\nATOM   2172  CG1 ILE A 281       3.134  18.698  -9.197  1.00 14.98           C\\nATOM   2173  CG2 ILE A 281       3.661  16.246  -9.530  1.00 14.98           C\\nATOM   2174  CD1 ILE A 281       4.570  19.186  -8.947  1.00 14.98           C\\nATOM   2175  N   LEU A 282       0.421  14.662  -9.029  1.00 15.26           N\\nATOM   2176  CA  LEU A 282       0.009  13.279  -8.754  1.00 15.26           C\\nATOM   2177  C   LEU A 282      -0.883  13.158  -7.500  1.00 15.26           C\\nATOM   2178  O   LEU A 282      -0.735  12.227  -6.706  1.00 15.26           O\\nATOM   2179  CB  LEU A 282       1.240  12.341  -8.713  1.00 15.26           C\\nATOM   2180  CG  LEU A 282       1.936  12.135 -10.073  1.00 15.26           C\\nATOM   2181  CD1 LEU A 282       3.237  11.347  -9.873  1.00 15.26           C\\nATOM   2182  CD2 LEU A 282       1.021  11.434 -11.093  1.00 15.26           C\\nATOM   2183  N   GLY A 283      -1.793  14.128  -7.312  1.00 19.60           N\\nATOM   2184  CA  GLY A 283      -2.720  14.209  -6.188  1.00 19.60           C\\nATOM   2185  C   GLY A 283      -2.103  14.670  -4.858  1.00 19.60           C\\nATOM   2186  O   GLY A 283      -2.782  14.580  -3.835  1.00 19.60           O\\nATOM   2187  N   SER A 284      -0.847  15.150  -4.851  1.00 16.38           N\\nATOM   2188  CA  SER A 284      -0.152  15.627  -3.658  1.00 16.38           C\\nATOM   2189  C   SER A 284       0.164  17.122  -3.779  1.00 16.38           C\\nATOM   2190  O   SER A 284       0.655  17.585  -4.808  1.00 16.38           O\\nATOM   2191  CB  SER A 284       1.122  14.796  -3.425  1.00 16.38           C\\nATOM   2192  OG  SER A 284       1.736  15.181  -2.210  1.00 16.38           O\\nATOM   2193  N   ALA A 285      -0.083  17.847  -2.678  1.00 20.92           N\\nATOM   2194  CA  ALA A 285       0.254  19.251  -2.458  1.00 20.92           C\\nATOM   2195  C   ALA A 285       1.675  19.422  -1.878  1.00 20.92           C\\nATOM   2196  O   ALA A 285       2.039  20.527  -1.479  1.00 20.92           O\\nATOM   2197  CB  ALA A 285      -0.756  19.805  -1.437  1.00 20.92           C\\nATOM   2198  N   LEU A 286       2.489  18.355  -1.854  1.00 19.64           N\\nATOM   2199  CA  LEU A 286       3.899  18.364  -1.482  1.00 19.64           C\\nATOM   2200  C   LEU A 286       4.710  17.500  -2.448  1.00 19.64           C\\nATOM   2201  O   LEU A 286       4.162  16.685  -3.192  1.00 19.64           O\\nATOM   2202  CB  LEU A 286       4.102  18.097   0.032  1.00 19.64           C\\nATOM   2203  CG  LEU A 286       3.737  16.698   0.582  1.00 19.64           C\\nATOM   2204  CD1 LEU A 286       4.802  15.625   0.286  1.00 19.64           C\\nATOM   2205  CD2 LEU A 286       3.513  16.781   2.100  1.00 19.64           C\\nATOM   2206  N   LEU A 287       6.032  17.697  -2.431  1.00 14.07           N\\nATOM   2207  CA  LEU A 287       6.982  17.015  -3.294  1.00 14.07           C\\nATOM   2208  C   LEU A 287       7.369  15.687  -2.621  1.00 14.07           C\\nATOM   2209  O   LEU A 287       8.117  15.668  -1.642  1.00 14.07           O\\nATOM   2210  CB  LEU A 287       8.204  17.943  -3.478  1.00 14.07           C\\nATOM   2211  CG  LEU A 287       7.864  19.330  -4.089  1.00 14.07           C\\nATOM   2212  CD1 LEU A 287       9.090  20.251  -4.093  1.00 14.07           C\\nATOM   2213  CD2 LEU A 287       7.259  19.239  -5.499  1.00 14.07           C\\nATOM   2214  N   GLU A 288       6.800  14.584  -3.137  1.00 13.13           N\\nATOM   2215  CA  GLU A 288       6.912  13.222  -2.608  1.00 13.13           C\\nATOM   2216  C   GLU A 288       8.313  12.666  -2.884  1.00 13.13           C\\nATOM   2217  O   GLU A 288       8.845  12.868  -3.975  1.00 13.13           O\\nATOM   2218  CB  GLU A 288       5.901  12.323  -3.349  1.00 13.13           C\\nATOM   2219  CG  GLU A 288       4.418  12.681  -3.123  1.00 13.13           C\\nATOM   2220  CD  GLU A 288       3.902  12.300  -1.737  1.00 13.13           C\\nATOM   2221  OE1 GLU A 288       4.121  11.138  -1.336  1.00 13.13           O\\nATOM   2222  OE2 GLU A 288       3.267  13.168  -1.103  1.00 13.13           O\\nATOM   2223  N   ASP A 289       8.889  11.952  -1.904  1.00 11.67           N\\nATOM   2224  CA  ASP A 289      10.288  11.514  -1.891  1.00 11.67           C\\nATOM   2225  C   ASP A 289      10.461  10.039  -1.483  1.00 11.67           C\\nATOM   2226  O   ASP A 289      11.579   9.635  -1.169  1.00 11.67           O\\nATOM   2227  CB  ASP A 289      11.132  12.519  -1.077  1.00 11.67           C\\nATOM   2228  CG  ASP A 289      11.101  12.351   0.444  1.00 11.67           C\\nATOM   2229  OD1 ASP A 289      10.057  11.933   0.988  1.00 11.67           O\\nATOM   2230  OD2 ASP A 289      12.153  12.631   1.052  1.00 11.67           O\\nATOM   2231  N   GLU A 290       9.391   9.231  -1.512  1.00 13.47           N\\nATOM   2232  CA  GLU A 290       9.405   7.803  -1.187  1.00 13.47           C\\nATOM   2233  C   GLU A 290       8.974   6.958  -2.395  1.00 13.47           C\\nATOM   2234  O   GLU A 290       8.320   5.930  -2.233  1.00 13.47           O\\nATOM   2235  CB  GLU A 290       8.639   7.503   0.124  1.00 13.47           C\\nATOM   2236  CG  GLU A 290       9.094   8.377   1.316  1.00 13.47           C\\nATOM   2237  CD  GLU A 290       8.784   7.796   2.700  1.00 13.47           C\\nATOM   2238  OE1 GLU A 290       7.918   6.902   2.799  1.00 13.47           O\\nATOM   2239  OE2 GLU A 290       9.443   8.249   3.661  1.00 13.47           O\\nATOM   2240  N   PHE A 291       9.400   7.381  -3.597  1.00 10.30           N\\nATOM   2241  CA  PHE A 291       9.485   6.598  -4.826  1.00 10.30           C\\nATOM   2242  C   PHE A 291      10.905   6.760  -5.366  1.00 10.30           C\\nATOM   2243  O   PHE A 291      11.273   7.865  -5.762  1.00 10.30           O\\nATOM   2244  CB  PHE A 291       8.507   7.118  -5.897  1.00 10.30           C\\nATOM   2245  CG  PHE A 291       7.042   7.062  -5.549  1.00 10.30           C\\nATOM   2246  CD1 PHE A 291       6.335   5.861  -5.735  1.00 10.30           C\\nATOM   2247  CD2 PHE A 291       6.368   8.222  -5.114  1.00 10.30           C\\nATOM   2248  CE1 PHE A 291       4.946   5.836  -5.529  1.00 10.30           C\\nATOM   2249  CE2 PHE A 291       4.978   8.193  -4.906  1.00 10.30           C\\nATOM   2250  CZ  PHE A 291       4.265   7.000  -5.123  1.00 10.30           C\\nATOM   2251  N   THR A 292      11.688   5.675  -5.445  1.00  9.64           N\\nATOM   2252  CA  THR A 292      12.990   5.679  -6.116  1.00  9.64           C\\nATOM   2253  C   THR A 292      12.792   5.717  -7.650  1.00  9.64           C\\nATOM   2254  O   THR A 292      11.730   5.319  -8.133  1.00  9.64           O\\nATOM   2255  CB  THR A 292      13.809   4.420  -5.716  1.00  9.64           C\\nATOM   2256  OG1 THR A 292      13.295   3.234  -6.286  1.00  9.64           O\\nATOM   2257  CG2 THR A 292      13.978   4.232  -4.207  1.00  9.64           C\\nATOM   2258  N   PRO A 293      13.809   6.135  -8.432  1.00 10.12           N\\nATOM   2259  CA  PRO A 293      13.893   5.910  -9.886  1.00 10.12           C\\nATOM   2260  C   PRO A 293      13.533   4.489 -10.375  1.00 10.12           C\\nATOM   2261  O   PRO A 293      12.867   4.367 -11.401  1.00 10.12           O\\nATOM   2262  CB  PRO A 293      15.322   6.307 -10.271  1.00 10.12           C\\nATOM   2263  CG  PRO A 293      15.770   7.274  -9.180  1.00 10.12           C\\nATOM   2264  CD  PRO A 293      14.944   6.923  -7.948  1.00 10.12           C\\nATOM   2265  N   PHE A 294      13.900   3.434  -9.621  1.00 11.40           N\\nATOM   2266  CA  PHE A 294      13.542   2.043  -9.917  1.00 11.40           C\\nATOM   2267  C   PHE A 294      12.059   1.740  -9.639  1.00 11.40           C\\nATOM   2268  O   PHE A 294      11.476   0.973 -10.399  1.00 11.40           O\\nATOM   2269  CB  PHE A 294      14.414   1.070  -9.089  1.00 11.40           C\\nATOM   2270  CG  PHE A 294      15.879   1.018  -9.493  1.00 11.40           C\\nATOM   2271  CD1 PHE A 294      16.244   0.757 -10.832  1.00 11.40           C\\nATOM   2272  CD2 PHE A 294      16.888   1.243  -8.533  1.00 11.40           C\\nATOM   2273  CE1 PHE A 294      17.599   0.780 -11.216  1.00 11.40           C\\nATOM   2274  CE2 PHE A 294      18.241   1.291  -8.921  1.00 11.40           C\\nATOM   2275  CZ  PHE A 294      18.597   1.066 -10.264  1.00 11.40           C\\nATOM   2276  N   ASP A 295      11.427   2.358  -8.622  1.00 11.59           N\\nATOM   2277  CA  ASP A 295       9.990   2.222  -8.336  1.00 11.59           C\\nATOM   2278  C   ASP A 295       9.118   2.871  -9.411  1.00 11.59           C\\nATOM   2279  O   ASP A 295       8.076   2.317  -9.761  1.00 11.59           O\\nATOM   2280  CB  ASP A 295       9.599   2.896  -7.004  1.00 11.59           C\\nATOM   2281  CG  ASP A 295      10.271   2.353  -5.757  1.00 11.59           C\\nATOM   2282  OD1 ASP A 295      10.650   1.162  -5.737  1.00 11.59           O\\nATOM   2283  OD2 ASP A 295      10.362   3.159  -4.810  1.00 11.59           O\\nATOM   2284  N   VAL A 296       9.544   4.032  -9.934  1.00 11.32           N\\nATOM   2285  CA  VAL A 296       8.849   4.761 -10.989  1.00 11.32           C\\nATOM   2286  C   VAL A 296       8.894   3.970 -12.299  1.00 11.32           C\\nATOM   2287  O   VAL A 296       7.844   3.782 -12.903  1.00 11.32           O\\nATOM   2288  CB  VAL A 296       9.440   6.182 -11.139  1.00 11.32           C\\nATOM   2289  CG1 VAL A 296       8.842   6.941 -12.342  1.00 11.32           C\\nATOM   2290  CG2 VAL A 296       9.224   6.992  -9.848  1.00 11.32           C\\nATOM   2291  N   VAL A 297      10.072   3.453 -12.690  1.00 12.99           N\\nATOM   2292  CA  VAL A 297      10.244   2.542 -13.824  1.00 12.99           C\\nATOM   2293  C   VAL A 297       9.402   1.261 -13.666  1.00 12.99           C\\nATOM   2294  O   VAL A 297       8.770   0.851 -14.636  1.00 12.99           O\\nATOM   2295  CB  VAL A 297      11.757   2.257 -14.039  1.00 12.99           C\\nATOM   2296  CG1 VAL A 297      12.057   1.070 -14.978  1.00 12.99           C\\nATOM   2297  CG2 VAL A 297      12.474   3.511 -14.574  1.00 12.99           C\\nATOM   2298  N   ARG A 298       9.355   0.662 -12.460  1.00 12.33           N\\nATOM   2299  CA  ARG A 298       8.642  -0.586 -12.178  1.00 12.33           C\\nATOM   2300  C   ARG A 298       7.121  -0.432 -12.326  1.00 12.33           C\\nATOM   2301  O   ARG A 298       6.496  -1.273 -12.969  1.00 12.33           O\\nATOM   2302  CB  ARG A 298       8.975  -1.083 -10.753  1.00 12.33           C\\nATOM   2303  CG  ARG A 298       8.575  -2.551 -10.513  1.00 12.33           C\\nATOM   2304  CD  ARG A 298       8.732  -3.029  -9.067  1.00 12.33           C\\nATOM   2305  NE  ARG A 298      10.122  -2.865  -8.624  1.00 12.33           N\\nATOM   2306  CZ  ARG A 298      10.566  -1.955  -7.742  1.00 12.33           C\\nATOM   2307  NH1 ARG A 298      11.881  -1.830  -7.544  1.00 12.33           N\\nATOM   2308  NH2 ARG A 298       9.732  -1.151  -7.072  1.00 12.33           N\\nATOM   2309  N   GLN A 299       6.535   0.636 -11.755  1.00 14.69           N\\nATOM   2310  CA  GLN A 299       5.099   0.905 -11.805  1.00 14.69           C\\nATOM   2311  C   GLN A 299       4.658   1.444 -13.179  1.00 14.69           C\\nATOM   2312  O   GLN A 299       3.574   1.081 -13.635  1.00 14.69           O\\nATOM   2313  CB  GLN A 299       4.730   1.895 -10.677  1.00 14.69           C\\nATOM   2314  CG  GLN A 299       3.236   2.304 -10.662  1.00 14.69           C\\nATOM   2315  CD  GLN A 299       2.746   3.044  -9.415  1.00 14.69           C\\nATOM   2316  OE1 GLN A 299       1.565   3.370  -9.336  1.00 14.69           O\\nATOM   2317  NE2 GLN A 299       3.607   3.317  -8.436  1.00 14.69           N\\nATOM   2318  N   CYS A 300       5.486   2.258 -13.859  1.00 16.70           N\\nATOM   2319  CA  CYS A 300       5.185   2.806 -15.188  1.00 16.70           C\\nATOM   2320  C   CYS A 300       5.386   1.782 -16.326  1.00 16.70           C\\nATOM   2321  O   CYS A 300       4.904   2.036 -17.429  1.00 16.70           O\\nATOM   2322  CB  CYS A 300       6.073   4.038 -15.474  1.00 16.70           C\\nATOM   2323  SG  CYS A 300       5.538   5.463 -14.475  1.00 16.70           S\\nATOM   2324  N   SER A 301       6.038   0.630 -16.071  1.00 19.23           N\\nATOM   2325  CA  SER A 301       6.217  -0.470 -17.031  1.00 19.23           C\\nATOM   2326  C   SER A 301       5.471  -1.755 -16.623  1.00 19.23           C\\nATOM   2327  O   SER A 301       5.361  -2.664 -17.444  1.00 19.23           O\\nATOM   2328  CB  SER A 301       7.721  -0.719 -17.285  1.00 19.23           C\\nATOM   2329  OG  SER A 301       8.356  -1.403 -16.225  1.00 19.23           O\\nATOM   2330  N   GLY A 302       4.929  -1.824 -15.394  1.00 22.50           N\\nATOM   2331  CA  GLY A 302       4.077  -2.904 -14.903  1.00 22.50           C\\nATOM   2332  C   GLY A 302       4.788  -4.241 -14.657  1.00 22.50           C\\nATOM   2333  O   GLY A 302       4.243  -5.282 -15.022  1.00 22.50           O\\nATOM   2334  N   VAL A 303       5.990  -4.225 -14.049  1.00 19.43           N\\nATOM   2335  CA  VAL A 303       6.775  -5.424 -13.713  1.00 19.43           C\\nATOM   2336  C   VAL A 303       6.061  -6.245 -12.621  1.00 19.43           C\\nATOM   2337  O   VAL A 303       5.534  -5.684 -11.659  1.00 19.43           O\\nATOM   2338  CB  VAL A 303       8.190  -5.024 -13.215  1.00 19.43           C\\nATOM   2339  CG1 VAL A 303       9.098  -6.225 -12.878  1.00 19.43           C\\nATOM   2340  CG2 VAL A 303       8.924  -4.086 -14.186  1.00 19.43           C\\nATOM   2341  N   THR A 304       6.084  -7.576 -12.779  1.00 22.50           N\\nATOM   2342  CA  THR A 304       5.494  -8.561 -11.877  1.00 22.50           C\\nATOM   2343  C   THR A 304       6.612  -9.402 -11.242  1.00 22.50           C\\nATOM   2344  O   THR A 304       6.418 -10.262 -10.384  1.00 22.50           O\\nATOM   2345  CB  THR A 304       4.477  -9.431 -12.663  1.00 22.50           C\\nATOM   2346  OG1 THR A 304       5.106 -10.218 -13.659  1.00 22.50           O\\nATOM   2347  CG2 THR A 304       3.345  -8.616 -13.310  1.00 22.50           C\\n\",\"pdb\");\n",
       "\tviewer_15885999372043362.setStyle({\"cartoon\": {\"colorscheme\": {\"prop\": \"b\", \"gradient\": \"sinebow\", \"min\": 0, \"max\": 130}}});\n",
       "\tviewer_15885999372043362.setBackgroundColor(\"0xeeeeee\");\n",
       "\tviewer_15885999372043362.zoomTo();\n",
       "viewer_15885999372043362.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x1a27ac0090>"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "struct = non_cov_prot_structs[0]\n",
    "p = py3Dmol.view(500, 500)\n",
    "buff = StringIO()\n",
    "prody.writePDBStream(buff, struct)\n",
    "p.addModel(buff.getvalue(), 'pdb')\n",
    "p.setStyle({'cartoon': {'colorscheme':{'prop':'b','gradient':'sinebow','min':0,'max':130}}})\n",
    "p.setBackgroundColor('0xeeeeee')\n",
    "p.zoomTo()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Select diverse receptor structures for docking"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([140, 144, 163, 145, 142, 166, 189, 165,  49,  41])"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# identify residues that interact with ligands the most\n",
    "contacts = []\n",
    "for struct in non_cov_structs:\n",
    "    contact_atoms = struct.select('sidechain within 4 of resname LIG').copy()\n",
    "    contact_resnums = set(contact_atoms.getResnums())\n",
    "    contacts += list(contact_resnums)\n",
    "key_contacts = np.argsort(np.bincount(contacts))[-10:]\n",
    "key_contacts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "# extract coordinates of those residues in each structure\n",
    "selstr = f'resnum {\" \".join(key_contacts.astype(str))}'\n",
    "contact_resid_structs = [ s.select(selstr).copy() for s in non_cov_structs ]\n",
    "contact_resid_coords = [ s.getCoords() for s in contact_resid_structs ]\n",
    "contact_resid_coords_flat = np.vstack([ c.flatten() for c in contact_resid_coords ])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Define a function to calculate pairwise RMSD for all pairs of structures"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "def rmsd_matrix(structure_list):\n",
    "    rmsd_df = pd.DataFrame()\n",
    "\n",
    "    for i in range(len(structure_list)):\n",
    "        for j in range(i + 1):\n",
    "            struct1 = structure_list[i]\n",
    "            struct2 = structure_list[j]\n",
    "\n",
    "            rmsd = prody.calcRMSD(struct1, struct2)\n",
    "            rmsd_df.loc[\n",
    "                struct1.getTitle().split()[0],\n",
    "                struct2.getTitle().split()[0],\n",
    "            ] = rmsd\n",
    "\n",
    "    # reflect over diagonal\n",
    "    rmsd_df = rmsd_df.fillna(0) + rmsd_df.T.fillna(0)\n",
    "    return rmsd_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Cluster the active site residues and visualize the clusters.  Use the slider below to set the number of clusters. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "be3b3772a2dd4b5a83eadd530646d018",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "interactive(children=(IntSlider(value=3, description='n_clusters', max=6, min=1), Output()), _dom_classes=('wi…"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "color_list = ['red', 'blue', \"cyan\", \"green\", \"yellow\", \"purple\"]\n",
    "@interact(n_clusters=(1,len(color_list),1))\n",
    "def cluster_structs(n_clusters=3):\n",
    "    global num_clusters, cluster_ids, cluster_centers, rmsd_df\n",
    "    num_clusters = n_clusters\n",
    "    km = KMeans(n_clusters=n_clusters)\n",
    "    rmsd_df = rmsd_matrix(contact_resid_structs)\n",
    "    cluster_ids = km.fit_predict(rmsd_df.values)\n",
    "    cluster_centers = km.cluster_centers_\n",
    "\n",
    "    letters = string.ascii_uppercase[0:num_clusters]\n",
    "\n",
    "    cluster_chain_ids = [letters[x] for x in cluster_ids]\n",
    "    for struct, ch_id in zip(contact_resid_structs, cluster_chain_ids):\n",
    "        struct.setChids(ch_id)\n",
    "\n",
    "    cluster_dict = {}\n",
    "    for i in range(0, num_clusters):\n",
    "        chain_id = letters[i]\n",
    "        cluster_dict[chain_id] = {'chain': chain_id}\n",
    "\n",
    "    p = py3Dmol.view(500, 500)\n",
    "    for struct in contact_resid_structs:\n",
    "        buff = StringIO()\n",
    "        prody.writePDBStream(buff, struct)\n",
    "        p.addModel(buff.getvalue(), 'pdb')\n",
    "\n",
    "    for i in range(0, num_clusters):\n",
    "        chain_id = letters[i]\n",
    "        ch = {'chain': chain_id}\n",
    "        p.setStyle(ch, {'stick': {'color': color_list[i]}})\n",
    "\n",
    "    p.setBackgroundColor('0xeeeeee')\n",
    "    p.zoomTo()\n",
    "    return p"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Find the structure closest to each cluster center.  Calculate the Euclindean distance between each cluster center adn the RMS values for each ligand."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dm = cdist(rmsd_df.values,cluster_centers)\n",
    "min_idx = np.argmin(dm,axis=0)\n",
    "center_list = [0] * dm.shape[0]\n",
    "for x in min_idx:\n",
    "    center_list[x] = 1\n",
    "center_list"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Display the cluster centers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_15885999374619782\"  style=\"position: relative; width: 500px; height: 500px\">\n        <p id=\"3dmolwarning_15885999374619782\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.csb.pitt.edu/build/3Dmol.js');\n}\n\nvar viewer_15885999374619782 = null;\nvar warn = document.getElementById(\"3dmolwarning_15885999374619782\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_15885999374619782 = $3Dmol.createViewer($(\"#3dmolviewer_15885999374619782\"),{backgroundColor:\"white\"});\n\tviewer_15885999374619782.addModel(\"REMARK AtomGroup Mpro-x1249_1 Selection 'heavy' Selection 'resnum 140 144 ...6 189 165 49 41'\\nATOM      1  N   HIS B  41      14.650  -7.926  22.230  1.00 13.87           N\\nATOM      2  CA  HIS B  41      13.823  -7.340  23.285  1.00 15.60           C\\nATOM      3  C   HIS B  41      12.545  -8.129  23.627  1.00 16.87           C\\nATOM      4  O   HIS B  41      11.844  -7.744  24.560  1.00 16.77           O\\nATOM      5  CB  HIS B  41      13.534  -5.854  22.982  1.00 14.47           C\\nATOM      6  CG  HIS B  41      12.515  -5.491  21.928  1.00 15.42           C\\nATOM      7  ND1 HIS B  41      11.142  -5.684  22.109  1.00 17.40           N\\nATOM      8  CD2 HIS B  41      12.708  -4.853  20.722  1.00 17.40           C\\nATOM      9  CE1 HIS B  41      10.572  -5.164  21.034  1.00 15.33           C\\nATOM     10  NE2 HIS B  41      11.455  -4.661  20.175  1.00 19.10           N\\nATOM     11  N   MET B  49      13.706  -2.787  29.739  1.00 29.98           N\\nATOM     12  CA  MET B  49      13.943  -1.912  28.590  1.00 29.15           C\\nATOM     13  C   MET B  49      14.609  -0.571  28.927  1.00 26.60           C\\nATOM     14  O   MET B  49      15.189   0.036  28.031  1.00 25.80           O\\nATOM     15  CB  MET B  49      12.623  -1.621  27.837  1.00 30.26           C\\nATOM     16  CG  MET B  49      12.082  -2.787  27.000  1.00 32.45           C\\nATOM     17  SD  MET B  49      13.239  -3.579  25.851  1.00 33.44           S\\nATOM     18  CE  MET B  49      14.050  -2.165  25.074  1.00 33.54           C\\nATOM     19  N   PHE B 140       2.896   1.636  13.365  1.00 17.93           N\\nATOM     20  CA  PHE B 140       3.332   0.607  14.298  1.00 20.03           C\\nATOM     21  C   PHE B 140       2.752   0.916  15.684  1.00 21.14           C\\nATOM     22  O   PHE B 140       3.115   1.923  16.289  1.00 22.91           O\\nATOM     23  CB  PHE B 140       4.874   0.574  14.320  1.00 20.30           C\\nATOM     24  CG  PHE B 140       5.592   0.114  13.058  1.00 18.87           C\\nATOM     25  CD1 PHE B 140       5.555  -1.229  12.630  1.00 19.74           C\\nATOM     26  CD2 PHE B 140       6.370   1.038  12.337  1.00 17.90           C\\nATOM     27  CE1 PHE B 140       6.335  -1.648  11.533  1.00 17.89           C\\nATOM     28  CE2 PHE B 140       7.166   0.623  11.256  1.00 18.27           C\\nATOM     29  CZ  PHE B 140       7.162  -0.726  10.863  1.00 18.40           C\\nATOM     30  N   ASN B 142       1.821  -1.198  19.668  0.56 26.48           N\\nATOM     31  CA  ASN B 142       2.417  -2.243  20.507  0.56 26.68           C\\nATOM     32  C   ASN B 142       1.965  -3.636  20.021  0.56 26.02           C\\nATOM     33  O   ASN B 142       0.793  -3.851  19.711  0.56 25.71           O\\nATOM     34  CB  ASN B 142       2.011  -2.024  21.980  0.56 27.74           C\\nATOM     35  CG  ASN B 142       2.840  -2.883  22.945  0.56 28.63           C\\nATOM     36  OD1 ASN B 142       4.068  -2.885  22.888  0.56 30.53           O\\nATOM     37  ND2 ASN B 142       2.172  -3.663  23.796  0.56 29.52           N\\nATOM     38  N   SER B 144       3.668  -4.829  17.280  1.00 18.03           N\\nATOM     39  CA  SER B 144       4.114  -4.764  15.886  1.00 17.43           C\\nATOM     40  C   SER B 144       5.615  -5.010  15.690  1.00 15.69           C\\nATOM     41  O   SER B 144       6.051  -5.093  14.545  1.00 14.72           O\\nATOM     42  CB  SER B 144       3.764  -3.393  15.281  1.00 18.29           C\\nATOM     43  OG  SER B 144       4.633  -2.367  15.707  1.00 19.08           O\\nATOM     44  N   CYS B 145       6.413  -5.132  16.761  1.00 15.69           N\\nATOM     45  CA  CYS B 145       7.847  -5.401  16.658  1.00 15.29           C\\nATOM     46  C   CYS B 145       8.093  -6.788  16.050  1.00 13.30           C\\nATOM     47  O   CYS B 145       7.323  -7.720  16.274  1.00 14.53           O\\nATOM     48  CB  CYS B 145       8.502  -5.268  18.037  1.00 16.75           C\\nATOM     49  SG  CYS B 145       8.473  -3.516  18.490  1.00 20.24           S\\nATOM     50  N   HIS B 163      11.190  -4.252  12.514  1.00 12.38           N\\nATOM     51  CA  HIS B 163      10.801  -3.170  13.411  1.00 12.40           C\\nATOM     52  C   HIS B 163      11.144  -3.530  14.862  1.00 12.46           C\\nATOM     53  O   HIS B 163      10.901  -4.654  15.299  1.00 13.08           O\\nATOM     54  CB  HIS B 163       9.284  -2.937  13.300  1.00 13.31           C\\nATOM     55  CG  HIS B 163       8.840  -1.750  14.115  1.00 12.88           C\\nATOM     56  ND1 HIS B 163       9.347  -0.466  13.887  1.00 13.44           N\\nATOM     57  CD2 HIS B 163       7.975  -1.709  15.186  1.00 12.81           C\\nATOM     58  CE1 HIS B 163       8.800   0.277  14.838  1.00 13.91           C\\nATOM     59  NE2 HIS B 163       7.982  -0.403  15.637  1.00 13.31           N\\nATOM     60  N   MET B 165      11.529  -0.379  17.421  0.56 17.34           N\\nATOM     61  CA  MET B 165      11.426   0.789  18.284  0.56 18.49           C\\nATOM     62  C   MET B 165      10.636   1.913  17.626  0.56 18.32           C\\nATOM     63  O   MET B 165      10.825   2.195  16.447  0.56 17.48           O\\nATOM     64  CB  MET B 165      12.853   1.213  18.681  0.56 19.37           C\\nATOM     65  CG  MET B 165      12.911   2.345  19.713  0.56 20.63           C\\nATOM     66  SD  MET B 165      14.534   2.586  20.470  0.56 22.61           S\\nATOM     67  CE  MET B 165      15.533   2.916  19.001  0.56 21.50           C\\nATOM     68  N   GLU B 166       9.813   2.601  18.425  1.00 18.20           N\\nATOM     69  CA  GLU B 166       9.285   3.926  18.137  1.00 18.85           C\\nATOM     70  C   GLU B 166      10.041   4.904  19.047  1.00 19.98           C\\nATOM     71  O   GLU B 166      10.109   4.693  20.258  1.00 22.11           O\\nATOM     72  CB  GLU B 166       7.772   3.935  18.447  1.00 20.82           C\\nATOM     73  CG  GLU B 166       7.080   5.268  18.103  1.00 23.10           C\\nATOM     74  CD  GLU B 166       5.557   5.147  18.125  1.00 25.20           C\\nATOM     75  OE1 GLU B 166       4.988   5.336  19.218  1.00 28.14           O\\nATOM     76  OE2 GLU B 166       4.980   4.872  17.051  1.00 25.14           O\\nATOM     77  N   GLN B 189      14.865   1.907  25.997  1.00 25.35           N\\nATOM     78  CA  GLN B 189      13.973   3.017  26.312  1.00 27.27           C\\nATOM     79  C   GLN B 189      14.804   4.102  27.019  1.00 27.08           C\\nATOM     80  O   GLN B 189      14.929   4.126  28.243  1.00 28.41           O\\nATOM     81  CB  GLN B 189      12.833   2.496  27.216  1.00 30.78           C\\nATOM     82  CG  GLN B 189      11.769   3.570  27.537  1.00 34.40           C\\nATOM     83  CD  GLN B 189      10.614   3.102  28.430  1.00 39.02           C\\nATOM     84  OE1 GLN B 189       9.799   3.924  28.840  1.00 47.54           O\\nATOM     85  NE2 GLN B 189      10.530   1.810  28.758  1.00 42.29           N\\n\",\"pdb\");\n\tviewer_15885999374619782.addModel(\"REMARK AtomGroup Mpro-x0161_1 Selection 'heavy' Selection 'resnum 140 144 ...6 189 165 49 41'\\nATOM      1  N   HIS C  41      14.763  -7.835  22.045  1.00 28.84           N\\nATOM      2  CA  HIS C  41      13.968  -7.276  23.142  1.00 31.13           C\\nATOM      3  C   HIS C  41      12.691  -8.076  23.472  1.00 32.24           C\\nATOM      4  O   HIS C  41      11.921  -7.646  24.329  1.00 29.90           O\\nATOM      5  CB  HIS C  41      13.664  -5.781  22.899  1.00 34.50           C\\nATOM      6  CG  HIS C  41      12.682  -5.411  21.814  1.00 34.53           C\\nATOM      7  ND1 HIS C  41      11.338  -5.746  21.891  1.00 38.93           N\\nATOM      8  CD2 HIS C  41      12.840  -4.642  20.677  1.00 39.19           C\\nATOM      9  CE1 HIS C  41      10.736  -5.201  20.842  1.00 38.96           C\\nATOM     10  NE2 HIS C  41      11.599  -4.543  20.075  1.00 39.05           N\\nATOM     11  N   MET C  49      14.132  -2.640  30.082  0.64 60.16           N\\nATOM     12  CA  MET C  49      14.134  -1.739  28.933  0.64 58.53           C\\nATOM     13  C   MET C  49      14.990  -0.465  29.054  0.64 53.68           C\\nATOM     14  O   MET C  49      15.429   0.028  28.019  0.64 49.11           O\\nATOM     15  CB  MET C  49      12.666  -1.419  28.575  0.64 60.88           C\\nATOM     16  CG  MET C  49      12.017  -2.529  27.730  0.64 63.12           C\\nATOM     17  SD  MET C  49      10.204  -2.586  27.717  0.64 68.43           S\\nATOM     18  CE  MET C  49       9.790  -0.927  27.131  0.64 66.21           C\\nATOM     19  N   PHE C 140       3.007   1.744  13.384  1.00 37.49           N\\nATOM     20  CA  PHE C 140       3.463   0.698  14.293  1.00 38.56           C\\nATOM     21  C   PHE C 140       2.932   0.989  15.704  1.00 39.07           C\\nATOM     22  O   PHE C 140       3.370   1.951  16.334  1.00 41.58           O\\nATOM     23  CB  PHE C 140       5.007   0.642  14.254  1.00 37.63           C\\nATOM     24  CG  PHE C 140       5.669   0.159  12.971  1.00 36.48           C\\nATOM     25  CD1 PHE C 140       5.693  -1.206  12.620  1.00 39.45           C\\nATOM     26  CD2 PHE C 140       6.337   1.086  12.148  1.00 38.54           C\\nATOM     27  CE1 PHE C 140       6.421  -1.642  11.496  1.00 37.66           C\\nATOM     28  CE2 PHE C 140       7.089   0.654  11.043  1.00 37.04           C\\nATOM     29  CZ  PHE C 140       7.149  -0.715  10.727  1.00 36.39           C\\nATOM     30  N   ASN C 142       1.921  -1.071  19.698  1.00 47.30           N\\nATOM     31  CA  ASN C 142       2.495  -2.105  20.568  1.00 46.42           C\\nATOM     32  C   ASN C 142       2.176  -3.512  20.032  1.00 41.58           C\\nATOM     33  O   ASN C 142       1.047  -3.786  19.622  1.00 39.86           O\\nATOM     34  CB  ASN C 142       1.965  -1.974  22.015  1.00 50.77           C\\nATOM     35  CG  ASN C 142       2.443  -0.711  22.741  1.00 59.66           C\\nATOM     36  OD1 ASN C 142       1.632   0.092  23.194  1.00 66.65           O\\nATOM     37  ND2 ASN C 142       3.759  -0.556  22.907  1.00 62.03           N\\nATOM     38  N   SER C 144       3.831  -4.656  17.318  1.00 36.24           N\\nATOM     39  CA  SER C 144       4.196  -4.585  15.901  1.00 35.20           C\\nATOM     40  C   SER C 144       5.670  -4.921  15.625  1.00 33.83           C\\nATOM     41  O   SER C 144       6.043  -5.084  14.467  1.00 35.46           O\\nATOM     42  CB  SER C 144       3.911  -3.172  15.367  1.00 34.73           C\\nATOM     43  OG  SER C 144       4.879  -2.232  15.782  1.00 32.65           O\\nATOM     44  N   CYS C 145       6.510  -5.013  16.667  1.00 35.69           N\\nATOM     45  CA  CYS C 145       7.941  -5.291  16.575  1.00 35.66           C\\nATOM     46  C   CYS C 145       8.189  -6.669  15.946  1.00 32.33           C\\nATOM     47  O   CYS C 145       7.437  -7.611  16.183  1.00 30.52           O\\nATOM     48  CB  CYS C 145       8.558  -5.202  17.979  1.00 38.48           C\\nATOM     49  SG  CYS C 145       8.369  -3.512  18.618  1.00 40.38           S\\nATOM     50  N   HIS C 163      11.350  -4.191  12.299  1.00 27.50           N\\nATOM     51  CA  HIS C 163      10.955  -3.120  13.203  1.00 26.43           C\\nATOM     52  C   HIS C 163      11.315  -3.487  14.648  1.00 27.22           C\\nATOM     53  O   HIS C 163      11.045  -4.603  15.091  1.00 25.19           O\\nATOM     54  CB  HIS C 163       9.437  -2.889  13.089  1.00 26.25           C\\nATOM     55  CG  HIS C 163       9.008  -1.675  13.870  1.00 26.44           C\\nATOM     56  ND1 HIS C 163       8.431  -1.740  15.126  1.00 27.07           N\\nATOM     57  CD2 HIS C 163       9.177  -0.335  13.608  1.00 27.92           C\\nATOM     58  CE1 HIS C 163       8.313  -0.485  15.565  1.00 27.90           C\\nATOM     59  NE2 HIS C 163       8.740   0.419  14.696  1.00 26.74           N\\nATOM     60  N   MET C 165      11.606  -0.426  17.305  0.64 32.66           N\\nATOM     61  CA  MET C 165      11.171   0.628  18.212  0.64 33.02           C\\nATOM     62  C   MET C 165      10.604   1.843  17.491  0.64 33.31           C\\nATOM     63  O   MET C 165      10.932   2.115  16.338  0.64 31.72           O\\nATOM     64  CB  MET C 165      12.305   0.995  19.194  0.64 32.18           C\\nATOM     65  CG  MET C 165      13.527   1.608  18.500  0.64 32.64           C\\nATOM     66  SD  MET C 165      14.935   1.818  19.598  0.64 34.74           S\\nATOM     67  CE  MET C 165      15.999   2.711  18.462  0.64 34.00           C\\nATOM     68  N   GLU C 166       9.799   2.603  18.237  1.00 34.87           N\\nATOM     69  CA  GLU C 166       9.390   3.960  17.932  1.00 35.31           C\\nATOM     70  C   GLU C 166      10.244   4.887  18.814  1.00 33.74           C\\nATOM     71  O   GLU C 166      10.412   4.629  20.006  1.00 31.96           O\\nATOM     72  CB  GLU C 166       7.895   4.065  18.291  1.00 41.50           C\\nATOM     73  CG  GLU C 166       7.224   5.383  17.866  1.00 47.32           C\\nATOM     74  CD  GLU C 166       5.698   5.270  17.876  1.00 48.16           C\\nATOM     75  OE1 GLU C 166       5.156   4.929  18.949  1.00 54.03           O\\nATOM     76  OE2 GLU C 166       5.095   5.517  16.810  1.00 44.24           O\\nATOM     77  N   GLN C 189      15.262   1.844  25.918  0.64 42.39           N\\nATOM     78  CA  GLN C 189      14.408   2.980  26.242  0.64 44.21           C\\nATOM     79  C   GLN C 189      15.259   4.096  26.861  0.64 43.15           C\\nATOM     80  O   GLN C 189      15.527   4.109  28.062  0.64 42.40           O\\nATOM     81  CB  GLN C 189      13.306   2.507  27.216  0.64 45.84           C\\nATOM     82  CG  GLN C 189      12.311   3.616  27.611  0.64 47.89           C\\nATOM     83  CD  GLN C 189      11.076   3.090  28.344  0.64 49.07           C\\nATOM     84  OE1 GLN C 189      11.099   2.015  28.938  0.64 51.43           O\\nATOM     85  NE2 GLN C 189       9.984   3.855  28.299  0.64 50.03           N\\n\",\"pdb\");\n\tviewer_15885999374619782.addModel(\"REMARK AtomGroup Mpro-x0946_1 Selection 'heavy' Selection 'resnum 140 144 ...6 189 165 49 41'\\nATOM      1  N   HIS A  41      14.710  -7.840  22.317  1.00 18.32           N\\nATOM      2  CA  HIS A  41      13.913  -7.244  23.394  1.00 19.86           C\\nATOM      3  C   HIS A  41      12.626  -8.012  23.747  1.00 20.66           C\\nATOM      4  O   HIS A  41      11.915  -7.600  24.664  1.00 21.18           O\\nATOM      5  CB  HIS A  41      13.630  -5.755  23.098  1.00 19.27           C\\nATOM      6  CG  HIS A  41      12.673  -5.388  21.990  1.00 20.09           C\\nATOM      7  ND1 HIS A  41      11.352  -5.847  21.940  1.00 22.19           N\\nATOM      8  CD2 HIS A  41      12.865  -4.496  20.956  1.00 20.05           C\\nATOM      9  CE1 HIS A  41      10.816  -5.249  20.891  1.00 19.82           C\\nATOM     10  NE2 HIS A  41      11.674  -4.454  20.260  1.00 22.03           N\\nATOM     11  N   MET A  49      13.882  -2.919  29.927  1.00 31.82           N\\nATOM     12  CA  MET A  49      14.031  -2.035  28.771  1.00 31.74           C\\nATOM     13  C   MET A  49      14.725  -0.704  29.077  1.00 29.69           C\\nATOM     14  O   MET A  49      15.306  -0.118  28.168  1.00 24.34           O\\nATOM     15  CB  MET A  49      12.656  -1.728  28.133  1.00 32.88           C\\nATOM     16  CG  MET A  49      12.059  -2.863  27.293  1.00 35.06           C\\nATOM     17  SD  MET A  49      13.141  -3.584  26.031  1.00 31.94           S\\nATOM     18  CE  MET A  49      13.877  -2.123  25.263  1.00 35.13           C\\nATOM     19  N   PHE A 140       2.880   1.651  13.435  1.00 22.54           N\\nATOM     20  CA  PHE A 140       3.310   0.616  14.365  1.00 24.46           C\\nATOM     21  C   PHE A 140       2.727   0.922  15.751  1.00 26.90           C\\nATOM     22  O   PHE A 140       3.087   1.930  16.357  1.00 26.45           O\\nATOM     23  CB  PHE A 140       4.854   0.563  14.373  1.00 23.49           C\\nATOM     24  CG  PHE A 140       5.549   0.099  13.101  1.00 21.45           C\\nATOM     25  CD1 PHE A 140       5.491  -1.246  12.677  1.00 22.18           C\\nATOM     26  CD2 PHE A 140       6.323   1.014  12.362  1.00 22.01           C\\nATOM     27  CE1 PHE A 140       6.251  -1.678  11.573  1.00 21.32           C\\nATOM     28  CE2 PHE A 140       7.097   0.585  11.270  1.00 21.15           C\\nATOM     29  CZ  PHE A 140       7.081  -0.768  10.889  1.00 21.40           C\\nATOM     30  N   ASN A 142       1.784  -1.175  19.726  1.00 31.09           N\\nATOM     31  CA  ASN A 142       2.337  -2.224  20.597  1.00 30.71           C\\nATOM     32  C   ASN A 142       1.949  -3.637  20.116  1.00 26.11           C\\nATOM     33  O   ASN A 142       0.788  -3.897  19.803  1.00 25.70           O\\nATOM     34  CB  ASN A 142       1.892  -2.022  22.065  1.00 31.96           C\\nATOM     35  CG  ASN A 142       2.321  -0.699  22.714  1.00 36.20           C\\nATOM     36  OD1 ASN A 142       1.608  -0.174  23.565  1.00 37.08           O\\nATOM     37  ND2 ASN A 142       3.492  -0.165  22.361  1.00 39.70           N\\nATOM     38  N   SER A 144       3.686  -4.828  17.345  1.00 22.28           N\\nATOM     39  CA  SER A 144       4.121  -4.799  15.944  1.00 21.42           C\\nATOM     40  C   SER A 144       5.628  -5.001  15.735  1.00 19.66           C\\nATOM     41  O   SER A 144       6.053  -5.102  14.586  1.00 19.27           O\\nATOM     42  CB  SER A 144       3.715  -3.470  15.284  1.00 21.85           C\\nATOM     43  OG  SER A 144       4.524  -2.393  15.698  1.00 22.80           O\\nATOM     44  N   CYS A 145       6.439  -5.082  16.801  1.00 19.64           N\\nATOM     45  CA  CYS A 145       7.873  -5.357  16.705  1.00 18.47           C\\nATOM     46  C   CYS A 145       8.120  -6.743  16.088  1.00 17.42           C\\nATOM     47  O   CYS A 145       7.356  -7.679  16.319  1.00 16.69           O\\nATOM     48  CB  CYS A 145       8.510  -5.245  18.096  1.00 20.79           C\\nATOM     49  SG  CYS A 145       8.524  -3.507  18.605  1.00 21.75           S\\nATOM     50  N   HIS A 163      11.286  -4.233  12.499  1.00 17.71           N\\nATOM     51  CA  HIS A 163      10.871  -3.163  13.395  1.00 17.35           C\\nATOM     52  C   HIS A 163      11.219  -3.511  14.849  1.00 19.33           C\\nATOM     53  O   HIS A 163      10.938  -4.617  15.307  1.00 16.74           O\\nATOM     54  CB  HIS A 163       9.351  -2.950  13.262  1.00 17.29           C\\nATOM     55  CG  HIS A 163       8.925  -1.726  14.024  1.00 16.00           C\\nATOM     56  ND1 HIS A 163       8.412  -1.774  15.307  1.00 16.24           N\\nATOM     57  CD2 HIS A 163       9.078  -0.391  13.733  1.00 15.40           C\\nATOM     58  CE1 HIS A 163       8.349  -0.517  15.741  1.00 16.45           C\\nATOM     59  NE2 HIS A 163       8.712   0.378  14.834  1.00 15.89           N\\nATOM     60  N   MET A 165      11.468  -0.399  17.440  0.66 22.15           N\\nATOM     61  CA  MET A 165      11.117   0.684  18.354  0.66 23.00           C\\nATOM     62  C   MET A 165      10.518   1.899  17.661  0.66 22.25           C\\nATOM     63  O   MET A 165      10.756   2.140  16.480  0.66 20.46           O\\nATOM     64  CB  MET A 165      12.335   1.055  19.227  0.66 23.71           C\\nATOM     65  CG  MET A 165      13.494   1.678  18.438  0.66 24.59           C\\nATOM     66  SD  MET A 165      15.076   1.627  19.299  0.66 27.43           S\\nATOM     67  CE  MET A 165      15.516   3.376  19.245  0.66 26.35           C\\nATOM     68  N   GLU A 166       9.803   2.693  18.461  1.00 22.09           N\\nATOM     69  CA  GLU A 166       9.390   4.052  18.167  1.00 22.61           C\\nATOM     70  C   GLU A 166      10.228   4.980  19.057  1.00 21.48           C\\nATOM     71  O   GLU A 166      10.400   4.712  20.247  1.00 22.84           O\\nATOM     72  CB  GLU A 166       7.890   4.152  18.496  1.00 24.75           C\\nATOM     73  CG  GLU A 166       7.200   5.455  18.079  1.00 28.35           C\\nATOM     74  CD  GLU A 166       5.680   5.325  18.176  1.00 30.02           C\\nATOM     75  OE1 GLU A 166       5.174   5.454  19.312  1.00 32.16           O\\nATOM     76  OE2 GLU A 166       5.046   5.087  17.126  1.00 26.42           O\\nATOM     77  N   GLN A 189      14.918   1.700  26.112  1.00 25.38           N\\nATOM     78  CA  GLN A 189      14.000   2.767  26.491  1.00 27.66           C\\nATOM     79  C   GLN A 189      14.797   3.869  27.207  1.00 25.80           C\\nATOM     80  O   GLN A 189      14.919   3.909  28.431  1.00 30.93           O\\nATOM     81  CB  GLN A 189      12.872   2.185  27.370  1.00 30.39           C\\nATOM     82  CG  GLN A 189      11.847   3.244  27.835  1.00 33.53           C\\nATOM     83  CD  GLN A 189      10.540   2.699  28.420  1.00 37.04           C\\nATOM     84  OE1 GLN A 189       9.701   3.485  28.853  1.00 45.65           O\\nATOM     85  NE2 GLN A 189      10.343   1.379  28.456  1.00 41.56           N\\n\",\"pdb\");\n\tviewer_15885999374619782.setStyle({\"chain\": \"A\"},{\"stick\": {\"color\": \"red\"}});\n\tviewer_15885999374619782.setStyle({\"chain\": \"B\"},{\"stick\": {\"color\": \"blue\"}});\n\tviewer_15885999374619782.setStyle({\"chain\": \"C\"},{\"stick\": {\"color\": \"cyan\"}});\n\tviewer_15885999374619782.setBackgroundColor(\"0xeeeeee\");\n\tviewer_15885999374619782.zoomTo();\nviewer_15885999374619782.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_15885999374619782\"  style=\"position: relative; width: 500px; height: 500px\">\n",
       "        <p id=\"3dmolwarning_15885999374619782\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "      resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.csb.pitt.edu/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_15885999374619782 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_15885999374619782\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_15885999374619782 = $3Dmol.createViewer($(\"#3dmolviewer_15885999374619782\"),{backgroundColor:\"white\"});\n",
       "\tviewer_15885999374619782.addModel(\"REMARK AtomGroup Mpro-x1249_1 Selection 'heavy' Selection 'resnum 140 144 ...6 189 165 49 41'\\nATOM      1  N   HIS B  41      14.650  -7.926  22.230  1.00 13.87           N\\nATOM      2  CA  HIS B  41      13.823  -7.340  23.285  1.00 15.60           C\\nATOM      3  C   HIS B  41      12.545  -8.129  23.627  1.00 16.87           C\\nATOM      4  O   HIS B  41      11.844  -7.744  24.560  1.00 16.77           O\\nATOM      5  CB  HIS B  41      13.534  -5.854  22.982  1.00 14.47           C\\nATOM      6  CG  HIS B  41      12.515  -5.491  21.928  1.00 15.42           C\\nATOM      7  ND1 HIS B  41      11.142  -5.684  22.109  1.00 17.40           N\\nATOM      8  CD2 HIS B  41      12.708  -4.853  20.722  1.00 17.40           C\\nATOM      9  CE1 HIS B  41      10.572  -5.164  21.034  1.00 15.33           C\\nATOM     10  NE2 HIS B  41      11.455  -4.661  20.175  1.00 19.10           N\\nATOM     11  N   MET B  49      13.706  -2.787  29.739  1.00 29.98           N\\nATOM     12  CA  MET B  49      13.943  -1.912  28.590  1.00 29.15           C\\nATOM     13  C   MET B  49      14.609  -0.571  28.927  1.00 26.60           C\\nATOM     14  O   MET B  49      15.189   0.036  28.031  1.00 25.80           O\\nATOM     15  CB  MET B  49      12.623  -1.621  27.837  1.00 30.26           C\\nATOM     16  CG  MET B  49      12.082  -2.787  27.000  1.00 32.45           C\\nATOM     17  SD  MET B  49      13.239  -3.579  25.851  1.00 33.44           S\\nATOM     18  CE  MET B  49      14.050  -2.165  25.074  1.00 33.54           C\\nATOM     19  N   PHE B 140       2.896   1.636  13.365  1.00 17.93           N\\nATOM     20  CA  PHE B 140       3.332   0.607  14.298  1.00 20.03           C\\nATOM     21  C   PHE B 140       2.752   0.916  15.684  1.00 21.14           C\\nATOM     22  O   PHE B 140       3.115   1.923  16.289  1.00 22.91           O\\nATOM     23  CB  PHE B 140       4.874   0.574  14.320  1.00 20.30           C\\nATOM     24  CG  PHE B 140       5.592   0.114  13.058  1.00 18.87           C\\nATOM     25  CD1 PHE B 140       5.555  -1.229  12.630  1.00 19.74           C\\nATOM     26  CD2 PHE B 140       6.370   1.038  12.337  1.00 17.90           C\\nATOM     27  CE1 PHE B 140       6.335  -1.648  11.533  1.00 17.89           C\\nATOM     28  CE2 PHE B 140       7.166   0.623  11.256  1.00 18.27           C\\nATOM     29  CZ  PHE B 140       7.162  -0.726  10.863  1.00 18.40           C\\nATOM     30  N   ASN B 142       1.821  -1.198  19.668  0.56 26.48           N\\nATOM     31  CA  ASN B 142       2.417  -2.243  20.507  0.56 26.68           C\\nATOM     32  C   ASN B 142       1.965  -3.636  20.021  0.56 26.02           C\\nATOM     33  O   ASN B 142       0.793  -3.851  19.711  0.56 25.71           O\\nATOM     34  CB  ASN B 142       2.011  -2.024  21.980  0.56 27.74           C\\nATOM     35  CG  ASN B 142       2.840  -2.883  22.945  0.56 28.63           C\\nATOM     36  OD1 ASN B 142       4.068  -2.885  22.888  0.56 30.53           O\\nATOM     37  ND2 ASN B 142       2.172  -3.663  23.796  0.56 29.52           N\\nATOM     38  N   SER B 144       3.668  -4.829  17.280  1.00 18.03           N\\nATOM     39  CA  SER B 144       4.114  -4.764  15.886  1.00 17.43           C\\nATOM     40  C   SER B 144       5.615  -5.010  15.690  1.00 15.69           C\\nATOM     41  O   SER B 144       6.051  -5.093  14.545  1.00 14.72           O\\nATOM     42  CB  SER B 144       3.764  -3.393  15.281  1.00 18.29           C\\nATOM     43  OG  SER B 144       4.633  -2.367  15.707  1.00 19.08           O\\nATOM     44  N   CYS B 145       6.413  -5.132  16.761  1.00 15.69           N\\nATOM     45  CA  CYS B 145       7.847  -5.401  16.658  1.00 15.29           C\\nATOM     46  C   CYS B 145       8.093  -6.788  16.050  1.00 13.30           C\\nATOM     47  O   CYS B 145       7.323  -7.720  16.274  1.00 14.53           O\\nATOM     48  CB  CYS B 145       8.502  -5.268  18.037  1.00 16.75           C\\nATOM     49  SG  CYS B 145       8.473  -3.516  18.490  1.00 20.24           S\\nATOM     50  N   HIS B 163      11.190  -4.252  12.514  1.00 12.38           N\\nATOM     51  CA  HIS B 163      10.801  -3.170  13.411  1.00 12.40           C\\nATOM     52  C   HIS B 163      11.144  -3.530  14.862  1.00 12.46           C\\nATOM     53  O   HIS B 163      10.901  -4.654  15.299  1.00 13.08           O\\nATOM     54  CB  HIS B 163       9.284  -2.937  13.300  1.00 13.31           C\\nATOM     55  CG  HIS B 163       8.840  -1.750  14.115  1.00 12.88           C\\nATOM     56  ND1 HIS B 163       9.347  -0.466  13.887  1.00 13.44           N\\nATOM     57  CD2 HIS B 163       7.975  -1.709  15.186  1.00 12.81           C\\nATOM     58  CE1 HIS B 163       8.800   0.277  14.838  1.00 13.91           C\\nATOM     59  NE2 HIS B 163       7.982  -0.403  15.637  1.00 13.31           N\\nATOM     60  N   MET B 165      11.529  -0.379  17.421  0.56 17.34           N\\nATOM     61  CA  MET B 165      11.426   0.789  18.284  0.56 18.49           C\\nATOM     62  C   MET B 165      10.636   1.913  17.626  0.56 18.32           C\\nATOM     63  O   MET B 165      10.825   2.195  16.447  0.56 17.48           O\\nATOM     64  CB  MET B 165      12.853   1.213  18.681  0.56 19.37           C\\nATOM     65  CG  MET B 165      12.911   2.345  19.713  0.56 20.63           C\\nATOM     66  SD  MET B 165      14.534   2.586  20.470  0.56 22.61           S\\nATOM     67  CE  MET B 165      15.533   2.916  19.001  0.56 21.50           C\\nATOM     68  N   GLU B 166       9.813   2.601  18.425  1.00 18.20           N\\nATOM     69  CA  GLU B 166       9.285   3.926  18.137  1.00 18.85           C\\nATOM     70  C   GLU B 166      10.041   4.904  19.047  1.00 19.98           C\\nATOM     71  O   GLU B 166      10.109   4.693  20.258  1.00 22.11           O\\nATOM     72  CB  GLU B 166       7.772   3.935  18.447  1.00 20.82           C\\nATOM     73  CG  GLU B 166       7.080   5.268  18.103  1.00 23.10           C\\nATOM     74  CD  GLU B 166       5.557   5.147  18.125  1.00 25.20           C\\nATOM     75  OE1 GLU B 166       4.988   5.336  19.218  1.00 28.14           O\\nATOM     76  OE2 GLU B 166       4.980   4.872  17.051  1.00 25.14           O\\nATOM     77  N   GLN B 189      14.865   1.907  25.997  1.00 25.35           N\\nATOM     78  CA  GLN B 189      13.973   3.017  26.312  1.00 27.27           C\\nATOM     79  C   GLN B 189      14.804   4.102  27.019  1.00 27.08           C\\nATOM     80  O   GLN B 189      14.929   4.126  28.243  1.00 28.41           O\\nATOM     81  CB  GLN B 189      12.833   2.496  27.216  1.00 30.78           C\\nATOM     82  CG  GLN B 189      11.769   3.570  27.537  1.00 34.40           C\\nATOM     83  CD  GLN B 189      10.614   3.102  28.430  1.00 39.02           C\\nATOM     84  OE1 GLN B 189       9.799   3.924  28.840  1.00 47.54           O\\nATOM     85  NE2 GLN B 189      10.530   1.810  28.758  1.00 42.29           N\\n\",\"pdb\");\n",
       "\tviewer_15885999374619782.addModel(\"REMARK AtomGroup Mpro-x0161_1 Selection 'heavy' Selection 'resnum 140 144 ...6 189 165 49 41'\\nATOM      1  N   HIS C  41      14.763  -7.835  22.045  1.00 28.84           N\\nATOM      2  CA  HIS C  41      13.968  -7.276  23.142  1.00 31.13           C\\nATOM      3  C   HIS C  41      12.691  -8.076  23.472  1.00 32.24           C\\nATOM      4  O   HIS C  41      11.921  -7.646  24.329  1.00 29.90           O\\nATOM      5  CB  HIS C  41      13.664  -5.781  22.899  1.00 34.50           C\\nATOM      6  CG  HIS C  41      12.682  -5.411  21.814  1.00 34.53           C\\nATOM      7  ND1 HIS C  41      11.338  -5.746  21.891  1.00 38.93           N\\nATOM      8  CD2 HIS C  41      12.840  -4.642  20.677  1.00 39.19           C\\nATOM      9  CE1 HIS C  41      10.736  -5.201  20.842  1.00 38.96           C\\nATOM     10  NE2 HIS C  41      11.599  -4.543  20.075  1.00 39.05           N\\nATOM     11  N   MET C  49      14.132  -2.640  30.082  0.64 60.16           N\\nATOM     12  CA  MET C  49      14.134  -1.739  28.933  0.64 58.53           C\\nATOM     13  C   MET C  49      14.990  -0.465  29.054  0.64 53.68           C\\nATOM     14  O   MET C  49      15.429   0.028  28.019  0.64 49.11           O\\nATOM     15  CB  MET C  49      12.666  -1.419  28.575  0.64 60.88           C\\nATOM     16  CG  MET C  49      12.017  -2.529  27.730  0.64 63.12           C\\nATOM     17  SD  MET C  49      10.204  -2.586  27.717  0.64 68.43           S\\nATOM     18  CE  MET C  49       9.790  -0.927  27.131  0.64 66.21           C\\nATOM     19  N   PHE C 140       3.007   1.744  13.384  1.00 37.49           N\\nATOM     20  CA  PHE C 140       3.463   0.698  14.293  1.00 38.56           C\\nATOM     21  C   PHE C 140       2.932   0.989  15.704  1.00 39.07           C\\nATOM     22  O   PHE C 140       3.370   1.951  16.334  1.00 41.58           O\\nATOM     23  CB  PHE C 140       5.007   0.642  14.254  1.00 37.63           C\\nATOM     24  CG  PHE C 140       5.669   0.159  12.971  1.00 36.48           C\\nATOM     25  CD1 PHE C 140       5.693  -1.206  12.620  1.00 39.45           C\\nATOM     26  CD2 PHE C 140       6.337   1.086  12.148  1.00 38.54           C\\nATOM     27  CE1 PHE C 140       6.421  -1.642  11.496  1.00 37.66           C\\nATOM     28  CE2 PHE C 140       7.089   0.654  11.043  1.00 37.04           C\\nATOM     29  CZ  PHE C 140       7.149  -0.715  10.727  1.00 36.39           C\\nATOM     30  N   ASN C 142       1.921  -1.071  19.698  1.00 47.30           N\\nATOM     31  CA  ASN C 142       2.495  -2.105  20.568  1.00 46.42           C\\nATOM     32  C   ASN C 142       2.176  -3.512  20.032  1.00 41.58           C\\nATOM     33  O   ASN C 142       1.047  -3.786  19.622  1.00 39.86           O\\nATOM     34  CB  ASN C 142       1.965  -1.974  22.015  1.00 50.77           C\\nATOM     35  CG  ASN C 142       2.443  -0.711  22.741  1.00 59.66           C\\nATOM     36  OD1 ASN C 142       1.632   0.092  23.194  1.00 66.65           O\\nATOM     37  ND2 ASN C 142       3.759  -0.556  22.907  1.00 62.03           N\\nATOM     38  N   SER C 144       3.831  -4.656  17.318  1.00 36.24           N\\nATOM     39  CA  SER C 144       4.196  -4.585  15.901  1.00 35.20           C\\nATOM     40  C   SER C 144       5.670  -4.921  15.625  1.00 33.83           C\\nATOM     41  O   SER C 144       6.043  -5.084  14.467  1.00 35.46           O\\nATOM     42  CB  SER C 144       3.911  -3.172  15.367  1.00 34.73           C\\nATOM     43  OG  SER C 144       4.879  -2.232  15.782  1.00 32.65           O\\nATOM     44  N   CYS C 145       6.510  -5.013  16.667  1.00 35.69           N\\nATOM     45  CA  CYS C 145       7.941  -5.291  16.575  1.00 35.66           C\\nATOM     46  C   CYS C 145       8.189  -6.669  15.946  1.00 32.33           C\\nATOM     47  O   CYS C 145       7.437  -7.611  16.183  1.00 30.52           O\\nATOM     48  CB  CYS C 145       8.558  -5.202  17.979  1.00 38.48           C\\nATOM     49  SG  CYS C 145       8.369  -3.512  18.618  1.00 40.38           S\\nATOM     50  N   HIS C 163      11.350  -4.191  12.299  1.00 27.50           N\\nATOM     51  CA  HIS C 163      10.955  -3.120  13.203  1.00 26.43           C\\nATOM     52  C   HIS C 163      11.315  -3.487  14.648  1.00 27.22           C\\nATOM     53  O   HIS C 163      11.045  -4.603  15.091  1.00 25.19           O\\nATOM     54  CB  HIS C 163       9.437  -2.889  13.089  1.00 26.25           C\\nATOM     55  CG  HIS C 163       9.008  -1.675  13.870  1.00 26.44           C\\nATOM     56  ND1 HIS C 163       8.431  -1.740  15.126  1.00 27.07           N\\nATOM     57  CD2 HIS C 163       9.177  -0.335  13.608  1.00 27.92           C\\nATOM     58  CE1 HIS C 163       8.313  -0.485  15.565  1.00 27.90           C\\nATOM     59  NE2 HIS C 163       8.740   0.419  14.696  1.00 26.74           N\\nATOM     60  N   MET C 165      11.606  -0.426  17.305  0.64 32.66           N\\nATOM     61  CA  MET C 165      11.171   0.628  18.212  0.64 33.02           C\\nATOM     62  C   MET C 165      10.604   1.843  17.491  0.64 33.31           C\\nATOM     63  O   MET C 165      10.932   2.115  16.338  0.64 31.72           O\\nATOM     64  CB  MET C 165      12.305   0.995  19.194  0.64 32.18           C\\nATOM     65  CG  MET C 165      13.527   1.608  18.500  0.64 32.64           C\\nATOM     66  SD  MET C 165      14.935   1.818  19.598  0.64 34.74           S\\nATOM     67  CE  MET C 165      15.999   2.711  18.462  0.64 34.00           C\\nATOM     68  N   GLU C 166       9.799   2.603  18.237  1.00 34.87           N\\nATOM     69  CA  GLU C 166       9.390   3.960  17.932  1.00 35.31           C\\nATOM     70  C   GLU C 166      10.244   4.887  18.814  1.00 33.74           C\\nATOM     71  O   GLU C 166      10.412   4.629  20.006  1.00 31.96           O\\nATOM     72  CB  GLU C 166       7.895   4.065  18.291  1.00 41.50           C\\nATOM     73  CG  GLU C 166       7.224   5.383  17.866  1.00 47.32           C\\nATOM     74  CD  GLU C 166       5.698   5.270  17.876  1.00 48.16           C\\nATOM     75  OE1 GLU C 166       5.156   4.929  18.949  1.00 54.03           O\\nATOM     76  OE2 GLU C 166       5.095   5.517  16.810  1.00 44.24           O\\nATOM     77  N   GLN C 189      15.262   1.844  25.918  0.64 42.39           N\\nATOM     78  CA  GLN C 189      14.408   2.980  26.242  0.64 44.21           C\\nATOM     79  C   GLN C 189      15.259   4.096  26.861  0.64 43.15           C\\nATOM     80  O   GLN C 189      15.527   4.109  28.062  0.64 42.40           O\\nATOM     81  CB  GLN C 189      13.306   2.507  27.216  0.64 45.84           C\\nATOM     82  CG  GLN C 189      12.311   3.616  27.611  0.64 47.89           C\\nATOM     83  CD  GLN C 189      11.076   3.090  28.344  0.64 49.07           C\\nATOM     84  OE1 GLN C 189      11.099   2.015  28.938  0.64 51.43           O\\nATOM     85  NE2 GLN C 189       9.984   3.855  28.299  0.64 50.03           N\\n\",\"pdb\");\n",
       "\tviewer_15885999374619782.addModel(\"REMARK AtomGroup Mpro-x0946_1 Selection 'heavy' Selection 'resnum 140 144 ...6 189 165 49 41'\\nATOM      1  N   HIS A  41      14.710  -7.840  22.317  1.00 18.32           N\\nATOM      2  CA  HIS A  41      13.913  -7.244  23.394  1.00 19.86           C\\nATOM      3  C   HIS A  41      12.626  -8.012  23.747  1.00 20.66           C\\nATOM      4  O   HIS A  41      11.915  -7.600  24.664  1.00 21.18           O\\nATOM      5  CB  HIS A  41      13.630  -5.755  23.098  1.00 19.27           C\\nATOM      6  CG  HIS A  41      12.673  -5.388  21.990  1.00 20.09           C\\nATOM      7  ND1 HIS A  41      11.352  -5.847  21.940  1.00 22.19           N\\nATOM      8  CD2 HIS A  41      12.865  -4.496  20.956  1.00 20.05           C\\nATOM      9  CE1 HIS A  41      10.816  -5.249  20.891  1.00 19.82           C\\nATOM     10  NE2 HIS A  41      11.674  -4.454  20.260  1.00 22.03           N\\nATOM     11  N   MET A  49      13.882  -2.919  29.927  1.00 31.82           N\\nATOM     12  CA  MET A  49      14.031  -2.035  28.771  1.00 31.74           C\\nATOM     13  C   MET A  49      14.725  -0.704  29.077  1.00 29.69           C\\nATOM     14  O   MET A  49      15.306  -0.118  28.168  1.00 24.34           O\\nATOM     15  CB  MET A  49      12.656  -1.728  28.133  1.00 32.88           C\\nATOM     16  CG  MET A  49      12.059  -2.863  27.293  1.00 35.06           C\\nATOM     17  SD  MET A  49      13.141  -3.584  26.031  1.00 31.94           S\\nATOM     18  CE  MET A  49      13.877  -2.123  25.263  1.00 35.13           C\\nATOM     19  N   PHE A 140       2.880   1.651  13.435  1.00 22.54           N\\nATOM     20  CA  PHE A 140       3.310   0.616  14.365  1.00 24.46           C\\nATOM     21  C   PHE A 140       2.727   0.922  15.751  1.00 26.90           C\\nATOM     22  O   PHE A 140       3.087   1.930  16.357  1.00 26.45           O\\nATOM     23  CB  PHE A 140       4.854   0.563  14.373  1.00 23.49           C\\nATOM     24  CG  PHE A 140       5.549   0.099  13.101  1.00 21.45           C\\nATOM     25  CD1 PHE A 140       5.491  -1.246  12.677  1.00 22.18           C\\nATOM     26  CD2 PHE A 140       6.323   1.014  12.362  1.00 22.01           C\\nATOM     27  CE1 PHE A 140       6.251  -1.678  11.573  1.00 21.32           C\\nATOM     28  CE2 PHE A 140       7.097   0.585  11.270  1.00 21.15           C\\nATOM     29  CZ  PHE A 140       7.081  -0.768  10.889  1.00 21.40           C\\nATOM     30  N   ASN A 142       1.784  -1.175  19.726  1.00 31.09           N\\nATOM     31  CA  ASN A 142       2.337  -2.224  20.597  1.00 30.71           C\\nATOM     32  C   ASN A 142       1.949  -3.637  20.116  1.00 26.11           C\\nATOM     33  O   ASN A 142       0.788  -3.897  19.803  1.00 25.70           O\\nATOM     34  CB  ASN A 142       1.892  -2.022  22.065  1.00 31.96           C\\nATOM     35  CG  ASN A 142       2.321  -0.699  22.714  1.00 36.20           C\\nATOM     36  OD1 ASN A 142       1.608  -0.174  23.565  1.00 37.08           O\\nATOM     37  ND2 ASN A 142       3.492  -0.165  22.361  1.00 39.70           N\\nATOM     38  N   SER A 144       3.686  -4.828  17.345  1.00 22.28           N\\nATOM     39  CA  SER A 144       4.121  -4.799  15.944  1.00 21.42           C\\nATOM     40  C   SER A 144       5.628  -5.001  15.735  1.00 19.66           C\\nATOM     41  O   SER A 144       6.053  -5.102  14.586  1.00 19.27           O\\nATOM     42  CB  SER A 144       3.715  -3.470  15.284  1.00 21.85           C\\nATOM     43  OG  SER A 144       4.524  -2.393  15.698  1.00 22.80           O\\nATOM     44  N   CYS A 145       6.439  -5.082  16.801  1.00 19.64           N\\nATOM     45  CA  CYS A 145       7.873  -5.357  16.705  1.00 18.47           C\\nATOM     46  C   CYS A 145       8.120  -6.743  16.088  1.00 17.42           C\\nATOM     47  O   CYS A 145       7.356  -7.679  16.319  1.00 16.69           O\\nATOM     48  CB  CYS A 145       8.510  -5.245  18.096  1.00 20.79           C\\nATOM     49  SG  CYS A 145       8.524  -3.507  18.605  1.00 21.75           S\\nATOM     50  N   HIS A 163      11.286  -4.233  12.499  1.00 17.71           N\\nATOM     51  CA  HIS A 163      10.871  -3.163  13.395  1.00 17.35           C\\nATOM     52  C   HIS A 163      11.219  -3.511  14.849  1.00 19.33           C\\nATOM     53  O   HIS A 163      10.938  -4.617  15.307  1.00 16.74           O\\nATOM     54  CB  HIS A 163       9.351  -2.950  13.262  1.00 17.29           C\\nATOM     55  CG  HIS A 163       8.925  -1.726  14.024  1.00 16.00           C\\nATOM     56  ND1 HIS A 163       8.412  -1.774  15.307  1.00 16.24           N\\nATOM     57  CD2 HIS A 163       9.078  -0.391  13.733  1.00 15.40           C\\nATOM     58  CE1 HIS A 163       8.349  -0.517  15.741  1.00 16.45           C\\nATOM     59  NE2 HIS A 163       8.712   0.378  14.834  1.00 15.89           N\\nATOM     60  N   MET A 165      11.468  -0.399  17.440  0.66 22.15           N\\nATOM     61  CA  MET A 165      11.117   0.684  18.354  0.66 23.00           C\\nATOM     62  C   MET A 165      10.518   1.899  17.661  0.66 22.25           C\\nATOM     63  O   MET A 165      10.756   2.140  16.480  0.66 20.46           O\\nATOM     64  CB  MET A 165      12.335   1.055  19.227  0.66 23.71           C\\nATOM     65  CG  MET A 165      13.494   1.678  18.438  0.66 24.59           C\\nATOM     66  SD  MET A 165      15.076   1.627  19.299  0.66 27.43           S\\nATOM     67  CE  MET A 165      15.516   3.376  19.245  0.66 26.35           C\\nATOM     68  N   GLU A 166       9.803   2.693  18.461  1.00 22.09           N\\nATOM     69  CA  GLU A 166       9.390   4.052  18.167  1.00 22.61           C\\nATOM     70  C   GLU A 166      10.228   4.980  19.057  1.00 21.48           C\\nATOM     71  O   GLU A 166      10.400   4.712  20.247  1.00 22.84           O\\nATOM     72  CB  GLU A 166       7.890   4.152  18.496  1.00 24.75           C\\nATOM     73  CG  GLU A 166       7.200   5.455  18.079  1.00 28.35           C\\nATOM     74  CD  GLU A 166       5.680   5.325  18.176  1.00 30.02           C\\nATOM     75  OE1 GLU A 166       5.174   5.454  19.312  1.00 32.16           O\\nATOM     76  OE2 GLU A 166       5.046   5.087  17.126  1.00 26.42           O\\nATOM     77  N   GLN A 189      14.918   1.700  26.112  1.00 25.38           N\\nATOM     78  CA  GLN A 189      14.000   2.767  26.491  1.00 27.66           C\\nATOM     79  C   GLN A 189      14.797   3.869  27.207  1.00 25.80           C\\nATOM     80  O   GLN A 189      14.919   3.909  28.431  1.00 30.93           O\\nATOM     81  CB  GLN A 189      12.872   2.185  27.370  1.00 30.39           C\\nATOM     82  CG  GLN A 189      11.847   3.244  27.835  1.00 33.53           C\\nATOM     83  CD  GLN A 189      10.540   2.699  28.420  1.00 37.04           C\\nATOM     84  OE1 GLN A 189       9.701   3.485  28.853  1.00 45.65           O\\nATOM     85  NE2 GLN A 189      10.343   1.379  28.456  1.00 41.56           N\\n\",\"pdb\");\n",
       "\tviewer_15885999374619782.setStyle({\"chain\": \"A\"},{\"stick\": {\"color\": \"red\"}});\n",
       "\tviewer_15885999374619782.setStyle({\"chain\": \"B\"},{\"stick\": {\"color\": \"blue\"}});\n",
       "\tviewer_15885999374619782.setStyle({\"chain\": \"C\"},{\"stick\": {\"color\": \"cyan\"}});\n",
       "\tviewer_15885999374619782.setBackgroundColor(\"0xeeeeee\");\n",
       "\tviewer_15885999374619782.zoomTo();\n",
       "viewer_15885999374619782.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x1a265ac0d0>"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "center_asite_structs = [x for (x,y) in zip(contact_resid_structs,center_list) if y]\n",
    "p = py3Dmol.view(500, 500)\n",
    "for struct in center_asite_structs:\n",
    "    buff = StringIO()\n",
    "    prody.writePDBStream(buff, struct)\n",
    "    p.addModel(buff.getvalue(), 'pdb')\n",
    "for i in range(0, num_clusters):\n",
    "    chain_id = string.ascii_uppercase[i]\n",
    "    ch = {'chain': chain_id}\n",
    "    p.setStyle(ch, {'stick': {'color': color_list[i]}})\n",
    "p.setBackgroundColor('0xeeeeee')\n",
    "p.zoomTo()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Write the cluster center representatives to disk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "copied diamond_non_cov_mpro/Mpro-x1249_1.pdb to 2020_05_04_09_45/cluster001.pdb\n",
      "copied diamond_non_cov_mpro/Mpro-x0161_1.pdb to 2020_05_04_09_45/cluster002.pdb\n",
      "copied diamond_non_cov_mpro/Mpro-x0946_1.pdb to 2020_05_04_09_45/cluster003.pdb\n"
     ]
    }
   ],
   "source": [
    "import os, shutil\n",
    "import time\n",
    "time_str = time.strftime(\"%Y_%m_%d_%H_%M\")\n",
    "try:\n",
    "    os.mkdir(time_str)\n",
    "except FileExistsError:\n",
    "    shutil.rmtree(time_str)\n",
    "    os.mkdir(time_str)\n",
    "center_files = [x for x,y in zip(non_cov_files,center_list) if y]\n",
    "for i, filename in enumerate(center_files,1):\n",
    "    outfile_name = f\"{time_str}/cluster{i:03d}.pdb\"\n",
    "    shutil.copy(filename,outfile_name)\n",
    "    print(f\"copied {filename} to {outfile_name}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Use Principal Compoent Analysis to Visualize the Active Site Coordinates"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PC_1</th>\n",
       "      <th>PC_2</th>\n",
       "      <th>structure</th>\n",
       "      <th>Cluster</th>\n",
       "      <th>Center</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>-1.438211</td>\n",
       "      <td>3.762896</td>\n",
       "      <td>x1249</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>4.275663</td>\n",
       "      <td>-3.352112</td>\n",
       "      <td>x0161</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>-1.062761</td>\n",
       "      <td>-2.323291</td>\n",
       "      <td>x0946</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        PC_1      PC_2 structure Cluster  Center\n",
       "3  -1.438211  3.762896     x1249       1       1\n",
       "17  4.275663 -3.352112     x0161       2       1\n",
       "20 -1.062761 -2.323291     x0946       0       1"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# perform PCA\n",
    "pca = PCA(n_components=2)\n",
    "pca_df = pd.DataFrame(\n",
    "    pca.fit_transform(contact_resid_coords_flat),\n",
    "    columns=[\"PC_1\",\"PC_2\"]\n",
    ")\n",
    "pca_df['structure'] = [ re.findall(r'x[0-9]+', f)[0] for f in non_cov_files ]\n",
    "pca_df['Cluster'] = [str(x) for x in cluster_ids]\n",
    "pca_df['Center'] = center_list\n",
    "pca_df.query('Center == 1')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Examine the variance explained by the principal components"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(array([0.21932545, 0.1582557 ]), 0.3775811490695934)"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pca.explained_variance_ratio_,sum(pca.explained_variance_ratio_)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "A few settings to make the plots look better"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "sns.set(rc={'figure.figsize': (12, 12)})\n",
    "sns.set(font_scale=1.5)\n",
    "sns.set_style('whitegrid')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Plot the PCs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<matplotlib.legend.Legend at 0x1a2788bc50>"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAt4AAALJCAYAAACDRyR8AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjMsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+AADFEAAAgAElEQVR4nOzde3zO9f/H8cfOsyOzGVkkRUyOIWf5LjnEECJymkNFqMYsqVRi+FpCDhnCOtAv5OzbEd8cah3ktMQwkykbdrKT6/fH1a6v2bCtXdt1bc/77bYb+3ze78/7de0z89r7en9ebxuDwWBARERERETMyra0AxARERERKQ+UeIuIiIiIlAAl3iIiIiIiJUCJt4iIiIhICVDiLSIiIiJSAuxLO4CScP36dVJSUnBwcMDGxqa0wxERERGRMshgMJCZmYmrqyu2tnnnt8tF4p2SksJvv/1W2mGIiIiISDlQp04d3N3d8xwvF4m3g4MDYPwiODo6Frjf4cOHadCggbnCkkLQvbAcuheWQ/fCcuheWA7dC8tRHu9FRkYGv/32myn3vFm5SLxzlpc4Ojri5ORUqL6FbS/mc6t7ERsbS1hYGAcPHgSgY8eOTJkyBS8vr1te6/jx4/Tt25cxY8bw/PPP5zq3Z88eFi9ezJEjR7C1taVRo0ZMnDiRxo0b52qXkJDAvHnz+Oqrr7h27Rr+/v689NJLedqVRfp3YTl0LyyH7oXl0L2wHOX1XtxqabMerhSrlpiYyNChQ/n5558ZOXIkw4cP56uvvmL48OFkZGTk2ycrK4vQ0FAyMzPznDt48CCjRo0iKSmJF154gbFjx3L27FkGDx7MoUOHTO2Sk5MZNGgQ27dvZ+DAgUyYMIH4+HiGDh2qZU0iIiKSr3Ix4y1l16pVq7hw4QKbN2+mdu3aADRq1Ijhw4ezceNG+vfvn6fP0qVLOXHiRL7Xe/vtt6lWrRrr1q2jQoUKAPTq1Ytu3boRHh7OypUrAXj//feJiYlhzZo1NG/eHIBu3boREBDA8uXLmT17tjleroiIiFgxzXiLVdu6dSstWrQwJd0ArVu3platWmzdujVP++joaBYvXsxzzz2X59yVK1c4fvw4Xbp0MSXdAN7e3jRv3pyffvoJMD6xvGHDBjp27GhKugF8fHyYPHkyDz30UHG+RBERESkjlHiLxdq9ezd169ZlwoQJuY5PmzaNunXrsmXLFmJjY/H398/T19/fnyNHjuQ6lrPEpE2bNvTs2TNPHzc3N3bs2MGwYcPynEtMTMTOzg6Ac+fOER8fT+vWrQFjIp6SkgLAoEGD8p1lFxEREVHiLRarffv29O7dmx07dvDLL78AsHfvXtatW8eAAQOoU6cOAL6+vnn6+vj4kJSURFJSkunY+++/z5kzZ5g+fXq+49nZ2XHPPffkud7x48f58ccfadKkCQBnzpwBoHLlyoSFhfHQQw/RtGlTHn30Ub766qt//sJFRESkTNIab7FooaGh7N27l1WrVtG7d2+mTZtGzZo1CQkJITo6GiDXspAcOU9Rp6am4u7uzokTJ1i0aBGvvvoqVatW5dy5cwUaPyUlhZCQEABGjx4NwNWrVwGYP38+9vb2TJ06FVtbWyIiIhg7diwRERGm2XAREZH8XLlyhb/++uuWhQDKAnt7e44dO1baYRQbR0dHvL298fT0LPI1lHiLRfP09OT1119n7Nix9OvXj/j4eNauXYuLiwsGg+GO/W1sbMjOzmbKlCk0a9asUMtA0tLSePbZZzl+/DhjxoyhRYsWAKYfklevXmXnzp2mf4CdOnXi0Ucf5d///rcSbxERuaVr164RHx+Pn58fFSpUKLO7aqekpODq6lraYRQLg8FAWloa586dw8nJCWdn5yJdxyoS76ysLJo2bUp6enqu4y4uLqYH3qTsCggIoHnz5nz//fcMHDiQpk2bAsb7D+T5vrjxmJubGxEREURHR/Phhx+SkJAA/G/WOi0tjYSEBCpWrJhra9erV68yZswYfvzxR5544gleeOEF07mccTt37pzrt14PDw86derEhg0bytQPGxERKV5//vknPj4+pv9PxPLZ2Njg4uKCt7c3f/75J3fffXeRrmMViXdMTAzp6emEhYVxzz33mI7fmChJ2ZWamsrp06cB4xrv1NRUXFxcuOuuuwDjD7CbXbx4EQ8PD1xcXNizZw+ZmZn069cvT7uIiAgiIiL48ssv8fPzA+DSpUsEBQVx7NgxnnzySaZPn55rNiJnDXh+G/R4eXlhMBhITU1V4i0iIvm6du0aVatWLe0wpAjc3d25dOlSkftbReJ9/PhxbG1teeyxx/Jdzytl27x58/jrr7+YPHkyc+bMYd68ebzyyit4eHjg5+eXp3oJwNGjR03b1IaEhJhmuHP89ddfTJo0icDAQHr16oWPjw9g3BgnJ+keNmwYoaGhea59//334+joyO+//57nXM5bULfbNVNERMq3rKws7O2tIgWTm9jb25OVlVX0/sUYi9kcO3aMGjVqKOkuh6KiooiMjKRTp04EBQVx6tQpIiMj6dKlCw899BCdO3dm9erVnDx50lTL+7vvviMmJoagoCAAUwJ+o5yHK+++++5c67HfeOMNjh07xpAhQ/JNusG41KRTp058+eWXnDhxgvvvvx8wbl3/1VdfERAQYCo9KCIikp+yuq67rPun980qEu/o6GgcHR0JCgrixx9/xN7enq5duzJ58mTc3NxKOzwxk/T0dKZOnYqXlxcDBgwAIDg4mC+++IKpU6fy+eefM2rUKDZt2sSwYcMYMWIE6enpLF++HH9/fwIDAws13smTJ9m0aRMeHh7Uq1ePTZs25WmTc81JkyZx8OBBhgwZwpAhQ3BwcGD16tU4Ozvz4osv/vMXLyIiIkViMBgs9hcbq0i8jx8/TnJyMv369eOZZ57h8OHDLFiwgJiYGFavXm2xX1z5Z3Lu8Zw5c0zrpStVqsSkSZOYOnUq77zzDiEhIaxdu5aZM2fy7rvv4uzsTEBAAJMnT8bR0bFQ4x08eBAwPlh5q9nunMTbz8+PdevWMWfOHCIiIjAYDDz00ENMnjy5yA9ciIiIlBW//PILK1as4OeffyYxMRFfX1/atWvHmDFjTM9K5WySl99u0kX19ddfs337dmbPnl1s1yxONoaC1GQrZQcPHsTT05O6deuajn3++edMmjSJFStW0KZNm9v2T09P5/Dhw+YOU0REROSO7O3tue+++0o7DLP58MMPCQ8Pp2XLljz++ON4e3sTExPDBx98QEZGBsuXL6dGjRo0bdqU5557jpEjRxbb2M888wzZ2dm8//77xXbNm/3+++93XOfdoEED054iN7KKGe+c+sk36tixI2CcDb9T4p3jVl+EW4mKiqJZs2YFbi/mo3thOXQvLIfuheXQvbAc1nAvjh07VmYrX0VFRTFv3jyGDRvGuHHjTK+zQ4cOdO3alcDAQMLCwli1ahUADg4Oxfq1yHnGypxfX0dHRxo1apTvuTtN9lp8Pb5Lly6xfv16YmNjcx2/du0aYFx6ICIiIiKlLyIigooVKzJx4sQ853x9fZkyZQqtWrXKM2P82WefUbduXS5cuJDreKdOnZg6darp8y1bttCzZ08aNmxIq1atCA4OJj4+HoCnn36affv2cfDgQerWrcuBAwcASExM5JVXXqFVq1Y0bNiQgQMHEhUVlWucunXrsmjRIvr06UPDhg1ZtmxZsXw9bmbxibeNjQ2vvvoqa9euzXV827Zt2NnZWfxvtSIiIiLlgcFgYO/evbRq1eqWKwx69erFmDFjilROMSoqismTJ9O5c2eWL1/OlClT2L9/P8HBwQC89tprPPjgg9SvX59PPvkEf39/0tPTGTZsGN988w0vvvgi7777Lp6engwbNoxDhw7luv6SJUvo2bMn77zzDv/6178K/wUoAItfauLl5cWgQYNYs2YNbm5uPPTQQ0RFRbFkyRIGDRpEzZo1SztEERERkXIvMTGR9PR00wZ3xS0qKgpnZ2dGjx5tKqBQsWJFfv31VwwGA/fddx9ubm5kZ2fTuHFjANatW0d0dDTr16/nwQcfBKB9+/b07duX8PBwVq5cabp+ixYtGDZsmFliz2HxiTcYN0Dx9fXl//7v/1i2bBm+vr6MHz++WBfji4iIiEjR5ayvzs7ONsv1mzdvTnh4OI8//jiPPfYYHTp0oG3btnTo0OGWffbt24evry/16tXLtbzlkUceYenSpWRkZJiS+Hr16pkl7htZReLt4ODAqFGjGDVqVGmHIiIiIiL58PT0xNXVlfPnz9+yTXJyMkCR9mFp0qQJy5YtY9WqVaxcuZJly5bh7e3NM888w9NPP51vn8uXL3PhwgX8/f3zPZ9T6hCMG+SZm1Uk3iJFcerUKcLClrJz53/JyrLD1hYqV3Zh8uQR9O3bCwcHh9IOUUREpExp27YtBw4cID09Pd/zq1at4r333mP79u25jufsyXLzbHlKSkquz9u1a0e7du1IS0tj//79rF69mrfeeosmTZrku1O1u7s7tWvXJiwsLN94SrpIh8U/XClSWOnp6QQGjqBVqxdZtqwTZ87sJi7uW2Jjv+Xnn1cyevRv1KrVli+++Lq0QxURESlThg8fzuXLl5k/f36ec+fPnycyMpKGDRvmeUYvZwb8jz/+MB07efIkly9fNn0+Z84c+vbti8FgoEKFCjzyyCOEhIQAmKqh5Cx3ydG8eXPOnz9PlSpVePDBB00fX375JWvWrCnxSTjNeEuZkpGRQYcOT/Dzz4NITx+YT4uqJCdPJTn5OQYO7M+qVRl07/5YiccpIiJSFjVp0oSxY8eycOFCfvvtN5544gkqVqzI8ePHiYiIwNbWljlz5uTp17JlS5ydnXn77beZMGECKSkpvPvuu1SsWNHUpnXr1kRERDBlyhR69uxJZmYmy5cvp1KlSqY9X9zd3fnhhx/Yt28f9evXp0+fPqxdu5bhw4ebds385ptvWLlyJePGjSvx3c814y1lyoQJr/HLL71ukXTfqBJ//fV/BAVNN9X/FBERkX/u+eefZ/HixWRlZTFjxgzGjBnDxx9/TJcuXdiwYQN33313nj4eHh4sWLCA7Oxsxo4dy/z58xk7dmyu5SNt2rRh3rx5nDhxgnHjxvHiiy/i4uLC6tWr8fDwAOCpp54yPRv43//+F1dXVyIjI2nUqBGzZs1i9OjR7Nmzh2nTpvH888+X2Nckh2a8pcxITU1l06ZvuXbt7QL28OCvvyby7rsrmDEj1KyxiYiIlCedOnWiZcuWt91BMjo6Otfn7du3p3379rmO9ejRI9fn3bp1o1u3bre8ZosWLfj669xLSb29vZk5c+Zt4705FnPRjLeUGR988DGXLw8ACv62UXZ2L1av3mS20kciIiIiOZR4S5mxadO3pKU9XshejmRkPMCZM2fMEpOIiIhIDiXeUmZcuZIEeBa63/XrHiQlJRV/QCIiIiI3UOItZYanpztwpdD9bG2v4u7uXvwBiYiIiNxAibeUGb17d6RChc2F7JWBo+PxPPVERURERIqbEm8pM55++kkqVvwEMBS4j53dBoYMCcxTcF9ERESkuCnxljLDxcWF3r0fwdl5eQF7XMHb+x0mTAgya1wiIiIioMRbyph33nmdJk024+QUeYeWCXh7P8GKFdOpUqVKicQmIiIi5ZsSbylTHBwc+Prr9XTr9g1VqvTExmYrcGON7vO4u0+nevUufPLJNLp161xaoYqIiEg5o50rpcxxcnLis8/e5/Tp08yZs4ytW98mO9seW1vw9nZjypQgevV6GQcHh9IOVURERMoRJd5SZt1zzz0sWvQ2ixaVdiQiIiJmZDDAvn1w8CAkJYG7O7RoAa1agU3Bd3MuDlu2bGHx4sXExsZSrVo1nn32WXr16lWiMVgyJd4iIiIi1igzEyIiYPZsuHjR+HlmJjg4GD+qVIHJkyEoyPi5mW3bto3g4GCGDh1K27Zt2bFjByEhITg7O9OlSxezj28NlHiLiIiIWJvkZOjaFX78EVJTc5/LyDB+xMTASy/Bhx/Ctm3g5mbWkMLDw+natSuhoaEANG3alJSUFObPn6/E+296uFJERETEmmRmGpPu77/Pm3TfLDXVuASlWzdjPzOJjY3l7NmzdO6cu2jBY489xqlTp4iNjTXb2NZEibeIiIiINYmIMM50p6cXrH16OkRFwYoVZgvp1KlTANSqVSvX8ZydoWNiYsw2tjVR4i0iIiJiLQwG45ruO8103yw11djPUPDdnQsjKSkJALeblrO4uroCkJycbJZxrY0SbxERERFrsW+f8UHKooiPN/Y3A8MdEnpbW6WcoMRbRERExHocPFj0tdpZWcZ14Wbg7u4OQEpKSq7jOTPdOefLOyXeIiIiItYiKanoiXdGhrG/GeSs7T579myu42fOnMl1vrxT4i0iIiJiLdzdi16T29HR2N8MatasiZ+fHzt27Mh1fNeuXdxzzz3cddddZhnX2qiOt4iIiIi1aNHCmHhnZBS+r709NG9e/DH9bezYsYSGhuLp6UnHjh3ZuXMn27dvJzw83GxjWhsl3iIiIiLWolUr446URSnP5+tr7G8mffr0ISMjgxUrVrB+/XqqV69OWFgY3bp1M9uY1kaJt4iIiIi1sLExbgP/0kuFKyno4mLsZ2NjvtiAAQMGMGDAAMD4oGVOOUEx0hpvEREREWsSFARNm4KTU8HaOzlBs2YwYoR545I7UuItIiIiYk0cHGD7duN6bxeX27d1cTG227at6A9lSrFR4i0iIiJibdzc4MsvYd48uPdecHU1zmzb2Bj/dHU1Hp83z9juph0lpXRojbeIiIiINXJwgDFjYPRo446U339vrNPt7m6c5X74YbOv6ZbCUeItIiIiYs1sbKB1a+OHWDQtNRERERERKQFKvEVERERESoASbxERERGREqDEW0RERESkBCjxFhEREREpAapqIiIiImLFDAZjNcGDB3NXE2zVStUELY0SbxERERErlJkJEREwezZcvGj8PDPTWN7bwQGqVIHJk407zJfGppXR0dE8/fTTfPnll1StWrXkA7BASrxFRERErExyMnTtCj/+CKmpuc9lZBg/YmLgpZfgww+NO8aX5OaVJ0+eZMKECWRlZZXcoFZAa7xFRERErEhmpjHp/v77vEn3zVJTjUtQunUz9jO3rKwsIiMj6devH+np6eYf0Moo8RYRERGxIhERxpnugua16ekQFQUrVpg3LoCoqCjmzp3LiBEjGD9+vPkHtDJKvEVERESshMFgXNN9p5num6WmGvsZDOaJK0ft2rX54osvGDduHHZ2duYdzAop8RYRERGxEvv2GR+kLIr4eGN/c/L29qZy5crmHcSKKfEWERERsRIHDxZ9rXZWlnFduJQeJd4iIiIiViIpqeiJd0aGsb+UHiXeIiIiIlbC3b3oNbkdHY39pfQo8RYRERGxEi1aFD3xtreH5s2LNx4pHCXeIiIiIlaiVSvjjpRF4etr7C+lR4m3iIiIiJWwsTFuA+/iUrh+Li7GfjY25olLCkaJt4iIiIgVCQqCpk3Byalg7Z2coFkzGDHCvHHdrGfPnkRHR1O1atWSHdiCKfEWERERsSIODrB9u3G9951mvl1cjO22bSv62nApPkq8RURERKyMmxt8+SXMmwf33guursaZbRsb45+ursbj8+YZ27m5lXbEAmBf2gGIiIiISOE5OMCYMTB6tHFHyu+/N9bpdnc3znI//LDWdFsaJd5Srk2bNo3Tp0+zZs2aPOf27NnD4sWLOXLkCLa2tjRq1IiJEyfSuHHjW17v+PHj9O3blzFjxvD888/nOvfDDz8QHh7O4cOH8fDwICAggOeffx4vL69if10iIlJ+2NhA69bGD7FsWmoi5db69etZt25dvucOHjzIqFGjSEpK4oUXXmDs2LGcPXuWwYMHc+jQoXz7ZGVlERoaSmY+W4odOHCAYcOGERMTw5gxYxg8eDA7d+5k0KBBXLlypVhfl4iIiFgmzXhLuZOdnc3ixYtZuHDhLdu8/fbbVKtWjXXr1lGhQgUAevXqRbdu3QgPD2flypV5+ixdupQTJ07ke7233noLOzs7Pv74Y2rUqAFAQEAAgYGBLFmyhJCQkGJ4ZSIiImLJNOMt5Up6ejq9e/dmwYIFBAYG4uvrm6fNlStXOH78OF26dDEl3QDe3t40b96cn376KU+f6OhoFi9ezHPPPZfn3Llz5/jtt98IDAw0Jd0AtWvX5pFHHmHDhg3F9OpERETEkinxljJj9+7d1K1blwkTJuQ6Pm3aNOrWrcvu3btJT08nOTmZ8PBwwsLCsLfP+6aPm5sbO3bsYNiwYXnOJSYmYmdnl+tYzhKTNm3a0LNnzzx94uPjAahTp06eczVq1CAxMZE//vijMC9VRERErJDVJd7jxo3j0UcfLe0wxAK1b9+e3r17s2PHDvbs2QPA3r17WbduHQMGDKB9+/a4ubmxa9cuunXrdsvr2NnZcc899+SZDT9+/Dg//vgjTZo0yXX8/fff58yZM0yfPj3f67n8XWQ1JSUlz7nLly8D8Oeffxb8hYqIiIhVsqrEe9OmTfznP/8p7TDEgoWGhuLj48Obb75JYmIi06ZNo2bNmqY11La2tvnOct9JSkqK6RqjR482HT9x4gSLFi0iJCTkljtz1a5dGzc3N3bu3InBYDAdT09PZ+/evQBkZGQUOiYREREAA/Ad8A7w5t9/fvf3cbEsVpN4x8fHM2PGDG07Krfl6enJ66+/zpkzZ+jXrx/x8fHMmjXLNOtcFGlpaTz77LMcP36c0aNH06JFC8D4kOaUKVNo1qwZ/fv3v2V/R0dHhg8fzpEjRwgODiY6Oppjx44xfvx40tLSAPIsXxEREbmTTGAJUBvoDIQAr/39Z+e/jy/5u11JuH79Oh999BE9evSgSZMm9OzZk5kzZ5KcnFxCEVg+q0m8X3nlFdq0aUOrVq1KOxSxcAEBAXTu3JnY2Fj69+9P06ZNi3ytq1evMmLECA4cOMATTzzBCy+8YDoXERFBdHQ0L730EgkJCSQkJHD16lXAmKwnJCRw/fp1AJ577jmGDh3Ktm3b6NmzJ7169cLGxoaRI0cCxl8YRERECioZ6AS8BMQAKUAGxlnujL8/j/n7/L/+bm9uy5cv580336Rjx44sWrSIwYMHs3HjxjzPXpVnVlFOcP369Rw5coQtW7Ywe/bs0g5HLFxqaipHjx4FjGu8U1NTizTjfenSJYKCgjh27BhPPvkk06dPx+aGLcD27NlDZmYm/fr1y9M3IiKCiIgIvvzyS/z8/LC1teXll19m9OjRnD59mmrVqlG9enXCw8Oxs7OjevXqRX/BIiJSrmQCXYHvgfQ7tE0FDgLdgC8BBzPFZDAYWL58OU8++SQvvfQSAI0aNcLX15cXXniBY8eOUa9ePTONbj0sPvGOi4tj5syZzJw5Uzv8SYHMmzePuLg4Jk+ezJw5c5g3bx6vvPJKoa6RnJxsSrqHDRtGaGhonjYhISGmGe4cf/31F5MmTSIwMJBevXrh4+MDwJYtW/Dx8aFly5Z4e3ub2n///ff4+/vj5ORUhFcqIiLlUQTwI3dOunOkA1HACmCMmWJKSUmhZ8+edO3aNdfxe++9F4CzZ88q8cbCE2+DwcDLL79Mhw4deOyxx/7x9Q4fPlzoPlFRUf94XCkeBbkX0dHRrF27lk6dOtG4cWM6dOjA2rVrqVWrFg888ECe9unp6SQlJeW59nvvvcexY8fo0qULnTt3vuXYNyfMjo6OgHHNtpOTk+l7btGiRWRkZDBz5kzTeu6ffvqJqKgonnnmGav7PrO2eMsy3QvLoXthOSz9Xtjb2+db6aogDEBYhQqk2hZutXAqMOv6dQalpWFzx9aFZ2NjY1qOeeNr27ZtGwDVq1cv8mu2NBkZGUX+HrPoxDsyMpLo6Gg2b95MVlYWgKkqRFZWFnZ2drne+r+TBg0aFGpmMSoqimbNmhUuaDGLgtyL9PR0pk6dSuXKlZk1axYeHh7MmjWLLl26sHr1aj7//PM899/JyQl3d/dc1z558iR79+7Fw8ODDh06cO7cuTxjBQYG5htDTtu77ror1zUnTpzI+PHjWbp0KZ07dyYuLo6VK1fStm1bxo8fb1UPV+rfheXQvbAcuheWwxruxbFjx3B1dS1S3++Aohag/dPWlkOurrQuYv/C2r9/P6tWrSIgIIAGDRqU0Kjm5+joSKNGjfI9l56eftuJXotOvHfu3EliYiJt27bNc87f35+ZM2fSp0+fUohMLNGCBQuIiYlhzpw5eHh4AFCpUiUmTZrE1KlTeeeddwq0NfvBgwcB44OV+S0xgVsn3rfy2GOPMW/ePJYtW8bMmTOpXLkyQUFBjBkzxqqSbhERKV0HKXqVkiyM68JLIvGOiopi3Lhx+Pn58dZbb5XAiNbBohPv6dOn53lbYtGiRRw7doyFCxfi5+dXSpGJJQoODiY4ODjP8b59+9K3b998+3z11Vd5jg0cOJCBAwcWKQY/Pz+io6PzPde9e3e6d+9epOuKiIgAJFH0xDvj7/7mtm3bNqZMmUKNGjVYsWIFlSpVKoFRrYNFJ945C/JvVLFiRRwdHXnwwQdLISIRERGR0uOOsTJJUbZdc/y7vzmtXLmSsLAwWrRowezZs6lSpYqZR7QuVlPHW0RERKS8a0HRSwLaA82LMZabrV+/nlmzZtG1a1eWL1+Ou7u503zrY9Ez3vmZNWtWaYcgIiIiUipaAVUwbo5TWL5/9zeHS5cuMWPGDKpXr86gQYM4evQoaWlpVKhQAYAaNWqoLDRWmHiLiIiIlFc2wGSMO1KmFqKfy9/9zFFKEIybyqWlpREXF8egQYPynJ89e3ahCxOURUq8RURERKxIEBBJwXauBHACmgEjzBhTr1696NWrV65jKSkpRS6bWFZpjbeUewaDgTlzFhAXF5fv+YULl3Py5MkSjkpERCR/DsB2jOu9Xe7Q1uXvdtsw33bxUnBKvKVcMxgMBAW9xPTph2nXrl+e5HvSpDeZOnUvHToM4vfffy+lKEVERHJzA74E5gH3Aq4YZ7Zt/v7T9e/j8/5u51Y6YcpNtNREyq2cpPvTT91JSfk3MTGHaNeuH3v2rDIkOgUAACAASURBVKd69epMmvQmy5df5OrVlVy9+jsdOw7mm2/Wct9995V26CIiIjgAY4DRwD6MS0+SMJYMbAE8jPnWdEvRaMZbyqUbk+6kpNcx/mhqREzMYtq168ezz05i+fKLXL787t/n7icubg0dOw7WzDcQGxvLuHHjaNGiBS1atGDy5MkkJCTkade3b1/q1q2b52P8+PG3vPbx48dp0KABCxYsKPK4IiLliQ3G3SgnAK/8/WcrlHRbIs14S7kUHx/Phg2bSUraT+4fTcbke+3aLSQnv3zTufu5dKkzixZ9QHj4myUbsAVJSkpi0qRJZGRkMHLkSLKzs4mIiCA6Opr169fj6OgIGH+5OXnyJAEBAXTu3DnXNapXr57vtbOysggNDSUzM+++bImJiQwdOvSO44qIiFgqJd5SLlWtWpXPP19F3769uXhxA1D5hrONSE5ulKePi8siOnaMYe7cVSUVpkXatm0bFy5cYPPmzdSuXRuARo0aMXz4cDZu3Ej//v0BOHfuHKmpqfzrX/8qcAmppUuXcuLEiXzPrVq1qkDjioiIWCotNZFyq127Nnz66UyqVOkNXLptW2PSvZ/PP1+FnZ1dyQRoofbt20eLFi1MyS9A69atqVWrFlu3bjUdy1mSc2O724mOjmbx4sU899xz+Z7funVrgcYVERGxVEq8pVzLSb69vLoDWfm2cXKKpE2b3eUi6d69ezd169ZlwoQJuY5PmzaNunXrsmXLFi5evIi/v3+evv7+/hw5csT0ec7MdU6inJp6660ecpaYtGnThp49e+Y5f+XKFWJjYws0roiIiKVS4i3l3ueff8X1682B/JPq9PQHOXEilgsXLpRsYKWgffv29O7dmx07drBnzx4A9u7dy7p16xgwYAB16tQBwNfXN09fHx8fkpKSSEpKAoyJt6urKzNnzqRJkyY0adKEgICAfGen33//fc6cOcP06dPzjSs+Pr7A44qIiFgqJd5Srk2a9Cbvvx9/Q/WS/DTk9OnFtG2bt853WRQaGoqPjw9vvvkmiYmJTJs2jZo1axISEkJKSgoAFSpUyNPPyckJ+N/M9u+//05KSgpJSUnMnj2bt99+G1dXV1588UU2btxo6nfixAkWLVpESEgIVatWzTemwowrZde0adN4+umn8z1X0Io3P/zwA4MGDaJRo0a0a9eO6dOn59tu3759DBw4kCZNmtCuXTtmzJhh+j4UEctkMBhKO4Q70sOVUm7lJN1XrizgzkWXGpmS771719+yKkdZ4Onpyeuvv87YsWPp168f8fHxrF27FhcXlwL9ULOxMX4t+/fvz/Xr1xk0aJDpXPfu3Xn88ceZM2cOPXr0AGDKlCk0a9bstg9HFmZcKZvWr1/PunXraNGiRZ5zBa14c+DAAYKCgvDw8GDMmDHY2dnxwQcfsH//fj7++GM8PT0BY9I9YsQI/P39CQ4O5o8//mD16tUcPnyYyMhIbG01ZyVyJ7/88gsrVqzg559/JjExEV9fX9q1a8eYMWPyfffyn0hKSmLGjBk88cQTNG/evFivXdyUeEu5dOHCBZYuXU5S0q/cnHS7uCzCyWkNiYlbubnayfnzXZk5810WLgwryXBLXE4JwF27djFw4ECaNm0KgIuLcXPi9PT0PH1yjrm5GfdHGzhwYJ42zs7OBAYGsnDhQn7//Xe+/fZboqOj+fDDD02zjlevXgUgLS2NhIQEKlasWKhxpWzJzs5m8eLFLFy48JZtClrx5q233sLOzo6PP/6YGjVqAMbv9cDAQJYsWUJISAgAc+bMoVq1aqxduxZnZ2cAqlWrxhtvvMGePXvo0KGDOV+yiNX74IMPCAsLo2XLlkyaNAkfHx9OnjzJ8uXL2bVrF2vXruWee+4ptvGio6PZsGEDvXv3LrZrmot+bZdyqWrVqkRGLsXbuw/Gfb6MXFwW8cgj+/nss9n4+PTixmonTk6RtGz5I//+9xslH3AJS01N5ejRo4BxjXfOMo677roLgD///DNPn4sXL+Lh4WFKkm/Fy8vLNMaePXvIzMykX79+tGrVilatWpl+cEZERNCqVSvOnz9fLOOK9UlPT6d3794sWLCAwMDAW86SFaTizblz5/jtt98IDAw0Jd1gfPj3kUceYcOGDaYxK1WqRP/+/U1JN2CaaY+Oji721ylSlkRFRTFr1iyGDh3KwoULefzxx2nZsiVPPfUUH330EVlZWbz++uulHWapUeIt5VaPHl1YsSIYb+/eQJIp6d60aRUdO7bn009nmpJvJ6dIWrT4lP/852PTmuKybN68ecTFxTF58mTOnTvHvHnzAPDw8MDHxyffKiJHjx6lQYMGgPFhyO7du+c7SxkTEwOAn58fISEhrFy5MtfHnDlzAAgMDGTlypX4+Pjg4eGBn5/fHccV63GnCjq7d+8mPT2d5ORkwsPDCQsLw94+75u0Ba14k/OAbs4DwjeqUaMGiYmJ/PHHHzg5OREREcEzzzyTq82xY8eA//3yKSL5i4iIoGLFikycODHPOV9fX6ZMmUKrVq3IyjJWElu3bh3dunWjQYMGdOrUiWXLluVaXjhlyhSCgoJYv349nTt3pkGDBgQGBpoKABw4cMC0pHHIkCG5ngP5z3/+Q58+fXjwwQdp27YtYWFhZGRkmM4vWLCALl268O6779K8eXPat29v9mc5tNREyjVj8g1DhrSmTZvGbNr0v5KB7du35dNPZ9Knz7+oX79WuUm6o6KiiIyMpH///gQFBXHq1CkiIyPp0qULDz30EC1atGDnzp2cPHnSNMP43XffERMTQ1BQEGD84Xr16lXWr1/PsGHDTMtAzp8/z2effUbLli3x8fHBx8cnz/jnzp0D4O6776Z169am4507d2b16tW3HVesR04FnQ0bNrBnzx7atWuXq4JO+/btuX79Ort27co34c5R0Io3Oe+I5Pef6uXLlwHjOyrVqlXLdS4uLo4DBw4QFhZGnTp1ePTRR4v8mkXKOoPBwN69ewkICMDJycmUXN+oV69epr8vXbqU8PBwhg4dSrt27fj111959913SUhIYMqUKaZ2v/zyC3/88QcTJkzAzc2N+fPnM378eHbv3o2/vz9vvPEGr776Kq+++iotW7YEYPPmzQQHB9OrVy8mTpzI2bNnmTdvHufOnWPBggWma8fGxvLf//6Xd955h6tXr+Lq6mrGr5ASbxF69OjC/v21uO+++/LU6W7fvi0HDvwffn5+5SLpTk9PZ+rUqXh5eREcHAxAcHAwX3zxBVOnTuXzzz+nR48e7N+/n2HDhjFixAjS09NZvnw5/v7+uXaofO211xg7diwDBgygX79+pKSkEBkZib29Pa+99lqhYxs1ahSbNm2647hiPUJDQ9m7dy9vvvkmn3zySa4KOgC2trZ3fJCxoBVvateujZubGzt37mT06NGmh3HT09PZu3cvQK6ZMDAm5J06dTJd/5VXXikXPwdEiioxMZH09PQCvTOUlJTEe++9x6BBgwgNDQWgbdu2uLi4EBYWxpAhQ0zXSUpKYsOGDdx9992A8XmjwYMHc+DAAQICAkyTMffddx/33XcfBoOBuXPn8sgjjxAW9r9nsqpWrcrYsWOJioqiWbNmwP/2kWjcuHGxfi1uRUtNRIC6devecnOc2rVrl5v/bBcsWEBMTAwhISF4eHgAUKlSJSZNmsTp06d555138PDwYO3atTzwwAO8++67fPDBBwQEBLB8+XJT9QgwPrS2aNEiKlSowNy5c1m5ciWNGzfmo48+KvBuljfy8vIq0LhiPXIq6Jw5c8ZUQWfWrFmFWq9f0Io3jo6ODB8+nCNHjhAcHEx0dDTHjh1j/PjxpKWlAeT5GWBjY2Na5lK7dm2GDx/Ozp07C/ciRcqRnH9D2dnZd2z7008/ce3aNTp16kRWVpbpo1OnTmRnZ7N//35TWx8fH1PSDZhKz+b8273ZqVOnuHDhQp5rt2vXDgcHB7777rtc7evVq1fo11pUmvEWEZPg4GDTTPeN+vbtS9++fQHjUpR7772X999//47XCwgIICAgoFAx+Pn53fIBtoKOK9bjVhV0CqowFW+ee+45rl69ypo1a9iyZQsAjzzyCCNHjuTf//63qZxgDk9PT7p16wZAly5dePzxx5k5cyaPPfZY4V6kSDnh6emJq6sr58+fv2Wb5ORk4H9LvEaMGJFvu4sXL5r+fvM7WjnvWF2/fj3fvjnXnjZtGtOmTbvtte3s7Ep0ck2Jt4iIlJr8KugUZsa7MBVvbG1tefnllxk9ejSnT5+mWrVqVK9enfDwcOzs7G5bn9/Z2ZmOHTuyZs0aEhISTNV5RCS3tm3bcuDAgXx/GQZj+c/33nuPl19+GYDw8PBcs9k5qlSpUuQY3N3dAeNytpwlJTeqVKlSka/9T2mpiYiIlJpbVdApqMJUvNmyZQsHDhzA29ubhx56yJRof//99/j7++Pk5MTJkyfp1KkTkZGRea6XkpJiWrYiIvkbPnw4ly9fZv78+XnOnT9/nsjISBo2bEi3bt1wcHDg4sWLPPjgg6aPrKwswsPD8/1l+lZuXiZWu3ZtvLy8iIuLy3XtSpUqMXfuXE6ePPmPX2dRacZbRERKxZ0q6BRUQSverFq1imvXrrFx40ZTpZRvvvmGqKgo0wNYNWvWJCkpiY8//ph+/fqZkuy4uDh27txJ8+bNtVmTyG00adKEsWPHsnDhQn777TeeeOIJKlasyPHjx4mIiMDW1pY5c+bg5eXFiBEjCA8PJzk5mWbNmnH+/HnCw8Nxd3fn/vvvL/CYOc8kffPNN3h6evLAAw8wceJEpk+fjq2tLe3bt+fy5cssWLCApKQk6tevb66Xf0dKvEVEpMQVpIJOQdddFrTizahRoxg/fjxjxoyhc+fOxMXFsXLlStq2bUuPHj0AsLe355VXXmHy5Mk8/fTT9OzZk8TERNNW8fmtFxWR3J5//nn8/f1ZvXo1M2bM4OrVq1SrVo0uXbowevRo0zKSF154AR8fHz788EOWLFlCxYoVadeuHS+++GKh1l3XqlWLxx9/nMjISPbu3cvmzZt58skncXNzY/ny5Xz44Ye4ubnRvHlzXnzxxXxL2ZYYQzlw7do1ww8//GC4du1aofr98MMPZopICkv3wnLoXlgOa74Xc+bMMdSpU8ewadOmXMfXr19vqFOnjmHWrFl5+jzyyCOGwYMH53u9kydPGkaOHGlo3Lix4eGHHzaEhIQYLl26lKfdli1bDD179jQ0atTI0KlTJ0N4eLghNTU1T7utW7caevfubfD39zc0b97c8PzzzxtOnTp1y9djzfeirLGGe3H06NHSDqFEJCcnl3YIZnG7+3ennFMz3iIiUuIKUkHnZl999dUtr1fQijfdu3ene/fud2zXrVs3U0UTEZHiosRbRIrNlStXWLp0NV9/HUVKSiqVK1di6NDu9OjR/ZZ10kVERMoLJd4i8o8lJSURFDSZPXuOkJAwlIyMVwEX4C+++modHh4zGDfuaSZPHmuqvyoiIlLeqJygiPwjCQkJNG/enc8+68qFC9+SkREE3AtUBRpw9eobnDu3h7ffPs/gwc8XaKdBERGRskiJt4gUmcFg4NFHn+LEibfIzu4J3Go225GrV99m82YvXn11bkmGKCIiYjGUeItIkX3zzTfExNTj+vX2BWqflPQ6ERGfkZaWZubIRERELI8SbxEpstdfX0xi4rOF6GHL5ctPsmbNJ2aLSURExFIp8RaRIsnKyiI6+g+gTqH6paU9zbJln5onKCkXDAYD3377LZ06DaBGjfb4+XWgRo12dO06hP379+s5AhGxWKpqIiJFcuXKFWxsirL7lxdXryYVezxSPsTExNCt23AuXmxEQsJrQD3TudjYX/j++8VUrz6N7ds/4K677iq9QEVE8qHEW0SKxLidb3oRel7Hzk5vtknhxcTE0K7dQOLiVnJjwv0/jbh0aQmXLv1Mq1ZPsG/f/yn5FhGLosRbRIrE1dUVe/tLQCbgUIieP9CgwQNmikrKKoPBQNeuw4iLWwXc6funMWfPvkf37sP56aed5g9OpJQZDAYOHTrEkSNHSE1NxcXFBX9/fxo2bFjieyds2bKFxYsXExsbS7Vq1Xj22Wfp1atXicZgyZR4i0iR2NjYMHhwT+bM2Uh2dr8C9/P2fo+pU18wY2RSFn3zzTdcvNiMOyfdOZpw7lx1oqKiaNasmTlDEyk1WVlZbNq0idWrV5OQkEBWVhZZWVnY29tjb2+Pl5cXQ4YMITAwEHt786d827ZtIzg4mKFDh9K2bVt27NhBSEgIzs7OdOnSxezjWwO93ysiRTZxYhCVKy/AOOtdEDH4+p6jcePG5gxLyiBjBZ1nCtXnr7+e5Y033jNTRCKlKzU1lWeeeYbw8HDi4uJIS0sjMzMTg8FAZmYmaWlpxMXFER4ezjPPPENqaqrZYwoPD6dr166EhobSrl07Xn75Zbp27cr8+fPNPra1UOItIkXm6+vLtGlDqVQpCMi6Q+vz+PoOYv36hSURmpQxp05doLAVdKA5v/zymznCESlVWVlZjB8/nqNHj3Lt2rXbtr127RpHjhxh/PjxZGXd6ed00cXGxnL27Fk6d+6c6/hjjz3GqVOniI2NNdvY1kSJt4j8I+PGBfHqq23x8ekKfAncXMotFUfHFfj59WH79kXUq5ffQ3Eit3f9etHWqV6/rv/mpOzZtGkTx48fJyMjo0DtMzMzOX78OJs2bTJbTKdOnQKgVq1auY7XrFkTMD4cLUq8RaQYTJw4mu++W8yIEdupVq0NVao8g7f3i/j6DuGeewJ4441kfv11B02aNCntUMVK2doWpTa3ATu768Uei0hpMhgMrF69+o4z3Te7du0aq1evNlud+6QkY5lYNze3XMddXV0BSE5ONsu41kYPV4pIsbjvvvuIiJhLWloap0+fJjU1lYoVK3LvvfeW+FP1Uvbcf/9dnDt3jPzLCN7KQZo0UQUdKVsOHTpEQkJCkfomJCRw6NAhGjVqVMxRcceE3tZWc72gGW8RKWYVKlSgXr16NGvWjNq1ayvplmLx2mvPUqnSkkL18fZ+j1dfHWumiERKx5EjR4q8VjsrK4ujR48Wc0RG7u7uAKSkpOQ6njPTnXO+vFPiLSIiFq99+/ZUrXoIOFLAHt9To8ZFVdCRMic1NfUfJd43J8bFJWdt99mzZ3MdP3PmTK7z5Z0SbxERsXg2NjZs374KP78g4PAdWv/APfeMZ+vWlSURmkiJcnFxKXJNbnt7e9Oa6+JWs2ZN/Pz82LFjR67ju3bt4p577tEusn/TGm8REbEKNWvW5L//XUf37sP54486XLr0LNDwhhY/4O39HjVq/MHWrRuoWrVqaYUqYjb+/v7Y29uTmVnQ/RP+x97envr165shKqOxY8cSGhqKp6cnHTt2ZOfOnWzfvp3w8HCzjWltlHiLiIjVqFGjBocOfcGBAweYPn0ux46dJjvbBlvb6zRuXIdXXx2rnSqlTGvYsCFeXl7ExcUVuq+XlxcNGza8c8Mi6tOnDxkZGaxYsYL169dTvXp1wsLC6Natm9nGtDZKvEVExKrY2Njw8MMPs337w6UdikiJs7GxYciQIYSHhxeqpKCzszNDhgwx+wPvAwYMYMCAAYDxQUtzLW2xVlrjLSIiImJFAgMDeeCBB3BwcChQe0dHR+rVq0dgYKCZI5M7UeItIiIiYkXs7e1599138ff3x9nZ+bZtnZ2d8ff3Z/78+UV+KFOKj+6AiIiIiJVxcXFhyZIlbNq0idWrV5OQkEBWVhZZWVnY29tjb2+Pl5cXQ4YMITAwUEm3hdBdEBEREbFC9vb2PPHEE/Tp04dDhw5x9OhR07pqf39/HnzwQW1iZmGUeIvVmzZtGqdPn2bNmjV5zsXGxhIWFsbBgwcB6NixI1OmTMHLywuAc+fO8a9//eu211+9ejUtW7akbt26t203c+ZM+vTpU8RXISIiUjQ2NjY0atTILFvBS/FS4i1Wbf369axbt44WLVrkOZeYmMjQoUPJyMhg5MiRZGdnExERQXR0NOvXr8fR0REvLy9mz56dp296ejpvvvkmlStX5oEHHgDIt53BYGDWrFlkZmbSvHnz4n+BIiIiUmYo8RarlJ2dzeLFi1m4cOEt26xatYoLFy6wefNmateuDUCjRo0YPnw4GzdupH///ri4uOT7lPeMGTPIyspi7ty5eHp6AuTbbtWqVSQmJjJ37lzuvvvuYnp1IiIiUhapqolYnfT0dHr37s2CBQsIDAzE19c333Zbt26lRYsWpqQboHXr1tSqVYutW7fe8vrR0dGsXbuW3r1789BDD92y3V9//cX8+fNp1aoVPXr0KPoLEhERkXJBibdYlN27d1O3bl0mTJiQ6/j7779P3bp12b17N+np6SQnJxMeHk5YWFi+T2pfuXKF2NhY/P3985zz9/fnyJEjt4whPDwcZ2dnJk6ceNtYlyxZwrVr15gyZUoBX52IiIiUZ0q8xaK0b9+e3r17s2PHDvbs2QPA3r17+frrrxkwYADt27fHzc2NXbt23XYL2vj4eIB8Z8N9fHxISkoiKSkpz7njx4+bxqpSpcotr5+QkMAnn3zCo48+aloDLiIiInI7SrzF4oSGhuLj48Obb75JYmIi06ZNw9fXl5CQEABsbW3vWI80JSUFgAoVKuQ55+TkBEBqamqecx999BF2dnYMHjz4ttf/9NNPycjIYOjQoQV6TSIiIubwxRdfcPHixXzPXbx4kS+++KKEI5LbsYrE22AwsGrVKh577DEaNmxIz5492bx5c2mHJWbi6enJ66+/zpkzZ+jXrx/x8fE8++yzuLi4FPgaBoPhjm1urm167do1Pv/8czp16kT16tVv2/fTTz+lfv36NGvWrMAxiYiIFKeIiAimTZvG0KFD8yTf8fHxDB06lGnTphEREVEq8UVHR+Pv78+FCxdKZXxLZBWJ99KlS5k9eza9evVi6dKltGnThuDgYLZt21baoYmZBAQE0LlzZ2JjY+nfvz916tQpVP+cJD09PT3PuZxjbm5uuY4fOHCA1NRUunTpcttr//7775w5c+aO7URERMwlIiKClStXkpmZSUJCQq7kOz4+nmHDhpGQkEBmZiYrV64s8eT75MmTTJgwgaysrBId19JZfOKdmZnJihUrGDhwIM8++yytWrUiJCSEFi1asHbt2tIOT8wkNTWVo0ePAsY13teuXStU/7vuuguAP//8M8+5ixcv4uHhkWcG/dtvv8XR0ZGOHTve9trffvstAI8++mihYhIRESkOOUl3zv+N2dnZpuT78OHDpqQ7OzsbML6jW1LJd1ZWFpGRkfTr1y/fya/yzuITbzs7O9asWcPo0aNzHXdwcNANLcPmzZtHXFwckydP5ty5c3zyySeF6u/h4YGfn1++1UuOHj1KgwYN8hz/8ccfadCgQZ6Z8Pza+fr6cu+99xYqJhERkX/qiy++YPny5XkmpHKS75uT7hzXrl1j+fLlZl/zHRUVxdy5cxkxYgTjx48361jWyOITb1tbW+rWrYuvry8Gg4G//vqLZcuW8d133/Hkk0+WdnhiBlFRUURGRtK/f3+CgoJ44okn2LVrFz/88EOhrtO5c2f27dvHyZMnTce+++47YmJi8lREyczM5Pfff6d+/fp3vO6xY8cK1E5ERKS4NWzYkIoVK2JnZ5fnXE6yfXPSDcaJzIoVK9KwYUOzxle7dm2++OILxo0bl2+M5Z3FJ9432rVrF23atOHf//43HTp0oGfPnqUdkhSz9PR0pk6dipeXF8HBwQAEBwfj6urK1KlTC/Uux6hRo/D09GTYsGGsXLmSJUuWMH78ePz9/fPsQvnHH3+QmZlJtWrVbnvNrKwszp8/f8d2IiIi5lClShVWrVqFl5dXgRNbOzs7vLy8+OCDD25bKrc4eHt7U7lyZbOOYc2sasv4+vXrs3btWqKjo5k/fz6jR4/mgw8+yFOd4lYOHz5c6DGjoqIK3UeK7qOPPiImJobnnnuOEydOmI4/9dRTLFu2jNDQUAYNGpSrT3p6OklJSfneq9DQUNasWUN4eDhOTk40adKEp556il9//TVXu99//x0w1ue+3T2/cuUKBoOB5OTkcv29UZ5fu6XRvbAcuheWw9Lvhb29vansbVG4ubnx3nvv8cwzz3D58uV8Z7hz5Mx0L168GFdX1380blGlpqaWyrjmkpGRUeTvMRtDQequWaCNGzcSEhLCRx99RNOmTW/bNj09ncOHD9OgQQNTDeeCiIqKUrk4C6F7YTl0LyyH7oXl0L2wHNZwL44dO0a9evX+8XVyHqS8k1WrVuX7bJO5ffTRR7z++ut8++23VK1atcTHN5fb3b875ZwWv9Tk8uXLbNy40bQTYY6cNba3KhovIiIiUlbFx8czadKkOy43sbOzY9KkScqXLITFJ97Xr19nypQpeapa/Pe//wUodH1nEREREWt2Y53u2y0zgdylBpV8lz6LX+Pt5eVlWt/r7OzMgw8+SFRUFEuXLqVfv34q6SYiIiLlxsWLFwucdOe4MfkuiQcs5dYsPvEG4wNy1apV49NPP2XBggVUrVqV8ePHExQUVNqhiYiIiJSYQ4cO3fKBSjs7O7Kzs01/3ig7O5vLly9z6NAhAgICSipcuYnFLzUB42Y5o0aNYufOnfz666/85z//YdSoUdjaWkX4IiIiIsUiICCAkSNH4uzsnOt4TsnAW5UadHZ2ZuTIkSWadPfs2ZPo6Ogy9WDlP6XMVURERMSKBAUFMXz4cFPyfWOd7gYNGuRJvp2dnRk+fLhWClgAJd5SphgMBvbv38/Agc/TuvUTtGvXnzFjphAdHV3aoYmIiBSbnOTbwcEhz+Y4vr6+puTbwcFBSbcFsYo13iIFsXXrLiZOfIvExPpcuhQE1AKy2bv3MBs3TqN69SRWr55TKrVMRUREiltQUBA1a9akYcOGeR6Y9PX15YMPPtCabgujxFvK9inelgAAIABJREFUhOXLI5ky5WMuXdoEVLrprC8XL/6LixdjCAh4mk8/nU3btq1LI0wREZFidbukukqVKkq6LYyWmojV27Pnv0yZEsmlS/9H3qT7RrWIj99Iv34vERsbW1LhiYiIiABKvKUMCA4O49KlxYBjAVp7Ex//GjNmLDJ3WCIiIrdkMBhKOwQpgn9635R4i1U7deoUp0/bATUL3Mdg6Mznn3/DtWvXzBeYiIjILTg4OJCWllbaYUgRpKWl4eDgUOT+SrzFqn388SYuXhxYyF62pKR0Zu/evWaJSURE5HaqVKlCXFwcqampmvm2EgaDgdTUVOLi4v7Rzp96uFKsWlzcJeChQvdLS6vKpUuXij8gERGRO/Dw8ADg/PnzZGZmlnI05pORkYGjY0GWgVoHBwcHfH19TfevKJR4i1VzdXUGCr9kxN7+Wp5dv0REREqKh4fHP0rgrEFUVBSNGjUq7TAsipaaiFVr2dIfd/d9he7n7n6A+vXrmyEiERERkfwp8Rar1rPn47i7bwWyCtHrPHfddZX777/fXGGJiIiI5KHEW6yag4MDTz3VHUfHNQXu4+Exl5dfHmXGqERERETyUuItVu+NNybxwAOR2Nr+545tK1RYTMeOifTt27sEIhMRERH5HyXeYvUqVKjA7t3/R9Om8/DwCAXO5tPqF7y8htO9+098+ukybGxsSjpMERERKedU1UTKBE9PT/bv38K2bTt4661xnDmThcFQE8jG1vYEjRr58dprz/Hwww8r6RYREZFSocRbygw7Ozt69OhOjx7duXTpEvHx8djZ2VGtWrUyX7JJRERELJ8SbymTKleuTOXKlUs7DBERERETrfEWERERESkBSrxFREREREqAEm8RERERkRKgxFtEREREpAQo8RYRERERKQGqaiIiInnExsYSFhbGwYMHAejYsSNTpvw/e3cel2WV93H8A7iiQqBoGuoohRY8udNobimi4YKUkkqmiGZlWRmKpjQWuaAFjks6KeNKLlSuVDqluTwaKU09qUmJpkDuoCDoDcL9/OFwjwQoINzc6Pf9evmauO5zzvU7XoP+OP6ucybj6OiYr92hQ4eIiIjg8OHD2NnZ4enpyWuvvVagXUpKCuHh4ezcuZPr16/j5ubGW2+9RevWrQFISkqiZ8+et41p1apVPPHEE2U4SxER81LiLSIi+aSmpjJixAiysrIYPXo0OTk5REZGEh8fT3R0NNWqVQMgNjaWwMBA7OzsGDt2LDY2NqxcuZLvvvuOdevWYW9vD8DVq1fx9/fn/PnzjBw5Ejs7O6KiohgxYgTR0dG4urri6OjInDlzCsRiMBgIDQ2lbt26tGzZ0qy/DyIiZU2Jt4iI5LNixQrOnj3L1q1bcXFxAaBVq1YEBASwadMm/Pz8AHj//fexsbFh3bp1NGnSBABPT098fHxYsmQJwcHBACxdupSTJ0+yevVqOnToAIC3tzeenp4sW7aMOXPmYGtri4+PT4FYZsyYwY0bN/jggw9MibyISGWlGm8REcknJiYGDw8PU9IN0KlTJ5o1a0ZMTAxwszTk119/xcfHx5R0A7i4uPDUU0+xceNGAIxGIxs3bqR79+6mpBvAycmJSZMm0b59+yLjiI+PZ82aNfj6+t62nYhIZaHEW0TkPrJnzx5atGjB66+/nu96SEgILVq0YNu2bSQmJuLm5lagr5ubG0eOHAHg3LlzALi6uhZo16RJE1JTUzlz5gxJSUmcO3eOTp06ATcT8YyMDAD8/f1Nq+eFiYiIoEaNGrzxxhulm6yIiIVR4i0ich/p2rUrvr6+fPXVV+zduxeAffv2sWHDBoYMGWJKpBs0aFCgr5OTE+np6aSnp2NrawtgSqJvdfnyZQAuXLjAqVOnAKhbty5hYWG0b9+etm3b0qtXL3bu3FlknMeOHWPXrl0MGTKE+vXr392kRUQshBJvEZH7zJQpU3ByciI0NJTU1FRCQkJo2rQpwcHBpkS6Zs2aBfpVr14dgMzMTFxcXKhduzbbt2/HaDSa2hgMBvbt2wdAVlYWaWlpAPz9739n9+7dTJ06lbCwMGrUqMG4cePYv39/oTGuXbsWGxsbnn/++TKdu4hIRVLiLSJyn7G3t2f69OmcOnWKwYMHc+7cOWbPno2trW2+JLooVlZWVKtWjYCAAI4cOUJQUBDx8fH88ssvjB8/nmvXrgFgY2NDVlYWAGlpaaxdu5ZnnnmGgQMHEhUVhZ2dHR9++GGB8a9fv86WLVvo0aMHDz30UNlOXkSkAinxFhG5D3l6euLl5UViYiJ+fn60bdsWwFRCYjAYCvTJu1a7dm0AXnnlFUaMGMEXX3zBgAEDGDhwIFZWVowePRq4meDnjefl5ZVvVxI7Ozt69OjBkSNHCpSrxMbGkpmZSZ8+fcp41iIiFUvbCYqI3IcyMzM5evQocLPGOzMzE1tbWxo1agTcrM/+s/Pnz2NnZ2dKpq2trXn77bd58cUX+f3332nYsCEPPfQQERER2NjY8NBDD5Geng5Q4ECdvGtGo5HMzExq1aplur57926qVatG9+7dy3raIiIVSiveIiL3ofDwcJKTk5k0aRJJSUmEh4cDN1einZ2dTbuX3Oro0aO4u7ubvt62bRuxsbHUq1eP9u3bm8pCDh48iJubG9WrV+eRRx6hWrVqHD9+vMB4SUlJVK9evUBS/sMPP+Du7m5aWRcRuVco8RYRuc/ExcURFRWFn58fgYGBPPvss0RFRXHo0CHgZlnIgQMHSEhIMPXZv38/J0+exNvb23RtxYoVhIaGcuPGDdO1b7/9lri4OPz9/YGbpSs9evTg22+/5bfffjO1S0xMZOfOnfTs2RMbGxvT9ezsbI4fP85jjz1WbvMXEakoKjUREbmPGAwGpk6diqOjI0FBQQAEBQXx9ddfM3XqVLZs2cKYMWPYvHkzI0eOZNSoURgMBpYtW4abm1u+0yXHjBnD+PHjGTt2LF5eXiQnJ7N8+XI6d+5M//79Te0mTpzI999/zwsvvMALL7xA1apVWbVqFTVq1GDChAn54jtz5gzZ2dk0bNjQPL8hIiJmpBVvEZH7yIIFCzh58iTBwcHY2dkB4ODgwMSJE/n999+ZN28ejo6OrFmzhpYtWzJ//nxWrlxpOt69WrVqprF69+5NeHg4Fy9eZNasWcTExBAYGMjChQvzrWI7OzuzYcMGOnToQGRkJIsXL+bRRx9l3bp1NG7cOF98eXuAq8xERO5FWvEWEbmPBAUFmVa6bzVo0CAGDRpk+rp58+YsXbr0juP17duXvn373rFd48aNmT9//h3bPf7448THx9+xnYhIZaQVbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETED7WoiIiIllpubS1paGrm5udjb2+fbPlBERAqnxFtERIrt+PHjzJq1hC+/3IvR2ACwxmg8S8eO/8Pf/jaO1q1bV3SIIiIWS4m3iIjcUW5uLq++OpVPPz3ChQuvA3MBq/98amTTpoPs3RtOx45WfPbZ0nwH7YiIyE2q8RYRkTsaNWoCq1fX4sKFzUBP/pt085//9uDSpVX861896d17GDk5ORUTqIiIBVPiLSIit7VlSwybNl3j6tVp5E+4CzIYXuD779vz4YcfmSc4EZFKRIm3iIjcVmjoYq5cmVbs9pmZ41m8eC1Go7EcoxIRqXyUeIuISJFOnDjB6dPVgcYl6GXLlSsd2blzV3mFJSJSKSnxFhGRIv3www9cudK9xP1SU59i7964sg9IRKQSU+ItIiJFyszMJCurZil62pKWllnm8YiIVGZKvEVEpEgODg7Y2l4qRc+LNGjwQJnHIyJSmSnxFhGRInXt2pU6dWKAkr0oWb9+NM88410+QYmIVFJKvEVEpEj29vZ07vwYEFuCXkk4O2fyyCOPlFdYIiKVkhJvERG5rffee5369acAxanZzsHR8XVmzZpQ3mGJiFQ6SrxFROS2Hn30URYseJ26dX2B1Nu0NODo+AITJz6Fl1dPc4UnIlJpVKnoAERExPL5+Q3EwcGOsWP7cflyB1JTxwItuHmSZSJ2dsuws9vB+++/yYgRz1VwtCIilkmJt4iIFEuvXj1ISHiK3bt3M3PmTE6fTsRoNFK/vhNvvfUC/fv/DRsbm4oOU0TEYinxFhGRYrOysqJ79+507969okMREal0LD7xzs3NZf369XzyySckJSVRt25devbsyWuvvUbt2rUrOjwRERERkWKx+MR72bJlzJs3j8DAQDp27MjJkyeZP38+x48fJzIysqLDExEREREpFotOvI1GI8uWLeO5557jrbfeAqBTp044ODjw5ptv8ssvv/Doo49WcJQiIiIiIndm0dsJZmRkMGDAAPr165fvevPmzQE4ffp0RYQlIiIiIlJiFr3iXbt2baZNm1bg+tdffw3Aww8/bO6QRERERERKxaJXvAvz008/8fHHH+Pp6YmLi0tFhyMiIiIiUixWRqPRWNFBFFdcXBwvvfQSTk5OREVF4eDgUKx+BoOBw4cPl3N0IiIiIiLg7u5O9erVC1y36FKTW33xxRdMnjyZv/zlLyxbtqzYSfetivpNKEpcXBzt2rUr8X2k7OlZWA49C8uhZ2E59Cwsh56F5bgfn8WdFnsrRanJ8uXLmTBhAq1btyYqKor69etXdEgiIiIiIiVi8Yl3dHQ0s2fP5umnn2bZsmXUqVOnokMSERERESkxiy41uXTpEjNmzOChhx7C39+fo0eP5vu8SZMmODo6VlB0IiIiIiLFZ9GJ9969e7l27RrJycn4+/sX+HzOnDn4+PhUQGQiIiIiIiVj0Yn3wIEDGThwYEWHISIiIiJy1yy+xltERERE5F6gxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImIESbxERERERM1DiLSIiIiJiBkq8RURERETMQIm3iIiIiIgZKPEWERERETEDJd4iIiIiImagxFtERERExAyUeIuIiIiImEGlS7x/+eUX3NzcOHv2bEWHIiIiIiJSbJUq8U5ISGDs2LHcuHGjokMRERERESmRSpF437hxg6ioKAYPHozBYKjocERERERESqxSJN5xcXF88MEHjBo1iqCgoIoOR0RERESkxKpUdADF4eLiwtdff03dunX5/PPPKzocEREREZESqxQr3vXq1aNu3boVHYbcZxITE3n11Vfx8PDAw8ODSZMmkZKScts+x44dw93dnQULFhT47MCBAwwdOpQ2bdrQpUsXZsyYQUZGRqnHExERkcqlUqx4l5XDhw+XuE9cXFw5RCKlYc5nkZ6eztSpU7lx4wZPP/00ubm5bNu2jR9//JHQ0FCqVCn4rZOTk0NISAjZ2dn88ccf+eI9cuQIM2fOpFmzZvj5+XHp0iXWrl3Ld999xzvvvIO1dcGfgW83XkWzpFjud3oWlkPPwnLoWVgOPYv87qvE293dnerVqxe7fVxcHO3atSvHiKS4zP0sIiIiSE1NZevWrbi4uADw9NNPExAQwKlTp/Dz8yvQZ9GiRSQnJwPQqFGjfPHOmDGDRo0asXHjRmrUqAFAmzZteO+998jMzKRbt24lGq8i6fvCcuhZWA49C8uhZ2E57sdnYTAYbrvQe9tSk59//pnAwEDatGlDhw4dGDt2LIcOHSq07ZYtW3j00UfvLloRCxETE4OHh4cp6Qbo1KkTzZo1IyYmpkD7+Ph4Fi9ezCuvvFLgM4PBgIODA35+fqakG8DDw8PUtyTjiYiISOVUZOL9008/MWzYMA4fPsyTTz6Jq6sre/bs4YUXXmDevHnmjFGkTO3Zs4cWLVrw+uuv57seEhJCixYt2LZtG4mJibi5uRXo6+bmxpEjR/Jdu3HjBlOmTOHJJ59kwIABBfpUr16dyMhIXnrppXzXf/nlF+DmanZJxhMREZHKqchSk3nz5tGgQQPWr19verHx2LFjBAcH849//INLly4RGhpqtkBFykrXrl3x9fVl48aN7N27ly5durBv3z42bNjAkCFDcHV1BaBBgwYF+jo5OZGenk56ejp16tQBYOnSpZw6dYqPPvqoWIc7JScnExsbS1hYGK6urvTq1Svf5yUdT0RERCqHIle8f/zxR4YNG5ZvN5GWLVuydu1aOnTowKeffsr06dPNEaNImZsyZQpOTk6EhoaSmppKSEgITZs2JTg42LTTSM2aNQv0y3tHIDMzE4DffvuNRYsWERwczIMPPnjH+16+fJkePXowZcoUDAYD06ZNy/feQUnHExERkcqjyMTbaDRiY2NT4LqtrS0ff/wxrVu3Zv369YSFhZVrgH/2zDPPEB8fr6RE7oq9vT3Tp0/n1KlTDB48mHPnzjF79mxsbW0xGo137G9lZUVOTg6TJ0+mXbt2hb5sWVS/iIgIwsLCcHFxISAggO3btwOUajwRERGpPIpMvB977DE+++wzsrKyCnxWo0YNPv74Y1q0aMGKFSuYPXs2OTk55RqoSFnz9PTEy8uLxMRE/Pz8aNu2LXDzh0u4+VLkn+Vdq127NpGRkcTHx/PWW2+RkpJCSkoKaWlpAFy7do2UlBRyc3Pz9be3t8fb25uBAwcSFRVFo0aNmDVrFkCpxhMREZHKo8ga75dffpkXX3yRPn364O3tzYgRI3BycjJ9XqdOHf75z38yatQoVq5ciZ2dnVkCFikrmZmZHD16FIB9+/aRmZmJra2t6WXHCxcuFOhz/vx57OzssLW1Ze/evWRnZzN48OAC7SIjI4mMjOSbb77B2dm50PvXqFGD7t27s3r1alJSUu56PBEREbFsRSbeXbp0YeXKlcycOZN//vOfhf7Tt6OjI1FRUbz77rts2bIFKyurcg1WpCyFh4eTnJzMpEmTmDt3LuHh4UybNg07OzucnZ0L7F4CcPToUdzd3QEIDg42rUjnuXjxIhMnTsTHx4eBAwfi5OREQkICY8aMITAwEH9//3ztMzIysLKyolq1asUeT0RERCqn2x6g4+HhwaZNm0hLS6N27dqFtqlVqxZz5sxh7NixHDx4sFyCFClrcXFxREVF4efnR2BgICdOnCAqKoo+ffrQvn17vLy8WLVqFQkJCaa9vPfv38/JkycJDAwEMCXgt0pKSgKgcePGdOrUCYCmTZuSnp7OunXrGDx4MNWqVQNu7m6yfft2OnToQO3atYs9noiIiFROtz1AJ4+dnV2hR1rfysXFhSFDhpRJUCLlyWAwMHXqVBwdHQkKCgIgKCgIOzs7pk6disFgYMyYMdjb2zNy5EiWL1/OkiVLGD9+PG5ubvj4+JToflWqVGHatGn8+uuvDB8+nKioKBYuXMigQYOwtrYmJCSkPKYpIiIiFqZYibfIvWTBggWcPHmS4OBg07sJDg4OTJw4kd9//5158+bh6OjImjVraNmyJfPnz2flypV4enqybNky04p1Sfj4+BAREUF2djazZs1i1apVdOjQgejoaNO+4SIiInJvu22pici9KCgoyLTSfatBgwYxaNAg09fNmzdn6dKlJRrb2dm50CPgAby9vfH29i6z8URERKRy0Yq3iIiIiIgZKPEWERERETGDu0q8L126pINzRERERESK4Y6J95o1a+jfvz83btwo8NnMmTPp0qULK1asKI/YRERERETuGUUm3kajkUmTJvH+++9z/vx5/vjjjwJtnJ2dsba2JiwsjAkTJpRroCJS9hITE3n11Vfx8PDAw8ODSZMmkZKScts+S5cuZfjw4WU23rFjx3B3d2fBggWlnoeIiEhlUGTiHR0dzZYtWxg2bBh79uyhSZMmBdq8+eabfPPNN/j4+PDll1+yadOmcg1WRMpOamoqI0aM4Mcff2T06NEEBASwc+dOAgICyMrKKrRPdHQ0u3btKrPxbty4wZQpU8jOzi6zeYmIiFiqIrcTjI6OpkOHDrzzzju3HaB69erMnDmT+Ph41q1bx8CBA8s8SBFLkpGRwdqVK9n12WemU1279O/P8DFjqFOnTkWHV2wrVqzg7NmzbN261XQ6Z6tWrQgICGDTpk34+fmZ2ubk5LB48WIWLlxYJuPl+cc//sFvv/1WxjMTERGxTEWueB8/fpyePXsWbxBra3r37q39huWelp2dzZSXX8brkUe49tZbzNy5kzWHDjH322+xefttnnZ15c0RI7h+/XpFh1osMTExeHh4mJJkgE6dOtGsWTNiYmJM1wwGA76+vixYsAAfHx8cHR3varw88fHxLF68mFdeeaUMZyUiImK5iky8bWxsSnRCn4ODwx2PlReprAwGA89260bzlSvZd+YMr12/TlPAHnAGxhoM7D17lvbr1jGgUycyMzMrNN49e/bQokULXn/99XzXQ0JCaNGiBdu2bSMxMRE3N7cCfd3c3Dhy5Ijpa4PBwNWrV4mIiCAsLKzQ7/MrV64Uezz4b4nJk08+yYABA0o7TRERkUqlyEy5adOmHD58uNgD/fzzzzRq1KhMghKxNOP9/Rn8ww+MuXYNqyLaWAH+WVm8fPgwoyu45Kpr1674+vry1VdfsXfvXgD27dvHhg0bGDJkiOmY+gYNGhTo6+TkRHp6Ounp6QDUrl2bHTt23PbUzXPnzhV7PLj5guapU6d49913Sz9JERGRSqbIxLtv375s3bq1WPWXv/32G1u3bqVr165lGpyIJUhMTCRp716GGwzFau+bnU3OTz9x7Nixco7s9qZMmYKTkxOhoaGkpqYSEhJC06ZNCQ4OJiMjA4CaNWsW6Fe9enUA06q9tbU1VaoU+ToIQInG++2331i0aBHBwcE8+OCDpZydiIhI5VNk4v3cc8/RqFEjhg8fzpYtWwo9KCc3N5etW7cSEBBArVq1GDFiRLkGK1IRPp47l7EXL5aoz8vnz7Nk5sxyiqh47O3tmT59OqdOnWLw4MGcO3eO2bNnY2tri9FovGN/K6ui1vYLKu54OTk5TJ48mXbt2hX6sqWIiMi9rMhlrFq1aplefAoODubdd9/Fzc0NJycncnNzuXTpEkeOHCEzM5OGDRuyaNEi6tevb87YRczi223bmJ6bW6I+3YCp335bLvGUhKenJ15eXuzYsYOhQ4fStm1bAGxtbYGb9dt/lnetdu3axb5PcceLjIwkPj6eTz75xLS/d1paGgDXrl0jJSWFBx54QO+LiIjIPem2/37cvHlztmzZQlRUFDExMfzwww+mEyyrVq1K69at8fLy4rnnnivRi5gilYn1jRvYlLCPFVClkH8lMrfMzEyOHj0K3KzxzszMxNbW1vQ+xoULFwr0OX/+PHZ2dqZkujiKO97evXvJzs5m8ODBBdpFRkYSGRnJN998g7Ozc7HvLSIiUlncvnATqFatGgEBAQQEBACQkpKCjY0N9vb25R6ciEUo5eqrlQWs2oaHh5OcnMykSZOYO3cu4eHhTJs2DTs7O5ydnQvsNgJw9OhR3N3dS3Sf4o4XHBxsWuHOc/HiRSZOnIiPjw8DBw7EycmpRPcWERGpLO6YeGdnZ3P8+HFu3LjBww8/XOQeviL3KoeGDfn91Cn+UoI+54HqDg7lFFHxxMXFERUVhZ+fH4GBgZw4cYKoqCj69OlD+/bt8fLyYtWqVSQkJJj23t6/fz8nT54kMDCwxPcrzniFJfRJSUkANG7cmE6dOpV2uiIiIhbvtktyK1asoFOnTjzzzDP4+fnx17/+lbCwMFO5icj9YMzbb/OxnV2J+vyzVi0CJk0qp4juzGAwMHXqVBwdHQkKCgIgKCgIOzs7pk6disFgYMyYMdjb2zNy5EiWL1/OkiVLGD9+PG5ubvj4+JT4nmU9noiIyL2myMR706ZNzJ49Gzs7O/z9/Rk+fDh/+ctfWLFiBXPmzDFnjCIVqo+3NzsfeIDzxWx/Gfjc3p5nKnDXjgULFnDy5EmCg4Ox+88PDQ4ODkycOJHff/+defPm4ejoyJo1a2jZsiXz589n5cqVeHp6smzZslK9s1HW44mIiNxrrIxF7APm5+eHtbU1K1euNO3DazQaefPNN9m1axcHDx6sNH+ZGgwGDh8+jLu7u2kuxREXF0e7du3KMTIprop+Ft8dOMBkHx8+v3CB2xVbpQHP1qvH5Kgoenp5mSs8s6roZyH/pWdhOfQsLIeeheW4H5/FnXLOIle8ExIS6N+/f75OVlZWjBw5kqysLE6cOFE+EYtYoL927Mh769fj3bAhq6pW5fqfPs8C1trY0LtBA4JWrLhnk24REREpvSJfrrx27Rp16tQpcN3Z2Rmj0VhgZwKRe13Xp57ii8OHWfHRR3RfupTmBgP2ubmkW1tzvFo1fEeMYMv48dqVQ0RERApVZOKdm5tb6Ml1NjY3dzQu7CRLkXudo6MjE6ZN44233yY5OZn09HRq1aqFs7Oz6XtDREREpDB33E5QRAqytramcePGFR2GiIiIVCK3TbwvX77MH3/8ke/alStXgJsH6fz5M/jvCXYZTye8AAAgAElEQVQiIiIiIvJft028Z86cycyZMwv9LG9v4FtZWVmZjqcWkcrt0KFDfLVhA1cuXsS2dm06eHrydN++FR2WiIhIpVVk4u3r62vOOETEQnz6yScsfO89Hr50iUEXL+IIZAJfr1zJ+/b2dOzdm1aLF1OliirVRERESqLIvzlnzZplzjhExAJMf/NNElesYOPly/z5wPvuaWm8k5bG/NWrGXTsGOu//rpE++KLiIjc7257ZLyI3D+WfPABF5YvZ1khSXeeakCQwcDggwcZ8+yz5gxPRESk0lPiLSJcv36dleHhzL9yhYKbiBbkbzCQ+/33/Pzzz+Uem4iIyL1CibeIEP3JJwy6coWS7EQ+7sIFPgoNLbeYRERE7jVKvEWEtQsXMjIzs0R9/gr8e+/e8glIRETkHqRtCe5CYmIiYWFhfP/99wB0796dyZMn4+joWGSfkJAQfv/9d1avXl3i8ZKSkujZs+dtY1q1ahVPPPFEaack96mMtDTqlrCPFfAAcO3aNWrWrFkOUYmIiNxblHiXUmpqKiNGjCArK4vRo0eTk5NDZGQk8fHxREdHU61atQJ9oqOj2bBhAx4eHqUaz9HRkTlz5hToazAYCA0NpW7durRs2bJc5iv3OKviVHYXdMNoxMamJAUqIiIi9y8l3qW0YsUKzp49y9atW3FxcQGgVatWBAQEsGnTJvz8/Extc3JyWLx4MQsXLryr8WxtbfHx8SnQd8aMGdy4cYMPPvgAe3v7Mp6p3A8aOjtz/PhxHi5BnxvA1apVC/0hU0RERApSjXcpxcTE4OHhYUqSATp16kSzZs2IiYkxXTMYDPj6+rJgwQJ8fHxo0KDBXY33Z/Hx8axZswZfX1/at29fBjP7r8TERF599VU8PDzw8PBg0qRJpKSkFGh34MABhg4dSps2bejSpQszZswgIyOjQLtDhw7h7+9Pq1at6NKlC++++26+8ZKSkmjRokWhv4YNG0aLFi2IjY0t0znKTaPffpslDzxQoj5brK3xHjKknCISERG592jFuxB79uxhzJgxPPHEE6xatcp0PSQkhA0bNvDhhx+SmJhI7969C/R1c3Nj9+7dpq8NBgNXr14lIiICb29vevToUaDPlStXij3en0VERFCjRg3eeOONkk7ztopbSnPgwAFGjRqFm5sbQUFBnDlzhlWrVnH48GGioqKwtr75s11sbCyBgYHY2dkxduxYbGxsWLlyJd999x3r1q3D3t7+tqU07733HvXq1VMpTTnp0bMn7zzwAKm32cP7VrnAonr1WP3mm+UdmoiIyD1DiXchunbtiq+vLxs3bmTv3r106dKFffv2sWHDBoYMGYKrqytAoavXTk5OpKenk56eTp06dahduzY7duy47fHa586dK/Z4tzp27Bi7du1i1KhR1K9f/26mXEBxS2nmzp1Lw4YNWbNmDTVq1ACgYcOGvPfee+zdu5du3boB8P7772NjY8O6deto0qQJAJ6envj4+LBkyRKCg4NVSlOBrK2tmblsGc8NGcLnFy9S+zZtjcCb9vY8/fLLNGrUyFwhioiIVHoqNSnClClTeOCBBwgNDSU1NZWQkBCaNm1KcHCwqYyisJ0c8o7QzvzP1mzW1ta3TbqBEo13q7Vr12JjY8Pzzz9fgpkVT3FKXwwGAw4ODvj5+ZmSbsD08mh8fDxws4Tk119/xcfHx5R0A7i4uPDUU0+xcePGIuPIK6Xp1q1bmZfSSH7de/Zk/Mcf412/Pju5mWD/2VHgWTs7nMaO5a2//c3MEYqIiFRuSryLYG9vz6hRozh16hSDBw/m3LlzzJ49G1tbW4zGwlKS/KxKsEtEaca7fv06W7ZsoUePHjz00EPFvhfcLKVp0aIFr7/+er7rISEhtGjRgm3btpGYmIibm1uBvm5ubhw5cgS4+UNBZGQkL730Ur42v/zyC4BpNTRvRT/vXwpu1aRJE1JTUzlz5kyhseaV0gwePLhEc5TS6efry/L9+9n6wgs82bAhIfb2zKtRgxl16tC7QQPe79mTfhERTAsLK9H/x0VERESlJrfVvn17vLy82LFjB0OHDqVt27YA2NraAjdXfP8s71rt2rf7x/r8SjNebGwsmZmZ9OnTp9j3yVOWpTS3Sk5OJjY2lrCwMFxdXenVq1e++RX2wuXly5cBuHDhAg0bNsz32a2lNA4Oxak8lrLg4uJCxMqVZGRkEBsby5UrV6hVqxajW7WiQYMGxMXFVXSIIiIilZIS79u4fv06R48eBWDfvn1kZmZia2trWsm9cOFCgT7nz5/Hzs7OlGwWR2nG2717N9WqVaN79+7Fvs+tpkyZwr59+wgNDWX9+vX5SmnySkTuVPpya+J9+fJl04ujNWvWZNq0aaa2Li4u1K5dm+3bt/Piiy+aVkoNBgP79u0DICsrq8C9bi2lOXv2bKnmKaVXq1atQl8GFhERkdJRqcltrF+/nuTkZCZNmkRSUhLh4eEA2NnZ4ezsbCq5uNXRo0dxd3cv0X1KM94PP/yAu7t7iVbWb2Vvb8/06dPLrJTGysqKiIgIwsLCcHFxISAggO3btwNQrVo1AgICOHLkCEFBQcTHx/PLL78wfvx4rl27BlDgEJa7KaURERERsURKvIsQFxfHjh078PPzIzAwkGeffZaoqCgOHToEgJeXFwcOHCAhIcHUZ//+/Zw8eRJvb+8S368k42VnZ3P8+HEee+yxUs7uJk9PT7y8vEhMTMTPz++uSmns7e3x9vZm4MCBREVF0ahRI2bNmmX6/JVXXmHEiBF88cUXDBgwgIEDB2JlZcXo0aNN/W91N6U0IiIiIpZIiXchDAYDU6dOxc7OjqCgIACCgoKws7Nj6tSpGAwGxowZg729PSNHjmT58uUsWbKE8ePH4+bmVuiWeHdSkvHOnDlDdnZ2gZroksrMzCxQSgOlK325VY0aNejevTtnzpwxHZBjbW3N22+/zd69e4mKimLnzp0sWbKEjIwMbGxsCqxq320pjYiIiIilUeJdiAULFnDy5En8/f2xs7MDwMHBgYkTJ/L7778zb948HB0dWbNmDS1btmT+/PmsXLkST09Pli1bVqojtEsyXt4LiaUtM8kTHh5+V6U0CQkJ9OjRg6ioqALtMjIysLKyMsW+bds2YmNjqVevHu3btzcl2gcPHsTNzc1UD57nbktpRERERCyNXq4sRFBQEEFBQQV2bxg0aBCDBg0yfd28eXOWLl1aorF37txZ5GfFHe/xxx83vQBZWnFxcURFRZlKaU6cOEFUVBR9+vQx7eayatUqEhISTHt555W+BAYGAtC0aVPS09NZt24dgwcPNiXZycnJbN++nQ4dOpgS5xUrVnD9+nU2bdpk2tf822+/JS4ujrCwsHyx5ZXSPPfcc3c1RxERERFLohXv+1BeKY2jo+NdldJUqVKFadOm8euvvzJ8+HCioqJYuHAhgwYNwtrampCQENM9x4wZw2+//cbYsWNZv3494eHhvPbaa3Tu3Jn+/fvni6+sSmlERERELIkS7/tQXilNcHDwXZfS+Pj4EBERQXZ2NrNmzWLVqlV06NCB6OjofAfm9O7dm/DwcC5evMisWbOIiYkhMDCQhQsXFtjRpKxKaUREREQsiUpN7kN5pTR/VtpSGm9v72Lt5NK3b1/69u17x3ZlUUojIiIiYmm04i0iIiIilUZiYiKvvvoqHh4eeHh4MGnSJNMuarc6cOAAQ4cOpU2bNnTp0oUZM2YUeor2oEGDaNGiRYFf48ePLzKGY8eO4e7uzoIFC0oUu1a8RURERKRSSE1NZcSIEWRlZTF69GhycnKIjIwkPj6e6OhoUznsgQMHGDVqFG5ubgQFBXHmzBlWrVrF4cOHiYqKwtr65tqz0WgkISHBdLbJrYo6wO/GjRtMmTKF7OzsEsevxFtEREREKoUVK1Zw9uxZtm7datp1rVWrVgQEBLBp0yb8/PwAmDt3Lg0bNmTNmjXUqFEDgIYNG/Lee++xd+9eunXrBkBSUhKZmZn07Nmz2Oew/OMf/+C3334rVfwqNSkH169fZ83y5bzg5YWPhwfDevRgfliY6aVBERERESm5mJgYPDw8TEk3QKdOnWjWrBkxMTHAzd3bHBwc8PPzMyXdAB4eHgD53iM7fvw4QL7xbic+Pp7FixfzyiuvlCr+SpN4b9u2jb59+/L444/z9NNPs2nTpooOqYCcnBxCg4J4qnlzzr76KtP/9S9WHjzInF27qPPOOwx89FFeGTKk0PoiERERkfvZnj17aNGiBa+//nq+6yEhIbRo0YJt27aRmJiIm5tbgb5ubm6mg/+qV69OZGQkL730Ur42v/zyC/DfE7oB08p1XuKdd4p3YfJKTJ588kkGDBhQihlWksT7iy++ICgoiM6dO7No0SI8PDwIDg7mq6++qujQTHJycvDv04daS5bwv2fOEJSZSXPgAcAZCMjK4tuzZ+n12Wf0e+IJrly5UsERi4iIiFiOrl274uvry1dffcXevXsB2LdvHxs2bGDIkCGmbYobNGhQoK+TkxPp6emkp6cX+Cw5OZnPP/+cGTNm4OrqSq9evUyf/fbbb9SqVYtZs2bRpk0b2rRpg6enp2n1/FZLly7l1KlTvPvuu6WeY6Wo8Y6IiODpp59mypQpAHTp0oUrV67w97//nT59+lRwdDdNeeklOu/fz6u3+UkJwPfGDWofO8YL3t5s/t//NVN0ZSc3N5cvv9zOjBlLOXPmErm5ULWqER+fnrz11ph8P0WKiIiIlMSUKVPYt28foaGhrF+/npCQEJo2bUpwcLCpRKRmzZoF+lWvXh24uWJdp04d0/XLly/To0cPU79p06aZ2sLNUpOMjAzS09OZM2cOaWlprFq1igkTJpCdnc3AgQOBmwn6okWLeOedd3jwwQdJSkoq1fwsfsU7MTGR06dPF3jTtHfv3pw4cYLExMQKiuy/Ll26xKFt2+6YdOfplZODU3w8Bw8eLOfIytaBA9/RvPmT+Pvv5cCBcH7/fTenT+8mIWEn8+Y9Ttu2gQwdOq5Ub/mKiIiI2NvbM336dE6dOsXgwYM5d+4cs2fPxtbWFqPReMf+VlZWBb6OiIggLCwMFxcXAgIC2L59u+lzPz8/3nnnHebPn0+vXr149tlnWb9+PY0bN2bu3Lnk5OSQk5PD5MmTadeunenlzdKy+MT7xIkTADRr1izf9aZNmwJw8uRJs8f0ZysWLSLg0qUS9Xnl0iUWv/deOUVU9nbt2oOPz2ROndrMlSszgb/c8mkVcnN9OXfuSzZtaoen53PcuHGjgiIVERGRyixva7/ExET8/Pxo27YtALa2tsDNlyf/LO/an0+9tre3x9vbm4EDBxIVFUWjRo2YNWuW6fOhQ4fi7++fr0+NGjXw8fHh4sWLHD9+3LRd4VtvvUVKSgopKSmkpaUBcO3aNVJSUsjNzS3W3Cw+8c6r1fnzb2StWrUAuHr1qtlj+rOYtWsZXMJV3rbAsbi48gmojF26dAl//0lcuPA5UP+2ba9fH8XBgz15663K80OFiIiIWI7MzEyOHj0K3KzxznvhMa+c9cKFCwX6nD9/Hjs7O1NyXpgaNWrQvXt3zpw5U+iBO7dydHQ0xbJ3716ys7MZPHgwHTt2pGPHjvj6+gIQGRlJx44d+eOPP4o1N4uv8b7TPyvkbYBeHIcPHy7x/eOKkRxnpqVR446tCqqSlUVsbCxVqlj2Y/j44yjOn38FcCxW+2vXXuaTTzzw8+udbxufu1WcZyHmoWdhOfQsLIeeheXQs7AcpXkWK1euJDk5mWHDhrF27VomT57MiBEjgJsvUR44cICuXbvm6/PDDz/QpEkT4uLiSE5OJiwsjP79++d7kRLg9OnTWFlZcfToUa5du8asWbPo2LEjzzzzTL52sbGxAKSkpDBgwAB69uyZ7/MrV67w0Ucf0blzZ7p06UJiYiLnzp278+SMFm7Xrl1GV1dX47Fjx/JdP3z4sNHV1dW4b9++O45x/fp146FDh4zXr18v0b0PHTpUrHbdXFyMRijxr24NGxpzc3NLFJO55eTkGJs0+asRrpVoejVrfmj8xz+Wl1kcxX0Wkt/p06eN48aNM3bo0MHYoUMH48SJE42XLl0q0G7//v3GIUOGGFu3bm3s3Lmz8f333zdevXq1QLuDBw8aBwwYYHz88ceNnTt3Nk6fPr3Q8Z599lmjq6trgV+vvfZauczTaCz7uf7888/GkSNHGlu1amVs06aNcezYscaEhATT54mJiYXO8dZf3333XbnN12jU94Ul0bOwHHoWlqM0z+LQoUPGli1bGkNCQoxGo9H49ttvG1u2bGk8ePCg0Wg0GmfPnm187LHHjMePHzf1+d///V+jq6urccOGDUaj0WjMzs42tm/f3tivXz+jwWAwtUtKSjK2adPG+Pzzz5uude7c2di1a1djenq66VpycrKxTZs2xuHDhxcZZ97fAfPnz893/U45p2UvtfLf2u7Tp0/TokUL0/VTp07l+7wiNXZx4UhCAgV3lSxaOmCsXbvASwCW5tSpUxgMD0MJ1/SvXRvIZ59N58UXR5ZLXHJnZX2sbmxsLIGBgdSsWZOxY8diY2PDypUr+e6771i3bh329vZA6Y7ftbS5njhxguHDh1OzZk3TIQnLly9n2LBhbN68mQYNGuDo6MicOXMKxGIwGAgNDaVu3bq0bNmyXOYrInIvMhgMTJ06FUdHR4KCggAICgri66+/ZurUqWzZsoUxY8awefNmRo4cyahRozAYDCxbtgw3NzfTyZNVqlRh2rRpTJo0ieHDhzNgwABSU1NNf86HhISY7vm3v/2NcePGMWTIEAYPHkxGRgZRUVFUqVKFv/3tb2U/yWL+AFKhevToYZwwYUK+a6+//rrRy8urWP3Le8V7z549xnGOjiVa7f6oenXj4nnzShRPRfjpp5+MTk7jSrGgf8n4xBMDyywOrWCUXHh4uPHRRx8tdFVg/fr1pmu+vr7Gp556ynjt2jXTtTVr1hhdXV2N3377relav379jI8//rgxJibGdO348eNGNzc34+zZs03XTp8+bXR1dTV+9tln5TW1Asp6ru+8847R1dXVeOTIEdO1n376yejq6ppvroV5//33863OlCd9X1gOPQvLoWdhOUr6LObOnWt0dXU1bt68Od/16OjofH/+JiQkGEePHm1s3bq18a9//asxODi40H/hjImJMfr6+hrd3NyMHTp0ML722mvGEydOFGj3r3/9yzho0CCju7u7sX379sZXXnkl398nhblnV7wBxo0bx5QpU7C3t6d79+588803fPnll0RERFR0aAB07tyZKY6OJKWk4FyM9hnAigce4OtRo8o7tLtWu3ZtrK1L8wJrGnXq1L5zMyk3dzpW18/Pz3SsrpeXV5HH6nbr1o2kpCR+/fVXnnvuuXwHF7i4uPDUU0+xceNGgoODgZIfv1sWynKuAElJSTg4OPDYY4+Z2j3++OM88MAD/Prrr0XGER8fz5o1a/D19aV9+/ZlPU0RkXtaUFCQaaX7VoMGDWLQoEGmr5s3b87SpUvvOJ63tzfe3t53bOfp6Ymnp2eJYnV2ds539HxxWfyuJgDPPPMM7777Lvv27WPcuHEcPHiQsLCwYv1mmoOVlRWLPv2UIfXrc6ey+mvAUEdH3l64MN8G75aqSZMmVK16DCjZri3Vq39B//6dyycoMfuxunkvjOSdGnarJk2akJqaypkzZ4CSHb9bHBVxhHDTpk25cuVKvrfeL1++THp6OvXrF72zT0REBDVq1OCNN94o+URFROSeVykSb4AhQ4awY8cOfv75Z7744gvTSUKWolWrVkRs3cqARo1YVq0af041bgCfW1vTq359RixciM8tP7lZsipVqjBsWD9sbLaUoJcRB4dPGDXK/85NpVTMfaxu3vZMGRkZBfpcvnwZ+O/2TiU5ftcS5wowevRoHnzwQSZMmMCxY8dM+7dWrVqV4cOHFxrnsWPH2LVrF0OGDLltci4iIvevSpN4VwYdPDzYcfQo199/H8+//IWhDRowzsmJEQ0a0Pmhhzj85pts+Pe/eXbo0IoOtUTefHM09eqFA8UrOale/ROefvqJAnuvS9maMmUKTk5OhIaGkpqamu9Y3bwE+U7H6t4q71jdKVOmYDAY8h2r6+LiQu3atdm+fXu+LT4NBgP79u0DICsrCyh4/O7MmTOpVasWEyZMYNOmTRY/V7i5+j127FgOHjyIj48PAwYM4MCBA3z44Yf5yk9utXbtWmxsbHj++edLNUcREbn3VYoa78rE3t6eVydOZFxQEElJSVy5coVatWrRuHFji9+vuygPPvggixdPYcyYQVy6tAGwK7Jt1aqbcHNbxeLFJVkhl9LIO1Z33LhxpmN116xZc9fH6mZlZbF69WoCAgKIiIigd+/eVKtWjYCAABYsWMCiRYuYPHkyubm5zJs3j2vXrgFgY2MD3Dx+Nzc3N99JYH379qVfv37MnTuX/v37m9pa4lwB5s2bx+LFi/Hw8MDPz4+cnBzWrVvHG2+8wfz58+nRo0e+8a5fv86WLVvo0aNHue3cIiIilZ9WvMuJlZUVjRs3xt3dnWbNmlXapDuPr28/Vq2aQMOGntjafgBcuuVTI7AbJ6chdO++gb17N+ZbPZTyY85jdV955RVGjBjBgQMHGDBgAAMHDsTKyorRo0eb+kPxjt+15LmmpaURGRmJu7s7K1asoH///gwcOJDVq1fj4uJCSEiIaXU/T2xsLJmZmfTp06dUcxMRkfuDEm8pNm9vL06c2MPChQ/y2GPP8dBD3WjYsBvOzk/i7/8Zu3f/jR07Prntca1Stsx5rK61tTVvv/02H330EVFRUezcuZMlS5aQkZGBjY3NHVd6bz1+tzTMNdfff/+drKws+vXrl29lvmrVqvTv35+LFy9y4sSJfGPs3r2batWq0b1791LNTUREykZOTg4XL17kzJkzXL9+vaLDKUCJt5RIjRo1CAh4niNHviYpaTd//LGbxMT9rFkzn0cffbSiw7vvhIeHk5yczKRJk0hKSiI8PBwAOzs7nJ2dTTt63Oro0aO4u7sDkJCQQI8ePYiKiirQLiMjAysrK9PhM9u2bSM2NhZ7e3vat29vSrQPHjyIm5sb1atX59y5c/Tt25eFCxcWGO/kyZPAzS2YLHmuefPNyckp0C43Nzff/+b54YcfcHd313sNIiIV5OTJk4wNDqbJk0/yP2PH0vaNN2jWpw/d/PzYtWtXscoSzUGJt0glFRcXR1RUFH5+fgQGBvLss88SFRXFoUOHAPDy8uLAgQMkJCSY+uzfv5+TJ0+atuJs2rQp6enprFu3Ll/5RHJyMtu3b6dDhw6mZHLFihWEhobmS0i//fZb4uLiTKUlDRo0IC0tjejoaK5e/e/LuH/88Qeff/45TzzxBE5OThY910ceeYT69euzcePGfOUrBoOBTZs24eDgwCOPPGK6np2dzfHjx4t86VJERMqP0WhkWlgYT7z0Ekt79OCP/fs5+9lnnF2/nrPffsue0FCe3byZv/brx5UrVyo6XGymT58+vaKDKG85OTmcP3+e+vXrl6jW+syZM/n29pWKo2eRn8FgYMyYMdjY2PDRRx9RvXp12rZtS3R0NPv378fPz4//+Z//4fPPP2fLli1YWVkRGxvLjBkzeOSRR5g2bRo2NjZYW1vToEED1q1bx4EDB8jKymLPnj2EhIRgNBqZP38+devWBW7WRUdFRZGQkIDRaGTHjh3MnDmTjh07EhQUZDpuvUmTJmzYsIFdu3aRnZ3N/v37TcfzLliwwFRyYqlztba2pnHjxnzyySd8/fXXZGdn8+9//5vp06eTkJDAe++9ly/JTk5OZsWKFfTu3dtUd24u+r6wHHoWlkPPwnKY41lMev99Fl+8SOqqVfDII/Cnl+mpV4/rffpwtn59tr76KiMHD6Zq1arlFs+dck4l3rehb17LoWeR37x58/jmm2949913efzxx4Gb2+k5ODjw+eefYzAY6NWrF0899RTHjh1j48aNHDlyhB49ehAWFpavJKJly5a4uLhw6NAhNm/ezNGjR3niiScIDw/n4YcfNrV7+OGHad68Obt372b79u0kJSUxZMgQ3nnnHVN5Btw8UezRRx/lp59+YsuWLRw+fJh27drx4Ycfluo0y4qYq4uLC+3atePnn39m8+bNxMbG4uzszLvvvouXl1e++E6dOkV0dDS9e/c2lbWYi74vLIeeheXQs7Ac5f0svv/+e4I+/5zLy5aB9e2LOIwuLqRWrcq59evpf8u5DWXtTjmnldFSil7KkcFg4PDhw7i7u5dot424uDjatWtXjpFJcelZWA49C8uhZ2E59Cwsh56F5SjvZ+E9YgRfvvkmtG5dvA65uTR88kkSdu4s9OyHsnCnnFM13iIiIiJSqaSkpPDvpKTiJ90A1tZcHjyYtZ9+Wn6B3SmECruziIiIiEgp/PLLLxieeKLE/a5168a3//53OURUPEq8RURERKRSuXbtGjmlKRepWZOrpTxPoiwo8RYRERGRSsXBwYFqhRycdkcXL1LfwaHsAyomJd4iIiIiUqm0atWKGt9/D9nZJepXd906Xujfv5yiujMl3iL3sZycHC5dukRKSkqhJzXeS3JyckhJSSE1NbXAyZMiIlK5VKlShaF9+2KzeXPxO6Wl4fDzz3Ts2LH8ArsDJd4i96Eff/yRF595hi7OzoxxdyfQzY0nnZ15zd+fo0ePVnR4Zer//u//eNnPj87OzgS6uRHw2GN0euihe3KuIiL3k7defJH6H34IxSk5MRp54M03eee117D68yE7ZlT802REpNLLysri5eeew7BvH+MvXqQDkPfHjxHY98knvLNjB859+/JhZCQ2NjYVGO3dyc7O5lV/f9J27WL8xYt8ROFzbfT000QsX16p5yoicj9q0KABn0ZE4Ovry/lPPoEmTQpvmJ3NA6+9RuBf/sJwPz/zBgAegokAACAASURBVPknWvEWuU/k5ubyQr9+dPrqK9ZcvIgH/01E+c9/dwE+vXiRJp9+ystDhlBZz9fKzc1l5IABtNm6lbUXL9KRoufa/LPPeHHw4Eo7VxGR+1mnv/6Vf330EY+NHEm9kSMhNhaysiAnB06fps577/FQ586EenjwQUhIRYerxFvkfrFs4UIe27+fwOvX79h2QkYGtXbs4LP1680QWdlb+fHHNN+3j5eKMdc3MjNx+OYbNnzyiRkiExGRsvb4449zZOdO/vXGG/itWoV737609PSkS1AQke7unNy3j1dHjaroMAGVmojcF4xGI6v+/ne+zMgodp/JaWkMnzmTQUOGlGNkZc9oNLL8ww/ZevVqsftMTktjyKxZPOfvX46RiYhIeWrdujXrFy2q6DBuSyveIveB/fv30/ryZeqUoE8DoN7Zsxw5cqS8wioX33//PY+lpmJfgj71gEbnz/N///d/5RWWiIiIEm+R+8EP+/fTPSWlxP26p6Tw7wo8Wrc0fvjuO7pfulTift0vXuTfP/xw1/dPTEzk1VdfxcPDAw8PDyZNmkRKIb/3KSkpTJs2jU6dOtG2bVuGDx/Ojz/+aPo8KSmJFi1a3PbXrbuynD59mnHjxtG+fXtatWrF8OHD+emnn+56PiIiUnZUaiJyH8hMS6MUB+tim5NDZnp6mcdTnjLT0mhUin62RiOpV67c1b1TU1MZMWIEWVlZjB49mpycHCIjI4mPjyc6Oppq1aoBcPXqVfz9/Tl//jwjR47Ezs6OqKgoRowYQXR0NK6urjg6OjJnzpwC9zAYDISGhlK3bl2a/OcN/pSUFIYNG0ZGRgYBAQHY2dmxevVqXnjhBdavX0/Lli3val4iIlI2lHiL3AccHnyQi1ZWUMKdOy5Wr85D9eqVU1Tl44EGDUo316pVqVu//l3de8WKFZw9e5atW7fi4uIC3DxdLSAggE2bNuH3n22sli5dysmTJ1m9ejUdOnQAwNvbG09PT5YtW8acOXOwtbXFx8enwD1mzJjBjRs3+OCDD0x70X766adcuHCBhQsX0qtXLwC8vLzo06cPH330EfPnz7+reYmISNlQqYnIfaBP//58VoqkcouDAz169CiHiMpPb29vPi/FXDc5OuLp6XlX946JicHDw8OUdAN06tSJZs2aERMTA9x8+XPj/7N393E93/sfxx9p6IJvEzGEYStbjlytbYahNIcRk5aZy6KNuZhFSGOaWbOVycbZmIvpxzibq9rBGeZizKbNrhwhccqIJkrxTfn+/uj0PVqhqG85nvfbzR99Pu/3+/P6fD+rvXr3fr8+69bRpUsXc9IN4OTkxOTJk2nfvv0Nx09ISGDlypX069evULuUlBQAnnrqKfOxBg0a8NBDD3HkyJE7uicRESk7SrxF7gGNGzfmvubNOV6KPj8ATdq1o3bt2uUVVrlwdnbG9uGHOVqKPj8D9Vu3xsnJ6YZtdu3ahaurK+PHjy90PCwsDFdXV2JjY0lOTsbNza1IXzc3N/Mm1ZSUFFJTU+nQoQOQn4hn/afazKBBg8yz4sWJiorCxsaGCRMmFDrepEkTAJKSkszHcnJySE1Nvek9iYiIZSnxFrlHvBYRwfjatcktQdsrwGQnJ159663yDqtcvBYRwYTatblagrZGILhOHSbOmXPTdp07d6Zfv35s3ryZ3bt3A7Bnzx7WrFmDv78/Li4uQP6b1P7MycmJzMxMMjMzOXnyJAC1a9cmIiKC9u3b07ZtW7p378727dtveP3Dhw+zY8cO/P39qfunGX0/Pz9atGhBaGgoBw8eJCkpidDQUM6fP09gYGAJPgUREbEEJd4i94inOnak74wZPO/oSPZN2l0E+teuzUtRUbRq1cpS4ZWpJzt0YMCsWfg5OnKzyuUZ5N/riLlzadOmzS3HnTp1Kk5OToSHh5Oenk5YWBhNmjQhJCTEPGtta1t0G2v16tUByM7OJiMjA4D333+fnTt3EhoaSkREBDY2NowZM4a9e/cWe+1Vq1ZhbW3Niy++WORczZo1GTt2LCdPnuT555+nR48ebNy4kdDQUJ5++ulb3peIiFiGNleK3EMCxo6ldr16dAsOpktGBiMvXqQpYAKOAB/VqsW3BgOzFi6k+1//WsHR3plho0fjWLcunhMnmu+1Gfn3egz46P77+cbBgRnR0fTo3btEYzo4ODBz5kzGjBnDgAEDSE1NZeXKldjZ2ZXolfNWVlbk5OQAkJGRwZYtW3BwyK843q1bN7p37857771nXoZS4MqVK2zcuJFu3brRsGHDIuOuXbuW6dOn4+LiwvDhw7GxsWHDhg28+eabVKlShRdeeKFE9yciIuVLibfIPaavnx99fH3Zunkz0999lzOnTwPg3LgxI6ZMIbJLF3O1jLtdH19fnn3uOfO9nv79d6ysrGjg7MzwyZN5x9OTKlVK94c/Ly8vvL292bp1KwMHDqRt27YA2NnZAfnl/v6s4FiNGjXM7by9vc1JN4DBYKBbt26sW7eOrKws7O3tzef2799PdnY2PXr0KDam999/n/r167N69Wpzv549ezJq1CjefvttvL29qXOXVacREflfpMRb5B5UpUoVevTsSY+ePSs6lHJX1veanZ1tfnHNnj17yM7Oxs7OjgYN8quHnzt3rkifs2fPYjAYsLOzM68Bd3R0LNLO0dERk8lEdnZ2ocR7586dVKtWjS5duhTpc/78ec6dO8fgwYML9QHo168fO3fu5ODBg3dcsUVERO6c1niLiJRCZGQkp06dYvLkyaSkpBAZGQnkz1g7Ozubq5dc79ChQ7Rs2RKAhx9+mGrVqnHs2LEi7VJSUqhevXqRpPyHH36gZcuW1KhRo0ifqlWrApCXl1fk3LVr1wBKtAxGRETKnxJvEZESio+PJyYmBj8/PwICAujfvz8xMTEcOHAAyF8+sm/fPhITE8199u7dS1JSEj3/M+NuZ2dHt27d+Prrrzl69L9FD5OTk9m+fTuenp5YW1ubj1+9epVjx47x6KOPFhtTzZo1cXd3Z8uWLYVeTX/t2jXWrFlD1apVS7RxVEREyp+WmoiIlIDRaCQ0NBRHR0eCg4MBCA4O5quvviI0NJSNGzcycuRINmzYwLBhwxgxYgRGo5HFixfj5uZW6C2UkyZN4rvvvmPIkCEMGTKEqlWrsmLFCmxsbJg4cWKh654+fZqrV69Sv379G8YWGhrKkCFD8PX1xd/fH1tbW+Li4vjxxx+ZOHGi1neLiFQSmvEWESmB6OhokpKSCAkJwWAwAFCrVi0mTZrEiRMnmDdvHo6OjqxcuZIWLVowf/58li9fbn4NfLVq1cxjOTs7s2bNGh577DGWLFnCwoULeeSRR1i9ejWNGjUqdN0LFy4AFLvMpIC7uzurVq3ioYceYtGiRbzzzjvk5OQwd+5cgoKCyuHTEBGR26EZbxGREggODjbPdF/P19cXX19f89fNmjXj448/vuV4jRo1Yv78+bds16pVKxISEm7Z7tFHH+Wjjz66ZTsREak4mvEWEREREbEAJd4iIiIiIhagxFtERERExAKUeIuIiIiIWIASbxERERERC1BVExERC8rJyeGPP/7AZDJRu3ZtqlevXtEhiYiIhSjxFhGxgIMHDzLrww/Z99tvUFCrOyWFdg8/zMwxY2jfvn3FBigiIuVOibeISDnKzc3lhTFj2H7pEn+MHw+PPQZWVubzcT/8wLfR0Tx57Rqff/xxoRftiIjI/xat8RYRKScmk4m+I0YQ27o1f8TEgIdHoaQbgLZt+WPpUv7ZrRs9Bg0iLy+vYoIVEZFyp8RbRKScLP+//2NXw4ZcfvnlW7Y1Dh3Kfnd3PliyxAKRiYhIRVDiLSJSTuZ+8gmZxbxm/kayx49n/ooVmEymcoxKREQqihJvEZFy8NNPP3G2cWOoXbvknWrWJN3dnX379pVfYCIiUmGUeIuIlIMfDx7kj06dSt3vfOfOfHfwYDlEJCIiFU2Jt4hIObh0+TImW9vSd7S1JSM7u+wDEhGRCqdyguUoOTmZiIgIvvvuOwC6dOnClClTcHR0LNTuwIEDREVF8euvv2IwGPDy8mLs2LG33c7X15dffvmlSDzPPPMM8+fPL+O7FJHiONWqRbXUVHJK2a/KuXPU/9P3tIiI/G9Q4l1O0tPTGTp0KDk5OQQGBpKXl8eSJUtISEhg7dq15lq9+/fvJyAgAIPBQFBQENbW1ixfvpxvv/2W1atX4+DgUKp2JpOJxMREvLy88Pb2LhRTw4YNLfshiNzDvLy8qDVgAKnjxpWqn9Pf/04vVTYREfmfpMS7nCxbtowzZ86wadMmmjdvDoC7uzvDhw9n/fr1+Pn5AfDmm29ibW3N6tWrady4MZD/P2wfHx8WLVpESEhIqdqlpKSQnZ2Np6cnPj4+lr5tEfmP2rVr065xY7788Udo06ZknQ4f5hGDgQYNGpRvcCIiUiG0xrucxMXF4eHhYU66ATp06EDTpk2Ji4sD8pPkI0eO4OPjY06mAZo3b07Xrl1Zt25dqdoBHDt2zHxORCrWnIkTcQoOhsuXb904J4faEyYQ8dpr5R+YiIhUCCXet2HXrl24uroyfvz4QsfDwsJwdXUlNjaW5ORk3NzcivR1c3Pjt99+AyA1NRUAFxeXIu0aN25Meno6p0+fLnE7gKNHjwL/TbyztUlLpMK0atWK98eMoU6/fpCefuOGmZnU9vUlYsgQPDw8LBegiIhYlBLv29C5c2f69evH5s2b2b17NwB79uxhzZo1+Pv7mxPkevXqFenr5OREZmYmmZmZ2NnZAZCVlVWk3YULFwA4d+5cidtBfuJtb2/PnDlzaNOmDW3atMHLy8s8yy4iljXwuedYFRJC8969cXzlFfj5ZzAaIScHDh2i1quv0rRHD5aOHk3ACy9UdLgiIlKOlHjfpqlTp+Lk5ER4eDjp6emEhYXRpEkTQkJCzAmybTGlxKpXrw7kz0Q3b96cGjVqsGXLlkJvqjMajezZsweAnJycEreD/KUmWVlZZGZm8s477/DWW29hb2/PxIkTWb9+ffl8GPeA5ORkXnnlFTw8PPDw8GDy5MmcP3++SLtff/2V4cOH07p1a9q2bctLL73E8ePHzedTUlJwdXW96b/9+/cXG8PZs2fx8PBgypQp5XafUj68unbl6O7dbPD359n336dl79607NWLnnPn8oWPD4l79tC7R4+KDlNERMqZNlfeJgcHB2bOnMmYMWMYMGAAqamprFy5Ejs7uxK97tnKyopq1aoxfPhwoqOjCQ4OZtSoUVy7do158+Zx+T9rQq2trUvcDsDPz49r164xaNAg87V69erFs88+y9y5c+ndu7e5rZRMSSvUHD9+nMGDB2Nra8vo0aMBWLp0KS+88AIbNmygXr16ODo68s477xS5htFoJDw8nNq1a9OiRYti43j99de5ePFi+d2olCsrKys6duxIx44dKzoUERGpIEq870BByb6tW7cycOBA2rZtC2BeGmI0Gov0KThWo0YNAEaPHk1GRgaffvopsbGxAHTt2pXAwEDee+89c5nAkrYbOHBgkWva2Njg4+PDggULOHbsGK6urmX5MfzPK2mFmuXLl5OdnU1MTAyPPvooAE888QQDBgxg2bJlhISEYGdnV2y1mdmzZ5Obm8u7775rfpbXW79+vfmvGyIiInJ3uquWmkRERDBs2LCKDsMsOzubQ4cOAflrvAs2MhaUAitYd329s2fPYjAYzMl5lSpVmDZtGrt37yYmJobt27ezaNEisrKysLa2NtfeLmm7Gyl4yY42W5ZeSSrUQP4yklq1apmTbsjfXHf//fdz5MiRG46fkJDAypUr6devH+3bty9y/ty5c7z11lvmWXQRERG5O901ifenn37KJ598UtFhFBIZGcmpU6eYPHkyKSkpREZGAmAwGHB2djZXL7neoUOHaNmypfnr2NhY9u/fT506dWjfvr05gf7+++9xc3MzrwkvSbvU1FR69erFggULilw3KSkJAGdn57L9EO5yZVWhBqBJkyZcvHix0NrvCxcukJmZSd26dW8YQ1RUFDY2NkyYMKHY8zNmzKB+/fqMHDmytLcnIiIilUilT7xTU1N57bXXeOutt6hZs2ZFh2MWHx9PTEwMfn5+BAQE0L9/f2JiYjhw4AAA3t7e7Nu3j8TERHOfvXv3kpSURM+ePc3Hli1bRnh4OLm5ueZjX3/9NfHx8YXWaZekXb169cjIyGDt2rVcunTJ3O7333/niy++4PHHH8fJyansP4y7WFlVqAEIDAzkgQceYOLEiRw+fJiEhARee+01qlatyuDBg4u9/uHDh9mxYwf+/v7FJucbN25k586dzJkzh6pVq5bVbYuIiEgFqPSJd1RUFIcOHWLp0qU88sgjFR0OkL9OOzQ0FEdHR4KDgwEIDg7GYDAQGhqK0Whk5MiRODg4MGzYMJYuXcqiRYsYN24cbm5uhdb4jhw5kqNHjxIUFMRnn31GZGQkY8eOpWPHjvTu3bvU7WbMmMGZM2fw9/dn+fLlfPjhhwwYMID77ruPGTNmWO5DuouURYUayF9iFBQUxPfff4+Pjw99+vRh3759vPfee4WWn1xv1apVWFtb8+KLLxY5l5aWxuzZswkMDLxhfxEREbl7VPrEOzAwkLi4OJ544omKDsUsOjqapKQkQkJCMBgMANSqVYtJkyZx4sQJ5s2bh6OjIytXrqRFixbMnz+f5cuX4+XlxeLFi81VMACeeeYZIiMjSUtLY86cOcTFxREQEMCCBQsKVR8paTsvLy8++OADbG1teffdd1m6dCmtW7dm1apVepvlDRRUqDl58qS5Qs3bb79dqgo1APPmzWPGjBm0bduWd999l4iICFq1asWECRPYvn17kX5Xrlxh48aNdOvWrdg1+jNnzqR27dqMGTPmzm9SREREKlyFVTXJzc1l7dq1Nzxft25dPD09eeihhywYVckEBwebZ7qv5+vri6+vr/nrZs2a8fHHH99yvF69etGrV68ya+fl5YWXl9ct28l/3WmFmoyMDJYsWULLli1ZtmyZ+ZehXr164evrS1hYGB07diz0S9f+/fvJzs6mRzH1m+Pi4vjnP//JwoULCy0bgvya7efPn6dmzZpafiIiInIXqbDE22g0MnPmzBue9/DwwNPTs0yv+euvv5a6T3x8fJnGILevPJ/FlStX+PHHHwHYtm0b3bt3x8bGxrzU5Oeffy5y/cOHD2NnZ8e//vUvEhMTycnJwd3dnYMHDxZq16ZNG1atWkVsbCxNmjQxH1+7di1Vq1bFwcGhyNgFLzt6+eWXi8QaFxdHXFwc06dPr7AlKPq+qDz0LCoPPYvKQ8+i8tCzKKzCEm97e3sSEhIses2WLVua1+WWRHx8PO3atSvHiKSkyvtZvPnmm6SlpTF58mTmzp3Ljh07mD59OpBfCeb8+fNFrn/mzBlat25Nu3btsLe3B/LXef+5XcEPnRYtWhRKlMPDw/nLX/5Cp06disQTEhLC8OHDixwfPnw4HTt2JCAgADc3t2Jrfpc3fV9UHnoWlYeeReWhZ1F53IvPwmg03nSiVy/QkXvenyvUHD9+nJiYGHr06EH79u3x9vZmxYoVJCYmmtfJF1SoCQgIAODhhx+mbt26rFu3jsGDB5t/wTMajaxfv55atWrx8MMPm6959epVjh07xvPPP19sTA899NANl1k5OTnRoUOHsvwIRERExAIq/eZKkfJUVhVqrK2tef311zl+/Di+vr4sX76cpUuX0r9/f44fP860adMKrcc+ffo0V69epX79+hVy3yIiImJ5SrzlnlaWFWq6d+/OJ598wv33309UVBTvv/8+BoOBjz76iD59+hS67oULF4D8jZkiIiJyb9BSkwp04cIF/vjjD6pVq0a9evUKJXFiGWVdoebJJ5/kySefvGW7Vq1a3dYeB0vvixAREZGyc1cl3p9++mlFh3DHcnNzid24kY/efBPT779TH8ixsiLRyooOPXrw0tSphdYCi4iIiMj/hrsq8b7bpaam4u/pSed//5uPMzO5/pUpJmD70qWM27SJLkFBTA4PN7+YRURERETuflrjbSHp6en079iRiN9+440/Jd0AVoAnEJeWxunoaN4ODa2AKEVERESkvCjxtpCQwEDCjh/H4xbtqgCRGRl8+9FH/PTTT5YITUREREQsQIm3BVy4cIGEvXt55tq1ErWvAkz94w8WhoeXb2AiIiIiYjFa420Bn370EcP++KNUfR4HJn/zDZmZmdSsWbN8ApMykZGRwY4dO0hPT8fW1hZ3d3datGhR0WGJiIhIJaMZbwv4afduOl69Wqo+VkDrvDyOHz9ePkHJHUtISOBlPz96ubpy+IUXuDZ8OGmDBzOzc2eecXfnizVrMJlMFR2miIiIVBKa8baAK5cvY3Mb/WyvXePy5ctlHo/cuS83bCBi1CjePnuWJ8j/RQmAq1cZc+4cZ86dI2LkSP6xZg2LPvsMa2vrCoxWREREKgPNeFuAY926pN5Gv1RraxwdHcs8Hrkz3+zezXuBgcSdPcuTXJd0X+cBICojg0f/8Q9eHT7cwhGKiIhIZaTE2wL6BgTwf7VqlapPFnDU3l4v06lkTCYTUwMCWJ2WRkle9v5qdjapX37JL7/8Uu6xiYiISOWmxNsCunbrxj6DgexS9Pm/atUYOHq0XqJTyezfv59Hz5/HqRR9xv/xBx+qQo2IiMg9T4m3BVhZWTEmNJRX77+fkmy1OwksqVePIaNGlXdoUkqfREQQVMoKNU+Sv8E2JyenfIISERGRu4ISbwt5ceRI6g8bxuj77+dm6dchwL9BA5b84x8YDAZLhScldCIxkUdL2ccKaAz8UcqEXURERP63qKqJBc2MiuKT5s3pHBFB94wMhmRk0BAwAt8Di+rW5eqDDxKzahXNmjWr4GilOCaTqdjNlLdiBVwr4QuURERE5H+TEm8LG/HKKwx9+WW+3LSJ8IULOXf2LNWrVaNFmzbMnTSJ5s2bV3SIchMNnJ05/uuvlPb1OKeA2rVrl0dIIiIicpdQ4l0BrK2t6d23L7379q3oUKSUBk+cyOJvv+XdCxdK3OcnoIm7OzY2t1PNXURERP5XaI23SCl08/TkGwcHLpWizweOjoyeMaPcYhIREZG7gxJvkVKoUqUK0957jyGOjlwtQfvPqlUjo21bnnjiiXKPTURERCo3Jd4ipdS7f396z5rFc7Vrc+IGbS4D79na8n+PPcayTZtUj11ERES0xlvkdgwfM4aH//IXJoSEYDp+HL9z53AymbgM7L7/fr5xcMA/KIjPJ03ivvv0bSYiIiJKvEVuW8fOnem4bx8nTpwg7osvSDpzBtuaNenxxBO84+lJlSr6g5KIiIj8lxJvkTv04IMPMmbixIoOQ0RERCo5TcmJiIiIiFiAEm8REREREQtQ4i0iIiIiYgFKvEVERERELECJt4iIiIiIBSjxFhERERGxACXeIiIiIiIWoMRbRERERMQClHiLiIiIiFiAEm8REREREQtQ4i0iIiIiYgFKvEVERERELECJt4iIiIiIBSjxFhERERGxACXeIiIiIiIWoMRbRERERMQClHiLiIiIiFiAEm8REREREQtQ4i0iIiIiYgFKvEVERERELECJt4iIiIiIBSjxFhERERGxACXeIiIiIiIWoMRbRERERMQClHiLiIiIiFiAEm8REREREQtQ4i0iIiIiYgFKvEVERERELECJt4iIiIiIBSjxFhERERGxACXeIiIiIiIWoMRbRERERMQClHiLiIiIiFjAfRUdgMi9Ijk5mYiICL777jsAunTpwpQpU3B0dCzUbt++fcyfP5/Dhw9To0YNevTowYQJE7C3t7+t8Xx9ffnll1+KxPPMM88wf/78srxFERERuQkl3iIWkJ6eztChQ8nJySEwMJC8vDyWLFlCQkICa9eupVq1akB+0j1ixAjc3NwIDg7m9OnTrFixgl9//ZWYmBiqVKlSqvFMJhOJiYl4eXnh7e1dKKaGDRta9kMQERG5xynxFrGAZcuWcebMGTZt2kTz5s0BcHd3Z/jw4axfvx4/Pz8A5s6dS/369Vm5ciU2NjYA1K9fn1mzZrF7926efvrpUo2XkpJCdnY2np6e+Pj4WPq2RURE5Dpa4y1iAXFxcXh4eJiTZIAOHTrQtGlT4uLiADAajdSqVQs/Pz9z0g3g4eEBQEJCQqnGAzh27BhAoXYiIiJSMZR4i9yhXbt24erqyvjx4wsdDwsLw9XVldjYWJKTk3FzcyvS183Njd9++w2A6tWrs2TJEl566aVCbf71r38B0KBBAwAuXrxYovEAjh49Cvw38c7Ozr7d2xQREZE7pMRb5A517tyZfv36sXnzZnbv3g3Anj17WLNmDf7+/ri4uABQr169In2dnJzIzMwkMzOzyLlTp07xxRdfMHv2bFxcXOjevTsAqampJR7v6NGj2NvbM2fOHNq0aUObNm3w8vIqNCsuIiIillHpE+9z584xffp0unbtSps2bXjuuef4xz/+UdFhiRQydepUnJycCA8PJz09nbCwMJo0aUJISAhZWVkA2NraFulXvXp1oOhM9IULF+jWrRtTp07FaDQyffp0c9vSjHfs2DGysrLIzMzknXfe4a233sLe3p6JEyeyfv36Mrp7ERERKYlKvbmyoGJDZmYm48aNo27dumzZsoUJEyaQl5fHs88+W9Ehyj3sz+X8XF1d+e677xgwYACpqamsXLkSOzs781KRN954g/nz5+Pl5cXYsWMLlf2zsrIqNPaxY8ewtrama9eunDlzhuHDhxMVFcUzzzzDxYsXAZgzZw5z5szBzc2N1157jdatWxcZz8/Pj2vXrjFo0CDzuV69evHss88yd+5cevfujbW1dfl8QCIiIlJIpZ7x3rVrF4cPH2bevHn069ePp556ilmzZtGpUyc+/vjjig5P7mEF5fwOHjxIYGAgw4cP51//+hc15HyrYwAAIABJREFUa9YkOTkZPz8/2rZty/79+5k9ezYATz31FC+++CJbtmxh0KBBXLx4EaPRCECNGjXMY+fm5jJ79mzy8vJo0aIFMTExNGjQgDlz5nDp0iXCw8MBaNeuHePHjyc1NZWhQ4dy5MiRIuMNHDiwUNINYGNjg4+PD2lpaebNlyIiIlL+KvWMt729Pc8//zx/+ctfCh1v1qwZ8fHxFRSVSPHl/FxdXXn55ZeB/DXe2dnZvPnmm1hbW5Obm0uLFi0ICgrCy8sLHx8fFi1axNmzZzEYDNjZ2ZnH/tvf/mbeFAn5iXKXLl349NNPiY6O5tSpUwA8+uijDB06lJ49e+Ll5cXixYvJzc0tMl5xCmbbtdlSRETEcir1jPeTTz7JrFmzCv0Z/urVq+zcuZOHH364AiOTe11x5fz27t0LQOPGjUlJSWHWrFkcOXKEvn374uzsbK420rx5c7p27cq6des4dOgQLVu2BCAxMZFOnTrxwQcfMHr06ELXK1jX/eWXX9KlS5dC4zk5OTF58mTat29faLzU1FR69erFggULisSflJQEgLOzc1l+LCIiInITFZZ45+bmsmrVqhv+27ZtW7H95s6dy4kTJxg1apSFI5Z7xe2UB4yPjycmJobGjRuTnp5O//79zZsXXVxc8Pb2Zt++fSQmJgKY2yUlJdGzZ08g/02SaWlp2NjY8Ne//tU89qlTp9iyZQvu7u6cPXuWDh064O3tzd69e/n1118BGDRoEM7OzoXGq1evHhkZGaxdu5ZLly6Zx/v999/54osvePzxx3FyciqHT1BERESKU2FLTYxGIzNnzrzheQ8PDzw9Pc1fm0wm5s6dy/LlywkICMDLy6vU1yxIUkpDS1oqD0s9C3t7ezp37szmzZv55JNPcHd35+eff2bNmjV4enqSm5sL5G/+jY+PJycnh6lTp1KzZk1cXV355z//SceOHYmLi+Py5cv861//wsvLi7///e8MGjSInj17mmfHGzRoQKNGjYiPj2f9+vXcd999ZGVlERQUBMC2bdtYsWIFJpOJ9u3b89NPP5nreOfl5dG/f39q1KhB69at+eGHH2jatKl5PMhPyKOiovDx8aFr165cuXKFrVu3AuDr63vbn6m+LyoPPYvKQ8+i8tCzqDz0LP7EdBcwGo2mV1991eTi4mKKiIgodf8rV66YDhw4YLpy5Uqp+h04cKDU15LyYelnceHCBdNTTz1l6t69u+n8+fOmLl26mLp3727Kysoy/fDDDyYXFxfTmjVrTCaTyTR37lyTi4uLacOGDabIyEiTi4uL6cyZM6ZVq1aZXFxcTE8++aTp2rVrpsTERFNgYKDJ3d3d1KJFC5OLi4tp27ZtJpPJZDpy5IjJzc3N9Nlnn5ni4uJMvXr1Mrm4uJj+8pe/mMaOHWs6fvy4KS4uzuTi4mLq3r276a9//atp0aJFpt69e5tcXV1NLi4upuHDh5v++OOPIvfyz3/+0+Tr62tq2bKlqX379qbRo0ebjh07dtufjb4vKg89i8pDz6Ly0LOoPO7FZ3GrnLNSb64EuHTpEkFBQfzwww9MmzaNoUOHVnRIcg9wcHBg5syZjBkzpkh5QJPJVKhtcHAwwcHBAERFRQH55fz8/f1JS0sjOjqa4OBgRo0axcSJE5k3bx4//vgjFy9epFatWuTl5TFlyhTatWuHn58fAK1atcLT05ORI0cyduxYAH766ScAMjIy2LJlCw4ODgQFBZGRkUH37t25ePFioRKFBby8vG7rL0QiIiJStir15sq8vDxefvllfvrpJ6KiopR0i0V5eXnh7e1dqDwgYK4YUlC673p/Luc3evRohg4dypdffkmfPn3o27cvVlZWBAYGAvkJ/pIlS0hISOC1117j/PnznD9/noyMDAAuX77M+fPnuXbtmvm63t7eODg4mK9pMBjo1q0bv/32m3kTpoiIiFQ+lXrGe/Xq1Xz33Xc8//zzPPDAAxw8eNB8zsrKCnd39wqMTv7XZWdnc+jQIeC/5QHt7Oxo0KABkP9W1T/7c3nAKlWqMG3aNEaNGsWJEyeoX78+DRs2JCoqCmtraxo2bMju3bu5evUqAwYMKDLekiVLWLJkCdu2bTO/Ir64WW1HR0dMJhPZ2dnY29uX2WcgIiIiZadSJ95btmwB4LPPPuOzzz4rdM7a2tqcFImUh8jISE6dOsXkyZOZO3cukZGRTJ8+HYPBUKic3/WuL+cHEBsbi5OTE48//jh16tQxH//+++9xc3OjevXqhISEmGe4C6SlpTFp0iR8fHzo27cvTk5OODo6Uq1atWJfepOSkkL16tWLTcpFRESkcqjUifeKFSsqOgS5RxWUB/Tz8yMgIIDjx48TExNDjx49aN++Pd7e3qxYsYLExERzLe+9e/eSlJREQECAeZxly5Zx5coVc8USgK+//pr4+HgiIiIACiXqBVJSUgBo1KgRHTp0MB/v1q0b27Zt4+jRo+Za9snJyWzfvh0vLy+9/l1ERKQSq9SJt0hFMBqNhIaG4ujoaN40GRwczFdffUVoaCgbN25k5MiRbNiwgWHDhjFixAiMRiOLFy/Gzc0NHx8f81gjR45k3LhxBAUF4e3tzalTp1i6dCkdO3akd+/epY5t0qRJfPfddwwZMoQhQ4ZQtWpVVqxYgY2NDRMnTiyzz0BERETKXqXeXClSEaKjo0lKSiIkJASDwQBArVq1mDRpEidOnGDevHk4OjqycuVKWrRowfz581m+fLn5te3VqlUzj/XMM88QGRlJWloac+bMIS4ujoCAABYsWHBbs9POzs6sWbOGxx57jCVLlrBw4UIeeeQRVq9eTaNGjcrsMxAREZGypxlvkT+5vjzg9Xx9ffH19TV/3axZMz7++ONbjterVy969ep1w/Mmk4kdO3awZM4cTv3735hMJmrVrs3sN97Ap3//Iu0bNWrE/PnzS3g3IiIiUlloxlukAn29bRtPNWtGrK8vr3/1FTuOHGHn0aMs/PZbDo8cSccHH2TF3/5W0WGKiIhIGdCMt0gF2fT55yx46SXi0tKo9adz9YHply4x+dIlgqZO5dzp07w2c2YFRCkiIiJlRTPeIhUgISGB90aP5otiku7rVQOWpKcTP38+/9i0yVLhiYiISDlQ4i1SAaJnzuTNs2cpyatuqgBR6enMmz69vMMSERGRcqTEW8TCMjMzOfj11zxVij71gNqnT+ulUSIiIncxJd4iZSgvL4+kpKRiz5lMJhITE/nmm2945tIlrEo59oBz5/jy73+/8yBFRESkQijxFikjeXl5DPfx4dl27di5fXuhcyaTifFDh9KzbVu2b9lCnStXSj1+HeBCamoZRSsiIiKWpsRbpAwUJN0eO3awJz2d1/39zcl3QdJdZ/16fsjI4PslS4i3Ku18N2QBdv95oY+IiIjcfZR4i9yh65PuV7KzqQWsP3eO1/39+XrbNnPS/XpmJvZAbGYmv+XmUtoaJTscHGjfpUvZ34CIiIhYhOp4i9yht2fMoO727bxy+bL5WEHy3WvAAP6am0tYZqb5nD3wlcmEO+AGNCvBNa4AuxwcmNO9e9kGLyIiIhajGW+ROzRizBj21q9P/J+O1wK+SU8vlHQXmGNry/1Vq/JgCa+xwNaW54OCqFJF37IiIiJ3K/1fXOQO1a9fn8/37GFss2ZFku/iVnLPrFmTrOefx3PECF6vWRPTLcZfWb06ex97jLEhIWUUsYiIiFQEJd4iZeBmyff1ZtasycX+/Yn85BPeXriQqiNH0tPJiX9YWXHtunYmYD8wuE4dtnp5sWrrVqytrcv3JkRERKRcKfEWKSMPPPAAbu3a8a2dXbHnTcDW++7D58UXsbKywsrKihnvvcfC775j10sv8WSDBvR84AF6P/AAHR54gE+ee47X/vlPVsTGUr16dcvejIiIiJQ5ba4UKQMmk4lxQ4bQaPNmxmRnF9vGCohLT6ffwIG8sXo1T3frBsCDDz7InA8/xPTBB2RnZ5OXl0fNmjWxuo2SgyIiIlJ5acZb5A4VJN1OGzbwejEbKa9XC1h37hwzrqvzXcDKygp7e3sMBoOS7ntAcnIyr7zyCh4eHnh4eDB58mTOnz9/0z5hYWEMHjy42HMHDhxg0KBBuLu706lTJ954441ix/P19cXV1bXIv3HjxpXJfYmIyI1pxlvkDr0XHo7VunW8npVV5Nx8Ozueys6m3XXHCpLvjr6+xP3wAw8++KClQpVKIj09naFDh5KTk0NgYCB5eXksWbKEhIQE1q5dS7Vq1Yr0Wbt2LWvWrMHDw6PIuf379xMQEIDBYCAoKAhra2uWL1/Ot99+y+rVq3FwcADyf0lMTEzEy8sLb2/vQmM0bNiwfG5WRETMlHiL3KH+gwczcPFiDmdl0eK64zNr1uTMs8/y2f79zD9+vFDyvcLensc6daJRo0aWDlcqgWXLlnHmzBk2bdpE8+bNAXB3d2f48OGsX78ePz8/c9u8vDwWLlzIggULbjjem2++ibW1NatXr6Zx48YAeHl54ePjw6JFiwj5T0WclJQUsrOz8fT0xMfHpxzvUEREiqOlJiJ3qGnTpqzauZNhjRpx+D/HCqqXLIyJ4fNvvmHcddVO3re350dPT5Z88YUqldyj4uLi8PDwMCfdAB06dKBp06bExcWZjxmNRvr160d0dDQ+Pj7Uq1evyFgpKSkcOXIEHx8fc9IN0Lx5c7p27cq6devMx44dO2Y+JyIilqfEW6QMXJ98v2Jvby4ZaGVlxQMPPGBOvl+ztVXS/T9u165duLq6Mn78+ELHw8LCcHV1JTY2luTkZNzc3Ir0dXNz47fffjN/bTQauXTpElFRUURERHDffUX/SJmamgqAi4tLkXONGzcmPT2d06dPA3D06FHgv4l39g02AouISPlQ4i1SRgqS7+bTppmT7gIFybfTpElKuv/Hde7cmX79+rF582Z2794NwJ49e1izZg3+/v7mBLm42WsnJycyMzPJ/M8m3Ro1arB161Z69ux5w+vZ/ad8ZVYxewwuXLgAwLlz54D8xNve3p45c+bQpk0b2rRpg5eXV6FZdhERKT9KvEXKUNOmTXl12rRiq5I88MADTHnjDSXd94CpU6fi5OREeHg46enphIWF0aRJE0JCQswJsq2tbZF+BfXaC2aiq1SpUuws9/WaN29OjRo12LJlCybTf9+DajQa2bNnDwA5OTlA/lKTrKwsMjMzeeedd3jrrbewt7dn4sSJrF+//s5vXEREbkqbK0VEypiDgwMzZ85kzJgxDBgwgNTUVFauXImdnV2h5PhGSlNOslq1agwfPpzo6GiCg4MZNWoU165dY968eVy+fBnA/Muen58f165dY9CgQeb+vXr14tlnn2Xu3Ln07t1bvxiKiJQjzXiLiJSDgpJ9ycnJ+Pn50bZtW+C/S0OMRmORPgXHatSoUaprjR49mqFDh/Lll1/Sp08f+vbti5WVFYGBgQDmcoIDBw4slHQD2NjY4OPjQ1pamnnzpYiIlA/NeIuIlIPs7GwOHToE5K/xzs7Oxs7OjgYNGgD/XXd9vbNnz2IwGMzJeUlVqVKFadOmMWrUKE6cOEH9+vVp2LAhUVFRWFtb37JGt6OjozlmEREpP5rxFhEpB5GRkZw6dYrJkyeTkpJCZGQkAAaDAWdn50LVSwocOnSIli1blvpasbGx7N+/nzp16tC+fXtzov3999/j5uZG9erVSU1NpVevXsXWA09KSgLA2dm51NcWEZGSU+ItIlLG4uPjiYmJwc/Pj4CAAPr3709MTAwHDhwAwNvbm3379pGYmGjus3fvXpKSkm5aweRGli1bRnh4OLm5ueZjX3/9NfHx8ealJfXq1SMjI4O1a9dy6dIlc7vff/+dL774gscffxwnJ6fbvWURESkBJd4iImXIaDQSGhqKo6MjwcHBAAQHB2MwGAgNDcVoNDJy5EgcHBwYNmwYS5cuZdGiRYwbNw43N7fbeqPkyJEjOXr0KEFBQXz22WdERkYyduxYOnbsSO/evc3tZsyYwZkzZ/D392f58uV8+OGHDBgwgPvuu48ZM2aU2WcgIiLFU+ItIlKGoqOjSUpKIiQkBIPBAECtWrWYNGkSJ06cYN68eTg6OrJy5UpatGjB/PnzWb58OV5eXixevJhq1aqV+prPPPMMkZGRpKWlMWfOHOLi4ggICGDBggWFqpR4eXnxwQcfYGtry7vvvsvSpUtp3bo1q1at0tssRUQsQJsrRUTKUHBwsHmm+3q+vr74+vqav27WrBkff/xxqcbevn37Dc/16tWLXr163XIMLy8vvLy8SnVdEREpG5rxFhERERGxACXeIiIiIiIWoMRbRERERMQClHiLiIiIiFiAEm8REREREQtQVRORO5CcnExERATfffcdAF26dGHKlCnmV3CXtF1KSgqenp43vdaKFSt4/PHHAfj55595//33+fHHH8nNzeWRRx5h9OjRPP3002V9i1JBLl68SFpaGlWrVqVevXpUr169okMSEZE7pMRb5Dalp6czdOhQcnJyCAwMJC8vjyVLlpCQkMDatWvN9ZhL0s7R0ZF33nmnyDWMRiPh4eHUrl2bFi1aAPmv9x48eDCOjo689NJLVK9enS+++IKgoCCio6Pp3r27RT8HKTt5eXls+vJLZv/tb6Tk5mJq0ACrq1exSkzE87HHCH35ZfN/ByIicvdR4i1ym5YtW8aZM2fYtGmT+eUj7u7uDB8+nPXr1+Pn51fidnZ2dsW+sXD27Nnk5uby7rvv4uDgAMC7776LtbU1a9asMb/i+/nnn6d3797MnTtXifddKi0tjW7PP8/JJ54gY+FCaNTovydNJlbu2sXmSZMY+thjzA0Lw8rKquKCFRGR26I13iK3KS4uDg8Pj0Jv/OvQoQNNmzYlLi6u1O3+LCEhgZUrV9KvXz/at28PgMlk4vvvv6djx47mpBvAxsaGrl27cvLkSc6fP1+WtykWkJGRQcfnnuPXWbPImD27cNINYGUFTz9N2saNfJSdzat6vbuIyF1JibdIMXbt2oWrqyvjx48vdDwsLAxXV1diY2NJTk7Gzc2tSF83Nzd+++03IH+dbknaFScqKgobGxsmTJhgPmZlZcXnn3/O5MmTi7RPT08HKPSKcLk7vBIWxrHXXsP01FM3b2hlReacOXx6+DD79++3THAiIlJmlHiLFKNz587069ePzZs3s3v3bgD27NnDmjVr8Pf3x8XFBYB69eoV6evk5ERmZiaZmZmkpqaWqN2fHT58mB07duDv70/dunULnWvUqBHOzs6FjqWlpfHVV1/RrFkz85IUuTtcunSJr+LjyevTp2QdrKw4P20ab3z4YfkGJiIiZU6Jt8gNTJ06FScnJ8LDw0lPTycsLIwmTZoQEhJCVlYWALa2tkX6FVSfyM7OLnG7P1u1ahXW1ta8+OKLt4wzNzeXkJAQsrOzCQoKKvkNSqWwbNUq0l94IX85SUm1bs2PKSlcvHix/AITEZEyp8Rb5AYcHByYOXMmJ0+eZMCAAaSmpvL2229jZ2eHyWS6ZX8rK6sSt7velStX2LhxI926daNhw4Y37ZuXl0dISAh79uzh2WefpW/fvre8nlQuu3/+mSsdO5a6X067dvz73/8uh4hERKS8KPEWuQkvLy+8vb1JTk7Gz8+Ptm3bAmBnZwfkl/v7s4JjNWrUKHG76+3fv5/s7Gx69Ohx09hycnKYMGECsbGxdOrUiTlz5pTy7qQyuHzlCtjYlLpfnq1tsf9diYhI5aXEW+QmsrOzOXToEJC/xrtgWUiDBg0AOHfuXJE+Z8+exWAwYGdnV+J219u5cyfVqlWjS5cuN40rKCiIrVu30rVrVz788ENz3XC5u9RzdIT/7AUojWqpqRgMhnKISEREyosSb5GbiIyM5NSpU0yePJmUlBQiIyMBMBgMODs7F1uV5NChQ7Rs2bJU7a73ww8/0LJlyyIz4QVyc3MZO3Yse/fupUePHkRHRyvpvouN6NuX2qtWla7TlSvYHDzIQw89VD5BiYhIuVDiLXID8fHxxMTE4OfnR0BAAP379ycmJoYDBw4A4O3tzb59+0hMTDT32bt3L0lJSfTs2dN8rKTtAK5evcqxY8d49NFHbxjXBx98wJ49e/D29iYyMpKqVauW1S1LBXjiiSeo9csvkJFR4j5V//53Anx9qVJFP8JFRO4menOlSDGMRiOhoaE4OjoSHBwMQHBwMF999RWhoaFs3LiRkSNHsmHDBoYNG8aIESMwGo0sXrwYNze3Qm+hLGk7gNOnT3P16lXq169fbFwXLlzgk08+oWrVqjzxxBPExsYWadO9e/ciy1ek8rKysmLm+PGMHTuW9GXLbl3d5NQp6i5YwNgvvyQpKckiMYqISNlQ4i1SjOjoaJKSkpg7d655HW2tWrWYNGkSoaGhzJs3j5CQEFauXMmcOXOYP38+NjY2eHl5MXny5EJLPxwdHUvUDvITayi64bLAzz//zJUrVwCYNWtWsW22bdumxPsuM8jXl9+OHWNRQADpCxfCf0pNFnHkCA8MG0bsokU4Ojoq8RYRucso8RYpRnBwsHmm+3q+vr74+vqav27WrBkff/zxLccrabtWrVqRkJBww/OdO3e+6Xm5e701ZQpNV64k/OmnyejWjYtDh4KzM1y9CvHx1F20COesLFYtW2Z+gZOIiNxdlHiLiFQSI198kREDB/KPLVtY8NZbpJ49S7Vq1Wjl6sqk2bOVcIuI3OWUeIuIVCLW1tY827Mnz/5p462IiNz9tCVeRERERMQClHiLiIiIiFiAEm8REREREQtQ4i0iIiIiYgGVPvE+e/YswcHBPPnkk7Rt25bRo0dz8uTJig6r1JKTk3nllVfw8PDAw8ODyZMnc/78+SLtDhw4wKBBg3B3d6dTp0688cYbd9TuemFhYQwePLjM7klu7OTJk0wbM4anHnyQpxs14ukmTXi2fXvW/f3v5ObmVnR4IiIiUgEqdVUTo9FIYGAgRqOR119/HRsbGz744ANefPFF4uLizC82qezS09MZOnQoOTk5BAYGkpeXx5IlS0hISGDt2rXml6js37+fgIAADAYDQUFBWFtbs3z5cr799ltWr16Ng4NDqdpdb+3ataxZswYPDw+L3vu9xmg0MmbgQM5+8w0vnTtHuMmE9X/Onfr3v1k8YgRvv/oq7376KZ26dKnIUEVERMTCKnXivWPHDhISEvj8889p2bIlAA8//DCenp5s2bKFAQMGVHCEJbNs2TLOnDnDpk2baN68OQDu7u4MHz6c9evX4+fnB8Cbb76JtbU1q1evpnHjxgB4eXnh4+PDokWLCAkJKVU7gLy8PBYuXMiCBQssecv3pKtXr+LbpQv+P/7IIKOxyPmGwIzMTF7JzMRvwACmxcTg6e1t+UBFRESkQlTqpSYdO3Zk1apV5qQboGrVqgDk5ORUVFilFhcXh4eHhznpBujQoQNNmzYlLi4OgJSUFI4cOYKPj485mQZo3rw5Xbt2Zd26daVqB/mzr/369SM6OhofHx/q1atX3rd6T3t93Dj6HDxYbNJ9vdrA52lphA4Zwrlz5ywTnIiIiFS4Sp1416hRg7Zt2wL5s4mHDx9mypQp1KpVi+7du1dwdPl27dqFq6sr48ePL3Q8LCwMV1dXYmNjSU5Oxs3NrUhfNzc3fvvtNwBSU1MBin0zXePGjUlPT+f06dMlbgf5ifelS5eIiooiIiKC++6r1H/guKtlZ2ezc8MGAq9cKVH7+4FxaWl8Mn9++QYmIhWipPt6rnezfTjnz59n+vTpdOjQgbZt2zJ48GAOHjx42+OJSMWosEwsNzeXtWvX3vB83bp18fT0NH89duxYduzYQZUqVZg9ezZ169a1RJi31LlzZ/r168e6devYvXs3nTp1Ys+ePaxZswZ/f39zglzcbLOTkxOZmZlkZmZiZ2cHQFZWVpF2Fy5cAODcuXMlble/fn1q1KjB1q1blXBbwOrly/G/cAGrUvTpn5dH52XLCJ45E2tr61t3EJG7Qkn39VzvZvtwLl26xKBBgzh79izDhg3DYDAQExPD0KFDWbt2bbETMdrXI1I5VVhGZjQamTlz5g3Pe3h4FEq8R44cydChQ9m4cSNTp04F4LnnnivVNX/99ddSxxkfH3/LNn/961/ZsWMHoaGhzJo1i9DQUOrVq4e3t7e5/9mzZ4uMlZ6eDuRvlqxZsya2trasW7eOdu3aYWWVn8Ll5OSwfft2AH755RceeuihErW7evVqkTiNRiOZmZkluqfKqDLHvXH5cqIuXy5Vn+pAs+xs4uLiaNiwYfkEVk4q87O41+hZVB4Fz+Kzzz7j9OnTREREmL+3bW1tmTNnDu+//z7dunUz97l27Rrr16/n888/Byj2Z/SaNWtISkpi+vTpPPLIIwCEhIQwYcIEIiIiGD16dKnGuxfci/dcWelZFFZhibe9vT0JCQklbt+uXTsAnnzySU6dOsXf/va3UifeLVu2pHr16iVuHx8fb77urcyePZsxY8bw5ptvkp6ezsqVK2nbti22trYANGnSpMhYu3btAqBVq1bUrVuXwMBAoqOjWbVqFaNGjeLatWt89NFH5OXlAfDoo4/Spk2bErf7s+rVq1OzZs0S31NlUppnURGsgKK1ZG6tjrU1Dz74IK1atSrrkMpNZX8W9xI9i8rj+mcREhLC448/Tp8+fczn27Vrx+rVq/n111+ZNGkSkD8ZMmDAABISEujbty/79u0r8jPaZDLx6quv0qVLF1588cVC15wyZQpVq1Y1ty/JePcCfV9UHvfiszAajTed6K3Ua7wPHTpk3nx4PTc3N86ePVsBEd2Yl5cX3t7eJCcn4+fnZ16bXrA0xFjMhruCYzVq1ABg9OjRDB06lC+//JI+ffrQt29frKysCAwMBDCXCSxpO7GcmgYDGbfR72KVKubnLyKV34329Xz88cel2tcDJduHk5KSQmpqKh06dADyE/GCpYYocg1DAAAXNElEQVSDBg0yV8Uq6XgiUrEqdeL97bff8tprr/Hvf//bfCwvL49vv/222DVtFSk7O5tDhw4BsGfPHrKzswFo0KABQLHVK86ePYvBYDAn51WqVGHatGns3r2bmJgYtm/fzqJFi8jKysLa2tr8J8uSthPL6ezjQ6yNTan65AAJVavSpEmT8glKRMpcwb6ezZs3s3v3biD/Z/6OHTtKta8HMO/D6dmz5w2vV/DCuNq1axMREUH79u1p27Yt3bt3Ny8vLFCS8USkYlXqxPu5556jfv36vPzyy2zevJmvv/6al156iSNHjjBx4sSKDq+QyMhITp06xeTJk0lJSSEyMhIAg8GAs7NzoVmOAocOHSpUKjE2Npb9+/dTp04d2rdvb06gv//+e9zc3MzLZEraTixn4LBh/N/992MqRZ8vrK3pO2SINlaK3GWmTp2Kk5MT4eHhpKenExYWRr169QgJCTHPRhcsM7xewc/mgomZKlWq3HJWOiMj/29p77//Pjt37iQ0NJSIiAhsbGwYM2YMe/fuNbctyXgiUrEqdeJ9//33s3LlSlxcXJg1axbjx4/nypUrLF++nMcff7yiwzOLj48nJiYGPz8/AgIC6N+/PzExMRw4cAAAb29v9u3bR2JiornP3r17SUpKKjQzsWzZMsLDwwu9Uvzrr78mPj6eQYMGlbqdWI69vT2d+vRhWQlnvTOA9+vUIeBPf64WkcrPwcGBmTNncvLkSQYMGEBqaiovv/wydnZ2mEy3/vW7YFN8SRS8syIjI4NVq1bx3HPP0bdvX2JiYjAYDLz33nu3fR8iYnmV/lfjhg0bEhUVVdFh3JDRaCQ0NBRHR0eCg4MBCA4O5quvviI0NJSNGzcycuRINmzYwLBhwxgxYgRGo5HFixfj5uaGj4+PeayRI0cybtw4goKC8Pb25tSpUyxdupSOHTvSu3fvUrcTywqPjqbvjz9i99NPPH+TFzylA3516jBzyZJKUxZTREqnYF/P1q1bGThwoHmJSWn29ZREwXje3t6F9u8YDAa6devGunXryMrKwt7e/rbvRUQsp1LPeN8NoqOjSUpKIiQkBIPBAPx/e/ceVVWZ8HH8B8oRRUR0hnEFFok6XlbmpbFMZcxQbkMqYIQ4ppCZ6fJ1QBPMUjNDozdMIZqJUme8YyNqXrNGrbe30Zp0LcprXhB1WlPoyHEEBM77B8ELoobl2RvZ389arpX72Qd+dJac39nneZ4teXt7a/r06Tp16pQWLVqkNm3aaMWKFerSpYsWL16s5cuXKygoSNnZ2bX2cw0ODtbrr7+u7777TqmpqdqyZYsSEhKUkZFRazpCfc+DsWw2m/66d6+2DR2qSB8f7ZRUUWP8n5Lmt2ypEF9fzVy9WsHh4SYlBfBzXbuup/iHm2fdyrqe+qiaK96mTZs6Y23atJHD4aieugKg4WvwV7wbumnTplVf6a4pOjpa0dHR1X/v0KGD3n777R/9euHh4QqvRyGr73k1XbsQB7efu7u7lm3erG+++UZ/XLBAc3fuVNPycsnVVS3atlX8c8/pk+houbm5mR0VwM9Qc11PWlqa1q5dq/79+9/Sup766NSpk2w2m44fP15nrKCgQM2aNbtuKQfQMFG8AScICAjQq/V4owXgznPtup4TJ07ovffe0+eff64HHnhAQ4cO1Z///Gd98803CggIkPT/63oSEhJu6Xu1aNFCgwcP1ocffqhjx46pU6dOkipvSf/RRx8pKCiITzqBOwhTTQAAqKcbrevx8PDQ888/r5KSEo0fP15eXl4aO3asli5dqrfeektTpkyps66nvqZPny5PT0+NGTNGWVlZys7OVlxcnNzd3RvcDl8Abo7iDQBAPd1oXc+oUaNueV1Pffn5+WndunX6zW9+o3feeUdZWVnq2rWr1qxZo/bt29/uHxGAEzHVBACAerrRup5BgwYpKSmp+u/1XddT083W4bRv316LFy++bV8PgDm44g0AAAAYgCveDcClS5f03po1yj98WI6KCvl17qyo2Fh5e3ubHQ0AAAC3CcXbROfOndP8xEQd3LNHsYWFGvjDTVeOu7lpxLx56vTQQ3o+PV3+/v7mBgUAAMDPRvE2yddff6344GAtKChQhqSaNxAefPWqnv7nP/VZbq5i//53ZW7erN59+pgVFQAAALcBc7xN8O233yo+JERrCgo0SLVLd00PSdpw/rwmRUQoPz/fuIAAAAC47SjeJlg8b56eLyiQfz3ObSdpwfnzSktJcXIqAIAzFRYWaurMmbpy5UqdMbvdrv9KSdGlS5dMSAbAKBRvg5WWlurDDRsU5nDU+zGBkr786CPZ7XbnBQMAOE1hYaEeHj5cWc2a6ZGRI2uVb7vdroEjRijL3V39hw+nfAONGMXbYB988IFCL13Srdzg10VS9IUL2pSb66xYAAAnqSrdx155RaWzZ+vLceOqy3dV6c5LTNTV2bN1KDmZ8g00YiyuNNj5ggJ1+AlXrjuUlOjYyZNOSAQAcJaapbtiwABJUmlUlL6UNCg6WqWlpcpLTFRZaKgkqXzoUB2S1H/4cP1Pbm713TEBNA4Ub4O5Nmmi8p/wuHJJTZrydAHAneSzffv0bfPmqnjwwVrHS6OidMBmU4W7u8qGDKk1Vh4YqIKXX9aBAwcUGBhoZFwATsZUE4N16NhRX7VufcuPy/PwkH+XLk5IBABwlrCQEM2LiVGb0aOlq1drjZVGRNQp3SouVtvHH1f6s89SuoFGiOJtsMDAQH3SqpWKb+Ex5ZK2eHkpLDzcWbEAAE4yOT5ec4ODr1u+a/mhdL82apTGPvGEcQEBGIbibTBXV1dFJyRohZtbvR/zV1dXDR05UjabzYnJAADOMjk+XskDB8p78uQbnuMdH6/5kZGUbqARo3ib4JmkJK3o2FH/W49zD0padO+9Spwzx8mpAADOYrfbtWrjRhUNH37Dcy5HRendnJzr7vMNoHGgeJvAw8ND63bvVkq3bnrTZtPl65xTLGlp06aa3KmT1u7erdY/YV44AMB8NbcMrNq95HpKo6J0ID5eg6KjKd9AI8U2GSbx8fHR1v37tTQrS0MzM9Xz0iV1KSqSi8Oh456e2ufpqZHjx2vrlCny9PQ0Oy4A4Ce4aemuqJBcXCr//KA0KkoHVLnV4O7169W8eXNjAwNwKoq3iVq0aKFJSUl6NjFR+/fvV35+vhwOh3r7+ur1hx6SqysfSADAnWzXRx/phJubyoKDaw8UF6ttTIwc7u4qXLFCqrHup3TYMB1eskSffvqpHn30UYMTA3AmincD4OLior59+6pv375mRwEA3EbDH3tMiceOadGECbr4xz9Krq7Vu5f8d1ycii5f1uy4OBWuXFlZvsvK5D1mjGY98QSlG2iEuKQKAIATzU5K0tQuXdR6wgTpypXq0v1kTEzlVoMhIWoTFycVF1eW7sBAJT7zjNmxATgBxRsAACerKt8tu3SpLt1Vqsq3R+fOlG6gkWOqCQAABpidlKSEmBj5+fnVGZscH6/hQ4dedwxA48EVbwAADHKzYk3pBho/ijcAAABgAIo3AAAAYACKNwAAAGAAijcAAABgAIo3AAAAYACKNwAAAGAAijcAAABgAIo3AAAAYACKNwAAAGAAijcAAABgAIo3AAAAYACKNwAAAGAAijcAAABgAIo3AAAAYACKNwAAAGAAijcAAABgAIo3AAAAYACKNwAAAGAAijcAAABgAIo3AAAAYICmZgcwgsPhkCSVlpbe8mNLSkpudxz8RDwXDQfPRcPBc9Fw8Fw0HDwXDYfVnouqrlnVPa/l4rjRSCNSVFSko0ePmh0DAAAAFtC5c2d5enrWOW6J4l1RUaHLly/Lzc1NLi4uZscBAABAI+RwOHT16lV5eHjI1bXujG5LFG8AAADAbCyuBAAAAAxA8QYAAAAMQPEGAAAADEDxBgAAAAxA8QYAAAAMQPEGAAAADEDxBgAAAAxA8b5F58+fV58+ffTmm2+aHcWS/vWvf2nWrFl65JFH1KtXL0VGRmrbtm1mx7KE999/X+Hh4erRo4dCQ0OVm5trdiRLqqio0OrVqxUREaFevXopKChIqampstvtZkezvMmTJ2vIkCFmx7Cs/fv3KzY2Vvfff78GDBigefPm6fLly2bHsqTVq1crNDRUPXv2VEREhDZt2mR2pAajqdkB7iQOh0MzZ87kBc4kpaWleuqpp1RUVKQpU6bIx8dHO3bs0NSpU1VeXq7f/e53ZkdstLZu3app06bpySef1IABA7Rr1y7NmDFD7u7uCgkJMTuepWRnZ2vRokVKSEhQv379dPLkSS1evFjHjx/XO++8Y3Y8y9q4caM++OAD3X333WZHsaQDBw5o3LhxGjx4sLKysnT69Gm9/vrrKiwsVHp6utnxLGXt2rWaM2eO4uPjNXDgQO3Zs0fTp0+Xm5ubQkNDzY5nOor3LVi1apVOnDhhdgzL2rt3rw4fPqycnBz16NFDktS/f3+dO3dOb7/9NsXbidLT0xUaGqqUlBRJ0sCBA/Xvf/9bb7zxBsXbQA6HQ9nZ2YqJiVFSUpIk6eGHH5a3t7f+8Ic/6NChQ+ratavJKa3n22+/1fz589WuXTuzo1jWa6+9pp49e+qNN96Qi4uLHn74YVVUVGjp0qW6cuWKmjdvbnZEy9iwYYMefPBBzZgxQ1Ll76i8vDytWrWK4i2mmtTbmTNn9Nprr2nevHlmR7EsDw8PxcTE6L777qt1vEOHDsrPzzcpVeN35swZ5efna+jQobWOBwcH68SJEzpz5oxJyazn8uXLeuyxx+q8yezQoYMk8e/AJLNmzVL//v3Vr18/s6NYUmFhoT7//HPFxsbKxcWl+nhcXJx27dpF6TZYSUmJPDw8ah1r3bq1Ll68aFKihoXiXQ8VFRVKTk5WaGioAgMDzY5jWf369dNLL71U6xfr1atXtWfPHnXq1MnEZI1b1ac89957b63j99xzjyTp5MmThmeyqpYtW2rWrFnq06dPreO7du2SJHXs2NGMWJaWk5Ojr776Si+88ILZUSzr6NGjcjgc8vLy0tSpU9WzZ0/16dNHs2fPVnFxsdnxLGfMmDH6+OOPtW3bNtntdm3fvl27d+/WsGHDzI7WIFh6qklZWZlycnJuOO7j46NHH31Uy5cvV0FBgd566y0D01lLfZ+La6WlpenUqVPKzMx0ZjxLKyoqklRZ+mqquqLBmgdzHTx4UH/6058UFBSkgIAAs+NYytmzZ5WamqrU1FS1adPG7DiWVVhYKElKTk7WkCFDlJWVpSNHjmjRokUqKSnRggULTE5oLeHh4frss880derU6mMjRozQU089ZWKqhsPSxbukpERz5sy54Xjfvn3l7++vRYsWafHixfL09DQunMXU57moWbwdDofS0tK0fPlyJSQkKCgoyICU1uRwOG467urKB2dm+eKLL/TMM8/Iz89PL7/8stlxLKVqsf1vf/tbBQcHmx3H0q5evSpJ6t27t2bPni2p8hNSh8OhhQsXatKkSWrfvr2ZES1l4sSJ+vLLL5WSkqJu3brp4MGDevPNN6s/sbM6SxdvDw8PHTly5Ibj5eXlio2NVUhIiPr376+ysrLqsYqKCpWVlalpU0v/L7xtfuy5qKm0tFTJycnasmWLEhIS9Nxzzzk5nbVVveG8dluuqivdvCE1x9atW5WcnCx/f39lZ2fL29vb7EiWsnLlSh05ckSbN2+ufm2oepNaVlamJk2a1JoWB+ep+vTt2qmgAwYM0IIFC3TkyBGKt0H+8Y9/6JNPPlFqaqoiIyMlVV44a9WqlV588UU9/vjj6ty5s8kpzcWlqps4f/68Dh48qNzcXHXv3r36jyQtWbKk+r9hHLvdrnHjxmnbtm2aOXMmpdsAVXO7r124d/r06VrjMM7SpUuVmJionj17auXKlfLx8TE7kuXs2LFDFy5c0IABA6pfG3Jzc5Wfn6/u3btrw4YNZke0DH9/f0mVF2VqqroSzhsg45w7d05S5acPNT3wwAOSpOPHjxueqaHhcu1N+Pj4aP369XWOR0dHKzY2VlFRUSaksq7y8nJNnDhRBw8eVHp6OtvYGeSee+6Rn5+ftm/fXuvmIDt37pS/v7/uuusuE9NZT05OjhYsWKCwsDAtXLhQNpvN7EiWNHfu3DqfAmVmZurQoUPKyMiQn5+fScmsJyAgQL6+vtq6datGjRpVffxvf/ubmjZtql69epmYzlqqLsR88cUX1W+IpMp91iXJ19fXjFgNCsX7Jmw2W52t66r4+PjccAzOsWbNGu3bt08xMTFq165d9T9kqfKKxv33329iusZt0qRJSklJkZeXlwYNGqQPP/xQ27Zt48YUBvv+++81f/58+fr6Ki4uTl9//XWt8bvvvptFfgap2sKxptatW9/0dQPO4eLiomnTpikxMVHTpk1TZGSk8vLylJWVpdGjR/NvwkDdu3dXUFCQ5s+fr6KiInXt2lV5eXnKzMxUYGAgr9OieOMOsmPHDkmVd8Vau3ZtrbEmTZrUKSG4fSIjI1VaWqp3331XOTk5at++vRYuXKiwsDCzo1nKxx9/rCtXrujs2bOKi4urM/7qq6+yZRcsKSwsTDabTZmZmZowYYLatm2rSZMmacKECWZHs5z09HRlZGRo2bJl+v777+Xr66v4+Hg9/fTTZkdrEFwcP7ZlAQAAAICfjcWVAAAAgAEo3gAAAIABKN4AAACAASjeAAAAgAEo3gAAAIABKN4AAACAAdjHGwAakeTk5Dq3K3d1dVXz5s0VEBCgUaNGacSIEXUeZ7fbtW7dOr3//vs6ffq0ysvL1bFjR40cOVIjR46Uq+utXad54YUXdOrUKf3lL3/5WT8PADQmFG8AaIRSUlLk7e0tSXI4HLLb7dq0aZOSk5N14cIFxcfHV5974sQJTZw4UWfPnlVERISioqJUWlqqXbt26cUXX9T+/fuVlpYmFxeXen3vnJwcrVu3Tn379nXKzwYAdyqKNwA0QkFBQfLz86t1LDo6WmFhYcrMzNTo0aNls9lUUlKiZ599VhcvXtT69evVpUuX6vPHjRunuXPnatWqVerRo4fGjBlz0+9ZXl6urKwsZWRkOOVnAoA7HXO8AcAi3N3dNXjwYNntdh07dkyStGrVKp08eVIpKSm1SneVGTNmyMvLS2vWrLnp1y4pKdGIESO0ZMkSDRs2TL/61a+c8jMAwJ2MK94AYCFV00XKy8slSVu2bFGLFi0UHh5+3fPd3d21bt063XXXXTf9uiUlJbLb7UpPT1dYWJgGDx58e4MDQCNA8QYAi6ioqNC+fftks9kUEBAgh8OhQ4cOqXfv3nJzc7vh4/z9/X/0a7ds2VI7d+5U06a8rADAjfAbEgAaoUuXLqmwsFBS5dXts2fPatmyZTp8+LDGjh0rDw8PFRYWqqysTL/85S9/9vdzdXW95Z1PAMBqKN4A0Ahdb8tAm82m3//+90pKSpKk6qJcNe0EAOBcFG8AaITS0tL0i1/8QlJlwW7VqpUCAgLUrFmz6nO8vLzk5uZWfWUcAOBcFG8AaIR69+5dZzvBa7m4uKhXr17Ky8tTWVnZDednp6en68yZM0pJSbkt01IAwKqYkAcAFjZkyBD95z//0ZYtW647XlxcrPXr1+vTTz9V69atDU4HAI0LxRsALCwmJka+vr569dVXdfTo0Vpj5eXlmjNnjr777juNHz/+pjufAAB+HFNNAMDCmjVrpoyMDMXHxys6OloRERG67777dPHiRW3fvl2HDh1SSEiIxo0bZ3ZUALjjUbwBwOK6deumjRs3atmyZdq7d6+2bt0qh8OhX//613rllVcUGRlZfeMdAMBP5+JwOBxmhwAAAAAaO+Z4AwAAAAageAMAAAAGoHgDAAAABqB4AwAAAAageAMAAAAGoHgDAAAABqB4AwAAAAageAMAAAAGoHgDAAAABqB4AwAAAAb4P3kP6vXwETWTAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 864x864 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pal = dict([(str(x),y) for x,y in zip(range(0,num_clusters),color_list)])\n",
    "ax = sns.scatterplot(x=\"PC_1\",y=\"PC_2\",data=pca_df,s=200,hue=\"Cluster\",palette=pal, style=\"Center\", edgecolor='black')\n",
    "ax.set(xlabel=\"PC 1\",ylabel=\"PC 2\")\n",
    "for i,row in pca_df.iterrows():\n",
    "    ax.text(row.PC_1-0.4,row.PC_2+0.2,row.structure)\n",
    "plt.legend(markerscale=2.5)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Plot the Chemical Strructures of the Ligands in Each Cluster"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "lig_buff = \"\"\"Mpro-x0072\tCS(=O)(=O)NCCC=1C=CC=CC1\n",
    "Mpro-x0104\tCC(=O)NCCC1=CNC=2C=CC(F)=CC12\n",
    "Mpro-x0107\tCC(=O)NC=1C=NC=CC1C\n",
    "Mpro-x0161\tCOC(=O)C=1C=CC(=CC1)S(=O)(=O)N\n",
    "Mpro-x0195\tCN1CCCC=2C=CC(=CC12)S(=O)(=O)N\n",
    "Mpro-x0305\tCCNC=1C=CC(C#N)=CN1\n",
    "Mpro-x0354\tCN1CCN(CC1)C(=O)COC=2C=CC(C)=CC2\n",
    "Mpro-x0387\tOC1CCN(CC=2C=CSC2)CC1\n",
    "Mpro-x0395\tCC1=NN=C(CN2CCC=C(F)C2)S1\n",
    "Mpro-x0397\tCN(CC=1C=C(C)ON1)C(=O)NC2CC2\n",
    "Mpro-x0426\tFC=1C=CC=CC1C(=O)NCCC=2C=CN=CC2\n",
    "Mpro-x0434\tO=C(NC=1C=CC=CC1)NC=2C=CC=NC2\n",
    "Mpro-x0540\tO=C(NCCC=1C=CN=CC1)NC2CCCCC2\n",
    "Mpro-x0678\tO=C(CC1CCCCC1)NC=2C=CC=NC2\n",
    "Mpro-x0874\tO=C([C@@H]1[C@H](C2=CSC=C2)CCC1)N\n",
    "Mpro-x0946\tBrC1=CC=C(S(N)(=O)=O)C=C1\n",
    "Mpro-x0967\tCC(=O)NC(Cc1ccc(cc1)O)C(=O)NCC#CBr\n",
    "Mpro-x0991\tCl.CCC(=N)N\n",
    "Mpro-x0995\tNC=1C=NC=NC1\n",
    "Mpro-x1077\tN#CC=1C=CC(=CN1)N2CCCOCC2\n",
    "Mpro-x1093\tCN1CCN(CC1)C(=O)CC1=CNC2=NC=CC=C12\n",
    "Mpro-x1249\tO=C(NCC=1C=CC(C#N)=CC1)N2CCOCC2\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABXgAAAMgCAIAAAAWZJ3AAAEAAElEQVR4nOzdd3zM9x8H8Pdd9kCGrfbeI7baMUpQI6UqWvXr1YxShK6gqlG0oajQIXajitiCImhwdghiRMiUIXvd3fv3x4drJEZy971cEq/nI3+Q3H3uneSb+36/78/78/7ImJkAAAAAAAAAAKQgN3YAAAAAAAAAAFB6INEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSMTV2AABQrF1NS7uelkZEDaysnMqUMXY4AAAAAABQ3CHRAACvcjopqYWNTRVz87KmeLsAAAAAAIDXw9IJAHiNQwkJf8XFJavVxg4EAAAAAABKAExRAsBrjKtSpaaFhUwmM3YgAAAAAABQAiDRAAAvlahSRWVnm8hkcmQZAAAAAACgYLB0AgBean109P74+H3x8cYOBAAAAAAASgwkGgDgxeJycnY8fiwj6mVnZ+xYAAAAAACgxECiAQBe7LeoqEyNpre9fUNra2PHAgAAAAAAJQYSDQDwAlHZ2bvj4uQymaJqVWPHAgAAAAAAJQkSDQDwAuuiorKZ+zs41LG0NHYsAAAAAABQkiDRAAB5hWdl7YuPl8tk4ytXNnYsUNROnKDQUCKigABjhwIAAAAAJRMSDQCQ17qNG6sxD3Z0rIlyhjfPjh00dSox06ZNxg4FAAAAAEomGTMbOwYAKEauX7/eokULcwuLu+HhVcuXN3Y4UERycmjXLvrpJ6pThzp3JhMTOnOGfH0pNpYqVjR2cAAAAABQoqCiAQCe4+npqdFoFJ98gizDG+LJE/rhB6pTh957j/79l4KDafRoOn6coqPpxAmqWZPmzSONxthRAgAAAOhNpVLl5ORgrr0IINFQ3N2+ffvEiROXL182diDwRrh48eLff/9tZWXl4eFh7FjeJNu20ZIlFB1dxC97/z7NmUO1a5OHBz16RPXrk7c3de5MRLRoESmVdPw4ZWXR/Pk0aBDFxhZxdAAAAACSSU5OXr58eb169WbOnOns7BwZGWnsiEo5JBqKr1OnTr333ntNmjQZPXq0k5PT2LFj09LSjB0UlHJff/01M7u7u1fFrpZFZu1aunqVBgygKVMoOLhoXvPff/9VKBbWr0+LF9OTJ+TsTPv3061bNG0arV5NdnZUuzbFx5OnJx08SJUr0/791Lw57d9fNNEBAADo6GTqyTRNGhEdSD5ARGfTzv4Q88OdrDtEdCzlmJGDAyO5c+fO1KlTq1Wr9tlnnz148OCPP/44duxY69atDx48aOzQSjMkGoqdtLS0NWvWNGnSpGvXrtu3bzc3N2/QoIGFhcXGjRvbt29//fp1YwcIpdaZM2f2799va2v7+eefGzuWN8nBg/TVV9S0KU2eTIcOGfSlNBrNnj17+vTp07lz53XrvnZyinN1pbNnKSCA3nmHZLIXPKVvX7p8mfr3p9hYcnGhadMoO9ugMQIAAOhu15NdiepEIvo97vcNCRsOJh8cYTfi68ivszl7a+JWY0cHRU1M3DZq1GjlypWpqaldunTx8/O7ffv2gAEDYmNjBwwYMG3atGxc2RiGqbEDkNq2bfTwIbm5UQncli8yMnLt2rUrV66Mj48noipVqigUiilTppQvXz4kJGTkyJHXrl1r27atl5fXtGnTjB1skYqPj3d0dDR2FKVETk5OampqUlJSenp6enr6kydP0tLS0tPTU1JSli9fTkSff/55hQoVjB3mm6RKFUpKImtrCg+nRo0M9CKpqam///778uXL7927R0QODg6ffvrplCmqV1euBATQH3/QL7/Qvn30ww/0zTe0YgUlJKz85pt+9evXN1CoUMROnKCqVal+fQoIoD59jB0NAIDe5kbMtZXbXsm4YiIz+b3m79Zy6621kWJ4s2RnZ+/evXvZsmVnz54lIgsLi9GjR8+aNat58+biAXv37l2xYsXs2bNXrFhx5syZrVu31qtXT88XZeYjR47cvHmzQoUKgwYNsrGx0ffbKOm4NPHx4blzOTiYhw/na9eMHU0hKJVKNzc3U9OneR8nJydfX9/s7Ozcj0lPT3d3dxcPGD58eGJiopGCLWrx8fH16tX74IMPMjIyjB1LyaDRaP7555/Ro0e/++67zs7O7dq1a9q0ae3ate3t7c3NzV/xbuDo6FimTJnbt28b+zt4k5w5w3/8wSNG8KRJPG0aG+YgP3TokJ2dnfgt16tXTyT1X/sslYobNGAirlWLT59mZg4K4oEDj8hkMltb2z/++MMQoULRmzqV+/VjjYbHjuWAAK5e/QUfffueqP4i7733XnBwsLG/AwCA/0x/OP1h9kNmHnF3hGek56PsR8y89vHaVHXq/x78z9jRgcHFxsZ6eXlVq1ZNXPZUqlTJw8MjIiLihQ8OCgqqXbs2EZUtW/bvvyN1ftHMzExfX1+RxTAzM5PJZA0bNrx06ZLOA5YOpSvRMHQop6UxMx87xkuXGjua18vKyvLz8+vYsaP4SzA3N3d1dT0truhzyczM/Pjjj+/fv8/MGzZssLW1JaJatWoFBQUZIegid/ToUWtrayLq0qVLdHS0scMpAWbMmNG+ffuXZRNMTEzs7e1r1KjRqFEjJycnZ2fnwYMHjxo1SqFQiDnqUaNGGfs7eJN068ZEfPQoh4WxvT136SL5Kzx8+HD48OGmpqYig6lSqQr+3LAw7tKFidjUlD09WaXi5ORkNzc3bcYzISFB8oChiE2dyqtW8Zo1PHYs797NRC/4cHJ68SpWkb68deuWsb8JKJ0SExPPnz+/bdu277777uOPP+7Xr9/Vq1dFu3iAl9EmGlzvud7NvDvwzkDFA8W8yHnM/MmDT4wdHRjQrVu33N3dxV0DEbVq1crHx+e185RJSUmjR4/u1OknInZz4wJMxDwnKirq66+/1tYCV6tWbcaMGc2aNSMiCwsLb29vjUaj+7dUwpWKRMN777GnJ6vVPGkSR0YyM69fz3v3GjusV4mOjs6dbKtYsaKHh8fDhw9f+OCvvvqKiOzs7Hbs2MHMt27datWq1Rt1+F6+fLlWrVrir/fcuXPGDqdYi4iIsLKykslkCxcu3LFjR0BAQFBQ0LVr1+7evZuQkJCZmfmK5z548ECksXbt2lVkAb/p6tVjIr55ky9dYiJu3lzyVzhx4oSok9Lt6dnZ7OHBcjnLZJqxY78Qb1N+fn6iRKJGjRqBgYGSxgtFbepUTkzkUaO4b19OT+cHD17w8fBhxoMX+eCDD4jIxcXF2N8ElHgJCQlKpdLPz8/Ly0uhUDg7O9epUyd/bsvc3HzGjBnGDhZKmHR1urFDAAPSaDQBAQEuLi4ymYyI5HK5s7Ozv79/oQb59dcca2sm4saN+cqVAj3l4kWeOfNHbbFwu3btNm/eLArSMzIytHXoQ4YMiY+P1+H7KgWKS6JB9/z0zz8zEZcrx48e8ZUr/P77Bq1A1l9ERMSYMWO0B2Xr1q3Xr1//6tu/pKSkkSNHise7ubmlpaW9gYfv48ePe/ToQUSWlpYbNmzQc7Tjx1msDzh8mE+ceJq83L9f7yiLgQkTJhDRyJEjdXu6t7c3EVWtWvXJkyfSBgYvZmPDRJyczPv2MRH36yf5K2zdupWIXF1d9RnkyBEeNGglETk4OPz999/MHBoaKgpnTE1Nf/jhB4mChaIWE8Nt23J4ON+7xw4O/33+/Hk+deq/j6CgxFPPe/z4MTPHxMSUK1eOiPaXjjdQKCpJSUlr1qyZNWvWsGHDWrZs+bKVzDY2Ni1atBg6dOisWbPWrFnz888/i8unP//809jfAQAUFytWrBDvGGXKlHF3d79z545u44SEcMuWTMSWluztzS+byVWrOSCAXVxYJuO2bQ/I5XIXF5eAgID8j9y5c6eDgwMRVa9e/eTJk7pFVaIVi0TD+vXr27RpExoaWuhnXrnCVlZMxFu2MDN/8QWfOlXokpei9dNPP1lbW7/ioHwZX19fKysrImratKlYE/v333/b29uLScVTp04ZLOTiIicnZ8qUKUQkk8k8PDzUarXOQ+Vekzx9OotSkhEjJAvVWMLCwszNzU1MTG7cuFHY527dujUwMFCtVnfu3JmIpkyZYogI4TmJiUzEtrbMzGvXMhGPGyf5iyxbtoyIpk2bxsxLlixZsGBBbGysDuM8fvx48ODB2oxnampqTk6Op6enTCb78MMPY2JiJI4bisT//sdEPGZM3s9XrfqapRNbt24Vj1yyZAkRNWrUKE9fIYCXefz4sYeHR54jyt7e3snJydXV1cPDw8fHJyAg4O7du/lP9D///DMR2draojkIADDz0aNHHR0d7ezsli1bpv8kWUYGu7s/PfG9+y7nmclNSeEVK56Woopp7pkzNffvP3jFgOHh4W+//TYRmZiYeHp6Fmr56gvduXNny5Ytr56iLj6Mn2hQqVSic0bZsmULl6JOSeGGDZmIJ01iZt60iYm4bFkuxmuGU1NTReGfaLhQWBcvXhSr6MuUKbNx40ZmDgsL69Spk5hU9PT01Ofeu6Tw8fExMzMjogEDBuj8hpJ7TfL06TxmDE+YwPXrSxupEXz00UdE9OGHHzLzwoULZ86cWcBbys2bNxNRgwYN0tPTQ0JCLCws5HI5SuINLjiYibhhQ2bmefOYiL/6SvIXEZuVLl68mJlr1KhBRHfv3tVtKI1G8/PPP1taWhLR6NGjxSdbtmxJRCdOnJAsYigqFy+yXM7m5py/x8KQIdy5838f779/tvPzjhw5Ih6ZlZXVsGFDIvrpp5+KOH4ooZYuXUpEtWrVWrhw4bZt286fP1+obi8ffvihOGElJSUZLkgAKBHWr18v5j8kHHP7drazYyJesOC/Cugvv+Ry5Z6mGOrW5eXLOTm5QKOJWRkTExMi6tGjx6NHj3QIKTg42MvLq0uXLiJnUbt27RJR2GX8RAMzJyUljRo1KvfSgII8K2vxdCbili05I4NDQ7lsWSbiX381dLT6uHHjhjg76jxCcnKyWBObZ1JRLpcTkbOzc1RUlIQBF08nT56sWLGi+EnevHmz4E9UqXj/fv799+fWJJeaiobbt2+bmpqamZnduXPn8ePHZcqUIaIzZ84U5Lk5OTmtW7cmorlz5zLzN998Q0QNGzbETh+GdfgwE3HPnszMCgUT8erVkr/I+++/T0QbN27UaDSi6jg9Xa/VqlevXu3QoYO2/59o16xLSRoYm2hFOnOmvuPs2bOHiOzs7HQrloE3ikajadCgARHt2bOnUE+8dOlSu3bt7t27l56eLk5Y77777pvQpgoAXmHhwoXay1cJ3b/PS5Y8VwH90Ueivo99fVmHFf/Hjh2rWrUqEZUvX76A734ajebff/+dPXt27n037ezsnJycRBXYgwevKqYoDopFokHw9fUVbUIbN2587XWbUz5+/KvyvOzJt3345k3OzOTWrZmI33uvaELV2aFDh0Q6gJkvXrx49uzZlJQUHcbx9fUVCxobNWp05coVZvb393d0dCSiVq1avQnn3bt374qGrg4ODgVZgXLrFnt6cq1aTMQODjxpEicmPl2TrE00DBnCJXoKX2TrPv30U342iV2oDm3nzp0zMTExNTW9cOFCVlZW06ZNieibb74xWLzAvH79f2XrLi5MxAZow9mtWzciOnr0aExMDBE5OjpKOLhGoxEFDgXZLxOKla1bmYgrVmRJ+rG88847RDRhwgQJxoJSLSAgQKxYLmwJ8bBhw4ioffv2mZmZoaGhojnIjz/+aKA4c8vM5Bs3eM8e/uknnjKFly8vgtcEgAKZOHEiEa1cudIQg+eugL5zhy9c0Gu02NjYgQMHijXg7u7uWVlZL3yYSqUKDAx0d3d/6623tPkFR0dHNzc3f3//rKwsjUYzaNAgIurWrZv+azEMqhglGpj58uXLogLTxsbm2rWXFoSkpwdfvGitVFJc3AZmVn81k4m4fv2ClrAYz7p164ho3LhxzNy/f38i2qvr7hjXrl1r0qQJEVlbW//111/M/PDhw1q1ajVv3vzAgQNSBl1cpaSkDB06VFQQeXl5vfAxSUlJ69eni/35xEeDBrxoEedP7+TkcK9ebGpaUi8grl27JpfLLSwswsPDIyMjra2tZTLZxYsXCzXIZ599JnJV2dnZQUFBcrnc1NQUmwAbzuGff97RvfuV779n5thhwzLr1FEZYFMVkQi/efPmpUuXiKi5pBtbPH78WOTXJRwTikB6OtesyUS8bp00A4aEhJiZmZmYmFy+fFmaEaEIZWdnh4aGvnaORxLDhw8nom+//ZaZ58yZs3379gJ290hISBD7UIh8ur+/v0wmMzU1lXbdVlJS0sWLF7dv3+7l5fXJJ58MHTqpRg2Wy59rWSKq0ACgOBC33AbaLi13BbQkNBqNt7e3KC91cnK6LRZmMDNzZmZmQECAu7t75cqVtfmFGjVqKBQKf3//PNsmxMbGVqlShYi+++47aSIzjOKVaGDm9PT0Tz75xN29rVIpu3/fTa3Oe0eoVmdcv95SqaSwsI+ZOSFh+9VTZbJGduNC3lMZhahIF7PEYsb46tWrOo8m9p6wtLTUXtiJG2+Rd3gTaDQaLy8vsWzkgw8+0Nb5q9XqwMBAhUJha2vbvfu/oneHmxsHBLy0haxKxR4eT68hFAqWpKlZSkpKRkZG0RSYvPvuu/Ss4d/kyZOJaEThl4KkpaWJy7glS5Zox2nfvn0xT5eWXKK5qbe3NzNXqlSJiCIiIiR/FVH9lJycvHfvXiLqJ+nGFleuXCGiJk2aSDgmFIHvvssh4tatWcLGPtOmTSOinrgJK96ysrLu3r0bEBDg4+Pj4eHh6urq5OQk6pKcnJyGDRtm0C2HoqKizMzMTE1NIyIi7ty5I5fLra2tC96g4fLly6L09ffff2dm0VGyUqVKuq15zq9NmzbPd6gkU1NLMzM2NeW6dblfP540iZct4yNHntu7CkqZPL/cUrY3WekjFlIplUpDDC4SDXl2ZdKfUqkUk0BlypT57bff/P393dzcRJWWUKdOHXd398DAwPw3Ebdu3Zo9e7ZGozl06JCYESzgKmmjKHaJBiE62vfiRRulkoKDG6WnP7eZaVjYeKWSgoMbqFTJWVkPLl92UCopJmaFsUItlHHjxhHRunXrmLls2bJElJiYqOeYufu6iUU7Z8+e1XPMksXPz0/cSnXu3Pns2bOenp61atUSf6hyufy99+Zs2MAF6/vBW7ey2ET37bdZnyb6SqVSoVCUKVNm/Pjx3bt3N/S6ZaVSKZPJbGxsoqKiHjx4IFo5XingLsDPO3r0qEwms7a2Dg0NTU5Orl69uvZOGCQnKoH9/PxycnLkcrmJiYnkOZ3ExEQisrW1Zea1a9cS0ccffyzh+AcOHCCiPn36SDgmGFp4eHiFCrV69Dh18qSUadCEhITy5cu/UcnuYi4xMVGpVG7btu277777+OOPu3fvXq1aNXoRuVxeo0YN0fxoyJAhhsuPL1iwgJ7ttjtz5kwi+t///leoETZu3EhElpaWFy5cUKvVffv2JaJOnTq9rAi5UHr16mVpadm0adMhQ4bMmDFj1apVBw8evHMnJ8/cQ0YGT5ny38ptMK59+/Zdv379+vXrUg2Ye1k+c+np5FVaVahQgYiio6MNMfijR1yjBg8dKv3I2gaFuTf3bdOmzcKFC19xMGdlZYkr859//pmfLZSuU6dOse2MW0wTDcyckXFTVC5cvGgZE6O9z1GHh0+7eNE6Pf2aRpMdEtJJqaTQUBfmktGVoHfv3kR06NAhcfVfpkwZaccXVwlvQj/IPNavXy8KjcSGFERUq1YtT0/Pe/fuFXaos2e5WjUm4tq1+cqVwk32hYeHf/vtt9qWLTKZTKST6tWrJ+EpMD+xDMfDw4OZx48fT0Rj8u9WV2Bubm5E1KNHD41Gs3//frE8R+d9CuAVOnToQESnTp0KDw8noqpVq0r+EsHBwUTUsGFDZp43bx4RfSXpxha//fYbPdvoBEoKcXEzatQoyUdevXo1EdWuXRt9ZI1r9+7dom1Tfubm5g0aNHjnnXemTJni7e29Z8+ekJAQsVNaWFiYSBV9//33hohKrVbXrFmTiI4cOZKZmSluD86fP1/YcT755BMiqlmzZlxcXExMjFjGPH36dP0jfPLkSZ4kS0YGBweznx97ebFCwc7OXKcOy+U8Zsx/K7fBWEJCQkR3mIYNG5qamioUCknmdXIvy2cuVXuTlT6ZmZkymczc3NxA++6dOcNE3KGDIcZmZv7f//5HRBUrVly2bFkBdyTcsWMHEVlYWFy+fDk7O7t9+/Yk9aYbEiq+iQZmVqszwsPdlUpSKunOnaEq1dPiuqysh8z88OF0pZKuXq2ekxNn1DALQXRavnHjxtWrV4moadOmEg6ekZFh0D+24qxnz55ENHDgQDc3t6pVq77//vv6/BAiI7ljRy5TJqlZs3YFmZrLzMz08/NzcXExNTUVV3LVqlXz8PC4ffv2o0eP2rVrJ5JKO3fu1DmkVzh16pSYso6NjQ0NDTU1NTUxMSnUZhx5xMXFiYyVqE0V9yT9+/eXLmR4SqSl79+/HxQURERt27aV/CUOHz5MRL169WJmhUJBRKsl3dhCzE9K3u0ZDOfMmTMymczKyiosLEzywVUqldjudOHChZIPDgV34sQJcSVap04dZ2dnhULh5eXl7+9/9+7dnFd2Sz98+LCJiYlcLj906JDkUfn7+xNR3bp1NRqNKExo1aqVDuNkZmaKE6uLi4tarQ4KCrKwsCAiPTd7S0hIUCqVfn5+Xl5eCoXC2dm5Tp06TZqk5e7OID7MzHjoUIlXbkOhxMXFTZo0SVx0OTo69uzZU+wd6OjouGrVqlcf5K+VZ1k+KhqKszt37ojstoHG9/NjIh4+3EDD84YNG3RIE4j0RJMmTdLT00NDQ8U2cxs3bjRQkPoo1okGIT5+66VLZZVKioyc//jxutjY1SrVE40m6+bNLhcumKWm/mvsAAtBrC1MSUkRi6WlvXm7ffu2qJ+RcMySQqxvDwsLO3v2LBG1bNlSzwEzMnjmzJ9EVcK8efNeVkeqVCrd3d21E0cWFhaurq55WrZkZGSMHTtWDOXp6Sl5SapIsnh6ejKz2Pq0sJWo+W3ZsoWIypUrFxER8fjxYzHvVDzfwkoutVptZmYmk8kyMjJEfnrw4MGSv4rYX1pUuLi4uJDUDZNEt2dRwgfFn1qtFndo4h3DEI4dOybKoMLDww30EvBaGRkZkZGRuj1XNJNydHQs4PRawYl260uXLmXmt99+m56tJNXBgwcPRPGFaCq5cuVKkXAPDg5+7XPVanVYWNjRo0d9fHxmz549fPjwli1bilnB/Lp3v9e8Ob/7Ls+cyb/8wocP8927nJNjqJXb8Fo5OTk+Pj7isiR3FUNISIio7iSiRo0a7de1ocLDh9ywIZ869d8vV5tocHVllYqf7ewMxcLx48eJqGvXrgYa/8cfmYinTTPQ8Ozr60tEY8eOZeaLFy8GBATExb1++jwtLa1Ro0ZENHnyZH621YCtrW3u1pLFRAlINDBzZuadsDBFZOS3T57sS04+Eh39AzNrNDkpKVK2GjY00Z7dwcGBmX/55Rci+uSTTyQc/+jRo0TUvXt3CccsERISEsQfmEaj+eOPPyQsCfbx8RH5chcXl9zLnyIjI729vcXEneDk5OTt7f2Kdwdvb2/RtHLkyJFpBewYUQBinzA7O7vExMTg4GC5XG5ubq7DgpH8Bg8eTM9W0or3QUdHxxh9GlfA86KiooiofPnyzPzzzz8T0cSJEyV/le+++46IZs+ezc8aJp2TdGMLcZzs2LFDwjHBcH799Vcieuuttwy6HaloPjIWNeUlk1qtFuXoHTp0EEsqJPHgwQMTExMLC4vY2NgbN27IZLJy5crpcxweOXJEFF+InbY+/PBDImrQoEHuk3V2dnb+zpdiyiePt99+297e3snJydXV1cPDw8fHJyAg4O7du3mqIxMS+Px53rqVDVDwAa939OjR5s2bi19Zr1698vdT9/f3r1u3rniAi4vLnTt3Cj54ejovWMA2NkzE77zz4scsWsSrV3MBNlWHIrJp0yYiev/99w00/uefMxH/8IOBhn86GyTWn/br14+IDh48WJAnXr16VTTxFbNHovq4Xbt2BdzEp8iUjESDkJz8z+3bfSIiPMXSiRLnwoUL2sn2L774gogWLFgg4fjiYC22q3QMJzAwUPx1MfPs2bOJaP78+VINfujQIXt7eyJq0aLFrVu3/P39XV1dtZ0gKleu7O7uXsDt3Pbv3y86yrZq1UqqouVOnTrRs/W0Ys+wKVOmSDJyRERE7dq1fXx8xH+7dOlSpkyZY8eOSTI48LM3hNq1a1+8eFHUvEj7hiAYemOLtm3bElFQUJCEY4KBJCcni92wtmzZYtAXunfvnqWlpUwm27t3b9zz4uNVcXGc++PwYTR4L3YSEhLq1q3Xvfv+KVMkK8H78ssvtZco4n1J/7OVWLrl4OBw79699PR0kUtt1aqVQqHo3bt3rVq1RDl9flWqVOnatetHH320cOHCrVu3njt3Lj4+Ps/gkZGRJ0+e/OOPP777zuf997ldO3Zw+G/1xIABesYOhRMaGurq6ip+ffXq1fPz83vZI7Ozs729vUWHLDMzM3d394L0yfP359q1n/5yXVz4ZfM1jx7xnDls4AbfUAheXl5ENGvWLAONP3IkE7HhzplifvSjjz5iZtHatuDL1pYtWybmqyIiIhITE0UHnC+//NJQseqkBCQasrMfqVRJzPzkyR5mdWrq2Xv3RsXHb0lJOWXs0Apn165dRDRo0CBmHjNmDBGtX79ewvHFGfeLL76QcMwSwcfHR5sOFMXh27dvl3D8W7duiQolW1tbcZIzNzcfPnz4nj17CrsOUDtU+fLljx8/rk9UKpVq6dKlRFShQoXk5OSsrKw+ffpYWVnpXC77wpfQ/rt79+5E9Mcff0g1+BsuMzPzjz/+EE1bGjRoIJPJ2rdvf+TIEclfSGx56+fnl52dbYiNLapWrUpEKJIvEU6dOlWuXLkWLVqIBVwqlerChQsGeq1p06a98B6vXr1reZa7d+6MBu/F0cWLGZaWTMSSrJnLyckR7xWnTp1KS0sT6Xt9tvcW1Gr1wIEDO3To8PDhQ2YODQ21trbOvQU9Ednb23fp0kV0qfDz81MqlcnJya8Yc/z48S1atMhd9WBl5SCTPT1cbW25ZUseNoyXLtUzdiio1NRUT09P0YbDxsbG09OzIL1mIyMjFQqFqCStUqWKj4/Py1p3Xbx48aOPborfb6tWfOKVpdLDhvGPP7IeXbBAYlOnTiVDbo7WpQsT8cmTBhqef//9dyIaN24cM/fp04eIDhd4y1yNRiPWo3Xv3l2lUgUGBooir6NHjxoq3MIzfWGut/hgzrl7d7hKFVev3h7m7Pv3PzQ1dTA1rfDgwXjmnKpVF1Su7EEkN3aYBSIay4v2bw8fPtT+WyqGGLNECAkJIaLGjRsT0Y0bN4ioSZMmEo7foEGDoKAghUJRpUqVgICAsWPHjhs3TrRL1GGo06dPjxw58siRI/369Vu9evXHH39cqBHu3bt35JnExMRKlSr17dtXtIE5fPhwWFiYmLGUhPY+4dSpUydOnChXrpyokwd9REdHr1q1as2aNXFxcURUs2bNunXr3r1799y5c1999VWdOnVq164t4ctFRkYSUbVq1UxMTIKDg+Pj4182xacDtVodExMjl8vzXNlD8ZSZmZmWlpaWlqZSqbKzs7t27Xrr1q2bN29KctbQaDR9+/bt06fPZ599ZmFh8eTJE7VabWVlZWVllfthZcuaOjg890RTUxo8mNau/e8zc+eSrS1duaJ/UKC71q0tV6wghYI++YSaNqXWrfUabefOnZGRkU2aNOnSpcuvv/6amJj49ttva2vgdSaXy7ds2WJhYSHuQhMTE9PT0+Vy+Y8//ti4ceO6devWqlVLW4FYQCEhIaJdt6OjY7169erWrVu3bt0GDXJq1zarW5fwVleUmHnjxo0eHh7R0dEymczNze2HH34o4OlGJBcUCsW0adNOnz796aefrl271tvbWzQHEeLj4xcsWLBq1Sp7+4Y1a16bPl0+ZQq9+gy5Y4ee3xNI7NGjR0Qktp4xBJUq3MKiUrVqFgYan5mJSCaT5fl3Qchkst9//71ly5YnTpxYtmzZ7Nmzv/jii2+//Xbs2LFXrlx52cZDRc3IiY7Xefjwc+3WEmp1ikaTqVanaTQ5ERGeSqVcqaRbt3plZ0swhRsfH7948eK9e/caruPdrFmz6FmVe506dYgoNDRUwvFFF5x9+/ZJOGaJIGqN/P39MzIyTExMzMzMJNlPOz+ppoJVKpWHh4f4A1QoFK9dTxUeHr5+/Xo3N7c8SQRxui1fvvzjx48lCexlevToQUTz5s0z6KuUerdv33Z3d9fed7Vp08bX11cUxQQFBYn3hLJly0r4FqRWq8VBInlTN0Gc4CtVqmSIwUFyWVlZDRs2JKKffvqJmUeOHEnSLW0VjYeqV6+elpZ28eJF0S/mVgHapr2iwTv6rhnd+PFMxPXqcWKiXuOIvb1XrlzJz9ZbGeJaS+zurGcR9dmzZ5VKZaKe3zDo7ezZsx07dhSny3bt2p05c0a3cTQajZ+fn6gql8lkrq6uYWFhWVlZS5cuFatZzczMZsyY8eTJq+pcoNgS7Y0NtH5TrVabm5vLZLLMTIPcVvCzxknjx4/nZ++ThS1uPXDggEwmMzMzCwoKysnJ6dy5MxENHTrUMPEWWrFONDx5sl+plF24YJqScopZfft2n9DQQdrNLJOSDly+XFGppCtXqly/rnuu4datW+7u7jY2NuKKmYiGDx9uiHOMaNSxadMm7YGbnp4u4fhiGl//WsQSJ7pPn/Pdu0ffuXP78uXqZcs2btzY2BEVyKZNm0Qfl27duuXf+Tk5OTkgIMDDw8PJySl3drNixYqurq4+Pj7i1lEkWcSeAgZy6NAhInJ0dCzIKkd4ocDAQBcXF/F7lMvlLi4uAfl6SSUkJIwYMUL8lj///PeUFL1eMSMjw9fXt1GjRhYWFra2tqNHj07Rc8RnHj58qB3q3LlzImMiychQBPbs2UNEdnZ2sbGxDx8+FCe+k3pXhSYlJYmzp1i21q1bt4Lf7+Xp3p+7wTv6rhldRgY7OTERDxrEOu+YdOfOHZlMZm1tnZiYeOnSJXFCKUj1e6E8efLExsZGJpMVJL0FxdyECRPEGbN69epbtmzRf7uu1NTUL7/8Ulx0WVtba6dtBg4cqM924GB04lf56NEjQwwuakINOpsiNowQu8X16tWLiHRY+DBt2jR6tjb//v37okFJwZdgGFTxTTRkZT28fLm8UklRUV7MHBm5QKmky5crZmf/18ksJyfm9u2+a9fOl8vZ3Z0L1WhTreY9e/Y4OzuL9zKZTNa/f/8vv/xSJDhr1aoleXpMJJlOnjyZnZ29YsUKyXcXEwfWkydPpB22uEtKYpmMLS1ZpeLNm5koq+S0wzxz5oyYcK5bt25wcHB6erpILnTp0kXsdiHY2Ng4Ozt7eXkplco8p9uwsDDROcLf399AQYopBS8vLwONX4plZWX5+vpqy4MtLS3d3NxCQkJe8RRfX9+qVZtUq5ZTqxbrNn8TERExd+5ch2fl6VWqVBFFxY0aNbp06ZJu34hw6dIlNzc3MzMzMR/Oz/rOuLi46DMsFDGxocCECRP42UaGrVu3ftnq5QKaPn06EXXp0kWj0WzdulWkRPU/GaHvWnEgckBEvGRJoZ/75MmTCxcuiDV3YputJ0+eeHt7a99DJLRixQoi6tOnj+QjQ9H76aefrKysPDw8Xt1Qo7BWrVrVrFkzExOTJk2aODg4vPfeexIODkUvOzvbxMRE8uZTWmI2xcnJyRCDC2vXrtW+PYrt6nVou56RkTF//nyxpZ1arW7RokXFihULstdvESiuiQaVKmfK6JBDVW/f7s+sTkk5eeGCqVIpT0rK24pTo1EtWZJlYsJE3L07FySllZzMPj7cuDG3bNmNiGxtbRUKhfb3cevWrVatWhGRhYWFt7e3/mlULbGCyEA1zImJiURUpkwZQwxerAUFPW3gw8xffslE/PXXxo6pEMLDw9u0aSOyCeKGULCwsOjZs+fChQv//fffV7ecFF1na9SoIe35WNi9ezcRVahQQar58DeEuJjWLhqsVKmSp6dnQfZGZuabN7NatWIiNjfnJUsKMYt4+TLPmeNvbm4uXrRTp07bt29XqVQ3btwQ72lmZmaenp6FvaVUq9X+/v5i+QwRmZqafv755+JLYldOhUJRqAHBuEJCQszMzExMTC5fvpyWllazZs2uXT/x9dV9l8HQ0FBzc3O5XK5UKtPT00WJ8rp16/QPFX3XionDh9nJiV+9UWBMDJ8+zRs28Hff/fjBBx907NixfPny2jOanZ3d8uXLDRpks2bNiOivv/4y6KtA0cjOzn7w4IHkw7733ntEtHz58nv37okCQ8lfAorS5s2biah8+fJ65spf5u+//yaiIUOGGGJwYc2aNUT06aef8rN1yv/8848+Ay5atEjk+iXsDa+P4ppo+OorJtI0a5KTHatSJVy9WlOppEeP5r7s4cePc7VqTMTly/PevS8d9c4d/uwzLlv2affggQMPLV26NP8qiczMTHd3d1HpMGTIkPybHhWWUql0c3OztbW1tbXdsGGDnqO90JUrV4ioadOmhhi8WPv9dybi0aOZmYcONewuNIaRkZHRrVu3qlWrymQyJycnd3d3Pz+/gq9TUKvVYpPLadOmSRuYRqMRN6iGa+db+oSFhU2cOFHbsbxly5a+vr6FbRqSnc2eniyXMxH37s2v3Y8yMJBdXFgmYzu7hDJlyrq4uJw69dymPBkZGdr3NGdn5wKefjIzM319fbWtVcuUKePu7i52Zk1JSfHx8alSpUq9evUMMTkJBiXKLHv27MnMf/6ZSsSVK7POS6NEiYS4TvL09BQlEgaaXwKjOH78abrn8OHn9h9dt45HjOBWrbhMmf+2EWnR4r9me9bW1s2bNxdVXWITSgNFePLkSSKqXLlycdtDHooVkWjYtm2bSqVCoqHkEuWiLVq0oGf7wfXs2VParbsFUSc1efJkyUfWEu2NRI2hWHV44tUbn7zSuXPnzMzMZDLZnj17pItRL8Uy0fDPP2xiwnI5HznCGk2a9+SLQeY3b3bRaF41r/v4MQ8YwET81Vcv+GpKCg8e/PTCnYh79OC//+ZXXwjt3LlT7MBUo0aNPFftBZSZmbl+/frWz5o1a4vhP/30U/27M4SGhrq7u2uXlu3du5eI+vfvr+ewJc+sWUzE337LzNywIRPx5cvGjqnQFi5cSESfffaZbk+/evWqmZmZXC7X7UB9mT///JOIqlatKm0zkdJNFNqJMnJ/f399SqL27eOKFZmIK1Z8ca1WRgavW8dNmjx9WytThqdN4/v3X1qvfujQIbFUp2LFiq/uGhsTE+Pl5aVdxVqlShVPT8+EhARmfvjw4ezZs+3s7MSXXF1ddf4GwVgSEhKaN3fq1s1vxw7WaLhrVybi2bN1GWrfvn1EVLZs2aioKG3TB32uk6AYmjr1vy1Ic3frVCj+yy/Y23PbtjxyJC9Z8tfvv/9+4sQJ7XW/RqMZNmyYSLwa6GzywQcfENHXJaqeEYoeEg0lXXR0tKenp3bft6pVq2q3gatYseLBgwelfbnZs2cT0aJFi6QdNrfVq1cT0cSJE5m5a9eupEfXpMTERLFtmYeHh6Qx6qX4JRpiYrhKFSbi+fOZmZctYyJV745ZWa+vodJoeO5cFiugRQuM3Kn39u3ZwoJdXbngvRcePHggGiuYmpoWquQ4Ojray8urWrVq4i+hYsWKHh4eDx8+9PX1FVOdjRs3vnbtWkHjeF5gYKCrq6vYo06bZhMpsTexhnngQCbiHTs4K4tNTdnEhEvgXbF4L9OnD8KXX35JRI0aNcrMzJQkJJVKJXYMXbNmjSQDvjkWLVp0/fp1SYaKieEBA3jEiBd/9fvvn17iv/UWL15coLbw0dHRYnsamUzm7u6ev9Ti1q1b48eP167iadeu3bZt28TinUuXLikUCtFMi4icnJx8fX0xcV1CrV7NRFy7Nmdk8MWLLJezuXmht3jIzs4W21j8+OOP/Kzh8ahRowwSMRjP1Km8ahWvWfM00TBmDE+YwPXr8+nTvGULnz3Lr10Wlpyc3KhRIyJyM0ATpbi4OEtLS7lcLuqtAF4GiYYS7PJlViiWODuLK5DWrVv7+PiInrKxsbGisO5lFzY6Ey26DbcdITOvWrWKiCZNmsTMYu/VwMBA3YZ6//33xWWbgbbe003xSzSI6veePVml4nPn2NycZTLetauAz86dd+fnN8q6elWXnlI5OTmenp5yuVyUHEdFRb368RcuXMh9Ld6qVSsfH5/cKfzr16+LlYRWVlaFqkhPS0vz8fFp2rSpGNnS0nL8+PFXrlxhZpVKNXz4cCL6Vkzsv1Fq12YivnGDr11jIq5f39gB6eLTTz8lotWrVzNzx44dO3funJpauCXTmZmZosR9vsjQ6c3X15eIatWqVazesN5AGg0fPMi3bzPnq1uOjeUuXXjTpsL1wdVoNN7e3qKVg5OTU54O7WJFYu6tMTQaTUBAQJ5dM06fPi3dtwhGoFJxy5ZMxN99x8w8bhwTcWFTBKJBTL169TIzM8+cOSOTyaysrHCzV/rk3oI092VVoVy9elVMtEjSvyO3H374gZ51XAd4BSQaSh61mv39uVcvMa+SaG8/6r338s/5576wadu2bWhoqJ4vK2ZWzMzM3n777bNnz+o52iusXLlSO23cpUsXItKtNvn48Q1EVK5cOcOtUNNN8Us0hIbyO+9wZCQnJj69h5w+veDPzp13Z34u9a6PgIAAsXdXpUqVXrhfiOiX5vws0/ayHeyE9PT0Tz75RDzSzc3ttW32IiMjPT09tX2VKleu7OnpKTZETEpK8vb2rlWrFhGNHDlSz67yJU9mJpcrx+bmnJ3Nf/7JRGzIli2GI2YCN2/erNFo5HK5TCZ7dQPIFzpz5ozYu17/TrPZ2dl169YlovXr1+s5FOjvZXXL+jh//ny9evXyZzxVKtWXX355+/ZtfmWPBigFjh1jIra25vBwjoriSZM4JqYQT4+NjRUraPbt26dWq8Vm5pLvpgTFQe4tSHPvP1pYGzZsENMkFy5ckSo2jUbToEEDItr7igZdAMyMREOJ4+fH9es/Ld0sW5Y/+4xfeRd97tw5ce1atmzZzZs36/CCarX677//Fr0SRD275InRPERT7SlTpjDz6tWrPTw8dLjKSk+/dvGi1T//dP7rrz8NEKNeil+iQWvECCbidu24MBOqufPuzJJdlDPzw4cPxZFnYmLi6empLRgW7eVr1KghDsqyZcu6u7sXZGsJX19f0cKkYcOGL0sQiC6SZmZmuWuVRa+jkJCQiRMnigWxRFS/fv0tJa0JomREmcm//7JCwT4+xo5GFwMGDBDXSUlJSURka2ur2zgTJkwgog4dOryioD0zMzMhIeHu3bvXrl07e/ZsQEDA33//vWnTphs3bmgf4+PjQ0QNGjTQId8BkntZ3bKekpOT3dzcxBvI8OHDRQsGQfRoqFq1qvhq7h4NUJoMG8ZET/PyhSXqsJydnZn5119/JaK33nqrsKVY8KaZNGlSly5rGjRQP34szYCHDx8moho1amAZF7wWEg3FVO5aTWbeupV/+IGjovi3356u8fPy4oJdgSQlJYkVBGIqt+CnpOTkZB8fH7HCS3s3VwQzK6Lf5NSpU3UeQa3OuH69hVJJYWHFcfl8MUs0aI8tZt69m2vWLOya0dx5d2a9Uu/5qVQqT09P0RyhR48egYGB7u7u2vby9evX9/b2LtRl1s2bN1u2bCkS/PmXUcybN0+MbGZm9v777wcFBXG+MmYi6tKli5+f3xt9is192JRMolzq5MmTDx8+JKJq1arpNk5SUpLYUvHnn3/WfvLAgQONGzeuWbOmvb29tiNpftojMCsrS9TI/PlnscuMvpkkqVt+GT8/PzEvXaNGjcDAwNu3b7u7u1tZWYmjonXr1trkJpQ+9+6xpSXLZKzDmtDDhw+3bNkyJCQkOTlZ9A19c5PdUGBZWeoOHZiI+/dnSTakE+tGFy5cKMFYUNq5u0e0bRuzc2ecWs1t28Z06FCYIi4wnNxXNj4+PHcuBwfz8OGsVPLOna9p3f8ivr6+YiK2UaNGl1/XIf7evXseHh6i/T8R1alTx8vLK/+OhAayfPlyImrfvv3Vq1d1GyEs7BOlkoKDm6jVadLGJonilGjIfWyJRonFcjY1ICBAdG4Xa4HkcvmAAQMOHTqkW3t5se2cOLiHDh2ae87w2rVrdnZ27u7u4eHh/Gw/OW2PBgsLCzc3N507SpYe+Q+bEki07bhy5UpwcDARNW7cWOehxP4jNjY22mVau3btyp1QMDMzs7e3r1WrVpMmTdq2bevs7DxkyJDRo0fvF4nkZ+96zZs3N9C+xFBYUtUtv0xoaKioexfLdsQ/hgwZgr0D3gRz5zIRjxun+wizZs0iok6dOumzxwq8OR4+5AoVmIgLuM5Go9E8fPjw+PHjv/7669y5c997771dz/p2RUVFmZmZmZqaGmJnOyh93nuPiXjbNlapmIhR0FBc5K7VHDqU09KYmY8d46VLdR7yxo0bYv/LF07lCqJsXDsDZ5SJ2/DwcG0JRu3atd3d3QMDAwt+Mk1I2K5U0sWLlunpxXTHveKUaJDo2CoC0dHRCoVi1qxZU6ZMuVXYPt0vsmPHDjGpWLNmzTNnzmg/L/rwiWSbg4ND7jLmuNd2eX5DlJzD5hXE0puwsLDTp08TUceOHfUZTRQHvvPOO+K/SUlJ169fv3//fkJCwmunptPT08VuKbt379YnBihZcnJyPvnkk/Lly5ubm7u5ueVeRwOlW0oKe3g8t1tTody5c8fCwkIul587d07y2KC0Onbs6SbmL9xm98wZXrWKp0/nwYO5aVOuVy8rT/3dnDlzxCMXLFhA2GQXCgyJhmIqd0XDpEkcGcnMvH4969d4JfdU7rvvvhsfHy8+n52d7efn16lTJ/Elc3NzV1fXoIJvSSi1M2fOKBSKChUqaN/i6tWrN2fOnJs3L776iVlZDy5dslcqKTa2+G4P99I6aiOoUoWSksjamsLD6dkimeKpUqVKYhG7VIYNG+bk5DRq1KigoKBu3bp9+eWX33zzjbh0W7Fixc6dO8VyMicnJ3d399GjR7+iAP6NU3IOm1cQrRnKli0bEhJCROXKldNntFWrVh07duzAgQPr168fOHBgSkpKdnb2kydP7ty5k5iYmJ6enp6enpycnJKSkp6enpaW9uTJE/GP5OTkqKioqKio9u3bDxo0SJrvDUoCU1PTPn36rFu3bsiQIaJhG7whbG0pPZ0++4wOHKBNm6h1a7p8+QUPMzVNUqnO5//8t99+m5WVNX78eFEUA1AQPXvSt9/SF1/QmDGkVFKdOs999aefaPv2//5rYmJetWrNmjWr1qtXr27duvXq1RMHm0aj+e2334hItAsBgBJPJqNPP6XPPyd7ezIzIy8vfQaztLRcvnx5jx49xo8fv2vXrosXL65du/bmzZvLli0T65QrVKjw8ccfT5kyRSw6NpZOnTp16tTpl19+OXPmzPbt27dv337nzh0vL6/atY9lZ8fY2Q2xt3e1te1CJMvzxMjIr9TqRHv7ERUqFN/3QBkzGzuGZ65eJS+v/46tZztEvjmysrJmz54tVte3b98+IyPj2rVrRGRhYTFy5Mhp06a1adPG2DEWPyX/sGFmU1NTZs7Ozt6xY8eoUaNcXV39/Pz0GXPdunVim9XMzMzCPnfBggX9+/fHbcObxtvbe/r06VOmTBE9kOHN4e5OjRqRiQmdOUPDh9OQIS94jJNT/IUL5fN/vmrVqhkZGTdu3BArCgEKiJmGDqXdu6ljRzpzhmS5LqF/+43OnqW6dZ9+1KtHZcr891WNRvPo0aO7d+/+/fffK1eurFu3bmhoqEyW9xIcIL+RI8nPj7ZtoxEjyNSU5HJSq40dE7xQWho963avv7CwMNHqztbWNjU1lYiaN28+bdq0Dz74wLL43TWo1epTp07t2vX3mDH+RGHik+bmNe3tR9jbj7Cx6aDNOGg06VFRCypXnmNiYmesaF+rOE2Mt2hBW7ZIe2yVLBYWFsuXL+/du/e4ceMSExNDQ0MrVar00UcfTZ06VVSzwwuU/MMmJSVFo9HY2tqampqK0gY9KxqISKlUEpG1tbWNjU25cuVsbW2tra1tbW3t7Oysra2tra1zf7JcuXLik3Z2djY2NtWqVSuT+7IO3gxRUVFEpN1pAt4oo0fTxImUkEAVKlDv3i94QPXqOXZ2L/hCy5Ytv/76a7H0D6DgZDLy9aWhQ2nRIsqTJRg/nsaPJyLKyaGwMDp9miIjj167tvfOnTt37969d+9eVtbTxRT9+/cfOnQosgwApY2kl/S1atU6efLkN998U61atQMHDkyfPr13797F9n3DxMSke/fu3bt3J/opNfVMYuJfT57syM5+EBOzLCZmmbl59UaNziYl7WPOcXAYXa2aXkUfRaA4JRqEEnu7KJXBgwdfunQpOzs7KCjovffeEy0n4TVK8mGTnJxMz5ILuf+ts5UrV65du9bS0vLgwYMoTIACioiIICQa3mCLFlHbttSpEx058sKvVyZ68RcAdFOuHB07RidOUGgo1a9PAQFUtiwFBtLdu3TnDt29S+HhTyecu3WLP3nSW/vEypUrizUUH330UY8ePYwUPgCUGGZmZt9//z0RTZkyxdixFJzc1vZtW9u3q1f3zsi4npi4PSFhs4lJufj49VZWLeVyi7i4tZUqzTJ2kK9R/BINQCRaA9arV8/YgUBR0DZoyPNv3QQGBn7++ecymez3339HlgEKLjIykpBoeCOtWEFEZGdH8fHGDgXePDt20O3bT1uEVKlCixf/9yUTE6pVi+rWpc6dGw8cuFj0aKhbt66tra3x4gUAKGpWVk2trJpWreqpUj3OyLgRHb3IxqZz+fL/M3Zcr4dEA4CR5V4uoWdFQ1hY2PDhw7Ozs7/44gvtfjkABSESDVilBQBFbPBgWruWiKhnT8rIoHr1nnZnqF2bntV0NidqbswQoeQ7cYJmzaJly+jyZTp9mm7dIhsbOnCA3nnH2JEBFJTM1LSiRnOufv2DaWnKiIhZtWtvNXZIr4FEA4CR5U4uqNVqS0tL3SoaUlNTBw8e/Pjx4/79+4tNvwAKDhUNAGAU2hYh/fpRv37GjgZKKW3tzJdfkqMjzZhB1arRZ58h0QAlDHP2/fsfmpo6WFo2NnYsr4dEA4CR5V4usXLlypUrV+owiEZDkyenMds0adLkzz//NDExkThKKNVSUlJSUlJEQ1BjxwIAbxzRIgTAoLS1M0Q0dy7Z2tKVK0YNCKDw7OyGlSvnwqySy62NHcvrIdEAYGQi0aBto62befNow4ZKlSv/c+pUvD4tHuDNhE6QAGAUaBECRUZbO+PoSN9/T2+9Ra6uxo4JoPBkMnOZrGTsFSA3dgAAb7q2bdv279/f39+/cePGixcvjo2NLewIf/9NCxeSiQn9/rtl3bpYYw+FhgYNAABQ6i1aRErlf/8trlscApQSSDQAGFmbNm369+9fqVKlmzdvzpkzp0aNGiNHjgwICNBoNAV5+pUrNHYsMdOPP2KpIegIDRoAAKAUW7GC7Oyodm2Kj6cff6S33iIi8vMzdlgApRoSDQDGN23atIiIiICAAFdXV7Va7efnN2TIqGbNeM4cun//VU+Mj6dhwygtjcaOJXf3ogoXSh0kGgAAAABAQujRAFAsmJiYODs7Ozs7P3r06I8//rhwoe7u3SYhIbR0KfXtS+PH06BB2o2+nmKmYcPo3j3q3Pm//kYAOkCiAQAAAAAkhIoGgOLlrbfe+vrrr//+e3RAAI0aRaamdOAAjRhB1avTwYPPPVImo6lTqV492r6dLCyMFC6UCkg0AAAAAICEkGgAKI7kcnJ2pq1bKTqafHyoVSt6/JgSEyk0lIgoIICI6ORJeucdCgnB/kygL+w6AQAAAAASQqIBoFizsyOFgi5douBg+vdfmjqVmGnTJiKiXbsoMZFMTen3340dJZRwqGgAAAAAAAmhRwNAydCkCRHR4MHPtWOYO5dsbVHRAHph5ujoaCKqUqWKsWMBAAAAgNIAiQaAkmT0aJo4kRISnv73++/prbfI1dWoMUEJp9Fo1q1bFx8fb2NjY+xYAAAAAKA0QKIBoIRZtIjatn3uMzKZkUKBUsHExGTMmDHGjgIAAAAASg8ZMxs7BgAAAAAAAAAoJdAMEgAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGioUCCg4MnT57cv3//RYsWqVQqY4cDpQSOKzAEHFdgCDiuwBBwXIEh4LgCQ8BxVVhINLxefHz8O++8Exwc7ObmtnXr1kmTJhk7IigNcFyBIeC4AkPAcQWGgOMKDAHHFRgCjisdmBo7gBJgx44dMpnsxIkTRGRmZjZ27NiffvrJxsbG2HFByYbjCgwBxxUYAo4rMAQcV2AIOK7AEHBc6QAVDa93586ddu3aiX/Xr18/Kyvr4cOHxg0JSgEcV2AIOK7AEHBcgSHguAJDwHEFhoDjSgdINLxeWFiYnZ2d+HeNGjWIKCUlxZgBQamA4woMAccVGAKOKzAEHFdgCDiuwBBwXOkAiYbXs7a2fvz4sfh3eno6EdWtW9eoEUFpgOMKDAHHFRgCjiswBBxXYAg4rsAQcFzpAImG16tevfr9+/fFv2/fvl25cmUHBwfjhgSlAI4rMAQcV2AIOK7AEHBcgSHguAJDwHGlAyQaXm/IkCHBwcH+/v5hYWHz58/v3LmzsSOC0gDHFRgCjiswBBxXYAg4rsAQcFyBIeC40oGMmY0dQwng6em5cOFCZq5bt25gYGDlypWNHRGUBjiuwBBwXIEh4LgCQ8BxBYaA4woMAcdVYSHRUFCJiYkxMTGNGjUydiBQquC4AkPAcQWGgOMKDAHHFRgCjiswBBxXhYJEAwAAAAAAAABIBj0aAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAAAAAkg0QDAAAAAAAAAEgGiQYAAAAAAAAAkAwSDQAAAAAAAAAgGSQaAAAAAAAAAEAySDQAAAAAAAAAgGSQaAAAAAAAAAAAySDRAAAAAAAAAACSQaIBAAAAAAAAACSDRAMAAAAAAAAASAaJBgAAAAAAAACQDBINAAAAAAAAACAZJBoAAAAAAAAAQDJINAAAAAAAAACAZJBoAAAAAAAAAADJINEAAAAAAAAAAJJBogEAAAAAAAAAJINEAwAAAAAAAABIBokGAAAAAAAAAJAMEg0AAAAAAAAAIBkkGgAAAAAAAABAMkg0AAAAAAAAAIBkkGgAAAAAAAAAAMkg0QAAAAAAAAAAkkGiAQAAAAAASozExMTExERjRwEAr4JEAwAAAAAAlBh9+vRxcHBQKpXGDgQAXgqJBgAAAAAAKDHS0tKIyMbGxtiBAMBLmRo7AKCraWnX09KIqIGVlVOZMsYOBwAAAACg+EKiAaD4Q6LB+E4nJbWwsalibl7WFL8OAAAAAIBXQaIBoPjD0oli4VBCwl9xcclqtbEDAQAAAAAo1pBoACj+MIVeLIyrUqWmhYVMJjN2IAAAAAAAxZdarc7KypLL5RYWFsaOBQBeChUNxUKWRpOq0bCxwwAAAAAAKM605QyYogMozpBoML6JVat+ce9er8uXH2VlGTsWAAAAAIDiKz09nbBuAqDYQ6IBAAAAAABKBjRoAOlt20ZLllB0tLHjKFXQowEAAKDUCgwMPHv2rPr5ZsPM/OTJkzyPzMnJSU1NFf9u167d//73v6KJEACgUESiwdra2tiBQGmxdi2FhdEHH9CUKTRvHjVrZuyASgkkGgAAAEqn5cuX//333ydPnizsE0+cOPHPP/9s3rzZEFEBAOgDFQ0gsYMHadMmsramyZPp0CEkGqRSMhINUVFR8+fPT01N3bRpk7FjAQAAKAHUavV33333+PHjcePGVaxYMfeXZDKZnZ1dnsebmZnZ2toSUWZm5uzZs0NDQ5csWVK1atUiCxgAoCCQaACJValCSUlkbU3h4dSokbGjKT1KRqLBxMTE19c3Ozt7/vz5devWNXY4AAAAxV1gYODjx4/r16//+++/F/a5x44d2717944dO6ZOnWqI2AAAdIZEA0hm61ZKTaXhw+nzz8nenszMyMvL2DGVHiWjGWTFihXff/99jUazYsUKY8cCAABQAuzatYuIhg8frsNzXV1diWj79u3ShgQAoD8kGkAyP/5ICgVpNLR2LcXGUpMmZGlp7JhKj5KRaCCiGTNmyGSy3377LT4+3tixSK9Keno9lYozM40dCAAAlBL+/v5E9O677+rw3MGDB1tZWZ0+fToiIkLisAAA9INEA0gjNZUuXyZTU+rYkU6fpr/+osIXAMIrlJhEQ7Nmzfr06ZOWlvbrr78aOxbpHXFz+7Njx6yYGGMHAgAApcGlS5fu379ftWrV9u3b6/D0MmXK9O/fX6PR/PXXX5LHBgCgs8TExNjYWEKiAfR36hSpVNS2Ldnakuia3L27sWMqVUpMooGIpk+fTkQ///xzdna2sWMBAMjr9OnT9+/fv3nzprEDKY6OHTsWEhISFRVl7EDeFGLdxJAhQ2QymW4jYPXEm+bevXunTp3avHlzWFiYsWMBeIG0tLTFixfXrVt37969hEQD6O/ECaJnyQXx727djBlPqVOSEg39+/dv0aJFREQErnugWNi2jZYsoehoY8cBxhcRETF27NiuXbsOHTq0efPmn376aVxcnLGDKi5CQkL69+/fu3fvwYMH169ff968eZlYJmZ4ItGg27oJYfDgwTY2NmfOnAkPD5cqKii2tm7d2rx5888++2zMmDG1a9euUqXKoEGDFi9efOrUqaysLGNHB2+6rKys5cuX16lTZ86cOYmJiY8fPyaiu3fvGjsuKOFEFUO3bpSRQUolmZhQly7GjqlUKUmJBiJyd3cnoh9//NHYgcAbb+1aunqVBgygKVMoONjY0YDRpKenz5s3r0GDBhs3brSysrKysmLmtWvXNm7ceM2aNWq12tgBGlNcXNzEiRObN29+6NAhR0dHe3v7tLS0+fPnN23adOfOncaOrjQLCwu7evVquXLlevToofMgNjY2/fv3Z+YdO3ZIFxoUOyqVasaMGaNHj05PT7e1tR00aJCjo2N0dPTevXvnzJnTtWtXe3v77t27z507d8+ePaWyTxYUZyqVasOGDU2aNPnss89iY2M7dux45MiRoKCg9u3b//nnnwMGDHjw4IGxY4SSKT39aXKhc2f691/KyqKWLSnfxs+gFy5RMjMzK1euTET//POPsWORUsOGDYno5s2bxg4ECmzoUE5LY2Y+doyXLjV2NGAc/v7+tWrVEu+lLi4u9+/fZ+YbN27069dPfLJRo0YHDhwwdphGkJOT4+PjU758eSIyNTVVKBSPHz9m5mPHjrVo0UL8cHr27HnlyhVjR1o6LV26lIhGjx6t5zjbtm0jok6dOkkSFRRDcXFxzs7O4u/Uy8tL+/m7d+/6+vq6u7s7OTnJ5c9NStWpU8fNzc3b21upVKrVaiMGD6WcWs2bN68bMkQceK1atdqzZw8zZ2dn+/r6Vq5c2d7enoisra29vLxUKpWxwwXpXb9+PSAg4MmTJ4YYPP3YMTYxYScnZmZPTybi6dMN8ULFypUrV77//vtvv/123759Go3G0C9XwhINzDxv3jwiGjx4sLEDkRISDSXPpEkcGcnMvH49791r7GigqF24cKFr167i6qdNmzYnT57M8wB/f/86depocxB37941SpxGceTIkWbNmonvvXfv3teuXcv9VbVa7evrW7FiRSKSy+Vubm4xMTHGCrW0Egenn5+fnuOkpaXZ2NjIZLKwsDBJAoNi5dKlSyJVWqFChVfM38TFxe3Zs2fu3LndunWztrbOnXRwdHR0cXG5cOFCEUZdOMeP8+3bzMyHDxs7FCiUgABu3ZqJsmvWbNW0qa+vr0qlUqvVGzdurFu3rjj8pkyZMnLkSPHvdu3aXb581dhBv97Nmzfd3Ny++uqr/v373xaHpoS2buUffuCoKD5xglNTmZn375f4JQwvIyMjICDAw8OjUaNGRFStWrXy5cv7+vpK/kLffPNNtTJl/ObPZ+b4999nU1P++2/JX6U4SE9PDwgIcHd3r1Gjhvh7sbKyIqKPPvrI0Bm6kpdoiI2NtbKykslkISEhxo5FMuJN88aNG8YOBArsyhV+/32eNImnTeOMDEmGvH79+oQJE6oX3pdfflma/hyKubi4OHd3dxMTE3GR7e3trVKpsrOzf/zxR3d399yPzMrK8vb2Llu2LBGZm5u7u7snJSUZK+yicfv2bdFBkIjq168vbnSTk5PnzJmz9PnCn4SEBA8PD3NzcyKys7Pz8vLKysoyUtSlTUxMjImJiYWFRXJysv6jiV/osmXL9B/qFZRKZXh4+PHjxw36KpBb+ubNlSpWJKLOnTtHREQU8Fk5OTnBwcE+Pj5ubm61a9cWf+x9+vQRU80GdevWre+///6FJ8F33z1WvTrn/1i0iKdO5X79WKPhsWMNHSBI5MgR7tCBiZiIa9bk339nlYqZAwICWrVqpa0W9PX1FQU1+/btq1mzpkwmb9cuzt396f11MRQWFjZu3Dhx8SASdlZWVosWLcrOzpbmBXx8eO5cDg7m4cN57Fh++JCZecQIaQY3vPv3769atWrAgAHiHlioWLFimzZtiMjExOS3336T9hW7d+9ORLt27crMzLSysqpka5sSFyftSxjXnTt3VqxY0a9fP0tLS+2PtHLlyh9//PGCBQvKlClDRO+9955kR+CLlLxEAzMrFAoimjhxorEDkUBkZKRCoTA1NXV0dGzevHn+eVEojhYs4M8+44QEvn6dP/uMly/Xf8jIyMgyZco4ODhQ4Tk6OlauXFmSmwp4hezsbG9v73LlyhGRmZmZu7u7KOfbv3+/yLvL5fL86cKIiAiFQiFqj6tUqeLj41Mqi41TUlI8PT0tLCyIyNbW1tPTMzMzU61W//HHH1WqVCGismXL5s+z3Lx5c8CAAeIwbtCgwV4UB0lh3bp1RDRw4EBJRvPz8yOijh07SjJafhqNxtvb29zcvEaNGjKZzN3dPTMz00CvBU9lZ7O7OxPtbt9+woQJ+uT4wsLC3NzciKh9+/YSBphfRkZGnTp1XnaK7N79iLgtzfMxaxZPncqrVvGaNaU50XD/yqmg3euundiZk1XC/3aUSu7V6+kvr3JlXrGCs7KYWXP4cK/OncXvunbt2uvXr88zDZuSkrJgwSETEybievX46FEjxf8SsbGxHh4e4mbPzMxMoVDcuHFDoVCILYGaN2/+77//SvAyuZf0duvGY8bwhAlcv74EIxuMSqVSKpWenp5OTk65N0hq0qSJh4dHQEBATk4OM3t5eRGRTCb76aefpHrpzMxMS0tLmUwWFxd38uRJ8YuQanAjUqlUgYGBHh4eTk5O2p+nXC53cnLy8PAIDAzUXoKeO3dOvKMOHDgwQ6IZ0/xKZKLh5s2bcrnc2tpaLPotoTIzM728vEQ+ydzc3NHRUfwVjRo1Kjw83NjRwSvVrMlEfP8+nz7NRCzFAuYPP/xQ/LU/KKSwsLBOnToRkYeHh/5hwMsEBAQ0adJEvGU7Oztfv36dXzKB/0Lnz5/v8qyVcdu2bU+dOlWEsRuWWApRqVIlerYUIjo6mpnPnTvX+dmlYbt27U6fPv2yEV74swWdDRw4kIjWrVsnyWhpaWm2trYymUy0IJFWfHz8kGcLsDt06GBqakpErVu3Rn2fAT1+/PRezsKC167Vf7y0tDSxEurIkSP6j/Yy3333HRE1atTo7t27+c+DDx8mPXjA+T8SE3nqVE5M5FGjuG9fw0VnZHtXz4m+d/3G6b0H130TH3k/8k4JWEHwYlu2MBE7OLCnJ4u5k3//FYfrzu7dK1So4OXl9YpE5MWL7OT0NE3h6srF4RYhPj7e09NTXOrL5XJXV9fQ0FDtV0+cOKGdpVAoFLpPF2VkcGLic0t63323OFc0PH782M/Pz83NzS5X50UbGxsXFxcfH59Hjx7lf8rq1avFhM38+fMliSF3cmHhwoVENHnyZElGNorY2FjxIxWTYXl+pNqatejo6PXr12uTyxcuXBC9tHr06JGSkmKIwEpkooGZxSTYwoULjR2Ijvz9/bVrzFxcXO7cuZOWlubl5WVra0tEVlZWHh4emKAuvqRONCiVSrlcbm5url2wFxQUdOp1Ll68KB584cIF8fRbt27pGQnkl3vWvWHDhvv27WPmxMREDw+PPBP4rx5Ho9H4+fmJBXIymczV1fXBgwcGjFu7VtOQgoKCOnToIH447du3F9MyuYs4qlatWpAijpdVi0BhpaSkWFpayuXyKOl+9WIV9JIlS6QaUDh79qyovS9btuy2bduY+dy5c/Xr1yciS0tLb2/vIuhT9ca5ePHp+atqVZZkEpWZmRctWkREPXv2lGrAPKKjo8UatEOHDjFzVlZWnrNhUNCjU6c494f2/VgkGkJD2caGS1cb8f/sXT1n3y9fbPx61Pn9vjt/mnbyT29jR1R44oQVEcE//cTizf/CBX7nnadpg/Ll072909PTXztMdjYvWsRWVkzElSpxbKzBA3+Z1NRULy8v7Y20s7Pz5cuXcz/gypUrarU6PT3d09NTrCKsWrXqjh07CvcyOTns68u1a/OkSc8t6Z069WmiwdVVuu9JAj/88EO7du1y95dt0qTJzJkzjx079rIC/sDAwMOHDzPzxo0bxcITSebVvv32WyKaMmUKM/fp04ek6GpkLAMHDsxdD9K8eXMPD48TJ06IehBmDg4O9vLy6tKli/jJH81V83Pt2jWxzUKvXs6GuO8sqYmGgIAAIqpUqZLhij0MJM9Ny/7n27Q8evTIzc1NHC7VqlXz9fXFlVZxJGmiQaPRvP3220Q0d+5c7SftCrC/TsOGDbWPHzduHBENGTJEn0ggP09PTzHL6uDgsGLFipycnJdN4BdQSkrK3LlzRQmlra2tIfobMT+/VvP5XowSGjNmjDgUa9SosW3bNo1Gk56enrtQy93dvVAJ0+jo6PHjx4sTYaVKlY4Wt/rXkkCsdOjatauEY/71118kaW28WC5hZmZGRO3atcvdKjUtLU3sY01E/fr1ixQTdCCJjRuf3oF16cKS/mCTkpJE///AwEAJh9UaO3YsEQ0bNkz89+HDh3nOhvmXTty799wIa9YwETdrxs8uvEuVvavnZKQmZaQmbf32Q/8VnzPzrp8++2fzUlWOAZdeSynPCSskhN3cWC5nIra1ZQ8PTkws1Hh377KzM7u5GSba18nKyvLx8RE3byLFcP78+TyPefDgga2tbadOnYKDg5n56tWr2pS9i4vLC6f081KrefNmrl//6RHfvv3Tg7vY9qhgXr16tWi4YGlp6ezs7OXl9dr+Ynfu3ClbtqyFhYW/vz8zb926VZw4Jk+erOf9kTa5kJOTIy5aSujpZsSIEbVr1xY/Um9vb+0M1pMnT/z8/D766CNRcSZYWVkNGDAgT1HtvXv36tWr16mTn5OT9HVAJTXRwMyiJcz69euNHUhB5W5+Zm9vn6f5WUpKyqxZs8RikLNnz4pieHERdubMGeNFDS8iaaJh48aN4rYq9wr2/v37d36d999/X/v46OhoMRt88OBBfYKB3C5dujRkyBCZTKbdGeH48ePaZlQdOnQICgrSbeTw8HCRUvzf//5XoEuKwjL89qtHjx7t0qWLpaWlh4eHqLjz9/fXNodzcXG5l+dKv8DEjh5mZmYzZ84scalkoxs9ejQRLZX0l56RkSHmkyVZ8hMbG/vOO+8QkejIkJWVlZqa+tFHHwUEBGgf4+fnZ29vb2ZmcvZsj6QkvKdJITubW7ViIp40iQ3Q+uurr74iogEDBkg+cv6Kv9jY2Dxnw3HjTnTuzLk/8twyZGU9vSNbuVLyAI1v7+o5/j/P3LvK48Khzf4/z4y+f2PeoKrzXKqsntLr0c3iuyHIf/KcsLp0YSK2tuZZs1jX5nwaDR869NyGI0WwD4PYd1N7HuzUqdOxY8de+MhTp05Vq1ZNZOS//vrrjIwMtVrt4+Mj7njt7Oy8vb1fUQmYtHcvN2/+NMXQqBH/+ScX+1lJjUYjZmjWrl2bJn7dBXvW9OnTicjExGTDhg3MvGfPHjFVo1AodO54lTu5EBQUlGfqrgQRnSbkcrl2ccTdu3e9vb2dnZ3F/aZQs2ZNhULh5+f3srmfBw+y6tZlIm7ZkqXdB6wEJxp+++03IipfvvyYMWNWrFgRFBRUbDtIiSnQChUqaKdAY/OVc82dO5eedV/PzMzUaDRil2BxNebm5iZhHSzoS7pEQ1pamqil1z9lJprlNG7c2KD9Y98ooqne8OHDxX+9vb21b9mSlNi1bNmSiF7RvEB3ht9+debMmUT0xRdfiP9OnDhR/HBatWql/94BGo1G3NnGx8frHembpX379pInHMPCwipUqCBmRapUqTJ48OAFCxYcOHAgrvD3AP/880/VqlXFuVv0/rx69apYpVy3bt2cXNPN4eHhu3b9T6kkpVIWHj5NrUbKSVfaVVT377OBSqiY4+LiRC8P7Zo+Sbyw4k83O3cyEdvb63zrWpI8CA5aOaHrPJcq8wZV3bNyVla6QVZfSybPCevoUZ40iQu8E8rL5NlwZPp0A3YtUKvVfn5+YuUXETVr1uy1FwlJSUnaHazq1asnWpxEREQMHTpUDNK1a9f8E/6BgYHdunWb1KIFE3H16uzjU1KqdG7cuCHOIDo819PTk3JtPHHgwAGxM8Xo0aNzdPr2r127NmTIkOrVqy9ZsmTQoEEibaHDOEZ3/PhxImrZsqX47/Xr17XJBVNT0x49eixevFhUzbxWVBQ3a8ZE3LDh078USZTgRMOyZctyr0gRecH27dtPnjzZ19c3NfUmc7FI7x07dqxFixYiwp49e165cuWFD7t165bo4EVEDRo0EDtFpaamenp6itSdjY2Np6cn5veKA9U73dWtGmkiHqgvBalbNVKNHabzUGIWqE2bNvrvRJCVldWgQQMi+vnnn/UcCgSRaBg/frz4b3h4ePny5T09PQuyUrQgROGSQRINBth+NQ+RaPjhhx/Ef0+fPq3d7FOS8UUlNhINhSVS8FZWVqtWrZJk5d22bdtE0ueF67nq1KkzcuTIpUuXnjhx4tWtpNRqtZeXl7iq7tatmyjk8fX1Fdu8NW7c+OrV/B3sNDEx3hcumCuVFBzcJD39cv5h4TWKZBWV8M03S7t29Rk1Ssopnw0bNuSv+NNZ375MxFOn6j9SCZCTlXHEd9GCd2vMc6ny40dOt84eNnZEL2eYE1aeDUemTzfgPgxdu3YVb4mNGzfevn17wd97T58+3bRpU+2cosje+vv7i3oHS0tLT09PUf585syZnj17ilepXLly/K+/conaE3rNmjVENGrUKGbW4V5Gu/GEt7c3M584cUKUJAwePLiA08wZGRkHDx6cOnWqtkceEYlBunXr9rLak2JuwYIFRDQ115ta27ZtP/zwwz///DOxkAuOmDkm5mndW61afOeONBGW1ERDfHy82JNjxYoVa9as+fjjj5s1ayauYIioRg07pVJ26VKZmze7hIe7x8X5ZmbqWMSrD1EdLUKqXr16QRZjBwQEiHccInJ2dhZZqNDQUG1n+wKOAwZ19WpNpZKysu6npJxWKikkRMeKhvDwcGtra5lMJtW61l27dhGRvb19id6QpfjIk2hgZv1TDDt37uzZs+eKFSvYoImGgQO5f39OTuaTJ7l/f160SPJXyJNoYCl+OCtWrOjZs+fOnTsZiQZdpaena3sc9O3bV59FpxkZGdqhhg4dmpCQoFQqFQpF7h25czMxMWnSpImbm5u3t3dgYGDuS8mYmJi+ffvSs+US2dnZSUlJ77//vniim5tb6svXFaelKYODGyqVdPGiZVSUF3Mp3B3WgAy/ikorKoqtrFgmkyyhIWHFn3D9OpuasokJvyCpVUrF3L/x6+cu81yqfP0/FxcXl4cSzlRKTurmAnk2HDFcRcPKlSubN29esWJFHx8fHSbYs7Ozvby8xPtqpUqVxEV+fHz8uHHjxHxqw4YNu3XrJt4tHRwcvLy8Cr70oPgQb/irV69m5ho1arRo0SKmkDX6q1atEj+QBQsWMPP58+fFneCAAQNecfkRHR3t6+vr6uoqMuaCo6Ojq6urj4/Phg0bxE/+008/LYlbjzs7OxPRX3/9JdWAiYncqRMTcZUqXLBKiNcoqYmGCRMmiFvx3J9MSUk5fvz4Dz/84O096erV6kol5f64erX6vXuj1erUx4/XxcauVqkM2NI8Ozs7d7+3RYsWFXxZh+i+LuaOxHa74qYxICCgWbNm4i+kd+/ed6TKNUHhSZVoGDFiBBGNGTNGwtj69etHz/rogp7yJxr0t2LFCu0vyICJBrGfeE4O//nn052+pJY/0aC/yZMn07OSHCQa9LFjxw6xZXLFihVFfVxhhYSEiKU9YgOI3F+KiIj4+uuvxWrbVxA1ht9+++2RI0fEMsCKFSuKNR1KpbJevXpiNmnz5s2vDUatTg0L+0Scyu/de/+1j4f/GH4VVW4TJzKRZE34vvzySyJycnKS8AZg8mQm4oEDS94dhc40atWZPX9UcHQQ8xDr1q17Q7qMi0TDvXtsZ8c///zfho+Snw8//vhjIvLx8dFnkNDQ0F69eok3T7HTOTOfPHmyUaNGpqam5ubmNjY2Hh4eOkxTFxPVq1cnouvXr9+5c0dkTHT4u167dq1oFy02nrh48aJYlt69e/fc3QdUKpVSqfT09HRycspd/N6kSRMPD4+AgIDc+aD9+/fruRDDWLKzs21sbGQyWWFTNq+Wmsq9enGlSrxx43NdTnRTIhMNwcHBpqampqam165dS05O/uWXX154ZGRnRz154h8R4Rka6nL5cnlxQxgVtejJk33JyUeio6W8Ps7j2rVrzZs312cHu/j4eO3aLQcHB1GQLHo9VKxY0dzcfM2aNZKHDQUkSaLh1KlTMpnMyspK2j0Or1+/bmZmZmJi8qI6ZCgcJBpeAYmGYi4qKqp///6iiEChUBRqBszX19fGxkbMpOXZkk0rMzNz48aN7dq1e3W6gYjEdWGvXr0iIyPFfhOiSVWbNm1ybyn/WomJOy9fLh8X90cRzBaUHleucIcO/PbbPHGigVZR5fbgAZubs4kJF+YX+2KSV/wJcXHs7BxWtWqH3bt3Szhs8RcRETFs2DDxJ/n222/fuHEj91czMzPnzJmjQ9eVEmH3bibiQYMMNb7YAFjs0asPjUbj4+MjJhrLli0rWrMlJyeLd9H8zd1KEJFcKF++vEaj+f3334lo6NChug21ZcsWsRfYzJkzNRpNSEiIWGby9ttv37t3z8/Pz83NTVw/CNbW1mI7hleU82gXYrz33nslqM3ZmTNniKhx48aSj5yWxnfu5O1yopsSmWgQW5JMmzaNmefMmUNEbq/Pn2syMkJSU88lJ/9z+3afiAjPrCwD1o998MEHRDR//nw9x7l06VL37t3Fn0rLli1FyioyMlLMMkkRKehC/0SDWq12cnKiZwVg0poyZYq4rJd85DcNEg2vgERD8Zf7rr5JkyaXLl167VPS09M/+eQTcdJxc3N7ddsFISgoaPTo0bkbXOdhYmLy3XffqdXqJ0+eaJcBKhQKHfo3q1QJRTNbUKpUr85ELGlG+xU++oiJ+NNP9R3HEBV/gngTrlOnTrHtIG4427ZtE7VIlpaW3377rfaeav78+URUoUKFTZs2GTdCQ9iyhYl41ChDjS86rO3Zsyc1NdXJyalPnz76jBYVFeXm5vbxxx+L/yYlJYm8gxSRGo1oHiR2qP3www+JKE+hXKH4+/tbWFjQs/UOt2/fFmuschcvNGzYcMaMGUeOHMkqWCeL06dPlytXztzcdsKE8JLyxvD9998T0YQJEww0fp4uJ7opeYmGHTt2iEn+uLi4u3fvWlpaymSys2fPFvDpT57sYVanpp69d89gbznPEg1SvV/7+/vXqVOnc+fOototIyMDiQbj0j/RIJriVK9e3RAL7RISEsqXL09EYq076AyJhldAoqGkEBV2RGRhYeHl5fWKatXr16+LBXpWVlaFLQOOiory9PTUbh2vJZPJxO4S4eHh4lrQwcFBn7emopktKFWKNtEQGsomJmxurtcLBgYGymQya2traSv+BJVKJf4iFi9erOdQW7du7ZvPJ5/807cv5/kIC5MkdgkkJia6u7vL5fKuXbs2a9ZMbKB+9+5dMYdHRD169Lh165axw5TSunVMxJKeyZ/To0cPIjp27FhUVBQRVapUSf8xtTkgMbmo22YNxYdILixfvpyZxfafem5Ps2/fPktLy1atWokVExcuXJDL5ba2ts7Ozl5eXnkKdgpIqVT26XOfiPv04RLRBENsFL1lyxYDjZ+ny4lu5C+bgiiesrLo11+71Ks35Ntvv3V0dJw5c2ZmZubYsWPFhl4FwZx9+7ZzRMQsc/NahoxUSoMGDbp+/fqff/6ZZ5cNKKGSk5PnzZtHRMuWLRMd16Vlb28vtgL6/PPPs7KyJB//zWFt7dKxY3jFil8bOxAA3TVr1uzs2bOiBeOcOXP69esnrlzz2LBhQ7t27YKDg5s0aXLu3DmFQlGoV6lcufK8efMePHiwefPmDh06aD/PzOLi+6233mrZsmW7du3Onz//7rvv6vztaDSp9esfLFduQETELJ0HAcOpV49GjKDsbFq5UscRNBrNZ599xsxz584VySlpmZiYiL2KFy5cKA5Ond27d+9wPsHBUYcPU56PlBRpgtefnZ3d8uXLjx49mp6eHhwc/Pbbb0+dOrVChQqHDx/28/MrX7788ePHW7VqtXjxYrVabexgpZGWRkRkY2O48dOIyMbGRvsP/cc0MzPLPbghrhWL0okTJ+jZfkP3798vV66cdj8+3QwYMED8uYklD/fu3dNoNB06dAgICPDw8GjcuLEOYzo5OS1fXqtqVQoIoH79KDlZ9/DS0tJ2796dnJyck5Oj+yivpFarxdIJsQGw4SxaREql7k8vYYmGn36iAwcq2djsVCg+/eeff3bu3ClaLRZ8BDu7YRpNakrKyXLlBhkuTslZWlq+9dZbxo4CnmrQ4GjTpjfMzKpZW7du2vRGnTpbC/X0efPmRUdHd+nSRZSGGsLEiRObN29+7949cTkFuklPrxwUVD02tqaxAwHQi5WV1fLlyw8cOFC5cuUjR460atXK399f+9Xk5OT333//ww8/TE9Pd3NzO3/+vLbxcGGZm5uPHj06KCjo3Llzbm5uvXv3JqLt27cTkUwm27hx45kzZ+rUqaPP95Kaevrmza7x8b9bWupyKQlF4MsvadEi+uorHZ++bt26CxcuVK9efcaMGZLG9Z9evXoNHjw4JSXlK52jJCKi999//2A+Cxf2PHiQ8nzUqiVR6BLp0aPHqVOnPD09TU1NV65c2bBhwx07dri6ut66dUuhUGRkZMyZM8fJyen8+fPGjlQCJTHRkH9wCccsYg8fPgwLCytXrlzz5s2PHz9ORF27dtVuFKizrl27ik6QRHTy5Eki0i4211njxnTqFNWuTadOUa9eFBdXuKffu3dv7dq1gwYNKl++/Lvvvjto0KChQ4dmZmbqGdULXb58OSkpqV69eqLLpiHY2dGCBSSTUXy87oOYShePwcXE0PffExH98INMJjP5+uuunTvvdnEJrVq1aqHGMTN7i+h8Tk6EQaI0PFHXwMzGDqQEO3mSnJzIxoYOHKB33iEiOnGCqlal+vUpIID69KHUVMpdCmBunm1l9d98hKmpg0xmHh/vy5zj4DCaWaVSveCvMCVFptHk/TXdv39/9erVcrl8+fLlhitRETM2vXv3/u677wYNGpS/P7yFmdwk34vLZPLoe8ExYSE2duUbdXzH1NzCQOEBQNHr16/flStXxo0bt3///iFDhri5uf3yyy8hISGjRo26e/du2bJl16xZo91yUk/t2rXbsGFDYmJi5cqVjx079vjx4woVKpQrV07/kVNTA9PSzlSsONnBYbT+o4EhNG9OCQkUE0Nly1JAAHXsSNnZL3iYTJbKnLfmLiUlRVTk/fjjjwadxf3xxx8PHTq0fv36UaNGtWnTprBPl8lkzFy2bNm2bds+/3n7FxYLnz9Pb7313zVGcWBpaTlv3rzhw4crFIqgoKARI0a4uLj88ssvPj4+Q4cOnThx4pUrVzp37jxp0qTvvvvO1tbW2PHqrsgSDaJABomGPERyoVu3biYmJiIjoN2tUypSJRqIqHZtOn6cevemCxeoe3cKCKBX32VmZdHJk3T+vP/69TNDQ0PFJ01MTFq1anX58uXk5OQhQ4bs3LlT8nczbZGI1MNSjRpUuzYR0W+/UWQk/e9/eg1YkhINHh6UnExDh1LfvvTLL3T6tGnt2oOnTy/0OObm1Yio5CYaQH+7dlGdOmRjQ7///jTRsGMH3b5NBw7Qpk3Upw9NmECbN//3eFfXCA+P/2bhTE0dK1X63MqqpVxu8fjxLxERc1/4KqNHl799+wXpUDMzs/fee080gzScXr169ezZ88yZM02bNs3/1c9H9yqTHJLnk1Zl7Jt2Hdy2v1tC1P2jGxa1GzguKz2lSt3mBo0TDMHGwkKlUqXLZLvk8tHm5iNMTTe//klQ+lWsWHHv3r3e3t5z587duHHjkSNH4uLicnJy2rZtu23btrp160r7cvb29r179z5w4MCuXbu0PSb1kZMTk5YWJJNZlKyaxDdQ7lPqihW0d+8LHtOxo09Q0Mz8n7eysmrdurXhKv6EunXrjh8/3sfHp2/fvjo8vUqVKi9cdlGz5u0HD+rn//zo0RQf/981RjGxadOmESNGnDlzZt26dTNnzty7d29gYOD8+fOnTp16/fr1BQsWLF26dMWKFXv27Pnll1/E5tklUZElGtLT0wmJhnxyJxfE7bEkGQGtxMTEa9euWVhYFHwd/avVqEEnTpCzM924QX//TVOmvOAxsbF08CDt3UuHD1NSErVubRsaGurg4NC7d29nZ+dBgwZVqVIlJCTE2dn58OHD/fv337t3b9myZSUJTzBQymb0aIqMpJs3ycSEIiOpQgXSaRnKf0rM0omLF2njRjI3p8WLKTGRvvmGiGjpUrK0LPRQZmbViCg7G4mGN9rcuTRxIl258t9nBg+mtWuf/rtMGXJw+O/D1lZjauqg/TAxcbCx6RQb652SEmhvPyr3l3J/2NnZO+RjY2OTk5Mj2ghrNJoNGza0b99enJz0l5iY2KpVqw0bNogWLCkpKdnZ2ba2tvnDsLK2sSpjl/+DiC4c2nzh4CbHanVP/ul958IxSQKDIpaVlSVaSak1muzs7ByVytgRQXEhk8mmT59+4cKFJk2apKWlqdVqd3f306dPS55lEMQ2E2L1hP6SknYzq8uW7WtiUkaSAcFwtKfUsmWfO59qP2xsTPOfm8qUKZOdnZ2WlqZSqYjoyJEj7dq1Cw8PlyQktVrdvXv3xYsXZ2dnE1FSUpJarbayssofxmuVK1fuJZ83eeE3K5c/d41RHPz9999ubm7NmjU7evSoQqG4efPmsGHDkpKSPvvss+7duz948MDLyysoKKhVq1b379+fO3euRqMxdsg6srFZ17Xrp+XLnzLQ+Fg68Wra5EJ0dPTt27dtbW1bt24t4fiBgYGiQYOlDveEL1G1Kn3/PX35JU2ZQgEBREQnTz7NWO3eTU5OVLkyffghbd9OSUnUsiUNHNg1MDAwNjbWz89PoVBUqVKFiBo3bvzPP/9Ur149MDCwV69e8fqsQHgeM586dYqkTtmEhj5NLjRoQCdOEBF160b61l7r25KySGg0/PbbTMRz5jAzT5vGRNyzp46jxcVtUCrp3r33JYwwj9mzt3Xv7rF9u14tVV9GrPaxsLAwxOBviOnTWeynO2IEq1R861ahe6vqvH1JdHS0qB8+ePCgdpPLb7/9tvDfxAtMnTqVnm1suXHjRiKqVKlSUlJSwUfYu3pORmpSRmrS1m8/9F/xuSRRlVyGaFVdNLtOiNWPOTk5f/75JxG5YtcJyGf37t1E5OjoaNBXSUxMNDc3NzExEdsz6yk0dIBSSY8f/6b/UG+OZf37f9e+fdzL95A3BJ3blWdlZdWvX5+IVq5cyczDhw+nAm1hXiA///wzEdWpUycjI0OpVMrlcnNz89u3b0sy+KtJ0r9dWmfPnhXdWGQy2fjx4xMSEpjZ39+/WrVqRGRmZubh4ZGZmZmTk7N48WKlUmnseHUnVoRt3rzZEIOr1WqZTCaTyTQazZYtW4holKQbaW7atImIRo8eLeGYRUkU/tja2movSPpK/Tfw+eefE9HXX38t7bBTp3K/fqzRPN3cMfeNg5MTW1uzszN7e3N4+GvGCQsLq1evHhE1adIkMjJSktiuXLlCRNWrV5dkNC1x0TtiBDPz2LFMxMuX6ztmyVg6cegQnTpFlSvTF19QWhpt20YmJvTTTzqOJpZOGLSiITJy5IkTI/Vc1vIy6NEgIZmMfviB7Ozo4UMiokWL6Pnlli+WnByQmnoqPn6LqWlZIllqaqCtbdcCvmKlSpXmzp07Z86cGTNmXL582dvbu1u3bt9///3YsWP1bK8dEhKyZs0aExOTn376KSMj48svvyQiLy+vwhZrBfzxrVxu0qB934jbl/SJBwCKs8TERCJKSEjIzMyUcCIoDzs7uz59+uzbt2/nzp2ffvqpPkNpNKkpKcdkMhM7O6ybKATv69cfPnw4RqNxLKpXPHmSjh6lDz8s6Ck1N3Nz8x9++GHo0KFff/31qFGjlixZsm/fvk2bNikUCj2bqycmJs6fP5+Ili1bZmFh8dlnn2k0ms8//1zkNXRz4cKFf//9N88nbWz6paXlHVP0g9PhB2I47du3v3Tp0rJly+bNm/fbb7/t3bv3hx9+GDt2bNeuXT09PVeuXLl48eJ9+/atXbt29uzZxg5WLwYtCkhPT2dmGxsbmUyGiob8RDnD22+/bWpqKmEnhfwvIfkiAqK8VUhz55KtLV25QocOUdWqZFGwPmY1a9YMDAzs06dPcHBwz549jxw5on93f/GTFPuqSujkSSIi8YMUFQ0S/K70zVQY2PHjLHLNX3/Nf/319JPx8bxtm+5jZmTcVCrp2rW6EsT3EmPGMBFv3GiQwcWGhebm5gYZ/c3z6BHPmcOxsYV4ytWrNZVKysy8nZR0VKmkkJBOhXpF7YyNmLmVasZGrJ+cPHkyM3/99ddE1KZNG7VareewbzJUNLwCKhpKusWLF4vLgLt37xr0hdavX09EvXv31nOchIRtSiXdvNlNkqjeHKIn+YMHD4rsFZ2dmYgXLNB9BNE3YerUqcz8xRdfEJGTk5Oep7MpU6aQfhV/+X333Xf5r6s7ddpCxHk+rl3T53UM686dO87OziL4AQMGhIWFMXNgYKDYI1AmkykUCj1/UMYltr8JCAgwxODR0dFEVLFiRWYW+3y5u7tLOP6PP/5IRGLD15Jo4sSJRLRo0SJm3r9/v0KhuHDhgoTjp6SkmJqampmZpaamSjgs56tCyl3RoIOYmJiWLVsSUa1ate7cuaNnbKKFzbp168R/MzIygoOD9RyTmRs21BDx5cscHs5EbG/P+t9DFPceDTt20NSpxEwPHtDw4U8/6eBAI0fqPqZcXm3v3s7LltlJESCUeO7uVLEiJSQU+okymZlcrss0oJixIaJvvvkmLi5O9NbetGmTWHClG39//0OHDtnb28+bN+/hw4fLli2TyWTe3t5yeXH/Gy+2TpygNm3o5k3SqVPYS02aNCk9PV1cOpQmjx49EpsX6DPITz/9lJ6eLq5LwNBiYmLEPx6Kgi6DGTJkiIWFxfHjx7WvqJsrV86o1WZ2du9KFBcYxLlzdOQIlS374g5qBfTTTz+ZmpquXr362rVrc+fOrVq16oULF0QNuW5CQkJ8fHzEfkzp6ek6V/zl0bZt28n59O3bYPJkyvPhWGT1JIVXt27dw4cP+/r6Ojo67t+/v0mTJosXL+7UqdPFixe//PJLU1PTtWvXNm3a9ODBg9K+7okTJJr0BwT8twD+wAFpX4TIwEUBuQdHRUN+Yu69a9euRPTOO+/4+PjosMnLK5w6dUqlUrVt29ZAP6JFi0ipfO4zuvUsqFix4vHjxzt27BgWFta1a9cbN27oMEh6evrevXsnTZp06NAhIhJLn3JyclxdXTt27PjPP//oEtkzDx48uH+/bOfOs5s3p3//TbC15bffJgnuIfTPfxjU1Km8ahWvWfN0hYxUxKnFcHNlqGgo3URFQ1bW/ZSU0zpUNAiiAEHMbIuLHp1nbLKysho0aEBEy5cvZ2bRfe2DDz7QYSjQyrM8TysujidN4qgoCV7C0BUNWVlZ27ZtI8NUNIhVkYsXLxb/HTduHBFZWFh4eHgkJyfrPz4qGgztgw8+EJcBmzZtMvRrDRo0iIhWr16t8whZWVnlypWzt7eLjb0vXVxvhCKuaBg8mIl47lx9xxH1TaIAwdfXl/QrQMhd8ffVV18RKv5eJDo62s3NTbwttG7dWvRluHbtWseOHWvVqmVlZeXi4vJQumYfuU+yek4Xv1rz5s2J6MqVK9IPzXz16lUiatasGT+rvlm4cKGE48+ZM4eeVQSURL6+vhYWFqNGjcrJyTHE+OLn4+HhYYjBJZeamtqrVy/xblbwA/L+/fs+Pj6urq55dplt3LhxRESESqUaO3YsEVlbWx8+fFjn2MTb7ODBg5n5f//7n6mp5c8//6nzaFolINFgiPY5TZo0IaKrV69KOWgubm5MxM96/0sMiQajkyTRcP36dVNTUxMTk6tXr6alpYkGDb6+vjoMJUqgGzdunJ2dferUKZlMZmVlVZSFsqXSy7KcU6YwEZcpw15enJmp4+AajcbX17dMmTKOjo7du3e/f/++3vE+p2PHjlWqVBk0aFC1atVq1KgxV/+r/uf9888/1apVq1KlSp06dUSiJDIyUqFQiARH+fLlvb29VSqVboNnZWV5e3vb2tp26NBBkpwFvJC2WNrLy8vQr7VhwwYi6qlzD2fmAwcOEFGrVq0kjOoNUZSJhuBglsvZ2pr1b/2ZkJDg6OhIRLt37xYt5Ynoiy++0GGoXbt2EZG9vf3jx4/Dw8Otra1lMllgYKC+IZZSe/bsEcdM5cqVMzMzmVmtVv/6669lypQhIjs7Ox8fH41Go/8L5T7JTp/OY8bwhAlcv77+A+dVp04dItK/Xv2FRJOODh06MPPcuXNFnywJxxdNvpfr35TPSI4ePSpqDUaOHCk2w5JW586diWjfvn2Sj2wgaWlpYmmYvb19UFDQyx6mUqkCAwM9PDxEz3itJk2aeHh4+Pv7516IoVKpxo8fLyZ7du7cqVtgYoSlS5cys1jffe7cOd2Gyq1kJBru3WMHBw4Lk2CtiNCnTx8iOnDggDTD5WPQRIPYlsnMzMwgo0MBSJJo4OdnbMSFuA4zNjExMWIbiwMHDmi3sVigz+pYYObns5yZmfy//3FICDNzaCgPGfJ02W3durxrV6GvtwIDA7VnDnNzc5GHnj9/vugppaesrKwlS5aIoi1x209ETZs21SfPndu9e/eGP1vGJq475XL5Rx99JHopK5VKbcO2Nm3a6HApv2PHDnFRSETb9GnGA68jZvno2Up4g0pOTra0tJTL5REREeIzYWFhSqVSqVQeOXIk4HmHDh3ye0b7figaSc6bN8/QoZY+RZloGDmSiXjaNGlGW758ORHVrVs3MzNTbBJhYWERGhpaqEG0FX8rVqzgZ2ubx4wZI02IpVRycrK7u/uG569iIyMjtW/+Xbp0uX79up6vkvska7iKhrt374pT1fnz5yUempmZjxw5or2QM4SPP/6YiH799VcDjV8Ezp075+DgQEQDBw7MyMiQcOS0tDSxq1FiYqKEwxpaVlbW0KFDiahcuXKnTp3K/SWxQaabm5udnZ02uWBjY+Pi4uLj46M9gTJzYmJix44diah69eq3b9/WaDSfffaZuD308/PTISqxNcb58+cjIyPp2UYh+n6rxT/RoHX2LJcvzxMm6DWItuLro48+olxdNCSHREPpJlWiQTtjs2vXLo1G06VLFx1mbMRJSBQ7+fj4iDedtLQ03UICrdxZzqVLmYjNzPizz1iczo4e5RYtmIi7dv21Z8+eBSyBe/TokZubm9g4plq1ar6+vuHh4Xk+o89MUUBAgCjXIiJnZ+fg4OCAgADR0Et8Rp9Lw7S0NE9PT7E9gbW1taenZ1xcnPYzNjY2np6eGRkZGo3Gz8+vZs2a4kVdXFwKWK8REhLSv39/8axGjRoZLhEMQsWKFcVPe+jQoUXwcoMHD6ZnexYy8/8Kti2TaHClVqvFtuSXL18uglBLGZFoMFAnvNxCQ9nEhM3MWKqcRk5OjkiHiSVaoqp/+PDhhRrEy8uLnq/4s7a2RsWfzvz9/UXPfO3+lzoMcu0af/XVcydZbaLB1ZXVar51S4JQY2JiPDw8LCwsRLTW1tZeXl46l9q9jFifKNWWjRqN5s8//+zXr19WVpb4zMiRI4lo69atkoxvLBcuXChfvjwR9ejRIyUlRaphRZbHyclJqgGLjEqlEm9o1tbWhw4dUiqVXl5eXbp0yd1YrU6dOu7u7gEBAdqDIY+UlJQ8CzG++eYbIjIxMfn9998LFY9ILpQpUyYnJ2fr1q1E1L9/fwm+zxKUaPj3X7a2ZiL+7rtCP1etVvv7+zs7O1tbWz9+/JifLYk33NzI2bPs58dhYQYZHIkG42vWhMvY8oMwTdC/XMZW01v3emCxB0GdOnW0Mzbm5uYFn7G5ePGieMqtW7eSkpIqV65MRLrlMuEV4uPZ3Z1NTJiIHRzY25tVKs7J4Z9/1jRo0ISITE1NJ0+e/IqGAunp6V5eXmJ9nZWVlYeHR+5z7YkTJ1q3bi1OLR98EHTxYqEjvH6d3dwS5HJTcUl98OBB7ZcyMzMXL14spnTq1x84ezYXdpmzRsO+vuziMoeIZDKZm5vbo0ePtF8VuRIRfPXq1cXyn7S0NO33a21tnef7zSM+Pt7d3d3U1JSIHBwcvL29DbSYE7RUKpW24KVdu3ZF8Iqiz3/37t3FfxctWuTk5OTk5NSrVy/n5/Xp08f1GdEA/8yZM0RUq1atIoiz9Dly5Iipqamtre327dsN+kITJkzt3v3z6dMLs4fT64gbiTJlykRGRkZFRYlarYLXZ2kr/g4ePKit+Pv2228ljPAN9OTJE3d3d3FHVL9+/WPHjhX8uWFhrFA8PZm+7Ne4YgWbmbGHB+s8+f348eOZM2daWVmJm66RI0eKNjFE1L59e6nylVlZWT4+PuXLl69Zs6aZmZmnp6eeSwP27NnTqlUrEaf2RtHFxYWI/P39pQjZmG7cuFG1alUievvtt3VotqLRaM6fPz9//vxBgwZpPynuq2fMmCFppEVEpVKJaW9ra2ttcsHa2nrgwIGrV68OK9g9ZO6FGGfPnuVnqVWZTFao5TYpKSlbtmwRZV+5NwrRX4lJNDDz7t1sasoyGRe8gCg1NXXlypWiGoSI7Ozsjh49ysyrV68mok8++cQQcWq35JSoVDkvJBqMr2ZNJuL79/n0aSbiTjpWNDCzSqXSZ8ZG7Bs8a9YsZp4xYwYRdenSRZLFk5DfjRvcr9/TRRONG7OYcY+Pj588ebL2JvmFl/L+/v61atUS70Ivm+FXq9Vr167t2/cTIpbL+X//K+gi54QEdndnU1Mm4r59t3h7e7/wWicqKurjjz9u3z6aiCtX5t9+K+hKtKAg7tBBbHQU37t3/5ctKTx27FiLFi3E96gt8QgLCxOtScVdYv5ri5ycHHGhJpI1CoVC5ILB0MSWbELlypWL4BWTk5OtrKxyr54ouNmzZxPR9OnTDRFYqZeRkSF6tcpkMg8PD8lndIXw8HBRw3xbXABJR9TCfPzxx8y8cOFCImratGkBc5G5K/7WrFlDqPiTTmH3v3z4kN3d2cKCidjcnBUKjox88SNnz2a5nIm4USM+caJwUaWkpHh5eYnskkwmc3Fx0dYb7tu3T5TamZqauru767MbYnZ29i+//FKtWjVthl38o02bNrrt3Xjq1Knu3buLQd566y1vb28xiR0QEFCxYkU7O7vjx4/rHG3xce/evdq1a4sahAKe61NTU/39/RUKhfanTUTXnu0WK35ou3btMmTUBqTRaGbPnj1nzpxatWopFAo/Pz8dyj1yL8QQPbNWrlwpk8lkMplouFBYojBWqj7lJSnRwMw+PkzEJib899+veWR0NHt6cqtWt0XOtXbt2l5eXmINT3h4uPiV1KhR45oBtjZ+WbN6SYgecubm5hUqVJB+dCgg6RINzHz06FHtjE1ERESLFi127NhRkCdqNJr169e3atUqKSkpNDTUwsJCLpcbaBUiaG3fzrVqMRHXrBn63nsj7927x8w3b94cMGAAEZ08eTL3gy9cuCB2dSKi1q1b5/lqfikp7On59FLMxoY9PV/Vb1KtZl9frliRRW7Cze31uYkLF7hr16e5kjZt+NXhRESwQvH0gq9qVfbxeU1uQq1W+/r6ioJ8uVzu5uYWExPDzEFBQR06dMi/KPrIkSNicyYi6t27tyHejeFlrly5or1ok8vlL6vMlNa7775LRM7Ozh4eHh4eHjNmzFDk4+bm5pqPqMc5Udh7DsjFx8fHzMyMiPr162eIzVwmTZpEhul9cOfOHXF2O3fuXEZGRrt27X755ZcCpksOHDjQokWL27dvo+LPELKzs728vMTahCpVqrysZCY2Nvarr7ysrdVEbGrK48e/fnHN6dPctCkTsUzGbm4cF/f6YDIzM318fCpVqiTe1pydncWuGbmlpaV5eHiIYq46dero0LpIrVb7+fmJVnlE1Lx5c3FE/fPPP6IViMhiFPx28ezZs6JmgYjKly/v5eUlujUFBASIBqhUunrTPHjwQMz+NmnSJPJlqSbm69evL1mypGfPnuJdS6hRo8aECRP27NkjfkSZmZmWlpYymSyuIMdHMaZ/CWdOTs6YMWOIyMbGRqyS8/HxEfe/hd2P4/Tp06KjvFRXBSUs0cDM33zDRGxlxc+3z/jPpUvs5sbm5k8vpj/44Iu//vpLnJPOnTs3atQoMfco1hWLgmepjtHkZF6z5rk+uo8f8x9/SNbD8vz586K9KhEdOXJEmkFBB5ImGvj5GRvdiLtchUKhZyRQEBkZvHAhd+36nngn+eKLL8RVRe4OvXFxce7u7uKCxtHRsVC7MNy6xQMHPn0Ha9qUX/hur1Zzx45PH9OrFxd83y6Nhrds4bfeenoN9/33L37Yv/+yrS0TsaUlf/EFFzzJnpCQ4OHhIZpc2tvbe3l5ZWVlqdXq3PtH3L59W1vpUL9+fVz6F73Dhw9TLpLve/JCe/fuHTZsGBWetbX10qVLDTQV/+Y4efKkuNmuUaNG/nswfURFRVlZWclkMgOlC2fNmkVEnTp10rleb/r06YSKP8O4fft2jx49xJ+qi4tLeHi49ku56ws6d97m6lqI5gtZWTx//tO0+8CBN16x8Cc7O9vHx0eU5RNR586d//nnn1eMfPHiRW0/ZldX14KX0QUEBGiXNjRq1MjPzy/34ZSenu7h4SFuMWrXrp17AeMLXb9+3dXVVXRoKlOmjIeHh6gKCQoK6t27t3iVChUqeHl5SdtA0eiioqLEHEPDhg3z7JZ6/fr1SZMmiaoHwdTUtFu3bl5eXtqNAnNycsR2DPXr1y9fvryDgwN2p2JmlUolCrgsLCxEicf69etNTExMTU1f+7ackZEREBDg4eHRqFEjcWxLuF1oyUs08LPt5Rwd+caNvF/655+nV96mpvzee/zvv8y5ejSIo9bExMTFxeXw4cP5r4Z1DkmjYT8/rlGDiXjAgP/66E6cWKCZw9d6/Pix9qalSpUqPj4+uOQyJqkTDdoZG7HCqrDEPUPZsmWjoqL0jAQKLiIiQqFQiJyx+KsUu7JnZ2d7e3uLSyszMzN3d/cnT57oMP6RI9ysGb+iWnzePK5enXXaEZXT0tjLi8uW5ZfdbmRlcYMG7OLCd+/qMr62xENcTOzdu1d8PiUlxdPTU8yAif6RuvUSAz2JbW60imyrvzt37ng9s2zZMp98NmzY4JePVHumwKNHj8Q0qaWlpW67Kb/Q559/ToVv01hwycnJohvoli1bdHh6aGioubm5XC6XNr0CWhqNxsfHR3TQKFeunLe3d3Jy8vfffy/2GiCiQYMGXbmiSxIqJIRdXDLKlq1ORIMHD85zXyrqC7Tro1u0aFHAtHVOTo63t7fYdrFixYqv/VsICAho166deJUaNWr4+Pi8bBb68uXL2ke6urrGxr6gZUloaOjo0aPFxYONjc0XX3yRkJDAzMHBwdrUg4ODg6enZ2m9hY6JiREpm5o1a+bedlRU+BJR+fLlXV1dfX19xU+GmR8+fOjj4zNkyBDxWxNEH6j27dtrH/Ym02g07u7uRGRubi4Sc3/++eemTZte9viwsLDVq1e7uLjkbhJRoUKFCXruvPC8EploUKl42DAm4rfe4lyZU2ZmtZpbtWJ3dxbTM6mp/PPP3Lv30ykUOzu72bNn536fetnVcKH8+y+3b/80wdGxI48c+V8f3a1buXr1pzOHo0bp0opZ3LSIt29x06JDDxWQmNSJBn62CLlly5aHDx8OKIyDBw+Ks+yyZcv0DwMKK3edUdu2bb29vXPv+6DnBmDZ2ZyW9lzblxMnWCws3b+fMzJ075UlPHmSt6dM7vF1So88J/cuGL179160aJGoaxULK6Kjo/V9AdDVkiVLcicaSnpLcyi4zMxMsV86ESkUCv2rduPj4w26faDw66+/ElHlypX37NlTqFNkQECA2NHp008/NVx4wMwPHz4cMmRI7jtAIurVq9eZM2f0GVYsGRZbdGl3jtBoNP7+/trGQI0bN85TX1AQd+/e1c5BvvPOOy9sv/fvv/+Kxv70rL7gtclxkcUQPwF7e3sfH588gf3222/iblChUIjlA/fu3VMoFGI20cbGxsPDo2Tt16iDxMTETp06iXkasbsQM2dnZ8+fP//s2bNi2kalUimVSk9PTycnJ5F/EZo0aeLh4REQEHDnzp2CLMR4c2g0mpkzZxKRiYnJ+vXr8z/gtT9SyVtxl8hEAzOnpz9dady0KedJY4k/Z9GjwdGRibhLl425ezTkl2dPuILfGxRkDbOYORRFyFZW7OHBBU9Q5tma7kb+Eg4wio0bedUqTk7mqCheter1LUMKICkpqUqVKjVq1KDCq127doMGDYpmlTXkJzZ0FL87cTHUsGHDffv2STV+7rYvkm82nqenjOTj567vEHcj7du3/1cUm4HxiGsRrSVLlhg7IihSPj4+oqKze/fuMQVsPJtPUlLSoUOHRNn8O++8I22EeajV6jZt2uSuqS64atWqOTg46PxtQqH4+/vXrl17xIgR7du3l3CvhMjIyOHDh4tfaPPmzbX9ferWrbtx40a1HkuU/fz8REPiPPtfXrt2TbvEz9HRUds9oYDu3bvXr18/8fTu3bvfyrVoJCcnZ+7cuWKNSXh4uEKhEAsuROrhzSlNTU1NFUmcihUr5t4jPC4uzs/PT6FQiKVegrW1tbOzs7e3d/jzM8yvWIjxxtJuPPHzzz+Lzzx+/NjPz8/Nzc3e3l77I7WxsXFxcfHx8cm9j5jkZMysw7t2cZCQQN26UUgIbdtGz94KiIguXaKffqI//6TsbCKiLl1oxgzVu+/Kc+9Nml9OTs7q1as9PT2TkpLMzMwmTpy4YMECcXH8QpmZmd7eq37+eXpkpNzSkj7/nObOpVzlPHmFh5OHB/35JzFT796nx48PHzVqVO5kUh63bt2aMWPG/v37iahhw4Y//vijtvICiott2+jhQ3Jzo1xvhfp48uTJihUrTp48WdgnDhkyxM3Nzc7OTpIwQDepqaleXl5t27YNDw+fNGmSuG6QhLs7NWpEJiZ05gw5OtLjx2RrS0eP0u3bEg/u60szZkg8vhATE7NkyZLWrVubmZlpS0PBiMaOHSv2mxQmTJjwyy+/GDEeKHqnT58eMWJEdHR09erVd+zYoa33frXIyMjTp0+fOnXqwoUL586dy8nJIaKPPvpIoVCI+UnDefLkyc6dOzdv3lzYJ7Zp0+aLL77AKbLI5OTkqFQqsbuktD755JO///5bJpMlJiY6ODiMGzfu22+/FWvx9BETEzNr1izxltiqVasvv/zS399/8+bNGo3G1tZ28uTJc+fOfcUdwSts37598uTJjx8/trKymj179hdffCESfEQUFxe3dOnS5cuXZ2ZmmpmZjRo1av78+bql0kqu9PT0YcOGHTp0yN7eftWqVeHh4UeOHDlx4oR4YyGiOnXqODs7u7i49O3b92W/6NjY2L59+165cqVWrVpHjhypW7duEX4HxdQPP/zg4eEhk8n69+8fHx+vVCo1Go34UuPGjQcMGDBgwICuXbvm7rVpKIbLYRSB8HD+66//Sn/37+fevZ/r0VDYBe8F7N/m7+9fp04dIurefWeh1jAHBnL79urGjZ3+396dx0VV/X8c/wyLLAruprmmouK+r2kuWGZqi2GZYbZhamK2YfX9hlZfQyvDfm1YWahZoZahuWcuuYO7gvuKiggqyA5zfn8cnMgVmDuMwOv56A/F4cyB7szc+76f8zki0qFDhxuWk92wlVrBfgwUgdBQ9dZbau9eNXiwols+bGns2H/avtiiosEyuLJBRQPuTP369ct7JlCmTJn8lASjhDl9+nTnzp1FxNXV9fvvv7/hYzIyMjZt2jRt2rTBgwfrRgkWzs7OnTt3Hj9+vOFbWgI3pBP8TZs2BQYGitFtQRYvXqwrE/VqZTc3tzfeeMP6bvGJiYn+/v46Xm/ZsuWWLVsSEhKCgoL0szg4OPj6+pbmV1B6erpuiK6b9IuIk5NTt27dgoOD819gnpCQ0LFjRxFp2br18ZLVO7PQvvrqK2dnZ71e1dXV1cfHJzg4ODo6uoinUbyDBi1v6e+gQcrDQwUEqButtMqv7du39+jRQx/ubdq0ybul1q5du3r16qX/qWXLlqtXry7o4HqxmS4HMplMvr6+liqgm20OhzvRo48qvRf36tWqUBvVAvmkswDd9sUSBPj6Gj+4UsaPjzuTvry8hpeX12+//WbvqaFIpaenv/DCC/oA8Pf3z8zMVEpdvnx55cqVQUFBPj4+19yX9vT09PHxCQoKWrlyZYEqyQErpaen61RUKfXDDz+IyHCjt5FPTk4eP378t99+O3r06NjYWANHXr58ua5WcHR01O0MTSbTo48+amlPUJplZmZ+8cUXr7zyir+//8KFC6/oTlEFdPny5Qcfeqj/vHl9d+06yFuTUkqpo0eP/vLLL4sWLUrRFyz2UEKCBst2ksePK6NaJeplZvqTdcCAAdu3b7cUO1SqVKlAm9Vd78qVK0FBQTq9063Xly1bZulq06tXr135360OdjF6tNKNZ374QRWqhygA2MXy5cvr1aunP24aNmzYqVOn7777rkWLFpYPoO3bt9t7jihSX375pa6jbNSoUePGjfMmCyaTqWnTps8///zMmTNjYmLsPVOUXgkJCbrUVyn1xRdfiMioUaPsPakCSE1NDQoKqlOnjpeXl4+PT979sGGItJycMQcPtouM7LVjx55CpRUwXDHu0WARECDvvSejRklioixfbuTIaWlpH3300ZQpU1JTU8uWLZuSkuLs7Dx69OigoKC87TQK7fDhw6+//vrvv/8uIg4ODmazuX79+h999FHhdhpHkdq9W4KDpWJFcXaW4GC5WvEFAHes06dPv/3223opcoMGDS5evOjq6hoUFPT8888rpUJDQydOnHjhwoUad911fNCgMhMnytWt6VHibdy40dfXt2nTpqtWrXJ3d2/Tpk27du3uvffeXr166VZ5gH2dOnWqTp06tWrVOnXq1EcfffTmm2++/vrr12ygc+dLSUnJzMw05CIC18tS6u2jR/+6dKmco+P0hg1bXd39BPZScoKGixelfXtJSDB+/NOnTwcGBjZr1mz9+vXTpk2zbANhlFWrVk2dOrV58+ZVq1YdP368K5esxUhKyq1agALAnSEtTT7+OPvzz73Pnz/s6en53//+d+DAgcOHD9+6dauItG3bdtq0affdd9+lS5c++OCDZlFRz65ZI2XLSmCgvPaa5NlkGyXYlStXTp06deXKlTZt2hjYzhYwRExMjLe3d+PGjWNiYiZOnDhp0qSgoKCJEyfae164s5iVmnjixJKEBFcHh08aNOjk6WnvGZVqJSFoAAAAN7NokYwbJ8eOSc+eUbVrT58yZYru6qeUmj9//ptvvnn8+HER8fHxmT59etOmTeXkSfnPf2TOHFFKataUd9+VF16QW+7cBAA2FRUV1b59+7Zt20ZFRb3xxhsff/zx1KlT33jjDXvPC3ccs1LvnzixKCGhjIPDR/XrdyvUjiEwBOcNAACUTAcPSv/+MmiQHDsmbdrI+++3mzVrlmXvAN2QeP/+/e+//365cuVWrVrVpk2bw++9Jx4eMmuWbNokXbpIbKyMHCkdO0rBd94FAKOkpKSIiO6kmPfPwDUcTKZ369UbWq2aSSQ2M/On8+d/On8+KjnZ3vMqjQgaAAAogaZNk+bNZelSqVxZvv5atm2Te++9wcPc3Nz+85//HDt2LCAgoH/btg2DgqRhQ5kyRdq0kQ0bJDxc6taVqCgJCyvynwAAcrXOzIxv1y6sSRMRCfD03HTffX0rV7b3pHCHMom8Wrv2j97eCVlZdVxcOnl41GVluj0QNAAAUAK1aCE5OeLnJ9HRMnKkODre6sFVqlSZPn36/K+/lt69JTFRJkyQVq3kjz/E11f27ZNJk+T994tq4gBwLc9Ll6pERd2TmCgiTQ4f7rx2rdet39RQuplE6rm6isjyxMT5Fy4k5eTYe0alEUEDAADF1bp1kpIiIrJ0ae5X1q6VQ4dy/3zwoMyaJVWr5nc0x1at5M8/ZeVKadZMYmJk4EDp00cOHZJ335W775aff5aPPpJz54z+IQDgdvQ7nV4ukffPwC09W6PG67Vq3UNFgz0QNAAAUFwtXCgXL4qIzJyZ+5UFC2TsWFFK5syRBg0KNaiPj+zYISEhUqmSrF4tjzwi2dkyY4bs3i39+8vLL8vevUbNHwDyhaABheJoMjmYTCZ7T6N0YvsiAACKsbfeknLlZNeuf74yaJDMmGHdoM7OMm6c+PnJe+9Jx47i5CTLlsmcOeLuLmPGyPLl0ry5dU8AAAVB0ICCG3X33faeQqlG0AAAQDH24YdSq5b4+kpOjhw5IiLy1FMyapQkJlo9dKVKEhKS++caNeTyZXF3l5MnpUkTq4cGgIIgaACKG4IGAACKPZNJpk6VChXk1CkRkcmTpX17Q59g5Eh57TWpWFGcnSU42NChAeB2CBqA4oagAQCA4mratNw/hIdLbKx8/rnMmCEVKkiFCpKQYOgztWwpc+dKSgon9wDsgKABKG5MSil7zwEAAFhr8GC5917p318aN7b3VADAcCkp4uQkLi4SHS2pqdK69W227QVgVwQNAAAAAO54P/8sp06Jn59Ur27vqQC4Dba3BAAAAHBnY5NdoFihRwMAAACAOxub7ALFChUNAAAAAO5sepNdETbZBYoFejQAAAAAuLPt3i3Bwf9ssuvqau8JAbgVggYAAAAAxQGb7ALFBEEDAAAAAAAwDD0aAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQga8mXv3r1jxozp16/f5MmTs7Oz7T0dlBAcV7AFjivYAscVbEQpNWzYsL///tveE0GJwnEFW+C4KhCChttLSEh48MEH9+7d6+fn99NPP40ePdreM0JJwHEFW+C4gi1wXMFGFixYMGLEiLlz58bHx9t7Lig5OK5gCxxXBeVk7wkUAwsWLDCZTGvXrhURZ2fn4cOHf/rpp2XLlrX3vFC8cVzBFjiuYAscV7CRhQsXJiUlOTo62nsiKFE4rmALHFcFRdBwe4cPH+7QoYP+s5eXV0ZGxqlTp5o0aWLfWaG447iCLXBcwRY4rmAjs2fPFhFPT097TwQlCscVbIHjqqBYOnF7x48fr1Chgv5znTp1RCQ5OdmeE0KJwHEFW+C4gi1wXAEAgAIhaLg9d3d3y1Kc1NRUEWnQoIFdZ4SSgOMKtsBxBVvguAIAAAVC0HB7tWvXPnbsmP7zwYMHq1evXqlSJftOCSUAxxVsgeMKtsBxBQAACoSg4fYefvjhvXv3RkREHD9+fNKkSV27drX3jFAScFzBFjiuYAscVwAAoEBoBnl77du3f/fddx999FGlVIMGDcLDw+09I5QEHFewBY4r2ALHFWzKZDKZTCZ7zwIlDccVbIHjKv9MSil7z6F4uHjxYlxcHE22YSyOK9gCxxVsgeMKAADkE0EDAAAAAAAwDD0aAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAAAAAACAYQgaAKBUS0hISE9Pt/csAAAAUHIQNABAqfbiiy+2aNEiKirK3hMBAABACeFk7wkAAOxm2bJlv/32m4eHR/Xq1e09FwAAAJQQVDQAQCmVlpY2ZswYEXn//fdr1qxp7+kAAACghCBoAIBS6n//+9/Ro0dbtGih4wYAAADAECallL3nAAAoaocOHWrRokVWVtbff//dpUsXe08HAAAAJQcVDQBQGo0aNSojI+OFF14gZQBgvJ9/lo8+knPn7D0PAIB9EDQAQKnz88/Z2dljqlev9eGHH9p7LgBKnBkzZPdu6d9fXn5Z9u6192wAAHbA0gkAKF0uX5YmTeTcOZk1K83Pz83e0wFQ4jz2mMyZI+7u8tdfsn27vPaavScEAChqVDQAQOnyzjty7pzce688/TQpAwAbqFFDLl8WETl5Upo0sfdsAAB2QEUDAJQiUVHSqZOYTBIZKa1a2Xs2AEqkbdvko4+kalURkf/+VypWFBcXe88JAFCkqGgAgNLCbJYxYyQnR159lZQBgG3MnSsdO4qzs0ydKidPSo0asnKlvecEAChqBA0AbiUnJ+ePP/44f/78ZV0Hi+Lsq69kyxapXVv++197TwVASeXoKCKSkyNly4qTk4hIdrZ9ZwQAKHoEDQBu7MCBAxMnTmzQoMGAAQMGDhzYoUOH6Ohoe08KhZedLdOmiYh89pmUK2fv2eAmlFIZGRn2ngVghbzhAkEDAJRWBA0A/iUpKenbb7/t1q1bkyZNJk2adOLECS8vr/Pnzx86dKhr165Lly619wRRYGvXyqFD4uQkU6bIp5/KI4/Ye0K4keTk5BkzZrRs2XLs2LFdu3Y9evSovWcEFErecMFS3QAUrZycnCNHjth7FkCp5mTvCQC4U0RFRc2YMWPu3LlXrlwREU9Pz4cffnj48OF9+vTJyMh48cUX58yZ89BDD7377rtBQUEmk8ne8y2NzGbz2bNnx48fLyLlypVzdnZ2cHAoX768iJQrV8HZeYKIVKggJpO4uIi7u4hIpUry++9y8KAsXSqLFklYmF1/ANzI/v37v/jii9mzZycnJ4vIyZMnk5KS2rdvHxYWNnDgQHvPDiggKhpgVzk5OePGjTt16lRERETTpk19fX2HDh3auHFje88LKHXYdQIo7WJjY+fMmfPtt98ePnxYf6Vdu3b+/v5PPfVUuTwV9kqpqVOnvv3222az+cknn/zuu+/c9YUsilBoaOh777135syZ6/+pSpVqFy7EXf91T0955hlp0kQcHWXjRoKGO0l29q+///75F1+sWbNGKWUymXr27DlmzJhevXqNHDly/vz5JpNp7NixH3/8sbOzs73nCuTbsmXy4IPSr58sXSrPPCOzZskPP8gzz9h7WigV0tLSnnrqqYULFzo7O7u6uur0VkTatm07ZMgQX1/f+vXr23eGQOlBRQNQSmVkZERERMyaNWvZsmXZ2dkiUrNmzaeffvrFF19s0KDBNQ8OCwtr3LhxYGBgixYtnnrqqZ9//jkmJmbhwoV169a1x9xLqYSEhHfeeSchIeG1117r1KlTcnJydnZ2Tk5OUlKSiOTkOCcliYhcvCgikp4uaWkiIjpMfuopGTVKEhPtNnn8y/nz8v338tVXGytV+mvHDg8Pj6FDh7788sstWrTQ/x4eHv7ZZ5+9+eabn3322ebNm8PDw3mtodi4vqKBpRMoEomJiQ8//PDff/9dsWLFhQsXdurUacWKFfPmzYuIiNi+ffv27dsnTJigaxyGDRvm5eVl7/kCJZ0CUMpERkYGBARUqVJFvwm4uLj4+vpGRERkZ2ff8PHbtm1zdnZ2cXH5/vvvlVIHDhxo0qSJiFStWlXfiUXRGD58uIg88MADBf3GsWPVxYvq6FFVqZLKybHF1JBvmzapYcOUi4sSUSInH374//7v/y5fvnzDx27durVevXoiUqVKlaVLlxbxTIHCubxuXULbtieGD1dKHX/77VNdupyZNcvek0LJd/ToUb0+ol69etHR0Xn/KT09PSIiws/Pz9PT03IF1LRp06CgoEOHDtlrwkCJZ4egYc2aNYcOHdq/f3/RP3VplpqaGhwc/Pzzz3fp0iUsLMxsNtt7RrCP48ePW+oG27VrFxIScuHChVt/S3Z2dmBgoP4Wf3//zMzMhISEvn376pDi22+/LZqZl3Lr1q0zmUwuLi4HDhwo3AgXL6qAANW3r7HzQv6kp6uZM1W7djpfUI6O6pFH1MqV6nZvxfHx8f369RMRk8kUGBh4szQQuHOsX79eRO69916l1KhRo0Tkyy+/tPekUMJt3br1rrvuEpEWLVqcPn06Ozt7yZIl1z+MxAEoSkUaNJw6dcrPz09EmjVr5uTk5O/vf/78+aKcQKllKbu1NPDr2rXr1q1b7T0vOzh9+vSyZcsuXbpk74nYTevWrU0m09NPP713794CfeOcOXNcXV1FpEePHufPn78+fbDRhKGUysrKatmypYi89957hR4kKUlVq6ZE1OLFBk4Nt/TTT2rqVHX2rEpOVuXLKxFVoYIKCFDHjuV/DLPZHBwc7OjoKCI9e/Y8c+aMzaaL21i7Vg0dqjhzubWNGzeKSJcuXZRSY8eOFZHPPvvM3pNCSbZ8+XIPDw8R8fHxuXz5ckpKyqBBg0wmU1hY2M2+JTU1dcGCBU8++aSlF5XJZOrSpQv3QXEnspxLFDdFFDSkpKT897//dXNzE5Fy5crde++9Tk5OIlKpUqXp06dziWI70dHRDzzwgH4PbdOmzdq1a8PCwmrUqKHfUn19fY8fP27vORaRtLS0Z555plq1amXKlHF1dfX19V25cmVpq+zIycnRL8PCRS0bN26sXr26iDRo0GDPnj1KqR9//FEP2KNHj7i4OKPni1wffvihiHh5eaWlpVkzzqefKhHl7a2ysoyaGm4uNFS99Zbau1cNHqz27FHTp6uwMJWeXrjB1qxZo9+6q1WrtnLlSmNniltLS1NhYapVq9x6lA8/VEqpNWvUwYNKKbVihX1nd8fZunWriHTo0EEppbfImTZtmr0nhTvOokWLPvvss3vuuScgIGD9+vWFPh+bOXOmbpf7zDPP6IrLbt266UuMdevW3fbb09LSdI2Dh4eHk5PTk08+edsyT6BIXXMuUawURdAQERGhV5nqK9sTJ04opWJiYh566CF9AdyoUaPw8PAimEmpkpiYGBAQYAl0QkJCLDW3SUlJEyZM0Heny5Yt+/7776emZth3trZ27NixNm3a6J+3TZs2lsoOLy+vDz/8sPTcIdT7StSqVavQI5w+fbpjx446Mfz111+VUps2bdLXP/Xr199T3N4Bi4UTJ07oWy43LAQtkIwM1aCBElEzZxoyNdzSo4+qlBSllFq9Wn38sfXjxcXF6SVLjo6OQUFBOfTbsL1Dhy6+8kpuMYqIqlFDvfuuio1VSqmxY9UDDyizWQ0fbu9Z3mGioqJEpG3btkqp119/XUQ++ugje08Kd5bQ0FAnJyddqKU1aNDgrbfe2r59e/4HMZvNQUFB+tsDAwPNZvORI0d0m4Z77rnnmjYNt5WSkqIrB5ctW1bAnwawJaPPJYqSbYOGHTt29OjRQ78FtG3bdv369dc8YOXKlc2aNdMP8PHx2b17t03nU0rk5OSEhYVVrVpVRPQSlfj4eKXUunXr8p6Ynjx50s/Pz2QyNWjQtG7drLCw2y4WLq7Wrl2rV+41bNhQrxc4depUcHCwzr9ExMHBwcfHJzw8vMQX1yxatEhE+vbtq5R6/fXXu3bt+ueffxZ0EF0bYlk0npOTExsb26lTJ50+LFiwwAYTL9UGDRokIkOHDjVktLlzlYiqWTP3Yws2NHq00iHmDz8YtV4lOzs7KCjIwcFBRB566CHuvNlITk7OypUrfX19y5ev5eFhFlHt2qnQUKUrio4cUbt3q7Fj1RdfqK+/Jmi41q5du0SkZcuWSqkJEyaIyIe6CAT4dzrw5ptvrl+/PiAg4O6777YkDvXq1dM1DrceJysr68UXX9TB69dff62U2rp1a7Vq1fSxd/r06ULM7ZVXXhGRDz74oDA/GGAjNjiXKDK2ChoSEhICAgJ0VFm5cuW8t9OvkZmZGRoaqhvg07jBen/99ZdOZEWkd+/e+g7z4cOHfX19ReSbb7655vFr164dPHijvlfTrZvats0ek7al0NBQXVPXr1+/xMTEvP9kOZUsU6aM/o1Vr149ICCgBN+WnzJliogEBAQope69914RWbVqVeGGCgkJ0S9wX1/fK1eupKWlPf300yLi7e1d4vOaorRkyRIR8fT0jNV3Ua1mNud2JOTM3+Z27VI9e6revdVzzynr1rxcY9WqVTo8rV279oYNGwwcGRcuXJg6deo999yjPxTc3d3feWffjh1KKWU2q5Urla+vcnRUPj65m7k8+aS6/347z/lOs2/fPhFp2rSpUuqdd96x1ZWbLRct5+TkHDt27KBeGwPjXJ8OaDk5OTpx0AWSt00ckpOTH3zwQV2munjxYqXUsmXLdJuGvn373mwfn5tJSUnZsGFDTk7O7NmzReTRRx+18scEjLRrlxo6VI0ercaNU1euFK/Nw4wPGrKysizBgbOzc0BAQH5Wg+tgQtf5V6xYMTg4OCOjhBfzG87Sa1OffeoWOMnJyW+99ZaLi4u+Vsn7tm5hNquwMFW9uhJRJpPy9VUnThT57PNITU2dOnWq9csZ0tLU66//n77x/vbbb9+izDgxMTE0NNQS0IhIu3btQkNDr1y5YuUc7jQjRowQka+++kopVblyZRGx5vJ1yZIlFSpUyFsW27VrV0dHx5nU5RskNTVVX/AY20pt1Solory8zly4kHj7R8Majz2mRNQvvxg+8KlTp7p27aov4UrJGoqMjIy4uDibnhssWrRIN53RFXCffPKJjqcvXlTTpikvr9wFFK6u6tln1Zgx/+wai7xiYmJ0JfzRo0efe+45EZk4caLBz2HLRctRUVFdunSpWbOmu7t77969C1TMj1u4Ph24niVx0A2hNEsfB/2YM2fOtG3bVt8cioyMVEp99913+gpixIgRhbjVobfi2rdvX3R0tD6FtubHRGmTmZl5/vz5gwcPbtmyZfny5b/88sssAzf0/fFH1aOHCg1VV66otm2ViIqJMWxw2zM4aFi9enWLFi30+0KfPn0K2tY+Ojq6f//++tsbNWq0aNEiY6dXUumtK/Uqbnd396CgoLS0NLPZHB4eXqdOHX2l7efnd/aWwX9SkpowIXdz93Ll7Hm3UyfKVi5nOH1adeyoypZNbtmy07x58/L5XZGRkf7+/pYWxJ6env7+/ret3ytG9AKHtWvXxsXF6R/QygFjYmJGjRplqVfSS6VW0BvNILrquG3btoZvavjSS7PLli07fvx4Y4fFtdq0USLKNlv8ZGZmuru7i0iJX0Bx9uzZ4ODgWrVq9evXr3bt2rZbQX3+/Hk3Nzf90aNfdLt27RozJqts2dyIoV49NWWKio9XShX7jqoRERH9+vV75JFHgoKCUlNTDRz5/PnzAwcOdHNz02tjvb29586da9joV66ozZv/tWh5yhQ1e7Yy4k0yPj7+hRde0EuTqlatqpN0BweHZ599tnCl+LA4c+aMbpVlSQduLTs7+88//xw5cqReCKw1bdp0zJgxtWrV0knWwYMHLQsxTCZTUFBQ4eY2ePBgEdFbv5cvX15Ebn3CnF/FdqcAWMTFxS1cuPD777//9NNPg4KCAgIChg8fPnDgwO7du7do0aJWrVqWS4a83N3d+/btu2/fPgNmMG2aElH6bK1TJyWiNm82YNiiYljQoBf8W+4DWNPc8ZrGDbatYy/+7wI37LUZFRWlC+NFpH379vmvrT15Uvn5KZNJ+fvbraX2pk2bBg8ebFnOUKNGjcDAwAMHDuR/hHXr1F13KRHVoIHavbvAnSeSkpK++eabzp07W94y8vOhWCzoT9Dz58//9ddfcnX7MQPpUqZTp04ZO2zpdODAARcXFwcHh02bNhk++K5duxwcHMqUKXP48GHDB8c/KlZUIjbaDjExMVFEPDw8bDH4HWLNmjW+vr76XqWI6HcYBweHiRMn2qiOQ6c22dnZERERPj4+JpOpe/cDemlheHhuuHDggAoMVLVrqwIWaN8p9uzZc9999+U9La5Tp87s2bMN2YZpzZo1lhtOrVq18vT0FBEnJ6eXXnrJgJ2JIiJUvXqqUiX13HP/LFp+/vnc3XSsuDulm1vlrcZNSkq6ePFiYGCg7pzt7u4eGBhY0Jp8aPv27dP3vRo2bHjo0KECfW92draucdDrxUTExcWlQ4cO58+fz8rK0ms2nZ2dv//++0JPb/LkyXJ1VWnPnj1F5GYFFwVQnHcKgLZhw4ZXX331+hzhGs7OzlWrVvXy8urYseP9998/ZMiQJk2aiEjXrl0N+JyaOVOJqBEjlFKqXz8lopYutf5HKzLGBA1Tp07Vb8QeHh7BwcHphd2+yyIjI+OTTz7RWbKzs/P//d//GTLPaxX/d4GoqCi9gUK7du10mnDhwgVLd4waNWqEhoYW4ihfs0adO2fnltpxcXEff/yxt7e3fhmbTKZnn90bFnb7DnahoapMGSWi7rvP2tP7vXv3jh8/vly5cg4ODhcvXrRqrDvA6dOnRaRy5cpKqS+//FJEnnvuOQPHj4+PFxFPT8/StmmoLZjN5t69e4vIqFGjbPQUOhoeNmyYjcaHunxZiaiyZW00vO7t36JFCxuNb0dpaWlhYWGW5WyOjo4DBgxYuXJlTk5OcHCw/oDr1auXMXcd/+3MmTMTJ060LBQvX778f/7zna5UzcpSCxao3r2VyZRb4FDsWt9euXIlKChI5/h6O6pVq1bp+8z6RGLNmjWFHvzMmTO6w7SINGjQQBelJiQkBAYG6mcsW7Zs4asnoqOVj0/u7719exUR8c+i5V9+UfXq5f7Tgw+qAtbSKqW2bdumd1MSkd69e19zH/LEiROWn+vuu+8ODQ01vMSsZNu4caNeqtm5c2fdm7xwsrKyli9frl+bli1+R48eXa5cuT/++MOaGa5YsUJEunXrpq7ukzJp0iRrBlSqeO8UcEcrwjvEOpBt0aLFiBEjXnnllYkTJ06fPj0sLCwiImLdunW7d+8+depUcnLy9d946dKl2rVri8gnn3xi7SR++02JqEceUUqpJ59UIsrAAjHbMyBoOHDgwJAhQ/LeTjeKpaPk8OHDjxw5YuDIuYr/u0Djxo1dXFymTZuWk5OTmZkZEhKib1nrPN7K6P0OaamtlzPcdVeDcuXMIsrTU/n5qRvuIp+erl54Ifdkw9/fsLpWfRK2uViVKt2Q/ijt3r27Umrs2LFi9JZja9euFZFOnToZOGapFRYWJiLVqlW7poOpgU6dOuXm5mYymUpMwc4dZ+dOJaKaNbPR8AsWLBCRQYMG2Wh8uzh06FBgYGClSpX0Vd9dd90VGBh48uTJvI/566+/9MVGtWrVCt3O9nobNmx44okndPNgEWnevPlXX32lTyLPnTv3f/+3vVat3M8XDw81enQhrmftSa+m1DXnDg4Ofn5+lku+nJyc77//vmbNmvoHf+SRRw4ePHnr0a6Rmam+/nq5bsXn5ub23nvvpf27++mBAwd0R2oRqVWrVsFugaSkqKCg3IWdFSuqkJB/VklY+ihlZKiQEFWhghJRDg7Kzy+f1yH63oxeK1GzZk3d3Mpi586d48ePT0hIUEpt2bLFUijq7e1twB3v0mH+/Pn6TuSjjz5qyAodf39/Efn000/1X7Oysgq6jeX1EhMTTSaTu7t7VlbWTz/9ZMD76tGjxXqnALvQXXhiYmK2bd6sli1TP/2kvvpKffihevNN5e+vfH3VE08U5R3iv//+WweySUlJhfj2FStWmEwmFxcXaxdQrFmTe+9UKTVqlBJRX35p1YBFy4CgQZ8QP/TQQ9YPdUO6gt2alP2miv+7gG6Wo+/qpKene3l5iYiPj48h64LuqJbaSUlZ33yjunTJPc8TUS1bqunTVULCP4/5+GMlotzd1Y8/GvnUTz75pIhcc/5RHE2fPl1ERo4cqZTq06ePMcWBeXz99dciMkLXd8EKJ06c0C9tI/sJ3chrr72m7wzb9FlKL30jYsAAGw3/ySefWMp9izu9B9CAAQP0fWN9dz0sLEz36Dl8+PDrr7+ed0lmXFycj4+PiDg5OQUFBVlTnqqrJ1q1auXm5lapUiUHBwddPaErs3TS7ebmVrlyE1dX5eWlgoOVzdI/W9m5c6flIrlDhw5bb9Q0JCUlJTg42NPT082tUp06Wf7+Kp8LHdasUc2bq0qVLlSqVHnAgAFHjx692SM3btzYpUuXAlVPrJo/X+mAx8FBjRypbt2O5Px5NWaMcnJSIps6dvzggw9ucWWrO5frO+1lypQJCAi4/s7k/fffr680pk2bpruQRkRE6MaB+lxr165dt/0RSrOQkBAd4gQEBBi11ik0NFREnn76aUNGs9D/W3fv3n3o0CERqV69euHHOntWOTqqJk3Uk0+ql15S48YZu+tQSXL//fc3b968Zs2aZcuWtaxEcDCZlKPjP6f7lv/Kli3KO8S6qvT9998v6Ddu3779119/VVf7r3fp0sWqGih9x6JVK6WUeustJaL+97/Cj1bkDAgaZs2aJSJ+fn7WD3VDernUX3/9ZfzQefcLKZ7vAvqujmWDhhUrVixZssSowYcPV0FB6vPP76yW2nv3qvHjVdWquW87Pj7/9JJYskQ9/bQyvD+07jP09ttvGzxukXvppZdEJCQkRCml96y+xRlhIYwbN05EpkyZYuCYxdr1XU5SU9XJk2rHDrVsmfrxRzV9uvroo19Hjx79+OOP9+zZs3nz5tWrV9c3VD08PNq3b2/rRSiffPJJxYoVReSbb745cuTIkSNHLly4kJiYeIv9VvL+UGvX5t5QNO5dp2T59FMlol5+2UbD67qkadOm2Wj8onHp0qWQkBDLdpKurq5+fn47duxQeXYg1msl2rZtm/cbs7Ozg4KC9GXMgAEDEvKmzvlz6NCh8ePH60Wa+tLik08+0T3/UlNTZ86c2a5dO8vajUceeeSvv84Xu2VhFy9etGzpVb169duWEpw5c+add7bok/wKFdTHH6tbrIU9dUoNGZL7WdyokVq16vbtEnVhhW4spa/Vb9Y1PCYmRl/nH+3WTbVrp/LfqiY6Wg0a1KV5cxGpXbv2rFmzrv+R161bZ1mY4+Pjc7Nb4vv27bN0KK9Tp45uFpiRkRESEmLpE3nbTtulk9lsfuONN8S6Ho03pJeMNWnSxMAxlVJDhgwRkZkzZ5rNZl1RVfg9uf7v//6pdR81SjVrduMqXCila6y0MmXKVKtWrXHjxp06dcoaNCg3ppkwQU2ZokJDVXi4WrGiyO4Qb9iwQUTKly9/8eLFmJiYX375JZ9J2d69e52dnT09PU+cOGHIAoqcsyfTnrr3ytsPK6WurP/q/KfdLm4sZUGD3iPA8HDRolevXiKyevVq44euXVtVrKji4tTSpapiRTV0qPFPYWP6ctGaHQpvYelSJaL69bPF2NbKyFDz5ql+/dSPP9q8l4Suo3vsscdsMnoR0ltCLF++/PLly7pK0NhuaiNHhrZrd/8ff/xp4JjFWt4j8/nnlaVxfd7/unX7Uq6jN9jz8fGx6fT0UpoKFSro9VbXaNbsWV0iXrGiqlJF1a+v6tdX/v7/+qHGj1e67+fjj9t0psVV8DvvPNi+/crQUBuNP3DgQBHRd06Ko4MHDz777LO6rFqubiepu+Fcvnw5NDS0adOm+p9cXFx8fX1v2NV48eLF+qZ07dq1N27cmJ/nvSa/kKv7Geet9n/nnXf0P1WtWvWtt94ydllo0TCbzWFhYdWqVRMRJyenfO41ru3frwYMyH2Pql1bhYWpaxKWzEwVEqI8PHKrCIOCbpVHXM9SPSEizs7O/v7+eftEXr58efz48TpyrVq16vwffijEvvGrV6/WOyDq/7+Wm1V5G0nUr18/P53LV65caWlv2alTp7///lsZ2HuiJEpPT3/iiSf0K9fIDUeUUkplZma6urqaTKb8H8/5MWXKFBEZM2aMulry+fvvvxdyrB49lIj68UeVk6PuvluJKNYn3kRUVNSePXtOnz6d3+3k894hnjHDdhs99u3bV6626hg6dKgUpG3H448/LiJ9+vQxm83WL6DIzr4cGSk7dngqpeLjZ0RGyvHjLxZuKLswIGiYM2eO2LKjmK5d+fNPG1y9lCunRFRysoqIUCJq4EDjn8LG9KJKY3ddunhRJSYqs1ktW6ZE1AMPGDi2Tdi6l8SOHTtEpGnTpjYZvQjpPaJOnTq1adMmEWnTpo2x4+v6VjYxsMh7ZD73nBJRrq6qVi3VqpXq21cNHarGjlUhIVs+//zzX375ZfXq1bt37z5z5kxGRkZ8fLyrq6uDg8OxY8dsNLf09PTGjRuLyOTJk6Oiou66665atWrVr1+/UqVKFStWdHd3b9ny5etjkUGD/vVDjR+vnn5avfSS8vKy0TSLt4cfflhE5s+fb6Pxn3zy+x49Ru3cWbAW7neIpKSkqVOnSp6djHVxaXR0dEBAgKWMtmbNmkFBQbduIHfy5MmuXbvqy+ng4OBb1AFdunQpNDTU0mNY5xc3jCeOHTvWpUuXsLAw65tb28X27dv170REevTosXv37kIMsnSpatYs97XfpYv69dfcaqaff1aNG+d+3ddXnSxYP4d/xMfHW3pXV6xYMTg4OC0tLSIiQt8DvKaRRCFcUz3Rp0+fCRMm6HTDzc1NbwSez6H0nhR61wPdkkw3DouJiRk0aJCl5MF279jFy8KFC00mU8WKFdeuXWuL8Tt06CBGVzr/+eefcrXJVGBgoIi8++67hRlIr5twdVWXL6u1a5WIuueea4M6WOnKldyCwa5dDdnX9hr6JLl8+fKJiYkHDx50dHR0cXHJ/35q8fHxOuH95ptvlAELKMxRUU6RkSazOSsxMTwyUo4c8S3UOPZhQNDw448/ishTTz1l/VA3pJNFAxs+/cPTU4moy5fVokU2XUlrO7royNjdBCtXViLqwgW1fLkSuSMaNNyarXtJpKSk6I0As4rztukXLlwQkXLlypnN5u+++87w1+zly8pkUq6utnjPL67yHpmJiepGnYlzZWdnnzt3bt++fWvXrtXrhPWWEG+99ZaN5jZx4kQRadasWWZm5qZNmxwcHNzd3S9ct/758mWVmKjOn1dHjqgjR9TJk//6oahouLVWrVqJLTfH1feTi12/AE2fydWuXVtfsFm2k7TU1HTr1i08PDyf77pZWVmBgYH6NvXDDz98/SZB27dv9/f3t+QX9evXDw4OtuYi9o6VmJiYd+cpXe1f6NFyclRYmKpeXXl7qzFj/qlmuu8+5eVlzCZru3fvfuCBB/T/F8sylm7duunlM9bLWz3h7u4uIrduJHELes8OXXGm2zrom+qrV69u2bJllSpV/vOf/xgy5+JOr3mZN2+ejcYfNWqUGN3N+tKlSw4ODq6urpmZmeHh4SLSv3//wgw0fboSUboG9uWXlYiaMMHAeSLXpUuqdm0lYotODf369bMkTcOGDbOUuuSfLoU2agHFzp2VIyMlK+vC5csrIyPl4EHblrsay4CgYe7cuSIy1GbrDvSZx0pbLHAqX16JqEuX1B9/KBFVuPcUu9LH7slC31C4kSpVlIiKj1crVigR1bevgWPbxNixKixMjRypKlSw1VN07Ph4+/b/OXSoeJ7RK6WUWrdunYh07NhRKfXDDz80atToww8/NHD8LVv+6VYDbcQIdfCgOnr0ny4nOTnq449VYKB67jk1aJDq1k0NHrxbF35b6PrwjRs3ikiVKlXyf88t/w4dOqRLT//666/s7Gy9r8o777yTn+/VQYP+ocaPV1u2qBUrlG9xiteLjl6TUojeAflx4YISUeXL22LsoqDPwx6/mlF9//33+vgvX778uHHjDhw4UIgxFy5cqHuO1K1bd8uWLUqpjIyM8PBwS35xTfVEyZOVlaV72jk7O7/xxhs33HetEJKSVHT0v6qZzpxRmZmGjJ1r5cqVrVq1Gjp0aOXKlUNCQoxd1qeUOnPmjD5fmjp1qpVDnThx4qmnntKpVs2aNfVb9MGDB0WkQYMGRky22NPXZqE2WzX27bffisiTTz5p7LC6n/qOHTuOHj0qIlWrVi3EIBmv+akyZdTcuf+sm4iKMnaeyLVihTKZlIuLMqL/vUVkZKTJZPL09ExISDh06JCTk5Ozs/Px48cLOk7eBRR//PGHiLi6uhZuAcWePQ0iIyU9/VBKyrbISNm/v30hBrEXA4IGfbpg+AveQq+TWWFpp2YgvRPSxYtqyZLc7ZeLmzp16oiIsctHdZ/F8+fVypW53RbvfP36KRFV6PV0t3X//UpERUTYavwioLeEeOaZZ2w0/vffK5Hi2OfEhvz8lIi6ZrsSN7d/LUbo0uWUvv6pVq2at7d39+7d9QWSUko3opszZ47hE9PtzfT+IJ9++qm+MMvvCsk8YmKUm5uqUkUZulS2hEhISBARDw8PG42/bZsSUa1b22h4m/vwww9F5PXXX9d/TUlJ6d69+9dff12I4zCvQ4cO6eDM1dV14MCBevcWfat8/Pjxhw4Vy2Um+ffpp5/WqlWrefPmN+uwaA1bFw9mZ2enpaWl6JbyNmDsOtxt27bdd999r732mv7rsWPHRKRevXqGDF7cTZs2TUT8/f1tNP7OnTtFpGHDhsYOq5fi63J3vdS0oLfxMjNPR0Y67P37LnNqUs6O9crZWZE92dSzzyoR1bmzgcW0+gRJ33oZPny4iLz00kuFGMfABRT797ePjJSUlG3p6YcjI2XPnuJ0UDmI1XSsq5SyfqiiHl9vo6XUP38obmzxyymOv5UmTUREYmJsO/6BA7Ya39bi4uJmzJjh7Ox86dIlGz1FdLSIyNW1zxARWbdORKR16399ccIEmTxZvv1WFi6U9evl++8rx8fHZ2dnx8XF7d+/f926dR07dtSP1LuEfPXVV8bOav1vvy1ZsqRSpUpTp049d+6cXkPx+eef591cKp8aN5YuXeTCBZk61dg5lgTHjx8XEctmCjYYX0Tk6vLz4kf/fizr593d3detWzdy5MhCHId5NWzYcNOmTQEBAenp6Zs3bz537lyTJk1CQkJOnz49bdq0hg0bWj3xO9qhQ4dOnz49atSoZs2a2egpJk+WyEibjOzo6Ojq6qpXN9iC3nojOztbbxuh93sutPbt269Zs2by5MnXDG79PEuA9u3bi0ikjQ4UkebNm7u7ux85ciQxMdHAYUeMGPH111/rAijdSbSgP8LFi/NEzG417jW5eZyu9PPeVeWSvvE3cIa41qefSu3asnmzhIQYMt727duXLl1atmzZcePGHTlyZO7cuc7Ozm+++WYhhqpSpYp+k3nttddOnjwZEhJSu3btTZs2FeKdx9m5qpNTVbM5w8mpsqfn/R4evW79+LVr5dAhEZGVK2XdOklJERFZurQQP4QBCBqK2yX1vxE0aI0bi9gyCLD1+LajlPrhhx+aNm26fft2k8n0+++/+/v7Z2ZmGv5EBA3XOH1aTpyQihWlefN/ff3dd+Wtt+T55+Xhh+Xee6VxY7cqVaroF/I1hg0bVqlSpQ0bNmzfvt2waSUndx879kSXLl9//HHVqlUDAgIuX7782GOPDRgwoHDjffihmEzy6ady6pRhcywZrrmQtsH4IiUoaDCQi4vL9OnTa9WqFR8fP3v27Ojo6HHjxlmZXxQXNr3c/ewzCQqS4GCJjbXF8Dan+1bk5OSkpaWNHz9eZ6xW0rtOCEHDv7Vr187R0XHPnj3p6em2GN/R0bFVq1bqaq9uo9x///0jR47Ub0pNmjQRka+//vrs2bP5H+HixXkiUrGir4j50qVfM8pedOrU18AZ4lrly0toqIic+Oqro/ra2joTJ05USo0dO7Zq1ar/+9//srOzR4wYUegbBk8++eTjjz+elJT03HPPeXp66srid955Z//+/QUap2HDJS1aHEtPj05M/Kl+/fC6db+5/jHZ2XLhghw+LDExsmCBjB0rSsmcObJwoVy8KCIyc2bhfghrleqgYZeXV5S3d7bZnOTkFOXtfahKFcOfwtZs8ctp3/7n7t3nKJXk7HyoS5dP69T5wcDBbaRoKhpsN76NHD9+vF+/fs8++2xiYmK/fv2mTZvm7u7+zTff9OrV69y5c8Y+l37bJGiwWLtWRKRbN3Eo7Lusm5ubLtubMWOGYdN6912Jja2Tne37zDNHV6/+/ddfy5UrF2LFrYCOHeXxxyUtTd57z7A5lgxJSUnu7u6xsbFZWVm2GJ+g4XqxsbFpaWkiYjab4+PjTSbTY489ZuD4dz59LW27y91Zs2TGDLHNxaPNWbIAW4QClhTDwDGLL3d39yZNmmRlZe3evdtGT6GXFtqoaOLo0aMLFy6sUqXKihUratWqde+9906fPv3MmTO3/q7MzNNXrmxycHAvX/6h5OQ1WVlnXVwauLu3scUM8Y8HH/zrrbdaxMX5jRhh5Qtw586dixcvLlu27Pjx40+cOPHjjz86Ozu/9dZb1oz51VdfVatW7c8///zuu+/69+8/YsSI9PT0F154IT9TNZuvZGaeSkvbc+XK+nPnPnJ2vtvVtdGFCzPi4788fnzEkSOPHDhw3/79raZOXebpKc7OUrWqeHnJ44+LiAwaJJYzx7feklGjZNcua36OwivVQYPPkSPto6MviWwxm9tHR485f97wp7A1W/xytm9/df16P6VScnJiN2169fjxMAMHtxFdcVB8g4aMjIzVq1eHhISsWbPGkP+bZrN5xowZLVq0WLFiRcWKFUNDQ5cuXTpmzJgNGzbUrVt348aN7du337ZtW6FGluXLZdasa78eEiLBweLlZf3cS4j160VEune3apAxY8aYTKbZs2df1Im0lfbskS++EEdHCQ2VzMz6I0de8vL6ccoU3SOt0IKDpUwZ+f572bfPgDmWGH5+fvXq1YuKiurVq9fp06cNH79YBw1KqRMnTohI3bp1DRx28ODB7u7uW7Zs0dvE3nXXXbarw78z6Uto213uOjmJiBTT2/aWfMEWcQwVDdfQqyeioqJsNL4OGmwx/ubNmzt37nzixIkqVar079+/TJkyGzZseOWVV+rUqdOrV68vv/wyLi7uht948WK4iCpf/iEHh3K6tKFSpScNnx6u1zYwsELFihs3brTmromITJo0SSk1ZsyYatWqvf/++5mZmX5+flauf8y7gOLUqVOWBRQvv/xyeHh4aGjolClTJkyYMHLkyCeeeOKBBx7Yt++BvXsb7dpVdft25x07PPbsqbN/f8sDB3o4OlY4fz4kOXl9xYpDk5NXJySEXbr0+5Ur69LSdovEJSeLo6NUqSING4r+UH3qKVmzRvQtxQ8/lK++klatrPk5rGB9m4f58+eLyODBg60f6oZ0W44//vjD8JF1r5fz58+vXLlSRHyKRdvDf2vQoIGIHD582MAx7777bhGJjY1ds2aNiNx3330GDm47urPn+fO2Gl/vhXrd9n8GiI2N7dy5s7Ozsz5TqV27dmBgYCE63Frs2bOnU6dO+gXu6+t7/t+/lPj4+J49e4qIq6tr2DWNCm/p5EkVHKzuuUeJqMqVlWV3+TVrcjdXt0XD1uKrXZscEbVxo7Xj6PWin332mbUD5eSorl2ViHrlFaWUCgpSIqpZM0N6x48Zo0TUoEHWj1SibN26Vd+xr1KlypIlS4wdfM8eNW+eOnfO2FGLSGxsrIhUq1bN2GF1563Y2Nj169eLSOfOnY0d/86nb75NnjzZRuPfdZcSKa5HnW4CP2/ePL140NnZ2cDBk5KSxJbNX4udzz77TESef/55G42/Z88eEbnnnnuMHXbhwoU6nbz//vuTkpKUUikpKREREX5+fpblV7/91jMmpltcXEhm5tm83xsX9+muXdUTE8PN5uydO6tFRkpq6k5jp4ebWbFihclkcnFxKdy2DkqpvXv36n2+4+LiTpw4UaZMGUdHx4P67NZqTz75ZGBgYHp6ulJq4cKFTk5Orq6uN7wqj4pqHBkp+r/t28vt3l17374WMTHdz537SKmcK1e2HD36ZFLSqgsXfrh4cWFy8trU1F1XrpxPSvrX012zO5jehlzvDpadrQq1p1PhGRA0LFiwQEQe03vG2sBDDz0kIosXLzZ8ZH1SEhcXt2rVKhHp06eP4U9ha7q1lbGdtGvWrCkip0+fXrt2rYj06NHDwMFtp2NHJaLWrbPV+B06KBH1998GD7tx48YaNWqISM2aNV944YVatWrp9xpHR8cBAwYsXLgwsyDXgZmZmcHBwS4uLiJy9913//rrrzd8WFZW1ssvv6yfKCAg4NYbiSUnJ3///fcjRhw0mXI3SvDyUh98oPS+aTk5auzYfzZXR674+MwyZTe1f9n6q3j9Buvt7W02m60aaMYMJaKqV1eXLqlDh5SrqzKZ1F9/WTs/pZRScXGqTp34++4bt852r8Di6cKFCw8++KCImEymwMBAo3ZVLO7p3oYNG+TqbrtGuXLlislkcnV1zcnJmT17tthyM6w71n//+18Ree+992w0fs2aSkSdPm2j4W3rySefFJGffvrJbDbrl6SBg6ekpIiIu7u7gWMWa3qH5lY22/I6JyenXLly+mah/spPP/00ZMiQ+fPnp6amFm7Mb7/9Vt/vefbZZ68/9bp8+fLs2bMHDhy4Y8c9+jowKsrxwIE+8fGhWVnx+jFmc7bZnJWWdmDnzmp79zYq9E+HQnj22Wd1vly4z9nBgweLiN5H5pVXXhFDt2nLe/6mOzVUrVp1yJAh/v7+gYGBwcHBX3/99S+//LJ8+fL4+Mj09INZWefN5qy8I1y8uODo0adPngw4c2aSNTOZPFl9+aVaudKaMQrGgKDh119/FZFHH33U+qFuSHcpW7RokeEj33XXXSJy7ty5P//8U0R69+5t+FPYmt7116jITdPXuqdOnVq3bp2IdO/e3cDBbeeNN77y9m43c+ZsG43/3Xfqo4+UoRuJqlmzZulQs0ePHnFxcUqpnJyclStX+vr6WlpMVa9ePSAgYPfu3bcdbePGjU2bNtXnT/7+/pcvX77140NDQ52dnUXkwQcfvHjx4vUPiIyM9Pf39/DwEJFWrQLc3JSvr1q5Uuk3TEuBw6OP/rO5OnL9+qsSUUa8pWRlZemX5OrVqws/yoULqkoVJaJ++UUppfr3VyLqueesn57Fe++9rz/jrQ1EShyz2RwSEqJfa/fdd9+ZM2esHDA7u9inez/++KOIDBkyxMAx9+7dKyKNGzdWSr333nsiMmHCBAPHLxZ0g8OgoCAbjV+3rhJRVtTb2dOwYcNEZPbs2UopBwcHEbl1yF4gGRkZIlKmTBmjBizuUlNTnZ2dHR0dbbdf6b333isiy5Yt03/Vka6IuLm5DRgwICwsLP975ZrN5qCgIH36dNuXT3b2pQsXwg4dGhAVVeZq4uB08GDf9PTD8fHfnD//ZXb2JbM5OyPjmBU/HArs0qVLeh3oxx9/nPfrSUlJJ0+e3LVr19q1axcuXPjDDz+EhIQEBQWNGzdu+PDhgwYN6t69e9OmTV1cXFxcXM6ePauUSklJmTZtmrHXVlpmZqZeizF//vz8f1daWszp02+dP/9VTk7KmTMTo6M7JiUVcpve06fVhAk2rP6+ngFBw8KFC0Xk4Ycftn6oGxo4cKCIREREGD6y3mH77Nmzq1evFpFevXoZ/hS21qhRIxE5cLUOJi4uLjEx0cox77333vr161uqT++9916rp1kU/ve//0meXdkNZ+wtxKysrMDAQP2hqLeBuOYBiYmJoaGhrfKsqWrXrl1oaGiyLiT4t5SUlMDAQL3utGHDhvm/Il23bp2u6/Hy8tq/f3/ef3r44Yf185pMpu7du8+c+b1+5itXVFiYuu8+ZSlwaNXKtpurF0vjxysRZdDp/qRJk0TEV9e9Fc6ZM2rQoNz/Qz//rERUpUrGftRcuXJF1+YU6OOz9FizZo3+/VSrVm1Fod5EsrNVRITy8VGBgWrs2H/SvWtqJouFDz74QEQCAwMNHHPx4sUi8sADDyilnn/+eRH5+uuvDRy/WAgOTqtQIfE//zHs+vka9esrEXXkiI2Gt61nnnlGRH744QellI7yMzIyjBpc98VwcHAwasASoGXLliKy0foFhDehbzu/ohcDKqV3EOzWrZtlF6d8Jg4ZGRlPP/20iDg5Oc2YMSP/E8jOvmhJHHburHLmzAeXLv2RlLTq3LmpZrMBaxJRIPPnz+/WrZvO+9q0adOwYcMqVaroEpX8qFChQpkyZWJiYmw6Sd3bu2nTpgVKOS9fXhoZKQcPPqCUOnbs2chIuXBhZuEm8Nhjato0ZeOf8l8MCBp+//13ERlks+W5gwYNEpHff//d8JH1ad+ZM2f++usvEenZs6fhT2FrjRs3FhHLC2Po0KEVK1YMDg425OPz77//FpFu3bpZP1QR0BXmAwYMsNH4Bt5CjI+P79Wrl4i4urp+//33t35w3rICEfH09PTz81uZp+xp6dKluqGak5NTYGBgWlpagSZz8uRJ3VTJ09Mzb5w3bdq0u+++OzAw0JLpbtp05vnnlYdHbr5Qtqx65hm1Zs2/FoMhV/v2SkStWmXIYGfPntUtPE5bWbKckqKSknKrn7/5xpC55fXVV1+JSKNGjQq03qf0OH/+fN++fUXE0dExKCgo/+cZZ8+qSZPU3XfnvvTq11cvv5yb7vn4qFatlJ+fMuSuYd44de1apU/OjW4uoZRSL774ooh8+eWXBo75+eefi8jIkSOVUn369Ml7q7P0+PBDJaJsV8nRqJESKeolvkbR8dO3336rlNLr8I292a6vbynpsnjuuefEkAZDN5KRkXHffffp0/imTZsGBQVZzoSvTxzc3d114nD9//Hk5OR+/fqJSLly5QrdSSc7OzE5+e+kpL8OHuwbGxuUkXFyz576uo9DVlacVT8n8sdsNutg66GHHnriiSfyJggeHh61a9du2bJljx49Hn744WeeeWbcuHETJ04MCQkJCwv7/fff161bt3v3br20qtArL/IjMzOzfv36IvKLri3NtytXNkdGSnR0J6XUqVPjIyPl3Llptpmj8QwIGiIiIkRk4MCB1g91Q3rpxG+//Wb4yLrr4fbt23U/S1t0PTx8+HBsbGx+6t4LISMjQ/dT+Ouvv/Rfdd84/bZr/TmWvmzo2rWrAXO1vX379un7+TYaP+8txJ49VUCAKtz/1e3bt+tcoFatWlu3bs3ndyUnJ3/77bddunSxvHW2aNFi8uTJI0aM0H9t3bp1ZGRkYSakVHJysl6c5ujoGBwcrL+Ynp6u323PnDkTEhLSokWLGjU6ODoqEdWunQoNLZb3UYtIcrJyclJOTupG5SeFM2TIELGmIvqnn9TUqersWZWSoiZMUPfdp4yrGbbIzs729vY2/AKyJMnOzg4KCtJl23369Dl3u656kZHK31+5uuZGDI0aqeBgdfHiP+meh0fuv7Ztq6xvCpw3TrV0kHr8cWuHvZ7+qDK2Qebrr78uIh9++KFSSp/MHSimF8RW+OgjJaJsVtinHnjA19u73f79xbKkwd/fX0RCQ0OVUjq7TzL0Y0wvjyJmtfjiiy/E0IXuFpcuXdJ3a9zc3Dw9PS3nRW3atJk8ebKlP/rRo0enTJmib6VouuLJIjY2tnXr1iJSo0aNqKgoq2e1SLfrO3Sof1SU89VVFc6HDvWfOzfT6mpj3Iq+11i7du2MjIwff/zRZDI98sgj8fHxWVlZt/9mpdTNV14Y6LvvvhMRb2/vgi7aSkuLiYwU3fXjzJlJkZESG/uubeZoPAOChnXr1jVv3rxy5crh4eHWj5ZXTk5OWFiYh4dHrVq1vL29/za6Ed+zzz5bo0aN2rVre3h41KtXz9hljSkpKUFBQa6urh07dnRwcPDz87vtOWWBrFy5Ui/I9/DwcHNzCwwM1B+Zlq+LiI+Pz549ewoxeGxsrJ+fn8lkqlSpUv4vhu0rIyPDycnJ0dEx3bIdghGystSrr6pFi3LP7J98UnXqlHvSbzKp7t3VDz+ofC8DVHPmzHFzcxORe++9t3DHQ0xMTGBgoF7voKMBNze34OBgKyNYs9kcHBysr3+GDh2ampqanp4+b968/v376xUZIlKzZs0PPzxf+k7dC275ciWiOnWycpjvvvvuiSee0B9IuuqqRo0ahTmLDQ1Vb72l9u5Vgwcr/YZgg5RB0x/21apVM/YMvoT5888/dYegWrVqbdiw4foHpKenh4WFtW7d2ts7UkQ5OKgBA/5pj3KN7dtVw4ZKRHl6Kis/hPPGqePHq6efVi+9pLy8rBrzhnQb4+joaAPH1HsK/PTTT9nZ2c7OziaTqaDlXSXAtGn/bCxjCy1atBARG907sbXRo0eLyBdffKGUqlixoohYv9Q0L91xqRQedTezZcsWEWnWrJmxw+ZNB7Zv356dnb1+/fqAgIAqVapYAgVd42CJGo8fP65rHD7//HPLOHv37q1Tp45+sDX7fCmlDh166MSJkRcufG9p15ednXh1VYVzeLifiHJ0VN26qZAQFR9v1Y+P65nNZh0n6ZscOoT64IMPCjqO9VtX3EJ2drZe7T537tyCfm9WVlxkpOzcWU0pFRc3PTJSTp4MMHyGNmJA0KCUGjt2rH5t9+3bt3BXttdbu3atfivR19IiYjKZnn766VP6Jot14uPjX3rpJX0FVaFCBb1pTZkyZV5//fVLly5ZObjZbJ49e7Yul3BwcGjdurXOucuXL//RRx9Zv6ghJibG0vOmcePGPj4+ujysVq1as2fPNpvNmZmZISEhFSpUEBFnZ2d/f//z+V6MnZaW9sEHH+hfiJubW1BQUKH79xY93Rpz7969Rg144YLy8VEiqmpV9dJL/ywQ2LhR5V1E4Omp/P3V1q23ugK8bVOGAsnIyJg/f76OG37++Wdrhsrr119/1W2c69SpU7lyZT1bFxcXX1/fJUuW2K6crKS5ckUtW6aWLi30ADk5OZajRd/1jY2NLVu2bJ8+ffL/Wv7Ho4/mFtavXq1sFtVrZrNZ190888wzHDC3cOrUKb2a1MnJKTg42FJuffjw4ddee01fBYlIv34B77yjbvuhl5SkhgzJfTvy91eF+5C5dOmfOPX++21Y0ZCTk+Pi4mIymYwtXNcnmps3bz5x4oSI3H333QYOXlx89pkSUWPH2mr8Nm3aiMj27dtt9QS2FBAQICLTp09XeXY3N3B8/el5wz5KpVN6enqZMmUcHBwM/J3s2bNH33Zu2rTpiX93505PT9f7UJYvX/6axMGyAtTyTrt69Wp9ktylS5d46y7909OPRkaatm8vl5OTmpS06vjxF69c2Wz516ys+HXrDvr4KF0QKqIM3WwHSl3tFVirVq309HS9pVH58uVv2OP8tqzcuuIWfvjhBxHx8vIqxMhmc0ZkpERFlVFKXbgwKzJSjh3zM3Z6tmNM0JCVlRUaGqrfuPXde91Cv3BOnz6tb6fr40avqgoKCtK3gt3d3QMDAwv9tqWnqoNPZ2fngICAS5cuXbhwISAgQOcOlSpVCgkJKfQRtm3bdkt9e+fOnXU5wMGDB319ffUXGzZsWOjSjytXrgQFBenNCytUqGDpxbB169auXbvq8du3b79+/XqlVEJCQkBAgO6DUqlSpfw0boiIiNAVpyIyYMCAo0ePFm6e9qJX2SxYsMCQ0Xbtym18VbWqWrPmBg9IS1Ph4crHJ7ctYs+eE729vYODg68/d7lw4YJeM+zi4qIXiBpCd0LSlTjbt28PCgratGmTlWPu3r27UqVKenFH06ZNb/jj4DYs6xQKJS0tbejQofoSVMfzUVFROrgs5M44o0crvdPBDz8oG+wTfI3w8HB9/LRu3dr6A7IEy8rKCgoK0p90AwcO/O2333x9fS0FRLr5a/5zXrNZhYSoMmWUiHr88QsnT57M5zemp6vwcNW1q7rvvptuvm31Rhn/curUKRGpXr265SuG1L/obPTcuXN6Y+bisujPWF9+qUTUqFG2Gr9Dhw4iUlzqHK8xfvx4EZk2bZrK0wvcwPH1hWvhLm9KqrZt24qIPim13p9//qlDhPvuu+8W1SiWxCHvqgqdOOjN4OfNm6fLTx577DHr76WdPRscGSlHjz6llDp+/PnISImNDbr+YefPq6++Ur17qylTiq4bTimh35d0uYo+2Z40qZB7QNpoAUV2drbuqad3vSmEw0uaHv+puTnjSurBZQnv90j67mUDp2dTxgQNWmJiYmBgoO7lqy+DC1rEnpqaGhwcrFNhd3f3a26nHz9+fMiQIfq0rH//RT/+eOM60ltYvVp17pxdrVpLEenTp881xRfbt2+/7777RKR+/YeaNzcX9DV/4YIKCFDt268QkRo1aoSGhl6zCGfVqlW6VYmI9OrVa+fOnfkf3Gw2h4WF6Y/GG0Y5ZrPZcn6vY4Jjx44ppaKjo/OWPyy+yWVG3iqJJk2aFNMeWq+//nrZsmXr1q17s90Z8u+XX1TZsrkrn2+7peX+/eq111TLlh0sJQBPPPHE8uXL9QGwY8eOevXq6Ttsxl53LVq0SK5uC/L222/L1R2ArZGRkaFjuEIUd0GpG61TKIj4+Hi9ZZeHh4euZVi6dKk+W+rWrVshb7zs2qWGDlWjR6tx45Tty3r19uk6D3VwcBg9ejRn3rfw22+/6esT/dHm6ur67LPPFrrfyrZtqlWr7Nq1e1SuXPmPP/649YOPHlVvvqkqV869z1a1qrpw4QYP+/135eamQkIKN6Mb0PsZde7cWf81MTGxfv36AQEB1tR5JSUliYibm5vZbNY3joYNG2bQfIuTGTOUiHrxRVuN37lzZxEppgHiG2+8ISJTp05VSi1atCg8PNyai8ycnJzQ0NDx48dbvqKjrgs3fBWVVrrt66effmr9UOHhqkOHNS4uLoMHD87n+pS0tLTffvtt6NCh+rJCv802aNBAv9mOHz/ekP1N9+9vFxkpFy/+ZjZn7txZOTJS0tJuU3hfZN1wSoM1S5boE+y0tDR9+lG+fHlrVkXZYgHF7Nmz9Z3m/PeMuJZu4336tFq/XomoYrIhoDI2aNAOHDhguXvv5eWV/7v3ERERenPRvNfJ11u/fr2v7zh9YtSli9qyJV+DHzumBg/OPZ165JGYW+xhMW/evAcfPHv1kerQodsPnpmpPvlElS+vRJSLi5o8+bub3Z/R9RS64j3/jRu2bdtmqZLo2LHj5s2bb/bIlJQUS1Jzw8YNTz/99DXfcvHiRUs8VLFixZCQkMK/DOwtIyPD0mzW09PT399/Sz6Pjzyys3MCA3OLFJ57TuU/K9PLGfI2Nahbt+7gwYN1JU63bt2MvXmilEpKStKbEVy+fPnPP//U95CtHHPevHki0qJFC0NmWBrlXacQFFSgzQD27t2rM6l77rlH7zYaEhKiD6cRI0ZYu+oq/61ErKPbSXTv3t1SfnXXXXeFhYXRjP1mjhw58u677z777LNTpkyx/irlwoWE/v3761Pqt99++/rqPLPZvGzZspEjYx0c/inlDQu76Xvda6/lPszPz9qDKD09PTw8vHnz5lWrVq1du/aZM2eUUr/++quuvOvevXuh91XZvXu3iHh7eyulJk6cKCLvvPOOVXMtnubNU/XrqzfftNX4OgY16gZ1EZswYYJcbRdqpS1btrRv316/ynbt2qW/qE/tjO3GVdyFhoaWK1euQoUKAQEB69evL/SnwIcf5p6VTZ4cWYhB0tLSdI2Dh4eHh4dHo0aNpkyZUriZXEOvm9ixwyMnJ1XvQbhvX/PbfleRdcMpFTp2PNehw+IZM5RS999/v1jTNvsqYxdQZGdnN2nSRK7urVtIzZsrEbVnj9qzR4mo5rc/zO4QxgcN2sqVK5s3b64vt3r37m15I76h6OjoBx54QD+4devWa9euvfXgZrMKD1d16uQ25PP1Vbdo45KSooKClJubElHu7ioo6Pa39DIyVEiI8vRUIsrZWQUEqFv0bVi1SjVrlnse5uOj9u+/zeDqaumHPgUvV65cUFDQzUo/zpw54+/vr1v03X333fk8Wc+79uTuu+/WtRUZGRkff/xx3g4Xutdm3tSjBBTJp6WlhYeHW/pW6FPP/Nf/X758edCgQT16fOnkpK5uv1BgsbGxwcHBDRo0EBG9nqhZs2ZGfapdQy/zXrRoUXp6upubm8lksmbVklJK7/Nko/2oSoW86xTat1c1a6p58/LzfStWrNBFoZ07dz537lx2dvbLL7+sT2SN7VNra8uXLxeR+++/Xyl14MABy1Y4PXr0MLB/Cm7BbDaHhITo3kA9evSIjY3VX798+XJoaGizZs1EpGPH911clK+vyk+T5TlzVLlySkQ1aVKIMh2llDp+/PiECRMs3dp0Gl6jRo01a9YopbZu3aojtipVqhSunu7w4cOjRo3673//q5TSe/HMmDGjMBMtzvLWY9tIz5495epGV8XOf/7zH/3WZE2/Rr0oVee/+qxMXW3+7e7uXqFChfvuu6+YNsu0hYyMjODgYLnqnnvuefPNNwtUsZWdrcaMyT3bnzrV2vmkpqZu2bLFwNZj52KnREbK0aPDlFLHjz8XGSlnzty+aP9m3XCys4vr3rF288cfSkRVr65SUlK2bHmheXNPT0/rm7wau4Bi7ty5ItKgQQOr7uPee68SUevXq9OnlYiqWdP6iRUNWwUNKn+NGxITE/P2EShQc4TkZPX227k7e3l4qJtdSD7wQO471NNPq6unW/ly9qx67jml7/kMGXLjxxw7ppyclIhq3LjArd/yln5c37hBN3TUJdNlypQJCAgo6CrWGzZusNiyZUunTp30v/bs2fPWSVBxpHdn0K3d9e9wwIAB4eHht3id79mzR/dCv/vumuvXW/s5VKtWLRGZNWuW3uGpS5cuVg54Q0FBQSLyyiuvKKX69u0r1vWGPHXqlKOjY5kyZazsjVSqWdYpjBypWrWylFHdegXON998oy8LH3/88dTU1OTkZN1wxMXF5ccffyyyuRvi+g2Pw8PDdaCp2+LQLK1orF27Vrf2qFq16tdffz1y5Ejd5VcXWwUHf1SgV/m+fappUyWiypZVP/5YgHrj9evX+/r66k95EWnbtm1oaOjx48f1+5Wjo2NQUFBOTk58fLxOOU0mU2BgoDX3kfTF8MqVKws9QjGVtx7bFk6cOKGPqFWrVtnkCWwsMTGxR48eIlK7du1C1FjpezN5O3zps7Kff/5Zf9w7ODjolf9OTk6jR48uAXduDJGTk/P3338HBATog0dr0KDB1KkRt20qmpamfH1zi4WNa3ttqM6dsnzapx5YYjZn7txZKTJS0tJuf7/xZt1wJk9WX36pSt9blxW6dVMi6pNPlFKqf38lct6gu3oFXUBx8eLFY8eO7dixY/Xq1QsXLrR8PScnR993nzlzplUTGjBAiahFi9SVK7mfxMWEDYMG7WaNG/Rbto4hnJyc/P39C3dtc+qU8vNTzz13094qf/6p2rZVhS70i4pSvXqpmTP/daMg7/hvv60+/lgVem3pqlWr9H5RuvRDN25YuXKl3o5eRAYMGHDkSCH3rNaNG/T+PSaTydfX99ixY9f32izk1IuD7OzslStX5j3NrVmzZmBg4PW/0oiICH0/uXXr1jdbtlMgOsdZt27duXPn9MFv/ZjXW7dunVxd6aDvG7xoxfLcSZMmiciTTz5p3ARLK/0GkZOjQkNzi6N0PdV1yx/MZrNOi0QkICAgJyfn9OnTurV75cqV161bZ4fJW0evvhk8eHDeL168eNFyG7BWrVrz58+31/RKlfXr17dq1Uq//+hjrFu3brfOW28hNVW98IJeHPqsn5/frfeMSEpKCg0NtRQ2lilTxtfXN+/Ff3Z2dlBQkK7X69OnT1xcnN5kVx8kPXv2LPRaM92r6FB+1j2WIGbzv+qxlTKyH0t6evoHH3zg7u4uIm+88YZh4xa5VatW6VeEiHTo0OG29bMW27Zt69ixo+VUTV97HDhwwFKN27Zt240bN+rO4vp8Q9erstulRU5Ojt6HUicOLVuuF1F166qAALV+/Q16riUk5F5FVqqk7tBPwiNHcjceS0szL1+W1bf9hZm+1ox3+rSaMOGm901xrWXLlIiqUkUlJ6uoKGUyqXLlDNw+9IYLKD777LMXXnhh8ODBffr0adu2bf369StWrGipodZMJpPlQ/aXX37R4b61q1/9/JSImjVLKZXb9rmAbRDtxeZBgxYTE6Nv0ImIl5fXxIkTLe/1vXr1sr7MLDv7Vr1VrF8afM2NAmN7t2RlZf3f//2fbiPk6Oiob6qLSLNmzQy5b5CcnPzOO+/oTgGurq56yYa7u/ukSZOK0daVVsq7nEHfefDx8QkLC0tNTdVnt/p8d+jQoUbtuPbMM8/I1fJdvVmdlYsabigrK8vDw8NkMp05c2bbtm367axwQ5nNZv37KYV3Am0rNlY98URuaUPz5nEbN+b9x82bN+sqEr14b8eOHfrmmJeXl2VHruLlxx9/FJGnnnrq+n+KiorS3aF1hGrl1uW4rXfffVdfxn/33Xc1a9Zs2bKl9Snq7NkH9W3bNm3a3PBi/sCBA4GBgZZoo0aNGoGBgTdrvrBq1Spdd1a7du0NGzYopf76668aNWqISLVq1Qr6CairJ8qVK9ekSRPrd5IuRqKiVJcuauDAf+qxz59XVaveZuFnPq1atSrvnY8Tt22PfGfTd7n0MaZ/oltnUjo70GcIlnszegswfQvt+mrc6OhoS71qrVq1rm8NXsrl5OSsXbvu5ZdzqlfP/WAUUV5e6p131M6d/9w1nD1bVa+u7rlHRUfbe8Y387//5XavUUqNGKFE1HvvWTPeY4+padNUTIwxsyv59GqCjz5S6uoN/7ffNnD4Gy6g0LV416tQoUK9evVat27dq1evRx999MqVK0ops9msNwH45ptvrJ3N2LFKROl1zQ0aqNq1VUKCtWMWiSIKGjRL4wZ9Ua2r14wa3Ka9Va65UWCL3i2Wxg3du3fPu3WlUU6dOuXn51ezZs26deveotdmyWY2m//888+nnnpKnyjrQ1FvOePk5KR3vTLK5MmT5eo2ELpTd/5vnhSIbvw2Z86cnJwcXdVZuBKY1auTuncf2bx5K06JbOKvv5S3d1bNmlXLlr1mHdmMGTP0mucFCxboe4Z9+vQpvjs1zJw5U0SeffbZG/6rbtWuF4XprYVK1QVhERs+fLiIfPfdd0opDw8PMWjvvX379unuwh4eHpa1Wjk5OREREXn74+SzeuLUqVN6lZ+Tk1NwcLDZbI6Li9OtPZycnPTCiluPkJSU9MUXX+gpiUiZMmVKzyL5+Hj1wgu5azxr1vynHnvmzNzmeXfdpb7+WhVuabA+bdC/1caNGy9fvtzo6duN7pytXxTOzs7+/v43XOnwxx9/6FJHV1fX//73v/reTEREhL780IuCb1aNu2rVqtatW+vfXvv27W10AlCs5eSo9etVQICqUSM3bvD0VKNH/3Njb+dOdUf31mzdWomoiAiVmakqVVIi+erTBkOsWKFEVOXKKjlZbd+uTCZVtqzh1SDXL6D4/fffv/nmm/nz569atSoqKurIkSOJiYk3W4c1f/58EalTp44B5zmnT6tDh4qsq7eBijRoUFdbD0RERHzwwQfGVpTdrLeK4YMroysa8jp48OD58+dtt0I+MTGx+F7AGOjSpUuhoaF6h+du3bpVrlzZ8Nv4CxYs0HdL1L+rGwz3ySefWK7rBg8eLCKhoaGFGOfJJ5WImjSJrQFsJi1t8Sef6JCrUqVK19zmCgkJ0TfNnnvuOWv2+bO7r776SkRGjhx5i8ecOXPGcgHTqFGjYrrk+87XvXt3EVm9evX58+dFpHz58kaNnJSUZNnix8/P73//+59lc2UPDw9/f/8CXepnZWUFBgbqhOLhhx9OTEzMysqaMGGC/sot9o84ePBgYGCgLhkTkerVqwcGBubteVyC5eSosDBVpco/Xauv6eO0bZvq0SP3+q1JE5XvHcCUurrXuG7qUbZs2ZIaCMbGxvr7++vVOhUrVrx+U/aTJ0+WLVvWx8cnOjpaKRUdHW25n9m+fftbbAGmFbR6otTKzlarV6uXXlKvvHLtjb071+HDSkRVqKDS09XixUpEtWpl7zmVJrpHqG7b/vDDSkQFBtrieW64gCIrKys+Pv7QoUNbt25dsWLFL7/8EhoaGhwcPGHChJEjRz7xxBP3339/x44ddUz51VdfGTabn35SU6cqo/ews6miDhps52a9VQwfXCnjx4e9REZGHjt2TO+yZqx9+/aJSMOGDdW/qxsMt2vXLp2YKqW+/PJLERlys+alN5eQoFxdlYPDrVsWwgCHDx9+8MEH9Xln27Ztt27dmpWVNXr0aCmGG0zc0PTp00UkICDgto9cvXq13mvgmoYOMIq+73r06NGtW7eKSJs2bYwdf/r06WXKlNHr8kTEy8srODi40B2/f//9dx0Z1K1bV+9MvHjx4iZNmly/7CInJ2flypUDBgywVE+0a9cuLCysWCd0BbJhg7lNm9wQ4YEHbtWpfv581bBh7iP79lV79hy97eARERH169e3XBufPHnSyKnfefbt26cLA+Xqpux570/qCsFLly5Z2o1Vrlw5JCQk/6V/+ayegHbNjb0715dfKpHcOOSZZ5SI+uADe8+plFm+XCUnq507c8sZbLA8WSl18eLFmjVr6qquFi1a1K5dW7+W88NkMs2aNetmGwsWWGioeusttXevGjy4kPs/2UPJCRqAO0pGRoaTk5Ojo2NaWtpvv/0mIkMfecQWT2Q2m6tXry4iBw4cOHjwoD4NKujyh+nTlYjq188WE8QNhIeH648uJycnfSvY3d29ZLRInDp1qoi8/vrrt37Yrl27LIFLIaIx3FZmZqajo6Ojo2NmZmZ4eLiIPProo4Y/y5YtWxYsWDBs2LBVq1YVYn/7axw/fly30XVxcQkJCVFKXfNWdvHixZCQEJ1PiYirq6ufn1/J2zXpFnTLgJ49l4ioWrVUfpafZmaq0FBVtapq3XqNg4ODr6/vzfosHD58+KGHHtK/21atWq0vdBvtYmjlypWWztydOnX6++q+r2azOSwsTHcS0WslLly4UIjxb1s9Ae2aG3t3tP37VUyMyshQFSooEbamLFKWe/vbt6sePZRt7uRpS5Ys0XeDLJycnKpUqdKwYcMOHTr07dt3yJAh/v7+gYGBH3744ddff/3zzz8vW7Zsy5YtBw4cMHIx8qOPKt1FbvVqZcS+m0WDoAGwlSkPPfT7ffcl7d+fEh2dU6WKatDARk80dOhQEfnyyy+VUvoUfPttt436N73ScN4828wPN6K3Xnd2dq5du3blypW3bt1q7xkZ44MPPrh1ufupU6csJ9zlypULDAws6N69yI/Dhw/L1e6wU6ZMEZHx48fbe1K3l56eHhAQoE/mhg0bZtkMNTIy0t/f31I90bBhw+Dg4IRi0g3LEFlZWSEhIboW19OzygcfpBaoc3FCgnr33en6nnzZsmUnTpx4Jc9y39TU1KCgIL2wq0KFCiEhIVZt+V486U3ZdaagN+pavHhxt27d9CHXvXt36yOtHTt29OnTx3IMb9q0yZCZlzDbtqknnlDFoLzPcq27d6/6/HN7z6Y0uf7evo3L2ZKTk9euXbt79+6TJ0/a7Yxl9Gil669/+EEtXmyfORQcQQNgMwMHKhE1f77KyFBOTsrR0cgNx/L49ttvLfXnzz33XNWqVX///ff8f/vWrbktdbi/UvReeeUVERk1apS9J2KY//73vyIyadKk6/9Jt7zV14q6hPjcHd3pq3hbuXKliPTs2VMppe/GTJ8+3d6Tyq8FCxboK+pGjRp98MEHls1KHBwcBg0atHz5cuurJ4qXtWvX6u7lImJpGVAIBw8e9PX11UtO7r777tDQ0Ozs7IiICJ1Qm0wmPz+/Uv6q1KskdOZSrlw5EalRo8acOXMMPOR09YSrqys779zQmjVKRLVvb+953FrxrGMvIYrnvX1r7dqlhg5Vo0erceNsdDVhCwQNgM28/vo/y/YaNVIiNvo0OnHihL4NlZ2dnZSUVNDzoZEjlYh69VVbTA238d1338nN92gojgIDA0UkWLdouiojIyM0NLRq1aqWW4WHDx+21wxLibOzZ4d16/bbhAlKqcDnnqtXq9YfERH2nlQB7N+/v1mzZnp9hIiUL18+ICDg6NHbtxgoYXTnVB0N1K9fP7xATR1vYt26dZbsRr8qRaRt27bcYLc4fvz4U0899dVXX7366quXL182fPysrKzb9pIstZKTlYODKlPmzr75UTqvde8QxfPevjGK28YTTrfuYwGg8Bo3FhE5cCD3zwcPyoED0ry54c9Tp06dhg0bHj58ePv27ZbTx/y7dEkcHeW55wyfF25P70gaHx9v74kYJiMjQ0RcXFz0X81m84IFCyZMmHD06FER6dOnz9SpU/WGL7Cp6jExwzdskL59RSR406bg06flamuDYsHb23vr1q1hYWGpqamVKlUaOnSoZVviUiU2NvbHH390dXV98803J0yYYMgvoXv37lu2bJk/f35gYGDr1q3XrFkTFBT08ssv6wVNEJG6dev++OOPthvfyclJtyPB9cqVk8aNJTpa9uyR9u3tPZubqVFDLl8Wd3c5eVKaNLH3bEqZkSPltdekYkVxdpbgYHvPpmiVLWvvGRQMQQNgM/qzRwcNTZrIokUSE2Ojp/Lx8Tl8+PC777778MMPe17l4eFRoULN8uWreXqK079f62vXyt13i5eXrFwpo0fL5MlSv74sXSpX2/OhiOjbiRcuXLD3RAyjgwa9FHzVqlVvvvnmjh07RKRp06YTJ0709fW18/xKj+PHRUTq1ROl5MSJ3D8XK+7u7qNGjbL3LOysffv2n3/++YMPPljP0P99urBowIABZrM5KyurQoUKBg4OWKl9e4mOlqioOzhoKM3XunbXsqXMnSspKcXuqrsUImgAbEYHDTExotS/qhtswNfXNy4u7rffflu2bFner3fr9tWGDS+JiJubeHqKh4eULy8jRsjBg3LwoCxdKnPmSOXK8uqrIiIzZxI0FDVd0VCSgobMzEwROXfuXJ8+fVavXi0itWvX/s9//vP8889zv7RIWYKG8+clNVUqV5Z8b8qFO4rt0hZLc03gjtKuncyeLVFR9p7HLXCta3f85osDggbAZqpUkcqVJSFBzp37J3Swjd69e1erVq1Ro0YXL15Muio5Oblq1aoVK0pSkqSlSVqaxMWJiDzwgIjIoEEyY0but7/1lpQrJ7t22Wh2uKmSFzSkpaWJyAcffKCUqlSp0ttvv/3yyy9bVlKg6Bw7JiJyzz3/JA4AUBzoQobISHvP47a41gVuiaABsKUmTWTDBomJkVatpGdPseXS9ObNmwffvH4vJUWSkiQ5WZKSpHJl+fRTeeopGTVKEhOlcmX58EOpVUuoai96FSpUcHZ2vnz5cmZmpl5uUNwNGDDg77//Pnv27KhRoyZNmkRJtn1kZMi5c+LsLDVryqZNIgQNAIqNng36twAAGi1JREFU1q3F0VESErLT05Wrq7O9pwOgkAgaAFuaMkUcHeXAAYmMlJ9+kurV7TWRsmWlbFmpUeNfX5w8Wdq3l2bNcv9qMhX9vEo7k8lUqVKluLi4xMTE6vY7PAw0dOjQzp07Ozk51a5d295zKcVOnBCzWerVE0fH3NIGggYAxUTZstKp08CNGxfv2rWZrplA8eVg7wkAJVq3brJ7txw4IP37y8svy9699p5Qrs8+kwoV5J57JCFBpk2TWrVERMLD7T2tUqnkbTxxzz33kDLYWWysyNVwoXh2ggRQmjVqVEVEIovB8gkAN0VFA2Bjy5bJnDni7i5jxsjy5bbY3hLFWsnbeAL216uXpKbKpUsiIgMGiIeHdOli5ykBQL61a9fuhx9+iLqjG0ICuA2CBsDG2GwZt1Ty+kHijuDmJr//LqdOiZ+fPPSQvWcDAAXQvn17oaIBKOZYOgHYmN5secwY2bFD+vSx92xwxyFogE3MmCG7d99pi7YAID9at27t7Oy8f//+lJQUe88FQCFR0QDYGJst45ZKXo8G3BFYtAWg2HJ1dW3atOmuXbt27drVtWtXe08HQGFQ0QAUCVIG3IQOGhISEuw9EZQsetGWCIu2ABRHrJ4AijsqGgDAnlg6AZvQi7YqVhRnZwkOtvdsAKBgHnjggaysLG9vb3tPBEAhmZRS9p4DAJRey5cv79ev3/333798+XJ7zwUlDou2AACAPbB0AgDsiR4NsCFSBgAAYA8EDQBgT9WrV2/RokXjxo3tPREAAADAGCydAAAAAAAAhqGiAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagAQAAAAAAGIagIb+UUsOGDfv777/tPRGUKBxXsAWOK9gCxxUMt3nz5mHDhvXs2XPatGnnzp2z93RQQnBcwRY4rgqKoCFfFixYMGLEiLlz58bHx9t7Lig5OK5gCxxXsAWOKxju9OnTvXv3vnjx4vPPP7948eL+/fvbe0YoCTiuYAscV4XgZO8JFA8LFy5MSkpydHS090RQonBcwRY4rmALHFcw3Pr166tXr75kyRIR8fb27tChQ3R0tLe3t73nheKN4wq2wHFVCAQN+TJ79mwR8fT0tPdEUKJwXMEWOK5gCxxXMFy3bt0WLFig/7x58+bKlSs3aNDAvlNCCcBxBVvguCoElk4AAACgqNWpU6dNmzY5OTlTp0599dVXX3vttTJlyth7Uij2OK5gCxxXhUBFAwAAAOzgwoULDz744IkTJ+bOnfv444/bezooITiuYAscVwVF0AAAAICidunSpb59+9atW3fFihUVK1a093RQQnBcwRY4rgqBoAEAAABFbcGCBUePHg0NDY2NjY2NjRWRhg0burq62nteKN44rmALHFeFQNBQACaTyWQy2XsWKGk4rmALHFewBY4rGGjz5s1JSUmdOnWyfCUyMrJdu3Z2nBJKAI4r2ALHVSGYlFL2ngMAAAAAACgh2HUCAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAYhqABAAAAAAAY5k4PGjZs2HDs2LGYmBh7T+ROtHr16ujo6LNnz9p7IgAAAAAA5Lpzg4bY2Njhw4d379790UcfbdGixciRIy9cuGDvSd0poqOj+/Xr16dPn0GDBnl5eU2cODE9Pd3ekwIAAAAA4I4MGlJTUydOnNioUaPZs2e7ubm5ubkppWbMmOHt7f3111/n5OTYe4L2dOHChVGjRrVo0WL58uWVK1euWLFiSkrKpEmTmjVr9ttvv9l7dgAAAACA0u6OCxoWLVrUrFmzSZMmpaamDhgwYN++fZs2bdqzZ88DDzygr7GbN2++bNkye0/TDrKzsy1pi8lk8vf3j4mJ2bp16+rVq1u2bHn06NHHHnusd+/eu3fvtvdMAQAAAACll0kpZe855Nq+ffsrr7yyfv16EWnbtm1ISEj37t3zPmDRokWvvPLK0aNHRWTAgAHTp0+vX7++feZa5P78889XXnll7969ItKnT5+QkJDmzZtb/tVsNs+ZM+eNN944f/68g4PDsGHDPv7442rVqtlvvgAAAACAUuqOqGhISEgYN25cx44d169fX7ly5ZCQkK1bt3bu3PnTTz8dN26c5WEDBw6Mjo4OCQnx9PRcvHixt7f3uHHjkpKS7DjzInDo0KEhQ4b4+Pjs3bvXy8srPDx81apVeVMGEXFwcBg+fHhMTExgYKCTk9Ps2bMbN248ZcqUzMxMe00bAAAAAFA62bmiISsr68svvwwKCrp8+bKzs/OoUaPee++98uXLL1269NVXX42JiXFwcNi7d6+3t3fe7zpz5sykSZO+/fZbs9lco0aNiRMnvvDCCw4Od0RoYqArV658/PHHwcHBGRkZ5cqVe+2119566y0XF5dbf9eBAwdeffXVJUuWiEijRo2mTZv20EMPFcl8AQAAAACwa9CwatWqcePG7d+/X0R8fHymT5/etGnTQ4cOvfPOO/PmzRMRLy+v//3vf76+vjf89sjIyFdeeWXDhg0i0r59+5CQkG7duhXl/G1HL4V488034+Li9FKIjz766K677sr/CDf83dpsvgAAAAAA5LJPFcCBAwceeuihvn377t+/v3Hjxn/88cfKlSvvvvvuCRMmtGjRYt68eeXKlQsKCtqzZ8/NUgYRad++/fr168PDw+vUqRMZGdm9e/chQ4acPHnShvP++Wf56CM5d86GTyGyZcuWrl27PvPMM3FxcR07dtywYcOsWbMKlDKIiI+Pz86dO0NCQsqXL79q1arWrVuPGzfu8uXLNpozAAAAAACaHSoaJk6c+L///S87O7tSpUoTJ04cNWqUg4ODNTfwr1y5Mnny5E8//TQ9Pb1cuXJffPHF8OHDjZ/3jBly/LgMGyZBQTJxovy7S4JR/Pz85syZIyJ16tSZOnXqkCFDTCaTNQPGxcW9884733//vdlsvuuuu+bOndu7d2+DJgsAAAAAwLWKuqJh586dO3fuzMnJ8fPzi46OHjt27IYNG9q1a6dv4Hfq1Gnjxo0FvYFfrly5yZMnHzx40M/PLyUlZf369bGxscZPfdky+c9/pFkzGTNGli83fnyR1atXHzt2zNXVNTAwcN++fU888YSVKYOI3HXXXd9+++22bdu6d++emJi4dOnS9PR0Q2YLAAAAAMD1ijpoiIyM/P333x977LFZs2ZVq1Zt+vTpPXv23LlzZ926dcPDwzdv3typU6fCjVy7du1Zs2a1bNny22+/PXHihLHTFhGpUUP00oOTJ6VJE+PHF1m6dOmGDRteffXV4ODgcuXKGThy27Zt165d6+bm9vHHH6empho4MgAAAAAAedmnR0OFChX0Hx577LEqVaoEBQVFR0ffoh1D/rm7u1s/yI2NHCmvvSZjxsiOHdKnj62eJc8vx1gmk8nR0dEWIwMAAAAAYOFk36evXbv2yZMn3dzcrBlk4cKFn3322aOPPjp27FijJnYDb78tOTkSGio7d8qjj0qPHvLWWzZ8OgAAAAAAiiE7Bw0iYmXKICKnTp3666+/mjVrZsh8bmrZMsnJETc3OXtWli0TDw/bPh0AAAAAAMWQfZZOAAAAAACAEomgAQAAAAAAGIagAQAAAAAAGKaoezS4uw/o3PlktWrmIn5eAAAAAABQBIo6aEhNrb55s9i6byMAAAAAALALlk4AAAAAAADDEDTkV1kXF5cyZXJMpgUODi5lygxzsv/OoAAAAAAA3Gm4Ws6vjIyMnJwcpVSO2ZyZmZmVnW3vGQEAAAAAcMehogEAAAAAABiGoAEAAAAAABiGoAEAAAAAABimSHs0rF0rbdtKTIzs2mXksKNHj37hhRec6M4IAAAAAIC9FWlFw4IF8vbb0qiR/PHHv76ekCBjxsi5c4Uc1tHR0c3NzdnZ2foZ3pbZbFZK2WhwPbLtxgcAAAAAwNaKeunEoEEyY8a1X5w4Ub78Uho1kilTJCOjkCMrpWbNmrV3797KlSu//fbbx48ft26m1+rQoUONGjUef/zx1157rU6dOg0bNjR2/DVr1vz88881atQIDQ3duHGjsYNnZmZOnz49KyurU6dORZPIAAAAAABKp6IOGp56StaskXPnJCNDXnxRYmJERMaNk4cfluRkmTBBmjWT338v8C39v//+u0OHDs8880xycnJycvLatWubNWv23nvvpaWlWT/nzMzMjz/+eP/+/WfPnl2yZElsbOzJkycjIiJWrlxp/eAicuzYsccff7xXr16xsbFXrlw5evRo9+7dn3322bNnzxoy/q+//urt7f3KK69cuXJl/PjxHh4ehgwLAAAAAMD17NAMcvJkiYyUzz+Xb7+Vli1l/HipUkUWLpQ//5SWLeXIEfnkk5m9e/fevXt3fkaLjY0dPnx4jx49oqKiatasGRYWdvjwYT8/v7S0tKCgIC8vr1mzZlmzGGHVqlVt2rR54403kpKSfHx8du3atXLlSm9v73379t1///19+/bdv39/oQdPTU2dOHFi06ZNFyxY4O7uHhQUdOzYsaCgoDJlyvzwww9eXl4TJ05MT08v9PgxMTEPPvjg4MGDjx492qRJk6VLlz7xxBOFHg0AAAAAgNtTdpKQoAIClKOjElGVKqmQEJWdrbKy1P/9n7lRo6Yi4uTkNGbMmISEhJuNkJqaGhwcXK5cORFxc3MLDAxMTk62/OvatWvbtGmjf8ZhwzZv317gGe7bp/z8Eh0cnETE29t72bJlln/KzMwMCQnx9PQUEWdn54CAgEuXLhVocLPZHB4eXqdOHRExmUy+vr4nTpyw/OvJkyf9/Pz05GvXrh0WFlbQySckJAQEBOgGmZUqVQoJCcnKyiroIAAAAAAAFJTdggZt/371wANKRIkob2+1dKlSSiUkJIwZM8ZykTxv3rzrvzEiIqJevXr6UnzAgAHHjh27/jE5OTkzZsy4//4XRZSDg3rhBRUXl69ZJSaqgADl5KRE1P33zw0JCcnMzLz+YfHx8QEBAY6OjiJSuXLlkJCQ7Ozs/Iy/bdu2rl276sm3b9/+77//vuHDVq9e3bJlS/2wXr167dq1Kz+DZ2VlhYaGVqlSRYc1/v7+8fHx+flGAAAAAACsZ+egQZs3T9Wrp0RU3bqHhgx54ujRo0qpmJiY/v37i8i6devyPjgqKqp79+768rtNmzbX/Ov1kpNVUJBycVEiqmxZFRSk0tNv+uCcHBUWpqpVUzqb8PO7fTaRdz5t27a99XxiY2P9/f0dHBxERPd9zMnJucXjc3JywsLCqlWrJiIODg5+fn5xt5zQqlWrmjdvrifTp0+fPXv23Gb2AAAAAAAY6o4IGpRSaWnqgw9U9+5DRMTV1fXtt9/W6yC2bt1qecyFCxcKV0GglDpwQD30UG7pRLNmKiPjBo/JyVGdO+c+pndvlb8Cgly3rbDIyMgICQnRjRjLlCkTEBBw+fLlfA6emJgYGBhYpkwZEalYsWJwcHDGdT/AwYMHfX199QS8vLzCw8MLMHsAAAAAAAxypwQN2s1u+OueCOXLly90TwRt1SrVvLkaP/6mD5g4UdWurQreEkEppVJSUm7WMyIiIqJ+/fqWGOLIkSOFGN9S4iEijRs3Xrx4sf56cnJyUFCQi4uLiJQtWzYoKCj9FjUbAAAAAADYkklZsSODjURGRo4bN27jxo0i0r59+6effnrGjBl6cwcfH5/p06c3bdq00INnZUlWlmzbJnffLV5esnKluLhIu3ZStqwsXSq9eomIuLoWfvKxsbFvvfXWnDlzlFI1a9Z8+eWX16xZs3z5chHx9vaeNm1av379Cj+6yKpVq8aNG6d/G3369OnTp8/06dPj4uIcHByGDRv20Ucf3XXXXdaMDwAAAACANe7EoEFElFLz589//fXXT548Wbly5YSEhMaNG0+bNs1yS99KAQFy8KAsXSojRkjlyvLqq1Krlvj6yrx5hgwvW7duHTdu3ObNm8uXL3/58uVKlSq9++67L7/8sl73YaWsrKwvv/wyKCjo8uXLHh4eycnJHTt2nD59eufOna0fHAAAAAAAa9yhQYN25cqV4ODg9u3bnzx5cvTo0XofCkMEBEiTJuLoKBs3SuXKEh8v5crJn3/KwYNGPYOYzeaZM2dWqFBh27ZtEyZMqFixomFDi4hIXFzcRx991KZNG2dnZ19fX5PJZOz4AAAAAAAUwh0dNNhOQIC8956MGiWJidKsmfEVDQAAAAAAlE4O9p6APU2eLJGR//yVmgAAAAAAAKxUSisaAAAAAACALZTqigYAAAAAAGAsggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYggYAAAAAAGAYgoZ82bx587Bhw3r27Dlt2rRz587ZezoAAAAAANyhCBpu7/Tp071797548eLzzz+/ePHi/v3723tGAAAAAADcoZzsPYFiYP369dWrV1+yZImIeHt7d+jQITo62tvb297zAgAAAADgjkNFw+1169ZtwYIF+s+bN2+uXLlygwYN7DslAAAAAADuTAQNt1enTp02bdrk5ORMnTr11Vdffe2118qUKWPvSQEAAAAAcCdi6US+XLhw4cEHHzxx4sTcuXMff/xxe08HAAAAAIA7FEHD7V26dKlv375169ZdsWJFxYoV7T0dAAAAAADuXAQNt7dgwYKjR4+GhobGxsbGxsaKSMOGDV1dXe09LwAAAAAA7jgEDbe3efPmpKSkTp06Wb4SGRnZrl07O04JAAAAAIA7k0kpZe85AAAAAACAEoJdJwAAAAAAgGEIGgAAAAAAgGEIGgAAAAAAgGEIGgAAAAAAgGEIGgAAAAAAgGEIGgAAAAAAgGH+Hw2KVnh7nuoVAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<PIL.PngImagePlugin.PngImageFile image mode=RGB size=1400x800 at 0x1A265AC110>"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "smiles_df = pd.DataFrame([x.split(\"\\t\") for x in lig_buff.split(\"\\n\")],columns=[\"Name\",\"SMILES\"])\n",
    "smiles_df.Name = [x.split(\"-\")[1] for x in smiles_df.Name]\n",
    "combo_df = smiles_df.merge(pca_df,left_on=\"Name\",right_on=\"structure\")\n",
    "combo_df.sort_values(\"Cluster\",inplace=True)\n",
    "combo_df['Mol'] = [Chem.MolFromSmiles(x) for x in combo_df.SMILES]\n",
    "MolsToGridImage(combo_df.Mol,molsPerRow=7,legends=list(combo_df.Cluster))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Are There Other Homologous Structures to Learn From?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'SGFRKMAFPSGKVEGCMVQVTCGTTTLNGLWLDDVVYCPRHVICTSEDMLNPNYEDLLIRKSNHNFLVQAGNVQLRVIGHSMQNCVLKLKVDTANPKTPKYKFVRIQPGQTFSVLACYNGSPSGVYQCAMRPNFTIKGSFLNGSCGSVGFNIDYDCVSFCYMHHMELPTGVHAGTDLEGNFYGPFVDRQTAQAAGTDTTITVNVLAWLYAAVINGDRWFLNRFTTTLNDFNLVAMKYNYEPLTQDHVDILGPLSAQTGIAVLDMCASLKELLQNGMNGRTILGSALLEDEFTPFDVVRQCSGVT'"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# pull out the protein sequence for a blast search\n",
    "ref = non_cov_prot_structs[0]\n",
    "seq = ref['A'].getSequence()\n",
    "seq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Blast searching NCBI PDB database for \"SGFRK...\"\n",
      "@> WARNING Blast search time out.                    \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Could not reach the NCBI PDB database\n"
     ]
    }
   ],
   "source": [
    "# use prody to blast search the PDB\n",
    "blast_record = prody.blastPDB(seq, expect=10)\n",
    "if blast_record:\n",
    "    blast_df = pd.DataFrame(blast_record.getHits()).T\n",
    "    homolog_df = blast_df[\n",
    "        (blast_df['percent_identity'] > 80) & \n",
    "        (blast_df['percent_coverage'] > 80)\n",
    "    ]\n",
    "    homolog_df[['evalue','percent_identity','percent_coverage','pdb_id','chain_id','title']]\n",
    "else:\n",
    "    print(\"Could not reach the NCBI PDB database\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6LU7 6M2Q 6M2N 6W63 6Y84 6M03 5R7Z 5R8T 5RGP 5RGQ 5RGN 5RGO 5RGL 5RGM 5RGJ 5RGK 5RGR 5RGS 5R81 6Y2E 5R82 6Y2F 5R80 5R7Y 5R83 6Y2G 5R84 5RG0 5RFO 5RFP 5RG1 5RFM 5RFN 5RFK 5RFL 5RFI 5RFJ 5RFW 5RFX 5RFU 5RFV 5RFS 5RFT 5RG2 5RFQ 5RFR 5RG3 7BQY 5RFY 5RFZ 5RGH 5RGI 5RGG 6YB7 5REN 5REO 5RF0 5REL 5REM 5REJ 5REK 5REH 5REI 5REV 5RF7 5REW 5RF8 5RF5 5RET 5REU 5RF6 5RF3 5RER 5RF4 5RES 5RF1 5REP 5RF2 5REZ 5RF9 5REX 5REY 5RFG 5RFH 5RFE 5RFF 5RFC 5RFD 5RFA 5RFB 5RE6 5RE7 5RE4 5RE5 5RE8 5RE9 5REF 5REG 5RED 5REE 5REB 5REC 5REA 3VB6 3VB5 3VB4 3VB3 2GX4 3VB7 3IWM 3SZN 2Z94 2GTB 2Z9G 2Z9L 2Z9K 2Z9J 3TIU 3TIT 2GT7 2GT8 2A5I 2A5K 2BX3 2BX4 2A5A 5N5O 2H2Z 2ZU4 2ZU5 2Z3E 2Z3D 2Z3C 2HOB 5N19 1Q2W 1UK2 1UK3 1UK4 1WOF 2C3S 1UJ1 2DUC 1Z1I 2D2D 3SN8 2AMD 3SNE 3SND 3SNC 3SNB 2GZ7 3V3M 2GZ8 2AMQ 2V6N 2GZ9 3TNT 3TNS 3E91 3F9E 2VJ1 3M3V 3EA8 3EA7 2PWX 2QCY 4HI3 3M3T 2ALV 5C5N 5C5O 3AW0 3AW1 3AVZ 4TWY 4TWW 3ATW 3F9H 2QC2 3F9F 4MDS 4WY3 3M3S 5B6O 1Z1J 2Q6G 3EA9 3EAJ 2OP9 3SNA 2QIQ 3F9G 3FZD 3D62 5WKM 5WKK 5WKL 5WKJ 5C3N 4RSP 4YLU 4WMF 4WME 4WMD 4YOJ 4YO9 4YOI 4YOG 2YNB 2YNA 6JIJ 3D23 5HYO 5GWZ 4XFQ 4ZUH 5ZQG 4ZRO 5EU8 3TLO 5NH0 6FV2 6FV1 5GWY 1P9U 4F49 2AMP 1LVO 1P9S 2ZU2 2LIZ 2K7X 3EBN 2Q6D 2Q6F\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "236"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# The BLAST PDB database is out of date\n",
    "# so do a manual search\n",
    "with open('PDB_seq_search_results.txt') as f:\n",
    "    pdb_ids = f.read().strip().split(',')\n",
    "\n",
    "# one of these structures doesn't have a PDB file available\n",
    "pdb_ids.remove('6Y7M')\n",
    "    \n",
    "print(' '.join(pdb_ids))\n",
    "len(pdb_ids)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Local PDB folder is set: '/Users/pwalters/software/mpro_analysis/PDB_Mpro_structures'\n",
      "@> A plain folder structure will be assumed.\n",
      "@> 236 PDBs were parsed in 272.65s.\n"
     ]
    }
   ],
   "source": [
    "# create local directory for new structures\n",
    "struct_dir = './PDB_Mpro_structures'\n",
    "if not os.path.exists(struct_dir): os.mkdir(struct_dir)\n",
    "prody.pathPDBFolder(folder=struct_dir)\n",
    "\n",
    "# download structures into our directory\n",
    "PDB_Mpro_structs = prody.parsePDB(pdb_ids)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Let's align all these structures"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Checking AtomGroup 6LU7: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6LU7 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 6LU7 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 74.94\n",
      "@> After alignment  RMSD: 1.54\n",
      "@> Checking AtomGroup 6M2Q: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6M2Q (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 115.90\n",
      "@> After alignment  RMSD: 0.36\n",
      "@> Checking AtomGroup 6M2N: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6M2N (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 6M2N (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 6M2N (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain D from 6M2N (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 85.51\n",
      "@> After alignment  RMSD: 0.65\n",
      "@> Checking AtomGroup 6W63: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6W63 (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.80\n",
      "@> After alignment  RMSD: 0.58\n",
      "@> Checking AtomGroup 6Y84: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6Y84 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 6M03: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6M03 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 11.91\n",
      "@> After alignment  RMSD: 0.40\n",
      "@> Checking AtomGroup 5R7Z: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R7Z (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 5R8T: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R8T (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.17\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RGP: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGP (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RGQ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGQ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.28\n",
      "@> After alignment  RMSD: 0.23\n",
      "@> Checking AtomGroup 5RGN: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGN (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5RGO: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGO (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RGL: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGL (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RGM: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGM (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.22\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5RGJ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGJ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RGK: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGK (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5RGR: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGR (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RGS: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGS (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 5R81: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R81 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.64\n",
      "@> After alignment  RMSD: 0.31\n",
      "@> Checking AtomGroup 6Y2E: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6Y2E (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.47\n",
      "@> After alignment  RMSD: 0.38\n",
      "@> Checking AtomGroup 5R82: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R82 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.26\n",
      "@> After alignment  RMSD: 0.23\n",
      "@> Checking AtomGroup 6Y2F: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6Y2F (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 100% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 38.00\n",
      "@> After alignment  RMSD: 0.66\n",
      "@> Checking AtomGroup 5R80: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R80 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.39\n",
      "@> After alignment  RMSD: 0.25\n",
      "@> Checking AtomGroup 5R7Y: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R7Y (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.25\n",
      "@> After alignment  RMSD: 0.21\n",
      "@> Checking AtomGroup 5R83: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R83 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.17\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 6Y2G: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6Y2G (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 100% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 6Y2G (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 303 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 303 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 34.56\n",
      "@> After alignment  RMSD: 0.87\n",
      "@> Checking AtomGroup 5R84: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5R84 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.65\n",
      "@> After alignment  RMSD: 0.33\n",
      "@> Checking AtomGroup 5RG0: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RG0 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RFO: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFO (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.39\n",
      "@> After alignment  RMSD: 0.37\n",
      "@> Checking AtomGroup 5RFP: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFP (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.91\n",
      "@> After alignment  RMSD: 0.29\n",
      "@> Checking AtomGroup 5RG1: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RG1 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 5RFM: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFM (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.29\n",
      "@> After alignment  RMSD: 0.21\n",
      "@> Checking AtomGroup 5RFN: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFN (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5RFK: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFK (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.28\n",
      "@> After alignment  RMSD: 0.25\n",
      "@> Checking AtomGroup 5RFL: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFL (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.25\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5RFI: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFI (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.23\n",
      "@> After alignment  RMSD: 0.23\n",
      "@> Checking AtomGroup 5RFJ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFJ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5RFW: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFW (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.23\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5RFX: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFX (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.27\n",
      "@> After alignment  RMSD: 0.23\n",
      "@> Checking AtomGroup 5RFU: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFU (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RFV: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFV (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5RFS: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFS (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 5RFT: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFT (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RG2: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RG2 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.17\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5RFQ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFQ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.25\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5RFR: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFR (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.23\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5RG3: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RG3 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 7BQY: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 7BQY (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 100% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 7BQY (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 38.07\n",
      "@> After alignment  RMSD: 0.47\n",
      "@> Checking AtomGroup 5RFY: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFY (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.30\n",
      "@> After alignment  RMSD: 0.27\n",
      "@> Checking AtomGroup 5RFZ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFZ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RGH: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGH (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RGI: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGI (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.14\n",
      "@> After alignment  RMSD: 0.14\n",
      "@> Checking AtomGroup 5RGG: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RGG (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.26\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 6YB7: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6YB7 (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 303 residues match with 100% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 303 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.17\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5REN: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REN (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.35\n",
      "@> After alignment  RMSD: 0.33\n",
      "@> Checking AtomGroup 5REO: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REO (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5RF0: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF0 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5REL: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REL (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5REM: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REM (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.32\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 5REJ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REJ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5REK: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REK (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5REH: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REH (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.44\n",
      "@> After alignment  RMSD: 0.33\n",
      "@> Checking AtomGroup 5REI: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REI (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.29\n",
      "@> After alignment  RMSD: 0.21\n",
      "@> Checking AtomGroup 5REV: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REV (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.15\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RF7: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF7 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.25\n",
      "@> After alignment  RMSD: 0.24\n",
      "@> Checking AtomGroup 5REW: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REW (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RF8: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF8 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RF5: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF5 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RET: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RET (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5REU: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REU (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RF6: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF6 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.15\n",
      "@> After alignment  RMSD: 0.14\n",
      "@> Checking AtomGroup 5RF3: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF3 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.12\n",
      "@> After alignment  RMSD: 0.11\n",
      "@> Checking AtomGroup 5RER: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RER (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.28\n",
      "@> After alignment  RMSD: 0.24\n",
      "@> Checking AtomGroup 5RF4: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF4 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.15\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RES: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RES (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.17\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RF1: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF1 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.23\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5REP: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REP (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RF2: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF2 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.18\n",
      "@> After alignment  RMSD: 0.16\n",
      "@> Checking AtomGroup 5REZ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REZ (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.24\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5RF9: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RF9 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.35\n",
      "@> After alignment  RMSD: 0.34\n",
      "@> Checking AtomGroup 5REX: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REX (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.27\n",
      "@> After alignment  RMSD: 0.27\n",
      "@> Checking AtomGroup 5REY: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REY (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.28\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5RFG: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFG (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.26\n",
      "@> After alignment  RMSD: 0.20\n",
      "@> Checking AtomGroup 5RFH: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFH (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.35\n",
      "@> After alignment  RMSD: 0.34\n",
      "@> Checking AtomGroup 5RFE: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFE (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RFF: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFF (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RFC: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFC (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RFD: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFD (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.17\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RFA: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFA (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 100% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.16\n",
      "@> After alignment  RMSD: 0.15\n",
      "@> Checking AtomGroup 5RFB: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RFB (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5RE6: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RE6 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.21\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5RE7: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RE7 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5RE4: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RE4 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.25\n",
      "@> After alignment  RMSD: 0.21\n",
      "@> Checking AtomGroup 5RE5: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RE5 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.43\n",
      "@> After alignment  RMSD: 0.24\n",
      "@> Checking AtomGroup 5RE8: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RE8 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.30\n",
      "@> After alignment  RMSD: 0.24\n",
      "@> Checking AtomGroup 5RE9: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RE9 (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.44\n",
      "@> After alignment  RMSD: 0.42\n",
      "@> Checking AtomGroup 5REF: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REF (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.19\n",
      "@> After alignment  RMSD: 0.17\n",
      "@> Checking AtomGroup 5REG: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REG (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.24\n",
      "@> After alignment  RMSD: 0.22\n",
      "@> Checking AtomGroup 5RED: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5RED (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.22\n",
      "@> After alignment  RMSD: 0.18\n",
      "@> Checking AtomGroup 5REE: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REE (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.29\n",
      "@> After alignment  RMSD: 0.23\n",
      "@> Checking AtomGroup 5REB: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REB (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.36\n",
      "@> After alignment  RMSD: 0.32\n",
      "@> Checking AtomGroup 5REC: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REC (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 5REA: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5REA (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 100% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 0.20\n",
      "@> After alignment  RMSD: 0.19\n",
      "@> Checking AtomGroup 3VB6: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3VB6 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3VB6 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain E from 3VB6 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 3VB6 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 42.10\n",
      "@> After alignment  RMSD: 1.03\n",
      "@> Checking AtomGroup 3VB5: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3VB5 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3VB5 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain E from 3VB5 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 3VB5 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 42.16\n",
      "@> After alignment  RMSD: 1.01\n",
      "@> Checking AtomGroup 3VB4: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3VB4 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3VB4 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain E from 3VB4 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 3VB4 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 42.22\n",
      "@> After alignment  RMSD: 0.93\n",
      "@> Checking AtomGroup 3VB3: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3VB3 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3VB3 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 42.38\n",
      "@> After alignment  RMSD: 1.07\n",
      "@> Checking AtomGroup 2GX4: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GX4 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.55\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 3VB7: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3VB7 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3VB7 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain E from 3VB7 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 3VB7 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 42.17\n",
      "@> After alignment  RMSD: 1.02\n",
      "@> Checking AtomGroup 3IWM: 8 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3IWM (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3IWM (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 3IWM (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain D from 3IWM (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain H from 3IWM (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 3IWM (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain G from 3IWM (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain E from 3IWM (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 40.99\n",
      "@> After alignment  RMSD: 16.30\n",
      "@> Checking AtomGroup 3SZN: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SZN (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 34.68\n",
      "@> After alignment  RMSD: 1.67\n",
      "@> Checking AtomGroup 2Z94: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z94 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.57\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 2GTB: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GTB (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 299 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 33.52\n",
      "@> After alignment  RMSD: 0.94\n",
      "@> Checking AtomGroup 2Z9G: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z9G (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.59\n",
      "@> After alignment  RMSD: 1.71\n",
      "@> Checking AtomGroup 2Z9L: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z9L (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 2Z9L (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 80.72\n",
      "@> After alignment  RMSD: 0.66\n",
      "@> Checking AtomGroup 2Z9K: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z9K (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain B from 2Z9K (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 62.59\n",
      "@> After alignment  RMSD: 1.00\n",
      "@> Checking AtomGroup 2Z9J: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z9J (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2Z9J (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 66.91\n",
      "@> After alignment  RMSD: 1.02\n",
      "@> Checking AtomGroup 3TIU: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3TIU (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 39.85\n",
      "@> After alignment  RMSD: 1.67\n",
      "@> Checking AtomGroup 3TIT: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3TIT (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.53\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 2GT7: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GT7 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 2GT7 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 52.05\n",
      "@> After alignment  RMSD: 0.95\n",
      "@> Checking AtomGroup 2GT8: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GT8 (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 298 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 298 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 29.94\n",
      "@> After alignment  RMSD: 1.27\n",
      "@> Checking AtomGroup 2A5I: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2A5I (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.42\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 2A5K: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2A5K (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain B from 2A5K (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 83.23\n",
      "@> After alignment  RMSD: 0.80\n",
      "@> Checking AtomGroup 2BX3: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2BX3 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 300 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 75.42\n",
      "@> After alignment  RMSD: 1.25\n",
      "@> Checking AtomGroup 2BX4: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2BX4 (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 298 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 298 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 30.87\n",
      "@> After alignment  RMSD: 0.73\n",
      "@> Checking AtomGroup 2A5A: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2A5A (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.42\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 5N5O: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5N5O (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 32.36\n",
      "@> After alignment  RMSD: 1.71\n",
      "@> Checking AtomGroup 2H2Z: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2H2Z (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 114.05\n",
      "@> After alignment  RMSD: 1.65\n",
      "@> Checking AtomGroup 2ZU4: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2ZU4 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.50\n",
      "@> After alignment  RMSD: 1.65\n",
      "@> Checking AtomGroup 2ZU5: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2ZU5 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.54\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 2Z3E: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z3E (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain I from 2Z3E (len=2) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 34.73\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 2Z3D: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z3D (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain I from 2Z3D (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.44\n",
      "@> After alignment  RMSD: 1.65\n",
      "@> Checking AtomGroup 2Z3C: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Z3C (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2Z3C (len=2) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.97\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 2HOB: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2HOB (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2HOB (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 113.61\n",
      "@> After alignment  RMSD: 1.65\n",
      "@> Checking AtomGroup 5N19: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5N19 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 34.75\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 1Q2W: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1Q2W (len=295) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 295 residues match with 96% sequence identity and 97% overlap.\n",
      "@>   Comparing Chain B from 1Q2W (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 295 residues match with 96% sequence identity and 97% overlap.\n",
      "@> Alignment is based on 295 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 55.25\n",
      "@> After alignment  RMSD: 0.77\n",
      "@> Checking AtomGroup 1UK2: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1UK2 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 1UK2 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 92.79\n",
      "@> After alignment  RMSD: 1.26\n",
      "@> Checking AtomGroup 1UK3: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1UK3 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 1UK3 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 71.02\n",
      "@> After alignment  RMSD: 1.40\n",
      "@> Checking AtomGroup 1UK4: 5 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1UK4 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 1UK4 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain G from 1UK4 (len=5) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=2%).\n",
      "@>   Comparing Chain H from 1UK4 (len=5) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=2%).\n",
      "@>   Comparing Chain K from 1UK4 (len=5) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=2%).\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 55.43\n",
      "@> After alignment  RMSD: 0.88\n",
      "@> Checking AtomGroup 1WOF: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1WOF (len=309) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 1WOF (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 95% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 96.63\n",
      "@> After alignment  RMSD: 1.16\n",
      "@> Checking AtomGroup 2C3S: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2C3S (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 36.21\n",
      "@> After alignment  RMSD: 0.95\n",
      "@> Checking AtomGroup 1UJ1: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1UJ1 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@>   Comparing Chain B from 1UJ1 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 55.49\n",
      "@> After alignment  RMSD: 0.89\n",
      "@> Checking AtomGroup 2DUC: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2DUC (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2DUC (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 55.08\n",
      "@> After alignment  RMSD: 1.13\n",
      "@> Checking AtomGroup 1Z1I: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1Z1I (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 50.22\n",
      "@> After alignment  RMSD: 0.80\n",
      "@> Checking AtomGroup 2D2D: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2D2D (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2D2D (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 73.99\n",
      "@> After alignment  RMSD: 0.75\n",
      "@> Checking AtomGroup 3SN8: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SN8 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.54\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 2AMD: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2AMD (len=309) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 2AMD (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 95% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 97.62\n",
      "@> After alignment  RMSD: 1.20\n",
      "@> Checking AtomGroup 3SNE: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SNE (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain H from 3SNE (len=4) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 47.74\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 3SND: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SND (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3SND (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 3SND (len=1) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@>   Comparing Chain D from 3SND (len=1) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 48.06\n",
      "@> After alignment  RMSD: 1.72\n",
      "@> Checking AtomGroup 3SNC: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SNC (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain H from 3SNC (len=4) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 48.05\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 3SNB: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SNB (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain H from 3SNB (len=4) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.53\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 2GZ7: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GZ7 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 52.18\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 3V3M: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3V3M (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 36.11\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 2GZ8: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GZ8 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 52.21\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> After alignment  RMSD: 1.67\n",
      "@> Checking AtomGroup 2AMQ: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2AMQ (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2AMQ (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 2AMQ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain D from 2AMQ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 73.72\n",
      "@> After alignment  RMSD: 0.91\n",
      "@> Checking AtomGroup 2V6N: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2V6N (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 53.52\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 2GZ9: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2GZ9 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 48.03\n",
      "@> After alignment  RMSD: 1.67\n",
      "@> Checking AtomGroup 3TNT: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3TNT (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 48.54\n",
      "@> After alignment  RMSD: 1.72\n",
      "@> Checking AtomGroup 3TNS: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3TNS (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 34.77\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 3E91: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3E91 (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain B from 3E91 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 45.58\n",
      "@> After alignment  RMSD: 0.97\n",
      "@> Checking AtomGroup 3F9E: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3F9E (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 297 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 297 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 53.54\n",
      "@> After alignment  RMSD: 4.61\n",
      "@> Checking AtomGroup 2VJ1: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2VJ1 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2VJ1 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 300 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 43.48\n",
      "@> After alignment  RMSD: 1.15\n",
      "@> Checking AtomGroup 3M3V: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3M3V (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3M3V (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 303 residues match with 96% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 45.72\n",
      "@> After alignment  RMSD: 1.24\n",
      "@> Checking AtomGroup 3EA8: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3EA8 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 49.62\n",
      "@> After alignment  RMSD: 1.61\n",
      "@> Checking AtomGroup 3EA7: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3EA7 (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain B from 3EA7 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 45.71\n",
      "@> After alignment  RMSD: 0.88\n",
      "@> Checking AtomGroup 2PWX: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2PWX (len=289) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 289 residues match with 96% sequence identity and 95% overlap.\n",
      "@> Alignment is based on 289 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 37.63\n",
      "@> After alignment  RMSD: 4.55\n",
      "@> Checking AtomGroup 2QCY: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2QCY (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.96\n",
      "@> After alignment  RMSD: 4.65\n",
      "@> Checking AtomGroup 4HI3: 2 chains are identified\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4HI3 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 4HI3 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 36.99\n",
      "@> After alignment  RMSD: 0.75\n",
      "@> Checking AtomGroup 3M3T: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3M3T (len=307) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 35.57\n",
      "@> After alignment  RMSD: 4.75\n",
      "@> Checking AtomGroup 2ALV: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2ALV (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 61.96\n",
      "@> After alignment  RMSD: 1.21\n",
      "@> Checking AtomGroup 5C5N: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5C5N (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.85\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 5C5O: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5C5O (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 5C5O (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 300 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 38.70\n",
      "@> After alignment  RMSD: 0.68\n",
      "@> Checking AtomGroup 3AW0: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3AW0 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3AW0 (len=4) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.68\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 3AW1: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3AW1 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3AW1 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 22.84\n",
      "@> After alignment  RMSD: 0.56\n",
      "@> Checking AtomGroup 3AVZ: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3AVZ (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3AVZ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 300 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.55\n",
      "@> After alignment  RMSD: 0.69\n",
      "@> Checking AtomGroup 4TWY: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4TWY (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.90\n",
      "@> After alignment  RMSD: 1.68\n",
      "@> Checking AtomGroup 4TWW: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4TWW (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 4TWW (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 38.80\n",
      "@> After alignment  RMSD: 1.69\n",
      "@> Checking AtomGroup 3ATW: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3ATW (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3ATW (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 3ATW (len=2) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain D from 3ATW (len=2) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Alignment is based on 300 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 38.60\n",
      "@> After alignment  RMSD: 0.60\n",
      "@> Checking AtomGroup 3F9H: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3F9H (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 3F9H (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 300 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 300 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 41.79\n",
      "@> After alignment  RMSD: 1.38\n",
      "@> Checking AtomGroup 2QC2: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2QC2 (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain B from 2QC2 (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 43.52\n",
      "@> After alignment  RMSD: 1.22\n",
      "@> Checking AtomGroup 3F9F: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3F9F (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 3F9F (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 299 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 29.31\n",
      "@> After alignment  RMSD: 1.21\n",
      "@> Checking AtomGroup 4MDS: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4MDS (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 303 residues match with 96% sequence identity and 100% overlap.\n",
      "@> Alignment is based on 303 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 37.79\n",
      "@> After alignment  RMSD: 1.09\n",
      "@> Checking AtomGroup 4WY3: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4WY3 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 51.79\n",
      "@> After alignment  RMSD: 1.66\n",
      "@> Checking AtomGroup 3M3S: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3M3S (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3M3S (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 45.43\n",
      "@> After alignment  RMSD: 1.01\n",
      "@> Checking AtomGroup 5B6O: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5B6O (len=310) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 5B6O (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 37.99\n",
      "@> After alignment  RMSD: 1.98\n",
      "@> Checking AtomGroup 1Z1J: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1Z1J (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 1Z1J (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 30.94\n",
      "@> After alignment  RMSD: 1.99\n",
      "@> Checking AtomGroup 2Q6G: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Q6G (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 304 residues match with 96% sequence identity and 100% overlap.\n",
      "@>   Comparing Chain B from 2Q6G (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain C from 2Q6G (len=11) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=4%).\n",
      "@>   Comparing Chain D from 2Q6G (len=8) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=3%).\n",
      "@> Alignment is based on 304 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 98.11\n",
      "@> After alignment  RMSD: 1.07\n",
      "@> Checking AtomGroup 3EA9: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3EA9 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 45.63\n",
      "@> After alignment  RMSD: 0.71\n",
      "@> Checking AtomGroup 3EAJ: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3EAJ (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 3EAJ (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 302 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 302 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 49.54\n",
      "@> After alignment  RMSD: 0.85\n",
      "@> Checking AtomGroup 2OP9: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2OP9 (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain B from 2OP9 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 299 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 75.47\n",
      "@> After alignment  RMSD: 1.11\n",
      "@> Checking AtomGroup 3SNA: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3SNA (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@>   Comparing Chain H from 3SNA (len=4) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=1%).\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 24.14\n",
      "@> After alignment  RMSD: 0.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> Checking AtomGroup 2QIQ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2QIQ (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 301 residues match with 96% sequence identity and 99% overlap.\n",
      "@> Alignment is based on 301 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 47.50\n",
      "@> After alignment  RMSD: 0.86\n",
      "@> Checking AtomGroup 3F9G: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3F9G (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 297 residues match with 96% sequence identity and 98% overlap.\n",
      "@>   Comparing Chain B from 3F9G (len=295) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 295 residues match with 96% sequence identity and 97% overlap.\n",
      "@> Alignment is based on 297 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 57.07\n",
      "@> After alignment  RMSD: 0.95\n",
      "@> Checking AtomGroup 3FZD: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3FZD (len=296) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 296 residues match with 96% sequence identity and 97% overlap.\n",
      "@> Alignment is based on 296 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 73.51\n",
      "@> After alignment  RMSD: 1.02\n",
      "@> Checking AtomGroup 3D62: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3D62 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tMatch: 299 residues match with 96% sequence identity and 98% overlap.\n",
      "@> Alignment is based on 299 atoms matching 'calpha'.\n",
      "@> Before alignment RMSD: 73.82\n",
      "@> After alignment  RMSD: 0.60\n",
      "@> Checking AtomGroup 5WKM: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5WKM (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5WKM (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 5WKK: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5WKK (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5WKK (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 5WKL: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5WKL (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5WKL (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 5WKJ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5WKJ (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5WKJ (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 5C3N: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5C3N (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=24%, overlap=99%).\n",
      "@>   Comparing Chain B from 5C3N (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=24%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5C3N (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=97%).\n",
      "@>  Comparing Chain B from 5C3N (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=97%).\n",
      "@> Checking AtomGroup 4RSP: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4RSP (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@>   Comparing Chain B from 4RSP (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4RSP (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain B from 4RSP (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 4YLU: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4YLU (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@>   Comparing Chain B from 4YLU (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain C from 4YLU (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain D from 4YLU (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4YLU (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain B from 4YLU (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain C from 4YLU (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@>  Comparing Chain D from 4YLU (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 4WMF: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4WMF (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=24%, overlap=100%).\n",
      "@>   Comparing Chain B from 4WMF (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@>   Comparing Chain C from 4WMF (len=289) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=24%, overlap=94%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4WMF (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain B from 4WMF (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain C from 4WMF (len=289) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=53%, overlap=93%).\n",
      "@> Checking AtomGroup 4WME: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4WME (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 4WME (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@>   Comparing Chain C from 4WME (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain D from 4WME (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4WME (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain B from 4WME (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain C from 4WME (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain D from 4WME (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 4WMD: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4WMD (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=24%, overlap=99%).\n",
      "@>   Comparing Chain B from 4WMD (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@>   Comparing Chain C from 4WMD (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=100%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4WMD (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=97%).\n",
      "@>  Comparing Chain B from 4WMD (len=304) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@>  Comparing Chain C from 4WMD (len=303) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=52%, overlap=98%).\n",
      "@> Checking AtomGroup 4YOJ: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4YOJ (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 4YOJ (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4YOJ (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain B from 4YOJ (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@> Checking AtomGroup 4YO9: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4YO9 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 4YO9 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=24%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4YO9 (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain B from 4YO9 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@> Checking AtomGroup 4YOI: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4YOI (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 4YOI (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4YOI (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain B from 4YOI (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@> Checking AtomGroup 4YOG: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4YOG (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 4YOG (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4YOG (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@>  Comparing Chain B from 4YOG (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@> Checking AtomGroup 2YNB: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2YNB (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 2YNB (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2YNB (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain B from 2YNB (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@> Checking AtomGroup 2YNA: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2YNA (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@>   Comparing Chain B from 2YNA (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2YNA (len=306) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain B from 2YNA (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@> Checking AtomGroup 6JIJ: 6 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain B from 6JIJ (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain C from 6JIJ (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain A from 6JIJ (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain D from 6JIJ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain E from 6JIJ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 6JIJ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain B from 6JIJ (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain C from 6JIJ (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain A from 6JIJ (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=98%).\n",
      "@>  Comparing Chain D from 6JIJ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain E from 6JIJ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain F from 6JIJ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 3D23: 8 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain B from 3D23 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=99%).\n",
      "@>   Comparing Chain A from 3D23 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=98%).\n",
      "@>   Comparing Chain C from 3D23 (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=98%).\n",
      "@>   Comparing Chain D from 3D23 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=98%).\n",
      "@>   Comparing Chain H from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain E from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain G from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain B from 3D23 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=50%, overlap=98%).\n",
      "@>  Comparing Chain A from 3D23 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=51%, overlap=97%).\n",
      "@>  Comparing Chain C from 3D23 (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=50%, overlap=98%).\n",
      "@>  Comparing Chain D from 3D23 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=50%, overlap=98%).\n",
      "@>  Comparing Chain H from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain F from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain E from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain G from 3D23 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 5HYO: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5HYO (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=99%).\n",
      "@>   Comparing Chain B from 5HYO (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5HYO (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain B from 5HYO (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@> Checking AtomGroup 5GWZ: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain B from 5GWZ (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=98%).\n",
      "@>   Comparing Chain A from 5GWZ (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=98%).\n",
      "@>   Comparing Chain D from 5GWZ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain E from 5GWZ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain B from 5GWZ (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain A from 5GWZ (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain D from 5GWZ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain E from 5GWZ (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 4XFQ: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4XFQ (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=98%).\n",
      "@>   Comparing Chain B from 4XFQ (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=97%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4XFQ (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 4XFQ (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=47%, overlap=97%).\n",
      "@> Checking AtomGroup 4ZUH: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4ZUH (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=98%).\n",
      "@>   Comparing Chain B from 4ZUH (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=97%).\n",
      "@>   Comparing Chain C from 4ZUH (len=8) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=25%, overlap=3%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4ZUH (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 4ZUH (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=97%).\n",
      "@>  Comparing Chain C from 4ZUH (len=8) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=88%, overlap=3%).\n",
      "@> Checking AtomGroup 5ZQG: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5ZQG (len=292) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=96%).\n",
      "@>   Comparing Chain B from 5ZQG (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=13%, overlap=97%).\n",
      "@>   Comparing Chain C from 5ZQG (len=7) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=2%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5ZQG (len=292) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=47%, overlap=96%).\n",
      "@>  Comparing Chain B from 5ZQG (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=97%).\n",
      "@>  Comparing Chain C from 5ZQG (len=7) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=86%, overlap=2%).\n",
      "@> Checking AtomGroup 4ZRO: 8 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain B from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain C from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain D from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain E from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain F from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain G from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain H from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain C from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain D from 4ZRO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain E from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain F from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain G from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain H from 4ZRO (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 5EU8: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5EU8 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain B from 5EU8 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5EU8 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 5EU8 (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 3TLO: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3TLO (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=99%).\n",
      "@>   Comparing Chain B from 3TLO (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 3TLO (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=99%).\n",
      "@>  Comparing Chain B from 3TLO (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=99%).\n",
      "@> Checking AtomGroup 5NH0: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5NH0 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=98%).\n",
      "@>   Comparing Chain B from 5NH0 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=99%).\n",
      "@>   Comparing Chain C from 5NH0 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5NH0 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@>  Comparing Chain B from 5NH0 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@>  Comparing Chain C from 5NH0 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@> Checking AtomGroup 6FV2: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6FV2 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=99%).\n",
      "@>   Comparing Chain B from 6FV2 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=98%).\n",
      "@>   Comparing Chain C from 6FV2 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 6FV2 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=99%).\n",
      "@>  Comparing Chain B from 6FV2 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@>  Comparing Chain C from 6FV2 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@> Checking AtomGroup 6FV1: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 6FV1 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=99%).\n",
      "@>   Comparing Chain B from 6FV1 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=98%).\n",
      "@>   Comparing Chain C from 6FV1 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 6FV1 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@>  Comparing Chain B from 6FV1 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@>  Comparing Chain C from 6FV1 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@> Checking AtomGroup 5GWY: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 5GWY (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=22%, overlap=98%).\n",
      "@>   Comparing Chain B from 5GWY (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=23%, overlap=99%).\n",
      "@>   Comparing Chain D from 5GWY (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain E from 5GWY (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 5GWY (len=298) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=98%).\n",
      "@>  Comparing Chain B from 5GWY (len=305) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=44%, overlap=98%).\n",
      "@>  Comparing Chain D from 5GWY (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain E from 5GWY (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@> Checking AtomGroup 1P9U: 8 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1P9U (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain B from 1P9U (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain C from 1P9U (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain D from 1P9U (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain E from 1P9U (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain F from 1P9U (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain G from 1P9U (len=6) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@>   Comparing Chain H from 1P9U (len=6) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=0%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 1P9U (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 1P9U (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain C from 1P9U (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain D from 1P9U (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain E from 1P9U (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain F from 1P9U (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain G from 1P9U (len=6) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=2%).\n",
      "@>  Comparing Chain H from 1P9U (len=6) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=2%).\n",
      "@> Checking AtomGroup 4F49: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 4F49 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain B from 4F49 (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain C from 4F49 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain D from 4F49 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 4F49 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 4F49 (len=297) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain C from 4F49 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain D from 4F49 (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@> Checking AtomGroup 2AMP: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2AMP (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain B from 2AMP (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2AMP (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 2AMP (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@> Checking AtomGroup 1LVO: 6 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1LVO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain B from 1LVO (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain C from 1LVO (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain D from 1LVO (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain E from 1LVO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@>   Comparing Chain F from 1LVO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 1LVO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain B from 1LVO (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain C from 1LVO (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain D from 1LVO (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=99%).\n",
      "@>  Comparing Chain E from 1LVO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@>  Comparing Chain F from 1LVO (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=46%, overlap=98%).\n",
      "@> Checking AtomGroup 1P9S: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 1P9S (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain B from 1P9S (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 1P9S (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> \tFailed to match chains (seqid=43%, overlap=97%).\n",
      "@>  Comparing Chain B from 1P9S (len=299) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=97%).\n",
      "@> Checking AtomGroup 2ZU2: 2 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2ZU2 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@>   Comparing Chain B from 2ZU2 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=14%, overlap=99%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2ZU2 (len=300) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=97%).\n",
      "@>  Comparing Chain B from 2ZU2 (len=301) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=97%).\n",
      "@> Checking AtomGroup 2LIZ: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2LIZ (len=120) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=95%, overlap=39%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2LIZ (len=120) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=95%, overlap=39%).\n",
      "@> Checking AtomGroup 2K7X: 1 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2K7X (len=120) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=95%, overlap=39%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2K7X (len=120) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=95%, overlap=39%).\n",
      "@> Checking AtomGroup 3EBN: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 3EBN (len=100) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@>   Comparing Chain B from 3EBN (len=100) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@>   Comparing Chain C from 3EBN (len=101) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@>   Comparing Chain D from 3EBN (len=101) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 3EBN (len=100) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@>  Comparing Chain B from 3EBN (len=100) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@>  Comparing Chain C from 3EBN (len=101) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@>  Comparing Chain D from 3EBN (len=101) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=96%, overlap=33%).\n",
      "@> Checking AtomGroup 2Q6D: 3 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Q6D (len=296) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=12%, overlap=97%).\n",
      "@>   Comparing Chain B from 2Q6D (len=307) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=12%, overlap=99%).\n",
      "@>   Comparing Chain C from 2Q6D (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=11%, overlap=98%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2Q6D (len=296) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=44%, overlap=96%).\n",
      "@>  Comparing Chain B from 2Q6D (len=307) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=44%, overlap=96%).\n",
      "@>  Comparing Chain C from 2Q6D (len=302) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=43%, overlap=95%).\n",
      "@> Checking AtomGroup 2Q6F: 4 chains are identified\n",
      "@> Checking AtomGroup Mpro-x0995_1 Selection 'heavy' Selection 'protein': 1 chains are identified\n",
      "@> Trying to match chains based on residue numbers and names:\n",
      "@>   Comparing Chain A from 2Q6F (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=12%, overlap=97%).\n",
      "@>   Comparing Chain B from 2Q6F (len=295) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=12%, overlap=97%).\n",
      "@>   Comparing Chain D from 2Q6F (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@>   Comparing Chain C from 2Q6F (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=0%, overlap=1%).\n",
      "@> Trying to match chains based on global sequence alignment:\n",
      "@>  Comparing Chain A from 2Q6F (len=294) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=95%).\n",
      "@>  Comparing Chain B from 2Q6F (len=295) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=45%, overlap=96%).\n",
      "@>  Comparing Chain D from 2Q6F (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n",
      "@>  Comparing Chain C from 2Q6F (len=3) and Chain A from Mpro-x0995_1 Selection 'heavy' Selection 'protein' (len=304):\n",
      "@> \tFailed to match chains (seqid=100%, overlap=1%).\n"
     ]
    }
   ],
   "source": [
    "# create local directory for aligned structures\n",
    "struct_dir = './PDB_Mpro_structures_aligned'\n",
    "if not os.path.exists(struct_dir): os.mkdir(struct_dir)\n",
    "\n",
    "# perform alignment and generate new files\n",
    "ref = non_cov_prot_structs[0]\n",
    "for struct in PDB_Mpro_structs:\n",
    "    alignment = prody.matchAlign(struct, ref)\n",
    "    if alignment:\n",
    "        aligned_struct = alignment[0]\n",
    "        outfile = f'{struct_dir}/{struct.getTitle()}_aligned.pdb'\n",
    "        prody.writePDB(outfile, aligned_struct)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "file_list = glob.glob(\"./PDB_Mpro_structures_aligned/*.pdb\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6bf06097e0fa443da1f9bf624ab0814e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "interactive(children=(Dropdown(description='pdb_file', options=('./PDB_Mpro_structures_aligned/5RFY_aligned.pd…"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "@interact(pdb_file = file_list)\n",
    "def show_pdb(pdb_file):\n",
    "    p = py3Dmol.view(500, 500)\n",
    "    buff = open(pdb_file).read()\n",
    "    p.addModel(buff, 'pdb')\n",
    "    p.setStyle({'cartoon': {'color':'spectrum'}})\n",
    "    p.setBackgroundColor('0xeeeeee')\n",
    "    p.zoomTo()\n",
    "    return p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  },
  "pycharm": {
   "stem_cell": {
    "cell_type": "raw",
    "metadata": {
     "collapsed": false
    },
    "source": []
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
